<?xml version="1.0"?>
<html>
  <body>
    <div class="article fulltext-view" itemprop="articleBody"><span class="highwire-journal-article-marker-start"/><h1 id="article-title-1" itemprop="headline">Changes in gene expression in the <em>leafy cotyledon1</em> (<em>lec1</em>) and <em>fusca3</em> (<em>fus3</em>) mutants of <em>Arabidopsis thaliana</em> L.
               </h1>
               <div class="contributors">
                  <ol class="contributor-list" id="contrib-group-1"><li class="contributor" id="contrib-1" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="name" itemprop="name"><a class="name-search" href="/search?author1=Carlos+M.+Vicient&amp;sortspec=date&amp;submit=Submit">Carlos M. Vicient</a></span><a id="xref-fn-1-1" class="xref-fn" href="#fn-1">
                           <sup>1</sup></a>, 
                     </li>
                     <li class="contributor" id="contrib-2" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="name" itemprop="name"><a class="name-search" href="/search?author1=Natacha+Bies%E2%80%90Etheve&amp;sortspec=date&amp;submit=Submit">Natacha Bies‐Etheve</a></span><a id="xref-fn-2-1" class="xref-fn" href="#fn-2">
                           <sup>2</sup></a> and 
                     </li>
                     <li class="last" id="contrib-3"><span class="name"><a class="name-search" href="/search?author1=Michel+Delseny&amp;sortspec=date&amp;submit=Submit">Michel Delseny</a></span></li>
                  </ol><ol class="affiliation-list"><li class="aff"><a id="aff-1" name="aff-1"/><address>Laboratoire de Physiologie et Biologie Moléculaire des Plantes, Centre National de la Recherche Scientifique UMR 5545, Université
                           de Perpignan, Avenue de Villeneuve, 66860 Perpignan Cedex, France
                        </address>
                     </li>
                  </ol><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="1999-05-27"><span class="received-label">Received </span>May 27, 1999.
                     </li>
                     <li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2000-01-21"><span class="accepted-label">Accepted </span>January 21, 2000.
                     </li>
                  </ul></div>
               <div class="section abstract" id="abstract-1" itemprop="description">
                  <div class="section-nav">
                     <div class="nav-placeholder"> </div><a href="#sec-1" title="Introduction" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>Abstract</h2>
                  
                  <p id="@hwp:id"><em>Arabidopsis thaliana</em> L. <em>leafy cotyledon1</em> (<em>lec1</em>) and <em>fusca3</em> (<em>fus3</em>) mutants show multiple phenotypic defects during seed development. In this report the effects of these mutations are examined
                     at the molecular level. The patterns of protein accumulation in <em>lec1</em> and <em>fus3</em> seeds are severly altered. In <em>lec1</em> seeds the steady‐state mRNA levels of several late embryogenesis genes were reduced. Different patterns of expression were
                     observed, indicating the occurrence of several regulatory pathways. The effect of <em>lec1</em> mutations on the expression of the late‐embryogenesis abundant <em>AtEm1</em> gene was examined in detail. In <em>lec1‐1</em> seeds, the <em>AtEm1</em> gene was expressed at a higher level than in the wild type and earlier in development. The activity of an <em>AtEm1</em> promoter/<em>β</em>‐<em>glucuronidase</em> reporter gene construct in transgenic <em>A</em>.<em>thaliana</em> plants was studied. Changes in promoter activity in <em>lec1‐1</em> with respect to wild‐type seeds were correlated with changes in corresponding mRNA steady‐state levels. <em>fus3‐2</em> mutation produced similar changes in <em>AtEm1</em> promoter activity as <em>lec1‐1</em>, which is consistent with the hypothesis that LEC1 and FUS3 might act in the same regulatory pathway. Transgenic analysis
                     using 5′–promoter deletions demonstrated that at least two regions of <em>AtEm1</em> gene promoter interact with the LEC1‐dependent transcriptional regulatory pathway. In spite of expression of the <em>AtEm1</em> promoter and accumulation of <em>AtEm1</em> mRNA, the corresponding Em1 protein does not accumulate in <em>lec1‐1</em> seeds. The ABA inducibility of the <em>AtEm1</em> promoter was not affected by the <em>lec1</em> mutation.
                  </p>
                  
               </div>
               <h3 class="kwd-header">Key words</h3>
               <ul class="kwd-group KWD"><li class="kwd"><span><a class="kwd-search" href="/search?fulltext=%0A++++++++++abi3%0A++++++++&amp;sortspec=date&amp;submit=Submit&amp;andorexactfulltext=phrase">
                           <em>abi3</em>
                           </a></span></li>
                  <li class="kwd"><span><a class="kwd-search" href="/search?fulltext=Em+genes&amp;sortspec=date&amp;submit=Submit&amp;andorexactfulltext=phrase"><em>Em</em> genes</a></span></li>
                  <li class="kwd"><span><a class="kwd-search" href="/search?fulltext=gene+expression&amp;sortspec=date&amp;submit=Submit&amp;andorexactfulltext=phrase">gene expression</a></span></li>
                  <li class="kwd"><span><a class="kwd-search" href="/search?fulltext=late+embryogenesis&amp;sortspec=date&amp;submit=Submit&amp;andorexactfulltext=phrase">late embryogenesis</a></span></li>
                  <li class="kwd"><span><a class="kwd-search" href="/search?fulltext=seed+development.&amp;sortspec=date&amp;submit=Submit&amp;andorexactfulltext=phrase">seed development.</a></span></li>
               </ul><div class="section" id="sec-1">
                  <div class="section-nav"><a href="#abstract-1" title="Abstract" class="prev-section-link"><span>Previous Section</span></a><a href="#sec-2" title="Materials and methods" class="next-section-link"><span>Next Section</span></a></div>
                  
                  <h2>Introduction</h2>
                  
                  <p id="@hwp:id">Mutational approaches have begun to yield insights regarding the molecular mechanisms controlling seed development. ABA‐insensitive
                     mutations have demonstrated the role of this hormone in seed maturation (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Leung and Giraudat, 1998</a>). However, other factors are clearly involved (de Vries <em>et al</em>., 1998; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Harada <em>et al</em>., 1998</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wobus and Weber, 1999</a>). One of the most severe alterations of seed maturation in higher plants has been observed in the <em>leafy cotyledon</em> (<em>lec</em>) mutants (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Meinke, 1992</a>). Three different mutants with a phenotype similar to the original <em>lec1</em> mutant have been reported, <em>lec1</em>, <em>lec2</em> and <em>fus3</em> (Baumlein <em>et al</em>., 1994; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Keith <em>et al</em>., 1994</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lotan <em>et al</em>., 1998</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Luerβen <em>et al</em>., 1998</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Meinke, 1992</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Meinke <em>et al</em>., 1994</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">West <em>et al</em>., 1994</a>). <em>Leafy cotyledon1</em> (<em>lec1</em>) mutant embryos present strong morphological alterations after the torpedo stage. The <em>lec1</em> embryos show reduced and almost transparent hypocotyls with highly vacuolated cells, rounded cotyledons that remain green
                     unusually late in development and an abnormal accumulation of anthocyanin is often observed prior to desiccation (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Meinke, 1992</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Meinke <em>et al</em>., 1994</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Parcy <em>et al</em>., 1997</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">West <em>et al</em>., 1994</a>). The cotyledons of <em>lec1</em> embryos have a vascular system intermediate between normal cotyledons and leaves, having mature xylem elements and stomata
                     that, in wild‐type embryos, appear only after germination. The <em>lec1</em> seeds are desiccation intolerant and occasionally viviparous at maturity. The <em>lec1</em> embryos can germinate and give rise to morphologically normal plants only if they are harvested before seed desiccation.
                     The plants produced are viable and phenotypically normal except for the presence of leafy cotyledons with trichomes on the
                     adaxial surface and the production of 100% defective seeds. The mutant embryos contain less protein and lipid bodies than
                     in the wild‐type embryos (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Meinke <em>et al</em>., 1994</a>). <em>lec1</em> embryos present defects in suspensor morphology during early embryogenesis (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Lotan <em>et al</em>., 1998</a>). Two alleles of the <em>lec1</em> mutant have been identified, <em>lec1‐1</em> and <em>lec1‐2</em>. Both alleles show very similar phenotypes, general morphology, internal anatomy, and response in culture. The <em>lec1‐1</em> mutation is the result of a deletion overlapping the <em>lec1</em> gene and <em>lec1‐2</em> was produced by the insertion of a T‐DNA 115 bp upstream from the <em>lec1</em> coding region (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Lotan <em>et al</em>., 1998</a>). It is likely that <em>lec1‐1</em> could correspond to a null allele whereas it is conceivable that <em>lec1‐2</em> might be leaky. The <em>lec1</em> gene has been cloned and the deduced protein shows sequence similarity with the HAP3 subunit of the CCAAT box‐binding factor
                     (CBF) (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Lotan <em>et al</em>., 1998</a>). <em>lec1</em> is a member of a small gene family. <em>lec1</em> mRNA accumulation is restricted to seed development from preglobular to bent cotyledon stage and decays during seed maturation.
                  </p>
                  
                  <p id="@hwp:id">Another mutant, <em>fusca3</em> (<em> fus3</em>), has a similar phenotype to <em>lec1‐1</em> and <em>lec1‐2</em>, but the cotyledons are not so heavily altered in shape (Baumlein <em>et al</em>., 1994; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Keith <em>et al</em>., 1994</a>; <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Meinke <em>et al</em>., 1994</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Miséra <em>et al</em>., 1994</a>). <em>fus3</em> seeds are paler than <em>lec1</em> and the pattern of anthocyanin accumulation in the embryo is different. The <em>fus3</em> embryos accumulate anthocyanin mainly in the junction between cotyledons and the hypocotyl whereas <em>lec1</em> embryos accumulate anthocyanin mainly in the cotyledon margins. Moreover, trichomes in <em>fus3</em> cotyledons are not as prominent as in <em>lec1</em>. Three <em>fusca3</em> alleles have been isolated. <em>fus3‐1</em> and <em>fus3‐2</em> were identified by the accumulation of a purple pigment in late embryogenesis (Baumlein <em>et al</em>., 1994; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Miséra <em>et al</em>., 1994</a>), and a third allele, <em>fus3‐3</em>, was identified by screening of immature mutagenized seeds for their ability to germinate during the green cotyledon stage
                     (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Keith <em>et al</em>., 1994</a>). No significant differences between the alleles have been reported. The <em>fus3</em> gene is a member of the ABI3/VP1 B3 domain protein family (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Luerβen <em>et al</em>., 1998</a>). A third mutation is <em>leafy cotyledon2</em> (<em>lec2</em>) (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Meinke, 1992</a>). Alterations in the development of the <em>lec2</em> embryos are not as severe as in <em>lec1</em> and <em>fus3</em>. A single allele of <em>lec2</em> has been reported (<a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">Meinke <em>et al</em>., 1994</a>) and at present, the sequence and function of this gene are not known.
                  </p>
                  
                  <p id="@hwp:id">The complexity of <em>lec1</em> and <em>fus3</em> phenotypes indicates that LEC1 and FUS3 proteins may be required for the activation of a wide range of embryo specific genetic
                     programmes. The steady‐state mRNA levels of some storage protein and oleosin genes and their promoter activities are reduced
                     in <em>lec1</em> and <em>fus3</em> embryos (Baumlein <em>et al</em>., 1994; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kirik <em>et al</em>., 1996</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Parcy <em>et al</em>., 1997</a>), but the promoter activity of some late embryogenesis genes is not severely affected in <em>fus3</em> (Baumlein <em>et al</em>., 1994) and <em>lec1‐2</em> seeds (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">West <em>et al</em>., 1994</a>). In <em>lec1‐2</em> embryos some genes prevalently expressed during post‐germinative growth are active simultaneously with late‐embryogenesis
                     genes (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">West <em>et al</em>., 1994</a>) and the accumulation of the MADS domain protein AGL15 is reduced in <em>lec1‐2</em> but not in <em>lec1‐1</em> embryos (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Perry <em>et al</em>., 1996</a>). The expression of MYB genes have been found to be altered in <em>fus3</em> embryos (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kirik <em>et al</em>., 1998<em>a</em></a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21"><em>b</em></a>).
                  </p>
                  
                  <p id="@hwp:id">Further molecular analysis in the <em>leafy cotyledon</em> mutants should help to elucidate the regulatory networks that function during embryogenesis and seed maturation in <em>A</em>.<em>thaliana</em>. As shown for the <em>abi3</em> mutant (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Parcy <em>et al</em>., 1994</a>), monitoring a set of molecular markers should help to clarify the role of these regulatory loci in the control of seed development.
                     This consideration prompted the authors to analyse the effect of the <em>lec1‐1</em> and <em>lec1‐2</em> mutations on the expression of several genes previously used as genetic markers of seed development.
                  </p>
                  
               </div>
               <div class="section" id="sec-2">
                  <div class="section-nav"><a href="#sec-1" title="Introduction" class="prev-section-link"><span>Previous Section</span></a><a href="#sec-9" title="Results" class="next-section-link"><span>Next Section</span></a></div>
                  
                  <h2>Materials and methods</h2>
                  
                  
                  <h3>Plant material and growth conditions</h3>
                  
                  <p id="@hwp:id">The <em>lec1‐1</em> seeds were obtained from Dr David Meinke (Oklahoma State University), <em>lec1‐2</em> from Dr John Harada (University of California, Davis) and <em>fus3‐2</em> from Dr Simon Miséra (IPK, Gatersleben). Transgenic seeds containing an <em>AtEm1</em> promoter/<em>β‐glucuronidase</em> reporter gene chimeric construct (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hull <em>et al</em>., 1996</a>) were kindly provided by Gillian Hull (University of Perpignan), and plants containing <em>ABI3</em> promoter/<em>β‐glucuronidase</em> reporter gene construct by Dr Jérôme Giraudat (ISV, Gif sur Yvette). Plants were grown in a greenhouse (22 °C, 16 h day).
                     Homozygous <em>lec1‐1</em> mutant plants were obtained by plating surface‐sterilized immature embryos on MS medium (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Murashige and Skoog, 1962</a>) and transferring to soil after 15 d of <em>in vitro</em> culture. All transgenic plants were grown according to the French safety guidelines and the French Genetic Engineering Commission
                     approved these experiments.
                  </p>
                  
                  
                  
                  <h3>Total RNA isolation and Northern hybridization</h3>
                  
                  <p id="@hwp:id">RNA extractions were performed from frozen material as previously described (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Vicient and Delseny, 1999</a>). 2 μg of total RNA were fractionated on 1.5% agarose/2.2 M formaldehyde gels as described previously (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Farrell, 1993</a>) and stained with ethidium bromide to ensure equal loading. RNA was transferred to Hybond N<sup>+</sup> membranes (Amersham) by capillarity (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Sambrook <em>et al</em>., 1989</a>) and UV‐fixed using a Stratagene Stratalinker. Membrane prehybridization was performed using 0.25 M NaHPO<sub>4</sub> (pH 7.2), 7% SDS (p/v) and 1 mM EDTA to which 100 mg l<sup>−1</sup> of previously denatured calf thymus DNA were added. After 20 min prehybridization at 65 °C, the DNA probe was added to the
                     prehybridization buffer. Radioactive probes were generated using the Megaprime labelling kit (Amersham). Following overnight
                     incubation, membranes were washed twice for 20 min at 65 °C with 20 mM NaHPO<sub>4</sub> (pH 7.2), 1% SDS (p/v) and 1 mM EDTA. A last wash was performed for 10 min in the same buffer containing only 0.1% SDS and
                     at the same temperature. Signal detection was carried out using Kodak X‐ray film and Hyperscreen intensifying screens (Amersham).
                     The filters were finally hybridized with a radish 18S rRNA probe (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Grellet <em>et al</em>., 1989</a>) as a control for RNA loading.
                  </p>
                  
                  
                  
                  <h3>Protein extraction and analysis</h3>
                  
                  <p id="@hwp:id">Total protein was extracted from <em>A</em>. <em>thaliana</em> dry seeds using the following extraction buffer: 10 mM NaCl, 10 mM TRIS‐HCl pH 8.3 1 mM TLCK (tosyl‐<span class="sc">l</span>‐lysine‐chloromethylcetone from Sigma) as protease inhibitor. After centrifugation (20 min, 13 000 rpm), protein content was
                     determined using the Bradford method (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bradford, 1976</a>). Electrophoretic analysis, protein blotting and immunological detection were conducted as described previously (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bies <em>et al</em>., 1998</a>). Proteins were visualized by staining with Coomassie Brilliant Blue R250.
                  </p>
                  
                  
                  
                  <h3>Generation of crosses</h3>
                  
                  <p id="@hwp:id">For crosses, heterozygous mutant plants were used as female parents and homozygous transgenic plants carrying the promoter/<em>β</em>‐<em>glucuronidase</em> fusion as male parents. Closed flowers were stripped of sepals, petals and anthers just prior to stigma maturity. Two days
                     later the stigma was brushed with anthers of the male parental plants. Crosses were checked by the absence of mutant seeds
                     in the F<sub>1</sub> population. 75% of the F<sub>2</sub> population inherit the promoter/<em>β</em>‐<em>glucuronidase</em> fusion and 25% exhibit the mutant phenotype. Wild‐type and mutant seeds of the F<sub>2</sub> population of at least three transgenic crosses were assayed for GUS expression. Dry seeds of crosses were collected from
                     the same pods to minimize developmental and physiological differences, mutant seeds were separated from wild‐type seeds under
                     a binocular microscope and frozen in Eppendorf tubes. For the analysis of GUS activity during seed development siliques were
                     harvested from the same inflorescence. Seeds of consecutive siliques were pooled and mutant seeds were separated from wild‐type
                     seeds under a binocular microscope, transferred to Eppendorf tubes and frozen.
                  </p>
                  
                  
                  
                  <h3>Measurements of GUS activity</h3>
                  
                  <p id="@hwp:id">Fluorometric GUS assays were performed as previously described (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Devic <em>et al</em>., 1995</a>). GUS activity was standardized with respect to protein content in the extract determined by the Bradford method (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bradford, 1976</a>).
                  </p>
                  
                  
                  
                  <h3>ABA induction</h3>
                  
                  <p id="@hwp:id">Homozygous <em>lec1‐1</em> and wild‐type immature transgenic seeds were harvested at 12 d after pollination and incubated on solid MS medium (Sigma)
                     supplemented with 50 μM of ABA (mixed isomers, Sigma). Control seeds were incubated in parallel on MS medium without ABA.
                     Wild‐type and mutant immature seeds were extracted from the same siliques to avoid developmental or environmental differences.
                     Seeds were incubated during 24 h. GUS activity determinations were done in duplicate using 20 seeds per treatment obtained
                     from 4 independent crosses.
                  </p>
                  
                  
               </div>
               <div class="section" id="sec-9">
                  <div class="section-nav"><a href="#sec-2" title="Materials and methods" class="prev-section-link"><span>Previous Section</span></a><a href="#sec-17" title="Next Section" class="next-section-link"><span>Next Section</span></a></div>
                  
                  <h2>Results</h2>
                  
                  
                  <h3>Effects of the lec1 mutations on gene expression in mature seeds</h3>
                  
                  <p id="@hwp:id">The levels of accumulation of different mRNAs in dry seeds of <em>lec1‐1</em> and <em>lec1‐2</em> mutants were determined by RNA blot analysis (Fig. 1). The <em>lec1‐1</em> seeds were kindly provided by Dr David Meinke (Oklahoma State University) and <em>lec1‐2</em> by Dr John Harada (University of California, Davis). The parental ecotype of both mutant alleles, Wassilewskija, was included
                     as a control. Nine cDNA probes were used corresponding to genes expressed during late embryogenesis. Each probe detected a
                     single band on RNA gel blot analysis. <em>AtEm1</em> and <em>AtEm6</em> probes correspond to well‐characterized <em>A</em>. <em>thaliana</em> class I <em>lea</em> (Late Embryogenesis Abundant) genes (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Finkelstein, 1993</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gaubier <em>et al</em>., 1993</a>), <em>M10</em> and <em>M17</em> correspond to a new type of <em>lea</em> genes (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Raynal <em>et al</em>., 1999</a>) and the other probes correspond to clones obtained from the systematic analysis of <em>A</em>. <em>thaliana</em> dry seed cDNA libraries (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cooke <em>et al</em>., 1996</a>) with the following protein sequence similarities: <em>PAP260</em> with the tomato <em>lea25</em> gene (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cohen and Bray, 1992</a>); <em>PAP51</em> with the cotton <em>lea</em> gene <em>d113</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Baker <em>et al</em>., 1988</a>); <em>PAP240</em> with the rapeseed <em>Lea76</em> gene (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Harada <em>et al</em>., 1989</a>); <em>PAP85</em> with a broad bean vicilin (Bassuner <em>et al</em>., 1987); and <em>PAP137</em> with a protease inhibitor protein (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ogata <em>et al</em>., 1991</a>). The accession numbers are given in the legend to figure 1. <em>PAP260</em> and <em>PAP51</em> belong to the same multigene family but do not cross hybridize. Filters were also hybridized as a control with a radish 18S
                     rDNA probe (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Grellet <em>et al</em>., 1989</a>). The accumulation of <em>PAP260</em>, <em>AtEm6</em>, <em>M10</em>, <em>M17</em>, and <em>PAP137</em> mRNAs was severely reduced in both mutants alleles compared to wild‐type. <em>PAP240</em> and <em>PAP85</em> mRNAs abundance was also reduced although less severely. A different situation was found with <em>PAP51</em> and <em>AtEm1</em> probes. <em>PAP51</em> mRNA accumulation was reduced in the two mutant alleles, but the reduction was stronger in <em>lec1‐2</em> than in <em>lec1‐1</em>. <em>AtEm1</em> mRNA level was increased in <em>lec1‐1</em> compared with the control, but in <em>lec1‐2</em> seeds the level was similar to the control. Similar results were obtained in three repetitions of this experiment using RNAs
                     obtained in independent extractions.
                  </p>
                  
                  <p id="@hwp:id">
                     
                     
                  </p>
                  <div id="F1" class="fig pos-float odd">
                     <div class="fig-inline"><a href="995/F1.expansion.html"><img alt="  Fig.&#x2009;1. " src="995/F1.small.gif"/></a><div class="callout"><span>View larger version:</span><ul class="callout-links"><li><a href="995/F1.expansion.html">In this window</a></li>
                              <li><a class="in-nw" href="995/F1.expansion.html">In a new window</a></li>
                           </ul><ul class="fig-services"><li class="ppt-link"><a href="/powerpoint/51/347/995/F1">Download as PowerPoint Slide</a></li>
                           </ul></div>
                     </div>
                     <div class="fig-caption"><span class="fig-label">
                           <strong>Fig. 1.</strong>
                           </span> 
                        
                        <p id="p-14" class="first-child">RNA blot analysis of the steady‐state mRNA level of different LEA genes in dry seeds of <em>leafy cotyledon1</em> mutants. Probes correspond to: <em>AtEm1</em> and <em>AtEm6</em> (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Finkelstein, 1993</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Gaubier <em>et al</em>., 1993</a>); <em>M10</em> and <em>M17</em> to late‐embryogenesis‐abundant genes (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Raynal <em>et al</em>., 1999</a>); and the others to clones obtained from the systematic analysis of <em>A</em>. <em>thaliana</em> dry seed cDNA libraries (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cooke <em>et al</em>., 1996</a>): <em>PAP260</em> (EMBL Z27059); <em>PAP51</em> (Z27246); <em>PAP240</em> (Z27036); <em>PAP85</em> (Z27025); and <em>PAP137</em> (Z27036). 18S corresponds to a radish 18S rRNA probe (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Grellet <em>et al</em>., 1989</a>).
                        </p>
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                  </div>
                  
                  
                  
                  <h3>Effects of the lec1 and fus3 mutations on protein accumulation in mature seeds</h3>
                  
                  <p id="@hwp:id">The patterns of protein accumulation in wild‐type, <em>lec1‐1</em>, <em>lec1‐2</em>, and <em>fus3‐2</em> seeds were compared. Total proteins were obtained from 50 <em>lec1‐1</em>, <em>lec1‐2</em> and <em>fus3‐2</em> seeds. As controls, total proteins from 50 seeds of the corresponding wild‐type ecotypes were also extracted (Wassilewskija
                     for <em>lec1‐1</em> and <em>lec1‐2</em>, and Dijon for <em>fus3‐2</em>). For each sample, total proteins were extracted in the same volume of extraction buffer and equal volumes of the soluble
                     fraction of each extraction were electrophoresed in a SDS‐polyacrylamide gel and stained. The patterns of protein accumulation
                     obtained in the mutant seeds were different from those observed in the wild‐type, but roughly similar between the three mutants
                     (Fig. 2). The intensity of many of the bands present in the wild‐type seed extracts was substantially lower in the mutant
                     seed extracts particularly those corresponding to seed storage protein. On the other hand, some bands not detectable or detectable
                     at low intensity in the wild‐type extracts appeared clearly in the mutant ones. The changes in the patterns of protein accumulation
                     were similar to those previously observed for <em>fus3</em> seeds (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Kirik <em>et al</em>., 1996</a>; <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Keith <em>et al</em>., 1994</a>).
                  </p>
                  
                  <p id="@hwp:id">
                     
                     
                  </p>
                  <div id="F2" class="fig pos-float odd">
                     <div class="fig-inline"><a href="995/F2.expansion.html"><img alt="  Fig.&#x2009;2. " src="995/F2.small.gif"/></a><div class="callout"><span>View larger version:</span><ul class="callout-links"><li><a href="995/F2.expansion.html">In this window</a></li>
                              <li><a class="in-nw" href="995/F2.expansion.html">In a new window</a></li>
                           </ul><ul class="fig-services"><li class="ppt-link"><a href="/powerpoint/51/347/995/F2">Download as PowerPoint Slide</a></li>
                           </ul></div>
                     </div>
                     <div class="fig-caption"><span class="fig-label">
                           <strong>Fig. 2.</strong>
                           </span> 
                        
                        <p id="p-17" class="first-child">Protein accumulation in wild‐type and mutant seeds. Each lane of the SDS‐polyacrylamide gel was loaded with equal volumes
                           of total protein extracts from 50 seeds. Ws, wild‐type (Wassilewskija ecotype); L1, <em>lec1‐1</em>; L2, <em>lec1‐2</em>; F3, <em>fus3‐2</em>; Dj, wild‐type (Dijon ecotype). Molecular weight markers are indicated on the left in kilodaltons (kDa). The position of
                           storage proteins are indicated on the right.
                        </p>
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                  </div>
                  
                  
                  
                  <h3>Effect of the lec1‐1 mutation on AtEm1 and AtEm6 gene expression in developing seeds</h3>
                  
                  <p id="@hwp:id">Expression of <em>AtEm1</em> and <em>AtEm6</em>, two class I <em>lea</em> genes, was further investigated by examining accumulation kinetics of corresponding mRNAs during seed development in wild‐type
                     and <em>lec1‐1</em> siliques (Fig. 3). Wild‐type and homozygous <em>lec1‐1</em> mutant plants were grown in parallel. Total RNA was extracted from siliques at different stages of development and used in
                     RNA blot analyses. The accumulation of <em>AtEm1</em> mRNA in wild‐type plants was detected, in these experimental conditions, as early as 12 d after pollination (dap), with the
                     greatest accumulation at about 17 dap as previously reported (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Parcy <em>et al</em>., 1994</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bies <em>et al</em>., 1998</a>). In contrast, in the <em>lec1‐1</em> mutant seeds, <em>AtEm1</em> mRNA was already detected at 3 dap, much earlier than in wild‐type seeds. At 7 dap the level of mRNA accumulation was similar
                     to those observed in mature seeds. To control equal loading, the ribosomal RNA in the gel was stained with ethidium bromide
                     and visualized under UV light. The same RNA blots were used to analyse the expression of the <em>AtEm6</em> gene. In wild‐type plants, <em>AtEm6</em> mRNA starts to accumulate at 17 dap, with a maximum level in mature seeds, as expected. In the <em>lec1‐1</em> mutant the time‐course of <em>AtEm6</em> mRNA accumulation is similar to wild‐type, but the steady‐state level is strongly reduced.
                  </p>
                  
                  <p id="@hwp:id">
                     
                     
                  </p>
                  <div id="F3" class="fig pos-float odd">
                     <div class="fig-inline"><a href="995/F3.expansion.html"><img alt="  Fig.&#x2009;3. " src="995/F3.small.gif"/></a><div class="callout"><span>View larger version:</span><ul class="callout-links"><li><a href="995/F3.expansion.html">In this window</a></li>
                              <li><a class="in-nw" href="995/F3.expansion.html">In a new window</a></li>
                           </ul><ul class="fig-services"><li class="ppt-link"><a href="/powerpoint/51/347/995/F3">Download as PowerPoint Slide</a></li>
                           </ul></div>
                     </div>
                     <div class="fig-caption"><span class="fig-label">
                           <strong>Fig. 3.</strong>
                           </span> 
                        
                        <p id="p-20" class="first-child">Northern blot analysis of the steady‐state mRNA level of <em>AtEm1</em> and <em>AtEm6</em> genes at different stages during development of wild‐type and <em>lec1‐1</em> mutant seeds. The indicated probes were hybridized to total RNA isolated from developing Wassilewskija (W. type) and <em>lec1‐1</em> homozygous siliques. In these experimental conditions silique development lasted 21 d from pollination to dry seed. The siliques
                           harvested in this period were divided into five groups according to their stage of development (dap, days after pollination).
                           rRNA: to control equal loading, the ribosomal bands in the gel were stained with ethidium bromide and visualized under UV
                           light.
                        </p>
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                  </div>
                  
                  
                  
                  <h3>AtEm1 and AtEm6 protein accumulation in lec1‐1 seeds</h3>
                  
                  <p id="@hwp:id">During seed development, AtEm1 and AtEm6 proteins accumulate at the highest level in mature seeds, as has been monitored by
                     Western analysis using a specific immune serum (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Bies <em>et al</em>., 1998</a>). The immune serum used detected two protein bands with apparent molecular masses of 21 and 16 kDa in wild‐type mature seed
                     extracts. These values are higher than the calculated molecular weights and are due to delayed migration of Em proteins in
                     SDS gels (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Raynal <em>et al</em>., 1989</a>). Total protein extracts were prepared from wild‐type and <em>lec1‐1</em> seeds. 15 μg of soluble proteins from each sample were separated by SDS‐polyacrylamide gel electrophoresis and blotted into
                     a membrane. Immunodetection experiments using the anti‐AtEm antibody showed the presence of the two expected bands in the
                     wild‐type seed extracts (Fig. 4). However, neither of these bands was detected in lec1‐1 seed protein extract. The pre‐immune
                     sera did not react with any protein in the seed extracts. Because the amount of proteins loaded on the gel was the same for
                     the two samples and the total protein content is much lower in <em>lec1‐1</em> than in wild‐type seeds (Fig. 2), these data indicate that neither AtEm1 nor AtEm6 proteins accumulate to a significant level
                     in <em>lec1‐1</em> mature seeds. Moreover, bands of a similar or higher apparent mobility to AtEm1 protein were detected in the total protein
                     extracts of <em>lec1‐1</em> seeds (Fig. 2), indicating that an extensive non‐specific protein degradation is not the reason for the absence of detection
                     of the AtEm proteins.
                  </p>
                  
                  <p id="@hwp:id">
                     
                     
                  </p>
                  <div id="F4" class="fig pos-float odd">
                     <div class="fig-inline"><a href="995/F4.expansion.html"><img alt="  Fig.&#x2009;4. " src="995/F4.small.gif"/></a><div class="callout"><span>View larger version:</span><ul class="callout-links"><li><a href="995/F4.expansion.html">In this window</a></li>
                              <li><a class="in-nw" href="995/F4.expansion.html">In a new window</a></li>
                           </ul><ul class="fig-services"><li class="ppt-link"><a href="/powerpoint/51/347/995/F4">Download as PowerPoint Slide</a></li>
                           </ul></div>
                     </div>
                     <div class="fig-caption"><span class="fig-label">
                           <strong>Fig. 4.</strong>
                           </span> 
                        
                        <p id="p-23" class="first-child">Western blot analysis of the native AtEm1 and AtEm6 proteins. 15 μg of proteins extracted from wild‐type and <em>lec1‐1</em> seed were loaded in each line in a SDS‐polyacrylamide gel. Immunodetection was performed with a primary antibody which specifically
                           detects <em>A</em>. <em>thaliana</em> Em proteins (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Bies <em>et al</em>., 1998</a>). The pre‐immune serum hybridization gave negative results (not shown).
                        </p>
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                  </div>
                  
                  
                  
                  <h3>Gene promoter activity in leafy cotyledon seeds</h3>
                  
                  <p id="@hwp:id">The <em>AtEm1</em> promoter/<em>β</em>‐<em>glucuronidase</em> (<em>AtEm1</em>::GUS) reporter gene fusion (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Hull <em>et al</em>., 1996</a>) was introduced into the <em>lec1‐1</em>, <em>lec1‐2</em> and <em>fus3‐2</em> mutant plants by conventional genetic crossing. The <em>fus3‐2</em> seeds were kindly provided by Dr Simon Miséra (IPK, Gatersleben). Heterozygous mutant plants were crossed with transgenic
                     plants homozygous for the <em>AtEm1</em>::GUS construct. GUS activity was analysed in wild‐type and mutant F<sub>2</sub> dry seeds extracted from the same siliques of at least three independent crosses. Since both types of seeds develop within
                     the same pod, varying physiological conditions can be excluded as an explanation for the differences in the GUS activity.
                     As a control, the <em>ABI3</em> gene promoter fused to the <em>β</em>‐<em>glucuronidase</em> gene (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Parcy <em>et al</em>., 1994</a>) was also introduced in <em>lec1‐1</em> and <em>fus3‐2</em> genetic backgrounds and similarly analysed. <em>AtEm1</em> promoter activity was about 9‐fold higher in <em>lec1‐1</em> and <em>fus3‐2</em> seeds compared to wild‐type (Fig. 5). In contrast, a similar level of GUS activity was observed in <em>lec1‐2</em> seeds. These results with <em>lec1‐1</em> and <em>lec1‐2</em> seeds are consistent with the pattern of <em>AtEm1</em> mRNA accumulation (Fig. 1). On the other hand, the levels of GUS activity directed by the <em>ABI3</em> construct were not significantly affected in <em>lec1‐1</em> or <em>fus3‐2</em> seeds compared with wild‐type.
                  </p>
                  
                  <p id="@hwp:id">The time‐course of <em>AtEm1</em> and <em>ABI3</em> promoter activity during seed development was determined in wild‐type, <em>lec1‐1</em> and <em>fus3‐2</em> seeds by fluorometric assay (Fig. 6). In wild‐type <em>AtEm1</em>::GUS <em>A</em>. <em>thaliana</em> transgenic plants, GUS activity was detected only in the embryo during late stages of seed development (CM Vicient, unpublished
                     data). At all the studied stages of development (mid‐cotyledonary to mature seed) the level of expression of the <em>AtEm1</em>::GUS construct was higher in the <em>lec1‐1</em> and <em>fus3‐2</em> than in the wild‐type seeds, whereas the level of expression of the <em>ABI3</em>::GUS construct was similar. As the <em>lec1‐1</em> embryos are not easily distinguishable from wild‐type before the cotyledonary stage, it was not possible to perform fluorometric
                     assays at earlier stages. However, at the cotyledonary stage the level of AtEm1 promoter activity in <em>lec1‐1</em> and <em>fus3‐2</em> seeds was several folds higher than in wild‐type, suggesting that <em>AtEm1</em> promoter began to be active earlier in seed development in the mutant embryos than in wild‐type.
                  </p>
                  
                  <p id="@hwp:id">Abscisic acid (ABA) induces the accumulation of the <em>AtEm1</em> mRNA and the expression of the <em>AtEm1</em> promoter in <em>A</em>. <em>thaliana</em> immature seeds (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Gaubier <em>et al</em>., 1993</a>; <a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Bies <em>et al</em>., 1998</a>). ABA was analysed to discover whether it was also able to induce the <em>AtEm1</em> promoter in <em>lec1‐1</em> immature seeds. Wild‐type and <em>lec1‐1</em> immature <em>AtEm1</em>::GUS transgenic seeds extracted from the same siliques were incubated in the presence of 50 μM ABA. Wild‐type and <em>lec1‐1</em> seeds were incubated in parallel in the absence of ABA as a control. After 24 h, the activity of the <em>AtEm1</em> promoter in wild‐type seeds was 40% higher in ABA‐treated than in the untreated control seeds (Fig. 7). An increase was observed
                     in <em>lec1‐1</em> seeds by the same order of magnitude (60%), indicating that the <em>lec1</em> mutation does not alter ABA responsiveness.
                  </p>
                  
                  
                  
                  
                  <p id="@hwp:id">
                     
                     
                  </p>
                  <div id="F5" class="fig pos-float odd">
                     <div class="fig-inline"><a href="995/F5.expansion.html"><img alt="  Fig.&#x2009;5. " src="995/F5.small.gif"/></a><div class="callout"><span>View larger version:</span><ul class="callout-links"><li><a href="995/F5.expansion.html">In this window</a></li>
                              <li><a class="in-nw" href="995/F5.expansion.html">In a new window</a></li>
                           </ul><ul class="fig-services"><li class="ppt-link"><a href="/powerpoint/51/347/995/F5">Download as PowerPoint Slide</a></li>
                           </ul></div>
                     </div>
                     <div class="fig-caption"><span class="fig-label">
                           <strong>Fig. 5.</strong>
                           </span> 
                        
                        <p id="p-28" class="first-child">GUS activity of <em>AtEm1</em> and <em>ABI3</em> promoter‐GUS constructs in wild‐type and mutant seeds. GUS activity was measured in wild‐type and mutant F<sub>2</sub> dry seeds of the indicated genetic crosses. The results correspond to the average values of the number of independent crosses
                           indicated (lines). Error bars are the standard errors.
                        </p>
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                  </div>
                  
                  <p id="@hwp:id">
                     
                     
                  </p>
                  <div id="F6" class="fig pos-float odd">
                     <div class="fig-inline"><a href="995/F6.expansion.html"><img alt="  Fig.&#x2009;6. " src="995/F6.small.gif"/></a><div class="callout"><span>View larger version:</span><ul class="callout-links"><li><a href="995/F6.expansion.html">In this window</a></li>
                              <li><a class="in-nw" href="995/F6.expansion.html">In a new window</a></li>
                           </ul><ul class="fig-services"><li class="ppt-link"><a href="/powerpoint/51/347/995/F6">Download as PowerPoint Slide</a></li>
                           </ul></div>
                     </div>
                     <div class="fig-caption"><span class="fig-label">
                           <strong>Fig. 6.</strong>
                           </span> 
                        
                        <p id="p-30" class="first-child">Developmental pattern of expression of the <em>AtEm1</em> and <em>ABI3</em> gene promoters in wild‐type and mutant seeds. GUS expression under the control of the <em>AtEm1</em> and <em>ABI3</em> gene promoters was measured by fluorometric assays in wild‐type (thick line) and mutant seeds (broken line) dissected from
                           the same pool of siliques at different stages of development. Number 1 corresponds to the first pool of siliques, at the top
                           of the inflorescence in which normal and mutant seeds can be unambiguously differentiated based on the cotyledon morphology
                           (mid cotyledonary stage). Relative GUS activities are given at the basis of equal protein concentrations. The reference value
                           for both wild‐type and mutant seeds was the same: value 100 corresponds to the GUS activity obtained for wild‐type dry seeds.
                           In some of the graphs the scales for the wild‐type and the mutant seeds are different.
                        </p>
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                  </div>
                  
                  <p id="@hwp:id">
                     
                     
                  </p>
                  <div id="F7" class="fig pos-float odd">
                     <div class="fig-inline"><a href="995/F7.expansion.html"><img alt="  Fig.&#x2009;7. " src="995/F7.small.gif"/></a><div class="callout"><span>View larger version:</span><ul class="callout-links"><li><a href="995/F7.expansion.html">In this window</a></li>
                              <li><a class="in-nw" href="995/F7.expansion.html">In a new window</a></li>
                           </ul><ul class="fig-services"><li class="ppt-link"><a href="/powerpoint/51/347/995/F7">Download as PowerPoint Slide</a></li>
                           </ul></div>
                     </div>
                     <div class="fig-caption"><span class="fig-label">
                           <strong>Fig. 7.</strong>
                           </span> 
                        
                        <p id="p-32" class="first-child">ABA induction of <em>AtEm1</em> gene promoter in wild‐type and <em>lec1‐1</em> immature seeds. GUS expression under the control of the <em>AtEm1</em> gene promoter was measured by fluorometric assays in wild‐type (W. type) and mutant (<em>lec1‐1</em>) immature seeds dissected from the same pool of siliques after 24 h of incubation in MS medium (Control), or MS medium supplemented
                           with 50 μM of abscisic acid (ABA). The average and standard error of eight measures are shown. GUS activity is expressed as
                           a percentage with respect to the control seeds for each genotype on the basis of equal protein contents.
                        </p>
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                  </div>
                  
                  
                  
                  <h3>Deletion analysis of the putative cis‐acting elements responsible for the high level of AtEm1 promoter activity in lec1‐1
                     seeds
                  </h3>
                  
                  <p id="@hwp:id">In order to identify the regions in the <em>AtEm1</em> promoter involved in the regulation of the gene by the LEC1‐dependent regulatory pathway, 5′‐deletion constructs of the <em>AtEm1</em> promoter were introduced into <em>lec1‐1</em> genetic background. Homozygous transgenic plants were crossed with <em>lec1‐1</em> heterozygous plants. The −1443 construct corresponds to the <em>AtEm1</em> promoter construct used in previous experiments. All the plants of the different crosses were grown in parallel. GUS activity
                     was determined in wild‐type and <em>lec1‐1</em> F<sub>2</sub> seeds as in previous experiments. GUS stimulation was defined as the relative increment of GUS activity in <em>lec1‐1</em> seeds compared to the wild type. The results obtained (Table 1) can be summarized in four points: (1) in the −1443 construct
                     stimulation of GUS activity in the mutant was 10.6‐fold, similar to the previously observed values (Fig. 5); (2) this stimulation
                     in the GUS activity disappeared when the region −1443 to −197 was deleted; (3) 31.2 GUS stimulation was detected after the
                     deletion of the next 15 bases (−197 to −182); and (4) GUS activity and GUS stimulation were reduced to very low levels after
                     the deletion of the −182 to −119 region. These results suggest the existence of at least two regions in the <em>AtEm1</em> promoter involved in the interaction with LEC1 regulatory pathway.
                  </p>
                  
                  
               </div>
               <div class="section" id="sec-17">
                  <div class="section-nav"><a href="#sec-9" title="Results" class="prev-section-link"><span>Previous Section</span></a><a href="#sec-18" title="Discussion" class="next-section-link"><span>Next Section</span></a></div>
                  
                  
                  <p id="@hwp:id">
                     
                     
                  </p>
                  <div class="table pos-float" id="T1">
                     <div class="table-inline">
                        <div class="callout"><span>View this table:</span><ul class="callout-links"><li><a href="995/T1.expansion.html">In this window</a></li>
                              <li><a class="in-nw" href="995/T1.expansion.html">In a new window</a></li>
                           </ul></div>
                     </div>
                     <div class="table-caption"><span class="table-label">
                           <strong>Table 1.</strong>
                           </span> 
                        
                        <p id="p-35" class="first-child"><em>GUS activity of various</em> AtEm1 <em>promoter deletion constructs in wild‐type and</em> lec1‐1 <em>seeds</em></p>
                        
                        <p id="@hwp:id">GUS activity is given in relative values standardized by protein content. Standard errors are indicated. GUS stimulation was
                           calculated in terms of relative GUS activity in <em>lec1‐1</em> seeds divided by relative GUS activity in wild‐type seeds.
                        </p>
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                  </div>
                  
               </div>
               <div class="section" id="sec-18">
                  <div class="section-nav"><a href="#sec-17" title="Previous Section" class="prev-section-link"><span>Previous Section</span></a><a href="#ack-1" title="Acknowledgments" class="next-section-link"><span>Next Section</span></a></div>
                  
                  <h2>Discussion</h2>
                  
                  <p id="@hwp:id"><em>Leafy cotyledon1</em> mutants have been isolated on the basis of the morphological alterations in the embryo but, in addition to these alterations,
                     the <em>leafy cotyledon1</em> mutant embryos are also severely affected in several physiological aspects such as tolerance to desiccation, precocious germination
                     and accumulation of storage lipids and proteins (<a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Meinke, 1992</a>; <a id="xref-ref-26-6" class="xref-bibr" href="#ref-26">Meinke <em>et al</em>., 1994</a>; <a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">West <em>et al</em>., 1994</a>). This complex phenotype suggests that the expression of many genes may be affected. Supporting this hypothesis, the mRNA
                     steady‐state level of all the <em>lea</em> genes tested was altered in the mutant seeds. Although the exact function of the protein products of many of the <em>lea</em> genes is not known, they may be involved in the acquisition of tolerance to desiccation (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Delseny <em>et al</em>., 1994</a>). Interestingly, <em>leafy cotyledon1</em> seeds are intolerant to desiccation, so this phenotype could be correlated with an alteration in the expression of the genes
                     conferring to the embryo the tolerance to desiccation. Accordingly, it has been shown that AtEm1 and AtEm6 proteins are present
                     at an undetectable level in <em>lec1‐1</em> seeds. On the other hand, the mRNA accumulation of <em>PAP85</em>, an <em>A</em>. <em>thaliana</em> gene homologous to the broad bean vicilin gene, was reduced in <em>lec1‐1</em> and <em>lec1‐2</em> seeds. This is in agreement with the observation that mature protein bodies are not present in <em>lec1</em> embryos (<a id="xref-ref-26-7" class="xref-bibr" href="#ref-26">Meinke <em>et al</em>., 1994</a>). Moreover, the accumulation of three cruciferin mRNAs is reduced by at least 50% in the <em>lec1‐1</em> mutant (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Parcy <em>et al</em>., 1997</a>), a 7S storage protein gene promoter is not operative in <em>lec1‐2</em> embryos (<a id="xref-ref-39-6" class="xref-bibr" href="#ref-39">West <em>et al</em>., 1994</a>) and the expression of cruciferin and 2S albumin genes is reduced in <em>fus3‐2</em> seeds (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bäumlein <em>et al</em>., 1994</a>). A strong reduction was also observed in the accumulation of the storage proteins in total extracts of the mutant seeds.
                  </p>
                  
                  <p id="@hwp:id">LEC1, FUS3 and ABI3 proteins are key elements in the regulation of seed development. Accordingly to Meinke <em>et al</em>., LEC1 and FUS3 might act in the same regulatory pathway and separately from the ABI3 pathway (<a id="xref-ref-26-8" class="xref-bibr" href="#ref-26">Meinke <em>et al</em>., 1994</a>). Later, a study using double mutants of the three regulatory genes <em>ABI3</em>, <em>lec1</em> and <em>fus3</em> suggested that these three genes act in concert to regulate genes involved in seed maturation (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Parcy <em>et al</em>., 1997</a>). The results of this study support this scenario. <em>lec1‐1</em> and <em>fus3‐2</em> mutants have the same effect over the activity of the gene promoters tested, and the overall patterns of protein accumulation
                     in <em>fus3‐2</em> and <em>lec1</em> mature seeds are similar. These results are in agreement with the hypothesis that FUS3 and LEC1 are in the same regulatory
                     pathway. On the other hand, the activity of ABI3 promoter is not significantly affected in the <em>lec1‐1</em> or <em>fus3‐2</em> seeds. Moreover, the activity of the <em>AtEm1</em> gene promoter is ABA responsive in <em>lec1‐1</em> immature embryos and it is known that ABI3 is necessary for this response (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Parcy <em>et al</em>., 1994</a>). These data support the idea that ABI3 and LEC1/FUS3 proteins are in separate regulatory pathways. However, for example,
                     <em>AtEm6</em> or <em>M10</em>, require ABI3 and LEC1 for expression (<a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Parcy <em>et al</em>., 1994</a>), and the same was observed for the expression of two <em>MYB</em> genes (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Kirik <em>et al</em>., 1998<em>b</em></a>). These results indicate that the ABI3 and LEC1 pathways interact in the regulation of at least some genes.
                  </p>
                  
                  <p id="@hwp:id">Increasing observations illustrate the complex regulation of the expression of the <em>AtEm1</em> gene. In physiological conditions, <em>AtEm1</em> is strongly expressed only during late embryogenesis. The expression of <em>AtEm1</em> is reduced in the <em>abi3</em> and <em>aba1</em> mutants (<a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">Parcy <em>et al</em>., 1994</a>). On the other hand, it has been shown that <em>AtEm1</em> expression is increased in <em>lec1‐1</em> mutants, indicating that LEC1 protein may act as an inhibitor for <em>AtEm1</em> expression. This over‐expression behaviour of <em>AtEm1</em> in <em>lec1‐1</em> background was further substantiated in two independent ways. First, Northern blot kinetics revealed that the gene accumulates
                     much earlier in <em>lec1‐1</em> than in wild type. This observation is correlated with the fact that <em>lec1</em> begins to be expressed at early stages of development (<a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Lotan <em>et al</em>., 1998</a>). Second an <em>AtEm1</em>::GUS fusion is also expressed earlier and to a higher level in the <em>lec1‐1</em> and <em>fus3‐2</em> genetic backgrounds. The activity of the <em>ABI3</em> promoter is not affected in the same way. This is not the first example in which the expression of a gene is increased in
                     the <em>lec1‐1</em> mutant, for instance, the expression of the <em>AtMYB13</em> gene is also increased in <em>lec1‐1</em> embryos compared to wild type (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Kirik <em>et al</em>., 1998<em>a</em></a>). Interestingly, as in the case of <em>AtEm1</em>, the transcript of <em>AtMYB13</em> is not detected in the <em>abi3</em> mutant. The ABA response of the <em>AtEm1</em>::GUS construct, which is mediated by ABI3 is not significantly affected by the <em>lec1‐1</em> mutation. This suggests that ABI3 and LEC1 control two distinct regulatory pathways and that <em>AtEm1</em> is the target of both of them. A <em>fus3‐2</em> mutant as a recipient for the construction has been used as an independent control. It is interesting that the <em>AtEm1</em> promoter is over‐expressed as well in the <em>fus3‐2</em> background, suggesting that FUS3 might also repress <em>AtEm1</em>. The exact nature of the interaction between LEC1 and <em>AtEm1</em> promoter is not known. The 5′ promoter deletion analysis showed that, at least, two regions in the promoter are implicated
                     in the regulation of the <em>AtEm1</em> gene by the LEC1‐dependent regulatory pathway.
                  </p>
                  
                  <p id="@hwp:id">The increased expression observed in <em>lec1‐1</em> was not observed in <em>lec1‐2</em>. Some other differences in gene expression and protein accumulation have been reported between both alleles. For example,
                     differences in the accumulation of AGL15 (AGAMOUS‐like MADS domain) protein have been reported (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Perry <em>et al</em>., 1996</a>) and the expression of the <em>PAP51</em> gene is also differentially affected by the two allelic mutations. Although <em>lec1‐1</em> and <em>lec1‐2</em> alleles show very similar phenotypes, general morphology, internal anatomy and response in culture, it is likely that <em>lec1‐1</em> could correspond to a null allele whereas it is conceivable that <em>lec1‐2</em> might be leaky.
                  </p>
                  
                  <p id="@hwp:id">These results show that although <em>AtEm1</em> mRNA accumulates to high levels in <em>lec1‐1</em> the protein itself does not accumulate. These data suggest the existence of an additional post‐transcriptional regulation.
                     Some previous data also support this hypothesis. It is possible to induce the accumulation of <em>AtEm1</em> mRNA in response to ABA in immature seeds (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Gaubier <em>et al</em>., 1993</a>) and in leaves of transgenic plants ectopically expressing <em>ABI3</em> (<a id="xref-ref-31-7" class="xref-bibr" href="#ref-31">Parcy <em>et al</em>., 1994</a>), however, in both cases, the protein does not accumulate (<a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">Bies <em>et al</em>., 1998</a>). It remains unclear whether the protein is not translated, is degraded or both. In the second case, the degradation must
                     be something more or less specific because proteins of relative high molecular weights are still present in protein extracts
                     of <em>lec1</em> seeds. The existence of specific proteases degrading Em protein has been reported in wheat (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Taylor and Cuming, 1993<em>a</em></a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37"><em>b</em></a>) and there is evidence of its existence in <em>A</em>. <em>thaliana</em>(<a id="xref-ref-4-7" class="xref-bibr" href="#ref-4">Bies <em>et al</em>., 1998</a>).
                  </p>
                  
               </div>
               <p id="@hwp:id">
                  
                  
               </p>
               <p id="@hwp:id">
                  
                  
               </p>
               <div class="section ack" id="ack-1">
                  <div class="section-nav"><a href="#sec-18" title="Discussion" class="prev-section-link"><span>Previous Section</span></a><a href="#fn-group-1" title="Footnotes" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>Acknowledgments</h2>
                  
                  <p id="@hwp:id">We thank J Giraudat and F Parcy for the gift of the <em>ABI3</em>::GUS transgenic seeds and for fruitful and stimulating discussions. We are grateful for the gift of the <em>AtEm1</em>::GUS transgenic seeds from G Hull, and for the gift of mutant seeds from D Meinke, S Miséra and J Harada. We are grateful
                     to AH Schulman, R Cooke, S Prat and D Ludevid for their help in preparing this manuscript. CMV was the recipient of a postdoctoral
                     fellowship from the Spanish Instituto Nacional de Investigación y Tecnología Agraria y Alimentaria (INIA). This work was supported
                     by the CNRS and by the EC BIOTECH Program (Grant No. BIO4‐CT96‐0062), and benefits from the joint CNRS‐CSIC Laboratoire Européen
                     Associé Perpignan‐Barcelone for Plant Molecular and Cellular Biology.
                  </p>
                  
               </div>
               <div class="section fn-group" id="fn-group-1">
                  <div class="section-nav"><a href="#ack-1" title="Acknowledgments" class="prev-section-link"><span>Previous Section</span></a><a href="#ref-list-1" title="References" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>Footnotes</h2>
                  <ul><li class="fn" id="fn-1">
                        
                        
                        <p id="p-43"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">1</span> Present address and to whom correspondence should be sent. Institute of Biotechnology, University of Helsinki, Biocenter
                           1, PO Box 56, Viikinkaari 9, FIN‐00014 Helsinki, Finland. Fax: +358 9 708 59570. E‐mail:<span class="em-link" title="carlos.vicient@helsinki.fi"><span class="em-addr">carlos.vicienthelsinki.fi</span></span></p>
                        
                     </li>
                     <li class="fn" id="fn-2">
                        
                        
                        <p id="p-45"><a class="rev-xref" href="#xref-fn-2-1">↵</a><span class="fn-label">2</span> Present address: Department of Biology, McDonnell Hall Rm248, Washington University, St Louis, MO 63130, USA.
                        </p>
                        
                     </li>
                  </ul></div>
               <ul><li class="copyright-statement" id="copyright-statement-1">© Oxford University Press</li>
               </ul><div class="section ref-list" id="ref-list-1">
                  <div class="section-nav"><a href="#fn-group-1" title="Footnotes" class="prev-section-link"><span>Previous Section</span></a><div class="nav-placeholder"> </div>
                  </div>
                  <h2>References</h2>
                  <ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference in text" id="ref-1">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.1">
                           <div class="cit-metadata"><cite><strong>Baker J, Steele C, Dure III L.</strong><span class="cit-pub-date">1988</span>. Sequence and characterisation of 6 Lea proteins and their genes from cotton. <span class="cit-source">Plant Molecular Biology</span><span class="cit-vol">11</span><strong>,</strong><span class="cit-fpage">277</span>–291.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1007/BF00027385&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=A1988P994100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/>
                        
                        <div class="cit ref-cit ref-other no-rev-xref" id="cit-51.347.995.2">
                           <div class="cit-metadata"><cite><strong>Bassüner R, Van Hai N, Jung R, Saalbach G, Müntz K.</strong><span class="cit-pub-date">1987</span>. The primary structure of the predominating vicilin storage protein subunit from field bean seeds (<em>Vicia faba</em> L. var. Minor cv. Fribo). <span class="cit-source">Nucleic Acids Research</span> <span class="cit-vol">15</span><strong>,</strong> <span class="cit-fpage">9609</span>.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=PDF&amp;journalCode=nar&amp;resid=15/22/9609" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference in text" id="ref-3">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.3">
                           <div class="cit-metadata"><cite><strong>Bäumlein H, Miséra S, Luerβen H, Kölle K, Horstmann C, Wobus U, Müller AJ.</strong><span class="cit-pub-date">1994</span>. The <em>FUS3</em> gene of <em>Arabidopsis thaliana</em> is a regulator of gene expression during late embryogenesis. <span class="cit-source">The Plant Journal</span> <span class="cit-vol">6</span>, <span class="cit-fpage">379</span>–387.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1046/j.1365-313X.1994.06030379.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=A1994PF50000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference in text" id="ref-4">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.4">
                           <div class="cit-metadata"><cite><strong>Bies N, Aspart L, Carles C, Gallois P, Delseny M.</strong><span class="cit-pub-date">1998</span>. Accumulation and degradation of Em proteins in <em>Arabidopsis thaliana</em>: evidence for post‐transcriptional controls. <span class="cit-source">Journal of Experimental Botany</span><span class="cit-vol">49</span><strong>,</strong><span class="cit-fpage">1925</span>–1933.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=jexbot&amp;resid=49/329/1925" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference in text" id="ref-5">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.5">
                           <div class="cit-metadata"><cite><strong>Bradford MM.</strong><span class="cit-pub-date">1976</span>. A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein–dye
                                 binding. <span class="cit-source">Analytical Biochemistry</span><span class="cit-vol">72</span><strong>,</strong><span class="cit-fpage">248</span>–254.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1016/0003-2697(76)90527-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=942051&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=A1976BU74400029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference in text" id="ref-6">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.6">
                           <div class="cit-metadata"><cite><strong>Cohen A, Bray EA.</strong><span class="cit-pub-date">1992</span>. Nucleotide sequence of an ABA‐induced tomato gene that is expressed in wilted vegetative organs and developing seeds. <span class="cit-source">Plant Molecular Biology</span><span class="cit-vol">18</span><strong>,</strong><span class="cit-fpage">411</span>–413.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1007/BF00034969&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=1531034&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=A1992HB30400023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference in text" id="ref-7">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.7">
                           <div class="cit-metadata"><cite><strong>Cooke R, Raynal M, Laudié M, <em>et al</em>.</strong><span class="cit-pub-date">1996</span>. Further progress towards a catalogue of all <em>Arabidopsis</em> genes: analysis of a set of 5000 non‐redundant ESTs. <span class="cit-source">The Plant Journal</span> <span class="cit-vol">9</span><strong>,</strong> <span class="cit-fpage">101</span>–124.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1046/j.1365-313X.1996.09010101.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=8580968&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=A1996UC81100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/>
                        
                        <div class="cit ref-cit ref-other no-rev-xref" id="cit-51.347.995.8">
                           <div class="cit-metadata"><cite><strong>de Vries SC.</strong><span class="cit-pub-date">1998</span>. Making embryos in plants. <span class="cit-source">Trends in Plant Science</span><span class="cit-vol">3</span><strong>,</strong><span class="cit-fpage">451</span>–452.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1016/S1360-1385(98)01346-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference in text" id="ref-9">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.9">
                           <div class="cit-metadata"><cite><strong>Delseny M, Gaubier P, Hull G, Saez‐Vazquez J, Gallois P, Raynal M, Cooke R, Grellet F.</strong><span class="cit-pub-date">1994</span>. Nuclear genes expressed during seed desiccation: relationship with responses to stress. In: Basra AS, ed. <span class="cit-source">Stress‐induced gene expression</span>. Reading: Harwood Academic Publishers, <span class="cit-fpage">25</span>–59.</cite></div>
                           <div class="cit-extra"/>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference in text" id="ref-10">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.10">
                           <div class="cit-metadata"><cite><strong>Devic M, Hecht V, Berger C, Lindsey K, Delseny K, Gallois P.</strong><span class="cit-pub-date">1995</span>. Assessment of promoter trap as a tool to study zygotic embryogenesis in <em>Arabidopsis thaliana</em>. <span class="cit-source">Comptes Rendus de l'Académie des Sciences—Series III—Sciences de la vie</span><span class="cit-vol">316</span><strong>,</strong><span class="cit-fpage">1194</span>–1199.</cite></div>
                           <div class="cit-extra"/>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference in text" id="ref-11">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.11">
                           <div class="cit-metadata"><cite><strong>Farrell Jr RE.</strong><span class="cit-pub-date">1993</span>. Electrophoresis of RNA. In: Farrell Jr RE, ed. <span class="cit-source">RNA methodologies. A laboratory guide for isolation and characterization</span>. San Diego: Academic Press, <span class="cit-fpage">125</span>–157.</cite></div>
                           <div class="cit-extra"/>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference in text" id="ref-12">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.12">
                           <div class="cit-metadata"><cite><strong>Finkelstein RR.</strong><span class="cit-pub-date">1993</span>. Abscisic acid‐insensitive mutations provide evidence for stage‐specific signal pathways regulating expression of an <em>Arabidopsis</em> late embryogenesis‐abundant (lea) gene. <span class="cit-source">Molecular and General Genetics</span> <span class="cit-vol">238</span><strong>,</strong> <span class="cit-fpage">401</span>–408.</cite></div>
                           <div class="cit-extra"/>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference in text" id="ref-13">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.13">
                           <div class="cit-metadata"><cite><strong>Gaubier P, Raynal M, Hull G, Huestis GM, Grellet F, Arenas C, Pages C, Delseny M.</strong><span class="cit-pub-date">1993</span>. Two different <em>Em‐like</em> genes are expressed in <em>Arabidopsis thaliana</em> seeds during maturation. <span class="cit-source">Molecular and General Genetics</span> <span class="cit-vol">238</span><strong>,</strong> <span class="cit-fpage">409</span>–418.</cite></div>
                           <div class="cit-extra"/>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference in text" id="ref-14">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.14">
                           <div class="cit-metadata"><cite><strong>Grellet F, Delcasso‐Tremouysaygue D, Delseny M.</strong><span class="cit-pub-date">1989</span>. Isolation and characterization of an unusual repeated sequence from the ribosomal intergenic spacer of the crucifer <em>Sisymbrium irio</em>. <span class="cit-source">Plant Molecular Biology</span><span class="cit-vol">12</span><strong>,</strong><span class="cit-fpage">695</span>–706.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1007/BF00044160&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=A1989AH00400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference in text" id="ref-15">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.15">
                           <div class="cit-metadata"><cite><strong>Harada JJ, DeLisle AJ, Baden CS, Crouch ML.</strong><span class="cit-pub-date">1989</span>. Unusual sequence of an abscisic acid‐inducible mRNA which accumulates late in <em>Brassica napus</em> seed development. <span class="cit-source">Plant Molecular Biology</span> <span class="cit-vol">12</span><strong>,</strong> <span class="cit-fpage">395</span>–401.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1007/BF00017579&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=A1989U562700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference in text" id="ref-16">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.16">
                           <div class="cit-metadata"><cite><strong>Harada JJ, Lotan T, Fischer RL, Goldberg RB.</strong><span class="cit-pub-date">1998</span>. ...Response: embryos without sex. <span class="cit-source">Trends in Plant Science</span><span class="cit-vol">3</span><strong>,</strong><span class="cit-fpage">452</span>–453.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1016/S1360-1385(98)01347-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference in text" id="ref-17">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.17">
                           <div class="cit-metadata"><cite><strong>Hull GA, Bies N, Twell D, Delseny M.</strong><span class="cit-pub-date">1996</span>. Analysis of the promoter of an abscisic acid responsive late embryogenesis abundant gene of <em>Arabidopsis thaliana</em>. <span class="cit-source">Plant Science</span><span class="cit-vol">114</span><strong>,</strong><span class="cit-fpage">181</span>–192.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1016/0168-9452(96)04328-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference in text" id="ref-18">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.18">
                           <div class="cit-metadata"><cite><strong>Keith K, Kraml M, Dengler NG, McCourt P.</strong><span class="cit-pub-date">1994</span>. <em>fusca3</em>: a heterochromic mutation affecting late embryo development in <em>Arabidopsis</em>. <span class="cit-source">The Plant Cell</span><span class="cit-vol">6</span><strong>,</strong><span class="cit-fpage">589</span>–600.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=6/5/589" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference in text" id="ref-19">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.19">
                           <div class="cit-metadata"><cite><strong>Kirik V, Kölle K, Balzer HJ, Bäumlein H.</strong><span class="cit-pub-date">1996</span>. Two new oleosin isoforms with altered expression in seeds of <em>Arabidopsis mutant fus3</em>. <span class="cit-source">Plant Molecular Biology</span><span class="cit-vol">31</span><strong>,</strong><span class="cit-fpage">413</span>–417.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1007/BF00021803&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=8756606&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=A1996VC92700020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference in text" id="ref-20">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.20">
                           <div class="cit-metadata"><cite><strong>Kirik V, Kölle K, Wohlfarth T, Miséra S, Bäumlein H.</strong><span class="cit-pub-date">1998a</span>. Ectopic expression of a novel MYB gene modifies the architecture of the <em>Arabidopsis</em> inflorescence. <span class="cit-source">The Plant Journal</span> <span class="cit-vol">13</span><strong>,</strong> <span class="cit-fpage">729</span>–742.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1046/j.1365-313X.1998.00072.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=9681014&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000073121500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference in text" id="ref-21">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.21">
                           <div class="cit-metadata"><cite><strong>Kirik V, Kölle K, Miséra S, Bäumlein H.</strong><span class="cit-pub-date">1998b</span>. Two novel MYB homologues with changed expression in late embryogenesis‐defective <em>Arabidopsis</em> mutants. <span class="cit-source">Plant Molecular Biology</span> <span class="cit-vol">37</span><strong>,</strong> <span class="cit-fpage">819</span>–827.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1023/A:1006011002499&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=9678577&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000074815600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference in text" id="ref-22">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.22">
                           <div class="cit-metadata"><cite><strong>Leung J, Giraudat J.</strong><span class="cit-pub-date">1998</span>. Abscisic acid signal transduction. <span class="cit-source">Annual Review of Plant Physiology and Plant Molecular Biology</span><span class="cit-vol">49</span><strong>,</strong><span class="cit-fpage">199</span>–222.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1146/annurev.arplant.49.1.199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=000074266700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference in text" id="ref-23">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.23">
                           <div class="cit-metadata"><cite><strong>Lotan T, Ohto M, Yee KM, West MAL, Lo R, Kwong RW, Tamagishi K, Fisher RL, Goldberg RB, Harada JJ.</strong><span class="cit-pub-date">1998</span>. <em>Arabidopsis LEAFY COTYLEDON1</em> is sufficient to induce embryo development in vegetative cells. <span class="cit-source">Cell</span> <span class="cit-vol">93</span><strong>,</strong> <span class="cit-fpage">1195</span>–1205.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1016/S0092-8674(00)81463-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=9657152&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000074491100014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference in text" id="ref-24">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.24">
                           <div class="cit-metadata"><cite><strong>Luerβen H, Kirik V, Herrmann P, Miséra S.</strong><span class="cit-pub-date">1998</span>. <em>FUSCA3</em> encodes a protein with a conserved Vp1/ABI3‐like B3 domain which is of functional importance for the regulation of seed maturation
                                 in <em>Arabidopsis thaliana</em>. <span class="cit-source">The Plant Journal</span> <span class="cit-vol">15</span><strong>,</strong> <span class="cit-fpage">755</span>–764.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1046/j.1365-313X.1998.00259.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=9807814&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000076587300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference in text" id="ref-25">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.25">
                           <div class="cit-metadata"><cite><strong>Meinke D.</strong><span class="cit-pub-date">1992</span>. A homeotic mutant of <em>Arabidopsis thaliana</em> with leafy cotyledons. <span class="cit-source">Science</span> <span class="cit-vol">258</span><strong>,</strong> <span class="cit-fpage">1647</span>–1650.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=258/5088/1647" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference in text" id="ref-26">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.26">
                           <div class="cit-metadata"><cite><strong>Meinke DW, Franzmann LH, Nickle TC, Yeung EC.</strong><span class="cit-pub-date">1994</span>. <em>Leafy Cotyledon</em> mutants of <em>Arabidopsis</em>. <span class="cit-source">The Plant Cell</span> <span class="cit-vol">6</span><strong>,</strong> <span class="cit-fpage">1049</span>–1064.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=6/8/1049" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference in text" id="ref-27">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.27">
                           <div class="cit-metadata"><cite><strong>Miséra S, Müller AJ, Weiland‐Heidecker U, Jürgens G.</strong><span class="cit-pub-date">1994</span>. The <em>FUSCA</em> genes of <em>Arabidopsis</em>: negative regulators of light responses. <span class="cit-source">Molecular and General Genetics</span> <span class="cit-vol">244</span><strong>,</strong> <span class="cit-fpage">242</span>–252.</cite></div>
                           <div class="cit-extra"/>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference in text" id="ref-28">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.28">
                           <div class="cit-metadata"><cite><strong>Murashige T, Skoog F.</strong><span class="cit-pub-date">1962</span>. A revised medium for rapid growth and bioassays with tobacco tissue culture. <span class="cit-source">Physiologia Plantarum</span><span class="cit-vol">15</span><strong>,</strong><span class="cit-fpage">473</span>–497.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1111/j.1399-3054.1962.tb08052.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=A19621781C00014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference in text" id="ref-29">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.29">
                           <div class="cit-metadata"><cite><strong>Ogata F, Miyata T, Fujii N, Yoshida N, Noda K, Makisumi S, Ito A.</strong><span class="cit-pub-date">1991</span>. Purification and amino acid sequence of a bitter gourd inhibitor against an acidic amino acid‐specific endopeptidase of
                                 <em>Streptomyces griseus</em>. <span class="cit-source">Journal of Biological Chemistry</span><span class="cit-vol">266</span><strong>,</strong><span class="cit-fpage">16715</span>–16721.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=jbc&amp;resid=266/25/16715" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference in text" id="ref-30">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.30">
                           <div class="cit-metadata"><cite><strong>Parcy F. Valon C, Kohara A, Miséra S, Giraudat J.</strong><span class="cit-pub-date">1997</span>. The <em>ABSCISIC ACID‐INSENSITIVE3, FUSCA3</em>, and <em>LEAFY COTYLEDON1</em> loci act in concert to control multiple aspects of <em>Arabidopsis</em> seed development. <span class="cit-source">The Plant Cell</span> <span class="cit-vol">9</span><strong>,</strong> <span class="cit-fpage">1265</span>–1277.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=9/8/1265" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference in text" id="ref-31">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.31">
                           <div class="cit-metadata"><cite><strong>Parcy F, Valon C, Raynal M, Gaubier‐Comella P, Delseny M, Giraudat J.</strong><span class="cit-pub-date">1994</span>. Regulation of gene expression programs during <em>Arabidopsis</em> seed development: roles of the <em>ABI3</em> locus and of endogenous abscisic acid. <span class="cit-source">The Plant Cell</span> <span class="cit-vol">6</span><strong>,</strong> <span class="cit-fpage">1567</span>–1582.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=6/11/1567" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference in text" id="ref-32">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.32">
                           <div class="cit-metadata"><cite><strong>Perry SE, Nichols KW, Fernandez DW.</strong><span class="cit-pub-date">1996</span>. The MADS domain protein AGL15 localizes to the nucleus during early stages of seed development. <span class="cit-source">The Plant Cell</span><span class="cit-vol">8</span><strong>,</strong><span class="cit-fpage">1977</span>–1989.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=8/11/1977" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference in text" id="ref-33">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.33">
                           <div class="cit-metadata"><cite><strong>Raynal M, Depigny D, Cooke R, Delseny M.</strong><span class="cit-pub-date">1989</span>. Characterization of a radish nuclear genes expressed during late seed maturation. <span class="cit-source">Plant Physiology</span><span class="cit-vol">91</span><strong>,</strong><span class="cit-fpage">829</span>–836.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=91/3/829" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference in text" id="ref-34">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.34">
                           <div class="cit-metadata"><cite><strong>Raynal M, Guilleminot J, Gueguen C, Cooke R, Delseny M, Gruber V.</strong><span class="cit-pub-date">1999</span>. Structure, organization and expression of two closely related novel <em>lea</em> (late‐embryogenesis‐abundant) genes in <em>Arabidopsis thaliana</em>. <span class="cit-source">Plant Molecular Biology</span> <span class="cit-vol">40</span><strong>,</strong> <span class="cit-fpage">153</span>–165.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1023/A:1026403215270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=10394954&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000080431900015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference in text" id="ref-35">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.35">
                           <div class="cit-metadata"><cite><strong>Sambrook J, Fritsh EF, Maniatis T.</strong><span class="cit-pub-date">1989</span>. <span class="cit-source">Molecular cloning: a laboratory manual</span>. Cold Spring Harbor Laboratory Press.</cite></div>
                           <div class="cit-extra"/>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference in text" id="ref-36">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.36">
                           <div class="cit-metadata"><cite><strong>Taylor RM, Cuming AC.</strong><span class="cit-pub-date">1993a</span>. Selective proteolysis of the wheat Em polypeptide. <span class="cit-source">FEBS Letters</span><span class="cit-vol">331</span><strong>,</strong><span class="cit-fpage">71</span>–75.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1016/0014-5793(93)80299-A&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=8405414&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=A1993LZ25100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference in text" id="ref-37">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.37">
                           <div class="cit-metadata"><cite><strong>Taylor RM, Cuming AC.</strong><span class="cit-pub-date">1993b</span>. Purification of an endoproteinase that digests the wheat ‘Em’ protein <em>in vitro</em>, and determination of its cleavage sites. <span class="cit-source">FEBS Letters</span> <span class="cit-vol">331</span><strong>,</strong> <span class="cit-fpage">76</span>–80.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1016/0014-5793(93)80300-J&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=8405415&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=A1993LZ25100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference in text" id="ref-38">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.38">
                           <div class="cit-metadata"><cite><strong>Vicient CM, Delseny M.</strong><span class="cit-pub-date">1999</span>. Isolation of total RNA from <em>Arabidopsis thaliana</em> seeds. <span class="cit-source">Analytical Biochemistry</span> <span class="cit-vol">268</span><strong>,</strong> <span class="cit-fpage">412</span>–413.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1006/abio.1998.3045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=10075836&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000079508100033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference in text" id="ref-39">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.39">
                           <div class="cit-metadata"><cite><strong>West MAL, Yee KM, Danao J, Zimmerman JL, Fischer RL, Goldberg RB, Harada JJ.</strong><span class="cit-pub-date">1994</span>. <em>LEAFY COTYLEDON1</em> is an essential regulator of late embryogenesis and cotyledon identity in <em>Arabidopsis</em>. <span class="cit-source">The Plant Cell</span> <span class="cit-vol">6</span><strong>,</strong> <span class="cit-fpage">1731</span>–1745.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=6/12/1731" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference in text" id="ref-40">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-51.347.995.40">
                           <div class="cit-metadata"><cite><strong>Wobus U, Weber H.</strong><span class="cit-pub-date">1999</span>. Seed maturation: genetic programmes and control signals. <span class="cit-source">Current Opinion in Plant Biology</span><span class="cit-vol">2</span><strong>,</strong><span class="cit-fpage">33</span>–38.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1016/S1369-5266(99)80007-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=10047566&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000078361800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                  </ol></div><span class="highwire-journal-article-marker-end"/></div>
  </body>
</html>
