<?xml version="1.0"?>
<html>
  <body>
    <div itemprop="articleBody">
      <span class="highwire-journal-article-marker-start"/>
      <div class="hw-optimized-content"><span style="  font-size: 83.33%">
                     Plant Cell, Vol. 13, 229-244, February 2001, Copyright © 2001, American Society of Plant Physiologists
                     </span><h2>
                     Petunia <em>Ap2</em>-like Genes and Their Role in Flower and Seed Development
                     
                     
                  </h2><strong>
                     Tamara Maes<sup><a name="FNR1" id="FNR1"/><a href="#FOOTNOTES">1</a></sup>, 
                     Nancy Van de Steene, 
                     Jan Zethof, 
                     Mansour Karimi, 
                     Mariëlla D'Hauw, 
                     Gwenny Mares, 
                     Marc Van Montagu, and 
                     Tom Gerats
                     </strong><br/><span style="  font-size: 83.33%">
                     Laboratorium of Genetics, University of Ghent, Flemish Institute for Biotechnology, 9000 Ghent, Belgium
                     </span><p><span style="  font-size: 83.33%">
                        Correspondence to: 
                        Tom Gerats, <span class="em-link"><span class="em-addr">toger{at}gengenp.rug.ac.be</span></span> (E-mail), 32-9-2645349 (fax)
                        </span>
                     
                     
                     
                  </p>
                  
                  <p>
                     <a name="ABSTRACT" id="ABSTRACT"/>
                     <br/></p><table style="background-color: #e1e1e1;width: 100%;"><tr><td style="background-color: #ffffff;width: 5%;"><img width="10" height="21" alt="*" src="/icons/toc/rarrow.gif"/></td>
                           <th style="width: 95%;" align="left"><span style="  font-size: 120%">
                                 ABSTRACT</span></th>
                        </tr></table><table><tr><th align="left"><span style="  font-size: 83.33%"><a href="#top"><img alt="*" width="11" height="9" src="/icons/toc/uarrow.gif"/>TOP<br/></a><img alt="*" width="11" height="9" src="/icons/toc/dot.gif"/><span style=" color: 464c53; ">ABSTRACT</span><br/><a href="#INTRODUCTION"><img alt="*" width="11" height="9" src="/icons/toc/darrow.gif"/>INTRODUCTION<br/></a><a href="#RESULTS"><img alt="*" width="11" height="9" src="/icons/toc/darrow.gif"/>RESULTS<br/></a><a href="#DISCUSSION"><img alt="*" width="11" height="9" src="/icons/toc/darrow.gif"/>DISCUSSION<br/></a><a href="#METHODS"><img alt="*" width="11" height="9" src="/icons/toc/darrow.gif"/>METHODS<br/></a><a href="#REFERENCES"><img alt="*" width="11" height="9" src="/icons/toc/darrow.gif"/>REFERENCES<br/></a></span></th>
                        </tr></table><p>
                     
                     
                     
                     We have isolated three <em>Apetala2</em> (<em>Ap2</em>)-like genes from petunia and studied their expression patterns by in situ hybridization. <em>PhAp2A</em> has a high sequence similarity to the A function gene <em>Ap2</em> from Arabidopsis and a similar expression pattern during flower development, suggesting that they are cognate orthologs.
                     <em>PhAp2B</em> and <em>PhAp2C</em> encode for AP2-like proteins that belong to a different subgroup of the AP2 family of transcription factors and exhibit divergent,
                     nearly complementary expression patterns during flower development compared with <em>PhAp2A</em>. In contrast, all three <em>PhAp2</em> genes are strongly expressed in endosperm. The phenotype of the petunia A-type mutant <em>blind</em> cannot be attributed to mutations in the petunia <em>Ap2</em> homologs identified in this study, and reverse genetics strategies applied to identify <em>phap2a</em> mutants indicate that <em>PhAp2A</em> might not be essential for normal perianth development in petunia. Nevertheless, we show that <em>PhAp2A</em> is capable of restoring the homeotic transformations observed in flowers and seed of the <em>ap2-1</em> mutant of Arabidopsis. Although the interspecific complementation proves that <em>PhAp2A</em> encodes a genuine <em>Ap2</em> ortholog from petunia, additional factors may be involved in the control of perianth identity in this species. 
                     
                  </p>
                  
                  <p>
                     
                     
                     
                     <a name="INTRODUCTION" id="INTRODUCTION"/>
                     <br/></p><table style="background-color: #e1e1e1;width: 100%;"><tr><td style="background-color: #ffffff;width: 5%;"><img width="10" height="21" alt="*" src="/icons/toc/rarrow.gif"/></td>
                           <th style="width: 95%;" align="left"><span style="  font-size: 120%">
                                 INTRODUCTION</span></th>
                        </tr></table><table><tr><th align="left"><span style="  font-size: 83.33%"><a href="#top"><img alt="*" width="11" height="9" src="/icons/toc/uarrow.gif"/>TOP<br/></a><a href="#ABSTRACT"><img alt="*" width="11" height="9" src="/icons/toc/uarrow.gif"/>ABSTRACT<br/></a><img alt="*" width="11" height="9" src="/icons/toc/dot.gif"/><span style=" color: 464c53; ">INTRODUCTION</span><br/><a href="#RESULTS"><img alt="*" width="11" height="9" src="/icons/toc/darrow.gif"/>RESULTS<br/></a><a href="#DISCUSSION"><img alt="*" width="11" height="9" src="/icons/toc/darrow.gif"/>DISCUSSION<br/></a><a href="#METHODS"><img alt="*" width="11" height="9" src="/icons/toc/darrow.gif"/>METHODS<br/></a><a href="#REFERENCES"><img alt="*" width="11" height="9" src="/icons/toc/darrow.gif"/>REFERENCES<br/></a></span></th>
                        </tr></table><p>
                     
                     
                     The overall conservation of flower developmental mechanisms in different species is illustrated by several observations. First,
                     analogous homeotic mutants exist in different species, and generally their phenotypes result from mutations in homologous
                     genes. Analysis of homeotic flower organ mutants has resulted in the ABC model, which proposes that three types of functions
                     (A, B, and C) exist that determine the identity of developing primordia in a combinatorial manner (Coen and Meyerowitz 1991
                     <a href="#R1"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). 
                     
                  </p>
                  
                  <p>
                     
                     Second, changes in the expression pattern of a C function gene in a given species often have phenotypic consequences comparable
                     to similar alterations in the expression pattern of its homolog in other species (Mizukami and Ma 1992 <a href="#R2"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>;  Bradley et al. 1993 <a href="#R3"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>;  Tsuchimoto et al. 1993 <a href="#R4"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>;  Pnueli et al. 1994 <a href="#R5"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). 
                     
                  </p>
                  
                  <p>
                     
                     Third, flower developmental genes can be conserved to the extent that their products are functional across species barriers.
                     Overexpression of flower meristem and organ development genes can cause similar phenotypic changes in various species (Mandel
                     et al. 1992 <a href="#R6"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>;  Weigel and Nilsson 1995 <a href="#R7"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>), and B-type mutants have been complemented by the introduction of B-type genes from other species (Samach et al. 1997 <a href="#R8"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). 
                     
                  </p>
                  
                  <p>
                     
                     No data have been reported that allow a comparison of the function of the homologs of the A-class gene <em>Apetala2</em> (<em>Ap2</em>) among the different model systems. 
                     
                  </p>
                  
                  <p>
                     
                     At an early stage of flower development, the <em>Ap2</em> gene of Arabidopsis promotes the establishment of the floral meristem, in cooperation with <em>Apetala1</em> (<em>Ap1</em>), <em>Leafy</em> (<em>Lfy</em>), and <em>Cauliflower1</em> (<em>CAL1</em>;  Irish and Sussex 1990 <a href="#R9"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>;  Bowman et al. 1993 <a href="#R10"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). During floral development, <em>Ap2</em> is essential for the determination of the identity of sepals and petals. In the weak <em>ap2-1</em> mutant, leaf-like organs replace sepals, and petals exhibit antheroid characteristics; in strong <em>ap2</em> mutants, carpels are formed in the outer whorl of the flower, petals are absent, and the number of stamens is reduced (Komaki
                     et al. 1988 <a href="#R11"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>;  Kunst et al. 1989 <a href="#R12"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). The homeotic transformations found in <em>ap2</em> mutants can be explained in part by ectopic expression of <em>Agamous</em> (<em>Ag</em>), as observed in the two outer whorls of their flowers (Drews et al. 1991 <a href="#R13"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). Apart from <em>Ap2</em>, two other genes, <em>Leunig</em> (<em>Lug</em>) and <em>Curly Leaf</em> (<em>Clf</em>), are involved in the repression of <em>Ag</em> in the perianth (Liu and Meyerowitz 1995 <a href="#R14"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>;  Goodrich et al. 1997 <a href="#R15"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). <em>Ap2</em> also is required for normal seed coat development, and <em>ap2</em> mutants lack the epidermal plateau of the epidermis of wild-type seed (Jofuku et al. 1994 <a href="#R16"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>;  Leon-Kloosterziel et al. 1994 <a href="#R17"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>;  Modrusan et al. 1994 <a href="#R18"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). 
                     
                  </p>
                  
                  <p>
                     
                     <em>Ap2</em> was isolated by T-DNA tagging. It encodes a protein that belongs to a rapidly growing family of plant-specific transcription
                     factors that contains the AP2 domain, a conserved ∼60–amino acid region involved in DNA binding (Okamuro et al. 1997 <a href="#R19"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>;  Riechman and Meyerowitz 1998 <a href="#R20"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). The AP2 family of proteins consists of two main branches. One branch contains proteins that have a single copy of the AP2
                     domain. These are involved in different kinds of stress responses and include the ethylene responsive element binding proteins
                     and the C repeat binding factor CBF1 (Ohme-Takagi and Shinshi 1995 <a href="#R21"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>;  Stockinger et al. 1997 <a href="#R22"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). 
                     
                  </p>
                  
                  <p>
                     
                     <em>Ap2</em>, <em>Aintegumenta</em> (<em>Ant</em>), <em>Glossy15</em> (<em>Gl15</em>), and <em>Indeterminate spikelet 1</em> (<em>Ids1</em>) all contain two AP2 domains and belong to the second branch of the <em>Ap2</em>-like transcription factor family that plays a role in plant development. <em>Ant</em> is required for ovule and female gametophyte development (Elliott et al. 1996 <a href="#R23"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>;  Klucher et al. 1996 <a href="#R24"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>), <em>Gl15</em> is required for the maintenance of juvenile traits of epidermal cells in maize leaves from node 2 to node 6 (Moose and Sisco
                     1996 <a href="#R25"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>), and <em>Ids1</em> controls maize spikelet meristem fate (Chuck et al. 1998 <a href="#R26"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). 
                     
                  </p>
                  
                  <p>
                     
                     In this study, we report the isolation and partial characterization of three <em>Ap2-</em>like genes of petunia: <em>PhAp2A</em>, <em>PhAp2B</em>, and <em>PhAp2C</em>. To determine which of these three genes represents the <em>Ap2</em> ortholog in petunia, we compared their sequences with <em>Ap2</em>, analyzed their expression patterns during plant development by in situ hybridization, and applied a reverse genetics approach
                     to identify mutant alleles by transposon insertion mutagenesis. We also mapped the three gene copies to determine if one of
                     them could encode <em>Blind</em> (<em>Bl</em>), which appeared not to be the case. This A-type mutant from petunia has been described previously and has been suggested
                     to be caused by the loss of activity of the petunia <em>Ap2</em> ortholog (Vallade et al. 1987 <a href="#R27"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>;  Tsuchimoto et al. 1993 <a href="#R4"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). The phenotype of flowers of the <em>bl</em> mutant is similar to that seen in <em>ap2</em> and <em>clf</em> mutants in Arabidopsis and in <em>fistulata</em> (<em>fis</em>) mutants in <em>Antirrhinum</em> species (Motte et al. 1998 <a href="#R28"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). 
                     
                  </p>
                  
                  <p>
                     
                     In addition, we have demonstrated the functional conservation of the A-type genes <em>PhAp2A</em> and <em>Ap2</em> by complementing the <em>ap2-1</em> mutant of Arabidopsis with the petunia <em>PhAp2A</em> gene. 
                     
                  </p>
                  
                  <p>
                     
                     <a name="RESULTS" id="RESULTS"/>
                     <br/></p><table style="background-color: #e1e1e1;width: 100%;"><tr><td style="background-color: #ffffff;width: 5%;"><img width="10" height="21" alt="*" src="/icons/toc/rarrow.gif"/></td>
                           <th style="width: 95%;" align="left"><span style="  font-size: 120%">
                                 RESULTS</span></th>
                        </tr></table><table><tr><th align="left"><span style="  font-size: 83.33%"><a href="#top"><img alt="*" width="11" height="9" src="/icons/toc/uarrow.gif"/>TOP<br/></a><a href="#ABSTRACT"><img alt="*" width="11" height="9" src="/icons/toc/uarrow.gif"/>ABSTRACT<br/></a><a href="#INTRODUCTION"><img alt="*" width="11" height="9" src="/icons/toc/uarrow.gif"/>INTRODUCTION<br/></a><img alt="*" width="11" height="9" src="/icons/toc/dot.gif"/><span style=" color: 464c53; ">RESULTS</span><br/><a href="#DISCUSSION"><img alt="*" width="11" height="9" src="/icons/toc/darrow.gif"/>DISCUSSION<br/></a><a href="#METHODS"><img alt="*" width="11" height="9" src="/icons/toc/darrow.gif"/>METHODS<br/></a><a href="#REFERENCES"><img alt="*" width="11" height="9" src="/icons/toc/darrow.gif"/>REFERENCES<br/></a></span></th>
                        </tr></table><p>
                     
                     
                     <strong><em>PhAp2A</em> Is the Closest Ortholog of <em>Ap2</em>; <em>PhAp2B</em> and <em>PhAp2C</em> Belong to a Different Subfamily of Ap2-like Genes in Petunia</strong><br/>
                     
                     An 800-bp <em>PhAp2A-1</em> cDNA insert was used to probe petunia genomic DNA gel blots at medium stringency. Under these conditions, three hybridizing
                     bands can be observed, one of which has a higher intensity than the other two (<a href="#F1">Fig 1A</a>). When the blot was stripped and rehybridized with a 4-kb <em>PhAp2A</em> genomic subclone, only the strongest hybridizing band was observed (<a href="#F1">Fig 1B</a>). These results indicate that <em>PhAp2A</em> belongs to a gene family and that at least two other <em>PhAp2A</em> homologous gene copies exist in the petunia genome, which were termed <em>PhAp2B</em> and <em>PhAp2C</em>. 
                     
                  </p>
                  
                  <p>
                     
                     <a name="F1" id="F1"/>
                     
                     <br/></p>
                  
                  <center>
                     <table style="width: 95%;"><tr style="background-color: #e1e1e1;"><td>
                              
                              <table><tr style="background-color: #e1e1e1;"><td style="background-color: #ffffff;" align="center" valign="top"><a href="229/F1.expansion.html"><img src="229/F1.small.gif"/></a><br/><strong>View larger version</strong> (50K):<br/><a href="229/F1.expansion.html">In this window</a><br/><a href="229/F1.expansion.html" class="in-nw">In a new window</a><br/> 
                                       
                                    </td>
                                    <td valign="top"><strong>Figure 1.</strong>
                                       DNA Gel Blot Analysis of <em>PhAp2</em> Genes in Petunia (Line V23) and Amino Acid Alignments of Petunia <em>PhAp2</em> genes and Arabidopsis <em>Ap2</em>.
                                       
                                       <p>
                                          
                                          <strong>(A)</strong> Blot probed with the 800-bp <em>PhAp2A-1</em> cDNA fragment that contains the Ap2 domains (medium stringency).
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(B)</strong> Blot probed with the 4-kb genomic XbaI fragment of <em>PhAp2A</em> (high stringency).
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(C)</strong> Comparison of the overall amino acid sequences, the AP2 domains (shaded), linkers, and putative nuclear localization signals
                                          (in boldface) of <em>PhAp2A</em> (GenBank accession number 
                                          <a href="/cgi/external_ref?access_num=AF132001&amp;link_type=GEN">AF132001</a>), <em>Ap2</em> (ATU12546), <em>PhAp2B</em> (<a href="/cgi/external_ref?access_num=AF132002&amp;link_type=GEN">AF132002</a>), <em>PhAp2C</em> (partial sequence), <em>RAP2.7</em> (AF003100), and <em>ANT</em> (U40256). Amino acids in the AP2 domains, which are conserved in all mentioned sequences, are in white on a shaded background;
                                          the five conserved amino acids in the domains of <em>PhAp2B</em> and <em>RAP2.7</em> are white on black. Gaps are indicated by dashes. No putative nuclear localization signal could be identified in front of
                                          the first <em>Ap2</em> domain in the <em>Ant</em> gene; the overall homology of <em>Rap2.7</em> and <em>Ant</em> with the other <em>Ap2</em>-(like) genes outside the domain regions was too low to allow a meaningful comparison.
                                          
                                       </p>
                                       
                                    </td>
                                 </tr></table></td>
                        </tr></table></center>
                  
                  <p>
                     
                     The <em>PhAp2A</em> cDNA encodes a protein of 519 amino acid residues (<a href="#F1">Fig 1C</a>). Similarity searches using tBlastn revealed the Arabidopsis AP2 protein as the most homologous entry in the GenBank database.
                     PHAP2A shows 64% overall amino acid identity and 77% similarity with AP2. The homology of PHAP2A and AP2 is not limited to
                     the AP2 domains but extends through the reading frame of the two proteins. A serine-rich putative transcription activation
                     domain (amino acids 18 to 60) and a putative nuclear localization signal (amino acids 148 to 158;  Jofuku et al. 1994 <a href="#R16"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>) are conserved, as are the N and C termini of the proteins (<a href="#F1">Fig 1C</a>). The homology between the two proteins is especially high in the two AP2 domains (63 of 67 and 66 of 68 residues identical),
                     whereas the linker that connects them is completely conserved at the amino acid level (<a href="#F1">Fig 1C</a>). 
                     
                  </p>
                  
                  <p>
                     
                     The <em>PhAp2B</em> cDNA encodes a 457–amino acid protein (<a href="#F1">Fig 1C</a>). The AP2 domains and the linker of PHAP2A and AP2 are more closely related than those of the two petunia proteins PHAP2A
                     and PHAP2B. Interestingly, of the 21 amino acid substitutions that occur in this region between PHAP2A and PHAP2B, five amino
                     acids are similarly substituted in the two domains and two more in the linker of the predicted Arabidopsis RAP2.7 sequence
                     (<a href="#F1">Fig 1C</a>).  
                     
                  </p>
                  
                  <p>
                     
                     Outside of the AP2 domains, there is very little sequence conservation between PHAP2A and PHAP2B. Two conserved motifs (F/VDF/ILKT/V
                     and VTKEF/LFPV/LS) could be identified in the N-terminal part of PHAP2B and PHAP2C and in the Arabidopsis RAP2.7 (GenBank
                     accession number 
                     <a href="/cgi/external_ref?access_num=T20443&amp;link_type=GEN">T20443</a>;  Okamuro et al. 1997 <a href="#R19"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>), and another one could be identified in the C-terminal part of PHAP2B and RAP2.7 (PL/MFSV/TAAASSGF). These data suggest
                     that PHAP2B, PHAP2C, and RAP2.7 belong to a different subgroup than AP2 and PHAP2A (<a href="#F1">Fig 1C</a>). 
                     
                  </p>
                  
                  <p>
                     
                     No <em>PhAp2C</em> cDNA clone could be retrieved from the inflorescence or ovary cDNA library, although reverse transcription–polymerse chain
                     reaction (RT-PCR) using two <em>PhAp2C</em>-specific primers on young petunia inflorescences and flowers suggested that the gene is expressed. 
                     
                  </p>
                  
                  <p>
                     
                     <strong>Chromosomal Localization of <em>PhAp2A</em>, <em>PhAp2B</em>, and <em>PhAp2C</em> Shows That They Do Not Coincide with the <em>Bl</em> Locus</strong><br/>
                     
                     Flowers of the <em>bl</em> mutant of petunia have patches of stigmoid tissue at the tips of their sepals, and antheroids replace the flower limb. These
                     homeotic transformations are caused by ectopic expression of the C-type gene <em>pMADS3</em> (pMADS3 = FBP14;  Kater et al. 1998 <a href="#R29"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>) in <em>bl</em> leaves, sepals, and petals (Tsuchimoto et al. 1993 <a href="#R4"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). It has been proposed that <em>Bl</em> is a negative regulator of <em>pMADS3</em> and that <em>Bl</em> determines the expression boundaries of <em>pMADS3</em> (Tsuchimoto et al. 1993 <a href="#R4"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). Because <em>Ap2</em> is a negative regulator of the C-type gene <em>Ag</em> in Arabidopsis, we wanted to determine if one of its homologs in petunia coincides with the <em>Bl</em> locus. 
                     
                  </p>
                  
                  <p>
                     
                     We performed a linkage analysis for the three <em>PhAp2</em> genes by analyzing restriction fragment length polymorphism (RFLP), amplified fragment length polymorphism (AFLP;  Vos et
                     al. 1995 <a href="#R30"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>), and phenotypic markers in different backcrosses. According to the (V26 <span style="font-family: ariel,helvetica;  ">x</span> W137) <span style="font-family: ariel,helvetica;  ">x</span> W137 progeny analysis, <em>PhAp2A</em> is located on chromosome 4, <em>PhAp2B</em> is located on chromosome 2, and <em>PhAp2C</em> is located on chromosome 6. <em>PhAp2A</em> and the <em>Bl</em> locus, both on chromosome 4, appear to be loosely linked (see Methods for details). Thus, none of the <em>PhAp2</em> loci coincides with the <em>Bl</em> locus. 
                     
                  </p>
                  
                  <p>
                     
                     <strong>Expression of <em>PhAp2</em> Genes during Plant Development</strong><br/>
                     
                     The spatial and temporal distribution of the <em>PhAp2A</em>, <em>PhAp2B</em>, and <em>PhAp2C</em> mRNA and the mRNA expression of the petunia C-type gene <em>pMADS3</em> were analyzed by in situ hybridization of consecutive sections of wild-type inflorescences and flowers or flower parts (<a href="#F2">Fig 2A</a>,  <a href="#F2">Fig 2E</a>,  <a href="#F2">Fig 2I</a>, and  <a href="#F2">Fig 2M</a>) to <sup>35</sup>S-labeled <em>PhAp2A</em>, <em>PhAp2B</em>, <em>PhAp2C</em>, and <em>pMADS3</em> antisense riboprobes. The specificity of the probes used for the in situ hybridization experiments was tested by RNA gel
                     blot analysis (not shown). The different stages of flower development referred to in the text are described in  <a href="#T1">Table 1</a>. 
                     
                  </p>
                  
                  <p>
                     
                     <a name="F2" id="F2"/>
                     
                     <br/></p>
                  
                  <center>
                     <table style="width: 95%;"><tr style="background-color: #e1e1e1;"><td>
                              
                              <table><tr style="background-color: #e1e1e1;"><td style="background-color: #ffffff;" align="center" valign="top"><a href="229/F2.expansion.html"><img src="229/F2.small.gif"/></a><br/><strong>View larger version</strong> (109K):<br/><a href="229/F2.expansion.html">In this window</a><br/><a href="229/F2.expansion.html" class="in-nw">In a new window</a><br/> 
                                       
                                    </td>
                                    <td valign="top"><strong>Figure 2.</strong>
                                       In Situ Analysis of Expression of <em>PhAp2A</em>, <em>PhAp2B</em>, <em>PhAp2C</em>, and <em>pMADS3</em> during Flower Development.
                                       
                                       <p>
                                          
                                          Each row represents a set of consecutive 10-µm sections that were hybridized with the different <sup>35</sup>S-CTP–labeled antisense riboprobes (except for <strong>[E]</strong>, which represents a similar section).
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(A)</strong> Bright-field photograph of the top of the inflorescence with a transverse section through the inflorescence meristem and
                                          a stage 4 to 5 flower and an oblique section through a stage 6 flower and part of a stage 7 flower.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(B)</strong> Dark-field photograph of a section like that shown in <strong>(A)</strong> with <em>PhAp2A</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(C)</strong> Dark-field photograph of a section like that shown in <strong>(A)</strong> with <em>PhAp2B</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(D)</strong> Dark-field photograph of a section like that shown in <strong>(A)</strong> with <em>pMADS3</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(E)</strong> Bright-field photograph of a longitudinal section through a stage 7 flower.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(F)</strong> Dark-field photograph of a section like that shown in <strong>(E)</strong> with <em>PhAp2A</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(G)</strong> Dark-field photograph of a section like that shown in <strong>(E)</strong> with <em>PhAp2B</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(H)</strong> Dark-field photograph of a section like that shown in <strong>(E)</strong> with <em>pMADS3</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(I)</strong> Bright-field photograph of a transverse section through a stage 8 to 9 flower.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(J)</strong> Dark-field photograph of a section like that shown in <strong>(I)</strong> with <em>PhAp2A</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(K)</strong> Dark-field photograph of a section like that shown in <strong>(I)</strong> with <em>PhAp2C</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(L)</strong> Dark-field photograph of a section like that shown in <strong>(I)</strong> with <em>pMADS3</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(M)</strong> Bright-field photograph of a longitudinal section through an inflorescence with an early stage 9 flower and a stage 6 flower.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(N)</strong> Dark-field photograph of a section like that shown in <strong>(M)</strong> with <em>PhAp2A</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(O)</strong> Dark-field photograph of a section like that shown in <strong>(M)</strong> with <em>PhAp2C</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(P)</strong> Dark-field photograph of a section like that shown in <strong>(M)</strong> with <em>pMADS3</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          ac, connective tissue of the anther; ap, anther primordium; br, bract; ca, carpel; fp, flower primordium; ov, ovary; ow, ovary
                                          wall; pc, procambium; pe, petal; pl, placenta; se, sepal; st, stamen; vb, vascular bundle. Bars in <strong>(</strong><strong>A</strong><strong>)</strong>, <strong>(</strong><strong>E</strong><strong>)</strong>, <strong>(</strong><strong>I</strong><strong>)</strong>, and <strong>(</strong><strong>M</strong><strong>)</strong> = 100 µm.
                                          
                                       </p>
                                       
                                    </td>
                                 </tr></table></td>
                        </tr></table></center>
                  
                  <p>
                     
                     <a name="T1" id="T1"/>
                     
                     
                  </p>
                  
                  <center>
                     <table style="width: 95%;"><tr style="background-color: #e1e1e1;"><td>
                              
                              <table><tr style="background-color: #e1e1e1;"><td style="background-color: #ffffff;" align="center" valign="top"> <br/><strong>View this table:</strong><br/><a href="229/T1.expansion.html">In this window</a><br/><a href="229/T1.expansion.html" class="in-nw">In a new window</a><br/> 
                                       
                                    </td>
                                    <td valign="top"><strong>Table 1.</strong>
                                       Stages of Flower Development in <em>Petunia hybrida</em> Line W138
                                       
                                    </td>
                                 </tr></table></td>
                        </tr></table></center>
                  
                  <p>
                     
                     <strong>Spatiotemporal Expression Pattern of <em>PhAp2A</em> Is Complementary to That of <em>PhAp2B</em> and <em>PhAp2C</em> during Flower Development and Overlaps Partially with That of <em>pMADS3</em></strong><br/>
                     
                     In the wild-type petunia inflorescence, uniform <em>PhAp2A</em> expression could be observed in stage 1 to 2 inflorescence meristems and in young stage 4 to 5 flower buds. A weaker signal
                     was observed in the procambial cells of the developing stem. In bracts subtending stage 4 to 5 flower buds and in stage 6
                     sepals, the <em>PhAp2A</em> mRNA was not distributed uniformly but localized to a layer of ground parenchyma cells (<a href="#F2">Fig 2B</a>). The expression of <em>PhAp2A</em> in sepals and bracts decreased after stage 7 to 8 (<a href="#F2">Fig 2F</a>,  <a href="#F2">Fig 2J</a>, and  <a href="#F2">Fig 2N</a>). Petals exhibited strong <em>PhAp2A</em> expression throughout their development. In young petal primordia until stage 5 to 6, the signal had a uniform appearance
                     (<a href="#F2">Fig 2B</a>). In later stages, <em>PhAp2A</em> was expressed in a layer of cells in the ground parenchyma, as in sepals and bracts. This layer runs through the center of
                     the flower tube tissue, assumes a more adaxial position in the flower limb (<a href="#F2">Fig 2J</a> and  <a href="#F2">Fig 2N</a>), and passes around the dorsal side of the vascular bundle of the main vein of the petal (not shown). 
                     
                  </p>
                  
                  <p>
                     
                     Stamen primordia of stage 6 flower buds showed uniform expression that decreased by stage 7 and became concentrated in the
                     region where the vascular bundles of the filaments develop (<a href="#F2">Fig 2B</a>,  <a href="#F2">Fig 2F</a>, and  <a href="#F2">Fig 2J</a>). At stage 9, weak expression also was detected in the anther wall and in the connective tissue (not shown). A hybridizing
                     signal also was detected over mature pollen grains (<a href="#F3">Fig 3H</a>). Young carpel primordia of stage 7 flower buds exhibited strong <em>PhAp2A</em> expression (<a href="#F2">Fig 2F</a>). In stage 7 to 9 flower buds, the signal was strong in the central cell layers of the ovary wall, in the placenta, and in
                     ovule primordia (<a href="#F2">Fig 2J</a> and  <a href="#F2">Fig 2N</a>). When the ovules became stalked, the signal in the ovary wall and the placenta decreased. <em>PhAp2A</em> mRNA then could be detected mainly in the vascular bundle of the placenta, and a weak signal also could be seen in the vascular
                     bundle of the style and stigma (<a href="#F3">Fig 3B</a>). By stage 11, the hybridization signal in the ovules became restricted to the funiculus and the integument (<a href="#F3">Fig 3E</a> and  <a href="#F3">Fig 3K</a>). 
                     
                  </p>
                  
                  <p>
                     
                     <a name="F3" id="F3"/>
                     
                     <br/></p>
                  
                  <center>
                     <table style="width: 95%;"><tr style="background-color: #e1e1e1;"><td>
                              
                              <table><tr style="background-color: #e1e1e1;"><td style="background-color: #ffffff;" align="center" valign="top"><a href="229/F3.expansion.html"><img src="229/F3.small.gif"/></a><br/><strong>View larger version</strong> (105K):<br/><a href="229/F3.expansion.html">In this window</a><br/><a href="229/F3.expansion.html" class="in-nw">In a new window</a><br/> 
                                       
                                    </td>
                                    <td valign="top"><strong>Figure 3.</strong>
                                       Expression of <em>PhAp2</em> Genes and <em>pMADS3</em> in Third and Fourth Whorl Organs.
                                       
                                       <p>
                                          
                                          <strong>(A)</strong> Dark-field photograph of a section through carpels and part of the anthers of a stage 10 flower with <em>pMADS3</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(B)</strong> Dark-field photograph of a section like that shown in <strong>(A)</strong> with <em>PhAp2A</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(C)</strong> Dark-field photograph of a section like that shown in <strong>(A)</strong> with <em>PhAp2B</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(D)</strong> Dark-field photograph of a section like that shown in <strong>(A)</strong> with <em>PhAp2C</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(E)</strong> Dark-field photograph of a section through carpels and part of the anthers of a stage 11 flower with <em>PhAp2A</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(F)</strong> Dark-field photograph of a section like that shown in <strong>(E)</strong> with <em>PhAp2C</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(G)</strong> Dark-field photograph of a section through merged pollen sacs, just before dehiscence, with <em>pMADS3</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(H)</strong> Dark-field photograph of a section like that shown in <strong>(G)</strong> with <em>PhAp2A</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(I)</strong> Dark-field photograph of a section like that shown in <strong>(G)</strong> with <em>PhAp2C</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(J)</strong> Dark-field photograph of a section through ovary of stage 11 flower (detail) with <em>pMADS3</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(K)</strong> Dark-field photograph of a section like that shown in <strong>(J)</strong> with <em>PhAp2A</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(L)</strong> Dark-field photograph of a section like that shown in <strong>(J)</strong> with <em>PhAp2C</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          ac, connective tissue of the anther; aw, anther wall; fu, funiculus; in, integument; ov, ovule; ow, ovary wall; pe, petal;
                                          po, pollen grain; vb, vascular bundle. Bars in <strong>(</strong><strong>A</strong><strong>)</strong> to <strong>(</strong><strong>L</strong><strong>)</strong> = 100 µm.
                                          
                                       </p>
                                       
                                    </td>
                                 </tr></table></td>
                        </tr></table></center>
                  
                  <p>
                     
                     <em>PhAp2B</em> and <em>PhAp2C</em> expression was detected in the outer cell layers of young bracts subtending stage 1 to 2 inflorescence meristems. Stage 7
                     flowers expressed <em>PhAp2B</em> and <em>PhAp2C</em> in the epidermis of the sepals (<a href="#F2">Fig 2C</a> and  <a href="#F2">Fig 2G</a>). Expression was maintained in sepals at least until stage 9 to 10, by which time the signal was no longer localized specifically
                     in the outer cell layers and acquired a spotty appearance (<a href="#F2">Fig 2K</a> and  <a href="#F2">Fig 2O</a>). Weaker <em>PhAp2B</em> and <em>PhAp2C</em> expression also was detected in the outer cell layers around the main veins of stage 9 petals (<a href="#F2">Fig 2K</a>). The two genes were strongly expressed in the connective tissue of stage 8 to 9 stamens and around stage 10 to 11 as well
                     in the anther wall. No signal was observed over the vascular bundles and the filaments or, at this stage, over the developing
                     microspores (<a href="#F2">Fig 2K</a>,  <a href="#F2">Fig 2O</a>,  <a href="#F3">Fig 3C</a>, and  <a href="#F3">Fig 3D</a>). However, in mature anther filaments, a spotty <em>PhAp2C</em> signal was observed (not shown), and although the connective tissue degenerated and the hybridization signal disappeared
                     from the anther wall, <em>PhAp2B</em> and <em>PhAp2C</em> signals could be observed over the pollen grains (<a href="#F3">Fig 3I</a>). The fourth whorl gynoecia of stage 9 flowers exhibited low <em>PhAp2B</em> and <em>PhAp2C</em> expression. As the style elongated, the signal increased in intensity and became restricted to the epidermis of the ovary
                     wall (<a href="#F2">Fig 2K</a>,  <a href="#F3">Fig 3C</a>,  <a href="#F3">Fig 3D</a>,  <a href="#F3">Fig 3F</a>, and  <a href="#F3">Fig 3L</a>). We did not detect any <em>PhAp2B</em> or <em>PhAp2C</em> expression in the placenta or in the developing ovules during the stages of flower development described in this article.
                     
                     
                  </p>
                  
                  <p>
                     
                     The expression patterns of the <em>PhAp2</em> genes were compared with the pattern of the C function gene <em>pMADS3</em>. <em>pMADS3</em> expression first was detected in the center of stage 5 flowers, before any stamen primordia development was visible. The
                     expression in developing stamen primordia remained uniform until stage 7, when the anthers and filaments start to differentiate
                     (<a href="#F2">Fig 2D</a> and  <a href="#F2">Fig 2H</a>). Stage 8 to 9 stamens showed some <em>pMADS3</em> expression in the anther wall and in the connective tissue, but the strongest signal was observed over the vascular bundle
                     in the filament (<a href="#F2">Fig 2L</a> and  <a href="#F2">Fig 2P</a>). No signal was detected over the developing microspores (<a href="#F3">Fig 3G</a>). <em>pMADS3</em> was expressed strongly in carpel primordia. By stage 8, when the gynoecia fuse at the top, expression decreased in the ovary
                     wall but became very strong in the vascular bundles of the placenta and at the positions where the ovule primordia will arise
                     (<a href="#F2">Fig 2L</a> and  <a href="#F2">Fig 2P</a>). By stage 9, <em>pMADS3</em> was expressed in the vascular bundle of the style and the placenta and in the epidermal layer of the style and stigma. Ovule
                     primordia showed strong and uniform expression that persisted until the ovules were mature (<a href="#F2">Fig 2L</a>,  <a href="#F3">Fig 3A</a>, and  <a href="#F3">Fig 3J</a>). The expression pattern observed for <em>pMADS3</em> was in agreement with that reported by others (Kater et al. 1998 <a href="#R29"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). 
                     
                  </p>
                  
                  <p>
                     
                     <strong>Expression Patterns of <em>PhAp2A</em>, <em>PhAp2B</em>, and <em>PhAp2C</em> Coincide in Mature Seed</strong><br/>
                     
                     The expression of the <em>PhAp2</em> genes was investigated further in mature seed, germinating seedlings, and young plants. Strong expression was detected for
                     <em>PhAp2A</em>, <em>PhAp2B</em>, and <em>PhAp2C</em> in the endosperm of mature seed. The hybridizing signal in endosperm was distributed asymmetrically, in that the side where
                     the radicle emerges during germination did not show any expression (<a href="#F4">Fig 4A to 4D</a>). No expression could be detected in the mature embryo. Leaf primordia of young seedlings showed a clear signal for <em>PhAp2A</em>, as did the leaf primordia and the leaves of young plantlets (<a href="#F4">Fig 4E to 4H</a>). As was observed for the developing floral organs, the <em>PhAp2A</em> signal appeared to be distributed homogeneously in leaf primordia and to concentrate subsequently in a layer of ground parenchyma
                     cells of the developing leaves (<a href="#F4">Fig 4F</a>). 
                     
                  </p>
                  
                  <p>
                     
                     <a name="F4" id="F4"/>
                     
                     <br/></p>
                  
                  <center>
                     <table style="width: 95%;"><tr style="background-color: #e1e1e1;"><td>
                              
                              <table><tr style="background-color: #e1e1e1;"><td style="background-color: #ffffff;" align="center" valign="top"><a href="229/F4.expansion.html"><img src="229/F4.small.gif"/></a><br/><strong>View larger version</strong> (85K):<br/><a href="229/F4.expansion.html">In this window</a><br/><a href="229/F4.expansion.html" class="in-nw">In a new window</a><br/> 
                                       
                                    </td>
                                    <td valign="top"><strong>Figure 4.</strong>
                                       Expression of <em>PhAp2</em> Genes in Germinating Seed and Expression of <em>PhAp2A</em> in Young Seedlings and Young Vegetative Plants.
                                       
                                       <p>
                                          
                                          <strong>(A)</strong> Phase-contrast photograph of a stained longitudinal section through a germinating seedling.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(B)</strong> Dark-field photograph of a section like that shown in <strong>(A)</strong> with <em>PhAp2A</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(C)</strong> Dark-field photograph of a section like that shown in <strong>(A)</strong> with <em>PhAp2B</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(D)</strong> Dark-field photograph of a section like that shown in <strong>(A)</strong> with <em>PhAp2C</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(E)</strong> Bright-field photograph of a longitudinal section through a seedling with emerging leaf primordia.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(F)</strong> Dark-field photograph of a section like that shown in <strong>(E)</strong> with <em>PhAp2A</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(G)</strong> Bright-field photograph of a longitudinal section through the apex of a young vegetative plantlet.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(H)</strong> Dark-field photograph of a section like that shown in <strong>(G)</strong> with <em>PhAp2A</em> expression.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          co, cotyledon; en, endosperm; le, leaf; lp, leaf primordium; r, radicle. Bars in <strong>(</strong><strong>A</strong><strong>)</strong> to <strong>(</strong><strong>H</strong><strong>)</strong> = 100 µm.
                                          
                                       </p>
                                       
                                    </td>
                                 </tr></table></td>
                        </tr></table></center>
                  
                  <p>
                     
                     The expression of the <em>PhAp2A</em> gene thus differs markedly from that of <em>PhAp2B/PhAp2C</em> during vegetative and floral development; in contrast, the expression patterns of the three genes coincide in mature seed.
                     Mutants in which the functions of these genes have been abolished are required to reveal the roles of the <em>PhAp2</em> genes during plant development. 
                     
                  </p>
                  
                  <p>
                     
                     <strong>Transposon Insertion Mutagenesis of <em>PhAp2A</em></strong><br/>
                     
                     Based on the results of in situ gene expression analysis and sequence comparisons, <em>PhAp2A</em> was the only likely candidate to perform the orthologous <em>Ap2</em> function in petunia. To prove that <em>PhAp2A</em> functions as an A-type gene, we screened for loss-of-function alleles by transposon insertion mutagenesis. A PCR-based screening
                     method (Ballinger and Benzer 1989 <a href="#R31"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>;  Kaiser and Goodwin 1990 <a href="#R32"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>) was used to select for insertions of the nonautonomous transposable element <em>dTph1</em> in <em>PhAp2A</em> by using three available petunia populations (Koes et al. 1995 <a href="#R33"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>;  <a href="#F5">Fig 5A</a>). A set of <em>PhAp2A</em>-specific primers and a primer complementary to the terminal inverted repeat of the transposon were used; five heterozygous
                     insertion mutants were detected in a population of 3456 W138 plants (<a href="#F5">Fig 5B</a>). 
                     
                  </p>
                  
                  <p>
                     
                     <a name="F5" id="F5"/>
                     
                     <br/></p>
                  
                  <center>
                     <table style="width: 95%;"><tr style="background-color: #e1e1e1;"><td>
                              
                              <table><tr style="background-color: #e1e1e1;"><td style="background-color: #ffffff;" align="center" valign="top"><a href="229/F5.expansion.html"><img src="229/F5.small.gif"/></a><br/><strong>View larger version</strong> (38K):<br/><a href="229/F5.expansion.html">In this window</a><br/><a href="229/F5.expansion.html" class="in-nw">In a new window</a><br/> 
                                       
                                    </td>
                                    <td valign="top"><strong>Figure 5.</strong>
                                       Insertion Mutagenesis of the <em>PhAp2A</em> Gene.
                                       
                                       <p>
                                          
                                          <strong>(A)</strong> Screening for insertion mutants. DNA pools of 1080 plants, organized in pools of 10 block samples, 12 row samples, and 9
                                          column samples, were subjected to PCR using <em>PhAp2A</em> primer 002 in combination with the <em>dTph1</em> inverted repeat primer. The PCR products were analyzed by DNA gel blot analysis using a <em>PhAp2A</em>-specific probe. Two positive plants were identified with the following coordinates: block 5, row 6, column 1 = V2025-6 and
                                          block 9, row 10, column 9 = V2047-10. The column signal of the second positive plant was missing in this screen but was identified
                                          in a similar experiment using <em>PhAp2A</em> primer 003. Marks indicate positive signals.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(B)</strong> Positions of the <em>dTph1</em> insertions in <em>PhAp2A</em> and the positions of the primers used in the screening experiment. UTR, untranslated region.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(C)</strong> RT-PCR analysis of wild-type mRNA and <em>PhAp2A</em>(V2025-6) mRNA in the transposition-stabilized <em>act</em><sup>-</sup>/<em>act</em><sup>-</sup> background. Products were amplified from first strand cDNA by using <em>PhAp2A</em> primers 001 and 002. The wild-type product (WT; lane 1) is ∼300 bp smaller than the product from the homozygous insertion
                                          mutant (INS; lane 2), which contains the <em>dTph1</em> element (lane 3; PstI size marker).
                                          
                                       </p>
                                       
                                    </td>
                                 </tr></table></td>
                        </tr></table></center>
                  
                  <p>
                     
                     The positions of the <em>dTph1</em> elements were determined by cloning and sequencing of the fragments flanking the transposon insertions in <em>PhAp2A</em>. Three plants contained a <em>dTph1</em> insertion in intron sequences (T2194-14, intron VIII; V2047-10, intron I; V2193-7, intron III). The two other plants harbored
                     a <em>dTph1</em> element in the first exon, after amino acids 17 (V2116-10) and 141 (V2025-6) (<a href="#F5">Fig 5B</a>). None of the homozygous insertion mutants segregating in the progeny of the selfed heterozygous primary insertion mutants
                     had an altered phenotype. 
                     
                  </p>
                  
                  <p>
                     
                     We used RT-PCR with two gene-specific primers flanking the exon 1 insertion site to analyze the <em>PhAp2A</em> mRNA in the exon 1 insertion mutants V2025-6 and V2116-10. The mutant cDNA fragments are ∼300 bp longer than the wild-type
                     <em>PhAp2A</em> cDNA fragments. This suggests that there is no premature termination of the messenger due to transcription termination signals
                     in the element and that <em>dTph1</em> is cotranscribed with the <em>PhAp2A</em> messenger (shown for the V2025-6 insertion allele in  <a href="#F5">Fig 5C</a>). Also, wild-type and mutant cDNA fragments exhibited a comparable abundance in plants heterozygous for the insertion. Thus,
                     the <em>dTph1</em> insertions did not seem to affect the level of <em>PhAp2A</em> transcription. The <em>dTph1</em> element is 284 bp long and induces a target site duplication of 8 bp upon insertion. The translated nucleotide sequence of
                     the elements inserted in the <em>PhAp2A</em> gene contains stop codons in the three reading frames. Thus, theoretically, only 17 amino acids of the PHAP2A protein of
                     V2116-10 and 141 amino acids of the PHAP2A protein of V2025-6 would be translated, which should result in a functional knockout
                     of the <em>PhAp2A</em> gene. 
                     
                  </p>
                  
                  <p>
                     
                     In plants homozygous for the V2025-6 insertion, a low level of wild-type mRNA sometimes was detected by RT-PCR. To determine
                     whether this product was formed after excision of <em>dTph1</em> or by alternative splicing, we crossed this insertion allele into an <em>activator</em> minus background to stabilize the insertion. The <em>Activator</em> element controls transposition of <em>dTph1</em> elements (Gerats et al. 1990 <a href="#R34"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). Stabilization of the <em>dTph1</em> system was monitored by means of a <em>dTph1</em>-tagged reporter gene that controls anthocyanin synthesis (see Methods). Plants homozygous for the <em>PhAp2A</em>(V2025-6) allele and lacking the active <em>Activator</em> produced only the mutant mRNA and, again, exhibited a wild-type phenotype. The expression patterns of <em>PhAp2A</em> and <em>pMADS3</em> in plants homozygous for the <em>PhAp2A</em>(V2025-6) insertion allele were analyzed by in situ hybridization and did not differ from the patterns found in wild-type
                     petunia (results not shown). 
                     
                  </p>
                  
                  <p>
                     
                     <strong><em>PhAp2A</em> Complements the <em>ap2-1</em> Mutant of Arabidopsis</strong><br/>
                     
                     Because the <em>phap2a</em> insertion mutants we obtained did not exhibit a mutant phenotype, we decided to express <em>PhAp2A</em> in an <em>ap2-1</em> background in Arabidopsis to determine whether this might result in complementation. The <em>ap2-1</em> mutant is temperature sensitive. At 15°C, the sepals have leaf-like characteristics, and the petals are slightly phylloid;
                     at 25°C, the sepals are stigmoid leaves, and the petals are staminoid; and at 29°C, the sepals are carpelloid, and the petals
                     are absent, resembling the phenotype of the strong <em>ap2</em> mutants (Bowman et al. 1989 <a href="#R35"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). The binary vector pBIN19-35S <em>PhAp2A</em> was used to transform the <em>ap2-1</em> mutant (see Methods). Kanamycin-resistant calli and shoots were selected at 23°C, but rooted transformants were allowed to
                     develop at 26°C to induce a more extreme <em>ap2-1</em> phenotype. 
                     
                  </p>
                  
                  <p>
                     
                     All shoots from four of five independent transformants generated phenotypically wild-type flowers that produced phenotypically
                     wild-type seed (<a href="#F6">Fig 6</a>). Together with the sequence and expression data, the complementation data prove that <em>PhAp2A</em> encodes the <em>Ap2</em> ortholog of petunia. 
                     
                  </p>
                  
                  <p>
                     
                     <a name="F6" id="F6"/>
                     
                     <br/></p>
                  
                  <center>
                     <table style="width: 95%;"><tr style="background-color: #e1e1e1;"><td>
                              
                              <table><tr style="background-color: #e1e1e1;"><td style="background-color: #ffffff;" align="center" valign="top"><a href="229/F6.expansion.html"><img src="229/F6.small.gif"/></a><br/><strong>View larger version</strong> (111K):<br/><a href="229/F6.expansion.html">In this window</a><br/><a href="229/F6.expansion.html" class="in-nw">In a new window</a><br/> 
                                       
                                    </td>
                                    <td valign="top"><strong>Figure 6.</strong>
                                       Complementation of the Arabidopsis <em>ap2-1</em> Mutant by <em>PhAp2A</em>.
                                       
                                       <p>
                                          
                                          <strong>(A)</strong> Arabidopsis <em>ap2-1</em> flower (grown at 26°C). The flower is from a transformant that did not exhibit complementation.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(B)</strong> Arabidopsis <em>ap2-1</em>::<em>35S PhAp2A</em> flower (grown at 26°C) exhibiting full complementation toward the wild type.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(C)</strong> Scanning electron micrograph of the seed coat of a noncomplementing <em>ap2-1</em> transformant.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(D)</strong> Scanning electron micrograph of the seed coat of an <em>ap2-1</em>::<em>35S PhAp2A</em> transformant.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(E)</strong> Detail of <strong>(C)</strong>.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          <strong>(F)</strong> Detail of <strong>(D)</strong>.
                                          
                                       </p>
                                       
                                       <p>
                                          
                                          Bar in <strong>(C)</strong> and <strong>(D)</strong> = 100 µm; bar in <strong>(E)</strong> and <strong>(F)</strong> = 10 µm.
                                          
                                       </p>
                                       
                                    </td>
                                 </tr></table></td>
                        </tr></table></center>
                  
                  <p>
                     
                     <a name="DISCUSSION" id="DISCUSSION"/>
                     <br/></p><table style="background-color: #e1e1e1;width: 100%;"><tr><td style="background-color: #ffffff;width: 5%;"><img width="10" height="21" alt="*" src="/icons/toc/rarrow.gif"/></td>
                           <th style="width: 95%;" align="left"><span style="  font-size: 120%">
                                 DISCUSSION</span></th>
                        </tr></table><table><tr><th align="left"><span style="  font-size: 83.33%"><a href="#top"><img alt="*" width="11" height="9" src="/icons/toc/uarrow.gif"/>TOP<br/></a><a href="#ABSTRACT"><img alt="*" width="11" height="9" src="/icons/toc/uarrow.gif"/>ABSTRACT<br/></a><a href="#INTRODUCTION"><img alt="*" width="11" height="9" src="/icons/toc/uarrow.gif"/>INTRODUCTION<br/></a><a href="#RESULTS"><img alt="*" width="11" height="9" src="/icons/toc/uarrow.gif"/>RESULTS<br/></a><img alt="*" width="11" height="9" src="/icons/toc/dot.gif"/><span style=" color: 464c53; ">DISCUSSION</span><br/><a href="#METHODS"><img alt="*" width="11" height="9" src="/icons/toc/darrow.gif"/>METHODS<br/></a><a href="#REFERENCES"><img alt="*" width="11" height="9" src="/icons/toc/darrow.gif"/>REFERENCES<br/></a></span></th>
                        </tr></table><p>
                     
                     
                     In this study, we report the isolation and partial characterization of three <em>Ap2</em>-like genes from petunia: <em>PhAp2A</em>, <em>PhAp2B</em>, and <em>PhAp2C</em>. The PHAP2A protein is the closest AP2 homolog described so far. The 18–amino acid cores of the AP2 domain, which theoretically
                     are capable of forming an amphipathic α helix (Jofuku et al. 1994 <a href="#R16"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>), are identical in the two proteins. If the DNA binding specificity of the AP2-like proteins is determined by the amphipathic
                     α-helix core itself, PHAP2A should be able to bind the target sequences of AP2. In contrast with what has been observed for
                     other AP2-like proteins, the homology between PHAP2A and AP2 extends throughout the whole protein, suggesting that these genes
                     are orthologs. 
                     
                  </p>
                  
                  <p>
                     
                     Two closely related <em>Ap2</em>-like gene copies, <em>PhAp2B</em> and <em>PhAp2C</em>, detected by DNA gel blot hybridization were isolated. Based on the occurrence of similar amino acid substitutions in their
                     AP2 domains, PHAP2B and PHAP2C, like RAP2.7, and IDS1 belong to a different subgroup of AP2-like transcription factors than
                     AP2 and PHAP2A. 
                     
                  </p>
                  
                  <p>
                     
                     <strong>Expression Patterns of <em>PhAp2A</em> and <em>PhAp2B/PhAp2C</em> Suggest That These Genes May Have Both Overlapping and Unique Functions</strong><br/>
                     
                     Like <em>Ap2</em> and <em>Ant</em> in Arabidopsis, <em>PhAp2A</em> is expressed uniformly in young lateral organ primordia. As the organs mature, the signal in developing bracts, sepals, petals,
                     and the ovary wall gradually decreases in the outer layers of these organs and eventually disappears. 
                     
                  </p>
                  
                  <p>
                     
                     During flower development, <em>PhAp2B</em> and <em>PhAp2C</em> expression patterns are remarkably complementary to that of <em>PhAp2A</em>. <em>PhAp2B</em> and <em>PhAp2C</em> expression begins to appear in bracts, sepals, petals, and the ovary wall when <em>PhAp2A</em> expression is decreasing. In general, we observed little if any overlap between the expression of <em>PhAp2A</em> and that of <em>PhAp2B</em> and <em>PhAp2C</em>. Only some of the <em>Ap2</em>-like genes that have been described have been well characterized in terms of both expression and function. The <em>Gl15</em> gene has been described to control the identity of epidermal cells in maize (Moose and Sisco 1996 <a href="#R25"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). The expression patterns of <em>PhAp2B</em> and <em>PhAp2C</em> are compatible with a function in the development of the epidermis. The divergence of the amino acid sequences between PHAP2B/PHAP2C
                     and GL15, however, suggests that they are not orthologous. 
                     
                  </p>
                  
                  <p>
                     
                     <em>PhAp2A</em> also is expressed during vegetative development. A strong signal can be observed in leaf primordia of young seedlings that
                     at later stages decreases and concentrates in the inner layers of the developing leaves. Thus, the progression of <em>PhAp2A</em> expression during floral development is the same as that during vegetative development, supporting the idea that the floral
                     organs are nothing but specialized, modified leaves (von Goethe, 1790). 
                     
                  </p>
                  
                  <p>
                     
                     The homology between the AP2 protein of Arabidopsis and the PHAP2A protein of petunia, together with the resemblance of their
                     expression patterns, strongly suggests that the two genes are genuine orthologs. 
                     
                  </p>
                  
                  <p>
                     
                     Like <em>Ap2</em> and <em>Ant</em> of Arabidopsis, <em>PhAp2A</em> also is expressed during ovule development. The <em>PhAp2A</em> expression in integuments is in accordance with the role of <em>Ap2</em> in seed coat development observed in Arabidopsis. <em>PhAp2A</em>, <em>PhAp2B</em>, and <em>PhAp2C</em> show an identical expression pattern in mature and germinating seed. A strong, asymmetrical signal is observed in the endosperm,
                     in which it is excluded from the side of radicle emergence. This expression pattern resembles that of prolamin seed storage
                     proteins of maize (also called zeins), which start to be expressed coordinately in the endosperm of maize 8 to 10 days after
                     pollination. The promoters of these zein genes share a common regulatory element, the prolamin box (5'-TGTAAAG-3'), that is
                     a good candidate for being the <em>cis</em>-acting regulatory element controlling this expression pattern (Forde et al. 1985 <a href="#R36"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). Recently, an endosperm-specific factor belonging to the Dof family of zinc finger proteins that binds the prolamin box
                     and interacts with Opaque 2, a known transcriptional activator of zein gene expression, was isolated from maize (Vicente-Carbajosa
                     et al. 1997 <a href="#R37"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). A perfect prolamin box (TGTAAAG) and four related elements (TGHAAARK) were detected in the promoter of the <em>PhAp2A</em> gene (T. Maes, results not shown). At least one other Ap2-like gene, an <em>Ant</em> homolog of maize (GenBank accession number 
                     <a href="/cgi/external_ref?access_num=Z47554&amp;link_type=GEN">Z47554</a>), also is expressed in postpollination maize endosperm (Elliott et al. 1996 <a href="#R23"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>;  Daniell et al. 1996 <a href="#R38"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). 
                     
                  </p>
                  
                  <p>
                     
                     <strong><em>PhAp2A</em> and the Petunia Class A Function: Isolation of <em>PhAp2A</em> Insertion Alleles</strong><br/>
                     
                     In Arabidopsis, the class A gene <em>Ap2</em> fulfills two roles in the process of floral organ identity determination: a cadastral function consisting of repression of
                     the class C gene <em>Agamous</em> and an organ specification function in the perianth (sepal identity and determination of meristems in the second whorl).
                     
                     
                  </p>
                  
                  <p>
                     
                     To determine whether <em>PhAp2A</em> performs the same function in petunia that <em>Ap2</em> does in Arabidopsis, we screened for <em>dTph1</em> insertions in <em>PhAp2A</em>. Two of the five isolated <em>PhAp2A</em> insertion alleles had an insertion in the first exon of the gene. Surprisingly, W138 plants homozygous for these exon 1 insertions
                     in <em>PhAp2A</em> do not exhibit a mutant phenotype in floral development. 
                     
                  </p>
                  
                  <p>
                     
                     <strong>A Complete <em>PhAp2A</em> Knockout?</strong><br/>
                     
                     Because attempts to obtain phenotypic <em>phap2a</em> mutants through cosuppression or antisense strategies have been equally unsuccessful (T. Maes, unpublished results), our
                     first hypothesis is based on the assumption that the <em>dTph1</em> insertions in exon 1 of <em>PhAp2A</em> knock out all of the potential functions of PHAP2A. We showed that the sequences of <em>PhAp2B</em>/<em>PhAp2C</em>, and their expression patterns during flower development, are very distinct from those of <em>PhAp2A</em>, making the hypothesis that they exert the same function improbable. Thus, the absence of a phenotype in the <em>PhAp2A</em> insertion mutants seems to contradict the hypothesis that homologs of the <em>Ap2</em> gene of Arabidopsis play a crucial role as the class A function genes in the development of all angiosperm flowers. 
                     
                  </p>
                  
                  <p>
                     
                     Alternatively, the lack of a phenotype could be due to functional redundancy caused by the activity of another gene without
                     any sequence homology with the <em>Ap2</em> genes. This would imply that Arabidopsis would lack such a gene. Because the <em>bl</em> mutant shows analogous, albeit less pronounced, homeotic transformations reminiscent of A-type mutants of Arabidopsis, <em>Bl</em> is an obvious candidate for such a function. In Arabidopsis, at least two other mutants have been described that exhibit
                     flowers displaying homeotic transformations similar to those in <em>ap2</em> mutants: <em>lug</em> and <em>clf</em> (Liu and Meyerowitz 1995 <a href="#R14"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>;  Goodrich et al. 1997 <a href="#R15"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). <em>clf</em> mutations have pleiotropic effects on flowering time, leaf shape, and flower morphology. In <em>clf</em> flowers, sepals and petals show partial transformation toward carpels and stamens. CLF seems to act later in flower development
                     than AP2 and LUG (Liu and Meyerowitz 1995 <a href="#R14"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>;  Goodrich et al. 1997 <a href="#R15"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). 
                     
                  </p>
                  
                  <p>
                     
                     Another possibility is that the A function, as encoded by <em>Ap2</em> of Arabidopsis, does not exist as such in petunia. The changes in the identity of perianth organs of A-type mutants found
                     in petunia (<em>bl</em>) and in <em>Antirrhinum</em> species (<em>fistulata</em> and <em>stylosa</em>) can be attributed completely to the lack of cadastral activity, that is, to the lack of control of the expression domain
                     of the C-type genes (Tsuchimoto et al. 1993 <a href="#R4"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>;  Motte et al. 1998 <a href="#R28"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). Until now, none of the A function mutants in these two species has been attributed to a mutation in an <em>Ap2</em>-like gene. In addition, Arabidopsis is the only species in which a gene has been identified that controls both sepal and
                     petal specification and the expression of the class C gene <em>Agamous</em>. In Arabidopsis, <em>Ap2</em> is required to establish the morphological features of the sepal epidermis, which differ from those of leaves in that the
                     sepal epidermis does not have stellate trichomes (Bowman et al. 1989 <a href="#R35"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). In petunia, the inflorescence leaf epidermis and the sepal epidermis are indistinguishable, suggesting that the <em>Ap2</em> ortholog of petunia has no function in the specification of sepal cell identity. If the only role of the <em>PhAp2A</em> gene in petunia consists of directly or indirectly negatively regulating the C function in the perianth, mutations that exclusively
                     affect the transcription activation domain may not interfere with its function in flower development. This brings us to our
                     second hypothesis. 
                     
                  </p>
                  
                  <p>
                     
                     <strong>A Partial <em>PhAp2A</em> Knockout?</strong><br/>
                     
                     Theoretically, truncated versions of the PHAP2A protein could be translated from the mRNA of the exon 1 insertion mutants
                     we have obtained. In both cases, the truncated protein would be missing the putative transcription activation domain but would
                     retain the putative nuclear localization signal and the AP2 domains. Such a truncated protein presumably still could function
                     as a repressor and possibly regulate <em>pMADS3</em> expression. Because <em>PhAp2A</em> overexpression can complement the <em>ap2-1</em> mutation, the easiest way to verify whether we have obtained loss-of-function mutants, or if a truncated <em>PhAp2A</em> gene retains the ability to regulate <em>Ag</em>, would be to express a similarly truncated <em>PhAp2A</em> gene in the <em>ap2-1</em> mutant of Arabidopsis. 
                     
                  </p>
                  
                  <p>
                     
                     <strong><em>PhAp2A</em> Behaves as a Functional A-Type Gene in Arabidopsis</strong><br/>
                     
                     Promoter 35S-driven <em>PhAp2A</em> gene expression in Arabidopsis complements all mutant aspects of flower development of the <em>ap2-1</em> mutation. Ectopic expression of <em>Ag</em> and the consequent homeotic changes in the perianth are inhibited, and the expression of genes required for the determination
                     of sepal versus leaf identity is regulated correctly, in that normal sepals develop in the complemented flowers. In conclusion,
                     the complementation of the <em>ap2-1</em> mutant unequivocally proves our hypothesis that <em>PhAp2A</em> represents the functional ortholog of <em>Ap2</em>. Therefore, additional factors must be involved in the control of perianth identity in petunia. Although the general conservation
                     of developmental control systems is widely accepted, there are examples in which divergence has occurred. A case in point
                     seems to be the essential role played by <em>sonic hedgehog</em> (<em>Shh</em>) in the formation of body asymmetries in chick but not in mouse (Pagan-Westphal and Tabin 1998 <a href="#R39"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). Here, we might have another example in which two orthologs have greatly diverged in at least some aspects of their function.
                     Thus, either petunia <em>PhAp2A</em> has lost its perianth identity control function or Arabidopsis <em>Ap2</em> has gained such a function. Further analyses involving a range of species are needed to clarify this point in more detail.
                     
                     
                  </p>
                  
                  <p>
                     
                     <a name="METHODS" id="METHODS"/>
                     <br/></p><table style="background-color: #e1e1e1;width: 100%;"><tr><td style="background-color: #ffffff;width: 5%;"><img width="10" height="21" alt="*" src="/icons/toc/rarrow.gif"/></td>
                           <th style="width: 95%;" align="left"><span style="  font-size: 120%">
                                 METHODS</span></th>
                        </tr></table><table><tr><th align="left"><span style="  font-size: 83.33%"><a href="#top"><img alt="*" width="11" height="9" src="/icons/toc/uarrow.gif"/>TOP<br/></a><a href="#ABSTRACT"><img alt="*" width="11" height="9" src="/icons/toc/uarrow.gif"/>ABSTRACT<br/></a><a href="#INTRODUCTION"><img alt="*" width="11" height="9" src="/icons/toc/uarrow.gif"/>INTRODUCTION<br/></a><a href="#RESULTS"><img alt="*" width="11" height="9" src="/icons/toc/uarrow.gif"/>RESULTS<br/></a><a href="#DISCUSSION"><img alt="*" width="11" height="9" src="/icons/toc/uarrow.gif"/>DISCUSSION<br/></a><img alt="*" width="11" height="9" src="/icons/toc/dot.gif"/><span style=" color: 464c53; ">METHODS</span><br/><a href="#REFERENCES"><img alt="*" width="11" height="9" src="/icons/toc/darrow.gif"/>REFERENCES<br/></a></span></th>
                        </tr></table><p>
                     
                     
                     <strong>Plant Material</strong><br/><em>Petunia hybrida</em> line W138 contains a high copy number of the nonautonomous transposable element <em>dTph1</em>, which requires the presence of an (active) <em>Activator</em>, located on chromosome 1, to undergo transposition (Gerats et al. 1990 <a href="#R34"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>;  Huits et al. 1995 <a href="#R40"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). 
                     
                  </p>
                  
                  <p>
                     
                     Petunia lines W138 and W162 are homozygous for an unstable recessive mutation caused by an insertion of the nonautonomous
                     element <em>dTph1</em> in the <em>An1</em> gene. <em>An1</em> controls the expression of several structural genes in the anthocyanin pathway, and <em>an1</em> mutants have a white corolla. The formation of red or pink revertant sectors and spots in line W138 results from excision
                     of the <em>dTph1</em> element and can be used as a reporter system for transposition activity. Line W162 does not contain the <em>Activator</em> of the two-element system, and hence this line has white flowers (Huits et al. 1995 <a href="#R40"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). The <em>bl</em> mutant was in the R51 background. 
                     
                  </p>
                  
                  <p>
                     
                     <strong>DNA Gel Blot Analysis</strong><br/>
                     
                     DNA was extracted from petunia leaves as described by  Souer et al. 1995 <a href="#R41"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>. Twenty micrograms of DNA was digested and separated by agarose gel electrophoresis. Blotting and the composition of the
                     hybridization buffer were as described by Amersham. α-<sup>32</sup>P-dCTP–labeled DNA probes were synthesized using the Megaprime DNA labeling kit (Amersham). Hybridizations and washes were
                     performed at 60°C with 2 <span style="font-family: ariel,helvetica;  ">x</span> SSC, 0.1% SDS and either 0.5 <span style="font-family: ariel,helvetica;  ">x</span> SSC, 0.1% SDS (intermediate stringency) or 0.1 <span style="font-family: ariel,helvetica;  ">x</span> SSC, 0.1% SDS (high stringency; 1 <span style="font-family: ariel,helvetica;  ">x</span> SSC is 0.15 M NaCl and 0.015 M sodium citrate). 
                     
                  </p>
                  
                  <p>
                     
                     <strong>Isolation and Sequencing of the <em>PhAp2</em> Genes</strong><br/>
                     
                     The full-length Arabidopsis <em>Ap2</em> cDNA was used to screen 10<sup>5</sup> plaque-forming units (PFUs) of a petunia λGEM4 cDNA library made from flower buds at low stringency. Initially, four positive
                     clones were isolated that contained an
                     800-bp partial cDNA insert. This partial cDNA clone was used to screen 3 <span style="font-family: ariel,helvetica;  ">x</span> 10<sup>6</sup> PFUs of a Uni-ZAP XR cDNA library from inflorescence meristems at medium stringency. The 15 positive clones that were isolated
                     were analyzed by polymerase chain reaction (PCR) using two cPhAP2A-1–specific primers (p001 and p002). All clones corresponded
                     to the same gene copy. Four full-length <em>PhAp2B</em> cDNA clones were isolated by screening 3 <span style="font-family: ariel,helvetica;  ">x</span> 10<sup>6</sup> PFUs of a HybriZAP ovary cDNA library using a 300-bp PCR fragment amplified from a 1.1-kb <em>PhAp2B</em> genomic subclone as a probe. 
                     
                  </p>
                  
                  <p>
                     
                     The cPhAP2A-1 insert (a kind gift from Bart den Boer, Aventis Crop Sciences, Gent) was used to screen a V26 genomic library
                     in λ at high stringency. Positive clones were analyzed by restriction analysis and represented the same gene. A 4.0-kb XbaI
                     fragment
                     cross-hybridizing with the cPhAP2-1 insert and a 3.2-kb flanking fragment were subcloned in pUC18, and deletion clones were
                     generated using the method of  Henikoff 1984 <a href="#R42"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>. The <em>PhAp2A</em> gene contains nine introns; the positions of introns I to VI are conserved compared with those of the introns of the Arabidopsis
                     <em>Ap2</em> gene. 
                     
                  </p>
                  
                  <p>
                     
                     To isolate <em>PhAp2B</em> and <em>PhAp2C</em>, we constructed a new genomic library from line W138 plants in λGem11 (Promega). The library was screened with the cPhAP2A-1
                     cDNA insert, and washes were at intermediate stringency (see
                     DNA Gel Blot Analysis above). Twenty-nine clones were isolated in the primary screen, 15 of which were analyzed further. They
                     could be divided based on their restriction patterns into three groups that corresponded to the three gene copies observed
                     in DNA gel blots. A 1.2-kb SstI-XbaI <em>PhAp2B</em> fragment and a 1.1-kb EcoRI-HindIII <em>PhAp2C</em> fragment were subcloned and sequenced. 
                     
                  </p>
                  
                  <p>
                     
                     Sequences were determined by the dideoxy chain termination method (Sanger et al. 1977 <a href="#R43"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>) using fluorescent dye terminators and AmpliTaq FS in a cycle-sequencing protocol as described by the manufacturer (Roche
                     Molecular Systems Inc., Branchburg, NJ) on an AB1373A or an ABI1377 automatic sequencer. Gaps in the sequence were closed
                     using a primer-based strategy. 
                     
                  </p>
                  
                  <p>
                     
                     <strong>Chromosomal Localization of <em>PhAp2A</em>, <em>PhAp2B</em>, and <em>PhAp2C</em></strong><br/>
                     
                     We detected HindIII restriction fragment length polymorphisms (RFLPs) for the three PhAp2 copies between lines V26 and W137.
                     To determine the map positions of <em>PhAp2A</em>, <em>PhAp2B</em>, and <em>PhAp2C</em>, we crossed line V26 to line W137, and the F1 hybrid was backcrossed to line W137. A population of 61 individuals was used
                     to construct a genetic map. Linkage between 183 amplified fragment length polymorphism (AFLP) markers, obtained from 14 different
                     primer combinations (EcoRI plus three selective nucleotides, MseI plus three selective nucleotides), four phenotypic markers
                     (<em>Hf1</em> on chromosome 1, <em>Fl</em> on chromosome 2, <em>An11</em> on chromosome 3, and <em>Rt</em> on chromosome 6), and the <em>PhAp2</em> RFLPs in this backcross population, was analyzed by the JoinMap program (Stam 1993 <a href="#R44"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). The AFLP linkage groups identified in the progeny of the (V26 <span style="font-family: ariel,helvetica;  ">x</span> W137) <span style="font-family: ariel,helvetica;  ">x</span> W137 test cross could be anchored to their specific chromosomes by linkage to the phenotypic markers and by comparison with
                     the locations of 40 AFLP markers that had been located previously in a different set of crosses (Gerats et al. 1995 <a href="#R45"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>). 
                     
                  </p>
                  
                  <p>
                     
                     The linkage between <em>PhAp2A</em> and the <em>Bl</em> locus, both on chromosome 4, was investigated further using a mutant <em>PhAp2A</em> allele as a segregating marker. Plant T2194-14 is heterozygous for a transposon insertion in intron VIII (see Transposon
                     Insertion Mutagenesis of <em>PhAp2A</em>). T2194-14 was crossed to line R51 (<em>bl/bl</em>), and F1 hybrids that contained the <em>PhAp2A</em> insertion allele were selfed. All of the F1 plants were wild type. Of 96 F2 plants, 26 exhibited a <em>bl</em> phenotype. PCR analysis, using gene-specific primers flanking the transposon insertion, showed that seven of 20 <em>bl</em> F2 plants tested carried the <em>PhAp2A</em>(T2194-14) allele. Because <em>PhAp2A</em>(T2194-14) homozygotes are wild type, and <em>bl</em> segregated as a normal recessive mutation, and because seven of 20 <em>bl</em> mutants contain the insertion allele, we conclude that <em>PhAp2A</em> does not coincide with <em>Bl</em>. Moreover, <em>bl</em> mutants that are homozygous for the <em>PhAp2A</em>(T2194-14) insertion allele exhibit a phenotype similar to that of <em>bl</em> mutants that carry wild-type <em>PhAp2A</em> alleles. 
                     
                  </p>
                  
                  <p>
                     
                     <strong>In Situ Hybridization Studies</strong><br/>
                     
                     Petunia tissue (line W138) was fixed in 3.7% formaldehyde, 5% acetic acid, and 50% ethanol, cleared in Histo-clear (National
                     Diagnostics, Atlanta, GA), and embedded in paraffin. Consecutive 10-µm tissue sections were placed on sets of dimethylsilane-treated
                     (Sigma) microscope slides so that comparable sections could be hybridized with different probes. The cDNA inserts of cPhAp2A-2
                     and <em>pMADS3</em> were amplified from the plasmid using the T3 and T7 sequencing primers. <em>PhAp2A</em> and <em>pMADS3</em> antisense mRNA and sense mRNA (control) probes were <sup>35</sup>S-labeled mRNA probes synthesized from this PCR fragment by using the T7 and T3 RNA polymerase (Promega). An ∼300-bp fragment
                     (nucleotides 198 to 596 of the <em>PhAp2B</em> cDNA sequence shown in  <a href="#F3">Fig 3</a>) and a 300-bp fragment were amplified from genomic subclones of <em>PhAp2B</em> and <em>PhAp2C</em> by using the T7 primer and a degenerate, chimeric T3-exon 2 PhAp2 primer (5'-AATAACCCTCACTAAAGCAG/TATG/ATGAGATTCCC-3'). <sup>35</sup>S-labeled <em>PhAp2B</em> and <em>PhAp2C</em> antisense mRNA and sense mRNA (control) probes were synthesized from these fragments by using the T3 and T7 RNA polymerase.
                     The in situ hybridization itself was performed as described by  Angerer and Angerer 1992 <a href="#R46"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>. Hybridized slides were washed for 1 hr at room temperature in 2 <span style="font-family: ariel,helvetica;  ">x</span> SSC and 0.01 M DTT and for 1 hr at 48°C in 0.1 <span style="font-family: ariel,helvetica;  ">x</span> SSC, 0.01 M DTT, and 50% formamide. Dehydrated slides were dipped into photographic emulsion and exposed for 6 to 10 days
                     before development. 
                     
                  </p>
                  
                  <p>
                     
                     <strong>Transposon Insertion Mutagenesis</strong><br/>
                     
                     The screening for <em>dTph1</em> insertion mutants was performed as described by  Koes et al. 1995 <a href="#R33"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>. The primers used to screen for <em>dTph1</em> insertions in <em>PhAp2A</em> were as follows: p002, 5'-GTGCCGTGTCAAATC-3'; p003, 5'-CCGGAATTCACAATATCGTGGAGTTACCTT-3'; p004, 5'-CGGGATCCCTGATCTGCCCCATTGTCTGTA-3';
                     and p005, 5'-CGGGATCCCTTCAAAGAGACTGGAGAATGT-3'. The presence of a <em>dTph1</em> element in one or both alleles of individual plants was determined by PCR analysis using a set of two gene-specific primers.
                     p001 (5'-GGTAAACGAGTCGGATCTTTTTCGAC-3') and p010 (5'-CGGTACCAATGTGGGATC-TAAATGATTC-3') were used in combination with p002
                     to analyze the progeny of the primary V2025-6 and V2116-10 mutants, respectively. The PCRs were performed in 50 µL of 1 <span style="font-family: ariel,helvetica;  ">x</span> PCR buffer (Perkin-Elmer) supplied with MgCl<sub>2</sub> (1.5 mM) and 1 unit of Taq polymerase (Perkin-Elmer) and consisted of 35 cycles of 30 sec denaturing at 94°C, 30 sec annealing
                     at 55°C, and 1.5 min polymerization at 72°C. DNA fragments flanking the <em>dTph1</em> elements in <em>PhAp2A</em> were amplified by PCR, cloned in pGEM-T, and sequenced to determine their exact insertion positions. 
                     
                  </p>
                  
                  <p>
                     
                     <strong>Reverse Transcription–PCR</strong><br/>
                     
                     Total RNA from flower buds was treated with DNase, extracted with phenol/chloroform, and precipitated, and 1 µg was reverse
                     transcribed from an oligo-dT primer by using the Superscript preamplification system for first-strand synthesis (Gibco BRL).
                     The first-strand cDNA was used as a template for PCR amplification with either p001/p002 (V2025-6) or p010/p002 (V2116-10).
                     
                     
                  </p>
                  
                  <p>
                     
                     <strong>Overexpression of PhAp2A in <em>Arabidopsis thaliana</em></strong><br/>
                     
                     Primers naste22 (5'-GAGATCTCAATGTGGGATCTAAATGATTC-3') and naste23 (5'-CGAATTCGGTCATGAGGGTCTCATAAGAGA-3') were used to amplify
                     the <em>PhAp2A</em> coding sequences. The fragment obtained was cloned in pGEM-T (Promega), and the sequence was verified. An ∼1600-bp <em>PhAp2A</em> fragment was cut from this plasmid by using the BglII site in primer naste22 and the HincII site from the polylinker of pGEM-T.
                     The binary vector pBIN m-gfp5-ER (kindly provided by James Haseloff, Cambridge, UK) was digested with SacI, blunted, digested
                     with BamHI, and ligated to the ∼1600-bp BglII-HincII <em>PhAp2A</em> fragment to generate pBIN19-35S-<em>PhAp2A</em>. 
                     
                  </p>
                  
                  <p>
                     
                     The resulting overexpression vector pBIN19-35S-<em>PhAp2A</em> was introduced into <em>Agrobacterium tumefaciens</em> C58C1(Rif) (pMP90[Gent]) by triparental mating. The Arabidopsis <em>ap2-1</em> mutant (kindly provided by Maarten Koornneef, LUW, Wageningen) was transformed as described by  Clarke et al. 1992 <a href="#R47"><img alt="Down" width="8" height="7" src="/icons/ref-arrow.gif"/></a>. Kanamycin-resistant calli and shoots were selected at 23°C, and shoots and selfed progeny of primary transformants were
                     grown at 26°C under long-day conditions (16 hr light and 8 hr dark). 
                     
                  </p>
                  
                  <p>
                     
                     
                     
                  </p>
                  
                  <p>
                     
                     
                     
                  </p>
                  
                  <p>
                     
                     <a name="FOOTNOTES" id="FOOTNOTES"/>
                     <br/></p><table style="background-color: #e1e1e1;width: 100%;"><tr><td style="background-color: #ffffff;width: 5%;"><img width="10" height="21" alt="*" src="/icons/toc/rarrow.gif"/></td>
                           <th style="width: 95%;" align="left"><span style="  font-size: 120%">
                                 FOOTNOTES</span></th>
                        </tr></table><p>
                     
                     
                     <sup>1</sup> Current address: Centro de Investigacion y Desarrollo, Consejo Superior de Investigacion Científica, Jordi-Girona 18-26,
                     08034 Barcelona, Spain. <a href="#FNR1"><img alt="*" width="11" height="9" src="/icons/toc/uarrow.gif"/></a><br/></p>
                  
                  <p>
                     
                     <a name="ACKNOWLEDGMENTS" id="ACKNOWLEDGMENTS"/>
                     <br/></p><table style="background-color: #e1e1e1;width: 100%;"><tr><td style="background-color: #ffffff;width: 5%;"><img width="10" height="21" alt="*" src="/icons/toc/rarrow.gif"/></td>
                           <th style="width: 95%;" align="left"><span style="  font-size: 120%">
                                 ACKNOWLEDGMENTS</span></th>
                        </tr></table><p>
                     
                     
                     We thank Bart den Boer for starting the petunia work on <em>Ap2</em> by isolating cPhAp2A-1, Wilson Ardiles and Raimundo Villaroel for sequencing, Gerco Angenent for the kind gift of the <em>pMADS3</em> plasmid, Ronald Koes and Erik Souer for making their 3D screening libraries and the inflorescence cDNA library available,
                     the people of the microscopy group of Gilbert Engler for advice concerning in situ hybridizations, and Mario Pezzotti for
                     the carpel-specific cDNA library. T.M. and N.V.d.S. were supported by the Instituut voor Wetenschap en Technologie. 
                     
                  </p>
                  
                  <p>
                     
                     Received September 22, 2000; accepted December 7, 2000.
                     
                  </p>
                  
                  <p>
                     
                     <a name="REFERENCES" id="REFERENCES"/>
                     <br/></p><table style="background-color: #e1e1e1;width: 100%;"><tr><td style="background-color: #ffffff;width: 5%;"><img width="10" height="21" alt="*" src="/icons/toc/rarrow.gif"/></td>
                           <th style="width: 95%;" align="left"><span style="  font-size: 120%">
                                 REFERENCES</span></th>
                        </tr></table><table><tr><th align="left"><span style="  font-size: 83.33%"><a href="#top"><img alt="*" width="11" height="9" src="/icons/toc/uarrow.gif"/>TOP<br/></a><a href="#ABSTRACT"><img alt="*" width="11" height="9" src="/icons/toc/uarrow.gif"/>ABSTRACT<br/></a><a href="#INTRODUCTION"><img alt="*" width="11" height="9" src="/icons/toc/uarrow.gif"/>INTRODUCTION<br/></a><a href="#RESULTS"><img alt="*" width="11" height="9" src="/icons/toc/uarrow.gif"/>RESULTS<br/></a><a href="#DISCUSSION"><img alt="*" width="11" height="9" src="/icons/toc/uarrow.gif"/>DISCUSSION<br/></a><a href="#METHODS"><img alt="*" width="11" height="9" src="/icons/toc/uarrow.gif"/>METHODS<br/></a><img alt="*" width="11" height="9" src="/icons/toc/dot.gif"/><span style=" color: 464c53; ">REFERENCES</span><br/></span></th>
                        </tr></table><p>
                     
                     
                     <a name="R46" id="R46"/>
                     <strong>Angerer, L.M., and Angerer, R.C.</strong> (1992) <em>In situ</em> hybridization to cellular RNA with radiolabeled RNA probes. In Wilkinson D.G., ed. In Situ Hybridization: A Practical Approach.
                     Oxford, UK, Oxford University Press. 14–32.pp.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R31" id="R31"/>
                     <strong>Ballinger, D.G., and Benzer, S.</strong> (1989) Targeted gene mutations in <em>Drosophila</em>. Proc. Natl. Acad. Sci. USA <strong>86</strong>:9402-9406<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=86/23/9402">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R35" id="R35"/>
                     <strong>Bowman, J.L., Smyth, D.R., and Meyerowitz, E.M.</strong> (1989) Genes directing flower development in Arabidopsis. Plant Cell <strong>1</strong>:37-52<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=1/1/37">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R10" id="R10"/>
                     <strong>Bowman, J.L., Alvarez, J., Weigel, D., Meyerowitz, E.M., and Smyth, D.R.</strong> (1993) Control of flower development in <em>Arabidopsis thaliana</em> by APETALA1 and interacting genes. Development <strong>119</strong>:721-743<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=119/3/721">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R3" id="R3"/>
                     <strong>Bradley, D., Carpenter, R., Sommer, H., Hartley, N., and Coen, E.</strong> (1993) Complementary floral homeotic phenotypes result from opposite orientations of a transposon at the <em>plena</em> locus of <em>Antirrhinum</em>. Cell <strong>72</strong>:85-95<a href="/cgi/external_ref?access_num=10.1016%2F0092-8674%2893%2990052-R&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=A1993KG95500012&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=8093684&amp;link_type=MED">[Medline]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R26" id="R26"/>
                     <strong>Chuck, G., Meeley, R.B., and Hake, S.</strong> (1998) The control of maize spikelet meristem fate by the <em>APETALA2</em>-like gene <em>indeterminate spikelet1</em>. Genes Dev. <strong>12</strong>:1145-1154<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=genesdev&amp;resid=12/8/1145">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R47" id="R47"/>
                     <strong>Clarke, M.C., Wei, W., and Lindsey, K.</strong> (1992) High frequency transformation of <em>Arabidopsis thaliana</em> by <em>Agrobacterium tumefaciens</em>. Plant Mol. Biol. Rep. <strong>10</strong>:178-189.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R1" id="R1"/>
                     <strong>Coen, E.S., and Meyerowitz, E.M.</strong> (1991) The war of the whorls: Genetic interactions controlling flower development. Nature <strong>353</strong>:31-37<a href="/cgi/external_ref?access_num=10.1038%2F353031a0&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=1715520&amp;link_type=MED">[Medline]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R38" id="R38"/>
                     <strong>Daniell, T.J., Fordham-Skelton, A.P., Vergani, P., and Edwards, R.</strong> (1996). Isolation of a maize cDNA (Accession No. Z47554) (PGR gb-013) encoding APETALA2-like binding domains by complementation
                     cloning of an L-isoaspartyl methyltransferase-deficient mutant of escherichia coli. Plant Physiol. <strong>110,</strong> 1435.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R13" id="R13"/>
                     <strong>Drews, G.N., Bowman, J.L., and Meyerowitz, E.M.</strong> (1991) Negative regulation of the <em>Arabidopsis</em> homeotic gene <em>Agamous</em> by the <em>Apetala2</em> product. Cell <strong>65</strong>:991-1002<a href="/cgi/external_ref?access_num=10.1016%2F0092-8674%2891%2990551-9&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=A1991FR04700010&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=1675158&amp;link_type=MED">[Medline]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R23" id="R23"/>
                     <strong>Elliott, R.C., Betzner, A.S., Huttner, E., Oakes, M.P., Tucker, W.Q.J., Gerentes, D., Perez, P., and Smyth, D.R.</strong> (1996) AINTEGUMENTA, an APETALA2-like gene of Arabidopsis with pleiotropic roles in ovule development and floral organ growth.
                     Plant Cell <strong>8</strong>:155-168<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=8/2/155">[Abstract]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R36" id="R36"/>
                     <strong>Forde, B.G., Heyworth, A., Pywell, J., and Kreis, M.</strong> (1985) Nucleotide sequence of a B1 hordein gene and the identification of possible upstream regulatory elements in endosperm
                     storage proteins from barley, wheat and maize. Nucleic Acids Res. <strong>13</strong>:7327-7339<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=nar&amp;resid=13/20/7327">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R34" id="R34"/>
                     <strong>Gerats, A.G.M., Huits, H., Vrijlandt, E., Maraña, C., Souer, E., and Beld, M.</strong> (1990) Molecular characterization of a nonautonomous transposable element (<em>dTph1</em>) of <em>Petunia</em>. Plant Cell <strong>2</strong>:1121-1128<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=2/11/1121">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R45" id="R45"/>
                     <strong>Gerats, T., De Keukeleire, P., Deblaere, R., van Montagu, M., and Zethof, J.</strong> (1995) Amplified fragment length polymorphism (AFLP) mapping in <em>Petunia</em>: A fast and reliable method for obtaining a genetic map. Acta Hortic. <strong>420</strong>:58-61.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R15" id="R15"/>
                     <strong>Goodrich, H.J., Puangsomlee, P., Martin, M., Long, D., Meyerowitz, E.M., and Coupland, G.</strong> (1997) A polycomb group gene regulates homeotic gene expression in <em>Arabidopsis</em>. Nature <strong>386</strong>:44-51<a href="/cgi/external_ref?access_num=10.1038%2F386044a0&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=9052779&amp;link_type=MED">[Medline]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R42" id="R42"/>
                     <strong>Henikoff, S.</strong> (1984) Unidirectional digestion with exonuclease III creates targeted breakpoints for DNA sequencing. Gene <strong>28</strong>:351-359<a href="/cgi/external_ref?access_num=10.1016%2F0378-1119%2884%2990153-7&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=A1984TA27900009&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=6235151&amp;link_type=MED">[Medline]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R40" id="R40"/>
                     <strong>Huits, H.S.M., Wijsman, H.J.W., Koes, R.E., and Gerats, A.G.M.</strong> (1995) Genetic characterization of <em>Act1</em>, the activator of a non-autonomous transposable element from <em>Petunia hybrida</em>. Theor. Appl. Genet. <strong>91</strong>:110-117.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R9" id="R9"/>
                     <strong>Irish, V., and Sussex, I.M.</strong> (1990) Function of the <em>Apetala</em>-1 gene during <em>Arabidopsis</em> floral development. Plant Cell <strong>2</strong>:741-753<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=2/8/741">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R16" id="R16"/>
                     <strong>Jofuku, K.D., den Boer, B.G.W., Van Montagu, M., and Okamuro, J.K.</strong> (1994) Control of Arabidopsis flower and seed development by the homeotic gene <em>Apetala2</em>. Plant Cell <strong>6</strong>:1211-1225<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=6/9/1211">[Abstract]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R32" id="R32"/>
                     <strong>Kaiser, K., and Goodwin, S.F.</strong> (1990) "Site-selected" transposon mutagenesis of <em>Drosophila</em>. Proc. Natl. Acad. Sci. USA <strong>87</strong>:1686-1690<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=87/5/1686">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R29" id="R29"/>
                     <strong>Kater, M.M., Colombo, L., Franken, J., Busscher, M., Masiero, S., Van Lookeren Campagne, M.M., and Angenent, G.C.</strong> (1998) Multiple AGAMOUS homologs from cucumber and petunia differ in their ability to induce reproductive organ fate. Plant
                     Cell <strong>10</strong>:171-182<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=10/2/171">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R24" id="R24"/>
                     <strong>Klucher, K.M., Chow, H., Reiser, L., and Fischer, R.L.</strong> (1996) The AINTEGUMENTA gene of Arabidopsis required for ovule and female gametophyte development is related to the floral
                     homeotic gene APETALA2. Plant Cell <strong>8</strong>:137-153<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=8/2/137">[Abstract]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R33" id="R33"/>
                     <strong>Koes, R. et al.</strong> (1995) Targeted gene inactivation in <em>Petunia</em> by PCR based selection of transposon insertion mutants. Proc. Natl. Acad. Sci. USA <strong>92</strong>:8149-8153<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=92/18/8149">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R11" id="R11"/>
                     <strong>Komaki, M.K., Okada, K., Nishino, E., and Shimura, Y.</strong> (1988) Isolation and characterization of novel mutants of <em>Arabidopsis thaliana</em> defective in flower development. Development <strong>104</strong>:195-203<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=104/2/195">[Abstract]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R12" id="R12"/>
                     <strong>Kunst, L., Klenz, E., Martinez-Zapater, J., and Haughn, G.W.</strong> (1989) <em>Ap</em>2 gene determines the identity of perianth organs in flowers of <em>Arabidopsis thaliana</em>. Plant Cell <strong>1</strong>:1195-1208<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=1/12/1195">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R17" id="R17"/>
                     <strong>Leon-Kloosterziel, K.M., Keijzer, C.J., and Koornneef, M.</strong> (1994) A seed shape mutant of Arabidopsis that is affected in integument development. Plant Cell <strong>6</strong>:385-392<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=6/3/385">[Abstract]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R14" id="R14"/>
                     <strong>Liu, Z., and Meyerowitz, E.M.</strong> (1995) LEUNIG regulates AGAMOUS expression in <em>Arabidopsis</em> flowers. Development <strong>121</strong>:975-991<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=121/4/975">[Abstract]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R6" id="R6"/>
                     <strong>Mandel, M.A., Bowman, J.L., Kempin, S.A., Ma, H., Meyerowitz, E.M., and Yanofsky, M.F.</strong> (1992) Manipulation of flower structure in transgenic tobacco. Cell <strong>71</strong>:133-143<a href="/cgi/external_ref?access_num=10.1016%2F0092-8674%2892%2990272-E&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=A1992JQ62300013&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=1356631&amp;link_type=MED">[Medline]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R2" id="R2"/>
                     <strong>Mizukami, Y., and Ma, H.</strong> (1992) Ectopic expression of the floral homeotic gene AGAMOUS in transgenic <em>Arabidopsis</em> plants alters floral organ identity. Cell <strong>71</strong>:119-131<a href="/cgi/external_ref?access_num=10.1016%2F0092-8674%2892%2990271-D&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=A1992JQ62300012&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=1356630&amp;link_type=MED">[Medline]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R18" id="R18"/>
                     <strong>Modrusan, Z., Reiser, L., Feldman, K.A., Fisher, R.L., and Haughn, G.W.</strong> (1994) Homeotic transformation of ovules into carpel-like structures in Arabidopsis. Plant Cell <strong>6</strong>:333-349<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=6/3/333">[Abstract]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R25" id="R25"/>
                     <strong>Moose, S.P., and Sisco, P.H.</strong> (1996) GLOSSY15, an APETALA2- like gene from maize that regulates leaf epidermal cell identity. Genes Dev. <strong>10</strong>:3018-3027<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=genesdev&amp;resid=10/23/3018">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R28" id="R28"/>
                     <strong>Motte, P., Saedler, H., and Schwarz-Sommer, Z.</strong> (1998) <em>STYLOSA</em> and <em>FISTULATA</em>: Regulatory components of the homeotic control of <em>Antirrhinum</em> floral organogenesis. Development <strong>125</strong>:71-84<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=125/1/71">[Abstract]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R21" id="R21"/>
                     <strong>Ohme-Takagi, M., and Shinshi, H.</strong> (1995) Ethylene-inducible DNA binding proteins that interact with an ethylene-responsive element. Plant Cell <strong>7</strong>:173-182<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=7/2/173">[Abstract]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R19" id="R19"/>
                     <strong>Okamuro, J.K., Caster, B., Villaroel, R., Van Montagu, M., and Jofuku, K.D.</strong> (1997) The AP2 domain of Apetala2 defines a large new family of DNA binding proteins in <em>Arabidopsis</em>. Proc. Natl. Acad. Sci. USA <strong>94</strong>:7076-7081<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=94/13/7076">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R39" id="R39"/>
                     <strong>Pagan-Westphal, S.M., and Tabin, C.J.</strong> (1998) The transfer of left-right positional information during chick embryogenesis. Cell <strong>93</strong>:25-35<a href="/cgi/external_ref?access_num=10.1016%2FS0092-8674%2800%2981143-5&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000072910800007&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=9546389&amp;link_type=MED">[Medline]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R5" id="R5"/>
                     <strong>Pnueli, L., Hareven, D., Rounsley, S.D., Yanofsky, M.F., and Lifschitz, E.</strong> (1994) Isolation of the tomato AGAMOUS gene TAG1 and analysis of its homeotic role in transgenic plants. Plant Cell <strong>6</strong>:163-173<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=6/2/163">[Abstract]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R20" id="R20"/>
                     <strong>Riechman, J.L., and Meyerowitz, E.M.</strong> (1998) The AP2/EREBP family of plant transcription factors. Biol. Chem. <strong>379</strong>:633-646<a href="/cgi/external_ref?access_num=000074551300002&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=9687012&amp;link_type=MED">[Medline]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R8" id="R8"/>
                     <strong>Samach, A., Kohalmi, S.E., Motte, P., Datta, R., and Haughn, G.W.</strong> (1997) Divergence of function and regulation of class B floral organ identity genes. Plant Cell <strong>9</strong>:559-570<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=9/4/559">[Abstract]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R43" id="R43"/>
                     <strong>Sanger, F., Nicklen, S., and Coulson, A.R.</strong> (1977) DNA sequencing with chain-terminating inhibitors. Proc. Natl. Acad. Sci. USA <strong>74</strong>:5463-5467<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=74/12/5463">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R41" id="R41"/>
                     <strong>Souer, E., Quattrocchio, F., de Vetten, N., Mol, J., and Koes, R.</strong> (1995) A general method to isolate genes tagged by a high-copy- number transposable element. Plant J. <strong>7</strong>:677-685<a href="/cgi/external_ref?access_num=10.1046%2Fj.1365-313X.1995.7040677.x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=A1995QU55800013&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=7742861&amp;link_type=MED">[Medline]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R44" id="R44"/>
                     <strong>Stam, P.</strong> (1993) Construction of integrated genetic linkage maps by means of a new computer package: JoinMap. Plant J. <strong>3</strong>:739-744.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R22" id="R22"/>
                     <strong>Stockinger, E.J., Gilmour, S.J., and Thomashow, M.F.</strong> (1997) <em>Arabidopsis thaliana</em> CBF1 encodes an AP2 domain–containing transcriptional activator that binds to the C repeat/DRE, a cis-acting DNA regulatory
                     element that stimulates transcription in response to low temperature and water deficit. Proc. Natl. Acad. Sci. USA <strong>94</strong>:1035-1040<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=94/3/1035">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R4" id="R4"/>
                     <strong>Tsuchimoto, S., van der Krol, A.R., and Chua, N.-H.</strong> (1993) Ectopic expression of <em>pMADS3</em> in transgenic petunia phenocopies the petunia <em>blind</em> mutant. Plant Cell <strong>5</strong>:843-853<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=5/8/843">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R27" id="R27"/>
                     <strong>Vallade, J., Maizonnier, D., and Cornu, A.</strong> (1987) La morphogenèse floral chez le pétunia. I. Analyse d'un mutant à corolle staminée. Can. J. Bot. <strong>65</strong>:761-764.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R37" id="R37"/>
                     <strong>Vicente-Carbajosa, J., Moose, S.P., Parsons, R.L., and Schmidt, R.J.</strong> (1997) A maize zinc-finger protein binds the prolamin box in zein gene promoters and interacts with the basic leucine zipper
                     transcriptional activator Opaque2. Proc. Natl. Acad. Sci. USA <strong>94</strong>:7685-7690<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=94/14/7685">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R" id="R"/>
                     <strong>von Goethe, J.W.</strong> (1790) Versuch der Metamorphose der Pflanzen zu Erklären. Gotha, Germany, C.W. Ettinger.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R30" id="R30"/>
                     <strong>Vos, P., Hogers, R., Bleeker, M., Reijans, M., Van De Lee, T., Hornes, M., Frijters, A., Pot, J., Peleman, J., Kuiper, M.,
                        and Zabeau, M.</strong> (1995) AFLP: A new technique for DNA fingerprinting. Nucleic Acids Res. <strong>23</strong>:4407-4414<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=nar&amp;resid=23/21/4407">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a>.
                     
                  </p>
                  
                  <p>
                     
                     <a name="R7" id="R7"/>
                     <strong>Weigel, D., and Nilsson, O.</strong> (1995) A developmental switch sufficient for flower initiation in diverse plants. Nature <strong>377</strong>:495-500<a href="/cgi/external_ref?access_num=10.1038%2F377495a0&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=7566146&amp;link_type=MED">[Medline]</a>.
                     
                  </p>
                  
               </div>
      <span class="highwire-journal-article-marker-end"/>
    </div>
  </body>
</html>
