<?xml version="1.0"?>
<html>
  <body>
    <div itemprop="articleBody">
      <span class="highwire-journal-article-marker-start"/>
      <div class="hw-optimized-content">
                  <div class="auto-clean">
                     
                     
                     
                     
                     
                     	
                     	
                     	
                     	
                     	
                     	
                     
                     	
                     	 
                     	
                     	
                     	
                     		<span style="  font-size: 83.33%">
                        The Plant Cell, Vol. 15, 5-18, 
                        January 2003, Copyright Â© 2003,<br/><a href="/misc/terms.shtml">American Society of Plant Biologists</a></span><br/></div>
                  <h1 id="article-title-1" itemprop="headline">
                     LEAFY COTYLEDON1-LIKE Defines a Class of Regulators Essential for Embryo Development
                     
                     
                  </h1>
                  <div class="contributors">
                     Raymond W. Kwong<sup>a</sup><sup>,b</sup>, 
                     Anhthu Q. Bui<a name="RFN1" id="RFN1"/><sup><a href="#FN1">1</a></sup><sup>,c</sup>, 
                     Hyeseung Lee<a/><sup><a href="#FN1">1</a></sup><sup>,a</sup><sup>,b</sup>, 
                     Linda W. Kwong<sup>a</sup>, 
                     Robert L. Fischer<sup>d</sup>, 
                     Robert B. Goldberg<sup>c</sup> and 
                     John J. Harada<sup>2</sup><sup>,a</sup><sup>,b</sup></div>
                  <p>
                     <span style="  font-size: 83.33%"><sup>a</sup> Section of Plant Biology, Division of Biological Sciences, One Shields Avenue, University of California, Davis, California
                        95616<br/><sup>b</sup> Graduate Group in Plant Biology, University of California, Davis, California 95616<br/><sup>c</sup> Department of Molecular, Cell, and Developmental Biology, University of California, Los Angeles, California 90024-1606<br/><sup>d</sup> Department of Plant and Microbial Biology, University of California, Berkeley, California 94720
                        </span></p>
                  <p>
                     
                     <span style="  font-size: 83.33%"><sup>2</sup> To whom correspondence should be addressed. E-mail <span class="em-link"><span class="em-addr">jjharada{at}ucdavis.edu</span></span>; fax 530-752-5410</span></p>
                  <p>
                     
                     <a name="ABS" id="ABS"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        Abstract
                        
                        
                     </h2>
                     
                  </div>
                  <div class="section abstract" id="abstract-1" itemprop="description">
                     
                     <p>
                        Arabidopsis LEAFY COTYLEDON1 (LEC1) is a critical regulator required for normal development during the early and late phases
                        of embryogenesis that is sufficient to induce embryonic development in vegetative cells. <em>LEC1</em> encodes a HAP3 subunit of the CCAAT binding transcription factor. We show that the 10 Arabidopsis HAP3 (AHAP3) subunits can
                        be divided into two classes based on sequence identity in their central, conserved B domain. LEC1 and its most closely related
                        subunit, LEC1-LIKE (L1L), constitute LEC1-type AHAP3 subunits, whereas the remaining AHAP3 subunits are designated non-LEC1-type.
                        Similar to <em>LEC1</em>, <em>L1L</em> is expressed primarily during seed development. However, suppression of <em>L1L</em> gene expression induced defects in embryo development that differed from those of <em>lec1</em> mutants, suggesting that LEC1 and L1L play unique roles in embryogenesis. We show that <em>L1L</em> expressed under the control of DNA sequences flanking the <em>LEC1</em> gene suppressed genetically the <em>lec1</em> mutation, suggesting that the LEC1-type B domains of L1L and LEC1 are critical for their function in embryogenesis. Our results
                        also suggest that LEC1-type HAP3 subunits arose from a common origin uniquely in plants. Thus, L1L, an essential regulator
                        of embryo development, defines a unique class of plant HAP3 subunits. 
                        
                     </p>
                     
                  </div>
                  <div class="auto-clean">
                     
                     
                     <a name="SEC1" id="SEC1"/>
                     
                     
                  </div>
                  <div class="section-nav">
                     
                     <h2>
                        INTRODUCTION
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     The single-celled zygote of a flowering plant undergoes a series of controlled cell divisions and cell differentiation events
                     that lead to the formation of a mature, multicellular embryo that is metabolically quiescent and desiccated. Early in embryogenesis,
                     during the morphogenesis phase, the plant body is formed through the establishment of the shoot-root axis and the formation
                     of the embryonic tissue and organ systems (West and Harada, 1993<a href="#BIB47"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Goldberg et al., 1994<a href="#BIB15"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Laux and Jurgens, 1997<a href="#BIB26"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Jurgens, 2001<a href="#BIB22"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Later, during the seed maturation phase, the embryo acquires the ability to withstand desiccation, accumulates storage
                     macromolecules such as lipids and proteins, and becomes metabolically quiescent as a result of desiccation (reviewed by Bewley,
                     1997<a href="#BIB4"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Harada, 1997<a href="#BIB18"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Once environmental conditions are favorable, the seed germinates and the vegetative phase of the life cycle begins. 
                     
                  </p>
                  <p>
                     
                     Genetic studies have identified regulatory genes that play critical roles in embryogenesis during either the morphogenesis
                     or the maturation phases. For example, genes such as <em>WUSCHEL</em>, <em>SHOOTMERISTEMLESS</em>, <em>SCARECROW</em>, and <em>SHORT ROOT</em> (Dilaurenzio et al., 1996<a href="#BIB7"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Long et al., 1996<a href="#BIB30"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Mayer et al., 1998<a href="#BIB34"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Helariutta et al., 2000<a href="#BIB21"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) have been shown to be essential for the formation of the shoot and root apical meristems that define the embryonic axis
                     of developing Arabidopsis embryos. A different class of genes, including <em>ABSCISIC ACID INSENSITIVE3</em> (<em>ABI3</em>), <em>ABI4</em>, and <em>ABI5</em>, play important roles during the maturation phase of embryogenesis (Giraudat et al., 1992<a href="#BIB13"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Finkelstein et al., 1998<a href="#BIB11"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Finkelstein and Lynch, 2000<a href="#BIB10"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), preparing the embryo for desiccation and postgerminative growth. 
                     
                  </p>
                  <p>
                     
                     Another set of genes encoding Arabidopsis LEAFY COTYLEDON (LEC) proteins, LEC1, LEC2, and FUSCA3, are unique in that they
                     are the only known embryonic regulators required for normal development during both the morphogenesis and maturation phases
                     (reviewed by Harada, 2001<a href="#BIB19"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). For example, LEC1 is required to maintain suspensor cell fate, to specify cotyledon identity in the early morphogenesis
                     phase, and to initiate and/or maintain the maturation phase and inhibit precocious germination late in embryogenesis (Meinke,
                     1992<a href="#BIB35"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Meinke et al., 1994<a href="#BIB36"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; West et al., 1994<a href="#BIB48"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Parcy et al., 1997<a href="#BIB38"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Lotan et al., 1998<a href="#BIB31"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Vicient et al., 2000<a href="#BIB46"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Furthermore, ectopic postembryonic expression of <em>LEC1</em> is sufficient to confer embryonic characteristics to seedlings and to induce somatic embryo formation from vegetative cells
                     (Lotan et al., 1998<a href="#BIB31"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Because LEC1 is required for normal development both early and late during embryogenesis and is sufficient to confer embryogenic
                     competence to vegetative cells, it is a central regulator that acts far upstream in the regulatory hierarchy that controls
                     embryogenesis. We speculate that <em>LEC1</em> establishes a cellular environment that promotes embryo development and that this environment coordinates the early and late
                     phases of embryogenesis in flowering plants (Lotan et al., 1998<a href="#BIB31"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). A major goal of our research is to understand, at a mechanistic level, how LEC1 establishes competence to initiate embryo
                     development. 
                     
                  </p>
                  <p>
                     
                     Given the key role of LEC1 in the control of embryogenesis, we asked if genes related to <em>LEC1</em> also encode embryonic regulators. LEC1 shares significant sequence similarity with the HAP3 subunit of CCAAT binding factor
                     (CBF, also known as NF-Y; Lotan et al., 1998<a href="#BIB31"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). CBFs are eukaryotic transcriptional activators that serve diverse roles in dif-ferent organisms (Li et al., 1992<a href="#BIB28"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). In yeast, CBF activates a set of genes involved in mitochondrial respiration (Guarente et al., 1984<a href="#BIB16"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Keng and Guarente, 1987<a href="#BIB23"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Trueblood et al., 1988<a href="#BIB45"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Schneider and Guarente, 1991<a href="#BIB40"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), whereas mammalian CBFs are thought to act generally to enhance transcription rates, often in combination with other proteins
                     (reviewed by Maity and De Crombrugghe, 1998<a href="#BIB32"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Mantovani, 1999<a href="#BIB33"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). The transcription factor is a hetero-oligomeric complex consisting of at least three subunits, HAP2, HAP3, and HAP5, although
                     yeast possesses a fourth subunit, HAP4 (reviewed by Maity and De Crombrugghe, 1998<a href="#BIB32"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Mantovani, 1999<a href="#BIB33"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). HAP3 subunits are recognized by their central B domain, an â¼90âamino acid region of the protein that is conserved across
                     eukaryotic organisms. For example, the LEC1 B domain has 57 and
                     62% sequence identity with HAP3 subunits from yeast and mammals, respectively. Thus, LEC1 appears to encode a subunit of a
                     transcription factor that regulates the expression of genes required for embryo development. 
                     
                  </p>
                  <p>
                     
                     We used the LEC1 polypeptide sequence to identify other genes encoding Arabidopsis HAP3 (AHAP3) subunits. We show that the
                     subunit most closely related to LEC1, designated LEC1-LIKE (L1L), is required for normal embryo development. L1L and LEC1
                     have distinct functions in embryogenesis, but L1L can substitute functionally for LEC1 when expressed ectopically. Comparison
                     of the deduced amino acid sequences of L1L and LEC1 identified specific amino acid sequences that appear to be required for
                     the function of these proteins in regulating embryo identity and development. 
                     
                  </p>
                  <p>
                     
                     <a name="SEC2" id="SEC2"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        RESULTS
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     <strong>Arabidopsis HAP3 Proteins Are Encoded by a Gene Family That Can Be Divided into Two Classes</strong><br/>
                     
                     We used the amino acid sequence of LEC1 as a query to identify related Arabidopsis polypeptides. Database searches of the
                     sequenced Arabidopsis genome showed that there are nine genes encoding proteins that share significant sequence identity and
                     that the gene encoding L1L (At5g47670) is related most closely to LEC1. As shown in <a href="#FIG1">Figure 1</a>, sequence similarity among the 10 proteins is limited primarily to the central B domain, consistent with comparisons of HAP3
                     subunits from other organisms (Li et al., 1992<a href="#BIB28"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Only two of these putative proteins, At2g37060 and At3g53340, display sequence identity with each other in the N-terminal
                     A domain or the C-terminal C domain. Because the B domain has been shown to underlie HAP3 function in other organisms (Xing
                     et al., 1993<a href="#BIB51"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Kim et al., 1996<a href="#BIB24"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Sinha et al., 1996<a href="#BIB41"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) and because all of these Arabidopsis proteins possess residues that are conserved among HAP3 proteins, we conclude that
                     there are 10 AHAP3 subunits. 
                     
                  </p>
                  <p>
                     
                     <a name="FIG1" id="FIG1"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="5/F1.expansion.html"><img src="5/F1.small.gif"/></a><br/><strong>View larger version</strong> (79K):<br/><a href="5/F1.expansion.html">In this window</a><br/><a href="5/F1.expansion.html" class="in-nw">In a new window</a><br/>Â 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 1.</strong></span> Analysis of Arabidopsis HAP3 Subunits.
                        
                        <p>
                           
                           Amino acid sequence alignment of AHAP3 proteins. Residues highlighted in black and gray represent identical and similar amino
                           acids, respectively. B-domain residues shared between LEC1 (At1g21970) and L1L (At5g47670) but not with the other proteins
                           are highlighted in red. The B domain is underlined.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     Close examination of B-domain sequence alignments showed that L1L and LEC1 define a distinct class of AHAP3 subunits. The
                     two proteins share 83% sequence identity with each other but only 52 to 71% identity with the other eight AHAP3 subunits.
                     Furthermore, L1L and LEC1 share the amino acid residues highlighted in red in <a href="#FIG1">Figure 1</a> that differ from the residues that are conserved in the other eight AHAP3 subunits. On the basis of sequence identity within
                     the B domain, we define two classes of AHAP3 subunits: the LEC1-type and the non-LEC1-type. Thus, L1L is the AHAP3 most closely
                     related to LEC1, opening the possibility that L1L also may be an embryonic regulator. 
                     
                  </p>
                  <p>
                     
                     <strong><em>L1L</em> RNA Accumulates Primarily in Developing Embryos</strong><br/>
                     
                     We analyzed <em>L1L</em> gene expression to obtain clues about its role during development. <em>L1L</em> RNA was detected in RNA gel blot hybridization experiments in developing siliques but not in vegetative organs or in flowers,
                     as shown in <a href="#FIG2">Figure 2A</a>. This pattern of RNA accumulation closely resembled that of <em>LEC1</em>, which accumulates specifically in seeds and differed substantially from those of non-LEC1-type AHAP3 subunits, whose RNAs
                     are not limited to seed development (Edwards et al., 1998<a href="#BIB9"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Lotan et al., 1998<a href="#BIB31"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Gusmaroli et al., 2001<a href="#BIB17"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; M. Kim and J.J. Harada, unpublished results). However, differences between <em>L1L</em> and <em>LEC1</em> RNA accumulation patterns were observed. <a href="#FIG2">Figure 2A</a> shows that <em>L1L</em> RNA levels peaked at a later stage of embryogenesis than did <em>LEC1</em> levels. Furthermore, sensitive reverse transcriptaseâmediated (RT) PCR amplification experiments indicated that <em>L1L</em> RNA is present in all vegetative organs, although presumably at low levels, as shown in <a href="#FIG2">Figure 2B</a>. By contrast, <em>LEC1</em> RNA was not detected in vegetative organs in parallel experiments. These results suggest that L1L is likely to function primarily
                     during seed development and that it is expressed differently from <em>LEC1</em>. 
                     
                  </p>
                  <p>
                     
                     <a name="FIG2" id="FIG2"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="5/F2.expansion.html"><img src="5/F2.small.gif"/></a><br/><strong>View larger version</strong> (62K):<br/><a href="5/F2.expansion.html">In this window</a><br/><a href="5/F2.expansion.html" class="in-nw">In a new window</a><br/>Â 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 2.</strong></span> <em>L1L</em> RNA Is Detected Predominantly in Developing Siliques.
                        
                        <p>
                           
                           <strong>(A)</strong> Analysis of <em>L1L</em> and <em>LEC1</em> RNA levels with RNA gel blot hybridization experiments. Each lane contained 1 Âµg of poly(A) RNA from siliques with zygote-
                           to early-globular-stage seeds (S1), siliques with globular- to heart-stage seeds (S2), siliques with torpedo- to bent-cotyledon-stage
                           seeds (S3), siliques with mature green seeds (S4), 2-day-old seedlings (Sl), mature rosette leaves (Le), 3-week-old seedling
                           roots (Ro), stems (St), and unopened floral buds and inflorescences (Fl). Control represents the accumulation of a ribosomal
                           protein RNA.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(B)</strong> RT-PCR amplification of <em>L1L</em> RNA. Abbreviations are as in <strong>(A)</strong> with the following additions: ND, no DNA; GD, wild-type genomic DNA; and <em>lec1-1</em>, mutant siliques with torpedo- to bent-cotyledon-stage embryos.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     We localized <em>L1L</em> RNA using in situ hybridization experiments to determine where it functions in developing siliques. As shown in <a href="#FIG3">Figures 3A to 3C and 3E to 3G</a>, <em>L1L</em> RNA was detected at low but statistically significant levels in the developing embryo proper, suspensor, and endosperm at
                     early stages, including zygotes (data not shown). During the torpedo stage (<a href="#FIG3">Figures 3C and 3G</a>) and the linear cotyledon stage (<a href="#FIG3">Figures 3D and 3H</a>), <em>L1L</em> RNA became prevalent primarily in the outer cell layers of the embryo, similar to the distribution of <em>LEC1</em> RNA (Lotan et al., 1998<a href="#BIB31"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). <em>L1L</em> RNA became evenly distributed throughout the embryo at a high level by the bent-cotyledon stage (<a href="#FIG3">Figures 3I, 3J, 3M, and 3N</a>) and was present at low levels in mature-stage embryos (<a href="#FIG3">Figures 3K, 3L</a>, 3O, and 3P). This temporal pattern of RNA accumulation corresponds with the results from RNA gel blot analyses. Sense RNA
                     did not bind appreciably with the sections, showing the specificity of the hybridization reactions (data not shown). Together,
                     the RNA accumulation patterns suggest a role for <em>L1L</em> in embryogenesis. 
                     
                  </p>
                  <p>
                     
                     <a name="FIG3" id="FIG3"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="5/F3.expansion.html"><img src="5/F3.small.gif"/></a><br/><strong>View larger version</strong> (128K):<br/><a href="5/F3.expansion.html">In this window</a><br/><a href="5/F3.expansion.html" class="in-nw">In a new window</a><br/>Â 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 3.</strong></span> In Situ Detection of <em>L1L</em> RNA in Developing Embryos.
                        
                        <p>
                           
                           Wild-type seed sections were hybridized with an <em>L1L</em>-specific antisense probe. All sections were exposed for 10 days. <strong>(A)</strong> to <strong>(D)</strong> and <strong>(I)</strong> to <strong>(L)</strong> show bright-field micrographs, and <strong>(E)</strong> to <strong>(H)</strong> and <strong>(M)</strong> to <strong>(P)</strong> show dark-field micrographs. The sense RNA control did not bind appreciably with the sections. Bars = 50 Âµm.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(A)</strong> and <strong>(E)</strong> Globular-stage embryo.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(B)</strong> and <strong>(F)</strong> Heart-stage embryo.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(C)</strong> and <strong>(G)</strong> Linear cotyledon-stage embryo.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(D)</strong> and <strong>(H)</strong> Early bent-cotyledon-stage embryo.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(I)</strong> and <strong>(M)</strong> Bent-cotyledon-stage embryo.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(J)</strong> and <strong>(N)</strong> Late bent-cotyledon-stage embryo.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(K)</strong> and <strong>(O)</strong> Mature green-stage embryo.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(L)</strong> and <strong>(P)</strong> Mature yellowing-stage embryo.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     <strong><em>L1L</em> Is Required for Embryogenesis</strong><br/>
                     
                     Given that <em>L1L</em> is expressed primarily during embryogenesis, we used RNA interference (RNAi) experiments to determine if the suppression
                     of <em>L1L</em> RNA levels affected embryo development. Because <em>L1L</em> shares substantial identity with other AHAP3 subunits in the central B domain, <em>L1L</em>-specific nucleotide sequences encoding the C domain were used for targeted suppression (see Methods). Wild-type plants were
                     transformed with the <em>L1L</em> RNAi construct under the control of the <em>35S</em> promoter, and transgenic plants were recovered. Thirteen of 172 T1 transgenic lines produced defective T2 seeds. More specifically,
                     T1 plants from three independently derived lines segregated 30.1% (<em>n</em> = 1372), 31.9% (<em>n</em> = 1156), and 21.7% (<em>n</em> = 1327) defective T2 seeds. Although the RNAi construct was incompletely penetrant, these results suggest that L1L is required
                     for embryo development. We also found that 4 of 15 lines containing a <em>35S</em>:<em>L1L</em> transgene segregated defective seeds. Together, these results suggest that cosuppression of <em>L1L</em> gene expression induces defects in embryogenesis (see below). 
                     
                  </p>
                  <p>
                     
                     As shown in <a href="#FIG4">Figures 4B to 4D</a>, RNAi mutants arrested at a number of different embryonic stages with a range of morphological phenotypes. Some embryos arrested
                     at the globular stage (<a href="#FIG4">Figures 4B and 4C</a>) but had extra cells in the suspensor. Other mutants arrested at later embryonic stages and had reduced cotyledons (<a href="#FIG4">Figure 4D</a>). Seeds containing these defective embryos did not germinate, nor did immature seeds collected before desiccation germinate
                     in culture. However, the effects appeared to be limited to seed development, because no defects in the vegetative development
                     of the RNAi transgenic lines were detected. Suppression of <em>L1L</em> gene expression induced embryonic defects that differed from those caused by the <em>lec1</em> mutation (Lotan et al., 1998<a href="#BIB31"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     <a name="FIG4" id="FIG4"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="5/F4.expansion.html"><img src="5/F4.small.gif"/></a><br/><strong>View larger version</strong> (120K):<br/><a href="5/F4.expansion.html">In this window</a><br/><a href="5/F4.expansion.html" class="in-nw">In a new window</a><br/>Â 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 4.</strong></span> RNAi Suppression of <em>L1L</em> Gene Expression Induces Embryo Defects.
                        
                        <p>
                           
                           <strong>(A)</strong> Seed with a wild-type embryo at the bent-cotyledon stage. The seed was cleared and viewed with Nomarski optics.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(B)</strong> to <strong>(D)</strong> Cleared seeds containing defective embryos from lines containing the <em>L1L</em> RNAi constructs. Progeny segregating with a wild-type phenotype in the same silique were at the bent-cotyledon stage.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(E)</strong> to <strong>(H)</strong> <em>L1L</em> RNA accumulation in defective embryos. Sections were hybridized with an antisense <em>L1L</em> probe and exposed for 10 days.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(E)</strong> and <strong>(F)</strong> Bright- and dark-field micrographs of a defective embryo from a line containing the <em>L1L</em> RNAi construct.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(G)</strong> and <strong>(H)</strong> Bright- and dark-field micrographs of a wild-type embryo at the mature green stage.
                           
                        </p>
                        
                        <p>
                           
                           Bars = 50 Âµm in <strong>(A)</strong>, <strong>(D)</strong>, <strong>(E)</strong>, and <strong>(H)</strong> and 25 Âµm in <strong>(B)</strong> and <strong>(C)</strong>.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     To confirm that defects in embryo development resulted from the silencing of the <em>L1L</em> gene, we analyzed <em>L1L</em> RNA levels in transgenic lines using in situ hybridization experiments with <em>L1L</em>-specific probes that excluded sequences encoding the C domain. <em>L1L</em> RNA was not detected at significant levels in 60 of 62 embryos with a mutant phenotype (<a href="#FIG4">Figures 4E and 4F</a>) from four independent transgenic lines and was present only at a low level compared with the wild type in the 2 other mutant
                     embryos. Of embryos that segregated with a wild-type phenotype, 31% (<em>n</em> = 352) possessed high levels of <em>L1L</em> RNA, similar to embryos with a wild-type genotype (<a href="#FIG4">Figures 4G and 4H</a>), whereas the remainder had only intermediate levels. We interpret these results to indicate that very low levels of <em>L1L</em> RNA do not support embryo development but intermediate levels are sufficient for normal embryogenesis. The incomplete penetrance
                     and variable expressivity of RNAi suppression of <em>L1L</em> gene expression probably allowed us to recover viable progeny containing the transgene. We also demonstrated the specificity
                     of gene silencing by showing that <em>LEC1</em>, oleosin, and cruciferin storage protein RNAs were detected in RNAi embryos exhibiting a mutant phenotype as they were in
                     wild-type embryos (data not shown). Together, these data suggest strongly that <em>L1L</em> is essential for embryo development. 
                     
                  </p>
                  <p>
                     
                     <strong>Ectopically Expressed <em>L1L</em> Can Function in Place of <em>LEC1</em></strong><br/>
                     
                     To examine the functional relationship between L1L and LEC1, we asked if <em>L1L</em> could suppress the <em>lec1</em> mutation when expressed ectopically. <em>L1L</em> RNA was detected in <em>lec1-1</em> null mutants (<a href="#FIG2">Figure 2B</a>), indicating that the endogenous <em>L1L</em> gene cannot substitute completely for the <em>LEC1</em> gene. Although the spatial distribution of <em>L1L</em> RNA was similar to that of <em>LEC1</em>, there were temporal differences in accumulation during embryogenesis (<a href="#FIG2">Figures 2</a> and <a href="#FIG3">3</a>) (Lotan et al., 1998<a href="#BIB31"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Therefore, we fused the <em>L1L</em> coding region with 1997 and 774 bp of sequence 5' and 3', respectively, of the <em>LEC1</em> coding region and transferred the chimeric gene into <em>lec1-1</em> null mutants. The <em>lec1</em> mutation causes embryos to become intolerant of desiccation, and no mutant seeds germinate (Meinke, 1992<a href="#BIB35"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; West et al., 1994<a href="#BIB48"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Lotan et al., 1998<a href="#BIB31"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). As shown in <a href="#TBL1">Table 1</a> and <a href="#FIG5">Figure 5A</a>, transgenic <em>lec1-1</em> mutant seeds that were dried extensively (see Methods) produced viable seedlings, indicating that <em>L1L</em> expressed under the control of <em>LEC1</em> flanking DNA sequences was able to rescue the desiccation intolerance of <em>lec1</em> mutants. Moreover, no embryonic or postembryonic abnormalities were detected in <em>lec1</em> mutant plants containing the <em>L1L</em> transgene, and T1 plants segregated progeny with wild-type and mutant phenotypes at ratios indicating the presence of one,
                     two, or multiple transgenes (data not shown). By contrast, the expression of two genes encoding non-LEC1-type AHAP3 subunits,
                     At4g14540 and At3g53340, under the control of <em>LEC1</em> 5' and 3' flanking sequences did not rescue the desiccation intolerance of the <em>lec1</em> mutants significantly (<a href="#TBL1">Table 1</a>). Together, these results suggest that <em>L1L</em> but not non-LEC1-type <em>AHAP3</em> genes can function in place of <em>LEC1</em> when expressed ectopically. 
                     
                  </p>
                  <p>
                     
                     <a name="TBL1" id="TBL1"/>
                     
                     
                  </p>
                  <div class="table pos-float">
                     
                     <div class="table-inline">
                        <strong>View this table:</strong><br/><a href="5/T1.expansion.html">In this window</a><br/><a href="5/T1.expansion.html" class="in-nw">In a new window</a><br/>Â 
                        
                        
                     </div>
                     
                     <div class="table-caption">
                        <span class="fig-label"><strong>Table 1.</strong></span> Transgene Suppression of the <em>lec1-1</em> Mutation 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     <a name="FIG5" id="FIG5"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="5/F5.expansion.html"><img src="5/F5.small.gif"/></a><br/><strong>View larger version</strong> (121K):<br/><a href="5/F5.expansion.html">In this window</a><br/><a href="5/F5.expansion.html" class="in-nw">In a new window</a><br/>Â 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 5.</strong></span> Suppression of the <em>lec1</em> Mutation by <em>L1L</em>.
                        
                        <p>
                           
                           <strong>(A)</strong> A representative <em>lec1-1</em> seedling containing the <em>LEC1</em>:<em>L1L</em>:<em>LEC1</em> transgene that has survived seed desiccation. The transgene confers desiccation tolerance to <em>lec1</em> mutant embryos.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(B)</strong> A <em>lec1-1</em> seedling transformed with <em>35S</em>:<em>L1L</em>. The transgene allows <em>lec1-1</em> mutant embryos to withstand desiccation and confers embryonic characteristics to the seedling, including a lack of cotyledon
                           expansion, failure of hypocotyls and roots to extend, and production of cotyledon-like organs at the positions of leaves.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(C)</strong> and <strong>(D)</strong> Hybridization of cruciferin storage protein and oleosin probes, respectively, with embryo-like <em>35S</em>:<em>L1L</em> seedlings. Sections were exposed for 2 days.
                           
                        </p>
                        
                        <p>
                           
                           Bars <strong>=</strong> 300 Âµm in <strong>(A)</strong> and <strong>(B)</strong> and 100 Âµm in <strong>(C)</strong> and <strong>(D)</strong>.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     We also showed that L1L reproduced effects caused by LEC1 when both were expressed postembryonically. We fused the <em>L1L</em> coding region with the <em>35S</em> promoter from <em>Cauliflower mosaic virus</em> (Odell et al., 1985<a href="#BIB37"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) and transferred the construct into <em>lec1-1</em> mutants. Plants containing <em>35S</em>:<em>L1L</em> produced viable seedlings, indicating that the transgene could rescue the desiccation intolerance of <em>lec1</em> mutants. However, the seedlings did not resemble wild-type. Rather, as shown in <a href="#FIG5">Figure 5B</a>, transgenic seedlings developed thick and fleshy cotyledons and hypocotyls. Between 25 and 59% of the transgenic seedlings
                     developed multiple pairs of fleshy, cotyledon-like structures at positions normally occupied by leaves, whereas the other
                     seedlings remained arrested in their development, with only two cotyledons. Seedlings with a wild-type genotype transformed
                     with the <em>35S</em>:<em>L1L</em> construct displayed similar phenotypes (data not shown). These morphological defects reproduced those observed in transgenic
                     seedlings overexpressing <em>LEC1</em> (Lotan et al., 1998<a href="#BIB31"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), although we did not detect somatic embryos on <em>35S</em>:<em>L1L</em> seedlings. 
                     
                  </p>
                  <p>
                     
                     We used in situ hybridization experiments to determine if these fleshy seedlings express embryonic programs of development.
                     As shown in <a href="#FIG5">Figures 5C and 5D</a>, RNAs encoding cruciferin storage protein and oleosin lipid body protein, which normally accumulate specifically during embryo
                     development, were detected in these transgenic seedlings. Thus, ectopic expression of <em>L1L</em> reproduces the effects of <em>LEC1</em> overexpression by creating an environment sufficient to induce embryonic characteristics in vegetative organs. By contrast,
                     expression of a non-LEC1-type <em>AHAP3</em>, At4g14540, under the control of the <em>35S</em> promoter did not induce detectable developmental abnormalities (data not shown). Together, these results demonstrate that
                     L1L can confer embryonic characteristics to seedlings and support the idea that the LEC1-type B domain may underlie L1L and
                     LEC1 function in embryogenesis. 
                     
                  </p>
                  <p>
                     
                     <strong><em>L1L</em> Genes Are Present in Other Plants</strong><br/>
                     
                     Because L1L and LEC1 both are required for embryo development, we determined the extent to which LEC1-type AHAP3 subunits
                     are present in other organisms. As part of a study to obtain ESTs from the embryo proper of globular-stage scarlet runner
                     bean embryos at 6 days after pollination (A.Q. Bui, K. Weterings, and R.B. Goldberg, unpublished results), cDNA clones encoding
                     a HAP3 subunit with a LEC1-type B domain were identified. Because sequence analysis revealed that the predicted protein shared
                     94% sequence identity with the Arabidopsis L1L B domain and only 85% identity with that from LEC1, we named the cDNA <em>PcL1L</em>. 
                     
                  </p>
                  <p>
                     
                     To obtain clues about <em>PcL1L</em> function, we examined its temporal and spatial patterns of RNA accumulation. <a href="#FIG6">Figure 6A</a> shows that <em>PcL1L</em> RNA was detected only in developing embryos by RNA gel blot analysis. However, RT-PCR analysis showed that <em>PcL1L</em> RNA was present, presumably at low levels, at all developmental stages tested, including in vegetative organs, inflorescences,
                     and unfertilized ovules (<a href="#FIG6">Figure 6B</a>). Thus, the pattern of <em>PcL1L</em> RNA accumulation is more similar to that of <em>L1L</em> than that of <em>LEC1</em> in Arabidopsis (<a href="#FIG2">Figure 2</a>). In situ hybridization analysis showed that <em>PcL1L</em> RNA accumulated at high levels in the embryo proper and suspensor of preglobular-stage (5 days after pollination [DAP]) and
                     globular-stage (7 DAP) embryos but at very low levels in the endosperm of the seeds and integuments of the unfertilized ovules
                     (<a href="#FIG6">Figures 6C to 6E</a>). <em>PcL1L</em> RNA was present at the highest levels in the epidermal and subepidermal layers of the embryo proper at the globular stage
                     (<a href="#FIG6">Figure 6D</a>). This pattern is similar to the distribution of Arabidopsis <em>L1L</em> RNA observed at the torpedo and bent-cotyledon stages (<a href="#FIG3">Figures 3C and 3D</a>). The finding that scarlet runner bean appears to possess a protein with a LEC1-type B domain and that RNA encoding this
                     protein accumulates with a pattern similar to that of Arabidopsis <em>L1L</em> suggests that the two <em>L1L</em> genes are orthologous. 
                     
                  </p>
                  <p>
                     
                     <a name="FIG6" id="FIG6"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="5/F6.expansion.html"><img src="5/F6.small.gif"/></a><br/><strong>View larger version</strong> (96K):<br/><a href="5/F6.expansion.html">In this window</a><br/><a href="5/F6.expansion.html" class="in-nw">In a new window</a><br/>Â 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 6.</strong></span> <em>PcL1L</em> RNA Is Present Primarily in Developing Seeds.
                        
                        <p>
                           
                           <strong>(A)</strong> Gel blot analysis of <em>PcL1L</em> RNA accumulation. Twenty-five micrograms of total RNA was analyzed from leaves (Le), stems (St), 2-week-old seedling leaves
                           (Sl), 2-week-old seedling roots (SlRo), 2-week-old seedling stems (SlSt), inflorescences (In), ovules (Ov), 2-DAP seeds (Se
                           I), 4- to 5-DAP seeds (Se II), 6-DAP seeds (Se III), 12- to 14-DAP embryos (Em I), and 19- to 21-DAP embryos (Em II).
                           
                        </p>
                        
                        <p>
                           
                           <strong>(B)</strong> RT-PCR analysis of <em>PcL1L</em> RNA accumulation. Each lane corresponds to the RNA gel blot sample in <strong>(A)</strong>.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(C)</strong> to <strong>(F)</strong> Distribution of <em>PcL1L</em> RNA. Sections were hybridized with a <em>PcL1L</em> antisense probe <strong>(C)</strong> to <strong>(E)</strong> or a sense RNA control <strong>(F)</strong>. <strong>(C)</strong> and <strong>(D)</strong> were exposed for 4 days, whereas <strong>(E)</strong> was exposed for 47 days.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(C)</strong> Preglobular-stage seed. <em>PcL1L</em> RNA is high in the embryo proper and suspensor.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(D)</strong> Globular-stage seed. <em>PcL1L</em> RNA is at its highest levels in outer tissue layers of the embryo.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(E)</strong> Unfertilized ovule. <em>PcL1L</em> RNA is present at low levels throughout the ovule.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(F)</strong> Unfertilized ovule that does not bind sense RNA probe.
                           
                        </p>
                        
                        <p>
                           
                           Bars = 100 Âµm.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     Through database searches, we extended this analysis by identifying 12 other HAP3 subunits from other plants with conserved
                     amino acid residues characteristic of the LEC1-type B domain. Most were identified from embryo or seed-derived cDNA libraries,
                     although sequences from pine pollen cone and lotus root nodule cDNA libraries were identified. As shown in <a href="#FIG7">Figure 7A</a>, alignment of the B domains of these 15 plant HAP3 subunits revealed 17 amino acid residues that are shared between LEC1-type
                     B domains but that differ from residues conserved in non-LEC1-type B domains. By contrast, no conserved amino acid residues
                     were detected in the A and C domains of these 15 proteins, although some similarities were observed in these regions between
                     L1L, PcL1L, and some of the other plant L1L proteins (data not shown). The phylogenetic tree obtained by maximum parsimony
                     analysis (PAUP 4.0) shown in <a href="#FIG7">Figure 7B</a> indicates that the L1L proteins constitute a well-supported, monophyletic clade. Based on their sequence identity within
                     the B domain, their origin in seed RNA populations, and the ability of L1L to suppress the <em>lec1</em> mutation when expressed ectopically, we speculate that some of these <em>L1L</em> genes also may play roles in embryogenesis. 
                     
                  </p>
                  <p>
                     
                     <a name="FIG7" id="FIG7"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="5/F7.expansion.html"><img src="5/F7.small.gif"/></a><br/><strong>View larger version</strong> (47K):<br/><a href="5/F7.expansion.html">In this window</a><br/><a href="5/F7.expansion.html" class="in-nw">In a new window</a><br/>Â 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 7.</strong></span> Identification of L1L Proteins from Other Plants.
                        
                        <p>
                           
                           <strong>(A)</strong> Amino acid sequence alignment of the B domains of plant L1L proteins. Conserved amino acid residues are highlighted in gray,
                           and residues unique to L1L proteins are highlighted in black. Accession numbers are given at the end of Methods.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(B)</strong> Phylogenetic relationships between L1L and non-LEC1-type- HAP3 subunits. The cladogram illustrates the most parsimonious
                           consensus pattern of relationships obtained using maximum parsimony analysis. Bootstrap values generated with 1000 replicates
                           are indicated before the nodes. Nodes with bootstrap scores of &lt;50% are not shown. The high bootstrap values provide strong
                           support for the monophyletic L1L clade.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     <a name="SEC3" id="SEC3"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        DISCUSSION
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     <strong>L1L and LEC1 Constitute a Subclass of HAP3 Subunits</strong><br/>
                     
                     L1L and LEC1 display substantial sequence identity with HAP3 subunits of CBFs. Although a CBF from plants has not been isolated,
                     several lines of evidence suggest that L1L and LEC1 function as part of a CBF that regulates embryogenesis. First, in addition
                     to HAP3, paralogs of the other two CBF subunits required for DNA binding activity, HAP2 and HAP5, have been identified in
                     plants (Li et al., 1992<a href="#BIB28"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Albani and Robert, 1995<a href="#BIB1"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Edwards et al., 1998<a href="#BIB9"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Kusnetsov et al., 1999<a href="#BIB25"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Gusmaroli et al., 2001<a href="#BIB17"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Unlike yeast and mammals, which possess single genes for each subunit, plants possess families of subunits. For example,
                     there are 6, 10, and 8 genes encoding the AHAP2, AHAP3, and AHAP5 subunits, respectively, in Arabidopsis, opening the possibility
                     that different combinations of subunits may regulate diverse processes (Edwards et al., 1998<a href="#BIB9"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Gusmaroli et al., 2001<a href="#BIB17"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; M. Kim, H. Lee, R.W. Kwong, and J.J. Harada, unpublished results). Second, an AHAP2 gene has been shown to complement a
                     yeast HAP2 mutation, indicating that the Arabidopsis protein can function in a CBF (Edwards et al., 1998<a href="#BIB9"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Third, an AHAP5 subunit has been shown to interact with other nuclear proteins, presumably AHAP2 and AHAP3, to form a complex
                     that binds a double-stranded oligonucleotide containing a CAAT box (Kusnetsov et al., 1999<a href="#BIB25"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Fourth, loss-of-function mutations of <em>L1L</em> and <em>LEC1</em>, two HAP3 paralogs, have severe consequences on plant development, suggesting that these subunits play essential roles. 
                     
                  </p>
                  <p>
                     
                     Our results show that there are at least two distinct classes of AHAP3 subunits that differ in several respects. LEC1-type
                     and non-LEC1-type HAP3 subunits differ by 16 amino acid residues that serve as signatures of their B domains (<a href="#FIG1">Figure 1</a>) (Gusmaroli et al., 2001<a href="#BIB17"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Phylogenetic analysis suggests that HAP3 subunits possessing these signature residues have a common evolutionary origin
                     (<a href="#FIG7">Figure 7</a>). Residues at corresponding positions of yeast and mammalian HAP3 subunits are more similar to non-LEC1-type than to LEC1-type
                     AHAP3 subunits. This finding opens the possibility that L1L and LEC1 represent novel HAP3 subunits of CBF. Next, sequence
                     diversity between the two types of subunits appears to underlie the functional differences, because <em>L1L</em> but not two other genes that encode non-LEC1-type AHAP3 subunits suppressed the <em>lec1</em> mutation when expressed under the control of <em>LEC1</em> flanking sequences (<a href="#TBL1">Table 1</a>). Similarly, a non-LEC1-type <em>AHAP3</em> did not induce embryonic characteristics in seedlings when fused with the <em>35S</em> promoter, as did <em>L1L</em> and <em>LEC1</em> (<a href="#TBL1">Table 1</a>, <a href="#FIG5">Figure 5</a>). Finally, genes encoding L1L and LEC1 are expressed predominately or exclusively during seed development (<a href="#FIG2">Figure 2</a>) (Lotan et al., 1998<a href="#BIB31"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), whereas the non-LEC1-type <em>AHAP3</em> genes generally are expressed at high levels in nonembryonic tissues (Edwards et al., 1998<a href="#BIB9"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Gusmaroli et al., 2001<a href="#BIB17"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; M. Kim and J.J. Harada, unpublished results). In this regard, <em>LEC1</em>, <em>L1L</em>, and <em>PcL1L</em> exhibit similar spatial patterns of RNA accumulation in developing embryos (<a href="#FIG3">Figures 3</a> and <a href="#FIG6">6</a>) (Lotan et al., 1998<a href="#BIB31"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     <strong>The B Domain Underlies the Function of LEC1-Type HAP3 Subunits in Embryogenesis</strong><br/>
                     
                     We present strong evidence that the B domain of LEC1-type HAP3 subunits underlies their function in embryogenesis. Sequence
                     similarity between L1L and LEC1 was observed exclusively in the B domain and not in the A and C domains (<a href="#FIG1">Figure 1</a>). Suppression of the <em>lec1</em> mutation by <em>L1L</em> (<a href="#FIG5">Figure 5</a>, <a href="#TBL1">Table 1</a>) suggests that some or all of the 16 residues unique to LEC1-type B domains account for the ability of L1L to substitute
                     functionally for LEC1 when expressed ectopically. Most HAP3 subunits from other plants that possess LEC1-type B domains (<a href="#FIG7">Figure 7</a>) are present in embryos or seeds, consistent with the expression patterns of L1L and LEC1. This finding opens the possibility
                     that other L1Ls play important roles in seed development. However, two L1Ls are present in pollen cones and root nodules,
                     suggesting that the LEC1-type HAP3 subunit may function at other developmental stages. This class of HAP3 subunit is present
                     in gymnosperms and monocotyledonous and dicotyledonous angiosperms, but it has not been detected in nonplant organisms, suggesting
                     that the LEC1-type B domain evolved uniquely in plants. 
                     
                  </p>
                  <p>
                     
                     The central B domain of HAP3 subunits serves critical roles in CBF function. Studies of yeast and mammalian HAP3s show that
                     the B domain contains amino acid residues that account for its ability to interact with HAP2 and HAP5 subunits and for the
                     CBF to bind DNA (Xing et al., 1993<a href="#BIB51"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Kim et al., 1996<a href="#BIB24"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Sinha et al., 1996<a href="#BIB41"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). One explanation for the differences in the activities of L1L and LEC1 versus non-LEC1-type AHAP3 subunits is that LEC1-type
                     B domains may mediate interactions with specific AHAP2 and AHAP5 subunits to form a CBF that activates the genes required
                     for embryo development. Because AHAP2 and AHAP5 are encoded by six and eight genes, respectively, and most are expressed in
                     nonseed tissues (Gusmaroli et al., 2001<a href="#BIB17"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; H. Lee, M. Kim, and J.J. Harada, unpublished results), the possibility exists that defined combinations of AHAP subunits
                     confer specific functions to the transcription complex. However, this interpretation requires that the specific AHAP2 and
                     AHAP5 subunits present in embryos also are present in vegetative tissues, because ectopic expression of <em>L1L</em> and <em>LEC1</em> confers embryonic characteristics to vegetative tissues (<a href="#FIG5">Figure 5</a>) (Lotan et al., 1998<a href="#BIB31"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     A second possibility is that the LEC1-type B domain may recruit other transcription factors to the CBF that confer unique
                     specificity to the complex. CBFs have been shown to interact with other transcription factors to activate specific sets of
                     genes. For example, activation of the MHC class II gene promoter requires the binding of CBF and an X-box binding factor,
                     and activation of the 3-hydroxy-3-methylglutaryl-CoA synthase gene requires the binding of both CBF and sterol regulatory
                     element binding proteins (Wright et al., 1994<a href="#BIB50"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Linhoff et al., 1997<a href="#BIB29"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Dooley et al., 1998<a href="#BIB8"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). In addition, no Arabidopsis paralog of the HAP4 subunit that provides a transcriptional activation domain to the yeast
                     CBF has been identified, and transcriptional activation domains are not apparent in the HAP2 and HAP5 subunits as they are
                     in their mammalian counterparts (Forsburg and Guarente, 1989<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Coustry et al., 1996<a href="#BIB5"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Thus, a protein with transcriptional activation function may be recruited to the complex by L1L and LEC1. 
                     
                  </p>
                  <p>
                     
                     A final alternative is that B domain residues unique to LEC1-type HAP3 subunits may confer a novel DNA binding specificity
                     to the CBF that differs from that afforded by non-LEC1-type AHAP3 subunits. Thus, CBFs containing L1L and LEC1 would bind
                     and modulate the transcription of genes required for embryo development, whereas non-LEC1-type AHAP3s would not. This is the
                     simplest alternative, because there is no need to invoke novel interactions with other proteins. However, to our knowledge,
                     no HAP3 subunit has been identified that alters the binding specificity of CBFs in other organisms. 
                     
                  </p>
                  <p>
                     
                     <strong>L1L and LEC1 Have Distinct Functions during Embryogenesis</strong><br/>
                     
                     Although our studies have shown that <em>L1L</em> can substitute for <em>LEC1</em> if expressed ectopically (<a href="#TBL1">Table 1</a>, <a href="#FIG5">Figure 5</a>), other evidence suggests that L1L and LEC1 normally have distinct functions during embryogenesis. The first and most compelling
                     argument is that monogenic, loss-of-function mutations in either <em>L1L</em> or <em>LEC1</em> cause defects in embryo development. These results show that the endogenous genes cannot substitute for one another, although
                     we cannot exclude the possibility that the genes have partial overlaps in function. Consistent with this interpretation is
                     the finding that the suppression of <em>L1L</em> and <em>LEC1</em> gene expression induces different embryonic phenotypes. <em>lec1</em> mutants arrest at a late stage of embryo development, with complete though misshapen cotyledons and embryonic axes, and mutant
                     embryos can be rescued before desiccation to produce viable seedlings (reviewed by Harada, 2001<a href="#BIB19"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). By contrast, RNAi suppression of <em>L1L</em> caused embryos to arrest in their development as early as the globular stage, and mutant embryos cannot be rescued to produce
                     postembryonic plants (<a href="#FIG4">Figure 4</a>). We conclude that although <em>L1L</em> clearly is required for embryo development, it appears to play a fundamentally different role in embryogenesis than <em>LEC1</em>. 
                     
                  </p>
                  <p>
                     
                     With regard to the L1L mutant phenotype, RNAi suppression of <em>L1L</em> gene expression is characterized by incomplete penetrance and variable expressivity. Not all embryos containing the construct
                     exhibit a mutant phenotype, and those that do arrest at a number of different embryonic stages with a variety of defects (<a href="#FIG4">Figure 4</a>). However, transgenic embryos displaying a mutant phenotype possessed low to undetectable levels of <em>L1L</em> RNA, whereas those with a wild-type phenotype had intermediate to high RNA levels (<a href="#FIG4">Figure 4</a>). Therefore, defects in embryo development appear to result from the suppression of <em>L1L</em> expression. Although we have not yet identified an insertional mutation of <em>L1L</em>, we note the possibility that the RNAi suppression of <em>L1L</em> may not produce a mutant phenotype as severe as that of a genetic null mutation. The RNAi construct is controlled by the
                     <em>35S</em> promoter, and we have shown that this promoter does not become active detectably during embryogenesis until the globular
                     stage (J. Pelletier and J.J. Harada, unpublished results). Thus, <em>L1L</em> RNA may accumulate early during embryogenesis in RNAi lines, albeit at a very low level, and decline only after the globular
                     stage. Despite these qualifications, it is unlikely that a null <em>l1l</em> mutant would share similar characteristics with the <em>lec</em> class of mutants, because <em>l1l</em> mutant embryos arrest earlier in embryogenesis than do <em>lec1</em> mutants. 
                     
                  </p>
                  <p>
                     
                     Three other lines of evidence support the conclusion that <em>L1L</em> and <em>LEC1</em> have distinct endogenous functions. First, <em>L1L</em> RNA accumulates later in embryogenesis than does <em>LEC1</em> RNA (<a href="#FIG2">Figure 2</a>). Second, <em>L1L</em> RNA is present in developing seeds and at low levels in vegetative organs, whereas <em>LEC1</em> RNA is detected only in developing seeds (Lotan et al., 1998<a href="#BIB31"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) (<a href="#FIG2">Figure 2</a>). Third, <em>lec1</em> mutants display an abnormal phenotype even though <em>L1L</em> RNA is detected in the mutant seeds, indicating that the endogenous <em>L1L</em> gene is not sufficient to completely prevent defects induced by the <em>lec1</em> mutation (<a href="#FIG2">Figure 2</a>). 
                     
                  </p>
                  <p>
                     
                     There are several potential explanations to reconcile the findings that the endogenous <em>L1L</em> and <em>LEC1</em> genes do not act redundantly, yet <em>L1L</em> can be made to substitute functionally for <em>LEC1</em>. One hypothesis is that the specific pattern of <em>LEC1</em> gene expression is critical for its function. Although the distribution of <em>LEC1</em> and <em>L1L</em> RNAs in embryos is similar, there are differences in the timing of their accumulation (<a href="#FIG2">Figures 2</a> and <a href="#FIG3">3</a>) (Lotan et al., 1998<a href="#BIB31"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Similar situations have been described for two Arabidopsis MYB genes, <em>WEREWOLF</em> and <em>GLABROUS1</em>. Genes encoding these functionally equivalent proteins play different roles in plant development because they are transcribed
                     in distinct cell types (Lee and Schiefelbein, 2001<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Alternatively, increased dosage of the <em>L1L</em> gene and, by inference, increased <em>L1L</em> RNA levels in transgenic <em>lec1</em> mutants containing the <em>LEC1</em>:<em>L1L</em>:<em>LEC1</em> transgene may account for the suppression of the mutation. Dosage suppression has been described in microorganisms (Puziss
                     et al., 1994<a href="#BIB39"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). A third possibility is that because the accumulation of <em>LEC1</em> and <em>L1L</em> RNA does not differ substantially, RNA sequences in the <em>LEC1</em> 5' and/or 3' untranslated regions, which are included in the <em>LEC1</em>:<em>L1L</em>:<em>LEC1</em> gene (see Methods), may regulate LEC1 function at the translational level. Additional information is needed to distinguish
                     between these possibilities. 
                     
                  </p>
                  <p>
                     
                     In conclusion, we have shown that L1L, the AHAP3 subunit most closely related to LEC1, is a regulator of embryo development.
                     <em>L1L</em> is expressed predominately during embryo development, and it is required for the completion of embryogenesis. The ability
                     of L1L but not non-LEC1-type AHAP3 subunits to function in place of LEC1 when expressed ectopically implicates the B domain
                     as the region of L1L and LEC1 that is critical for their function. Mutagenesis studies are needed to define which of the 16
                     amino acid residues of LEC1-type AHAP3 subunits differentiate their functions from non-LEC1-type subunits. Although L1L can
                     function redundantly with LEC1 when expressed ectopically, the two subunits have distinct functions during embryogenesis.
                     Thus, L1L is a novel regulatory protein that plays an essential role during embryogenesis. 
                     
                  </p>
                  <p>
                     
                     <a name="SEC4" id="SEC4"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        METHODS
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     <strong>Plant Materials and Manipulations</strong><br/><em>lec1-1</em> mutants and wild-type plants of <em>Arabidopsis thaliana</em> (ecotype Wassilewskija) were grown as described previously (West et al., 1994<a href="#BIB48"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Seeds of the day-neutral scarlet runner bean (<em>Phaseolus coccineus</em> cv Hammond's Dwarf Red Flower) were grown in the greenhouse as described by Weterings et al. (2001)<a href="#BIB49"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. Seeds were germinated in vermiculite to obtain seedlings. Flowers were pollinated and collected at specific days after pollination
                     (DAP) as described previously (Weterings et al., 2001<a href="#BIB49"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     Approximately 500 unfertilized runner bean ovules were collected from young, open flowers. Approximately 90, 66, and 50 seeds
                     were collected from 2-DAP, 4- to 5-DAP, and 6-DAP pods, respectively. Approximately 100 cotyledon-stage embryos were isolated
                     from seeds of 12- to 14-DAP and 19- to 21-DAP pods. Seed and embryo stages were according to Weterings et al. (2001)<a href="#BIB49"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. Small young leaves, stems, and inflorescences were collected from lateral branches of flowering plants. True leaves, roots,
                     and stems were collected from 2-week-old seedlings. Upon collection, tissues were frozen immediately in liquid nitrogen and
                     stored at -80Â°C until use. 
                     
                  </p>
                  <p>
                     
                     <em>Agrobacterium tumefaciens</em> strain GV3101 containing transformation constructs was infiltrated into <em>lec1-1</em> and wild-type plants (Bechtold et al., 1993<a href="#BIB3"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Seeds from T0 plants were germinated on medium containing 60 Âµg/mL glufosinate ammonium to select for transgenic plants
                     (Finale; AgrEvo Environmental Health, Montvale, NJ). Plant genotypes were verified in PCR amplification experiments. 
                     
                  </p>
                  <p>
                     
                     <strong>Isolation and Preparation of cDNA and Genomic Clones</strong><br/>
                     
                     PCR was used to amplify the genomic fragments containing <em>AHAP3</em> genes. Primers that flanked the putative <em>L1L</em> open reading frame with the addition of BamHI and XbaI sites for subcloning purposes were used (BAMMNJ7-5, 5'-AGGATCCATGGAACGTGGAGGCTTCCAT-3';
                     and 3-MNJ7XBA, 5'-ATCTAGATCAGTACTTATGTTGTTGAGTCG-3'). The AHAP3 genes At4g14540 (3-224) and At3g53340 (3-180) were amplified
                     using primer combinations 3-224-F/3-224-R (5'-CCTATC-TCGAGATGGCGGATTCGGACAACGATTC-3'/5'-CCCGGTCTAGAT-TAAGAAAAATGATGGGAAAATTGATGTCC-3')
                     and AH3-180-F/AH3- 180-R (5'-CCCGGGGAGATCTATGGCGGATACGCCTTCGAGCCC-AGC-3'/5'-GGGCCCCTAGGCTTTTACCAGCTCGGCATTTCTTCA-CC-3'), respectively.
                     Nucleotide sequences of the genomic clones were verified. 
                     
                  </p>
                  <p>
                     
                     <em>L1L</em>, At4g14540, and At3g53340 genomic clones were inserted between the <em>LEC1</em> promoter and terminator within the plant transformation vector BJ49 (Gleave, 1992<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). The <em>LEC1</em> promoter/terminator cassette consists of 1992 bp of DNA 5' of the <em>LEC1</em> translation start codon plus 770 bp 3' of the <em>LEC1</em> stop codon (H. Lee and J.J. Harada, unpublished results). The <em>L1L</em> gene was fused with the <em>35S</em> promoter from <em>Cauliflower mosaic virus</em> and the octopine synthase terminator of the plasmid pART7 and transferred into the binary transformation vector pMLBART (Gleave,
                     1992<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     cDNA clone pPCEP112 was identified from a scarlet runner bean cDNA library by EST sequencing analysis. This cDNA library was
                     constructed with total RNA isolated from the embryo proper of 6-DAP seeds using the SMART PCR cDNA Library Construction Kit
                     (Clontech, Palo Alto, CA) (A.Q. Bui, K. Weterings, and R.B. Goldberg, unpublished results). 
                     
                  </p>
                  <p>
                     
                     <strong>Protein Sequence Analysis</strong><br/>
                     
                     Amino acid sequences were aligned with the PileUp program (Seqweb version 2.0.2; Accelrys, Burlington, MA), and alignments
                     were prepared with BOXSHADE (<a href="http://ch.EMBnet.org">http://www.ch.EMBnet.org</a>). Database searches were performed with the LEC1 protein sequence as a query (<a href="http://www.ncbi.nlm.nih.gov/blast/">http://www.ncbi.nlm.nih.gov/blast/</a> and <a href="http://www.arabidopsis.org/Blast/">http://www.arabidopsis.org/Blast/</a> [as of August 5, 2002]). The analysis identified the following AHAP3 genes: At2g47810, At1g09030, At2g37060, At3g53340, At2g38880,
                     At5g47640, At4g14540, At2g13570, and At5g47670 [the last of which we renamed LEC1-LIKE]. 
                     
                  </p>
                  <p>
                     
                     Parsimony trees of the B domains of HAP3 subunits were generated with CLUSTAL X (version 1.8; Thompson et al., 1997<a href="#BIB44"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) and the heuristic search algorithm of the PAUP program (version 4.0 beta; Swofford et al., 1996<a href="#BIB43"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). One hundred replicates were used for weighted analysis in generated consensus parsimony trees. For maximum parsimony analysis,
                     1000 iterations were used to create bootstrap percentages. 
                     
                  </p>
                  <p>
                     
                     <strong>RNA Analyses</strong><br/>
                     
                     Arabidopsis RNA was isolated as described previously (Stone et al., 2001<a href="#BIB42"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Total RNA from scarlet runner bean was isolated using the RNeasy Plant Mini Kit (Qiagen, Valencia, CA) and treated with
                     RNase-free DNase I (Boehringer Mannheim, Indianapolis, IN) according to the protocol of Ausubel et al. (1995)<a href="#BIB2"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. Approximately 25 Âµg of RNA was loaded on a formaldehyde gel, and RNA gel blot analysis was performed as described previously
                     (Harada et al., 1988<a href="#BIB20"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). In situ hybridization experiments with Arabidopsis and scarlet runner bean tissues were performed as described previously
                     (Dietrich et al., 1989<a href="#BIB6"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Weterings et al., 2001<a href="#BIB49"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     The presence of <em>L1L</em> RNA in an organ system was assessed using nonquantitative reverse transcriptaseâmediated PCR analysis. For Arabidopsis, first-strand
                     cDNA was generated from 5 Âµg of each RNA in a 20-ÂµL reaction volume using the Thermal Script reverse transcriptase system
                     (Invitrogen, Carlsbad, CA). One microliter of each reaction was amplified in a 20-ÂµL reaction volume according to the manufacturer's
                     specifications using primers for <em>L1L</em> (see above), <em>LEC1</em> (LP/UP, 5'-GACATACAACACTTTTCCTTAAAG-3'/5'-CAGCAA-CAACCCACCCCCAATG-3'), and a ribosomal protein gene (TIN1/TIN2, 5'-TTTGGTGGATGCCCCTGATA-3'/5'-TAATTTCCGAATCCA-AAATC-3')
                     (T. Lotan and J.J. Harada, unpublished results). Amplification products were fractionated by agarose gel electrophoresis.
                     
                     
                  </p>
                  <p>
                     
                     For scarlet runner bean RNA, first-strand cDNA was generated from 2 Âµg of each total RNA in a 20-ÂµL reaction using Superscript
                     II Reverse Transcriptase according to the manufacturer's specifications (Gibco BRL, Rockville, MD). PCR amplification was
                     performed using the primers PcL1L-F (5'-AGATTCTTCCTCCACATGCCAAGAT-3') and PcL1L-R (5'-CCTTAATCCCATCCATCCCCTTAAT-3') with 2
                     ÂµL of each reverse transcriptase reaction in a 50-ÂµL reaction volume. 
                     
                  </p>
                  <p>
                     
                     <strong>RNA Interference Suppression of <em>L1L</em></strong><br/>
                     
                     The primer combination 3LEFTXX (5'-TCTAGACTCGAGCTTAGCTGCAGTGCTGGG-3') and 3RIGHTBAM (5'-GGATCCTTGAACCAAGACGCATTACG-3') was
                     used to amplify a 500-bp fragment unique to the C domain of <em>L1L</em>. The fragment was placed in both orientations into the RNA interference vector pRNA69, which contains the <em>35S</em> promoter (J.F. Emery and J.L. Bowman, unpublished results). This construct then was placed into the pMLBART binary vector.
                     
                     
                  </p>
                  <p>
                     
                     Upon request, all novel materials described in this article will be made available in a timely manner for noncommercial research
                     purposes. 
                     
                  </p>
                  <p>
                     
                     <a name="SEC5" id="SEC5"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        Accession Numbers
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     Accession numbers for the CCAAT binding factor HAP3 subunits shown in <a href="#FIG7">Figure 7</a> are as follows: LEC1, <a href="/cgi/external_ref?access_num=AF036684&amp;link_type=GEN">AF036684</a>; L1L, <a href="/cgi/external_ref?access_num=AY138461&amp;link_type=GEN">AY138461</a>; PcL1L, <a href="/cgi/external_ref?access_num=AF533650&amp;link_type=GEN">AF533650</a>; barley (A), <a href="/cgi/external_ref?access_num=AL506199&amp;link_type=GEN">AL506199</a> and <a href="/cgi/external_ref?access_num=AL509098&amp;link_type=GEN">AL509098</a>; wheat, <a href="/cgi/external_ref?access_num=AY058921&amp;link_type=GEN">AY058921</a>; pine A, <a href="/cgi/external_ref?access_num=AW754604&amp;link_type=GEN">AW754604</a>; pine B, <a href="/cgi/external_ref?access_num=AW981729&amp;link_type=GEN">AW981729</a>; Argemone, <a href="/cgi/external_ref?access_num=AY058920&amp;link_type=GEN">AY058920</a>; rice, <a href="/cgi/external_ref?access_num=AU088581&amp;link_type=GEN">AU088581</a>; maize, <a href="/cgi/external_ref?access_num=AF410176&amp;link_type=GEN">AF410176</a>; barley B, <a href="/cgi/external_ref?access_num=BE603222&amp;link_type=GEN">BE603222</a>; Vernonia, <a href="/cgi/external_ref?access_num=AY058919&amp;link_type=GEN">AY058919</a>; soybean A, <a href="/cgi/external_ref?access_num=AY058917&amp;link_type=GEN">AY058917</a>; soybean B, <a href="/cgi/external_ref?access_num=AY058918&amp;link_type=GEN">AY058918</a>; and lotus, <a href="/cgi/external_ref?access_num=AW719547&amp;link_type=GEN">AW719547</a> and <a href="/cgi/external_ref?access_num=AW720671&amp;link_type=GEN">AW720671</a>. 
                     
                  </p>
                  <p>
                     
                     
                     <a name="ACK" id="ACK"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        Acknowledgments
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     We thank John Emery and John Bowman for the gift of pRNA69, Keith Lowe and Bill Gordon-Kamm for sharing unpublished data,
                     Kook-Hyun Chung and Neelima Sinha for help with the phylogenetic analysis, the ABRC for seeds, and Abeba Kiros, Diana Lee,
                     Julie Pelletier, and Kelly Matsudaira Yee for technical assistance. This work was supported by grants to R.B.G. and J.J.H.
                     from the U.S. Department of Energy and Ceres, Inc. 
                     
                  </p>
                  <p>
                     
                     
                     <a name="FN" id="FN"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        Footnotes
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     <a name="" id=""/>
                     Article, publication date, and citation information can be found at <a href="http://www.plantcell.org/cgi/doi/10.1105/tpc.006973">www.plantcell.org/cgi/doi/10.1105/tpc.006973</a>. 
                     
                  </p>
                  <p>
                     
                     <a name="FN1" id="FN1"/>
                     <sup><sup>1</sup></sup> These two authors contributed equally to this work. <a href="#RFN1"><img width="12" height="12" alt="Back" src="/icons/back.gif"/></a></p>
                  <p>
                     
                     Received August  7, 2002; 
                     
                     accepted October 13, 2002.
                     
                  </p>
                  <p>
                     
                     <a name="BIBL" id="BIBL"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        References
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     
                     <a name="BIB1" id="BIB1"/>
                     <strong>Albani, D., and Robert, L.S.</strong> (1995). Cloning and characterization of a <em>Brassica napus</em> gene encoding a homologue of the B subunit of a heteromeric CCAAT-binding factor. Gene <strong>167,</strong> 209â213.<a href="/cgi/external_ref?access_num=8566779&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB2" id="BIB2"/>
                     <strong>Ausubel, F.M., Brent, R., Kingston, R.E., Moore, D.D., Seidman, J.G., and Smith, J.A.</strong> (1995). Short Protocols in Molecular Biology, 3rd ed. (New York: John Wiley &amp; Sons).
                     
                  </p>
                  <p><a name="BIB3" id="BIB3"/>
                     <strong>Bechtold, N., Ellis, J., and Pelletier, G.</strong> (1993). In planta Agrobacterium mediated gene transfer by infiltration of adult <em>Arabidopsis thaliana</em> plants. C. R. Acad. Sci. Paris <strong>316,</strong> 1194â1199.
                     
                  </p>
                  <p><a name="BIB4" id="BIB4"/>
                     <strong>Bewley, J.D.</strong> (1997). Seed germination and dormancy. Plant Cell <strong>9,</strong> 1055â1066.<a href="/cgi/external_ref?access_num=10.1105%2Ftpc.9.7.1055&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=A1997XP13600008&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=12237375&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB5" id="BIB5"/>
                     <strong>Coustry, F., Maity, S.N., Sinha, S., and de Crombrugghe, B.</strong> (1996). The transcriptional activity of the CCAAT-binding factor CBF is mediated by two distinct activation domains, one
                     in the CBF-B subunit and the other in the CBF-C subunit. J. Biol. Chem. <strong>272,</strong> 14485â14491.
                     
                  </p>
                  <p><a name="BIB6" id="BIB6"/>
                     <strong>Dietrich, R.A., Maslyar, D.J., Heupel, R.C., and Harada, J.J.</strong> (1989). Spatial patterns of gene expression in <em>Brassica napus</em> seedlings: Identification of a cortex-specific gene and localization of messenger RNA encoding isocitrate lyase and a polypeptide
                     homologous to proteinases. Plant Cell <strong>1,</strong> 73â80.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=1/1/73">[Abstract/<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB7" id="BIB7"/>
                     <strong>Dilaurenzio, L., Wysocka-Diller, J., Malamy, J.E., Pysh, L., Helariutta, Y., Freshour, G., Hahn, M.G., Feldmann, K.A., and
                        Benfey, P.N.</strong> (1996). The <em>SCARECROW</em> gene regulates an asymmetric cell division that is essential for generating the radial organization of the Arabidopsis root.
                     Cell <strong>86,</strong> 423â433.<a href="/cgi/external_ref?access_num=10.1016%2FS0092-8674%2800%2980115-4&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=A1996VC30900010&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=8756724&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB8" id="BIB8"/>
                     <strong>Dooley, K.A., Millinder, S., and Osborne, T.F.</strong> (1998). Sterol regulation of 3-hydroxy-3-methylglutaryl-coenzyme A synthase gene through a direct interaction between sterol
                     regulatory element binding protein and the trimeric CCAAT-binding factor/nuclear factor Y. J. Biol. Chem. <strong>273,</strong> 1349â1356.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=jbc&amp;resid=273/3/1349">[Abstract/<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB9" id="BIB9"/>
                     <strong>Edwards, D., Murray, J.A.H., and Smith, A.G.</strong> (1998). Multiple genes encoding the conserved CCAAT-box transcription factor complex are expressed in Arabidopsis. Plant
                     Physiol. <strong>117,</strong> 1015â1022.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=117/3/1015">[Abstract/<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB10" id="BIB10"/>
                     <strong>Finkelstein, R.R., and Lynch, T.J.</strong> (2000). The Arabidopsis abscisic acid response gene <em>ABI5</em> encodes a basic leucine zipper transcription factor. Plant Cell <strong>12,</strong> 599â609.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=12/4/599">[Abstract/<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB11" id="BIB11"/>
                     <strong>Finkelstein, R.R., Wang, M.L., Lynch, T.J., Rao, S., and Goodman, H.M.</strong> (1998). The Arabidopsis abscisic acid response locus ABI4 encodes an APETALA 2 domain protein. Plant Cell <strong>10,</strong> 1043â1054.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=10/6/1043">[Abstract/<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB12" id="BIB12"/>
                     <strong>Forsburg, S.L., and Guarente, L.</strong> (1989). Identification and characterization of HAP4: A third component of the CCAAT-bound HAP2/HAP3 heteromer. Genes Dev.
                     <strong>3,</strong> 1166â1178.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=genesdev&amp;resid=3/8/1166">[Abstract/<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB13" id="BIB13"/>
                     <strong>Giraudat, J., Hauge, B.M., Valon, C., Smalle, J., Parcy, F., and Goodman, H.M.</strong> (1992). Isolation of the Arabidopsis <em>ABI3</em> gene by positional cloning. Plant Cell <strong>4,</strong> 1251â1261.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=4/10/1251">[Abstract/<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB14" id="BIB14"/>
                     <strong>Gleave, A.P.</strong> (1992). A versatile binary vector system with a T-DNA organisational structure conducive to efficient integration of cloned
                     DNA into the plant genome. Plant Mol. Biol. <strong>20,</strong> 1203â1207.<a href="/cgi/external_ref?access_num=10.1007%2FBF00028910&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=A1992KC40200023&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=1463857&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB15" id="BIB15"/>
                     <strong>Goldberg, R.B., de Paiva, G., and Yadegari, R.</strong> (1994). Plant embryogenesis: Zygote to seed. Science <strong>266,</strong> 605â614.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=266/5185/605">[Abstract/<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB16" id="BIB16"/>
                     <strong>Guarente, L., Lalonde, B., Gifford, P., and Alani, E.</strong> (1984). Distinctly regulated tandem upstream activation sites mediate catabolite repression of the CYC1 genes of <em>S. cerevisiae</em>. Cell <strong>32,</strong> 317â321.
                     
                  </p>
                  <p><a name="BIB17" id="BIB17"/>
                     <strong>Gusmaroli, G., Tonelli, C., and Mantovani, R.</strong> (2001). Regulation of the CCAAT-binding NF-Y subunits in <em>Arabidopsis thaliana</em>. Gene <strong>264,</strong> 173â185.<a href="/cgi/external_ref?access_num=10.1016%2FS0378-1119%2801%2900323-7&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000167654500003&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=11250072&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB18" id="BIB18"/>
                     <strong>Harada, J.J.</strong> (1997). Seed maturation and control of germination. In Advances in Cellular and Molecular Biology of Plants, Vol. 4, Cellular
                     and Molecular Biology of Seed Development, B.A. Larkins and I.K. Vasil, eds (Dordrecht, The Netherlands: Kluwer Academic Publishers),
                     pp. 545â592.
                     
                  </p>
                  <p><a name="BIB19" id="BIB19"/>
                     <strong>Harada, J.J.</strong> (2001). Role of Arabidopsis LEAFY COTYLEDON genes in seed development. J. Plant Physiol. <strong>158,</strong> 405â409.<a href="/cgi/external_ref?access_num=10.1078%2F0176-1617-00351&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000168326400002&amp;link_type=ISI">[Web of Science]</a></p>
                  <p><a name="BIB20" id="BIB20"/>
                     <strong>Harada, J.J., Baden, C.S., and Comai, L.</strong> (1988). Spatially regulated genes expressed during seed germination and postgerminative development are activated during
                     embryogeny. Mol. Gen. Genet. <strong>212,</strong> 466â473.<a href="/cgi/external_ref?access_num=10.1007%2FBF00330851&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=A1988N827300010&amp;link_type=ISI">[Web of Science]</a></p>
                  <p><a name="BIB21" id="BIB21"/>
                     <strong>Helariutta, Y., Fukaki, H., Wysocka-Diller, J., Nakajima, K., Jung, J., Sena, G., Hauser, M.T., and Benfey, P.N.</strong> (2000). The SHORT-ROOT gene controls radial patterning of the Arabidopsis root through radial signaling. Cell <strong>101,</strong> 555â567.<a href="/cgi/external_ref?access_num=10.1016%2FS0092-8674%2800%2980865-X&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000087375400012&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=10850497&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB22" id="BIB22"/>
                     <strong>Jurgens, G.</strong> (2001). Apical-basal pattern formation in Arabidopsis embryogenesis. EMBO J. <strong>20,</strong> 3609â3616.<a href="/cgi/external_ref?access_num=10.1093%2Femboj%2F20.14.3609&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000170149500001&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=11447101&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB23" id="BIB23"/>
                     <strong>Keng, T., and Guarente, L.</strong> (1987). Constitutive expression of the yeast <em>HEM1</em> gene is actually a composite of activation and repression. Proc. Natl. Acad. Sci. USA <strong>84,</strong> 9113â9117.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=84/24/9113">[Abstract/<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB24" id="BIB24"/>
                     <strong>Kim, I.S., Sinha, S., de Crombrugghe, B., and Maity, S.N.</strong> (1996). Determination of functional domains in the C subunit of the CCAAT-binding factor (CBF) necessary for formation of
                     a CBF-DNA complex: CBF-B interacts simultaneously with both the CBF-A and CBF-C subunits to form a heterotrimeric CBF molecule.
                     Mol. Cell. Biol. <strong>16,</strong> 4003â4013.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=mcb&amp;resid=16/8/4003">[Abstract/<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB25" id="BIB25"/>
                     <strong>Kusnetsov, V., Landsberger, M., Meurer, J., and Oelmuller, R.</strong> (1999). The assembly of the CAAT-box binding complex at a photosynthesis gene promoter is regulated by light, cytokinin,
                     and the stage of the plastids. J. Biol. Chem. <strong>274,</strong> 36009â36014.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=jbc&amp;resid=274/50/36009">[Abstract/<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB26" id="BIB26"/>
                     <strong>Laux, T., and Jurgens, G.</strong> (1997). Embryogenesis: A new start in life. Plant Cell <strong>9,</strong> 989â1000.<a href="/cgi/external_ref?access_num=10.1105%2Ftpc.9.7.989&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=A1997XP13600002&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=12237371&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB27" id="BIB27"/>
                     <strong>Lee, M.M., and Schiefelbein, J.</strong> (2001). Developmentally distinct MYB genes encode functionally equivalent proteins in Arabidopsis. Development <strong>128,</strong> 1539â1546.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=128/9/1539">[Abstract]</a></p>
                  <p><a name="BIB28" id="BIB28"/>
                     <strong>Li, X.-Y., Mantovani, R., Hooft Van Huijsduijnen, R., Andre, I., Benoist, C., and Mathis, D.</strong> (1992). Evolutionary variation of the CCAAT-binding transcription factor NF-Y. Nucleic Acids Res. <strong>20,</strong> 1087â1091.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=nar&amp;resid=20/5/1087">[Abstract/<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB29" id="BIB29"/>
                     <strong>Linhoff, M.W., Wright, K.L., and Ting, J.P.-Y.</strong> (1997). CCAAT-binding factor NF-Y and RFX are required for in vivo assembly of a nucleoprotein complex that spans 250 base
                     pairs: The invariant chain promoter as a model. Mol. Cell. Biol. <strong>17,</strong> 4589â4596.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=mcb&amp;resid=17/8/4589">[Abstract/<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB30" id="BIB30"/>
                     <strong>Long, J.A., Moan, E.I., Medford, J.I., and Barton, M.K.</strong> (1996). A member of the knotted class of homeodomain proteins encoded by the STM gene of Arabidopsis. Nature <strong>379,</strong> 66â69.<a href="/cgi/external_ref?access_num=10.1038%2F379066a0&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=8538741&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB31" id="BIB31"/>
                     <strong>Lotan, T., Ohto, M., Matsudaira Yee, K., West, M.A.L., Lo, R., Kwong, R.W., Yamagishi, K., Fischer, R.L., Goldberg, R.B.,
                        and Harada, J.J.</strong> (1998). <em>Arabidopsis</em> LEAFY COTYLEDON1 is sufficient to induce embryo development in vegetative cells. Cell <strong>93,</strong> 1195â1205.<a href="/cgi/external_ref?access_num=10.1016%2FS0092-8674%2800%2981463-4&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000074491100014&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=9657152&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB32" id="BIB32"/>
                     <strong>Maity, S.N., and De Crombrugghe, B.</strong> (1998). Role of the CCAAT-binding protein CBF/NF-Y in transcription. Trends Biochem. Sci. <strong>23,</strong> 174â178.<a href="/cgi/external_ref?access_num=10.1016%2FS0968-0004%2898%2901201-8&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000073724800006&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=9612081&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB33" id="BIB33"/>
                     <strong>Mantovani, R.</strong> (1999). The molecular biology of the CCAAT-binding factor NF-Y. Gene <strong>239,</strong> 15â27.<a href="/cgi/external_ref?access_num=10.1016%2FS0378-1119%2899%2900368-6&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000083291600002&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=10571030&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB34" id="BIB34"/>
                     <strong>Mayer, K.F.X., Schoof, H., Haecker, A., Lenhard, M., Juergens, G., and Laux, T.</strong> (1998). Role of <em>WUSCHEL</em> in regulating stem cell fate in the Arabidopsis shoot meristem. Cell <strong>95,</strong> 805â815.<a href="/cgi/external_ref?access_num=10.1016%2FS0092-8674%2800%2981703-1&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000077498800010&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=9865698&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB35" id="BIB35"/>
                     <strong>Meinke, D.W.</strong> (1992). A homoeotic mutant of <em>Arabidopsis thaliana</em> with leafy cotyledons. Science <strong>258,</strong> 1647â1650.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=258/5088/1647">[Abstract/<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB36" id="BIB36"/>
                     <strong>Meinke, D.W., Franzmann, L.H., Nickle, T.C., and Yeung, E.C.</strong> (1994). <em>leafy cotyledon</em> mutants of Arabidopsis. Plant Cell <strong>6,</strong> 1049â1064.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=6/8/1049">[Abstract]</a></p>
                  <p><a name="BIB37" id="BIB37"/>
                     <strong>Odell, J.T., Nagy, F., and Chua, N.-H.</strong> (1985). Identification of DNA sequences required for activity of cauliflower mosaic virus 35S promoter. Nature <strong>313,</strong> 810â812.<a href="/cgi/external_ref?access_num=10.1038%2F313810a0&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=3974711&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB38" id="BIB38"/>
                     <strong>Parcy, F., Valon, C., Kohara, A., Misera, S., and Giraudat, J.</strong> (1997). The ABSCISIC ACID-INSENSITIVE3, FUSCA3, and LEAFY COTYLEDON1 loci act in concert to control multiple aspects of Arabidopsis
                     seed development. Plant Cell <strong>9,</strong> 1265â1277.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=9/8/1265">[Abstract]</a></p>
                  <p><a name="BIB39" id="BIB39"/>
                     <strong>Puziss, J.W., Hardy, T.A., Johnson, R.B., Roach, P.J., and Hieter, P.</strong> (1994). MDS1, a dosage suppressor of an mck1 mutant, encodes a putative yeast homolog of glycogen synthase kinase 3. Mol.
                     Cell. Biol. <strong>14,</strong> 831â839.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=mcb&amp;resid=14/1/831">[Abstract/<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB40" id="BIB40"/>
                     <strong>Schneider, J.C., and Guarente, L.</strong> (1991). Regulation of the yeast <em>CYT1</em> gene encoding cytochrome c<sub>1</sub> by HAP1 and HAP2/3/4. Mol. Cell. Biol. <strong>11,</strong> 4934â4942.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=mcb&amp;resid=11/10/4934">[Abstract/<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB41" id="BIB41"/>
                     <strong>Sinha, S., Kim, I.S., Sohn, K.-Y., de Crombrugghe, B., and Maity, S.N.</strong> (1996). Three classes of mutations in the A subunit of the CCAAT-binding factor CBF delineate functional domains involved
                     in the three-step assembly of the CBF-DNA complex. Mol. Cell. Biol. <strong>16,</strong> 328â337.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=mcb&amp;resid=16/1/328">[Abstract/<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB42" id="BIB42"/>
                     <strong>Stone, S.L., Kwong, L.W., Yee, K.M., Pelletier, J., Lepiniec, L., Fischer, R.L., Goldberg, R.B., and Harada, J.J.</strong> (2001). LEAFY COTYLEDON2 encodes a B3 domain transcription factor that induces embryo development. Proc. Natl. Acad. Sci.
                     USA <strong>98,</strong> 11806â11811.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=98/20/11806">[Abstract/<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB43" id="BIB43"/>
                     <strong>Swofford, D.L., Olsen, G.J., Waddell, P.J., and Hills, D.</strong> (1996). Phylogenic inference. In Molecular Systematics, 2nd ed, C. Moritz, D.M. Hills, and B.K. Mable, eds (Sunderland, MA:
                     Sinauer and Associates), pp. 407â514.
                     
                  </p>
                  <p><a name="BIB44" id="BIB44"/>
                     <strong>Thompson, J.D., Gibson, T.J., Plewniak, F., Jeanmougin, F., and Giggins, D.G.</strong> (1997). The CLUSTALX Windows interface: Flexible strategies for multiple sequence alignment aided by quality analysis tools.
                     Nucleic Acids Res. <strong>25,</strong> 4876â4882.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=nar&amp;resid=25/24/4876">[Abstract/<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB45" id="BIB45"/>
                     <strong>Trueblood, C.E., Wright, R.M., and Poyton, R.O.</strong> (1988). Differential regulation of the two genes encoding <em>Saccharomyces cerevisiae</em> cytochrome oxidase subunit V by heme and <em>HAP2</em> and <em>REO1</em> genes. Mol. Cell. Biol. <strong>8,</strong> 4537â4540.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=mcb&amp;resid=8/10/4537">[Abstract/<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB46" id="BIB46"/>
                     <strong>Vicient, C.M., Bies-Etheve, N., and Delseny, M.</strong> (2000). Changes in gene expression in the leafy cotyledon1 (lec1) and fusca3 (fus3) mutants of <em>Arabidopsis thaliana</em> L. J. Exp. Bot. <strong>51,</strong> 995â1003.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=jexbot&amp;resid=51/347/995">[Abstract/<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB47" id="BIB47"/>
                     <strong>West, M.A.L., and Harada, J.J.</strong> (1993). Embryogenesis in higher plants: An overview. Plant Cell <strong>5,</strong> 1361â1369.<a href="/cgi/ijlink?linkType=PDF&amp;journalCode=plantcell&amp;resid=5/10/1361">[<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB48" id="BIB48"/>
                     <strong>West, M.A.L., Matsudaira Yee, K.L., Danao, J., Zimmerman, J.L., Fischer, R.L., Goldberg, R.B., and Harada, J.J.</strong> (1994). <em>LEAFY COTYLEDON1</em> is an essential regulator of late embryogenesis and cotyledon identity in Arabidopsis. Plant Cell <strong>6,</strong> 1731â1745.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=6/12/1731">[Abstract/<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB49" id="BIB49"/>
                     <strong>Weterings, K., Apuya, N.R., Bi, Y., Fischer, R.L., Harada, J.J., and Goldberg, R.B.</strong> (2001). Regional localization of suspensor mRNAs during early embryo development. Plant Cell <strong>13,</strong> 2409â2425.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=13/11/2409">[Abstract/<span style=" color: CC0000; ">Free</span>Â FullÂ Text]</a></p>
                  <p><a name="BIB50" id="BIB50"/>
                     <strong>Wright, K.L., Vilen, B.J., Itoh-Lindstrom, Y., Moore, T.L., Li, G., Criscitiello, M., Cogswell, P., Clarke, J.B., and Ting,
                        J.P.-Y.</strong> (1994). CCAAT box binding protein NF-Y facilitates in vivo recruitment of upstream DNA binding transcription factors. EMBO
                     J. <strong>13,</strong> 4042â4053.<a href="/cgi/external_ref?access_num=A1994PF60600014&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=8076600&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB51" id="BIB51"/>
                     <strong>Xing, Y., Fikes, J.D., and Guarente, L.</strong> (1993). Mutations in yeast HAP/HAP3 define a hybrid CCAAT box binding domain. EMBO J. <strong>12,</strong> 4647â4655.<a href="/cgi/external_ref?access_num=A1993MF99400019&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=8223474&amp;link_type=MED">[Medline]</a></p>
               </div>
      <span class="highwire-journal-article-marker-end"/>
    </div>
  </body>
</html>
