<?xml version="1.0"?>
<html>
  <body>
    <div class="article fulltext-view" itemprop="articleBody"><span class="highwire-journal-article-marker-start"/><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">© 2004 American Society of Plant Biologists</li>
               </ul><h1 id="article-title-1" itemprop="headline"><em>MUCILAGE-MODIFIED4</em> Encodes a Putative Pectin Biosynthetic Enzyme Developmentally Regulated by <em>APETALA2, TRANSPARENT TESTA GLABRA1</em>, and <em>GLABRA2</em> in the Arabidopsis Seed Coat<a id="xref-fn-1-1" class="xref-fn" href="#fn-1">1</a></h1>
               <div class="contributors">
                  <ol class="contributor-list" id="contrib-group-1"><li class="contributor" id="contrib-1" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="name" itemprop="name"><a class="name-search" href="/search?author1=Tamara+L.+Western&amp;sortspec=date&amp;submit=Submit">Tamara L. Western</a></span><a id="xref-fn-2-1" class="xref-fn" href="#fn-2">2</a>, 
                     </li>
                     <li class="contributor" id="contrib-2" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="name" itemprop="name"><a class="name-search" href="/search?author1=Diana+S.+Young&amp;sortspec=date&amp;submit=Submit">Diana S. Young</a></span>, 
                     </li>
                     <li class="contributor" id="contrib-3" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="name" itemprop="name"><a class="name-search" href="/search?author1=Gillian+H.+Dean&amp;sortspec=date&amp;submit=Submit">Gillian H. Dean</a></span>, 
                     </li>
                     <li class="contributor" id="contrib-4" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="name" itemprop="name"><a class="name-search" href="/search?author1=Wei+Ling+Tan&amp;sortspec=date&amp;submit=Submit">Wei Ling Tan</a></span><a id="xref-fn-3-1" class="xref-fn" href="#fn-3">3</a>, 
                     </li>
                     <li class="contributor" id="contrib-5" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="name" itemprop="name"><a class="name-search" href="/search?author1=A.+Lacey+Samuels&amp;sortspec=date&amp;submit=Submit">A. Lacey Samuels</a></span> and 
                     </li>
                     <li class="last" id="contrib-6"><span class="name"><a class="name-search" href="/search?author1=George+W.+Haughn&amp;sortspec=date&amp;submit=Submit">George W. Haughn</a></span><a id="xref-corresp-1-1" class="xref-corresp" href="#corresp-1">*</a></li>
                  </ol><ol class="affiliation-list"><li class="aff"><a id="aff-1" name="aff-1"/><address>Botany Department, University of British Columbia, 6270 University Boulevard, Vancouver, British Columbia, Canada V6T 1Z4</address>
                     </li>
                  </ol><ol class="corresp-list"><li class="corresp" id="corresp-1"><a class="rev-xref" href="#xref-corresp-1-1">↵</a><span class="corresp-label">*</span> Corresponding author; email <span class="em-link"><span class="em-addr">haughn{at}interchange.ubc.ca</span></span>; fax 604-822-6089. 
                     </li>
                  </ol></div>
               <div class="section abstract" id="abstract-1" itemprop="description">
                  <div class="section-nav">
                     <div class="nav-placeholder"> </div><a href="#sec-1" title="Next Section" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>Abstract</h2>
                  
                  <p id="p-1">The Arabidopsis seed coat epidermis undergoes a complex process of differentiation that includes the biosynthesis and secretion
                     of large quantities of pectinaceous mucilage, cytoplasmic rearrangement, and secondary cell wall biosynthesis. Mutations in
                     <em>MUM4</em> (<em>MUCILAGE-MODIFIED4</em>) lead to a decrease in seed coat mucilage and incomplete cytoplasmic rearrangement. We show that <em>MUM4</em> encodes a putative NDP-<span class="sc">l</span>-rhamnose synthase, an enzyme required for the synthesis of the pectin rhamnogalacturonan I, the major component of Arabidopsis
                     mucilage. This result suggests that the synthesis of monosaccharide substrates is a limiting factor in the biosynthesis of
                     pectinaceous seed coat mucilage. In addition, the reduced cytoplasmic rearrangement observed in the absence of a key enzyme
                     in pectin biosynthesis in <em>mum4</em> mutants establishes a causal link between mucilage production and cellular morphogenesis. The cellular phenotype seen in
                     <em>mum4</em> mutants is similar to that of several transcription factors (<em>AP2</em> [<em>APETALA2</em>], <em>TTG1</em> [<em>TRANSPARENT TESTA GLABRA1</em>], <em>TTG2 MYB61</em>, and <em>GL2</em> [<em>GLABRA2</em>]). Expression studies suggest that <em>MUM4</em> is developmentally regulated in the seed coat by <em>AP2, TTG1</em>, and <em>GL2</em>, whereas <em>TTG2</em> and <em>MYB61</em> appear to be regulating mucilage production through alternate pathway(s). Our results provide a framework for the regulation
                     of mucilage production and secretory cell differentiation.
                  </p>
                  
               </div>
               <div class="section" id="sec-1">
                  <div class="section-nav"><a href="#abstract-1" title="Abstract" class="prev-section-link"><span>Previous Section</span></a><a href="#sec-2" title="RESULTS" class="next-section-link"><span>Next Section</span></a></div>
                  
                  <p id="p-2">The mature seed coat, in addition to forming a protective layer around the embryo, can play roles in such processes as seed
                     dispersal and germination. One adaptation believed to act in these processes is the production of mucilage in the epidermal
                     cells of the seed coat. This specialization, known as myxospermy, is found in many families, including the Brassicaceae, Solanaceae,
                     Linaceae, and Plantaginaceae (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Grubert, 1981</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Boesewinkel and Bouman, 1995</a>). Seed imbibition in these plants leads to the formation of a gel capsule around the seed that is thought to aid hydration
                     and/or dispersal. Mucilage consists of pectins, a heterogeneous group of acidic polysaccharides that form a gel-like matrix.
                     In the primary cell wall, pectins form the matrix in which cellulose microfibrils and hemicelluloses are embedded. Key pectins
                     include homo-GalUA (HGA) and rhamnogalacturonan I (RGI). HGA consists of an unbranched chain of α-1,4-linked GalUA residues
                     whose carboxyl groups may be modified through methyl-esterification. RGI has a backbone of alternating α-1,2-linked Rha and
                     α-1,4-linked GalUA residues, with sugar side chains attached on varying numbers of Rha residues. Both HGA and RGI are believed
                     to be synthesized in the Golgi apparatus through the activities of glycosyltransferases using nucleotide sugars imported from
                     the cytoplasm (for review, see <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Ridley et al., 2001</a>). Chemical analysis of Arabidopsis mucilage has shown that it is composed largely of α-1,2 linked Rha and α-1,4 linked GalUA,
                     suggesting that it is comprised primarily of RGI (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Penfield et al., 2001</a>). Immunofluorescence studies of mucilage have demonstrated the presence of methylesterified HGA (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Willats et al., 2001</a>); however, the authors did not test for the presence of RGI.
                  </p>
                  
                  <p id="p-3">The production of mucilage in the epidermal cells of the Arabidopsis seed coat is part of a coordinated developmental process
                     that begins with cell growth, followed by biosynthesis and polar secretion of large quantities of pectin, formation of a cytoplasmic
                     column through cytoplasmic constriction and vacuolar contraction, and, finally, the synthesis of a secondary cell wall (columella;
                     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beeckman et al., 2000</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Western et al., 2000</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Windsor et al., 2000</a>). Seed coat mucilage is dispensable under laboratory conditions in Arabidopsis and <em>mum</em> (<em>mucilage-modified</em>) mutants affecting mucilage production can be identified by a simple screening method (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Western et al., 2001</a>). Mutations in one of these genes, <em>MUM4</em>, yield a phenotype where mucilage is not released from hydrated mature seeds.
                  </p>
                  
                  <p id="p-4">Several genes encoding putative transcription factors have been implicated in seed coat epidermal development because mutations
                     in these genes result in seeds that fail to release mucilage upon hydration. Mutants in <em>AP2</em> (<em>APETALA2</em>), in addition to their defects in floral morphogenesis, lack differentiation past the growth phase of mucilage secretory
                     cells (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Jofuku et al., 1994</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Western et al., 2001</a>). <em>TTG1</em> (<em>TRANSPARENT TESTA GLABRA1</em>), <em>TTG2</em>, and <em>GL2</em> (<em>GLABRA2</em>), which were originally identified through their role in trichome specification, have defects in both mucilage and columella
                     production in the seed coat (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Koornneef, 1981</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Penfield et al., 2001</a>; <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Western et al., 2001</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Johnson et al., 2002</a>). It has been shown at the genetic and molecular levels in trichomes and root hairs that TTG1 interacts with a basic helix-loop-helix
                     (bHLH) protein and a tissue-specific MYB protein to activate both <em>GL2</em> and <em>TTG2</em> (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Payne et al., 2000</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Johnson et al., 2002</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Schiefelbein, 2003</a>). Recently, mutations in <em>MYB61</em> have been found to specifically affect both mucilage and columella production in the Arabidopsis seed coat (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Penfield et al., 2001</a>).
                  </p>
                  
                  <p id="p-5">The objective of this study was to characterize the role of <em>MUM4</em> in the development of the Arabidopsis seed coat. Using positional cloning of <em>MUM4</em>, we demonstrate that <em>MUM4</em> encodes a putative NDP-<span class="sc">l</span>-Rha synthase. Mutations in this gene lead to reduced mucilage in the seed coat and an altered columella. Expression studies
                     show that <em>MUM4</em> is developmentally regulated during seed coat differentiation such that its transcript levels are increased at the time of
                     mucilage production. Furthermore, <em>MUM4</em> appears to be a downstream target of a cascade of transcription factors that includes AP2, TTG1, and GL2. These results demonstrate
                     the importance of mucilage production for the morphology of seed coat epidermal cells and suggest a regulatory framework for
                     the control of seed coat epidermal differentiation.
                  </p>
                  
               </div>
               <div class="section" id="sec-2">
                  <div class="section-nav"><a href="#sec-1" title="Previous Section" class="prev-section-link"><span>Previous Section</span></a><a href="#sec-9" title="DISCUSSION" class="next-section-link"><span>Next Section</span></a></div>
                  
                  <h2>RESULTS</h2>
                  
                  <div id="sec-3" class="subsection">
                     
                     <h3><em>mum4</em> Mutant Phenotype
                     </h3>
                     
                     <p id="p-6">Two alleles of <em>mum4</em> (<em>mum4-1</em> and <em>mum4-2</em>) were identified in a screen for mutants that lacked extruded mucilage as visualized by staining with Ruthenium red (compare
                        <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1, A with D</a>; <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Western et al., 2001</a>). A T-DNA insertion allele, <em>mum4-3</em>, was also identified. Because all alleles of this mutant had similar phenotypes, only that of <em>mum4-1</em> is described.
                     </p>
                     
                     <p id="p-7">
                        
                        
                     </p>
                     <div id="F1" class="fig pos-float odd">
                        <div class="fig-inline"><a href="296/F1.expansion.html"><img alt="  Figure 1. " src="296/F1.small.gif"/></a><div class="callout"><span>View larger version:</span><ul class="callout-links"><li><a href="296/F1.expansion.html">In this window</a></li>
                                 <li><a class="in-nw" href="296/F1.expansion.html">In a new window</a></li>
                              </ul><ul class="fig-services"><li class="ppt-link"><a href="/powerpoint/134/1/296/F1">Download as PowerPoint Slide</a></li>
                              </ul></div>
                        </div>
                        <div class="fig-caption"><span class="fig-label">
                              Figure 1.
                              </span> 
                           
                           <p id="p-8" class="first-child">Structure and development of wild-type (Columbia-2 [Col-2]) and <em>mum4-1</em> seed coats. A, Whole-mount wild-type seed stained with Ruthenium red. Note red staining capsule of mucilage surrounding seeds.
                              B, Scanning electron micrograph of wild-type seed coat. Note thickened radial cell walls (arrow) and central columella (arrowhead)
                              in each cell. C, Cross section of mature wild-type seed coat epidermis. The cells have burst to release mucilage, leaving
                              cell wall fragments attached to tip of columellae (arrow). D, Whole-mount <em>mum4-1</em> seeds stained with Ruthenium red. No mucilage is visible around seeds. E, Scanning electron micrograph of <em>mum4-1</em> seed coat. Columellae are not visible. F, Cross section of mature <em>mum4-1</em> epidermal cells. Flattened columellae (arrow) are apparent and intact primary cell walls (arrowhead) surround faint pink
                              staining mucilage. G to I, Cross sections of developing wild-type epidermal cells. G, 4 DPA. H, 7 DPA, note accumulation of
                              pink-staining mucilage in cells. I, 10 DPA, the center of each cell contains cytoplasm that is surrounded by a developing
                              secondary cell wall staining blue (arrow), whereas both are surrounded by intensely stained mucilage (dark pink). J to L,
                              Developing <em>mum4-1</em> epidermal cells. J, 4 DPA. K, 7 DPA, pink staining mucilage is only apparent in top portion of cells. L, 10 DPA, the cytoplasm
                              fills most of the cell in a dome shape, with a thin, blue secondary cell wall (arrow) forming around it. Pink staining mucilage
                              is found only in the upper corners of the cells. C and F to L, Toluidine blue-stained sections of tissue fixed under aqueous
                              conditions. Scale bars: A and D = 100 μm, B and E = 20 μm, and C and F to L = 10 μm.
                           </p>
                           
                           <div class="sb-div caption-clear"/>
                        </div>
                     </div>
                     
                     <p id="p-9">Viewed with scanning electron microscopy, mature <em>mum4</em> seed coats are marked not only by the absence of mucilage extrusion upon hydration but also by the absence of volcano-shaped
                        columellae (compare <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1, B with E</a>). Light microscopy of sections of mature seeds revealed that both columellae and mucilage are present in <em>mum4</em> but reduced in comparison with wild type (compare <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1, C with F</a>). No other phenotypes were observed.
                     </p>
                     
                     <p id="p-10">To more fully characterize the <em>mum4</em> seed coat epidermal cell defects, their development was studied with light microscopy (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1, G-L</a>). Under our growth conditions, seed maturation to dry seed takes approximately 18 d and has been staged by DPA. In terms
                        of embryo growth, 4 DPA is early to mid globular stage, 7 DPA is mid-torpedo stage, whereas 10 DPA is late upturned-U stage
                        (this study; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Western et al., 2000</a>). The early stages of <em>mum4</em> epidermal cell differentiation are similar to that of wild type: The cells grow, and mucilage starts to accumulate, accompanied
                        by the initiation of intracellular cytoplasmic rearrangement to form a column in the center of the cell (compare wild type,
                        <a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1, G and H</a>, with <em>mum4</em>, <a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1, J and K</a>). Differences, however, were obvious at 10 DPA. Although wild-type seed coat cells have dark-pink staining mucilage in the
                        extracellular space and a thin secondary cell wall surrounding a central column of cytoplasm (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1I</a>), the extracellular space containing mucilage in <em>mum4</em> seed coat epidermal cells is smaller with lighter staining mucilage and the secondary cell wall has formed around a partially
                        formed cytoplasmic column atop a large vacuole (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Fig. 1L</a>). Thus, the seed coat appears to have less mucilage, and the flattened columella appears to be the result of reduced cytoplasmic
                        and vacuolar constriction.
                     </p>
                     
                     <p id="p-11">The possibility of a change in mucilage quantity or composition in <em>mum4</em> versus wild-type seed coats was addressed by quantifying monosaccharides using gas chromatography. Because <em>mum4</em> seed coats release mucilage in the presence of ammonium oxalate, mucilage could be extracted from intact <em>mum4</em> seeds for direct comparison with wild-type mucilage. Trimethylsilane (TMS) derivatives were used because this method allows
                        detection of GalUA and neutral sugars simultaneously. The results showed that, for equal seed masses, less sugar was extracted
                        from <em>mum4</em> mucilage, suggesting that less mucilage is synthesized compared with wild-type seeds (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>). In addition, the decrease in monosaccharides in <em>mum4</em> mucilage was limited primarily to two sugars, GalUA and a single neutral sugar (identified as either Rha or Fuc; <a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2</a>; H<sub>0</sub> μ<sub>1</sub> = μ<sub>2</sub>, GalUA, T = 7.897, <em>P</em> &lt; 0.01; Rha, T = 6.960, <em>P</em> = 0.01). Because the identity of the neutral sugar could not be completely resolved using TMS derivatives, alditol acetate
                        derivatization, which gives a single derivative for each monosaccharide, was used to confirm that it is Rha (Fuc found only
                        at trace levels in wild-type mucilage; data not shown). These results are consistent with the suggestion that Arabidopsis
                        mucilage is comprised primarily of relatively unbranched RGI (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Penfield et al., 2001</a>).
                     </p>
                     
                     <p id="p-12">
                        
                        
                     </p>
                     <div id="F2" class="fig pos-float odd">
                        <div class="fig-inline"><a href="296/F2.expansion.html"><img alt="  Figure 2. " src="296/F2.small.gif"/></a><div class="callout"><span>View larger version:</span><ul class="callout-links"><li><a href="296/F2.expansion.html">In this window</a></li>
                                 <li><a class="in-nw" href="296/F2.expansion.html">In a new window</a></li>
                              </ul><ul class="fig-services"><li class="ppt-link"><a href="/powerpoint/134/1/296/F2">Download as PowerPoint Slide</a></li>
                              </ul></div>
                        </div>
                        <div class="fig-caption"><span class="fig-label">
                              Figure 2.
                              </span> 
                           
                           <p id="p-13" class="first-child">GalUA and Rha levels of wild-type (Col-2) and <em>mum4-1</em> mucilage. Comparison of GalUA and Rha levels detected in mucilage isolated from intact <em>mum4-1</em> seeds (white bars) versus wild-type mucilage (black bars, Col-2) using TMS derivatization and gas chromatography. The bars
                              represent micrograms of sugar per milligram of seed as determined by comparison with an internal standard. Analyses were done
                              in triplicate. Error bars = <span class="sc">sd</span>.
                           </p>
                           
                           <div class="sb-div caption-clear"/>
                        </div>
                     </div>
                     
                  </div>
                  
                  <div id="sec-4" class="subsection">
                     
                     <h3>Arabidopsis Mucilage Is Largely Composed of RGI</h3>
                     
                     <p id="p-14">To further investigate the effect of <em>mum4</em> mutations on mucilage production, immunofluorescence with antipectin antibodies was used. Whole-mount immunofluorescence
                        performed on wild-type Arabidopsis seeds using the polyclonal antipectin antibody α-poly-GalUA (PGA)/RGI gave significant
                        labeling of the mucilage without pretreatment (dry seeds added directly to fixative; <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3A</a>). This polyclonal antibody, although raised to RGI, has some cross-reactivity to HGA (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lynch and Staehelin, 1992</a>). However, JIM5 and JIM7 monoclonal antibodies specific for HGA with moderate or higher states of methylation, respectively,
                        failed to cross-react with the mucilage under the same conditions (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3C</a>; data not shown). Slight (JIM5; data not shown) or moderate (JIM7; <a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3D</a>) amount of labeling of extruded mucilage could be observed only after a pretreatment of imbibition with extensive shaking
                        before fixation. These results suggest that most of the binding found against wild-type mucilage can be attributed to RGI,
                        rather than HGA. When <em>mum4</em> seeds were pretreated by shaking in ammonium oxalate, α-PGA/RGI labeling of a thin ring around the seed was observed (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3B</a>), suggesting less mucilage is produced in <em>mum4</em> seeds compared with wild type.
                     </p>
                     
                     <p id="p-15">
                        
                        
                     </p>
                     <div id="F3" class="fig pos-float odd">
                        <div class="fig-inline"><a href="296/F3.expansion.html"><img alt="  Figure 3. " src="296/F3.small.gif"/></a><div class="callout"><span>View larger version:</span><ul class="callout-links"><li><a href="296/F3.expansion.html">In this window</a></li>
                                 <li><a class="in-nw" href="296/F3.expansion.html">In a new window</a></li>
                              </ul><ul class="fig-services"><li class="ppt-link"><a href="/powerpoint/134/1/296/F3">Download as PowerPoint Slide</a></li>
                              </ul></div>
                        </div>
                        <div class="fig-caption"><span class="fig-label">
                              Figure 3.
                              </span> 
                           
                           <p id="p-16" class="first-child">Immunofluorescence of wild-type (Col-2) and <em>mum4-1</em> mucilage with antipectin antibodies. A, Immersion immunofluorescence of whole wild-type seeds, labeled with α-PGA/RGI primary
                              and anti-rabbit Alexa 594 secondary antibody. Note autofluorescence of seed itself and stain of mucilage capsule surrounding
                              the seed. B, Immersion immunofluorescence of whole <em>mum4-1</em> seed, labeled with α-PGA/RGI after shaking in ammonium oxalate. The stained mucilage capsule is thin compared with wild type.
                              C and D, Immersion immunofluorescence of whole wild-type seeds, labeled with JIM7 as primary antibody, followed by anti-rat
                              fluorescein isothiocyanate secondary antibody. C, Seed treated as in A with no shaking pretreatment. No stain of mucilage
                              capsule. D, Seed pretreated by imbibition of extensive shaking in water. Stain of mucilage capsule is apparent. E to H, Immunofluorescence
                              of sections of 9-DPA seeds incubated with α-PGA/RGI or serum control. E, Wild-type seed incubated with α-PGA/RGI antibody,
                              epidermal cells are labeled on seed surface. F, Wild-type seed incubated with nonimmune normal rabbit serum control, no staining
                              of seed epidermis, same photographic conditions as E. G, Direct comparison of wild-type seed and H <em>mum4-1</em> seed show reduced α-PGA/RGI label in <em>mum4</em> mutant. Scale bars: A to F = 100 μm and G and H = 50 μm.
                           </p>
                           
                           <div class="sb-div caption-clear"/>
                        </div>
                     </div>
                     
                     <p id="p-17">To examine the mucilage in situ, samples were cryo-fixed/freeze substituted, resin embedded, sectioned, and then labeled by
                        indirect immunofluorescence. Labeling was seen on thick sections of developing wild-type seed coats stained with α-PGA/RGI
                        (9 DPA; <a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3E</a>), with fluorescence in the epidermal cells on either side of a central, unstained space. Control experiments including omitting
                        the primary antibody or substituting nonimmune, normal rabbit serum for the primary antibody, abolished the fluorescent signal
                        (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3F</a>). The similarity in staining pattern with α-PGA/RGI in sections compared with the pink staining substance in toluidine blue-stained
                        sections (e.g. <a id="xref-fig-1-9" class="xref-fig" href="#F1">Fig. 1I</a>) confirms the identity of the pink staining substance as pectinaceous mucilage. When thick sections of <em>mum4</em> seeds were treated with α-PGA/RGI, only a small amount of labeling was observed on the outer edges of the epidermal cells
                        (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig. 3H</a>) compared with wild-type seeds (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Fig. 3G</a>), once again suggesting a reduced amount of mucilage in <em>mum4</em> seeds.
                     </p>
                     
                  </div>
                  
                  <div id="sec-5" class="subsection">
                     
                     <h3>Molecular Cloning of <em>MUM4</em></h3>
                     
                     <p id="p-18">To identify why <em>mum4</em> seeds had reduced mucilage, we cloned <em>MUM4</em> by map-based cloning. <em>MUM4</em> was mapped to a 144-kb region spanning the bacterial artificial chromosomes F12M16 and T3F20. To identify the exact locus,
                        the <em>mum4-3</em> T-DNA insertion allele was probed with sequences derived from these bacterial artificial chromosomes, and an insertion was
                        identified in a region of T3F20 containing four predicted genes. PCR performed using gene-specific primers in combination
                        with a T-DNA right border primer led to the identification of a T-DNA insertion in the predicted gene <em>At1g53500</em>.
                     </p>
                     
                     <p id="p-19">Next, we isolated a putative full-length cDNA by reverse transcriptase (RT)-PCR using a series of primers upstream from the
                        predicted translation start site. Sequencing of a putative full-length <em>At1g53500</em> transcript revealed that the gene consists of three exons and two introns (including one in the 5′-untranslated region [UTR])
                        and encodes a transcript of 2,477 bp (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4A</a>).
                     </p>
                     
                     <p id="p-20">
                        
                        
                     </p>
                     <div id="F4" class="fig pos-float odd">
                        <div class="fig-inline"><a href="296/F4.expansion.html"><img alt="  Figure 4. " src="296/F4.small.gif"/></a><div class="callout"><span>View larger version:</span><ul class="callout-links"><li><a href="296/F4.expansion.html">In this window</a></li>
                                 <li><a class="in-nw" href="296/F4.expansion.html">In a new window</a></li>
                              </ul><ul class="fig-services"><li class="ppt-link"><a href="/powerpoint/134/1/296/F4">Download as PowerPoint Slide</a></li>
                              </ul></div>
                        </div>
                        <div class="fig-caption"><span class="fig-label">
                              Figure 4.
                              </span> 
                           
                           <p id="p-21" class="first-child">Structure of the <em>MUM4, RHM1</em>, and <em>RHM3</em> genes and complementation of <em>mum4-1</em> with <em>MUM4</em> coding region. A, Exonintron structures are based on comparison of the genomic and cDNA sequences. The coding region is shown
                              as black boxes, whereas the gray boxes represent 5′- and 3′-UTRs. The location of the two domains with similarity to bacterial
                              nucleotide sugar interconversion enzymes are represented as thin bars underneath the <em>MUM4</em> coding region.|, Location of the two EMS alleles (<em>mum4-1</em> and <em>mum4-2</em>); ∨, Location of the three T-DNA insertion alleles (<em>mum4-3, mum4-4</em>, and <em>mum4-5</em>). B, Complementation of <em>mum4-1</em> with pMUM4g. Mucilage capsule is observed around seed stained with Ruthenium red. C, <em>mum4-1</em> seed transformed with empty vector pGREEN0229. No mucilage release is observed. Scale bars = 100 μm in B and C.
                           </p>
                           
                           <div class="sb-div caption-clear"/>
                        </div>
                     </div>
                     
                     <p id="p-22">To confirm that <em>At1g53500</em> is <em>MUM4</em>, we sequenced this locus in <em>mum4-1, mum4-2</em>, and <em>mum4-3</em>. All three alleles contain mutations in the second exon. <em>mum4-1</em> and <em>mum4-2</em> contain point mutations leading to G to A transitions at nucleotides 595 and 886, respectively (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4A</a>). These missense mutations should result in a conserved Asp residue being replaced with an Asn in <em>mum4-1</em> and a Gly being replaced with an Arg in <em>mum4-2</em> (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5</a>). <em>mum4-3</em> contains a T-DNA insertion at nucleotide 1,637 (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4A</a>). The similar phenotype between the missense alleles and that of the <em>mum4-3</em> insertional allele that lacks detectable transcript (data not shown) suggests that these changes have a significant effect
                        on protein activity. A database search of the Salk Institute sequence-indexed Arabidopsis T-DNA insertion lines identified
                        two new alleles of <em>mum4, mum4-4</em>, and <em>mum4-5</em> (Salk_085051 and 038898, respectively), both of which also have insertions in the second exon (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4A</a>). Based on the Ruthenium red assay, both of these mutants have a phenotype (data not shown) similar to <em>mum4-1</em> (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Fig. 1D</a>). The wild-type <em>At1g53500</em> sequence plus 2.1 kb upstream and 0.4 kb downstream sequence was cloned to give the plasmid pMUM4g. Transformation of <em>mum4-1</em> plants with pMUM4g versus an empty vector control showed that the wild-type <em>At1g53500</em> genomic clone could rescue the mucilage phenotype (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4, B and C</a>). Together, these results demonstrate that <em>At1g53500</em> is <em>MUM4</em>.
                     </p>
                     
                     <p id="p-23">
                        
                        
                     </p>
                     <div id="F5" class="fig pos-float odd">
                        <div class="fig-inline"><a href="296/F5.expansion.html"><img alt="  Figure 5. " src="296/F5.small.gif"/></a><div class="callout"><span>View larger version:</span><ul class="callout-links"><li><a href="296/F5.expansion.html">In this window</a></li>
                                 <li><a class="in-nw" href="296/F5.expansion.html">In a new window</a></li>
                              </ul><ul class="fig-services"><li class="ppt-link"><a href="/powerpoint/134/1/296/F5">Download as PowerPoint Slide</a></li>
                              </ul></div>
                        </div>
                        <div class="fig-caption"><span class="fig-label">
                              Figure 5.
                              </span> 
                           
                           <p id="p-24" class="first-child">Alignment of the amino acid sequence of the MUM4 protein with those of RHM1 and RHM3. The putative 4,6-dehydratase and 4-reductase
                              domains within MUM4 are marked with single and double underlines, respectively, whereas the highly conserved NAD(P)<sup>+</sup> cofactor-binding (GxxGxxG) and active site catalytic couple (YxxxK) motifs are indicated above the sequence alignment. The
                              amino acid substitutions (<em>mum4-1</em> and <em>mum4-2</em>) and sites of T-DNA insertion (<em>mum4-3, mum4-4</em>, and <em>mum4-5</em>) are also marked.
                           </p>
                           
                           <div class="sb-div caption-clear"/>
                        </div>
                     </div>
                     
                     <p id="p-25"><em>MUM4</em> encodes a protein of 667 amino acids (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5</a>). Comparison with GenBank sequences suggested that MUM4 contains two domains: an N-terminal domain with similarity to bacterial
                        dTDP-<span class="sc">d</span>-Glc-4,6-dehydratases (44% identical, 61% similar to RfbB [COG1088] consensus) and a C-terminal domain with some similarity
                        to bacterial 4-reductases (20% identical, 34% similar to RfbD [COG1091] consensus) (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Tatusov et al., 2000</a>; Figs. <a id="xref-fig-4-6" class="xref-fig" href="#F4">4A</a> and <a id="xref-fig-5-3" class="xref-fig" href="#F5">5</a>). Both domains contain the conserved N-terminal NAD<sup>+</sup> binding (GxxGxxG) and active site catalytic couple (YxxxK) motifs found in members of the reductase/epimerase/dehydrogenase
                        protein superfamily (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Graninger et al., 1999</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Allard et al., 2001</a>). Both of the missense alleles, <em>mum4-1</em> and <em>mum4-2</em>, change codons of conserved amino acids (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig. 5</a>) in the N-terminal, putative NDP-<span class="sc">d</span>-Glc 4,6-dehydratase domain.
                     </p>
                     
                     <p id="p-26">A comparison with other Arabidopsis genes predicted from genome annotation revealed that <em>MUM4</em> is a member of a small gene family of putative nucleotide sugar interconversion factors that was initially described in an
                        in silico analysis by Reiter and Vanzin (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">2001</a>). The gene family consists of three genes, <em>RHM1</em> (<em>At1g78570</em>), <em>RHM2</em> (<em>MUM4</em>) and <em>RHM3</em> (<em>At3g14790</em>), which are 83% to 89% identical at the amino acid level (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Fig. 5</a>). This suggests that there are at least three genes with a similar function.
                     </p>
                     
                  </div>
                  
                  <div id="sec-6" class="subsection">
                     
                     <h3>Expression Pattern of <em>MUM4</em></h3>
                     
                     <p id="p-27">The expression pattern for <em>MUM4</em> mRNA in different tissues was determined qualitatively using RT-PCR at a saturating cycle number. As suggested by expressed
                        sequence tag data, <em>MUM4</em> transcripts are apparent in all tissues tested: stems, roots, leaves, seedlings, inflorescence tips, and siliques (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig. 6A</a>). A similar analysis was done for <em>RHM1</em> and <em>RHM3</em>. As with <em>MUM4</em>, both <em>RHM1</em> and <em>RHM3</em> transcripts were detected in all tissues (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 6A</a>). <em>RHM3</em>, however, unlike <em>MUM4</em> and <em>RHM1</em>, appeared to have only low expression in some tissues (leaves and inflorescences) because bands for these tissues were not
                        saturated in several trials, even at the high cycle number used for qualitative RT-PCR.
                     </p>
                     
                     <p id="p-28">
                        
                        
                     </p>
                     <div id="F6" class="fig pos-float odd">
                        <div class="fig-inline"><a href="296/F6.expansion.html"><img alt="  Figure 6. " src="296/F6.small.gif"/></a><div class="callout"><span>View larger version:</span><ul class="callout-links"><li><a href="296/F6.expansion.html">In this window</a></li>
                                 <li><a class="in-nw" href="296/F6.expansion.html">In a new window</a></li>
                              </ul><ul class="fig-services"><li class="ppt-link"><a href="/powerpoint/134/1/296/F6">Download as PowerPoint Slide</a></li>
                              </ul></div>
                        </div>
                        <div class="fig-caption"><span class="fig-label">
                              Figure 6.
                              </span> 
                           
                           <p id="p-29" class="first-child"><em>MUM4, RHM1</em>, and <em>RHM3</em> expression in Arabidopsis plants. A, Saturating, qualitative RT-PCR of RNA isolated from wild-type (Col-2) stem, root, leaf,
                              seedling, inflorescence, and siliques using gene-specific primers for <em>MUM4, RHM1</em>, and <em>RHM3</em>. The loading control is <em>GAPC</em>, which encodes cytosolic glyceraldehyde-3-phosphate dehydrogenase. B, Changes in <em>MUM4</em> expression during silique development. RNA gel-blot analysis using <em>MUM4</em> probe on total RNA from wild-type (Col-2) inflorescence, 4, 7, and 10 DPA wild-type siliques, and 7 DPA <em>ap2-7</em> siliques. Ethidium bromide-stained rRNA is shown as loading control.
                           </p>
                           
                           <div class="sb-div caption-clear"/>
                        </div>
                     </div>
                     
                     <p id="p-30">Because the only obvious phenotype in <em>mum4</em> plants is the defect in the seed coat epidermis, we looked at the expression of <em>MUM4</em> during seed coat differentiation (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig. 6B</a>). RNA was extracted from siliques at three stages of development: (a) before mucilage synthesis (4 DPA; see <a id="xref-fig-1-11" class="xref-fig" href="#F1">Fig. 1G</a>), (b) midmucilage synthesis (7 DPA; <a id="xref-fig-1-12" class="xref-fig" href="#F1">Fig. 1H</a>), and (c) after the completion of mucilage synthesis (10 DPA; <a id="xref-fig-1-13" class="xref-fig" href="#F1">Fig. 1I</a>), and subjected to northern analysis. This experiment showed that although <em>MUM4</em> is expressed throughout seed coat differentiation, transcript levels are highest at the time of mucilage synthesis (7 DPA;
                        <a id="xref-fig-6-4" class="xref-fig" href="#F6">Fig. 6B</a>). To test if the higher levels of <em>MUM4</em> detected at 7 DPA may be largely restricted to the seed coat, <em>ap2-7</em> mutants were used as a control in gel blots (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Fig. 6B</a>). <em>ap2</em> was chosen because the outer two layers of the seed coat of <em>ap2</em> mutant seeds fail to differentiate, whereas all other cell types of the seed (embryo, endosperm, and seed coat endothelium)
                        appear normal (<a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Western et al., 2001</a>). The low level of <em>MUM4</em> transcript seen in 7-DPA <em>ap2-7</em> siliques suggests that a large proportion of the <em>MUM4</em> transcript at 7 DPA in wild-type siliques may be specific to the outer layers of the seed coat.
                     </p>
                     
                  </div>
                  
                  <div id="sec-7" class="subsection">
                     
                     <h3>Maximal Accumulation of <em>MUM4</em> Transcript Requires Upstream Transcription Factors
                     </h3>
                     
                     <p id="p-31">Mutations in <em>AP2, TTG1, TTG2, GL2</em>, and <em>MYB61</em>, which encode putative transcription factors, lead to seeds with reduced mucilage in the seed coat. Furthermore, developmental
                        analyses have shown that the cellular phenotypes for <em>ttg1, gl2</em>, and <em>myb61</em> mucilage secretory cells are almost identical to those of <em>mum4</em> mutants, whereas the <em>ap2</em> mutant cellular phenotype is more severe (<a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Penfield et al., 2001</a>; <a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">Western et al., 2001</a>). Therefore, we tested whether the accumulation of <em>MUM4</em> transcript was altered in mutants for each of these genes. RNA was extracted from 7 DPA siliques of wild type and mutants
                        and subjected to northern hybridization (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig. 7A</a>). To control for ecotype differences between mutants isolated in Col-2 (<em>ap2-7</em> and <em>myb61-1</em>) versus L<em>er</em> (<em>ap2-1, gl2-1, ttg1-1</em>, and <em>ttg2-1</em>) backgrounds, both wild types were examined. Interestingly, <em>MUM4</em> appeared to be more highly expressed in the L<em>er</em> background, possibly because of background mutations affecting morphological differences in the seeds and/or siliques between
                        the two ecotypes. As mentioned above, the level of <em>MUM4</em> transcript was reduced in <em>ap2-7</em> mutant siliques compared with wild-type Col-2 siliques, a result that was confirmed with <em>ap2-1</em> in the L<em>er</em> background. Our analysis also indicated that the levels of <em>MUM4</em> RNA at 7 DPA are lower for <em>ttg1-1</em> and <em>gl2-1</em> siliques (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Fig. 7A</a>), suggesting that <em>AP2, TTG1</em>, and <em>GL2</em> each are required for maximum levels of <em>MUM4</em> expression at the time of mucilage production. Conversely, there was a slight increase in <em>MUM4</em> transcript levels in <em>myb61-1</em> and <em>ttg2-1</em> siliques compared with wild type (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Fig. 7A</a>). Thus, the requirement of <em>MYB61</em> and <em>TTG2</em> for mucilage biosynthesis is for some aspect other than the regulation of <em>MUM4</em> transcription.
                     </p>
                     
                     <p id="p-32">
                        
                        
                     </p>
                     <div id="F7" class="fig pos-float odd">
                        <div class="fig-inline"><a href="296/F7.expansion.html"><img alt="  Figure 7. " src="296/F7.small.gif"/></a><div class="callout"><span>View larger version:</span><ul class="callout-links"><li><a href="296/F7.expansion.html">In this window</a></li>
                                 <li><a class="in-nw" href="296/F7.expansion.html">In a new window</a></li>
                              </ul><ul class="fig-services"><li class="ppt-link"><a href="/powerpoint/134/1/296/F7">Download as PowerPoint Slide</a></li>
                              </ul></div>
                        </div>
                        <div class="fig-caption"><span class="fig-label">
                              Figure 7.
                              </span> 
                           
                           <p id="p-33" class="first-child">Expression of <em>MUM4</em> and genes encoding putative regulatory transcription factors during silique development. A, RNA gel-blot analysis using <em>MUM4</em> probe on total RNA from 7-DPA wild-type, <em>ap2-7, myb61-1</em> (Col-2), wild-type, <em>ap2-1, gl2-1, ttg1-1</em>, and <em>ttg2-1</em> (Landsberg <em>erecta</em> [L<em>er</em>]) siliques. Ethidium bromide stained rRNA is shown as loading control. B, Semiquantitative RT-PCR amplification of RNA isolated
                              from wild-type and <em>ap2-7</em> (Col-2) siliques at 4 and 7 DPA using gene-specific primers for <em>AP2, TTG1, TTG2, MYB61</em>, and <em>GL2</em>. The loading control is <em>GAPC</em>, which encodes cytosolic glyceraldehyde-3-phosphate dehydrogenase. C, Semiquantitative RT-PCR amplification of RNA isolated
                              from wild-type and <em>ttg1-1</em> (L<em>er</em>) 7-DPA siliques using gene-specific primers for <em>TTG2</em> and <em>GL2</em>. The loading control is <em>GAPC</em>, which encodes cytosolic glyceraldehyde-3-phosphate dehydrogenase.
                           </p>
                           
                           <div class="sb-div caption-clear"/>
                        </div>
                     </div>
                     
                  </div>
                  
                  <div id="sec-8" class="subsection">
                     
                     <h3>The Expression of Putative Regulatory Transcription Factors during Mucilage Secretory Cell Differentiation</h3>
                     
                     <p id="p-34">The <em>ap2</em> mutant phenotype differs from that of <em>ttg1, ttg2, gl2</em>, and <em>myb61</em> in that the differentiation of the mucilage secretory cells is halted before mucilage and columella production in <em>ap2</em> whereas the others resemble <em>mum4</em>, having low levels of mucilage and flattened columellae. Thus, phenotypically, <em>AP2</em> appears to be working upstream of the other genes. To determine whether AP2 acts by controlling the transcription of the
                        other genes, semiquantitative RT-PCR was used to determine the levels of <em>TTG1, TTG2, GL2</em>, and <em>MYB61</em> in <em>ap2-7</em> mutants versus wild type at 4 and 7 DPA. The results showed a significant decrease in transcript levels for <em>GL2</em> at both 4 and 7 DPA (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Fig. 7B</a>) and <em>TTG2</em> at 4 DPA (<em>TTG2</em> levels at 7 DPA too low to differentiate between wild type and <em>ap2</em>; <a id="xref-fig-7-5" class="xref-fig" href="#F7">Fig. 7B</a>). This result suggests that <em>AP2</em> is required for maximal levels of both <em>GL2</em> and <em>TTG2</em> expression during seed coat differentiation.
                     </p>
                     
                     <p id="p-35"><em>TTG1</em> has been shown to be required for maximal expression of both <em>TTG2</em> and <em>GL2</em> in leaves (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Johnson et al., 2002</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Schiefelbein, 2003</a>). To determine if the same is true in seed coats, the expression of both <em>GL2</em> and <em>TTG2</em> were tested in <em>ttg1-1</em> compared with wild-type <em>Ler</em> 7 DPA siliques using semiquantitative RT-PCR. The transcript levels for both <em>GL2</em> and <em>TTG2</em> were reduced in <em>ttg1-1</em> mutants (<a id="xref-fig-7-6" class="xref-fig" href="#F7">Fig. 7C</a>), suggesting that <em>TTG1</em> also acts upstream of <em>GL2</em> and <em>TTG2</em> in the seed coat.
                     </p>
                     
                  </div>
                  
               </div>
               <div class="section" id="sec-9">
                  <div class="section-nav"><a href="#sec-2" title="RESULTS" class="prev-section-link"><span>Previous Section</span></a><a href="#sec-13" title="MATERIALS AND METHODS" class="next-section-link"><span>Next Section</span></a></div>
                  
                  <h2>DISCUSSION</h2>
                  
                  <p id="p-36">The development of the mucilage secretory cells of the Arabidopsis seed coat is a complex process involving the biosynthesis
                     and secretion of a large quantity of pectinaceous mucilage, intracellular cytoplasmic rearrangement, and secondary cell wall
                     production. We have shown that the putative NDP-<span class="sc">l</span>-Rha synthase encoded by <em>MUM4</em> is required for wild-type levels of mucilage biosynthesis and columella formation. Expression of <em>MUM4</em> is up-regulated during seed coat differentiation and is regulated by several transcription factors required for seed coat
                     epidermal cell differentiation. These data validate the utility of the mucilage secretory cells as a model system for polysaccharide
                     biosynthesis, establish a regulatory framework for seed coat epidermal differentiation, and provide a causal link between
                     pectin biosynthesis and cell morphogenesis.
                  </p>
                  
                  <div id="sec-10" class="subsection">
                     
                     <h3><em>MUM4</em> Encodes a Putative NDP-<span class="sc">l</span>-Rha Synthase
                     </h3>
                     
                     <p id="p-37">Our data support the hypothesis that <em>MUM4</em> encodes an enzyme involved in RGI biosynthesis. First, analysis of <em>mum4</em> mucilage using antibodies and gas chromatography have demonstrated a significant reduction in RGI and its composite backbone
                        monosaccharides Rha and GalUA in comparison with wild-type seeds. Second, cloning of <em>MUM4</em> revealed a putative protein containing similarity to bacterial nucleotide sugar interconversion enzymes, suggesting that
                        it, too, is required for the production of activated sugars. The N-terminal portion of MUM4 is most similar to bacterial dTDP-<span class="sc">d</span>-Glc 4,6-dehydratases, the first of three enzymes required for the conversion of dTDP-<span class="sc">d</span>-Glc to dTDP-<span class="sc">l</span>-Rha (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Tonetti et al., 1998</a>). These data favor a role for MUM4 in the synthesis of NDP-<span class="sc">l</span>-Rha, a key step in the production of RGI. A reverse genetics approach taken by another group has come to a similar conclusion
                        regarding the putative enzymatic function of MUM4 (also designated <em>RHM2</em> [<em>RHAMNOSE BIOSYNTHESIS 2</em>]; B. <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Usadel et al., 2004</a>). However, another role in RGI biosynthesis cannot be ruled out without evidence of enzymatic activity in the conversion
                        of NDP-<span class="sc">d</span>-Glc to NDP-<span class="sc">l</span>-Rha.
                     </p>
                     
                     <p id="p-38">Gram-negative bacteria such as <em>Escherichia coli</em> encode three separate enzymes (4,6-dehydratase, 3,5-epimerase, and 4-reductase) to convert dTDP-<span class="sc">d</span>-Glc to dTDP-<span class="sc">l</span>-Rha (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Tonetti et al., 1998</a>). The putative MUM4 protein not only contains an N-terminal domain with similarity to the bacterial dTDP-<span class="sc">d</span>-Glc 4,6-dehydratases but also a C-terminal domain with some similarity to 4-reductases (this study; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Reiter and Vanzin, 2001</a>). There is a precedent in Arabidopsis for a bifunctional 3,5 epimerase, 4-reductase in the synthesis of GDP-<span class="sc">l</span>-Fuc from GDP-<span class="sc">d</span>-Man (GER1, GER2) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bonin and Reiter, 2000</a>). Therefore, based on sequence analysis, it is attractive to postulate a single, multifunctional protein acting in the conversion
                        of NDP-<span class="sc">d</span>-Glc to NDP-<span class="sc">l</span>-Rha in Arabidopsis (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Reiter and Vanzin, 2001</a>).
                     </p>
                     
                     <p id="p-39"><em>MUM4</em> is a member of a small gene family consisting of three genes (<em>RHM1, MUM4/RHM2</em>, and <em>RHM3</em>; <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Reiter and Vanzin, 2001</a>) that exhibit high identity at both nucleotide and amino acid levels. The ubiquitous expression of <em>RHM1</em> and <em>RHM3</em> can be postulated to provide a mechanism for the formation of the normal primary cell wall in <em>mum4</em> mutants and the residual mucilage present in <em>mum4</em> seed coats. Redundancy in genes coding for putative NDP-<span class="sc">l</span>-Rha synthases is unsurprising because of the importance of the Rha-containing pectins RGI and RGII in primary cell walls.
                        In fact, the presence of multiple genes coding enzymes involved in nucleotide sugar interconversions is common in plants (<a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Reiter and Vanzin, 2001</a>). The roles of MUM4 and its paralogs may not be completely redundant, however, because the three genes do not have identical
                        patterns/levels of expression throughout all plant tissues (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Fig. 6A</a>; Arabidopsis expressed sequence tag database at The Institute for Genomic Research [<a href="http://www.tigr.org">http://www.tigr.org</a>]; T.L. Western and G.W. Haughn, unpublished data).
                     </p>
                     
                  </div>
                  
                  <div id="sec-11" class="subsection">
                     
                     <h3>Developmental Regulation of <em>MUM4</em> during Seed Coat Secretory Cell Differentiation by AP2, TTG1, and GL2
                     </h3>
                     
                     <p id="p-40"><em>MUM4</em> transcript increases in differentiating siliques at the time of mucilage production. Two lines of evidence suggest that this
                        up-regulation occurs in the seed coat epidermis to support mucilage biosynthesis. First, the only obvious phenotypic defect
                        in <em>mum4</em> plants occurs in the seed coat epidermis. Second, <em>MUM4</em> expression is severely attenuated in siliques of <em>ap2</em> mutants that fail to differentiate the outer two layers of the seed coat. Such a specific up-regulation of a putative NDP-<span class="sc">l</span>-Rha synthase may be required to provide extra Rha for the production of the large quantity of RGI required for mucilage synthesis.
                        If so, the amount of this enzyme must be the limiting factor in Rha biosynthesis and the amount of Rha a limiting factor in
                        RGI biosynthesis.
                     </p>
                     
                     <p id="p-41">Our data indicate that <em>MUM4</em> transcription is decreased in <em>ttg1</em> and <em>gl2</em> mutants but is essentially wild type in <em>ttg2</em> and <em>myb61</em> mutants (<a id="xref-fig-7-7" class="xref-fig" href="#F7">Fig. 7A</a>). This result suggests a regulatory framework for mucilage secretory cell differentiation in which there are at least two
                        pathways controlling mucilage biosynthesis in the seed coat. In one of these pathways, it appears that TTG1 and GL2 control
                        the transcription of <em>MUM4</em> (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Fig. 8</a>). In other epidermal systems, such as trichomes and root hairs, TTG1 has been shown to interact with a tissue-specific MYB
                        protein through the bHLH protein GLABRA3 (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Payne et al., 2000</a>; <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Schiefelbein, 2003</a>) to activate <em>GL2</em>. Therefore, we favor a model in which a seed coat-specific complex of TTG1-bHLH-MYB causes activation of <em>GL2</em> and the subsequent up-regulation of <em>MUM4</em> during mucilage production (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Fig. 8</a>). Recent results suggest that EGL3 (ENHANCER OF GLABRA3) and/or TT8 (TRANSPARENT TESTA8) are the bHLH proteins acting with
                        TTG1 in the seed coat (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Zhang et al., 2003</a>). A possible MYB component of this trimeric complex, <em>MYB23</em>, is expressed in developing seeds and groups to the same subfamily of MYBs as the bHLH-TTG1 interacting proteins GLABROUS1
                        and WEREWOLF (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kirik et al., 2001</a>).
                     </p>
                     
                     <p id="p-42">
                        
                        
                     </p>
                     <div id="F8" class="fig pos-float odd">
                        <div class="fig-inline"><a href="296/F8.expansion.html"><img alt="  Figure 8. " src="296/F8.small.gif"/></a><div class="callout"><span>View larger version:</span><ul class="callout-links"><li><a href="296/F8.expansion.html">In this window</a></li>
                                 <li><a class="in-nw" href="296/F8.expansion.html">In a new window</a></li>
                              </ul><ul class="fig-services"><li class="ppt-link"><a href="/powerpoint/134/1/296/F8">Download as PowerPoint Slide</a></li>
                              </ul></div>
                        </div>
                        <div class="fig-caption"><span class="fig-label">
                              Figure 8.
                              </span> 
                           
                           <p id="p-43" class="first-child">Proposed genetic pathway for the regulation of mucilage production during seed coat secretory cell differentiation. AP2 and
                              a TTG1 complex with a bHLH protein (candidates include EGL3 and/or TT8) and a tissue-specific MYB transcription factor activate
                              <em>GL2</em> and <em>TTG2. GL2</em> acts upstream of <em>MUM4</em>. In contrast, both <em>TTG2</em> and <em>MYB61</em> appear to affect aspects of mucilage production independent from <em>MUM4</em>. For details, see text.
                           </p>
                           
                           <div class="sb-div caption-clear"/>
                        </div>
                     </div>
                     
                     <p id="p-44">Correct regulation of <em>MUM4</em> transcription at the time of mucilage production does not require <em>TTG2</em> or <em>MYB61</em> (<a id="xref-fig-7-8" class="xref-fig" href="#F7">Fig. 7A</a>). However, <em>TTG1</em> is required to activate <em>TTG2</em> in the seed coat (<a id="xref-fig-7-9" class="xref-fig" href="#F7">Fig. 7C</a>) and in other tissues (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Johnson et al., 2002</a>). This result places <em>TTG2</em> downstream from <em>TTG1</em> in a second pathway that controls mucilage biosynthesis in the developing seed. This alternate pathway, along with another
                        that involves <em>MYB61</em>, may regulate the expression of other proteins involved in RGI synthesis including an NDP-<span class="sc">d</span>-GlcUA 4-epimerase (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Feingold, 1982</a>; <a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Reiter and Vanzin, 2001</a>), a nucleotide sugar transporter or an RGI-backbone glycosyltransferase. Alternatively, <em>MYB61</em> and <em>TTG2</em> may be involved in the transport of newly synthesized RGI to the plasma membrane. Feedback inhibition of mucilage production
                        in the absence of secretion has been demonstrated in the root caps of the maize (<em>Zea mays</em>) mutant Ageotropic (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Millar and Moore, 1990</a>).
                     </p>
                     
                     <p id="p-45"><em>AP2</em> encodes a putative transcription factor that is required for the differentiation of the outer two layers of the seed coat
                        (<a id="xref-ref-31-7" class="xref-bibr" href="#ref-31">Western et al., 2001</a>). Not surprisingly, <em>ap2</em> seeds lack mucilage and fail to activate <em>MUM4</em> transcription beyond its baseline level of expression (<a id="xref-fig-7-10" class="xref-fig" href="#F7">Fig. 7A</a>). Although <em>AP2</em> is required for maximum <em>GL2</em> and <em>TTG2</em> transcript levels (<a id="xref-fig-7-11" class="xref-fig" href="#F7">Fig. 7B</a>), transcription of <em>TTG1</em> and <em>MYB61</em> was independent of <em>AP2</em> activity (<a id="xref-fig-7-12" class="xref-fig" href="#F7">Fig. 7B</a>). Our data are consistent with the hypothesis that AP2 functions in parallel with TTG1 to activate <em>GL2</em> and <em>TTG2</em> in the seed coat epidermis (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Fig. 8</a>).
                     </p>
                     
                  </div>
                  
                  <div id="sec-12" class="subsection">
                     
                     <h3>Columella Production during Seed Coat Secretory Cell Differentiation</h3>
                     
                     <p id="p-46">The <em>mum4</em> phenotype is characterized by both a reduction in the amount of mucilage produced and by a flattened columella in the seed
                        coat epidermis. The flattened appearance of the columella in <em>mum4</em> seeds appears to be the result of incomplete cytoplasmic rearrangement and vacuole constriction, leaving a dome of cytoplasm
                        over which the secondary cell wall is laid. Although the defect in mucilage production is readily explained by the putative
                        function of <em>MUM4</em> as an NDP-<span class="sc">l</span>-Rha synthase, the connection with the columella shape change is less obvious. At least two hypotheses can explain the dependence
                        of cell morphogenesis on mucilage synthesis. The formation of the apoplastic space and cytoplasmic column may be driven by
                        the pressure of accumulating mucilage in the extracytoplasmic space and/or the formation of an osmotic gradient between the
                        cell and the hydrophilic pectin may lead to the loss of turgor and size reduction of the vacuole. Such passive processes may
                        contribute to cell morphogenesis but seem too simple to account for the apparent complexity of the shape of the cytoplasmic
                        column. An alternate but not mutually exclusive explanation is that the synthesis of mucilage may represent or lead to the
                        production of an oligosaccharide signal that stimulates the cell to undergo morphogenesis (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dumville and Fry, 2000</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Ridley et al., 2001</a>). Determination of the exact processes involved would be facilitated by the isolation of mutants affected specifically in
                        the shaping of the columella.
                     </p>
                     
                  </div>
                  
               </div>
               <div class="section" id="sec-13">
                  <div class="section-nav"><a href="#sec-9" title="DISCUSSION" class="prev-section-link"><span>Previous Section</span></a><a href="#ack-1" title="Acknowledgments" class="next-section-link"><span>Next Section</span></a></div>
                  
                  <h2>MATERIALS AND METHODS</h2>
                  
                  <div id="sec-14" class="subsection">
                     
                     <h3>Plant Material and Growth Conditions</h3>
                     
                     <p id="p-47">Lines of Arabidopsis used were <em>mum4-1</em>, <em>mum4-2</em> (Col-2 ecotype; <a id="xref-ref-31-8" class="xref-bibr" href="#ref-31">Western et al., 2001</a>), <em>mum4-3</em> (see below), <em>mum4-4</em>, <em>mum4-5</em> (Salk_085051 and 038898, respectively, obtained from the Arabidopsis Biological Resource Center [ABRC], Ohio State University,
                        Columbus), <em>ap2-1</em> (L<em>er</em> ecotype; ABRC), <em>ap2-7</em> (Col-2; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kunst et al., 1989</a>), <em>gl2-1, ttg1-1, ttg2-1</em> (L<em>er</em>; ABRC), and <em>myb61-1</em> (Col-0; gift from Michael Bevan, John Innes Centre, Norwich, UK). Growth conditions were as described by Western et al. (<a id="xref-ref-31-9" class="xref-bibr" href="#ref-31">2001</a>).
                     </p>
                     
                     <p id="p-48"><em>mum4-3</em> was isolated by screening T-DNA mutagenized lines (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Feldmann and Marks, 1987</a>; ABRC) by staining with a 0.01% (w/v) aqueous solution of Ruthenium red.
                     </p>
                     
                  </div>
                  
                  <div id="sec-15" class="subsection">
                     
                     <h3>Staging of Flower Age</h3>
                     
                     <p id="p-49">Flowers were staged as by Western et al. (<a id="xref-ref-31-10" class="xref-bibr" href="#ref-31">2001</a>).
                     </p>
                     
                  </div>
                  
                  <div id="sec-16" class="subsection">
                     
                     <h3>Immunofluorescence</h3>
                     
                     <p id="p-50">Developing seeds for bright-field microscopy were prepared as described by Western et al. (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">2000</a>).
                     </p>
                     
                     <p id="p-51">For immunofluorescence, developing seeds were placed in copper hats in a 0.2 <span class="sc">m</span> Suc solution and high-pressure frozen using a Balzers HPM 101 (Balzers Instruments, Balzers, Liechtenstein). The samples
                        were freeze substituted over 5 d in 0.25% (v/v) glutaraldehyde/8% (v/v) dimethoxypropane in acetone using a dry ice/acetone
                        bath at -80°C, then gradually warmed to -20°C, 4°C, and then 20°C (3 h at each temperature) before embedding in LR White resin.
                        Sectioning and microscopy were performed as for bright field.
                     </p>
                     
                     <p id="p-52">Immersion immunofluorescence was performed according to the procedure of Willats et al. (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">2001</a>), except dry wild-type seeds were placed directly into the fixative. Alternatively, the seeds were imbibed for 24 h in buffer
                        with gentle shaking before fixation, all subsequent steps also being performed with shaking. Antipectin monoclonal antibodies
                        JIM5, JIM7, and polyclonal α-PGA/RGI at 1:5 and 1:10 (v/v) dilutions were used as primary antibodies. Pre-immune serum controls
                        were rat (JIM5 and JIM7) and rabbit (α-PGA/RGI). Secondary antibodies were 1:100 (v/v) dilutions in 5% (w/v) nonfat dry milk
                        (NFDM) of goat anti-rat IgG conjugated to fluorescein isothiocyanate (JIM5 and JIM7) and anti-rabbit IgG (α-PGA/RGI). Sections
                        were mounted in either anti-fade agent (JIM5 and JIM7; Slowfade Antifade, Molecular Probes, Eugene, OR) or 1:2 (v/v) glycerol:water
                        (α-PGA/RGI) before observation.
                     </p>
                     
                     <p id="p-53">For immunofluorescence, 0.5-μm sections were incubated in 50 m<span class="sc">m</span> EGTA for 1 h, blocked with 5% (w/v) NFDM for 20 min, then incubated for 1 h with primary antibodies at 1:5 and 1:10 (v/v)
                        dilutions in 5% (w/v) NFDM, and for 1 h in secondary antibodies diluted at 1:100 (v/v). Rinses were performed before and after
                        each incubation with 1× Tris-buffered saline/0.1% (v/v) Tween 20. Primary antibodies, secondary antibodies, pre-immune controls,
                        and mounting were done as described for immersion immunofluorescence.
                     </p>
                     
                  </div>
                  
                  <div id="sec-17" class="subsection">
                     
                     <h3>Scanning Electron Microscopy</h3>
                     
                     <p id="p-54">Dry-mounted seeds were prepared and observed as by Western et al. (<a id="xref-ref-31-11" class="xref-bibr" href="#ref-31">2001</a>).
                     </p>
                     
                  </div>
                  
                  <div id="sec-18" class="subsection">
                     
                     <h3>Gas Chromatography</h3>
                     
                     <p id="p-55">Mucilage was extracted from samples of approximately 5 mg of intact seeds by shaking in 0.2% (w/v) ammonium oxalate for 2
                        h at 30°C. No significant difference in mass was observed between Col-2 and <em>mum4-1</em> seed (four lots of 100 counted seed weighed in triplicate; Col-2 = 2.0 ± 0.3 mg, <em>mum4-1</em> = 2.2 ± 0.1 mg <span class="sc">sd</span>). Ten microliters of internal standard (5 mg mL<sup>-1</sup> myo-inositol) was added before precipitation with 5 volumes of absolute ethanol and drying under nitrogen gas. Derivatization,
                        gas chromatography, and identification of trimethylsilyl ethers were performed as described previously (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Western et al., 2000</a>).
                     </p>
                     
                     <p id="p-56">For alditol acetates, samples were hydrolyzed in trifluoroacetic acid for 2 h at 100°C, dried under nitrogen gas, then reduced
                        for 90 min at 40°C in 245 μL of 2.6 <span class="sc">m</span> ammonia and 700 μL of 2% (w/v) sodium borohydride in dimethylsulfoxide. After addition of 175 μL of acetic acid, the samples
                        were acetylated for 10 min at room temperature with 175 μL of 1-methylimidazole and 2.8 mL of acetic anhydride. Water (5.6
                        mL) was added, and the alditol acetates were extracted with 1.05 mL of dichloromethane (DCM). The recovered phase was dried
                        and resuspended in 200 μL of DCM, then re-extracted with 1 mL of water. This recovered DCM phase was used for analysis. Samples
                        were run on a gas chromatograph (model 5890A, Hewlett-Packard, Mississauga, ON, Canada) on a HP-23 glass capillary column
                        (30-m × 0.25-mm i.d.) with helium as the carrier gas. The temperature program was 2 min at 180°C, increase 10°C min<sup>-1</sup> up to 200°C, 5 min at 200°C, increase 10°C min<sup>-1</sup> to 250°C, and hold at 250°C for 10 min. Monosaccharides were identified through comparison with the retention times obtained
                        with individual sugar standards and a composite standard consisting of Rib, Fuc, Man, Gal, Glc, Ara, Rha, Xyl, GlcUA, and
                        GalUA.
                     </p>
                     
                  </div>
                  
                  <div id="sec-19" class="subsection">
                     
                     <h3>Positional Cloning of MUM4 and Determination of cDNA Sequence</h3>
                     
                     <p id="p-57">A mapping population of 519 plants derived from a cross between <em>mum4-1</em> and wild-type L<em>er</em> was used for progressive fine mapping using simple sequence length polymorphism and cleaved-amplified polymorphic sequence
                        markers generated from sequence information provided by the Arabidopsis Genome Initiative (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2000</a>) and Cereon (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Jander et al., 2002</a>). Genomic DNA was isolated from the T-DNA-tagged <em>mum4-3</em> mutant and wild-type Wassilewskija plants, and Southern hybridization was performed using radiolabeled probes derived from
                        F12M16 and T3F20. An insertion was detected in a 9.5-kb <em>Xho</em>I fragment from T3F20. Gene-specific primers for the four predicted genes on that fragment were used in PCR with a T-DNA right
                        border primer (KF-RB p1 5′-gttgaagttggcgagttcgt-3′). An insertion was identified in <em>At1g53500</em> using a 3′ primer (<em>At1g53500</em> p2 5′-tctgaaactgcctaggtggaa-3′).
                     </p>
                     
                     <p id="p-58">To confirm that the gene was <em>MUM4</em>, <em>mum4-1</em> and <em>mum4-2</em> were sequenced. Sequencing of the Salk Institute sequence-indexed Arabidopsis T-DNA insertion lines 085051 and 038898 revealed
                        two further alleles, named <em>mum4-4</em> and <em>mum4-5</em>, respectively. Molecular complementation of <em>MUM4</em> was carried out using a 5.2-kb <em>Sal</em>I/<em>Bst</em>Z17I fragment of T3F20, including <em>At1g53500</em> plus 2.1 kb upstream and 0.4 kb downstream sequence, into pGREEN0229 (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hellens et al., 2000</a>). <em>mum4-1</em> plants were transformed either with the genomic clone (pMUM4g) or the empty vector (pGREEN0229) using the <em>Agrobacterium tumefaciens</em> dipping method (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Clough and Bent, 1998</a>). Basta-resistant transformants were isolated by spraying young seedlings with 0.1% (v/v) glufosinate (Final EV150, AgrEvo
                        EH, Paris) in 0.1% (v/v) Silwet <span class="sc">l</span>-77, and putative transformants were verified by PCR analysis.
                     </p>
                     
                     <p id="p-59">The 5′-UTR of <em>MUM4</em> was located using RT-PCR with primers located up to 900 bp upstream from the predicted ATG in combination with an internal
                        primer. RNA was isolated from developing siliques according to the protocol of Downing et al. (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">1992</a>), with the addition of a sodium acetate wash to remove excess polysaccharides. One-microgram samples were treated with DNaseI
                        (Invitrogen Life Technologies, Carlsbad, CA) and reverse transcribed with SuperScript II Reverse Transcriptase (Invitrogen)
                        according to the manufacturer's instructions. RT-PCR using two sequential primers identified bands approximately 200 bp smaller
                        than predicted from genomic sequence. The longer fragment was isolated and sequenced.
                     </p>
                     
                  </div>
                  
                  <div id="sec-20" class="subsection">
                     
                     <h3>Analyses of Gene Expression</h3>
                     
                     <p id="p-60">RNA was isolated as described above. For RNA gel blots, 10 μg of total RNA was separated on formaldehyde gels, blotted to
                        Hybond-XL nylon membranes (Amersham Biosciences Corp., Baie d'Urfé, QC, Canada), and hybridized according to the manufacturer's
                        instructions. A <sup>32</sup>P-labeled <em>MUM4</em> probe corresponding to the degenerate region between the two predicted enzymatic domains was synthesized using PCR (<em>MUM4</em>p11 5′-ggaagactttctgatggatctagt-3′/<em>MUM4</em>p12 5′-gatcaaaaacttcaacgaagc-3′). 25S rRNA bands from the ethidium bromide-stained RNA gel were used as loading controls.
                     </p>
                     
                     <p id="p-61">RNA isolation and RT of DNase-treated RNA was performed as described above. Gene-specific primers surrounding an intron were
                        designed for each gene, with the exception of <em>TTG1</em>, which lacks an intron. Primers were as follows: <em>MUM4</em>, p1-ttcgtgtaaatgtcgcaggt/p2 5′-gagaagctcgtctagtacggtc-3′; <em>RHM1</em>, p1 5′-gagactatccgtgccaatgta-3′/p2 5′-taataactcgtccaacacagtc-3′; <em>RHM3</em>, p1 5′-ccgagtcaacgttgctgga-3′/p2 5′-ggtaagagttcatctagtatggtc-3′; <em>GAPC</em>, p1 5′-tcagactcgagaaagctgctac-3′/p2 5′-gatcaagtcgaccacacgg-3′; <em>MYB61</em>, p3 5′-tggggagacattcttgctg-3′/p4 5′-gatgggcttgtgtgtgtttg-3′; <em>GL2</em>, p1 5′-aacggtcactccaaggtcac-3′/p2 5′-agaaacccgcatgtcttgtc-3′; <em>TTG2</em>, p1 5′-gccatcttgtcctcttccac-3′/p2 5′-ccttgcgatactcctgcttc-3′; <em>AP2</em>, p1 5′-cagggaatcctactactccacaag-3′/p2 5′-atctgattgtgatgatgaggagag-3′; and <em>TTG1</em>, p1 5′-catcctccggtcacagaatc-3′/p2 5′-tttcggctctacatcgttcc-3′. Loading for all RT-PCRs was determined by standardizing against
                        <em>GAPC</em> test PCRs (at 22-23 cycles) for each RT reaction. Saturating, qualitative RT-PCR for <em>MUM4</em>, <em>RHM1</em>, and
                     </p>
                     
                     <p id="p-62"><em>RHM3</em> was done with 30 cycles. Semiquantitative RT-PCR (for <em>AP2</em>, <em>TTG1</em>, <em>TTG2</em>, <em>GL2</em>, and <em>MYB61</em>) was performed a minimum of four times for each set of primers using two to three different cycle numbers (from 25-30 cycles)
                        to confirm results.
                     </p>
                     
                  </div>
                  
                  <div id="sec-21" class="subsection">
                     
                     <h3>Accession Numbers</h3>
                     
                     <p id="p-63">The GenBank accession numbers for genes mentioned in this article are <a href="/external-ref?link_type=GEN&amp;access_num=AY328518">AY328518</a> (<em>MUM4</em> full-length cDNA), <a href="/external-ref?link_type=GEN&amp;access_num=AY042833">AY042833</a> (<em>RHM1</em>), and <a href="/external-ref?link_type=GEN&amp;access_num=AY078958">AY078958</a> (<em>RHM3</em>). <em>mum4-3</em> has been submitted to ABRC under the seed stock number CS6382.
                     </p>
                     
                  </div>
                  
                  <div id="sec-22" class="subsection">
                     
                     <h3>Distribution of Materials</h3>
                     
                     <p id="p-64">Upon request, all novel materials described in this publication will be made available in a timely manner for noncommercial
                        research purposes, subject to the requisite permission from any third party owners of all or parts of the material. Obtaining
                        permissions will be the responsibility of the requestor.
                     </p>
                     
                  </div>
                  
               </div>
               <div class="section ack" id="ack-1">
                  <div class="section-nav"><a href="#sec-13" title="MATERIALS AND METHODS" class="prev-section-link"><span>Previous Section</span></a><a href="#fn-group-1" title="Footnotes" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>Acknowledgments</h2>
                  
                  <p id="p-69">We thank Ms. Yeen Ting Hwang (University of British Columbia, Vancouver, BC Canada) and Mr. Adrian Wladichuk (University of
                     British Columbia, Vancouver, BC Canada) for technical assistance, Dr. Michael Bevan (John Innes Centre, Norwich, UK) for supplying
                     <em>myb61-1</em> seed, Dr. Mark Smith (University of British Columbia, Vancouver, BC Canada) and Ms. Michelle Fawcett (Carnegie Institution,
                     Stanford University, CA) for help with chemical analysis of mucilage, Drs. Andrew Staehelin (University of Colorado, Boulder)
                     and Paul Knox (University of Leeds, Leeds, UK) for gifts of antibodies, and the Salk Institute Genomic Analysis Laboratory
                     (La Jolla, CA, USA) for providing the sequence-indexed Arabidopsis T-DNA insertion mutants. We also thank Drs. Markus Pauly
                     (Max-Planck Institute for Molecular Plant Physiology, Golm, Germany) and Björn Usadel (Max-Planck Institute for Molecular
                     Plant Physiology, Glom, Germany) for communicating unpublished data and delaying publication of their results on <em>RHM2</em>. We thank members of the Ljerka Haughn (University of British Columbia, Vancouver, BC Canada) and George Kunst (University
                     of British Columbia, Vancouver, BC Canada) labs for comments on the manuscript.
                  </p>
                  
               </div>
               <div class="section fn-group" id="fn-group-1">
                  <div class="section-nav"><a href="#ack-1" title="Acknowledgments" class="prev-section-link"><span>Previous Section</span></a><a href="#ref-list-1" title="LITERATURE CITED" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>Footnotes</h2>
                  <ul><li class="fn fn-group-arthw-misc" id="fn-4">
                        <p id="p-68">Article, publication date, and citation information can be found at <a href="http://www.plantphysiol.org/cgi/doi/10.1104/pp.103.035519">www.plantphysiol.org/cgi/doi/10.1104/pp.103.035519</a>.
                        </p>
                     </li>
                     <li class="fn fn-group-arthw-misc" id="fn-1">
                        <p id="p-65"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">1</span> This work was supported by the Natural Science and Engineering Research Council of Canada (Discovery Grants to G.W.H. and
                           A.L.S. and a Cell Wall Multi-Network Grant to G.W.H.).
                        </p>
                     </li>
                     <li class="fn fn-group-arthw-misc" id="fn-2">
                        <p id="p-66"><a class="rev-xref" href="#xref-fn-2-1">↵</a><span class="fn-label">2</span> Present address: Biology Department, McGill University, 1205 Docteur Penfield Avenue, Montreal, QC, Canada H3A 1B1.
                        </p>
                     </li>
                     <li class="fn fn-group-arthw-misc" id="fn-3">
                        <p id="p-67"><a class="rev-xref" href="#xref-fn-3-1">↵</a><span class="fn-label">3</span> Present address: Ottawa Regional Cancer Centre, Centre for Cancer Therapeutics, Third Floor, 503 Smyth Road, Ottawa, ON,
                           Canada K1H 1C4.
                        </p>
                     </li>
                  </ul></div>
               <ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2003-10-30"><span class="received-label">Received </span>October 30, 2003.
                  </li>
                  <li xmlns:hwp="http://schema.highwire.org/Journal" class="rev-recd" hwp:start="2003-11-12"><span class="rev-recd-label">Revised </span>November 12, 2003.
                  </li>
                  <li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2003-11-20"><span class="accepted-label">Accepted </span>November 20, 2003.
                  </li>
                  <li class="published"><span class="published-label">Published </span>December 30, 2003.
                  </li>
               </ul><div class="section ref-list" id="ref-list-1">
                  <div class="section-nav"><a href="#fn-group-1" title="Footnotes" class="prev-section-link"><span>Previous Section</span></a><div class="nav-placeholder"> </div>
                  </div>
                  <h2>LITERATURE CITED</h2>
                  <ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference in text" id="ref-1">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.1">
                           <div class="cit-metadata"><cite><strong>Allard STM, Giraud M-F, Whitfield C, Graninger M, Messner P, Naismith</strong> <strong>JH</strong> (<span class="cit-pub-date">2001</span>) The crystal structure of dTDP-<span class="sc">d</span>-glucose 4, 6-dehydratase (RmlB) from <em>Salmonella enterica</em> serovar Typhimurium, the second enzyme in the dTDP-<span class="sc">l</span>-rhamnose pathway. <span class="cit-source">J Mol Biol</span> <span class="cit-vol">307</span><strong>:</strong> <span class="cit-fpage">283</span>-295</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1006/jmbi.2000.4470&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=11243820&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000167764900024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference in text" id="ref-2">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.2">
                           <div class="cit-metadata"><cite><strong>Arabidopsis Genome Initiative</strong> (<span class="cit-pub-date">2000</span>) Analysis of the genome sequence of the flowering plant <em>Arabidopsis thaliana</em>. <span class="cit-source">Nature</span> <span class="cit-vol">408</span><strong>:</strong> <span class="cit-fpage">796</span>-815</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1038/35048692&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=11130711&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference in text" id="ref-3">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.3">
                           <div class="cit-metadata"><cite><strong>Beeckman T, De Rycke R, Viane R, Inzé D</strong> (<span class="cit-pub-date">2000</span>) Histological study of seed coat development in <em>Arabidopsis thaliana</em>. <span class="cit-source">J Plant Res</span> <span class="cit-vol">113</span><strong>:</strong> <span class="cit-fpage">139</span>-148</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1007/PL00013924&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=000088205500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference in text" id="ref-4">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.4">
                           <div class="cit-metadata"><cite><strong>Boesewinkel FD, Bouman F</strong> (<span class="cit-pub-date">1995</span>) The seed: structure and function. <em>In</em> J Kigel, G Galili, eds, <span class="cit-source">Seed Development and Germination</span>. Marcel Dekker Inc., New York, pp <span class="cit-fpage">1</span>-24</cite></div>
                           <div class="cit-extra"/>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference in text" id="ref-5">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.5">
                           <div class="cit-metadata"><cite><strong>Bonin CP, Reiter W-D</strong> (<span class="cit-pub-date">2000</span>) A bifunctional epimerase-reductase acts downstream of the <em>MUR1</em> gene product and completes the de novo synthesis of GDP-<span class="sc">l</span>-fucose in <em>Arabidopsis</em>. <span class="cit-source">Plant J</span> <span class="cit-vol">21</span><strong>:</strong> <span class="cit-fpage">445</span>-454</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1046/j.1365-313x.2000.00698.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=10758496&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000086269900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference in text" id="ref-6">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.6">
                           <div class="cit-metadata"><cite><strong>Clough SJ, Bent AF</strong> (<span class="cit-pub-date">1998</span>) Floral dip: a simplified method for <em>Agrobacterium</em>-mediated transformation of <em>Arabidopsis thaliana</em>. <span class="cit-source">Plant J</span> <span class="cit-vol">16</span><strong>:</strong> <span class="cit-fpage">735</span>-743</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1046/j.1365-313x.1998.00343.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=10069079&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000078001900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference in text" id="ref-7">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.7">
                           <div class="cit-metadata"><cite><strong>Downing WL, Mauxion F, Fauvarque MO, Reviron MP, de Vienne D,</strong> <strong>Vartanian N, Giraudat J</strong> (<span class="cit-pub-date">1992</span>) A <em>Brassica napus</em> transcript encoding a protein related to the Kunitz protease inhibitors family accumulates upon water stress in leaves, not
                                 in seeds. <span class="cit-source">Plant J</span> <span class="cit-vol">2</span><strong>:</strong> <span class="cit-fpage">685</span>-693</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1046/j.1365-313X.1992.t01-11-00999.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=1302628&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=A1992JN08700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference in text" id="ref-8">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.8">
                           <div class="cit-metadata"><cite><strong>Dumville JC, Fry SC</strong> (<span class="cit-pub-date">2000</span>) Uronic acid-containing oligosaccharins: their biosynthesis, degradation and signalling roles in non-diseased plant tissues.
                                 <span class="cit-source">Plant Physiol Biochem</span> <span class="cit-vol">38</span><strong>:</strong> <span class="cit-fpage">125</span>-140</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1016/S0981-9428(00)00163-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=000086315600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference in text" id="ref-9">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.9">
                           <div class="cit-metadata"><cite><strong>Feingold DS</strong> (<span class="cit-pub-date">1982</span>) Aldo (and keto) hexoses and uronic acids. <em>In</em> FA Loewus, W Tanner, eds, <span class="cit-source">Encyclopedia of Plant Physiology</span>. Springer Verlag, Berlin, pp <span class="cit-fpage">3</span>-76</cite></div>
                           <div class="cit-extra"/>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference in text" id="ref-10">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.10">
                           <div class="cit-metadata"><cite><strong>Feldmann KA, Marks MD</strong> (<span class="cit-pub-date">1987</span>) <em>Agrobacterium</em>-mediated transformation of germinating seeds of <em>Arabidopsis thaliana</em>: a non-tissue culture approach. <span class="cit-source">Mol Gen Genet</span> <span class="cit-vol">208</span><strong>:</strong> <span class="cit-fpage">1</span>-9</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1007/BF00330414&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=A1987H853600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference in text" id="ref-11">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.11">
                           <div class="cit-metadata"><cite><strong>Graninger M, Nidetzky B, Heinrichs DE, Whitfield C, Messner P</strong> (<span class="cit-pub-date">1999</span>) Characterization of dTDP-4-dehydrorhamnose 3, 5-epimerase and dTDP-4-dehydrorhamnose reductase, required for dTDP-<span class="sc">l</span>-rhamnose biosynthesis in <em>Salmonella enterica</em> serovar Typhimurium LT2. <span class="cit-source">J Biol Chem</span> <span class="cit-vol">274</span><strong>:</strong> <span class="cit-fpage">25069</span>-25077</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=jbc&amp;resid=274/35/25069" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference in text" id="ref-12">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.12">
                           <div class="cit-metadata"><cite><strong>Grubert M</strong> (<span class="cit-pub-date">1981</span>) <span class="cit-source">Mucilage or Gum in Seeds and Fruits of Angiosperms: A Review</span>. Minerva Press, Munich</cite></div>
                           <div class="cit-extra"/>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference in text" id="ref-13">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.13">
                           <div class="cit-metadata"><cite><strong>Hellens RP, Edwards EA, Leyland NR, Bean S, Mullineaux PM</strong> (<span class="cit-pub-date">2000</span>) pGreen: a versatile and flexible binary Ti vector for <em>Agrobacterium</em>-mediated plant transformation. <span class="cit-source">Plant Mol Biol</span> <span class="cit-vol">42</span><strong>:</strong> <span class="cit-fpage">819</span>-832</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1023/A:1006496308160&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=10890530&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000087182000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference in text" id="ref-14">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.14">
                           <div class="cit-metadata"><cite><strong>Jander G, Norris SR, Rounsely SD, Bush DF, Levin IM, Last RL</strong> (<span class="cit-pub-date">2002</span>) <em>Arabidopsis</em> map-based cloning in the post-genome era. <span class="cit-source">Plant Physiol</span> <span class="cit-vol">129</span><strong>:</strong> <span class="cit-fpage">440</span>-450</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=129/2/440" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference in text" id="ref-15">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.15">
                           <div class="cit-metadata"><cite><strong>Jofuku KD, den Boer BGW, Van Montagu M, Okamuro JK</strong> (<span class="cit-pub-date">1994</span>) Control of <em>Arabidopsis</em> flower and seed development by the homeotic gene APETALA2. <span class="cit-source">Plant Cell</span> <span class="cit-vol">6</span><strong>:</strong> <span class="cit-fpage">1211</span>-1225</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=6/9/1211" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference in text" id="ref-16">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.16">
                           <div class="cit-metadata"><cite><strong>Johnson CS, Kolevski B, Smyth DR</strong> (<span class="cit-pub-date">2002</span>) TRANSPARENT TESTA GLA BRA2, a trichome and seed coat development gene of <em>Arabidopsis</em>, encodes a WRKY transcription factor. <span class="cit-source">Plant Cell</span> <span class="cit-vol">14</span><strong>:</strong> <span class="cit-fpage">1359</span>-1375</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=14/6/1359" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference in text" id="ref-17">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.17">
                           <div class="cit-metadata"><cite><strong>Kirik V, Schnittger A, Radchuk V, Adler K, Hülskamp M, Bäumlein H</strong> (<span class="cit-pub-date">2001</span>) Ectopic expression of the <em>Arabidopsis AtMYB23</em> gene induces differentiation of trichome cells. <span class="cit-source">Dev Biol</span> <span class="cit-vol">235</span><strong>:</strong> <span class="cit-fpage">366</span>-377</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1006/dbio.2001.0287&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=11437443&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000170046700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference in text" id="ref-18">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.18">
                           <div class="cit-metadata"><cite><strong>Koornneef M</strong> (<span class="cit-pub-date">1981</span>) The complex syndrome of <em>TTG</em> mutants. <span class="cit-source">Arab Inf Serv</span> <span class="cit-vol">18</span><strong>:</strong> <span class="cit-fpage">45</span>-51</cite></div>
                           <div class="cit-extra"/>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference in text" id="ref-19">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.19">
                           <div class="cit-metadata"><cite><strong>Kunst L, Klenz JE, Martinez-Zapater J, Haughn GW</strong> (<span class="cit-pub-date">1989</span>) AP2 gene determines the identity of perianth organs in flowers of <em>Arabidopsis thaliana</em>. <span class="cit-source">Plant Cell</span> <span class="cit-vol">1</span><strong>:</strong> <span class="cit-fpage">1195</span>-1208</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=1/12/1195" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference in text" id="ref-20">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.20">
                           <div class="cit-metadata"><cite><strong>Lynch MA, Staehelin LA</strong> (<span class="cit-pub-date">1992</span>) Domain-specific and cell type-specific localization of two types of cell wall matrix polysaccharides in the clover root
                                 tip. <span class="cit-source">J Cell Biol</span> <span class="cit-vol">118</span><strong>:</strong> <span class="cit-fpage">467</span>-479</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=jcb&amp;resid=118/2/467" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference in text" id="ref-21">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.21">
                           <div class="cit-metadata"><cite><strong>Millar I, Moore R</strong> (<span class="cit-pub-date">1990</span>) Defective secretion of mucilage is the cellular basis of agravitropism in primary roots of <em>Zea mays</em> cv. Ageotropic. <span class="cit-source">Ann Bot</span> <span class="cit-vol">66</span><strong>:</strong> <span class="cit-fpage">169</span>-178</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=annbot&amp;resid=66/2/169" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference in text" id="ref-22">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.22">
                           <div class="cit-metadata"><cite><strong>Payne CT, Zhang F, Lloyd AM</strong> (<span class="cit-pub-date">2000</span>) GL3 encodes a bHLH protein that regulates trichome development in <em>Arabidopsis</em> through interaction with <em>GL1</em> and <em>TTG1</em>. <span class="cit-source">Genetics</span> <span class="cit-vol">156</span><strong>:</strong> <span class="cit-fpage">1349</span>-1362</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=genetics&amp;resid=156/3/1349" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference in text" id="ref-23">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.23">
                           <div class="cit-metadata"><cite><strong>Penfield S, Meissner RC, Shoue DA, Carpita NC, Bevan MW</strong> (<span class="cit-pub-date">2001</span>) MYB61 is required for mucilage deposition and extrusion in the <em>Arabidopsis</em> seed coat. <span class="cit-source">Plant Cell</span> <span class="cit-vol">13</span><strong>:</strong> <span class="cit-fpage">2777</span>-2791</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=13/12/2777" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference in text" id="ref-24">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.24">
                           <div class="cit-metadata"><cite><strong>Reiter W-D, Vanzin GF</strong> (<span class="cit-pub-date">2001</span>) Molecular genetics of nucleotide sugar interconversion pathways in plants. <span class="cit-source">Plant Mol Biol</span> <span class="cit-vol">47</span><strong>:</strong> <span class="cit-fpage">95</span>-113</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1023/A:1010671129803&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=11554483&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000169866100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference in text" id="ref-25">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.25">
                           <div class="cit-metadata"><cite><strong>Ridley BL, O'Neill MA, Mohnen D</strong> (<span class="cit-pub-date">2001</span>) Pectins: structure, biosynthesis, and oligogalacturonide-related signaling. <span class="cit-source">Phytochemistry</span> <span class="cit-vol">57</span><strong>:</strong> <span class="cit-fpage">929</span>-967</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1016/S0031-9422(01)00113-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=11423142&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000169805000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference in text" id="ref-26">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.26">
                           <div class="cit-metadata"><cite><strong>Schiefelbein J</strong> (<span class="cit-pub-date">2003</span>) Cell-fate specification in the epidermis: a common patterning mechanism in the root and shoot. <span class="cit-source">Curr Opin Plant Biol</span> <span class="cit-vol">6</span><strong>:</strong> <span class="cit-fpage">74</span>-78</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1016/S136952660200002X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=12495754&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000180195000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference in text" id="ref-27">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.27">
                           <div class="cit-metadata"><cite><strong>Tatusov RL, Galperin MY, Natale DA, Koonin EV</strong> (<span class="cit-pub-date">2000</span>) The COG database: a tool for genome-scale analysis of protein functions and evolution. <span class="cit-source">Nucleic Acids Res</span> <span class="cit-vol">28</span><strong>:</strong> <span class="cit-fpage">33</span>-36</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=nar&amp;resid=28/1/33" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference in text" id="ref-28">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.28">
                           <div class="cit-metadata"><cite><strong>Tonetti M, Sturla L, Bisso A, Zanardi D, Benatti U, De Flora A</strong> (<span class="cit-pub-date">1998</span>) The metabolism of 6-deoxyhexoses in bacterial and animal cells. <span class="cit-source">Biochimie</span> <span class="cit-vol">80</span><strong>:</strong> <span class="cit-fpage">923</span>-931</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=9893952&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference in text" id="ref-29">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.29">
                           <div class="cit-metadata"><cite><strong>Usadel B, Kuschinsky AM, Rosso MG, Eckermann N, Pauly M</strong> (<span class="cit-pub-date">2004</span>) RHM2 is involved in mucilage pectin biosynthesis and is required for the development of the seed coat in Arabidopsis. <span class="cit-source">Plant Physiol</span> <span class="cit-vol">134</span><strong>:</strong> <span class="cit-fpage">286</span>-295</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=134/1/286" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference in text" id="ref-30">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.30">
                           <div class="cit-metadata"><cite><strong>Western TL, Skinner DJ, Haughn GW</strong> (<span class="cit-pub-date">2000</span>) Differentiation of mucilage secretory cells of the <em>Arabidopsis</em> seed coat. <span class="cit-source">Plant Physiol</span> <span class="cit-vol">122</span><strong>:</strong> <span class="cit-fpage">345</span>-355</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=122/2/345" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference in text" id="ref-31">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.31">
                           <div class="cit-metadata"><cite><strong>Western TL, Burn J, Tan WL, Skinner DJ, Martin-McCaffrey L, Moffatt</strong> <strong>BA, Haughn GW</strong> (<span class="cit-pub-date">2001</span>) Isolation and characterization of mutants defective in seed coat mucilage secretory cell development in <em>Arabidopsis</em>. <span class="cit-source">Plant Physiol</span> <span class="cit-vol">127</span><strong>:</strong> <span class="cit-fpage">998</span>-1011</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=127/3/998" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference in text" id="ref-32">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.32">
                           <div class="cit-metadata"><cite><strong>Willats WGT, McCartney L, Knox JP</strong> (<span class="cit-pub-date">2001</span>) In-situ analysis of pectic polysaccharides in seed mucilage and at the root surface of <em>Arabidopsis thaliana</em>. <span class="cit-source">Planta</span> <span class="cit-vol">213</span><strong>:</strong> <span class="cit-fpage">37</span>-44</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1007/s004250000481&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=11523654&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000168920900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference in text" id="ref-33">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.33">
                           <div class="cit-metadata"><cite><strong>Windsor JB, Symonds VV, Mendenhall J, Lloyd AL</strong> (<span class="cit-pub-date">2000</span>) <em>Arabidopsis</em> seed coat development: morphological differentiation of the outer integument. <span class="cit-source">Plant J</span> <span class="cit-vol">22</span><strong>:</strong> <span class="cit-fpage">483</span>-493</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1046/j.1365-313x.2000.00756.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=10886768&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000087868300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                        
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference in text" id="ref-34">↵</a>
                        
                        <div class="cit ref-cit ref-other" id="cit-134.1.296.34">
                           <div class="cit-metadata"><cite><strong>Zhang F, Gonzalez A, Zhao M, Payne CT, Lloyd A</strong> (<span class="cit-pub-date">2003</span>) A network of redundant bHLH proteins functions in all TTG1-dependent pathways of <em>Arabidopsis</em>. <span class="cit-source">Development</span> <span class="cit-vol">130</span><strong>:</strong> <span class="cit-fpage">4859</span>-4869</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=130/20/4859" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                        
                     </li>
                  </ol></div><span class="highwire-journal-article-marker-end"/></div>
  </body>
</html>
