<?xml version="1.0"?>
<html>
  <body>
    <div itemprop="articleBody">
      <span class="highwire-journal-article-marker-start"/>
      <div class="hw-optimized-content">
                  <div class="auto-clean">
                     
                     
                     
                     
                     
                     	
                     	
                     	
                     	
                     	
                     	
                     
                     	
                     	 
                     	
                     	
                     	
                     		<span style="  font-size: 83.33%"><em>The Plant Cell</em> 16:1206-1219 (2004)<br/>
                        © 2004 <a href="/misc/terms.shtml">American Society of Plant Biologists</a></span><br/></div>
                  <h1 id="article-title-1" itemprop="headline">
                     The Embryo MADS Domain Protein AGAMOUS-Like 15 Directly Regulates Expression of a Gene Encoding an Enzyme Involved in Gibberellin
                     Metabolism
                     
                     
                  </h1>
                  <div class="contributors">
                     Huai Wang<sup>a</sup>, 
                     Leonardo V. Caruso<sup>a</sup>, 
                     A. Bruce Downie<sup>b</sup> and 
                     Sharyn E. Perry<sup>a</sup><sup>,1</sup></div>
                  <p>
                     <span style="  font-size: 83.33%"><sup>a</sup> Department of Agronomy, University of Kentucky, Lexington, Kentucky 40546-0312<br/><sup>b</sup> Department of Horticulture, University of Kentucky, Lexington, Kentucky 40546-0312
                        </span></p>
                  <p>
                     
                     <span style="  font-size: 83.33%"><sup><sup>1</sup></sup> To whom correspondence should be addressed. E-mail <span class="em-link"><span class="em-addr">sperr2{at}uky.edu</span></span>; fax 859-257-7125.</span></p>
                  <p>
                     <a name="ABS" id="ABS"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        ABSTRACT
                        
                        
                     </h2>
                     
                  </div>
                  <div class="section abstract" id="abstract-1" itemprop="description">
                     
                     <p>
                        AGL15 (for AGAMOUS-Like 15) is a member of the MADS domain family of DNA binding transcriptional regulators that accumulates
                        to its highest amounts during embryo development. To better understand how AGL15 functions, a chromatin immunoprecipitation
                        approach was used to identify directly regulated genes. One DNA fragment that coprecipitated with AGL15 corresponded to a
                        portion of the regulatory region of a gene named <em>DTA1</em> (for <em>Downstream Target of AGL15-1</em>). The expression of <em>DTA1</em> was positively correlated with AGL15 abundance during embryogenesis. In this report, a <em>cis</em> element for response to AGL15 was identified, and the activity of DTA1 as a gibberellin (GA) 2-oxidase was confirmed. <em>DTA1</em> corresponds to <em>AtGA2ox6</em> and was renamed to indicate this identity. Further experiments related the function of AtGA2ox6 to regulation by AGL15. Constitutive
                        expression of <em>AGL15</em> and of <em>AtGA2ox6</em> altered endogenous GA amounts and caused GA-deficient phenotypes in <em>Arabidopsis thaliana</em> that could be at least partially rescued by application of biologically active GA. The phenotype of plants with decreased
                        expression of <em>AtGA2ox6</em> was the converse of plants overexpressing <em>AtGA2ox6</em> in terms of seed germination attributes and effects on somatic embryo production. 
                        
                     </p>
                     
                  </div>
                  <div class="auto-clean">
                     
                     
                     <a name="SEC1" id="SEC1"/>
                     
                     
                  </div>
                  <div class="section-nav">
                     
                     <h2>
                        INTRODUCTION
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     DNA binding transcriptional regulators are central players in the control of genetic programs. In plants, members of the MADS
                     domain family of regulatory proteins have key roles throughout the plant life cycle, with the best-characterized members involved
                     in inflorescence and floral development (reviewed in Riechmann and Meyerowitz, 1997<a href="#BIB37"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). MADS domain proteins bind to DNA elements called CArG motifs (for C-[A/T]rich-G) via the conserved 55 to 60 amino acid
                     residue MADS domain and control developmental programs by activating and/or repressing the expression of specific target genes
                     (reviewed in Riechmann and Meyerowitz, 1997<a href="#BIB37"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Theissen, 2001<a href="#BIB49"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Identification of the genes directly controlled by DNA binding proteins is of critical importance to understanding how
                     transcriptional regulators operate. In addition, determination of the function of the regulated gene products is crucial for
                     defining the events that lead from gene regulation to developmental consequence. However, identification of direct regulatory
                     targets of a transcription factor has been difficult to address by genetic means because of redundancy of gene functions,
                     essential roles for downstream gene products, and the interaction of different regulated gene products to generate a final
                     phenotype (Andrew and Scott, 1992<a href="#BIB1"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Graba et al., 1997<a href="#BIB11"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Deconvoluting gene regulatory networks is further complicated because a given gene may be regulated by different <em>trans</em>-acting factors, depending on the temporal or spatial context (Arnone and Davidson, 1997<a href="#BIB2"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Graba et al., 1997<a href="#BIB11"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Fickett and Wasserman, 2000<a href="#BIB8"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Other approaches such as differential display or microarray analysis can identify direct targets that respond to a transcriptional
                     regulator but may also isolate indirect targets. Chromatin immunoprecipitation (ChIP) is becoming a widely used approach to
                     obtain DNA fragments directly bound in vivo by a transcription factor and, therefore, potentially directly regulated (reviewed
                     in Orlando et al., 1997<a href="#BIB31"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Orlando, 2000<a href="#BIB30"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Shannon and Rao, 2002<a href="#BIB44"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). However, there are cases of in vivo binding of a transcriptional regulator to a <em>cis</em> element, without a corresponding change in gene regulation in the absence of other factors or signals, making assessment
                     of the consequence of binding important (Wyrick and Young, 2002<a href="#BIB56"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     ChIP has been used to isolate several direct downstream targets of AGAMOUS-Like 15 (AGL15), a member of the MADS domain family
                     (Wang et al., 2002<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). <em>AGL15</em> is currently the only identified member of the <em>Arabidopsis thaliana</em> MIKC subgroup of the MADS box family that is preferentially expressed during embryogenesis (Heck et al., 1995<a href="#BIB15"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Rounsley et al., 1995<a href="#BIB38"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Other members of the Arabidopsis MIKC subgroup are also expressed in developing embryos but are expressed at similar or
                     higher levels in other tissues and at other stages of development (e.g., Flanagan and Ma, 1994<a href="#BIB9"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Burgeff et al., 2002<a href="#BIB3"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). <em>AGL15</em> is expressed after germination but generally at much lower levels than during embryo development (Fernandez et al., 2000<a href="#BIB7"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). In all circumstances studied to date, AGL15 and putative orthologs in other species accumulate in nuclei of angiosperm
                     tissue developing in an embryonic mode, suggesting that AGL15 may be involved in promotion of the embryo phase of the plant
                     life cycle (Heck et al., 1995<a href="#BIB15"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Perry et al., 1996<a href="#BIB34"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>, 1999<a href="#BIB33"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Furthermore, constitutive expression of <em>AGL15</em> via the 35S promoter of <em>Cauliflower mosaic virus</em> (CaMV) supports the maintenance of development in an embryonic mode for extended periods of time in culture and enhances
                     the production of somatic embryo formation from the shoot apical meristems of seedlings in a liquid culture system (Harding
                     et al., 2003<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     <em>DTA1</em> (for <em>Downstream Target of AGL15-1</em>) was isolated by ChIP, is expressed in response to AGL15 (Wang et al., 2002<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), and encodes a protein with similarity to gibberellin (GA) 2-oxidases. GA 2-oxidases are a small family of enzymes in Arabidopsis
                     that catalyze conversion of some biologically active GAs to inactive forms. Does DTA1 function in this manner and, if so,
                     how does this function relate to AGL15? Experimental results presented in this article verify that <em>DTA1</em> is a direct downstream target of AGL15, identify a DNA binding site for AGL15 in the regulatory regions of <em>DTA1</em>, and demonstrate that DTA1 functions as a GA 2-oxidase with effects on seed germination and somatic embryogenesis. 
                     
                  </p>
                  <p>
                     
                     <a name="SEC2" id="SEC2"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        RESULTS
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     <strong><em>DTA1</em> Encodes a Protein That Functions as a GA 2-Oxidase</strong><br/>
                     
                     ChIP was described previously as an approach to isolate DNA fragments, to which AGL15 binds in vivo in embryonic culture tissue
                     constitutively expressing <em>AGL15</em> (Wang et al., 2002<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Harding et al., 2003<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). One such isolated fragment corresponded to the 5' regulatory region of a gene encoding an unknown protein that was named
                     <em>DTA1</em>. <em>DTA1</em> cDNA coding region and 5' and 3' untranslated regions were obtained as described previously (Wang et al., 2002<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Sequence analysis confirmed the correct prediction of the gene in the Arabidopsis database (gene locus At1g02400, protein
                     identification number AAG008911). Database searches indicated that the predicted gene product of <em>DTA1</em> has homology to GA 2-oxidases, including key conserved residues that would predict function as a GA 2-oxidase. The active
                     site core motif of 2-ketoglutarate–dependent dioxygenases involved in gibberellin metabolism was present, including His-203,
                     Asp-205, and His-260 that bind the active site Fe<sup>2+</sup>, and Arg-271 and Ser-273 that interact with 2-ketoglutarate (Hedden and Phillips, 2000<a href="#BIB18"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). DTA1 was more similar to the predicted products of <em>AtGA2ox1</em> through <em>AtGA2ox3</em> (similarity from 40.4% to 67.5%) than to <em>AtGA2ox7</em> (17.3%) and <em>AtGA2ox8</em> (21.5%). 
                     
                  </p>
                  <p>
                     
                     To test whether <em>DTA1</em> encodes a gene product with GA 2-oxidase activity, the full-length coding region was expressed in <em>Escherichia coli</em> as a C-terminal fusion to maltose binding protein. The recombinant fusion protein was present in the soluble portion of the
                     induced cell lysate and was used for enzyme assays. As shown in <a href="#TBL1">Table 1</a>, 17,17-<sup>2</sup>H<sub>2</sub>-GA<sub>4</sub> was converted to <sup>2</sup>H<sub>2</sub>-GA<sub>34</sub> as identified by gas chromatography–mass spectrometry (GC-MS). Also, <sup>2</sup>H<sub>2</sub>-GA<sub>1</sub> and <sup>2</sup>H<sub>2</sub>-GA<sub>20</sub> were converted to <sup>2</sup>H<sub>2</sub>-GA<sub>8</sub> and <sup>2</sup>H<sub>2</sub>-GA<sub>29</sub>, respectively (<a href="#TBL1">Table 1</a>; data not shown). Thus, <em>DTA1</em> encodes a functional GA 2-oxidase. Given the sequence similarity, chromosomal location (at the top of chromosome 1), and
                     the fact that DTA1 can catalyze 2ß-hydroxylation of gibberellins, <em>DTA1</em> corresponds to a gene with a product predicted to function as a GA 2-oxidase, <em>AtGA2ox6</em> (Hedden et al., 2002<a href="#BIB19"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>, and unpublished data therein), and has been renamed to indicate this identity. This gene also corresponds to <em>AtGA2ox4</em> in Ogawa et al. (2003)<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. 
                     
                  </p>
                  <p>
                     
                     <a name="TBL1" id="TBL1"/>
                     
                     
                  </p>
                  <div class="table pos-float">
                     
                     <div class="table-inline">
                        <strong>View this table:</strong><br/><a href="1206/T1.expansion.html">In this window</a><br/><a href="1206/T1.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="table-caption">
                        <span class="fig-label"><strong>Table 1.</strong></span> GA 2-Oxidase Enzyme Activity Assay of the Recombinant AtGA2ox6 Protein 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     <strong><em>AtGA2ox6</em> Is a Direct Downstream Target of AGL15</strong><br/>
                     
                     To confirm that the DNA fragment corresponding to the 5' regulatory region of <em>AtGA2ox6</em> was specifically selected in the ChIP experiments, enrichment tests were performed on independently generated ChIP populations
                     (Wang et al., 2002<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). If a DNA fragment is truly associated with AGL15 in vivo, it should be represented in greater abundance after selection
                     by ChIP than an unbound fragment, for example, from the coding region of <em>ubiquitin extension protein</em> (<em>UBQ6</em>; Callis et al., 1990<a href="#BIB4"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Multiplex PCR using primers that amplify the target and control fragments, on input (total) and on ChIP-selected DNA, revealed
                     that the ratio of <em>AtGA2ox6</em> fragment to control fragment was higher in the ChIP population compared with the input DNA (<a href="#FIG1">Figure 1A</a>). After two sequential immunoprecipitations using AGL15-specific antiserum, the <em>AtGA2ox6</em> fragment was represented in even greater abundance compared with the control fragment (<a href="#FIG1">Figure 1A</a>, ChIP2). Chromatin immunoprecipitation populations generated using preimmune serum did not show specific selection of the
                     <em>AtGA2ox6</em> fragment (data not shown), nor did a series of controls to verify that the immune serum–AGL15-DNA fragment association was
                     specific (Wang et al., 2002<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     <a name="FIG1" id="FIG1"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="1206/F1.expansion.html"><img src="1206/F1.small.gif"/></a><br/><strong>View larger version</strong> (44K):<br/><a href="1206/F1.expansion.html">In this window</a><br/><a href="1206/F1.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 1.</strong></span> AGL15 Interacts in a Sequence-Specific Manner with a Noncanonical CArG Motif Present in the Regulatory Region of <em>AtGA2ox6</em>.
                        
                        <p>
                           
                           <strong>(A)</strong> Enrichment of the <em>AtGA2ox6</em> fragment in a ChIP population selected using AGL15 antiserum. Oligonucleotide primer pairs to amplify the region of <em>AtGA2ox6</em> cloned in the ChIP population and to amplify control regions not expected to be bound by AGL15 (<em>UBQ6</em>) were used in multiplex PCR on total (Input) DNA diluted 125- and 625-fold, and on DNA recovered by immune precipitation
                           in ChIP after one (ChIP1) and two (ChIP2) sequential immunoprecipitations. PCR products were resolved on an 8% polyacrylamide
                           gel. Size of the PCR products is indicated at right. Numbers refer to the relative enrichment calculated as in Noma et al.
                           (2001)<a href="#BIB24"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(B)</strong> Autoradiography of EMSAs to assess the interaction of AGL15 with <em>AtGA2ox6</em> regulatory regions. A 47-bp fragment containing the noncanonical CArG motif CCAATTTAATGG was used in EMSAs. Lanes 1 to 4
                           contained increasing amounts of AGL15 with a C-terminal T7 tag (0, 40, 80, and 160 ng) expressed and purified from <em>E. coli</em> and incubated with the radiolabeled DNA fragment. Lane 5 added 0.5 µg of anti-T7 antibody to the reaction in lane 4. Lanes
                           6 and 7 contained incubation of 0 and 160 ng, respectively, of purified AGL15-T7 incubated with a modified radiolabeled DNA
                           fragment, in which the CC within the CArG motif was changed to a TT.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(C)</strong> Autoradiography of EMSAs to assess the sequence-specific interaction of AGL15 with the DNA fragment derived from <em>AtGA2ox6</em>. Competitive binding assays with AGL15-T7 (160 ng), <sup>32</sup>P-labeled DNA fragment from the <em>AtGA2ox6</em> promoter with the native CArG of CCAATTTAATGG, and nonradiolabeled competitor corresponding to either the native CArG or
                           a mutated form in which the CC was changed to a TT were analyzed on polyacrylamide gels. Unlabeled competitor was present
                           at 50-, 200-, and 800-fold compared with radiolabeled fragment (lanes 1 to 3, native competitor; lanes 4 to 6, modified competitor).
                           
                        </p>
                        
                        <p>
                           
                           <strong>(D)</strong> Quantitative analysis of native <em>AtGA2ox6</em>:<em>GUS</em> and mutated <em>mAtGA2ox6</em>:<em>GUS</em> in wild-type and <em>AGL15</em> ectopically expressing backgrounds. MUG assays were performed to quantitate the amount of GUS activity to determine expression
                           from the native <em>AtGA2ox6</em> regulatory region and from a modified form of the <em>AtGA2ox6</em> promoter, in which the CC in the putative AGL15 binding site was changed to a TT. Expression of the reporter gene was measured
                           in sibling seedlings that contained or lacked a <em>35S</em>:<em>AGL15</em> transgene (<em>n</em> = 16 to 21, except for <em>AtGA2ox6</em>:<em>GUS</em>, line 2, where <em>n</em> = 6 to 9; standard error of the mean is shown). Different letters indicate significant difference as assessed by Tukey's
                           mean separation test (SAS, 1999<a href="#BIB40"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) when wild-type and <em>35S</em>:<em>AGL15</em> siblings were compared within a reporter line.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(E)</strong> Enrichment of a DNA fragment from a putative Brassica ortholog of <em>AtGA2ox6</em> in a ChIP population selected using AGL15 antiserum. Oligonucleotide primer pairs were designed to amplify the regulatory
                           region of a putative <em>BnGA2ox</em> that is similar to <em>AtGA2ox6</em> and to amplify control regions not expected to be bound by AGL15 (<em>BnTUB</em>). Multiplex PCR was performed on total (Input) DNA diluted 125- and 625-fold, and on DNA recovered by immune precipitation
                           (ChIP) or preimmune precipitation (PI-ChIP) from developing Brassica seeds. PCR products were resolved on a 2% agarose gel.
                           Numbers refer to the relative enrichment calculated as in Noma et al. (2001)<a href="#BIB24"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(F)</strong> Response of expression of <em>AtGA2ox6</em> to AGL15 levels. RT-PCR products using oligonucleotide primers to assess the expression of <em>AtGA2ox6</em> in wild-type and <em>agl15</em> mutant 5- to 8-d developing seeds were analyzed on a 1.5% agarose gel. Tubulin served as a control. A representative gel
                           image is shown; numbers refer to the mean from quantitation of three biological replicates, normalized using tubulin and compared
                           with the wild type.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     The selected fragment did not contain any canonical CArG motifs of the form CC[A/T]<sub>6</sub>GG (reviewed in Riechmann and Meyerowitz, 1997<a href="#BIB37"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), and no canonical sequences were found nearby. However, in vitro binding site selection studies revealed that AGL15 preferentially
                     binds to sequences with a longer A/T-rich region, of the form [A/T]<sub>3</sub>C[A/T]<sub>8</sub>G[A/T]<sub>3</sub> (Tang and Perry, 2003<a href="#BIB48"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). A potential binding site for AGL15 with a longer A/T-rich region was present in the <em>AtGA2ox6</em> fragment (CC[A/T]<sub>8</sub>GG). To test whether AGL15 could associate with this sequence in vitro, electrophoretic mobility shift assays (EMSAs) were
                     performed with a <sup>32</sup>P-radiolabeled DNA fragment containing this site. AGL15 with a C-terminal T7 epitope was expressed in <em>E. coli</em>, and the protein purified via the T7 epitope. As shown in <a href="#FIG1">Figure 1B</a>, purified AGL15-T7 was able to retard the mobility of the probe with the noncanonical CArG motif (<a href="#FIG1">Figure 1B</a>, lanes 2 to 4; 40, 80, and 160 ng of AGL15-T7, respectively; lane 1, no AGL15-T7). Addition of a T7-specific antibody caused
                     a decrease in the intensity of the shifted band and appearance of supershifted complexes (<a href="#FIG1">Figure 1B</a>, lane 5), indicating that AGL15-T7 was responsible for retardation of the probe. Binding of AGL15 to the DNA fragment was
                     sequence specific. When the CC within the CArG motif was changed to TT, no binding was observed in EMSAs (<a href="#FIG1">Figure 1B</a>, lane 7, with 160 ng AGL15-T7; lane 6, no AGL15-T7). Additionally, a 50- to 800-fold excess of unlabeled DNA fragment with
                     the native binding site was able to compete efficiently with the radiolabeled native site for binding to AGL15 (<a href="#FIG1">Figure 1C</a>, lanes 1 to 3; 50-, 200-, and 800-fold unlabeled competitor, respectively), but the same amounts of mutated unlabeled fragment
                     had little effect (<a href="#FIG1">Figure 1C</a>, lanes 4 to 6; 50-, 200-, and 800-fold unlabeled mutated competitor). 
                     
                  </p>
                  <p>
                     
                     Although AGL15 was able to specifically bind in vitro to a probe containing the CC[A/T]<sub>8</sub>GG motif, in vitro binding does not necessarily indicate in vivo interaction or function. Enrichment tests indicated that
                     AGL15 interacted in vivo with the <em>AtGA2ox6</em> fragment that included this site (<a href="#FIG1">Figure 1A</a>). But does AGL15 truly contribute to regulation of <em>AtGA2ox6</em> through the CC[A/T]<sub>8</sub>GG element? To address this question, quantitative analysis of reporter constructs in response to AGL15 accumulation were
                     performed. Previous work using RT-PCR and reporter constructs demonstrated that expression of <em>AtGA2ox6</em> is upregulated in response to increased accumulation of AGL15 as found in <em>35S</em>:<em>AGL15</em> transgenic plants. The reporter construct (<em>AtGA2ox6</em>:<em>GUS</em> [ß-glucuronidase], previously named <em>DTA1</em>:<em>GUS</em> in Wang et al., 2002<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) used in these studies consisted of ∼2.3-kb sequence 5' of the translation initiation codon fused to the reporter gene <em>GUS</em> and includes the <em>AtGA2ox6</em> fragment isolated by ChIP. To quantitate the response to AGL15, transgenic lines carrying the reporter construct were crossed
                     to <em>35S</em>:<em>AGL15</em> plants, subsequent progeny separated visually for the presence or absence of the <em>35S</em>:<em>AGL15</em> transgene (Fernandez et al., 2000<a href="#BIB7"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Wang et al., 2002<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), and GUS activity in individual seedlings measured using 4-methylumbelliferyl-ß-<span style="  font-size: 69.44%">D</span>-glucuronide (MUG) assays. Seedlings with the <em>35S</em>:<em>AGL15</em> and reporter transgenes had significantly greater GUS activity than did siblings with only the reporter transgene and no
                     ectopic or increased AGL15 accumulation (<a href="#FIG1">Figure 1D</a>). These results were confirmed with two independent reporter lines, each carrying one reporter insert. 
                     
                  </p>
                  <p>
                     
                     To test whether the CC[A/T]<sub>8</sub>GG motif within the <em>AtGA2ox6</em> regulatory region was important for the increased expression in response to AGL15, a second reporter construct was generated
                     that was identical to the <em>AtGA2ox6</em>:<em>GUS</em> transgene except that the potential binding site for AGL15 was mutated. Specifically, the CC in the CC[A/T]<sub>8</sub>GG motif was changed to TT, to which AGL15 cannot bind in vitro (<a href="#FIG1">Figure 1B</a>, lane 7). The mutated transgenic lines (<em>mAtGA2ox6</em>:<em>GUS</em>) were crossed to <em>35S</em>:<em>AGL15</em> plants, and GUS activities measured in subsequent generations in seedlings with and without the <em>35S</em>:<em>AGL15</em> transgene. In contrast with the reporter with the native binding site, the mutated site showed little to no response to AGL15
                     amounts (<a href="#FIG1">Figure 1D</a>). Two independent mutated lines with one insert each were assessed this way. Therefore, the noncanonical CArG motif was a
                     relevant <em>cis</em> element for the response to AGL15 in vivo. 
                     
                  </p>
                  <p>
                     
                     Protein gel blots and immunohistochemical localization studies indicate that the embryonic culture tissue used for ChIP accumulates
                     similar amounts of AGL15 compared with <em>Brassica napus</em> and Arabidopsis zygotic embryos (Wang et al., 2002<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). However, it is still possible that somewhat higher levels of AGL15 occupy sites that would not be occupied in nontransgenic
                     tissue. It is not reasonable to collect enough Arabidopsis embryos or seeds of early enough developmental stage for ChIP experiments.
                     <em>B. napus</em> is closely related to Arabidopsis, and it is feasible to obtain sufficient tissue for ChIP. The nucleotide sequence corresponding
                     to the coding region of <em>AtGA2ox6</em> was used to search the Brassica DNA database. A sequence from <em>B. oleracea</em> was 87% identical over 401 nucleotides (accession number <a href="/cgi/external_ref?access_num=BZ033271&amp;link_type=GEN">BZ033271</a>). <em>B. napus</em> is believed to be an amphidiploid from hybridization of <em>B. oleracea</em> and <em>B. rapa</em> (Parkin et al., 1995<a href="#BIB32"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), so it is reasonable to expect a closely related sequence in <em>B. napus</em>. The predicted protein product of the identified gene fragment was 87% identical and 93% similar to the N-terminal 133 amino
                     acid residues of the Arabidopsis protein. The first intron–exon boundary was in the same position in both species, but no
                     contiguous Brassica DNA fragments 3' to this sequence could be identified in the database. However, searches to obtain sequence
                     5' to the coding region were successful and allowed assembly of ∼620 bp 5' to the start codon (accession numbers <a href="/cgi/external_ref?access_num=BH741917&amp;link_type=GEN">BH741917</a> and <a href="/cgi/external_ref?access_num=BZ081538&amp;link_type=GEN">BZ081538</a>). Overlap between database entries were at least 260 nucleotides in length and 100% identical. The region ∼430 nucleotides
                     5' to the start codon is ∼70% identical to that found in Arabidopsis. Interestingly, the 5' region of the Brassica sequence
                     contains three potential
                     binding sites for MADS domain proteins in similar positions as found in Arabidopsis, including a C-8-G type site of the form
                     GC[A/T]<sub>8</sub>GG. Oligonucleotide primers were designed corresponding to this region of the Brassica sequence and used to test for enrichment
                     of this fragment in a ChIP population derived from young <em>B. napus</em> seeds containing heart to young bent-cotyledon stage embryos. As shown in <a href="#FIG1">Figure 1E</a>, this DNA fragment was enriched in the immune-precipitated sample (ChIP) but not in the preimmune-precipitated sample compared
                     with a DNA fragment from the coding region of a ß<em>-</em>tubulin gene (accession number <a href="/cgi/external_ref?access_num=AF258790&amp;link_type=GEN">AF258790</a>). The enriched fragment was sequenced to verify identity (data not shown). 
                     
                  </p>
                  <p>
                     
                     To further confirm that regulation of <em>AtGA2ox6</em> by AGL15 occurred in the context of zygotic embryogenesis, where AGL15 was expressed only from the native promoter, accumulation
                     of <em>AtGA2ox6</em> mRNA in staged wild-type and <em>agl15</em> mutant seeds (5 to 8 d postanthesis) was compared by RT-PCR. <em>agl15</em> contains a T-DNA insertion in the first intron, and no correctly spliced full-length transcript can be detected by RT-PCR
                     (M.D. Lehti-Shiu and D.E. Fernandez, personal communication). As shown in <a href="#FIG1">Figure 1F</a>, decreased amounts of <em>AtGA2ox6</em> mRNA accumulated when no AGL15 was present. This agrees with previous results using staged developing siliques (Wang et al.,
                     2002<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     <strong>Expression of <em>AtGA2ox6</em> and Relation to AGL15</strong><br/>
                     
                     To gain insight into the developmental role of AtGA2ox6, the expression pattern of <em>AtGA2ox6</em> was analyzed by RNA gel blot using various developmental stages and tissue types from Arabidopsis (<a href="#FIG2">Figure 2A</a>). <em>AtGA2ox6</em> transcript amounts were very high in roots and relatively high in rosette leaves and stems. Signals were also clearly detected
                     from bulk siliques but barely detectable in cauline leaves and flowers. As shown in <a href="#FIG2">Figure 2B</a>, expression of <em>AtGA2ox6</em> in siliques was highest during the early stages of development (5 to 6 d after fertilization [daf]), and mRNA accumulation
                     from this gene declined afterward. During and after germination, mRNA could be detected by RT-PCR (<a href="#FIG2">Figure 2C</a>). Increased accumulation of <em>AtGA2ox6</em> transcript was induced by GA<sub>4</sub> or GA<sub>3</sub> treatment of plants, indicating a role for feed-forward regulation (<a href="#FIG2">Figure 2D</a>). 
                     
                  </p>
                  <p>
                     
                     <a name="FIG2" id="FIG2"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="1206/F2.expansion.html"><img src="1206/F2.small.gif"/></a><br/><strong>View larger version</strong> (37K):<br/><a href="1206/F2.expansion.html">In this window</a><br/><a href="1206/F2.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 2.</strong></span> Developmental Expression of <em>AtGA2ox6</em>.
                        
                        <p>
                           
                           <strong>(A)</strong> RNA gel blot probed to assess <em>AtGA2ox6</em> expression pattern. Total RNA (20 µg per lane) was electrophoresed through an agarose gel, transferred to membrane, and incubated
                           with <sup>32</sup>P-<em>AtGA2ox6</em> specific probe. The ethidium bromide–stained gel showing rRNA served as a loading control.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(B)</strong> RNA slot blot probed to assess <em>AtGA2ox6</em> expression during silique development. Poly(A<sup>+</sup>) RNA (1 µg) extracted from different stages of silique development was applied to membrane using a vacuum manifold and incubated
                           with <sup>32</sup>P<em>-AtGA2ox6</em> specific probe. The same blot was stripped and assessed for equal loading with a <sup>32</sup>P-2-ß-Tubulin (<em>TUB2</em>) specific probe.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(C)</strong> RT-PCR products using <em>AtGA2ox6</em> or 2-ß-Tubulin (<em>TUB2</em>) specific oligonucleotide primers and RNA from germinating seeds and establishing seedlings were analyzed on a 1.5% agarose
                           gel.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(D)</strong> Feed-forward regulation of <em>AtGA2ox6</em>. RT-PCR analysis to assess <em>AtGA2ox6</em> expression in rosette leaves from plants treated with 100 µM GA<sub>3</sub>, GA<sub>4</sub>, or water (mock treated). 2-ß-Tubulin (<em>TUB2</em>) was used as a control.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     To further investigate the expression of <em>AtGA2ox6</em> during seed development, the activity of the regulatory regions of this gene was assessed using a GUS reporter construct
                     (<em>AtGA2ox6</em>:<em>GUS</em>). Transgenic plants containing the reporter construct showed a pattern of GUS activity consistent with RT-PCR studies (Wang
                     et al., 2002<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; <a href="#FIG2">Figure 2C</a>) and RNA gel blot analysis (<a href="#FIG2">Figures 2A and 2B</a>). Developing seeds or embryos of different stages were stained for GUS activity and demonstrated that the 5' regulatory region
                     of <em>AtGA2ox6</em> was active during early embryogenesis, from at least globular stage (∼3 daf) through torpedo stage (∼6 to 8 daf; <a href="#FIG3">Figures 3A to 3C</a>). During early stages, GUS activity was also detected in other tissues of the seed (<a href="#FIG3">Figure 3A</a>). Decreased GUS activity was observed in bent cotyledon embryos and was nearly undetectable in the late maturing embryo (<a href="#FIG3">Figures 3D to 3F</a>). The regulatory regions of <em>AtGA2ox6</em> were reactivated in imbibed seeds (by 8 h after imbibition; <a href="#FIG3">Figure 3G</a>), with high GUS activity persisting throughout the seedling until 5 d after imbibition, when expression became more localized
                     to the root and the shoot apex (<a href="#FIG3">Figures 3H to 3L</a>). 
                     
                  </p>
                  <p>
                     
                     <a name="FIG3" id="FIG3"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="1206/F3.expansion.html"><img src="1206/F3.small.gif"/></a><br/><strong>View larger version</strong> (75K):<br/><a href="1206/F3.expansion.html">In this window</a><br/><a href="1206/F3.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 3.</strong></span> <em>AtGA2ox6</em>:<em>GUS</em> Expression during Embryogenesis, Germination, and Seedling Establishment.
                        
                        <p>
                           
                           Different stages of development from plants carrying a reporter transgene were stained for GUS activity.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(A)</strong> Globular stage embryo (arrowhead) within a developing seed.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(B)</strong> Late heart/early torpedo stage embryo within a developing seed.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(C)</strong> Isolated torpedo stage embryo.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(D)</strong> Bent-cotyledon stage embryo.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(E)</strong> and <strong>(F)</strong> Maturation stage embryos.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(G)</strong> to <strong>(I)</strong> Isolated embryos from seed at 8, 24, and 48 h postimbibition, respectively.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(J)</strong> to <strong>(L)</strong> Seedlings at 3 d (<strong>[J]</strong> and <strong>[K]</strong>) and 5 d <strong>(L)</strong> postimbibition.
                           
                        </p>
                        
                        <p>
                           
                           Blue arrowheads in <strong>(F)</strong>, <strong>(I)</strong>, and <strong>(J)</strong> indicate the root apical meristem. h, hypocotyl; c, cotyledons. Bars in <strong>(A)</strong> to <strong>(D)</strong> and <strong>(H)</strong> = 100 µm; in <strong>(K)</strong> and <strong>(L)</strong> = 0.5 mm.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     Plants that express <em>AtGA2ox6</em> ectopically and at higher levels and plants that have decreased expression of <em>AtGA2ox6</em> were obtained to assess in vivo function. The <em>AtGA2ox6</em> cDNA or the <em>AtGA2ox6</em> genomic region including introns were introduced into Arabidopsis plants under the control of the CaMV 35S promoter. A greater
                     percentage of transformants containing the genomic version of the gene showed a dwarf phenotype compared with transformants
                     containing the cDNA version of the transgene (∼70%, 19 out of 27 plants show an obvious phenotype for the genomic version
                     of the construct, as shown in <a href="#FIG4">Figures 4A and 4B</a>, lines 2 and 6, compared with ∼8%, 3 out of 37 plants, for the cDNA version of the construct). In accordance with the more
                     severe phenotype, the genomic-version
                     transgenic plants were found to accumulate higher amounts of <em>AtGA2ox6</em> mRNA (<a href="#FIG4">Figure 4C</a>; data not shown). Two independent lines were selected that carried one copy of the genomic version of the transgene and produced
                     different degrees of expression for further experiments as described below. 
                     
                  </p>
                  <p>
                     
                     <a name="FIG4" id="FIG4"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="1206/F4.expansion.html"><img src="1206/F4.small.gif"/></a><br/><strong>View larger version</strong> (46K):<br/><a href="1206/F4.expansion.html">In this window</a><br/><a href="1206/F4.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 4.</strong></span> Phenotypic Analysis of Arabidopsis with Altered Expression of <em>AtGA2ox6</em> or <em>AGL15</em>.
                        
                        <p>
                           
                           <strong>(A)</strong> and <strong>(B)</strong> Phenotype of two <em>35S</em>:<em>AtGA2ox6</em> transgenic lines (lines 2 and 6), a <em>35S</em>:<em>AGL15</em> transgenic line (<em>AGL15</em>), and a <em>ga2ox6</em> knockdown mutant (<em>ga2ox6-1</em>) compared with wild-type plants, at 5 <strong>(A)</strong> and at 7 to 9 weeks <strong>(B)</strong>.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(C)</strong> RNA gel blot to assess the amount of ectopic <em>AtGA2ox6</em> expression in <em>35S</em> transgenic plants (lines 2 and 6) compared with wild-type and <em>ga2ox6-1</em> plants. Total RNAs (7 µg) isolated from 3-week-old plants were analyzed by RNA gel blot with <em>AtGA2ox6</em> specific radiolabeled probe. rRNA is shown to assess equal loading.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(D)</strong> RT-PCR analysis of <em>ga2ox6-1</em>. Oligonucleotide primers corresponding to <em>AtGA2ox6</em> were used to assess the nature of the insertional mutation in <em>AtGA2ox6</em>. Oligonucleotides that amplify 2-ß-Tubulin (<em>TUB2</em>) were used as a control.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(E)</strong> Application of biologically active gibberellin rescued the <em>35S</em>:<em>AtGA2ox6</em> plants. GA<sub>3</sub> was applied to the severe overexpressing line 6, just before or at the onset of bolting. Phenotypes were compared with wild-type
                           and mock-treated line 6 plants 10 d after application of GA.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(F)</strong> Application of biologically active gibberellin (GA<sub>3</sub>) partially rescued the phenotype of <em>35S</em>:<em>AGL15</em> plants. GA<sub>3</sub> was applied when plants were 3.5 weeks old and were photographed at 5 weeks.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(G)</strong> Altered GA profiles in transgenic plants constitutively expressing <em>AGL15</em> or <em>AtGA2ox6</em> compared with the wild type. Endogenous GA concentrations in rosette leaves were obtained using GC-MS and internal standards.
                           Means from three replicates and standard error of the mean are shown. Different letters indicate significant difference as
                           assessed by Tukey's mean separation test (SAS, 1999<a href="#BIB40"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Uppercase and lowercase letters indicate different comparisons.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     In addition to generation of gain-of-function plants, a mutant containing a T-DNA insertion in the first intron of <em>AtGA2ox6</em> was isolated from the University of Wisconsin–Madison Arabidopsis Knockout Facility α-T-DNA population. To confirm whether
                     the mutant truly represents a knockout allele, RT-PCR was performed after obtaining
                     a plant that was homozygous for a single T-DNA insertion within <em>AtGA2ox6</em>. A very weak DNA band was clearly visible in the mutant after 40 cycles of PCR amplification. Although correctly spliced
                     mRNA was detected, the amount was much less than found in wild-type plants (<a href="#FIG4">Figure 4D</a>). Therefore, this allele, named <em>ga2ox6-1</em>, did not represent a complete knockout of the gene but rather was a knockdown mutation (Krysan et al., 1999<a href="#BIB21"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     As shown in <a href="#FIG4">Figures 4A, 4B, and 4E</a>, the <em>AtGA2ox6</em> overexpressing plants showed a GA-deficient phenotype (reviewed in Richards et al., 2001<a href="#BIB35"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). The plants were dwarf, dark green, flowered later than the wild type, showed decreased apical dominance, and remained green
                     longer. Line 6 showed a severe phenotype (<a href="#FIG4">Figures 4A, 4B, and 4E</a>) and accumulated more <em>AtGA2ox6</em> mRNA (<a href="#FIG4">Figure 4C</a>) than did line 2, which had a weaker phenotype (<a href="#FIG4">Figures 4A to 4C</a>). As summarized in <a href="#TBL2">Table 2</a>, plants with the <em>35S</em>:<em>AtGA2ox6</em> transgene produced an increased number of rosette leaves, more inflorescence branches, shorter internodes, and smaller fruit
                     size compared with the wild type. The number of seeds per silique was less than in the wild type, and the seeds were more
                     compactly arranged within the silique. Many aspects of the <em>35S</em>:<em>AtGA2ox6</em> phenotype were similar to that found when <em>AGL15</em> is expressed via the CaMV 35S promoter (Fernandez et al., 2000<a href="#BIB7"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; <a href="#FIG4">Figure 4A</a>; <a href="#TBL2">Table 2</a>). However, the delay in senescence and abscission of the floral organs reported for the <em>35S</em>:<em>AGL15</em> transgenic plants (Fernandez et al., 2000<a href="#BIB7"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) was not observed in the <em>35S</em>:<em>AtGA2ox6</em> plants. All of the aspects of the <em>35S</em>:<em>AtGA2ox6</em> phenotype could be rescued by spraying the plants with either GA<sub>4</sub> or GA<sub>3</sub> (<a href="#FIG4">Figure 4E</a>; data not shown). Some aspects of the <em>35S</em>:<em>AGL15</em> phenotype could be alleviated by spraying with biologically active GAs, in particular the darker green color, late flowering,
                     and decreased stature (<a href="#FIG4">Figure 4F</a>). The delay in floral organ senescence and abscission observed in the <em>35S</em>:<em>AGL15</em> plants was not rescued by spraying with GA<sub>3</sub> or GA<sub>4</sub> (data not shown). The <em>ga2ox6-1</em> plants exhibited only very mild phenotypes that were not significantly different from the wild type (<a href="#FIG4">Figures 4A and 4B</a>; <a href="#TBL2">Table 2</a>). 
                     
                  </p>
                  <p>
                     
                     <a name="TBL2" id="TBL2"/>
                     
                     
                  </p>
                  <div class="table pos-float">
                     
                     <div class="table-inline">
                        <strong>View this table:</strong><br/><a href="1206/T2.expansion.html">In this window</a><br/><a href="1206/T2.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="table-caption">
                        <span class="fig-label"><strong>Table 2.</strong></span> Phenotypic Comparison of the Wild Type to Overexpression Lines of <em>AtGA2ox6</em> and <em>AGL15</em> and to <em>ga2ox6-1</em> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     Because the phenotypes of both the <em>AtGA2ox6</em> and <em>AGL15</em> overexpressors suggested deficiencies in biologically active GAs, the amounts of select GAs in <em>AtGA2ox6</em> and <em>AGL15</em> overexpressors and wild-type Arabidopsis rosette leaves were determined using GC-MS. Compared with the wild-type control,
                     the major biologically active GA in Arabidopsis, GA<sub>4</sub> (Hedden, 2002<a href="#BIB17"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Ogawa et al., 2003<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), was significantly decreased in both the <em>AtGA2ox6</em> and <em>AGL15</em> overexpressor plants compared with the wild type (<a href="#FIG4">Figure 4G</a>; <em>n</em> = 3). GA<sub>34</sub> is the GA 2-oxidase metabolite of GA<sub>4</sub> (reviewed in Hedden and Phillips, 2000<a href="#BIB18"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) and was present at increased amounts in both <em>AtGA2ox6</em> and <em>AGL15</em> overexpressors, compared with the wild type (<a href="#FIG4">Figure 4G</a>). 
                     
                  </p>
                  <p>
                     
                     <strong>The Role of AtGA2ox6 during Seed Development and Germination</strong><br/>
                     
                     Because of the intriguing expression of <em>AtGA2ox6</em> during early stages of silique development (<a href="#FIG2">Figure 2B</a>) that was subsequently found to be localized predominantly in the developing embryo (<a href="#FIG3">Figures 3A to 3F</a>) and the second phase of expression during and after germination (<a href="#FIG2">Figures 2C</a> and <a href="#FIG3">3G to 3L</a>), experiments to determine the developmental consequences of <em>AtGA2ox6</em> expression on seeds were performed. Germination assays were used to test whether <em>AtGA2ox6</em> expression may influence seed dormancy. Seeds (14 d postharvest) from plants grown at the same time and under identical conditions
                     were sown on moist germination paper and incubated at 25°C under continuous light without any chilling treatment. As shown
                     in <a href="#FIG5">Figure 5A</a>, the percentage germination was negatively correlated with the amount of <em>AtGA2ox6</em> expression. Within 6 d, ∼38% of the seed from wild-type plants completed germination (<a href="#FIG5">Figure 5A</a>). However, a higher percentage of seed from plants homozygous for the knockdown mutation <em>ga2ox6-1</em> completed germination within the same time frame (∼54%; <a href="#FIG5">Figure 5A</a>). Seed from plants with higher than normal expression of <em>AtGA2ox6</em> completed germination to a lower percentage within this time period (only 9% to 13%), and the germination percentage was
                     related to the amount of expression of the transgene. Prechilling for 3 d increased the percentage germination of <em>35S</em>:<em>AtGA2ox6</em> seeds and could completely or nearly completely break the dormancy of wild-type, <em>35S</em>:<em>AGL15</em>, and <em>ga2ox6-1</em> seeds (<a href="#FIG5">Figure 5A</a>). Introduction of the <em>ga2ox6-1</em> knockdown mutation into <em>35S</em>:<em>AGL15</em> plants significantly increased the percentage of seeds capable of completing germination within 6 d, above that observed
                     for <em>35S</em>:<em>AGL15</em> seed grown and harvested under identical conditions (<a href="#FIG5">Figure 5B</a>). 
                     
                  </p>
                  <p>
                     
                     <a name="FIG5" id="FIG5"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="1206/F5.expansion.html"><img src="1206/F5.small.gif"/></a><br/><strong>View larger version</strong> (21K):<br/><a href="1206/F5.expansion.html">In this window</a><br/><a href="1206/F5.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 5.</strong></span> Germination Phenotype of Seeds from Gain-of-Function and Loss-of-Function <em>AtGA2ox6</em> Plants Compared with <em>35S</em>:<em>AGL15</em> and Wild-Type Plants.
                        
                        <p>
                           
                           <strong>(A)</strong> The amount of expression of <em>AtGA2ox6</em> was correlated with the degree of seed dormancy. Seeds were stored at room temperature for 14 d after harvest before performing
                           the germination tests.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(B)</strong> The <em>ga2ox6-1</em> knockdown mutant increased the percentage of <em>35S</em>:<em>AGL15</em> seed that could complete germination within 6 d. Seeds were stored at room temperature for 2 d after harvest before performing
                           the germination tests.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(C)</strong> The <em>ga2ox6-1</em> knockdown mutant seed was more resistant to the GA biosynthesis inhibitor paclobutrazol. Presence of a complementation transgene
                           in <em>ga2ox6-1</em> partially or completely rescued the resistance to paclobutrazol. Seeds were stored at room temperature for &gt;1 year before
                           performing germination tests.
                           
                        </p>
                        
                        <p>
                           
                           The percentage of seeds that completed germination was scored after 6 d at 25°C under continuous light. Standard deviation
                           (<strong>[A]</strong> and <strong>[C]</strong>) or standard error of the mean <strong>(B)</strong> is shown. Different letters indicate significant difference as assessed by Tukey's mean separation test (SAS, 1999<a href="#BIB40"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). In <strong>(A)</strong>, upper and lower case letters indicate different comparisons. In <strong>(C)</strong>, comparisons are between Wassilewskija (Ws) and <em>ga2ox6-1</em> at each paclobutrazol concentration.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     To test whether the observed seed germination results were as a result of different degrees of dormancy imposed by different
                     endogenous GA amounts, seed germination in response to biologically active GA<sub>4+7</sub> (reviewed in Hedden and Phillips, 2000<a href="#BIB18"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) or to a GA-biosynthesis inhibitor paclobutrazol (reviewed in Yamaguchi and Kamiya, 2002<a href="#BIB57"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) was investigated. The low percentage germination of the <em>35S</em>:<em>AtGA2ox6</em> seed could be completely rescued by the addition of 100 µM GA<sub>4+7</sub> (data not shown). The <em>ga2ox6-1</em> mutant seed was more resistant to paclobutrazol than was wild-type seed (<a href="#FIG5">Figure 5C</a>). Both wild-type and <em>ga2ox6-1</em> seed germination were completely inhibited when the concentration of paclobutrazol exceeded 20 µM, but the mutant seed consistently
                     displayed a higher percentage of germination than the wild type at lower concentrations of paclobutrazol. Transgenic <em>ga2ox6-1</em> lines containing a complementation construct, in which <em>AtGA2ox6</em> was expressed from its own regulatory regions, partially or completely rescued the resistance to paclobutrazol (<a href="#FIG5">Figure 5C</a>). 
                     
                  </p>
                  <p>
                     
                     <strong>AGL15, AtGA2ox6, GAs, and Somatic Embryogenesis</strong><br/>
                     
                     Recently, constitutive expression of <em>AGL15</em> by the 35S promoter was shown to enhance production of somatic embryos from the shoot apical meristem (SAM) when seeds completed
                     germination in liquid culture containing 2,4-D (Harding et al., 2003<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). To determine whether expression of <em>AtGA2ox6</em> was involved in production of SAM embryos, the effect of the knockdown mutant in the <em>35S</em>:<em>AGL15</em> transgenic background was investigated. As shown in <a href="#FIG6">Figure 6A</a>, inability to express normal amounts of <em>AtGA2ox6</em> repressed the ability of <em>35S</em>:<em>AGL15</em> to promote SAM somatic embryo development. Additionally, <em>ga2ox6-1</em> in the absence of the <em>35S</em>:<em>AGL15</em> transgene produced fewer SAM somatic embryos than did wild-type seedlings (<a href="#FIG6">Figure 6B</a>). However, seedlings expressing a cDNA version of <em>AtGA2ox6</em> via the 35S promoter showed enhanced production of SAM somatic embryos in three independent transgenic lines (<a href="#FIG6">Figure 6B</a>).  
                     
                  </p>
                  <p>
                     
                     <a name="FIG6" id="FIG6"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="1206/F6.expansion.html"><img src="1206/F6.small.gif"/></a><br/><strong>View larger version</strong> (22K):<br/><a href="1206/F6.expansion.html">In this window</a><br/><a href="1206/F6.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 6.</strong></span> Somatic Embryo Production from the Shoot Apex of Seedlings in Culture.
                        
                        <p>
                           
                           <strong>(A)</strong> The <em>ga2ox6-1</em> knockdown mutant decreased the percentage of <em>35S</em>:<em>AGL15</em> seedlings that produced somatic embryos from the shoot apex in liquid culture.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(B)</strong> Expression of <em>AtGA2ox6</em> altered frequency of somatic embryo production from the shoot apex. Wild-type seedlings were compared with the <em>ga2ox6-1</em> mutant and to three independent transgenic lines that constitutively expressed a cDNA version of <em>AtGA2ox6</em>.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(C)</strong> Biologically active GAs influenced production of somatic embryos from the shoot apex of wild-type seedlings and seedlings
                           that constitutively expressed <em>AGL15</em>. GA<sub>3</sub> or paclobutrazol was added to the culture media.
                           
                        </p>
                        
                        <p>
                           
                           The percentage of seeds that produced SAM embryos was scored after 21 d. Means and standard error of the mean are shown. The
                           numbers next to the bars in <strong>(A)</strong> and <strong>(B)</strong> indicate the number of experiments. All treatments performed in <strong>(C)</strong> were in triplicate. Different letters indicate significant difference as assessed by Tukey's mean separation test (SAS, 1999<a href="#BIB40"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). In <strong>(B)</strong>, Tukey's test was performed between Wassilewskija (WS), the wild type, and <em>ga2ox6-1</em>, and between WS, the wild type, and the <em>35S</em>:<em>GA2ox6</em> lines separately. Uppercase and lowercase letters in <strong>(C)</strong> indicate different comparisons.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     To further test the effect of biologically active GA on production of SAM somatic embryos, wild-type and <em>35S</em>:<em>AGL15</em> seeds were allowed to complete germination in the culture system in the presence of either a biologically active GA, GA<sub>3</sub>, or in the presence of paclobutrazol. As shown in <a href="#FIG6">Figure 6C</a>, a decrease in biologically active GA by addition of paclobutrazol caused an increase in the percentage of seedlings that
                     produced SAM embryos in both wild-type and in <em>35S</em>:<em>AGL15</em> seedlings. Conversely, the presence of GA<sub>3</sub> decreased production of somatic embryos in both cases. 
                     
                  </p>
                  <p>
                     
                     <a name="SEC3" id="SEC3"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        DISCUSSION
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     ChIP was described previously as a means to identify nucleotide sequences bound by a transcription factor in vivo as a step
                     toward identification of directly regulated genes (Wang et al., 2002<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). In this article, further evidence is presented to verify that <em>DTA1</em> is a direct downstream target of AGL15 and to demonstrate that the protein product of this gene is a functional GA 2-oxidase.
                     <em>DTA1</em> corresponds to <em>AtGA2ox6</em> as designated in Hedden et al. (2002)<a href="#BIB19"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a> and has been renamed to indicate this identity. 
                     
                  </p>
                  <p>
                     
                     <em>AtGA2ox6</em> had a somewhat different expression pattern during development than do the previously described GA 2-oxidases (Thomas et
                     al., 1999<a href="#BIB50"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Schomburg et al., 2003<a href="#BIB41"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Although not expressed exclusively during embryogenesis (<a href="#FIG2">Figures 2A and 2C</a>), during this phase of the life cycle, expression of <em>AtGA2ox6</em> parallels nuclear accumulation of AGL15, with the highest levels of expression and accumulation respectively occurring between
                     globular to early torpedo stage of development (<a href="#FIG2">Figures 2B</a> and <a href="#FIG3">3</a>; Heck et al., 1995<a href="#BIB15"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Rounsley et al., 1995<a href="#BIB38"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Perry et al., 1996<a href="#BIB34"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). After torpedo stage, expression of <em>AtGA2ox6</em> and accumulation of AGL15 both decline and are undetectable or almost undetectable in the nearly mature embryo (<a href="#FIG2">Figures 2B</a> and <a href="#FIG3">3</a>; Perry et al., 1996<a href="#BIB34"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     What developmental role would a GA 2-oxidase have during early embryogenesis? Biologically active GAs have been suggested
                     to have a role in embryogenesis between the torpedo and early cotyledon stages, when cells in the embryo axis elongate (Swain
                     et al., 1997<a href="#BIB46"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Hays et al., 2002<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Perhaps it is important to keep active GA concentrations low during earlier stages of embryo development, when cell division
                     is more active than cell elongation, and AtGA2ox6 has a role in contributing to control of GA levels during this part of the
                     life cycle. Although viable mature seed was obtained from the <em>ga2ox6-1</em> mutant, this mutation in the first intron resulted in a decrease in expression rather than a complete loss of function. In
                     addition, other GA 2-oxidases are expressed during silique development (Thomas et al., 1999<a href="#BIB50"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Ogawa et al., 2003<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) that may contribute to functional redundancy. It is also possible that there are more subtle changes early in embryogenesis
                     that are compensated during the later stages of seed development as has been observed in pea (<em>Pisum sativum</em>; Swain et al., 1997<a href="#BIB46"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), although no obvious differences were noted in morphogenesis in staged seeds (S. Perry, unpublished data). However, the
                     knockdown mutation had a clear effect on seed germination, with the <em>ga2ox6-1</em> mutant showing a decrease in seed dormancy compared with the wild type (<a href="#FIG5">Figure 5A</a>). Furthermore, <em>ga2ox6-1</em> seeds were more resistant to the effects of a GA biosynthesis inhibitor, paclobutrazol, than were wild-type seeds (<a href="#FIG5">Figure 5C</a>). Although mature embryos showed little expression of <em>AtGA2ox6</em> (<a href="#FIG3">Figure 3F</a>), imbibed seeds and establishing seedlings accumulated mRNA from this gene (<a href="#FIG2">Figures 2C</a> and <a href="#FIG3">3G to 3L</a>). The germination phenotype may result from the GA status established during seed development and/or GA production and metabolism
                     during germination. 
                     
                  </p>
                  <p>
                     
                     The knockdown mutant partially alleviated the increased dormancy observed in freshly harvested <em>35S</em>:<em>AGL15</em> seeds (<a href="#FIG5">Figure 5B</a>) and repressed the ability of <em>35S</em>:<em>AGL15</em> seedlings to produce somatic embryos from the SAM in liquid culture media containing 2,4-D (<a href="#FIG6">Figure 6A</a>). In a nontransgenic background, the knockdown mutant also reduced production of SAM embryos relative to wild-type seedlings
                     (<a href="#FIG6">Figure 6B</a>). Conversely, expression of the cDNA corresponding to <em>AtGA2ox6</em> via the 35S promoter led to enhanced production of SAM embryos in culture (<a href="#FIG6">Figure 6B</a>). Furthermore, the production of SAM embryos could be manipulated in wild-type and <em>35S</em>:<em>AGL15</em> seedlings by addition of the biologically active GA<sub>3</sub> or by the GA biosynthesis inhibitor, paclobutrazol (<a href="#FIG6">Figure 6C</a>), indicating that production of SAM embryos is negatively correlated with biologically active GA. 
                     
                  </p>
                  <p>
                     
                     Production of SAM embryos in liquid culture has been correlated with meristem size, with the larger SAMs of <em>amp1</em>, <em>clv1</em>, and <em>clv3</em> proposed to provide a larger pool of undifferentiated cells capable of following an embryonic default pathway when exposed
                     to 2,4-D (Mordhorst et al., 1998<a href="#BIB23"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), although mutants lacking a functional SAM are still able to produce somatic embryos from the cotyledonary axils (Mordhorst
                     et al., 2002<a href="#BIB22"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Seedlings constitutively expressing <em>AGL15</em> in wild-type and <em>amp1</em> mutant backgrounds did not initially have a larger meristem than the corresponding background lacking the transgene (Harding
                     et al., 2003<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). However, shortly after completion of germination, increased proliferation at the meristem in populations of seedlings showing
                     enhanced SAM embryo production was observed (Harding et al., 2003<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Because biologically active GAs have been reported to interfere with SAM function (Hay et al., 2002<a href="#BIB13"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), one possible explanation for previous and current observations is that reduction of active GAs promotes meristem activity,
                     and other factors allow somatic embryo development from this activated meristem. 
                     
                  </p>
                  <p>
                     
                     Another possible explanation involves transitions between developmental phases. Inhibition of GA biosynthesis enhances the
                     ability of the <em>pickle</em> (<em>pkl</em>) mutant to produce embryos from roots of seedlings, but addition of biologically active GA represses this phenotype (Ogas
                     et al., 1997<a href="#BIB25"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>, 1999<a href="#BIB26"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). PKL is a CHD3 chromatin-remodeling factor that has been proposed, along with a GA-modulated factor, to repress embryo identity
                     in seedlings (Ogas et al., 1999<a href="#BIB26"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Rider et al., 2003<a href="#BIB36"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Biologically active GAs are involved in other phase transitions, such as juvenile to adult vegetative development as well
                     as transition to reproductive development (Scott et al., 1999<a href="#BIB43"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Sakamoto et al., 2001<a href="#BIB39"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>, and references therein). Therefore, a decrease in biologically active GAs in <em>35S</em>:<em>AGL15</em>, <em>35S</em>:<em>AtGA2ox6</em>, or wild-type seedlings treated with paclobutrazol may prevent the transition from embryonic to postembryonic development,
                     enhancing production of somatic embryos from the SAM in culture. 
                     
                  </p>
                  <p>
                     
                     GAs have been reported to have effects on somatic embryo development in contexts other than <em>pkl</em> and outside of the meristem (for example, Hutchinson et al., 1997<a href="#BIB20"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Ondrej et al., 2002<a href="#BIB29"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Chen and Chang, 2003<a href="#BIB6"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Tokuji and Kuriyama, 2003<a href="#BIB51"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). In carrot (<em>Daucus carota</em>) hypocotyl explants, GA inhibits early stages of somatic embryogenesis, whereas uniconazole, an inhibitor of GA biosynthesis,
                     promotes secondary embryogenesis from primary embryos (Tokuji and Kuriyama, 2003<a href="#BIB51"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Interestingly, if GA is added at the time of 2,4-D treatment, instead of after, a slight stimulation of somatic embryo
                     production is observed, suggesting a role for GA during the earliest stages of development. This could explain observations
                     that expression of a genomic version of <em>AtGA2ox6</em> via the 35S promoter, which accumulates <em>AtGA2ox6</em> mRNA to higher amounts than the cDNA version of the transgene, nearly completely blocks production of SAM embryos (H. Wang,
                     unpublished data). Also consistent is the observation that the few seeds capable of completing germination in the presence
                     of high levels of paclobutrazol (1 and 10 µM) do not produce any SAM embryos (H. Wang, unpublished data). 
                     
                  </p>
                  <p>
                     
                     Although results presented in this report suggest that part of the developmental consequences of ectopic expression of <em>AGL15</em> (Fernandez et al., 2000<a href="#BIB7"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) may be as a result of increased expression of <em>AtGA2ox6</em> and, therefore, decreased concentrations of biologically active GAs, not all aspects of <em>35S</em>:<em>AGL15</em> phenotype were partially or completely rescued by the knock-down mutant. For example, the knockdown mutant does not reduce
                     production of SAM embryos in the <em>35S</em>:<em>AGL15</em> background to the level found in the wild-type background. It is important to note that <em>AtGA2ox6</em> is one of at least five family members that have overlapping patterns of expression. Potential binding sites for MADS domain
                     proteins are present in the 5' regulatory regions of other family members. If AGL15 is able to upregulate more than one family
                     member, a mutation in <em>ga2ox6</em> would not be sufficient to completely rescue the phenotypic aspect. Additionally, AGL15 most likely regulates other genes
                     that encode proteins that contribute to particular aspects of observed phenotype. The fact that ectopic expression of <em>AtGA2ox6</em> was not sufficient to phenocopy all aspects of the <em>35S</em>:<em>AGL15</em> phenotype also suggests that other regulated gene products must be involved. Regulatory networks, in general, are likely
                     to be very complicated, with different cohorts of transcriptional regulators orchestrating control of a given gene's expression
                     in different temporal and spatial contexts. This is the likely explanation why <em>AtGA2ox6</em> was expressed to high abundance in developmental contexts outside of the accumulation pattern of AGL15. 
                     
                  </p>
                  <p>
                     
                     However, <em>AtGA2ox6</em> has met all of the criteria to demonstrate direct regulation by AGL15 (reviewed in Andrew and Scott, 1992<a href="#BIB1"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Carey and Smale, 2000<a href="#BIB5"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). A direct interaction between AGL15 and regulatory elements of <em>AtGA2ox6</em> was documented in vitro and in vivo, both in Arabidopsis and Brassica (<a href="#FIG1">Figure 1</a>). Binding results in a regulatory consequence: Both increased and decreased amounts of AGL15 have been shown to affect expression
                     of <em>AtGA2ox6</em> (<a href="#FIG1">Figures 1D and 1F</a>; Wang et al., 2002<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). The increased accumulation of <em>AtGA2ox6</em> mRNA in response to AGL15 was not an indirect effect resulting from feed-forward control (reviewed in Hedden and Phillips,
                     2000<a href="#BIB18"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Olszewski et al., 2002<a href="#BIB28"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) because endogenous biologically active GA concentrations were lower in <em>35S</em>:<em>AGL15</em> plants than in the wild type (<a href="#FIG4">Figure 4G</a>). Additionally, many aspects of ectopic expression of <em>35S</em>:<em>AGL15</em> can be explained by increased expression of <em>AtGA2ox6</em>, and, in some cases, a knockdown mutation of <em>AtGA2ox6</em> partially alleviated the phenotype conferred by the <em>35S</em>:<em>AGL15</em> transgene. Finally, as shown in Wang et al. (2002)<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a> and <a href="#FIG1">Figure 1D</a>, regulatory sequences of <em>AtGA2ox6</em> conferred regulation in response to AGL15, and a specific <em>cis</em>-acting element responsible for this regulation was identified. Thus, the evidence is very strong that AGL15 directly regulates
                     expression of a gene encoding a GA 2-oxidase. GAs have profound effects on plant growth and development and <em>AtGA2ox6</em>'s expression pattern, and the phenotypes of altered expression indicate an important role in embryogenesis, seed development,
                     and germination. 
                     
                  </p>
                  <p>
                     
                     <a name="SEC4" id="SEC4"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        METHODS
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     <strong>Plant Material</strong><br/><em>Arabidopsis thaliana</em> ecotype Wassilewskija seeds were surface-sterilized and sown on MS germination medium (Invitrogen, Carlsbad, CA) with 50
                     µg/mL of kanamycin for transgenic seed. After chilling at 4°C for 2 d, seeds were transferred to 22°C under a 23-h-light/1-h-dark
                     cycle. Seedlings were transplanted 7 to 10 d later to ProMix BX (Premier Brands, Rivière-du-Loup, Canada). Plants were grown
                     under a 16-h-light (∼110 µmol m<sup>–2</sup> s<sup>–1</sup>)/8-h-dark regime. 
                     
                  </p>
                  <p>
                     
                     <strong>Enrichment Test for in Vivo Binding by AGL15</strong><br/>
                     
                     Enrichment tests for in vivo binding of AGL15 to a DNA fragment containing the CArG motif from the <em>AtGA2ox6</em> regulatory region were performed as described in Wang et al. (2002)<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. Primers specific for <em>AtGA2ox6</em> (5'-TCCATACGGTTGACAAATG-3' and 5'-CCTAGTTTGAAAATGACTAGACTG-3'), as well as primers that amplify a DNA fragment from the coding
                     region of <em>UBQ6</em> (5'-GGTGCTAAGAAGAGGAAGAAT-3' and 5'-CTCCTTCTTTCTGGTAAACGT-3'), were used in multiplex PCR on total (input) DNA and on DNA
                     fragments selected by ChIP using AGL15-specific antiserum. ChIP experiments were performed as in Wang et al. (2002)<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a> and were followed by a second immunoprecipitation of AGL15-DNA fragments as described in Weinmann et al. (2001)<a href="#BIB54"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. Briefly, protein A-Sepharose beads (Sigma, St. Louis, MO) were washed after the first immunoprecipitation as described in
                     Wang et al. (2002)<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>, and elution was performed in 50 mM NaHCO<sub>3</sub> and 1% SDS by agitating on a Vortex-Genie 2, vortex setting 3 (Fisher Scientific, Pittsburgh, PA), for 30 min at room temperature.
                     The beads were removed by centrifugation and 5 volumes of immunoprecipitation buffer (Wang et al., 2002<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) added to the eluant along with another aliquot of anti-AGL15 serum. After incubation overnight at 4°C with gentle mixing
                     on a rotating wheel, the sample was centrifuged in a microcentrifuge at top speed for 2 min at 4°C, and the supernatant moved
                     to a new tube with protein A-Sepharose. The remainder of the second immunoprecipitation was identical to the description in
                     Wang et al. (2002)<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. Quantitation was performed using ImageQuant TL (Amersham Biosciences, Piscataway, NJ). 
                     
                  </p>
                  <p>
                     
                     ChIP was also performed using <em>Brassica napus</em> cv Tower seeds containing heart to early bent-cotyledon stage embryos. Oligonucleotide primers were designed to the 5' region
                     of a putative Brassica ortholog of <em>AtGA2ox6</em> (5'-GTGGCAATAATTTGGTTTTTCAA-3' and 5'-TGTTGAAGAAGGTAAACCCATGT-3') and to a Brassica ß-tubulin (5'-CGAGAGGATCACAGCAATACAG-3'
                     and 5'-GGATCCATTCCACAAAGTAGGA-3') for use in enrichment tests. 
                     
                  </p>
                  <p>
                     
                     <strong>Electrophoretic Mobility Shift Assay</strong><br/>
                     
                     A DNA fragment corresponding to the <em>AtGA2ox6</em> regulatory region (from –389 to –343) and containing a putative but noncanonical CArG motif (–372 to –361) was generated
                     using two partially complementary oligonucleotides and extension by PCR. The two oligonucleotides were as follows: oligonucleotide
                     1, 5'-CTTGTACAACTTAAGAT<strong>CC</strong>AATTTAATGGAGATA-3', and oligonucleotide 2, 5'-TGACATGCATGTCTATCTCCATTAAATT<strong>GG</strong>ATCT-3'. A mutated version of the DNA fragment was created using oligonucleotide 3 (same as oligonucleotide 1 except that
                     the <strong>CC</strong> was changed to TT) and oligonucleotide 4 (same as oligonucleotide 2 except that the <strong>GG</strong> was changed to AA). The wild-type and mutant DNA fragments were labeled with <sup>32</sup>P-dATP and EMSA performed as described previously (Wang et al., 2002<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     <strong>Generation of GUS Reporter Lines and Activity Assays</strong><br/><em>AtGA2ox6</em>:<em>GUS</em> transgenic plants, in which the reporter gene <em>GUS</em> was under the control ∼2.3 kb of the 5' regulatory region of <em>AtGA2ox6</em>, were as described in Wang et al. (2002)<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. To generate the <em>mAtGA2ox6</em>:<em>GUS</em> reporter plant lines, all approaches were the same except that the CC was changed to TT within a CArG motif located at –372
                     to –361 using oligonucleotides 3 and 4 described above and the QuikChange site-directed mutagenesis kit (Stratagene, La Jolla,
                     CA). 
                     
                  </p>
                  <p>
                     
                     Transgenic lines containing a single insert were identified and were crossed to <em>AGL15</em> constitutively expressing plants that contained a single hemizygous copy of <em>35S</em>:<em>AGL15</em>. The F1 siblings with or without the <em>35S</em>:<em>AGL15</em> transgene can be easily separated visually at the seedling stage, using petiole and hypocotyl length, and cotyledon epinasty
                     (Fernandez et al., 2000<a href="#BIB7"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). GUS activities of individual seedlings in subsequent generations were measured by MUG assays using a Hoefer DynaQuant 200
                     fluorometer (Hoefer Scientific Instruments, San Francisco, CA) as described in Gallagher (1992)<a href="#BIB10"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. Histochemical staining for GUS activity was as in Fernandez et al. (2000)<a href="#BIB7"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>, followed in some cases by clearing of the seeds in Hoyer's solution as described in Schwartz et al. (1994)<a href="#BIB42"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. 
                     
                  </p>
                  <p>
                     
                     <strong><em>AtGA2ox6</em> Gain-of-Function and Loss-of-Function Plants</strong><br/>
                     
                     Confirmation of the coding region of <em>AtGA2ox6</em> was described previously (Wang et al., 2002<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Two different constructs were generated to drive constitutive expression of <em>AtGA2ox6</em>. One construct corresponded to the cDNA, whereas the other included all of the introns (i.e., genomic version). The cDNA
                     was obtained by reverse transcription from developing siliques, followed by PCR amplification as described previously (Wang
                     et al., 2002<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) and insertion into the <em>Sma</em>I site of pBluescript II KS+ (<em>pBSII KS+</em>) (Stratagene) to produce <em>pBSII KS-AtGA2ox6</em>. The insert was excised using <em>Xba</em>I (5' end of the gene) and <em>Sal</em>I and ligated to pBIMC that was digested with <em>Xba</em>I (35S promoter side) and <em>Xho</em>I. pBIMC is a derivative of pBI121 in which the <em>GUS </em>gene is replaced by a multiple cloning site (provided by Deane Falcone, University of Kentucky). The gene with introns was
                     obtained by PCR amplification from Arabidopsis genomic DNA and ligation into pBIMC as above. All constructs were confirmed
                     by sequencing. The plasmids were introduced into <em>Agrobacterium tumefaciens</em> strain GV3101, and Arabidopsis plants were transformed as described previously (Wang et al., 2002<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     To obtain loss-of-function plants, both the Alpha and Basta T-DNA insertion populations (Arabidopsis Knockout Facility, University
                     of Wisconsin–Madison) were screened for insertion alleles of <em>AtGA2ox6</em> according to the instructions on the facility's Web site (<a href="http://www.biotech.wisc.edu/Arabidopsis">http://www.biotech.wisc.edu/Arabidopsis</a>). A primer corresponding to the T-DNA left border JL-202 (Krysan et al., 1999<a href="#BIB21"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) and two gene-specific primers located at the 5' and 3' regions of the <em>AtGA2ox6</em> gene were used to identify potential mutant alleles. The <em>AtGA2ox6</em> specific primers to screen the T-DNA population were 5'-GCTTTATTCATCTTGGCAATAATTTTGGT-3' and 5'-AACAATCCAACAATCATATTTTCTTTTTCAA-3'.
                     The primers used for RT-PCR are identical to those used to obtain the cDNA (Wang et al., 2002<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     The transgene for complementation tests consisted of ∼2.3 kb 5' of the start codon, the entire coding region including introns,
                     and ∼0.8 kb 3' of the stop codon. The fragment was generated by PCR amplification from wild-type Arabidopsis Wassilewskija
                     genomic
                     DNA, cloned into pCAMBIA1300, and introduced into <em>A. tumefaciens</em> GV3101 and <em>ga2ox6-1</em> homozygous plants. Transformants were selected on media containing both kanamycin and hygromycin. 
                     
                  </p>
                  <p>
                     
                     <strong>In Vitro GA 2-Oxidase Activity Assay</strong><br/>
                     
                     The coding region of <em>AtGA2ox6</em> was excised from the <em>pBSII KS-AtGA2ox6</em> construct described above using <em>Xho</em>I and <em>Hin</em>dIII, and inserted into the pMal-c2x expression vector (New England Biolabs, Beverly, MA). <em>Escherichia coli</em> strain BL21-RIL (Stratagene) was transformed with the <em>pMal-c2x-AtGA2ox6</em> fusion construct and expression was induced by addition of 1 mM isopropyl-ß-<span style="  font-size: 69.44%">D</span>-thiogalactopyranoside to a culture with OD<sub>600</sub> of 0.5. After 3 h at room temperature, the cells were harvested and processed using CelLytic B-II bacterial cell lysis/extraction
                     reagent (Sigma) following the manufacturer's instructions. The presence of the maltose binding protein-AtGA2ox6 fusion protein
                     in the soluble portion of the cell lysate was confirmed by SDS-PAGE. 
                     
                  </p>
                  <p>
                     
                     The GA 2-oxidase activity assay was performed by incubating 80 µL of soluble fraction of the <em>E. coli</em> cell lysate with 200 ng of [17,17-<sup>2</sup>H<sub>2</sub>]GA<sub>1</sub>, GA<sub>4</sub>, or GA<sub>20</sub> (purchased from Lew N. Mander, Australian National University, Canberra, Australia) in a final reaction volume of 200 µL
                     containing 50 mM Tris-HCl, pH 7.5, 0.5 mM FeSO<sub>4</sub>, 5 mM 2-ketoglutarate, and 5 mM ascorbate. The pH of the reaction was reconfirmed after the GA was added. After incubating
                     at 30°C for 2 h, the reaction was stopped by addition of 1.8 mL of 0.1 N HCl to lower the pH to between 2 and 3. The products
                     were extracted twice with 2 mL of ethyl acetate (saturated with 1% acetic acid), dried under vacuum, resuspended in 100 µL
                     of methanol, and derivatized to methylester-trimethylsilyl ether (Me-TMSi). Derivatization was performed first with 10 to
                     20 µL of methanol and 50 to 100 µL of fresh CH<sub>2</sub>N<sub>2</sub> and incubated at room temperature for 30 min. After solvents were eliminated under N<sub>2</sub>, the sample was freeze-dried, followed by treatment with 50 µL of a mixture of dry pyridine and <em>N</em>,<em>O</em>-bistrimethylsilyl trifluoroacetamide (1:2) plus 1% trimethylchlorosilane (Pierce Chemical, Rockford, IL) and heated for 30
                     min at 80°C. One microliter was injected in splitless mode onto a DB-5ms column (30 m, 0.25 mm ID <span style="font-family: arial,helvetica;  ">x</span> 0.25 µm film thickness; Agilent/J&amp;W Scientific, Folsom, CA), fitted in a gas chromatograph (Trace GC; ThermoFinnigan, San
                     Jose, CA) with a capillary direct interface to a mass spectrometer PolarisQ ion trap system (ThermoFinnigan). The GC temperature
                     program was 50°C (1-min hold) to 150°C at 20°C min<sup>–1</sup>, then to 280°C at 6°C min<sup>–1</sup>, using helium as a carrier gas at 1 mL min<sup>–1</sup>. For product identification, full scan spectra analysis of [17,17-<sup>2</sup>H<sub>2</sub>]GAs were monitored at the retention times of authentic [17,17-<sup>2</sup>H<sub>2</sub>]GA standards. 
                     
                  </p>
                  <p>
                     
                     <strong>Quantification of Endogenous GA Concentrations</strong><br/>
                     
                     Rosette leaves were harvested from plants 5 weeks after the completion of germination, flash frozen in liquid N<sub>2</sub>, and freeze-dried. Approximately 500 mg dry weight was homogenized in liquid N<sub>2</sub>, extracted with 80% methanol and 1% acetic acid at 4°C overnight, and 10 ng of each of the following GAs, [17, 17-<sup>2</sup>H<sub>2</sub>]GA<sub>1</sub>, GA<sub>4</sub>, GA<sub>8</sub>, GA<sub>9</sub>, GA<sub>19</sub>, GA<sub>20</sub>, and GA<sub>34</sub> (L. Mander, Australian National University, Adelaide, Australia), were added as internal standards and allowed to equilibrate
                     for 30 min. GA extraction and purification was as described in Volmaro et al. (1998)<a href="#BIB52"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>, except the second HPLC using a Nucleosil (N[CH<sub>3</sub>]<sub>2</sub>) column was omitted, and no bioassay was performed to locate GA presence. Instead, fractions from 10 to 30 min were collected
                     from the C<sub>18</sub> reverse phase column (µBondapak, 300 <span style="font-family: arial,helvetica;  ">x</span> 3.9 mm; Waters Associates, Milford, MA) at a flow rate of 2 mL min<sup>–1</sup>; from 0 to 10 min with 10% methanol in 1% acetic acid, from 10 to 40 min with 10% to 73% methanol in 1% acetic acid, from
                     40 to 50 min with 73% methanol in 1% acetic acid, and from 50 to 60 min with 100% methanol (conditions based on experience
                     of R. Bottini with the same column). After solvent evaporation under low pressure, samples were converted to the Me-TMSi derivatives,
                     and 1 µL was injected in splitless mode in a GC-MS as described above. In selected ion monitoring mode, the parent ion and
                     at least four other characteristic ions (Hedden, 1987<a href="#BIB16"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Talon et al., 1990<a href="#BIB47"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) for both the deutero-isotope and the endogenous compound were monitored. The identity of endogenous GAs was established
                     by coincidence of the relative intensities of the five characteristic ions and the retention time for the endogenous compound
                     with those of the deutero-GAs used as internal standards. By comparison of the peak area for the parent ion of the endogenous
                     GA versus the parent ion of the corresponding deutero GA standard, the amount of free acid GA was calculated from triplicate
                     samples using Xcalibur software (ThermoFinnigan). 
                     
                  </p>
                  <p>
                     
                     <strong>RNA Gel Blot Analysis</strong><br/>
                     
                     RNAs used for expression pattern analysis were extracted from various organs of 5-week-old plants as described previously
                     (Wang et al., 2002<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). To stage siliques, flowers were tagged on the day that they opened and siliques collected at the appropriate time afterward.
                     <em>35S</em>:<em>AtGA2ox6</em> plants were assessed for the amount of expression of the transgene by extracting total RNA from 12-d-old seedlings grown
                     on germination media. Total RNA was subjected to electrophoresis in a 1% agarose gel in the presence of 0.5 M formaldehyde
                     and then transferred to Zeta-Probe GT blotting membrane (Bio-Rad, Hercules, CA) according to the manufacturer's instructions.
                     For slot blots, 1 µg of mRNA (obtained using the PolyATract mRNA isolation system; Promega, Madison, WI) was applied to Zeta-Probe
                     GT blotting membrane using a vacuum manifold (Bio-Rad). Filters were probed with <sup>32</sup>P-labeled DNA probe specific for <em>AtGA2ox6</em> or for ß-2 Tubulin (Snustad et al., 1992<a href="#BIB45"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) at 68°C in 0.5 M NaHPO<sub>4</sub>, pH 7.2, and 7% SDS for 16 h, followed with two 30-min washes at 68°C in 40 mM KHPO<sub>4</sub>, pH 7.2, and 5% SDS and one 30-min wash in 40 mM NaHPO<sub>4</sub>, pH 7.2, and 1% SDS at 68°C. Washed filters were exposed to a storage phosphor screen (Molecular Dynamics, Sunnyvale, CA)
                     and signal detected on a PhosphorImager (Model 445SI; Molecular Dynamics). 
                     
                  </p>
                  <p>
                     
                     <strong>RT-PCR Analysis</strong><br/>
                     
                     RNA was isolated from imbibed seeds or establishing seedlings using the hot borate method (Wilkins and Smart, 1996<a href="#BIB55"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) or TRIzol reagent (Life Technologies, Rockville, MD) respectively. To test for feed-forward regulation, 3-week-old plants
                     were sprayed with 100 µM GA<sub>3</sub> or GA<sub>4</sub>, and rosette leaves collected 16 h afterward for total RNA isolation. Semiquantitative RT-PCR was performed to evaluate the
                     <em>AtGA2ox6</em> expression amounts as described previously (Wang et al., 2002<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Quantitation was performed using ImageQuant TL. 
                     
                  </p>
                  <p>
                     
                     <strong>Germination Assays</strong><br/>
                     
                     The seeds used for germination assays were harvested from mature plants of different genotypes grown in the same tray under
                     the conditions described above. To ensure the plants matured at the same time, <em>35S</em>:<em>AtGA2ox6</em> and <em>35S</em>:<em>AGL15</em> seeds were germinated 1 to 2 weeks earlier depending on the particular line. All the germination experiments were performed
                     in 4-cm Petri dishes on two layers of germination paper (Stults Scientific Eng., Springfield, IL) soaked with 4 mL of water,
                     different concentrations of GA<sub>4+7</sub>, or paclobutrazol. After 6 d of incubation under continuous light at 25°C, the percentage of seeds completing germination
                     was assessed. In some experiments, seeds were chilled at 4°C for 3 d after imbibition and before incubation at 25°C. Each
                     line was tested in triplicate, with ∼100 seeds per plate. 
                     
                  </p>
                  <p>
                     
                     <strong>Seedling Liquid Culture System</strong><br/>
                     
                     The liquid culture system was essentially as described by Mordhorst et al. (1998)<a href="#BIB23"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a> with further details provided by Harding et al. (2003)<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>, except that GA<sub>3</sub> or paclobutrazol was added to the media in some experiments. 
                     
                  </p>
                  <p>
                     
                     <strong>Statistical Analysis</strong><br/>
                     
                     All comparisons of the effect of the mutations, transgenes, or treatments were performed using analysis of variance (the analysis
                     of variance procedure of SAS, 1999<a href="#BIB40"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) when the number of replications was equal. If replication number was not equal, the general linear model was used (SAS,
                     1999<a href="#BIB40"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). In either instance, if the test indicated that there were significant differences among means, Tukey's mean separation
                     test (SAS, 1999<a href="#BIB40"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) was used to distinguish among them. 
                     
                  </p>
                  <p>
                     
                     Sequence data from this article have been deposited with the EMBL/GenBank data libraries under accession numbers <a href="/cgi/external_ref?access_num=At1g02400&amp;link_type=GEN">At1g02400</a>, <a href="/cgi/external_ref?access_num=B7033271&amp;link_type=GEN">B7033271</a>, <a href="/cgi/external_ref?access_num=BH741917&amp;link_type=GEN">BH741917</a>, <a href="/cgi/external_ref?access_num=B7081538&amp;link_type=GEN">B7081538</a>, and <a href="/cgi/external_ref?access_num=AF258790&amp;link_type=GEN">AF258790</a>. 
                     
                  </p>
                  <p>
                     
                     
                     <a name="ACK" id="ACK"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        Acknowledgments
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     We thank Weining Tang, Lynnette Dirk, and Randy Dinkins for critical reading of the manuscript, as well as members of the
                     University of Kentucky Seed Biology Group for helpful suggestions. We also acknowledge two anonymous reviewers for critical
                     review of and suggestions on the manuscript. We thank Donna Fernandez and Melissa Lehti-Shiu (University of Wisconsin–Madison)
                     for providing seeds for the constitutively expressing <em>AGL15</em> plants and for the <em>agl15</em> plants, Deane Falcone for the pBIMC vector, Qilong Xu and Cong Zhu (University of Kentucky, Lexington) for materials for
                     RT-PCR experiments, and Abbott Biochemicals (Chicago, IL) for the kind gift of the GA<sub>4+7</sub>. In addition, we thank Jack Goodman at the University of Kentucky Mass Spectrometry Facility for obtaining mass spectrometry
                     data, Neil Fannin for assistance with the HPLC, and Lowell Bush and Harold Burton (University of Kentucky) for use of lab
                     facilities for a portion of this work. We also acknowledge Ruben Bottini (University of Cuyo, Argentina) for training in quantitation
                     of GAs. This research was supported by the National Science Foundation (Grant IBN-9984274 to S.E.P.) and by the University
                     of Kentucky. This article (03-06-077) is published with the approval of the Director of the Kentucky Agricultural Experiment
                     Station. 
                     
                  </p>
                  <p>
                     
                     
                     <a name="FN" id="FN"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        Footnotes
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     <a name="" id=""/>
                     The author responsible for distribution of materials integral to the findings presented in this article in accordance with
                     the policy described in the Instructions for Authors (<a href="http://www.plantcell.org">www.plantcell.org</a>) is: Sharyn Perry (<span class="em-link"><span class="em-addr">sperr2{at}uky.edu</span></span>). 
                     
                  </p>
                  <p>
                     
                     Article, publication date, and citation information can be found at <a href="http://www.plantcell.org/cgi/doi/10.1105/tpc.021261">www.plantcell.org/cgi/doi/10.1105/tpc.021261</a>. 
                     
                  </p>
                  <p>
                     
                     
                     Received January 26, 2004; 
                     
                     accepted February 20, 2004.
                     
                  </p>
                  <p>
                     
                     <a name="BIBL" id="BIBL"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        REFERENCES
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     
                     <a name="BIB1" id="BIB1"/>
                     <strong>Andrew, D.J., and Scott, M.P.</strong> (1992). Downstream of the homeotic genes. New Biol. <strong>4,</strong> 5–15.<a href="/cgi/external_ref?access_num=A1992HC44900002&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=1346973&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB2" id="BIB2"/>
                     <strong>Arnone, M.I., and Davidson, E.H.</strong> (1997). The hardwiring of development: Organization and function of genomic regulatory systems. Development <strong>124,</strong> 1851–1864.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=124/10/1851">[Abstract]</a></p>
                  <p><a name="BIB3" id="BIB3"/>
                     <strong>Burgeff, C., Liljegren, S.J., Tapia-Lopez, R., Yanofsky, M.F., and Alvarez-Buylla, E.R.</strong> (2002). MADS-box gene expression in lateral primordia, meristems and differentiated tissues of Arabidopsis thaliana roots.
                     Planta <strong>214,</strong> 365–372.<a href="/cgi/external_ref?access_num=10.1007%2Fs004250100637&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000173848200005&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=11855641&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB4" id="BIB4"/>
                     <strong>Callis, J., Raasch, J.A., and Vierstra, R.D.</strong> (1990). Ubiquitin extension proteins of Arabidopsis thaliana: Structure, localization, and expression of their promoters
                     in transgenic tobacco. J. Biol. Chem. <strong>265,</strong> 12486–12493.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=jbc&amp;resid=265/21/12486">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB5" id="BIB5"/>
                     <strong>Carey, M., and Smale, S.T.</strong> (2000). Confirming the functional importance of a protein-DNA interaction. In Transcriptional Regulation in Eukaryotes: Concepts,
                     Strategies and Techniques. (New York, NY: Cold Spring Harbor Laboratory Press), pp. 291–317.
                     
                  </p>
                  <p><a name="BIB6" id="BIB6"/>
                     <strong>Chen, J.T., and Chang, W.C.</strong> (2003). Effects of GA<sub>3</sub>, ancymidol, cycolcel and paclobutrazol on direct somatic embryogenesis of Oncidium <em>in vitro</em>. Plant Cell Tissue Organ Cult <strong>72,</strong> 105–108.<a href="/cgi/external_ref?access_num=10.1023%2FA%3A1021235700751&amp;link_type=DOI">[CrossRef]</a></p>
                  <p><a name="BIB7" id="BIB7"/>
                     <strong>Fernandez, D.E., Heck, G.R., Perry, S.E., Patterson, S.E., Bleecker, A.B., and Fang, S.-C.</strong> (2000). The embryo MADS domain factor AGL15 acts postembryonically: Inhibition of perianth senescence and abscission via
                     constitutive expression. Plant Cell <strong>12,</strong> 183–197.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=12/2/183">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB8" id="BIB8"/>
                     <strong>Fickett, J.W., and Wasserman, W.W.</strong> (2000). Discovery and modeling of transcriptional regulatory regions. Curr. Opin. Biotechnol. <strong>11,</strong> 19–24.<a href="/cgi/external_ref?access_num=10.1016%2FS0958-1669%2899%2900049-X&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000085404600003&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=10679343&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB9" id="BIB9"/>
                     <strong>Flanagan, C.A., and Ma, H.</strong> (1994). Spatially and temporally regulated expression of the MADS-box gene <em>AGL2</em> in wild-type and mutant Arabidopsis flowers. Plant Mol. Biol. <strong>26,</strong> 581–595.<a href="/cgi/external_ref?access_num=10.1007%2FBF00013745&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=A1994PQ59000005&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=7948914&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB10" id="BIB10"/>
                     <strong>Gallagher, S.R.</strong> (1992). Chapter 3: Quantitation of GUS activity by fluorometry. In GUS Protocols: Using the GUS Gene as a Reporter of Gene
                     Expression, S.R. Gallagher, ed (San Diego, CA: Academic Press), pp. 47–59.
                     
                  </p>
                  <p><a name="BIB11" id="BIB11"/>
                     <strong>Graba, Y., Aragnol, D., and Pradel, J.</strong> (1997). <em>Drosophila</em> Hox complex downstream targets and the function of homeotic genes. Bioessays <strong>19,</strong> 379–388.<a href="/cgi/external_ref?access_num=10.1002%2Fbies.950190505&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=A1997WZ25600005&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=9174403&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB12" id="BIB12"/>
                     <strong>Harding, E.W., Tang, W., Nichols, K.W., Fernandez, D.E., and Perry, S.E.</strong> (2003). Expression and maintenance of embryogenic potential is enhanced through constitutive expression of <em>AGAMOUS-Like 15</em>. Plant Physiol. <strong>133,</strong> 653–663.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=133/2/653">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB13" id="BIB13"/>
                     <strong>Hay, A., Kaur, H., Phillips, A., Hedden, P., Hake, S., and Tsiantis, M.</strong> (2002). The gibberellin pathway mediates KNOTTED1-type homeobox function in plants with different body plans. Curr. Biol.
                     <strong>12,</strong> 1557–1565.<a href="/cgi/external_ref?access_num=10.1016%2FS0960-9822%2802%2901125-9&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000178132000015&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=12372247&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB14" id="BIB14"/>
                     <strong>Hays, D.B., Yeung, E.C., and Pharis, R.P.</strong> (2002). The role of gibberellins in embryo axis development. J. Exp. Bot. <strong>53,</strong> 1747–1751.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=jexbot&amp;resid=53/375/1747">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB15" id="BIB15"/>
                     <strong>Heck, G.R., Perry, S.E., Nichols, K.W., and Fernandez, D.E.</strong> (1995). AGL15, a MADS domain protein expressed in developing embryos. Plant Cell <strong>7,</strong> 1271–1282.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=7/8/1271">[Abstract]</a></p>
                  <p><a name="BIB16" id="BIB16"/>
                     <strong>Hedden, P.</strong> (1987). Gibberellins. In Principles and Practice of Plant Hormone Analysis, L. Rivier and A. Crozier, eds (London, UK: Academic
                     Press), pp. 9–110.
                     
                  </p>
                  <p><a name="BIB17" id="BIB17"/>
                     <strong>Hedden, P.</strong> (2002). Gibberellin metabolism and its regulation. J. Plant Growth Regul. <strong>20,</strong> 317–318.
                     
                  </p>
                  <p><a name="BIB18" id="BIB18"/>
                     <strong>Hedden, P., and Phillips, A.L.</strong> (2000). Gibberellin metabolism: New insights revealed by the genes. Trends Plant Sci. <strong>5,</strong> 523–530.<a href="/cgi/external_ref?access_num=10.1016%2FS1360-1385%2800%2901790-8&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000166393400017&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=11120474&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB19" id="BIB19"/>
                     <strong>Hedden, P., Phillips, A.L., Rojas, M.C., Carrera, E., and Tudzynski, B.</strong> (2002). Gibberellin biosynthesis in plants and fungi: A case of convergent evolution? J. Plant Growth Regul. <strong>20,</strong> 319–331.<a href="/cgi/external_ref?access_num=10.1007%2Fs003440010037&amp;link_type=DOI">[CrossRef]</a></p>
                  <p><a name="BIB20" id="BIB20"/>
                     <strong>Hutchinson, M.J., KrishnaRaj, S., and Saxena, P.K.</strong> (1997). Inhibitory effect of GA3 on the development of thidiazuron-induced somatic embryogenesis in geranium (<em>Pelargonium x hortorum</em> Bailey) hypocotyl cultures. Plant Cell Rep. <strong>16,</strong> 435–438.<a href="/cgi/external_ref?access_num=10.1007%2Fs002990050255&amp;link_type=DOI">[CrossRef]</a></p>
                  <p><a name="BIB21" id="BIB21"/>
                     <strong>Krysan, P., Young, J., and Sussman, M.</strong> (1999). T-DNA as an insertional mutagen in Arabidopsis. Plant Cell <strong>11,</strong> 2283–2290.<a href="/cgi/ijlink?linkType=FULL&amp;journalCode=plantcell&amp;resid=11/12/2283">[<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB22" id="BIB22"/>
                     <strong>Mordhorst, A.P., Hartog, M.V., Tamer, M.K.E., Laux, T., and de Vries, S.C.</strong> (2002). Somatic embryogenesis from Arabidopsis shoot apical meristem mutants. Planta <strong>214,</strong> 829–836.<a href="/cgi/external_ref?access_num=10.1007%2Fs00425-001-0700-6&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000175309400002&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=11941458&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB23" id="BIB23"/>
                     <strong>Mordhorst, A.P., Voerman, K.J., Hartog, M.V., Meijer, E.A., van Went, J., Koornneef, M., and de Vries, S.C.</strong> (1998). Somatic embryogenesis in <em>Arabidopsis thaliana</em> is facilitated by mutations in genes repressing meristematic cell divisions. Genetics <strong>149,</strong> 549–563.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=genetics&amp;resid=149/2/549">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB24" id="BIB24"/>
                     <strong>Noma, K.-i., Allis, C.D., and Grewal, S.I.S.</strong> (2001). Transitions in distinct histone H3 methylation patterns at the heterochromatin domain boundaries. Science <strong>293,</strong> 1150–1155.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=293/5532/1150">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB25" id="BIB25"/>
                     <strong>Ogas, J., Cheng, J.-C., Sung, Z.R., and Somerville, C.</strong> (1997). Cellular differentiation regulated by gibberellin in the <em>Arabidopsis thaliana pickle</em> mutant. Science <strong>277,</strong> 91–94.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=277/5322/91">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB26" id="BIB26"/>
                     <strong>Ogas, J., Kaufmann, S., Henderson, J., and Somerville, C.</strong> (1999). PICKLE is a CHD3 chromatin-remodeling factor that regulates the transition from embryonic to vegetative development
                     in <em>Arabidopsis</em>. Proc. Natl. Acad. Sci. USA <strong>96,</strong> 13839–13844.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=96/24/13839">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB27" id="BIB27"/>
                     <strong>Ogawa, M., Hanada, A., Yamauchi, Y., Kuwahara, A., Kamiya, Y., and Yamaguchi, S.</strong> (2003). Gibberellin biosynthesis and response during Arabidopsis seed germination. Plant Cell <strong>15,</strong> 1591–1604.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=15/7/1591">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB28" id="BIB28"/>
                     <strong>Olszewski, N., Sun, T.-p., and Gubler, F.</strong> (2002). Gibberellin signaling: Biosynthesis, catabolism, and response pathways. Plant Cell <strong>14</strong> (suppl.), S61–S80.<a href="/cgi/external_ref?access_num=12045270&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB29" id="BIB29"/>
                     <strong>Ondrej, V., Navratilova, B., and Lebeda, A.</strong> (2002). Influence of GA<sub>3</sub> on the zygotic embryogenesis of <em>Cucumis</em> species <em>in vitro</em>. Biologia (Bratisl.) <strong>57,</strong> 523–525.
                     
                  </p>
                  <p><a name="BIB30" id="BIB30"/>
                     <strong>Orlando, V.</strong> (2000). Mapping chromosomal proteins <em>in vivo</em> by formaldehyde-crosslinked-chromatin immunoprecipitation. Trends Biochem. Sci. <strong>25,</strong> 99–104.<a href="/cgi/external_ref?access_num=10.1016%2FS0968-0004%2899%2901535-2&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000085797100002&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=10694875&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB31" id="BIB31"/>
                     <strong>Orlando, V., Strutt, H., and Paro, R.</strong> (1997). Analysis of chromatin structure by <em>in vivo</em> formaldehyde cross-linking. Methods <strong>11,</strong> 205–214.<a href="/cgi/external_ref?access_num=10.1006%2Fmeth.1996.0407&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=A1997XT37600008&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=8993033&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB32" id="BIB32"/>
                     <strong>Parkin, I., Sharpe, A., Keith, D., and Lydiate, D.</strong> (1995). Identification of the A and C genomes of amphidiploid <em>Brassica napus</em> (oilseed rape). Genome <strong>38,</strong> 1122–1131.<a href="/cgi/external_ref?access_num=A1995TL86300009&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=18470236&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB33" id="BIB33"/>
                     <strong>Perry, S.E., Lehti, M.D., and Fernandez, D.E.</strong> (1999). The MADS-domain protein AGAMOUS-like 15 accumulates in embryonic tissues with diverse origins. Plant Physiol. <strong>120,</strong> 121–129.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=120/1/121">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB34" id="BIB34"/>
                     <strong>Perry, S.E., Nichols, K.W., and Fernandez, D.E.</strong> (1996). The MADS domain protein AGL15 localizes to the nucleus during early stages of seed development. Plant Cell <strong>8,</strong> 1977–1989.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=8/11/1977">[Abstract]</a></p>
                  <p><a name="BIB35" id="BIB35"/>
                     <strong>Richards, D.E., King, K.E., Ait-ali, T., and Harberd, N.P.</strong> (2001). How gibberellin regulates plant growth and development: A molecular genetic analysis of gibberellin signaling. Annu.
                     Rev. Plant Physiol. Plant Mol. Biol. <strong>52,</strong> 67–88.<a href="/cgi/external_ref?access_num=10.1146%2Fannurev.arplant.52.1.67&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000169615600004&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=11337392&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB36" id="BIB36"/>
                     <strong>Rider, S.D., Henderson, J.T., Jerome, R.E., Edenberg, H.J., Romero-Severson, J., and Ogas, J.</strong> (2003). Coordinate repression of regulators of embryonic identity by PICKLE during germination in Arabidopsis. Plant J. <strong>35,</strong> 33–43.<a href="/cgi/external_ref?access_num=10.1046%2Fj.1365-313X.2003.01783.x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000183841600004&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=12834400&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB37" id="BIB37"/>
                     <strong>Riechmann, J.L., and Meyerowitz, E.M.</strong> (1997). MADS domain proteins in plant development. Biol. Chem. <strong>378,</strong> 1079–1101.<a href="/cgi/external_ref?access_num=A1997YD25600003&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=9372178&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB38" id="BIB38"/>
                     <strong>Rounsley, S.D., Ditta, G.S., and Yanofsky, M.F.</strong> (1995). Diverse roles for MADS box genes in Arabidopsis development. Plant Cell <strong>7,</strong> 1259–1269.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=7/8/1259">[Abstract]</a></p>
                  <p><a name="BIB39" id="BIB39"/>
                     <strong>Sakamoto, T., Kobayashi, M., Itoh, H., Tagiri, A., Kayano, T., Tanaka, H., Iwahori, S., and Matsuoka, M.</strong> (2001). Expression of a gibberellin 2-oxidase gene around the shoot apex is related to phase transition in rice. Plant Physiol.
                     <strong>125,</strong> 1508–1516.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=125/3/1508">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB40" id="BIB40"/>
                     <strong>SAS.</strong> (1999). Statistical Analysis Systems, Version 8. (Cary, NC: SAS Institute).
                     
                  </p>
                  <p><a name="BIB41" id="BIB41"/>
                     <strong>Schomburg, F.M., Bizzell, C.M., Lee, D.J., Zeevaart, J.A.D., and Amasino, R.M.</strong> (2003). Overexpression of a novel class of gibberellin 2-oxidases decreases gibberellin levels and creates dwarf plants.
                     Plant Cell <strong>15,</strong> 151–163.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=15/1/151">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB42" id="BIB42"/>
                     <strong>Schwartz, B.W., Yeung, E.C., and Meinke, D.W.</strong> (1994). Disruption of morphogenesis and transformation of the suspensor in abnormal suspensor mutants of Arabidopsis. Development
                     <strong>120,</strong> 3235–3245.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=120/11/3235">[Abstract]</a></p>
                  <p><a name="BIB43" id="BIB43"/>
                     <strong>Scott, D.B., Jin, W., Ledford, H.K., Jung, H.-S., and Honma, M.A.</strong> (1999). <em>EAF1</em> regulates vegetative-phase change and flowering time in Arabidopsis. Plant Physiol. <strong>120,</strong> 675–684.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=120/3/675">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB44" id="BIB44"/>
                     <strong>Shannon, M.F., and Rao, S.</strong> (2002). Of chips and ChIPs. Science <strong>296,</strong> 666–669.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=296/5568/666">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB45" id="BIB45"/>
                     <strong>Snustad, D.P., Haas, N.A., Kopczak, S.D., and Silflow, C.D.</strong> (1992). The small genome of Arabidopsis contains at least nine expressed beta-tubulin genes. Plant Cell <strong>4,</strong> 549–556.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=4/5/549">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB46" id="BIB46"/>
                     <strong>Swain, S.M., Reid, J.B., and Kamiya, Y.</strong> (1997). Gibberellins are required for embryo growth and seed development in pea. Plant J. <strong>12,</strong> 1329–1338.<a href="/cgi/external_ref?access_num=10.1046%2Fj.1365-313x.1997.12061329.x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000071560900010&amp;link_type=ISI">[Web of Science]</a></p>
                  <p><a name="BIB47" id="BIB47"/>
                     <strong>Talon, M., Koornneef, M., and Zeevaart, J.</strong> (1990). Endogenous gibberellins in Arabidopsis thaliana and possible steps blocked in the biosynthetic pathways of the semidwarf
                     <em>ga4</em> and <em>ga5</em> mutants. Proc. Natl. Acad. Sci. USA <strong>87,</strong> 7983–7987.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=87/20/7983">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB48" id="BIB48"/>
                     <strong>Tang, W., and Perry, S.E.</strong> (2003). Binding site selection for the plant MADS-domain protein AGL15: An <em>in vitro</em> and <em>in vivo</em> study. J. Biol. Chem. <strong>278,</strong> 28154–28159.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=jbc&amp;resid=278/30/28154">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB49" id="BIB49"/>
                     <strong>Theissen, G.</strong> (2001). Development of floral organ identity: Stories from the MADS house. Curr. Opin. Plant Biol. <strong>4,</strong> 75–85.<a href="/cgi/external_ref?access_num=10.1016%2FS1369-5266%2800%2900139-4&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000167318900012&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=11163172&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB50" id="BIB50"/>
                     <strong>Thomas, S.G., Phillips, A.L., and Hedden, P.</strong> (1999). Molecular cloning and functional expression of gibberellin 2-oxidases, multifunctional enzymes involved in gibberellin
                     deactivation. Proc. Natl. Acad. Sci. USA <strong>96,</strong> 4698–4703.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=96/8/4698">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB51" id="BIB51"/>
                     <strong>Tokuji, Y., and Kuriyama, K.</strong> (2003). Involvement of gibberellin and cytokinin in the formation of embryogenic cell clumps in carrot (<em>Daucus carota</em>). J. Plant Physiol. <strong>160,</strong> 133–141.<a href="/cgi/external_ref?access_num=10.1078%2F0176-1617-00892&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000181286800005&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=12685029&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB52" id="BIB52"/>
                     <strong>Volmaro, C., Pontin, M., Luna, V., Baraldi, R., and Bottini, R.</strong> (1998). Blue light control of hypocotyl elongation in etiolated seedlings of <em>Lactuca sativa</em> (L.) cv. Grand Rapids related to exogenous growth regulators and endogenous IAA, GA3, and abscisic acid. Plant Growth Regul.
                     <strong>26,</strong> 165–173.<a href="/cgi/external_ref?access_num=10.1023%2FA%3A1006185416997&amp;link_type=DOI">[CrossRef]</a></p>
                  <p><a name="BIB53" id="BIB53"/>
                     <strong>Wang, H., Tang, W., Zhu, C., and Perry, S.E.</strong> (2002). A chromatin immunoprecipitation (ChIP) approach to isolate genes regulated by AGL15, a MADS-domain protein that preferentially
                     accumulates in embryos. Plant J. <strong>32,</strong> 831–843.<a href="/cgi/external_ref?access_num=10.1046%2Fj.1365-313X.2002.01455.x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000179643100017&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=12472697&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB54" id="BIB54"/>
                     <strong>Weinmann, A.S., Bartley, S.M., Zhang, T., Zhang, M.Q., and Farnham, P.J.</strong> (2001). Use of chromatin immunoprecipitation to clone novel E2F target promoters. Mol. Cell. Biol. <strong>21,</strong> 6820–6832.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=mcb&amp;resid=21/20/6820">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB55" id="BIB55"/>
                     <strong>Wilkins, T.A., and Smart, L.B.</strong> (1996). Isolation of RNA from plant tissue. In A Laboratory Guide to RNA: Isolation, Analysis, and Synthesis, P.A. Krieg,
                     ed (New York: Wiley-Liss), pp. 21–41.
                     
                  </p>
                  <p><a name="BIB56" id="BIB56"/>
                     <strong>Wyrick, J.J., and Young, R.A.</strong> (2002). Deciphering gene expression regulatory networks. Curr. Opin. Genet. Dev. <strong>12,</strong> 130–136.<a href="/cgi/external_ref?access_num=10.1016%2FS0959-437X%2802%2900277-0&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000174356800002&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=11893484&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB57" id="BIB57"/>
                     <strong>Yamaguchi, S., and Kamiya, Y.</strong> (2002). Gibberellins and light-stimulated seed germination. J. Plant Growth Regul. <strong>20,</strong> 369–376.<a href="/cgi/external_ref?access_num=10.1007%2Fs003440010035&amp;link_type=DOI">[CrossRef]</a></p>
               </div>
      <span class="highwire-journal-article-marker-end"/>
    </div>
  </body>
</html>
