<?xml version="1.0"?>
<html>
  <body>
    <div class="article fulltext-view" itemprop="articleBody"><span class="highwire-journal-article-marker-start"/><h1 id="article-title-1" itemprop="headline">The AIP2 E3 ligase acts as a novel negative regulator of ABA signaling by promoting ABI3 degradation</h1>
               <div class="contributors">
                  <ol class="contributor-list" id="contrib-group-1"><li id="contrib-1"><span class="name"><a class="name-search" href="/search?author1=Xiuren+Zhang&amp;sortspec=date&amp;submit=Submit">Xiuren Zhang</a></span><a class="xref-" href="#fn-3" id="xref-fn-3-1">1</a>, 
                     </li>
                     <li id="contrib-2"><span class="name"><a class="name-search" href="/search?author1=Virginia+Garreton&amp;sortspec=date&amp;submit=Submit">Virginia Garreton</a></span><a class="xref-" href="#fn-3" id="xref-fn-3-2">1</a><span class="xref-sep">,</span><a class="xref-" href="#fn-4" id="xref-fn-4-1">2</a>,  and 
                     </li>
                     <li id="contrib-3"><span class="name"><a class="name-search" href="/search?author1=Nam-Hai+Chua&amp;sortspec=date&amp;submit=Submit">Nam-Hai Chua</a></span><a class="xref-" href="#fn-5" id="xref-fn-5-1">3</a></li>
                  </ol><ol class="affiliation-list"><li class="aff"><a id="aff-1" name="aff-1"/><address>Laboratory of Plant Molecular Biology, Rockefeller University, New York, New York 10021, USA</address>
                     </li>
                  </ol></div>
               <div class="section abstract" id="abstract-1" itemprop="description">
                  <div class="section-nav">
                     <div class="nav-placeholder"> </div><a href="#sec-1" title="Next Section" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>Abstract</h2>
                  <p id="p-1">The phytohormone abscisic acid (ABA) mediates many complex aspects of plant development including seed maturation, dormancy,
                     and germination as well as root growth. The B3-domain transcription factor abscisic acid-insensitive 3 (ABI3) is a central
                     regulator in ABA signaling, but little is known of how this factor is regulated. Here, we show that ABI3 is an unstable protein
                     and that an ABI3-interacting protein (AIP2), which contains a RING motif, can polyubiquitinate ABI3 in vitro. The AIP2 E3
                     ligase activity is abolished by mutations (C230S; C231S) in the RING motif and the AIP2 (C/S) mutant functions in a dominant-negative
                     manner. AIP2 has a stronger binding affinity for the B2 + B3 domain of ABI3 than the A1 + B1 domain, but only ubiquitinates
                     the latter. In double-transgenic plants, induced AIP2 expression leads to a decrease in ABI3 protein levels. In contrast,
                     ABI3 levels are elevated upon induced expression of the AIP2 RING mutant, which interferes with the endogenous AIP2 E3 activity.
                     An <em>aip2-1</em>-null mutant shows higher ABI3 protein levels compared with wild type after seed stratification, and is hypersensitive to
                     ABA, mimicking the ABI3-overexpression phenotype, whereas AIP2-overexpression plants contain lower levels of ABI3 protein
                     than wild type and are more resistant to ABA, phenocopying <em>abi3</em>. Our results indicate that AIP2 negatively regulates ABA signaling by targeting ABI3 for post-translational destruction.
                  </p>
                  <h3 class="kwd-header">Keywords</h3>
                  <ul class="kwd-group"><li class="kwd"><span><a class="kwd-search" href="/search?fulltext=Abscisic+acid-insensitive+3+(ABI3)&amp;sortspec=date&amp;submit=Submit&amp;andorexactfulltext=phrase">Abscisic acid-insensitive 3 (ABI3)</a></span></li>
                     <li class="kwd"><span><a class="kwd-search" href="/search?fulltext=ABI3-interacting+protein2+(AIP2)&amp;sortspec=date&amp;submit=Submit&amp;andorexactfulltext=phrase">ABI3-interacting protein2 (AIP2)</a></span></li>
                     <li class="kwd"><span><a class="kwd-search" href="/search?fulltext=ABA+signaling&amp;sortspec=date&amp;submit=Submit&amp;andorexactfulltext=phrase">ABA signaling</a></span></li>
                     <li class="kwd"><span><a class="kwd-search" href="/search?fulltext=polyubiquitination&amp;sortspec=date&amp;submit=Submit&amp;andorexactfulltext=phrase">polyubiquitination</a></span></li>
                     <li class="kwd"><span><a class="kwd-search" href="/search?fulltext=proteasomal+degradation&amp;sortspec=date&amp;submit=Submit&amp;andorexactfulltext=phrase">proteasomal degradation</a></span></li>
                  </ul></div>
               <div class="section" id="sec-1">
                  <div class="section-nav"><a href="#abstract-1" title="Abstract" class="prev-section-link"><span>Previous Section</span></a><a href="#sec-2" title="Results" class="next-section-link"><span>Next Section</span></a></div>
                  <p id="p-2">The phytohormone abscisic acid (ABA) regulates many complex processes of plant growth and development, including seed maturation,
                     the onset and maintenance of seed dormancy, and inhibition of phase transitions from embryonic to germinative growth and from
                     vegetative to reproductive growth (<a class="xref-" href="#ref-13" id="xref-ref-13-1">Finkelstein et al. 2002</a>). This hormone also plays a vital role in a plant's adaptation to assorted environmental stresses such as drought, salinity,
                     and cold, mainly by regulating stomatal aperture (<a class="xref-" href="#ref-13" id="xref-ref-13-2">Finkelstein et al. 2002</a>).
                  </p>
                  <p id="p-3">Genetic screens for ABA-response mutants have uncovered several molecular components of ABA signaling. One class of mutants
                     shows an enhanced response to ABA, and these ABA-hypersensitive mutants include <em>era1</em> (<a class="xref-" href="#ref-9" id="xref-ref-9-1">Cutler et al. 1996</a>), <em>ein2</em> (<a class="xref-" href="#ref-2" id="xref-ref-2-1">Beaudoin et al. 2000</a>), <em>fry1</em> (<a class="xref-" href="#ref-32" id="xref-ref-32-1">Lu and Fedoroff 2000</a>), <em>abh1</em> (<a class="xref-" href="#ref-18" id="xref-ref-18-1">Hugouvieux et al. 2001</a>), <em>sad1</em> (<a class="xref-" href="#ref-53" id="xref-ref-53-1">Xiong et al. 2001a</a>), and <em>hyl1</em> (<a class="xref-" href="#ref-54" id="xref-ref-54-1">Xiong et al. 2001b</a>). Genes affected in these mutants are involved in a variety of cellular processes like farnesylation, inositol signaling,
                     and RNA metabolism. ABA-insensitive mutants, which are tolerant/resistant to ABA-mediated growth inhibition, include <em>abi1</em> (<a class="xref-" href="#ref-25" id="xref-ref-25-1">Leung et al. 1994</a>), <em>abi2</em> (<a class="xref-" href="#ref-38" id="xref-ref-38-1">Rodriguez et al. 1998</a>), <em>abi3</em> (<a class="xref-" href="#ref-37" id="xref-ref-37-1">Parcy et al. 1994</a>), <em>abi4</em> (<a class="xref-" href="#ref-12" id="xref-ref-12-1">Finkelstein et al. 1998</a>), <em>abi5</em> (<a class="xref-" href="#ref-11" id="xref-ref-11-1">Finkelstein and Lynch 2000</a>; <a class="xref-" href="#ref-28" id="xref-ref-28-1">Lopez-Molina and Chua 2000</a>), and <em>abi8</em> (<a class="xref-" href="#ref-7" id="xref-ref-7-1">Brocard-Gifford et al. 2004</a>). <em>ABI1</em> and <em>ABI2</em>, which encode protein phosphatases, have been proposed to negatively regulate ABA signaling during seed dormancy and germination.
                     <em>ABI4</em> and <em>ABI5</em> encode two different transcription factors of the AP2 domain and bZIP domain families, respectively. Besides modulating ABA
                     seed sensitivity, these two factors also function in sugar and salt responses in early seedling growth (<a class="xref-" href="#ref-29" id="xref-ref-29-1">Lopez-Molina et al. 2001</a>; <a class="xref-" href="#ref-6" id="xref-ref-6-1">Brocard-Gifford et al. 2003</a>), and in lateral root branching in response to nitrate (<a class="xref-" href="#ref-47" id="xref-ref-47-1">Signora et al. 2001</a>). <em>ABI8</em> encodes a protein with no domains of known function but belongs to a small plant-specific protein family mediating ABA and
                     sugar responses essential for growth (<a class="xref-" href="#ref-7" id="xref-ref-7-2">Brocard-Gifford et al. 2004</a>).
                  </p>
                  <p id="p-4">Abscisic acid-insensitive 3 (ABI3) has been considered as a paradigm for regulation of seed-specific development, as this
                     factor determines ABA sensitivity and plays a central role in the establishment of desiccation tolerance and dormancy during
                     zygotic embryogenesis. <em>ABI3</em> transcript and protein are abundant in maturing and mature seeds, but disappear soon after germination. However, their levels
                     can be up-regulated by ABA or osmotic stress during the time period when post-germination growth arrest occurs (Lopez-Molina
                     et al. <a class="xref-" href="#ref-29" id="xref-ref-29-2">2001</a>, <a class="xref-" href="#ref-30" id="xref-ref-30-1">2002</a>). Recent data showed that <em>ABI3</em> is also involved in plastid development, bud dormancy, and flowering time (Rohde et al. <a class="xref-" href="#ref-40" id="xref-ref-40-1">2000a</a>,<a class="xref-" href="#ref-41" id="xref-ref-41-1">b</a>, <a class="xref-" href="#ref-42" id="xref-ref-42-1">2002</a>). Moreover, ABI3 has also been identified as a repressor of several plant tissues such as apical meristems in seeds and in
                     arrested seedlings, and axillary meristems associated with the rosette and cauline leaves of stem, and lateral root meristems
                     (<a class="xref-" href="#ref-39" id="xref-ref-39-1">Rohde et al. 1999</a>; <a class="xref-" href="#ref-5" id="xref-ref-5-1">Brady et al. 2003</a>). Therefore, it has been postulated that ABI3 might function as a general regulator imprinting the timing of developmental
                     transitions (<a class="xref-" href="#ref-41" id="xref-ref-41-2">Rohde et al. 2000b</a>).
                  </p>
                  <p id="p-5">Comparison of <em>Arabidopsis</em> ABI3 and its homologs of maize (VP1) (<a class="xref-" href="#ref-33" id="xref-ref-33-1">McCarty et al. 1991</a>), poplar (PtABI3) (<a class="xref-" href="#ref-42" id="xref-ref-42-2">Rohde et al. 2002</a>), and other plant species (<a class="xref-" href="#ref-13" id="xref-ref-13-3">Finkelstein et al. 2002</a>) revealed that this factor contains four highly conserved amino acid domains: A1 (an acid region at the N-terminal of the
                     protein) and three basic domains designated B1, B2, and B3 (<a class="xref-" href="#ref-14" id="xref-ref-14-1">Giraudat et al. 1992</a>). The B1 and B2 domains are implicated in nuclear localization and interaction with other proteins (<a class="xref-" href="#ref-14" id="xref-ref-14-2">Giraudat et al. 1992</a>; <a class="xref-" href="#ref-10" id="xref-ref-10-1">Ezcurra et al. 2000</a>; <a class="xref-" href="#ref-35" id="xref-ref-35-1">Nakamura et al. 2001</a>), whereas the C-terminal B3 binds specifically to the RY/Sph DNA elements in vitro (<a class="xref-" href="#ref-50" id="xref-ref-50-1">Suzuki et al. 1997</a>). It is believed that the multiple domains of ABI3 enable it to function either as an activator or a repressor depending
                     on the promoter context. The N-terminal part of the protein is responsible for ABA-dependent coactivation and repression activities
                     (<a class="xref-" href="#ref-17" id="xref-ref-17-1">Hoecker et al. 1995</a>; <a class="xref-" href="#ref-8" id="xref-ref-8-1">Carson et al. 1997</a>), whereas the C-terminal B3 is essential for activation of a subset of genes (<a class="xref-" href="#ref-8" id="xref-ref-8-2">Carson et al. 1997</a>). ABI3 also has the potential to recruit additional DNA-binding proteins to promoters (Li et al. <a class="xref-" href="#ref-26" id="xref-ref-26-1">1999</a>, <a class="xref-" href="#ref-27" id="xref-ref-27-1">2001</a>). Yeast two-hybrid screening using ABI3/VP1 as a bait identified several ABI3-interacting proteins, including a CONSTANS-related
                     factor; AIP2, a homolog of a <em>Drosophila</em> developmental protein called GOLIATH; the RPB5 subunit of RNA polymerases II; a homolog of the human C1 protein involved
                     in cell cycle control (<a class="xref-" href="#ref-21" id="xref-ref-21-1">Jones et al. 2000</a>; <a class="xref-" href="#ref-24" id="xref-ref-24-1">Kurup et al. 2000</a>); ABI5 (<a class="xref-" href="#ref-35" id="xref-ref-35-2">Nakamura et al. 2001</a>); and Rad23, a protein implicated in DNA repair (<a class="xref-" href="#ref-44" id="xref-ref-44-1">Schultz and Quatrano 1997</a>).
                  </p>
                  <div id="F1" class="fig pos-float">
                     <div class="fig-inline"><a href="1532/F1.expansion.html"><img alt="Figure 1." src="1532/F1.small.gif"/></a><div class="callout"><span>View larger version:</span><ul class="callout-links"><li><a href="1532/F1.expansion.html">In this window</a></li>
                              <li><a class="in-nw" href="1532/F1.expansion.html">In a new window</a></li>
                           </ul><ul class="fig-services"><li class="ppt-link"><a href="/powerpoint/19/13/1532/F1">Download as PowerPoint Slide</a></li>
                           </ul></div>
                     </div>
                     <div class="fig-caption"><span class="fig-label"><strong>Figure 1.</strong></span> 
                        <p id="p-7" class="first-child">ABI3 protein is unstable in vivo. (<em>A</em>) ABI3 degradation in vivo can be blocked by MG132. Two-week-old <em>35S-ABI3–6myc</em> seedlings were transferred to MS liquid medium in the presence of different concentrations of MG132 for 4 h before samples
                           were collected for Western blot analysis. (<em>B</em>) ABI3 protein is short-lived. ABI3 protein levels were analyzed at different times after transfer of <em>35S-ABI3–6myc</em> plants to MS liquid medium containing cycloheximide (CHX), in the presence or absence of MG132. The molecular mass of ABI3–6myc
                           is ∼150 kDa. A cross-reaction band (asterisk) is shown as a loading control. Relative intensities of ABI3–6myc normalized
                           with respect to the cross-reacting band are given at the <em>bottom</em>.
                        </p>
                        <div class="sb-div caption-clear"/>
                     </div>
                  </div>
                  <p id="p-8">Whereas the involvement of transcription factors ABI3/VP1 in plant development processes and ABA responses is well established,
                     little is known concerning how these factors themselves are regulated. The <em>ABI3</em> transcript contains a long 5′-untranslated region (UTR), and excision of this region dramatically increased <em>ABI3-GUS</em> reporter expression, suggesting that the 5′-UTR might regulate <em>ABI3</em> gene expression by a post-transcriptional mechanism, for example, through interaction with some unknown RNA-binding proteins
                     (<a class="xref-" href="#ref-36" id="xref-ref-36-1">Ng et al. 2004</a>). Here we demonstrate that <em>ABI3</em> expression can be regulated at the post-translational level. We show that <em>AIP2</em>, which encodes a C3H2C3 type of RING-motif protein (<a class="xref-" href="#ref-24" id="xref-ref-24-2">Kurup et al. 2000</a>; <a class="xref-" href="#ref-49" id="xref-ref-49-1">Stone et al. 2005</a>), is, indeed, an E3 ligase for ABI3 in vivo, and that AIP2 is a novel negative regulator of ABA signaling by targeting ABI3
                     to 26S proteasomes for regulated proteolysis.
                  </p>
               </div>
               <div class="section" id="sec-2">
                  <div class="section-nav"><a href="#sec-1" title="Previous Section" class="prev-section-link"><span>Previous Section</span></a><a href="#sec-10" title="Discussion" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>Results</h2>
                  <h3>The ABI3 protein is unstable</h3>
                  <p id="p-9">To investigate the stability of ABI3 in vivo, we constructed a cDNA encoding ABI3 tagged with 6myc at the C terminus, and
                     generated transgenic plants expressing <em>35S-ABI3–6myc</em>. All positive lines showed a similar phenotype: highly extended vegetative growth, delayed flowering, and low seed yield
                     compared with wild type (Col-0). We treated 2-, or 3-wk-old seedlings with different concentrations of a proteasome inhibitor
                     (MG132) for 4 h, and then measured the ABI3 accumulation levels. <a class="xref-" href="#F1" id="xref-fig-1-1">Figure 1A</a> shows that ABI3 protein was unstable as its levels were enhanced by two- to fivefold by MG132 depending on the concentration.
                     To investigate the half-life of ABI3 protein, we treated seedlings with cycloheximide to block protein synthesis. <a class="xref-" href="#F1" id="xref-fig-1-2">Figure 1B</a> shows that ABI3 had a half-life of ∼1–2 h in the absence of new protein synthesis. Under this condition, degradation of ABI3
                     can be blocked by MG132, suggesting that the protein is destroyed in vivo by 26S proteasomes.
                  </p>
                  <h3>AIP2 is an E3 ligase</h3>
                  <p id="p-10">The instability of ABI3 prompted us to investigate the mechanism of its post-translational regulation. Previous work using
                     yeast two-hybrid assays identified several ABI3-interacting proteins, one of which is AIP2 (<a class="xref-" href="#ref-24" id="xref-ref-24-3">Kurup et al. 2000</a>), a hypothetical protein containing a RING motif. As several proteins containing RING motifs function as E3 ligases in vitro
                     (<a class="xref-" href="#ref-52" id="xref-ref-52-1">Xie et al. 2002</a>; <a class="xref-" href="#ref-45" id="xref-ref-45-1">Seo et al. 2003</a>), we examined whether AIP2 also has E3 activity. In the presence of ubiquitin, ATP, E1, and E2, purified MBP-AIP2–3HA was
                     able to autoubiquitinate (<a class="xref-" href="#F2" id="xref-fig-2-1">Fig. 2A</a>), whereas MBP alone was inactive (data not shown). As a negative control we constructed an AIP2 RING-motif mutant by changing
                     residues Cys230 and Cys231 to Ser residues (C230S; C231S). The AIP2 (C/S) mutant protein was inactive (<a class="xref-" href="#F2" id="xref-fig-2-2">Fig. 2B</a>), indicating that an intact RING motif is essential for its E3 activity.
                  </p>
                  <h3>AIP2 interacts with ABI3 in vivo</h3>
                  <p id="p-11">To investigate the interaction between AIP2 and ABI3, we first examined the subcellular localization of AIP2 and ABI3 by transiently
                     expressing AIP2 tagged with yellow fluorescent protein (YFP) and ABI3 tagged with cyan fluorescent protein (CFP) in <em>Nicotiana benthamiana</em> and in <em>Arabidopsis</em> epidermal cells. Both proteins were localized predominantly in the nucleus, but they were also found in the cytosol (<a class="xref-" href="#F3" id="xref-fig-3-1">Fig. 3A</a>; data not shown). We then performed in vitro pull-down assays using purified MBP-AIP2 protein. Other bait proteins such as
                     MBP and MBP-SINAT5 (<a class="xref-" href="#ref-52" id="xref-ref-52-2">Xie et al. 2002</a>) and a prey protein, 6xHis-ABI5–6myc (<a class="xref-" href="#ref-31" id="xref-ref-31-1">Lopez-Molina et al. 2003</a>), were used as negative controls. <a class="xref-" href="#F3" id="xref-fig-3-2">Figure 3B and C</a> shows that MBP-AIP2 was able to pull down 6xHis-ABI3–6myc but not 6xHis-ABI5–6myc. This interaction was not only direct but
                     also specific as neither MBP nor MBP-SINAT5, another RING-motif E3 ligase, was active in this assay. To define protein domains
                     responsible for AIP2 and ABI3 interaction, we generated two truncated mutants of ABI3 protein, one with the A1 and B1 domains
                     (ABI3 ΔC), the other with the B2 and B3 domains (ABI3 ΔN) (<a class="xref-" href="#F3" id="xref-fig-3-3">Fig. 3B</a>). Pulldown assays showed that AIP2 interacted with both ABI3 truncated mutants, but its affinity for 6xHis-ABI3 ΔN–6myc was
                     higher than that for 6xHis-ABI3 ΔC–6myc (<a class="xref-" href="#F3" id="xref-fig-3-4">Fig. 3B,C</a>). Similar deletion analysis showed that the C-terminal region of AIP2 (amino acids 270–311), which contains the RING motif,
                     interacted with ABI3. These in vitro pull-down results were confirmed by yeast two-hybrid assays (data not shown).
                  </p>
                  <div id="F2" class="fig pos-float">
                     <div class="fig-inline"><a href="1532/F2.expansion.html"><img alt="Figure 2." src="1532/F2.small.gif"/></a><div class="callout"><span>View larger version:</span><ul class="callout-links"><li><a href="1532/F2.expansion.html">In this window</a></li>
                              <li><a class="in-nw" href="1532/F2.expansion.html">In a new window</a></li>
                           </ul><ul class="fig-services"><li class="ppt-link"><a href="/powerpoint/19/13/1532/F2">Download as PowerPoint Slide</a></li>
                           </ul></div>
                     </div>
                     <div class="fig-caption"><span class="fig-label"><strong>Figure 2.</strong></span> 
                        <p id="p-13" class="first-child">AIP2 is a ubiquitin E3 ligase. (<em>A</em>) Epitope-tagged recombinant AIP2 protein was purified from <em>E. coli</em> extracts. MBP-AIP2–3HA was assayed for E3 activity in the presence or absence of rabbit E1, human E2 (UbcH5b), and 6xHis-ubiquitin
                           (Ub). (<em>B</em>) AIP2 E3 activity is dependent on the integrity of its RING motif. MBP-AIP2–3HA and MBP-AIP2(C/S)–3HA were assayed for self-ubiquitination
                           in the presence of rabbit E1, human E2, and 6xHis-Ub. E2 was omitted in some reactions as control. Western blot analyses were
                           performed by using anti-HA antibody. (MBP) Maltose-binding protein. The arrow indicates either MBP-AIP2–3HA or MBP-AIP2(C/S)–3HA.
                        </p>
                        <div class="sb-div caption-clear"/>
                     </div>
                  </div>
                  <p id="p-14">To see whether AIP2 and ABI3 also interact in vivo, we generated double-transgenic lines harboring <em>35S-ABI3–6myc</em> and <em>XVE-AIP2(C/S)–3HA</em> for coimmunoprecipitation experiments. The RING-motif mutant of AIP2 was used to reduce possible degradation of ABI3 in vivo.
                     Upon induced expression of AIP2(C/S)–3HA, antibody to either AIP2–3HA or ABI3–6myc was able to coimmunoprecipitate the other
                     (<a class="xref-" href="#F3" id="xref-fig-3-5">Fig. 3D</a>), indicating their physical association in vivo.
                  </p>
                  <h3>Expression profiles of AIP2 and ABI3</h3>
                  <p id="p-15">We next compared the expression profiles of <em>AIP2</em> and <em>ABI3</em> by Northern blot analysis and by histochemical GUS staining of transgenic plants expressing a fusion gene comprising their
                     endogenous promoters and the β-glucuronidase (<em>GUS</em>) coding sequence. <em>AIP2</em> transcripts were ubiquitously present in all plant tissues, whereas <em>ABI3</em> was exclusively expressed in developing and mature siliques and seeds (<a class="xref-" href="#F4" id="xref-fig-4-1">Fig. 4A–C</a>). Furthermore, the <em>AIP2-GUS</em> fusion gene was expressed in vegetative tissues including cotyledons, the root cap, vascular tissues, the pericycle, and
                     the shoot apical meristem (SAM). <em>AIP2-GUS</em> activity could also be detected in reproductive tissues such as anthers, ovules, siliques, and seeds with the highest expression
                     in freshly stratified seeds (<a class="xref-" href="#F4" id="xref-fig-4-2">Fig. 4D</a>). The spatial pattern of <em>GUS</em> expression from the <em>ABI3</em> promoter was similar to that previously reported (<a class="xref-" href="#ref-37" id="xref-ref-37-2">Parcy et al. 1994</a>; <a class="xref-" href="#ref-36" id="xref-ref-36-2">Ng et al. 2004</a>; data not shown).
                  </p>
                  <p id="p-16">We investigated the effects of ABA on endogenous <em>AIP2</em> and <em>ABI3</em> gene expression. ABA treatment for 4 h or longer increased <em>AIP2</em> transcript levels at least two-fold in 4-wk wild-type (Col-0) plants compared with untreated control plants (<a class="xref-" href="#F4" id="xref-fig-4-3">Fig. 4B</a>). Transfer of ABA-treated plants onto MS media for a day decreased <em>AIP2</em> mRNA levels to the pretreated status (data not shown). Similar treatment with auxin did not modify the <em>AIP2</em> expression (data not shown). After stratification and during germination, <em>AIP2</em> and <em>ABI3</em> expression decreased gradually without ABA treatment. However, within the developmental window when growth arrest occurs
                     (Lopez-Molina et al. <a class="xref-" href="#ref-29" id="xref-ref-29-3">2001</a>, <a class="xref-" href="#ref-30" id="xref-ref-30-2">2002</a>), ABA clearly stabilized <em>ABI3</em> expression level but not that of <em>AIP2</em> (<a class="xref-" href="#F4" id="xref-fig-4-4">Fig. 4C</a>).
                  </p>
                  <h3>ABI3 is a substrate of AIP2 in vitro</h3>
                  <p id="p-17">The interaction of ABI3 with AIP2 in vitro and in vivo provides preliminary evidence that the former may be a substrate of
                     the latter. To study this further, purified ABI3 was used as a substrate in in vitro ubiquitination reactions. <a class="xref-" href="#F5" id="xref-fig-5-1">Figure 5A</a> shows that ABI3 was polyubiquitinated by the AIP2 E3 ligase in a reaction dependent on E1 and E2 activities. The differential
                     AIP2-binding affinity of the two different truncated ABI3 proteins prompted us to investigate their ability to serve as substrates
                     in vitro. Surprisingly, AIP2 specifically modified the ABI3 mutant containing the A1 + B1 domains but not the mutant containing
                     the B2 + B3 domains (<a class="xref-" href="#F5" id="xref-fig-5-2">Fig. 5A</a>, middle and right panels). The ABI3 polyubiquitination was dependent on the integrity of the AIP2 RING motif (<a class="xref-" href="#F5" id="xref-fig-5-3">Fig. 5B</a>, left panel). Moreover, wild-type AIP2 E3 activity was blocked by a fourfold to 10-fold excess of the AIP2(C/S) mutant, whereas
                     a 10-fold excess of MBP alone had no visible effect (<a class="xref-" href="#F5" id="xref-fig-5-4">Fig. 5B</a>, right panel). These results indicate that excess AIP2(C/S) mutant can compete with wild-type AIP2 for binding sites on ABI3
                     substrates, thus acting in a dominant-negative manner. The ubiquitination of ABI3 by AIP2 is specific, because no ABI3 ubiquitination
                     was detected either with SINAT5, an <em>Arabidopsis</em> E3 ligase that modifies NAC1 (<a class="xref-" href="#ref-52" id="xref-ref-52-3">Xie et al. 2002</a>), or with COP1, another E3 ligase that targets LAF1 (<a class="xref-" href="#ref-45" id="xref-ref-45-2">Seo et al. 2003</a>), phyA (<a class="xref-" href="#ref-46" id="xref-ref-46-1">Seo et al. 2004</a>), and HFR1 (<a class="xref-" href="#F5" id="xref-fig-5-5">Fig. 5C</a>; <a class="xref-" href="#ref-19" id="xref-ref-19-1">Jang et al. 2005</a>). Similarly, AIP2 was unable to ubiquitinate HFR1 (<a class="xref-" href="#ref-19" id="xref-ref-19-2">Jang et al. 2005</a>), and ABI5, a transcription factor that acts downstream of ABI3 in ABA signaling (<a class="xref-" href="#F5" id="xref-fig-5-6">Fig. 5D</a>; Lopez-Molina et al. <a class="xref-" href="#ref-29" id="xref-ref-29-4">2001</a>, <a class="xref-" href="#ref-30" id="xref-ref-30-3">2002</a>, <a class="xref-" href="#ref-31" id="xref-ref-31-2">2003</a>).
                  </p>
                  <div id="F3" class="fig pos-float">
                     <div class="fig-inline"><a href="1532/F3.expansion.html"><img alt="Figure 3." src="1532/F3.small.gif"/></a><div class="callout"><span>View larger version:</span><ul class="callout-links"><li><a href="1532/F3.expansion.html">In this window</a></li>
                              <li><a class="in-nw" href="1532/F3.expansion.html">In a new window</a></li>
                           </ul><ul class="fig-services"><li class="ppt-link"><a href="/powerpoint/19/13/1532/F3">Download as PowerPoint Slide</a></li>
                           </ul></div>
                     </div>
                     <div class="fig-caption"><span class="fig-label"><strong>Figure 3.</strong></span> 
                        <p id="p-19" class="first-child">Interaction of AIP2 and ABI3 in vitro and in vivo. (<em>A</em>) Colocalization of AIP2-YFP and ABI3-CFP in the nucleus and the cytosol in <em>N. benthamiana</em> cells. (<em>B</em>) Schematic diagram of full-length and truncated forms of ABI3 (ABI3 ΔC, A1 + B1; ABI3 ΔN, B2 + B3). Numbers refer to the
                           amino acid residues in the wild-type (WT) ABI3 protein. The prey proteins were double tagged with 6xHis and 6myc at N and
                           C termini, respectively, and were purified from <em>E. coli</em> using nitrilotriacetate resin. (<em>C</em>) In vitro pull-down assays of full-length or deletion mutants of ABI3 proteins with three bait proteins (MBP, MBP-SINAT5,
                           and MBP-AIP2). The bait proteins were purified from <em>E. coli</em> using amylose resin. Two micrograms of prey proteins were pulled down with the indicated bait proteins (2 μg each) using
                           amylose resin, and proteins were detected by anti-myc antibody. Another prey protein, ABI5, was used as a negative control.
                           (I) Input amount of prey proteins (asterisks). (<em>D</em>) Coimmunoprecipitation of ABI3 and AIP2 proteins in vivo. Two-week-old seedlings of transgenic seedlings carrying <em>35-ABI3–6myc/XVE-AIP2(C/S)–3HA</em> were treated overnight with 50 μM MG132 in the absence (lanes <em>1,4</em>) or presence (lanes <em>2,3,5</em>) of β-estradiol (25 μM). Total protein extracts were immunoprecipitated (IP) with monoclonal antibody to HA or His. (<em>Left</em> panel) Western blots were analyzed with a polyclonal antibody to myc to detect coimmunoprecipitated ABI3–6myc. (<em>Right</em> panel) In a parallel experiment, IP was done using a polyclonal antibody to myc. Western blots were analyzed with a monoclonal
                           antibody to HA to detect coimmunoprecipitated AIP2–3HA protein. (Input) Crude protein extracts. The arrowhead indicates the
                           cross-reaction with the heavy chain of the protein A-conjugated antibody.
                        </p>
                        <div class="sb-div caption-clear"/>
                     </div>
                  </div>
                  <h3>ABI3 levels are down-regulated by inducible expression of AIP2</h3>
                  <p id="p-20">The instability of ABI3 in vivo (<a class="xref-" href="#F1" id="xref-fig-1-3">Fig. 1</a>) raises the question of the identity of its cognate E3 ligase. Because of the observed in vitro and in vivo interaction between
                     ABI3 and AIP2 as well as the ubiquitination of ABI3 by the AIP2 E3 ligase in vitro, we hypothesized that AIP2 targets ABI3
                     for degradation in vivo. To test this hypothesis, we first analyzed ABI3 levels in <em>aip2-1</em>, a null mutant generated by T-DNA insertional mutagenesis (<a class="xref-" href="#F6" id="xref-fig-6-1">Fig. 6A</a>, panel a; <a class="xref-" href="#ref-1" id="xref-ref-1-1">Alonso et al. 2003</a>). AIP2 was detected as a band of ∼37 kDa in wild-type (Col-0) seedlings, but this band was absent from the <em>aip2-1</em> mutant (<a class="xref-" href="#F6" id="xref-fig-6-2">Fig. 6A</a>, panel b). In contrast, higher ABI3 protein levels were found in the <em>aip2-1</em> mutant compared with wild type (Col-0). Both mutant and wild-type seeds were growth-arrested in MS medium containing 3.0
                     μM ABA for 5 d post-stratification (<a class="xref-" href="#F6" id="xref-fig-6-3">Fig. 6A</a>, panel c). On the other hand, ABI3 levels were less abundant in the complemented line <em>35S-AIP2–3HA/aip2-1</em> (#1) compared with wild type (Col-0) (Figs. <a class="xref-" href="#F6" id="xref-fig-6-4">6A</a> [panel d], <a class="xref-" href="#F7" id="xref-fig-7-1">7D</a>) grown under the same conditions.
                  </p>
                  <div id="F4" class="fig pos-float">
                     <div class="fig-inline"><a href="1532/F4.expansion.html"><img alt="Figure 4." src="1532/F4.small.gif"/></a><div class="callout"><span>View larger version:</span><ul class="callout-links"><li><a href="1532/F4.expansion.html">In this window</a></li>
                              <li><a class="in-nw" href="1532/F4.expansion.html">In a new window</a></li>
                           </ul><ul class="fig-services"><li class="ppt-link"><a href="/powerpoint/19/13/1532/F4">Download as PowerPoint Slide</a></li>
                           </ul></div>
                     </div>
                     <div class="fig-caption"><span class="fig-label"><strong>Figure 4.</strong></span> 
                        <p id="p-22" class="first-child">Expression patterns of <em>AIP2</em> and <em>ABI3</em>.(<em>A</em>) Total RNA from different tissues of wild-type (Col-0) plants were analyzed for <em>AIP2</em> and <em>ABI3</em> transcripts levels. (<em>B</em>) Four-week-old wild-type (WT) seedlings were transferred to MS medium containing 50 μM ABA or an equal amount of methanol
                           only for the indicated time period (in hours) before RNA extraction. (<em>C</em>) Wild-type seeds in MS medium with or without ABA (5 μM) were stratified at 4°C for 3 d and then transferred to 24 h of continuous
                           light for the indicated days before RNA extraction. Ten micrograms of RNA per lane was used in <em>A</em>, and 5 μg of RNA per lane was used in <em>B</em> and <em>C</em>. The <em>top</em> and <em>middle</em> panels in <em>A</em>–<em>C</em> were probed for <em>AIP2</em> and <em>ABI3</em> transcripts, respectively. 18S rRNA levels were used as loading controls. (<em>D</em>) Histochemical localization of GUS activity in transgenic plants carrying <em>AIP2-GUS-GFP</em>. GUS expression was detected in germinating seeds at 0 d (panel <em>a</em>), 2 d (panel <em>b</em>), and 4 d (panel <em>c</em>) post-stratification, embryo (panel <em>b</em>), root caps (panels <em>d,g</em>), root vascular tissues (panels <em>e,h</em>), pericycle (panel <em>j</em>), cotyledons (panels <em>c,f,i</em>), shoot apical meristems (panel <em>k</em>), anthers (panel <em>l</em>), pollen sac (panel <em>m</em>), and siliques (panels <em>n,o</em>). Bars: Panels <em>a</em>–<em>c,f,i</em>–<em>k</em>, 1 mm; panels <em>d,e,g,h</em>, 2 mm; panels <em>m</em>–<em>o</em>, 0.5 mm; panel <em>l</em>, 5 mm.
                        </p>
                        <div class="sb-div caption-clear"/>
                     </div>
                  </div>
                  <p id="p-23">We next generated double-transgenic plants harboring the <em>35S-ABI3–6myc</em> and the β-estradiol-inducible <em>XVE-AIP2–3HA</em> transgenes. More than 30 independent double-transgenic lines were obtained that displayed similar phenotypes to single-transgenic
                     lines expressing only <em>35S-ABI3–6myc</em>. <a class="xref-" href="#F6" id="xref-fig-6-5">Figure 6B</a> shows that induced expression of AIP2–3HA resulted in a decrease in ABI3–6myc levels in all lines tested. Under these conditions,
                     expression of <em>ABI3–6myc</em> transcripts was comparable between induced and uninduced samples in all lines. In contrast, increased expression of the dominant-negative
                     <em>AIP2(C/S)–3HA</em> transgene enhanced ABI3–6myc accumulation by two- to fourfold in double-transgenic lines carrying <em>35S-ABI3–6myc</em> and <em>XVE-AIP2(C/S)–3HA</em> (<a class="xref-" href="#F6" id="xref-fig-6-6">Fig. 6D</a>).
                  </p>
                  <div id="F5" class="fig pos-float">
                     <div class="fig-inline"><a href="1532/F5.expansion.html"><img alt="Figure 5." src="1532/F5.small.gif"/></a><div class="callout"><span>View larger version:</span><ul class="callout-links"><li><a href="1532/F5.expansion.html">In this window</a></li>
                              <li><a class="in-nw" href="1532/F5.expansion.html">In a new window</a></li>
                           </ul><ul class="fig-services"><li class="ppt-link"><a href="/powerpoint/19/13/1532/F5">Download as PowerPoint Slide</a></li>
                           </ul></div>
                     </div>
                     <div class="fig-caption"><span class="fig-label"><strong>Figure 5.</strong></span> 
                        <p id="p-25" class="first-child">ABI3 is a substrate of AIP2 E3 ligase. (<em>A</em>) MBP-AIP2–3HA E3 activity was assayed in the presence or absence of rabbit E1, UbcH5b, 6xHis-ubiquitin, and 6xHis-ABI3–6myc
                           (<em>left</em> panel), 6xHis-ABI3(ΔC)–6myc (<em>middle</em> panel), and 6xHis-ABI3(ΔN)–6myc (<em>right</em> panel). (<em>B</em>) MBP-AIP2 (C/S)–3HA mutant protein has no E3 activity for 6xHis-ABI3–6myc (<em>left</em> panel) and blocks MBP-AIP2–3HA E3 activity by competing for substrates (<em>right</em> panel). Numbers indicate the relative amounts of proteins present in the reaction, where 1 represents 200 ng of MBP, MBP-AIP2–3HA,
                           or MBP-AIP2(C/S)–3HA. (<em>C</em>) Ubiquitination of ABI3 by AIP2 is specific. Reaction mixtures contained 400 ng of MBP-SINAT5 or 400 ng of MBP-COP1. (<em>D</em>) MBP-AIP2 has no E3 activity for 6xHis-HFR1–3HA (<a class="xref-" href="#ref-19" id="xref-ref-19-3">Jang et al. 2005</a>) and 6xHis-ABI5–6myc (<a class="xref-" href="#ref-31" id="xref-ref-31-3">Lopez-Molina et al. 2003</a>). Western blots were analyzed using polyclonal anti-myc antibody (for full-length and truncated forms of ABI3 and for ABI5)
                           and anti-HA antibody for HFR1. (MBP) Maltose-binding protein. The arrows indicate positions of nonubiquitinated substrates.
                        </p>
                        <div class="sb-div caption-clear"/>
                     </div>
                  </div>
                  <div id="F6" class="fig pos-float">
                     <div class="fig-inline"><a href="1532/F6.expansion.html"><img alt="Figure 6." src="1532/F6.small.gif"/></a><div class="callout"><span>View larger version:</span><ul class="callout-links"><li><a href="1532/F6.expansion.html">In this window</a></li>
                              <li><a class="in-nw" href="1532/F6.expansion.html">In a new window</a></li>
                           </ul><ul class="fig-services"><li class="ppt-link"><a href="/powerpoint/19/13/1532/F6">Download as PowerPoint Slide</a></li>
                           </ul></div>
                     </div>
                     <div class="fig-caption"><span class="fig-label"><strong>Figure 6.</strong></span> 
                        <p id="p-27" class="first-child">AIP2 promotes ABI3 protein degradation. (<em>A</em>) The different levels of ABI3 protein levels in wild-type (Col-0), <em>aip2-1</em> mutant, and <em>AIP2</em> overexpression lines. (Panel <em>a</em>) T-DNA insertion line (<em>aip2-1</em>) of the <em>AIP2</em> (At5g20910) gene. Exons are shown as gray boxes and introns as black lines. The predicted translation start (ATG) and stop
                           (TAA) codons are indicated. The 5′- and 3′-untranslated regions are represented by gray lines. The T-DNA insert site in the
                           <em>aip2-1</em> allele is shown in the <em>upper</em> part. (Panel <em>b</em>) Levels of AIP2 protein in 2-wk seedlings of wild type (Col-0) and <em>aip2-1</em>. (Panels <em>c,d</em>) ABI3 protein levels in wild-type (Col-0), <em>aip2-1</em> mutant, and <em>35S-AIP2–3HA/aip2-1</em> (line #1) seeds. Recombinant 6xHis-AIP2 and 6xHis-ABI3(ΔC)–6myc proteins purified from <em>E. coli</em> were used as positive controls. Tubulin levels were used as loading controls. Seeds were germinated on MS medium supplemented
                           with 3.0 μM ABA for 5 d before Western blot analysis. Endogenous AIP2 and ABI3 were detected with specific polyclonal antibodies
                           generated against the full-length recombinant proteins. The exposure used for panel <em>d</em> was longer than that used for panel <em>c</em>. (<em>B</em>) ABI3 protein levels are reduced after induction of <em>AIP2</em>. Transgenic plants containing <em>35S-ABI3–6myc</em> and <em>XVE-AIP2–3HA</em> (lines #1, #2, and #3) were treated with and without inducer (25 μM β-estradiol) for 16 h. (<em>C</em>) Induction of AIP2 expression suppresses ABI3 overexpression phenotype. Plants from the same double-transgenic line (line
                           #1) were grown on MS medium for 15 d and then transferred to a fresh medium without (#1 and #2) or with (#3 and #4) β-estradiol
                           (10 μM) for another 15 d. ABI3–6myc and AIP2–3HA levels were determined by Western blots as in <em>B</em>. Bar, 15 mm. (<em>D</em>) ABI3 protein levels are increased by induced expression of AIP2 (C/S) mutant protein. ABI3 and AIP2 protein levels were
                           analyzed by Western blots in three independent transgenic lines (#1, #2, and #3) treated with or without inducer (25 μM β-estradiol).
                           In all assays except <em>A</em>, ABI3 and AIP2 expression levels were detected by anti-myc and anti-HA antibodies, respectively. A cross-reacting band (asterisk)
                           was used to normalize loading. (<em>E</em>) ABA modulates ABI3 stability at the post-translational level by regulating <em>AIP2</em> expression. Two-week-old seedlings were treated with ABA (50 μM) or an equal amount of methanol before sampling at the indicated
                           times for RNA and Western blot analysis. The cross-reaction bands (arrow and asterisk) in the Western blot were used as loading
                           controls for endogenous AIP2 and ABI3–6myc, respectively. 18S rRNA was used as a loading control in Northern blots.
                        </p>
                        <div class="sb-div caption-clear"/>
                     </div>
                  </div>
                  <p id="p-28">To further confirm the effects of AIP2, we investigated whether AIP2–3HA induction can rescue ABI3 overexpression phenotypes
                     in the double-transgenic lines. ABI3 overexpression caused vigorous and extremely extended vegetative growth compared with
                     wild type (Col-0). Induction of AIP2–3HA expression reduced ABI3 levels and promoted earlier flowering in plants (#3 and #4)
                     compared with uninduced controls (#1 and #2) (<a class="xref-" href="#F6" id="xref-fig-6-7">Fig. 6C</a>). Under the same conditions, wild-type and <em>35S-ABI3–6myc</em> single-transgenic lines were not affected by the inducer in terms of flowering time (Supplementary Fig. S1). Taken together,
                     our results provide evidence that AIP2 is an E3 ligase for ABI3 in vivo.
                  </p>
                  <p id="p-29">Because ABA up-regulates endogenous <em>AIP2</em> transcripts and protein by two- to sixfold in wild-type (Col-0) (<a class="xref-" href="#F4" id="xref-fig-4-5">Fig. 4B</a>) and in <em>35S-ABI3–6myc</em> seedlings (<a class="xref-" href="#F6" id="xref-fig-6-8">Fig. 6E</a>), we determined whether ABI3 protein stability is decreased in <em>35S-ABI3–6myc</em> transgenic seedlings in the presence of ABA (<a class="xref-" href="#F6" id="xref-fig-6-9">Fig. 6E</a>). Consistent with our expectation, ABI3–6myc levels were down-regulated four- to eightfold upon ABA induction, as shown by
                     immunoblot analysis using anti-myc antibodies, whereas the transcript levels of <em>ABI3–6myc</em> were unaffected by ABA. Thus, our results confirm that ABA could modulate ABI3 stability at the post-translational level,
                     mainly through regulating <em>AIP2</em> expression.
                  </p>
                  <h3>AIP2 is a negative regulator of ABA signaling</h3>
                  <p id="p-30">Our identification of AIP2 as an E3 ligase for ABI3 prompted us to investigate further how AIP2 regulates ABI3 functions in
                     vivo in <em>ABA</em> signaling. To this end, we created transgenic lines expressing <em>35S-AIP2–3HA,</em> and analyzed their responses to ABA compared with those of the <em>aip2-1</em> mutant and <em>35S-ABI3–6myc</em> transgenic plants. No AIP2 transcript and protein were detectable in <em>aip2-1</em> when analyzed by Northern and Western blots (Figs. <a class="xref-" href="#F6" id="xref-fig-6-10">6A</a>, <a class="xref-" href="#F7" id="xref-fig-7-2">7D</a>). In contrast, <em>AIP2</em> transcript and AIP2 protein levels in <em>35S-AIP2–3HA</em> lines (#8 and #9) were at least 10-fold higher compared with wild-type plants (<a class="xref-" href="#F7" id="xref-fig-7-3">Fig. 7C</a>; data not shown). In germination and root growth inhibition assays, <em>aip2-1</em> mutant and <em>35S-ABI3–6myc</em> plants were hypersensitive to ABA compared with wild type, whereas <em>35S-AIP2–3HA</em> transgenic plants, like <em>abi3-1</em> mutants, were more resistant (<a class="xref-" href="#F7" id="xref-fig-7-4">Fig. 7A,B</a>; data not shown). The ABA-hypersensitive phenotype of <em>aip2-1</em> could be complemented with either <em>35S-AIP2–3HA</em> or the native <em>AIP2</em> gene, but neither by <em>35S-AIP2(C/S)–3HA</em> nor by the empty vector pBA002–3HA (<a class="xref-" href="#F7" id="xref-fig-7-5">Fig. 7D–F</a>; data not shown). Moreover, <em>35S-AIP2(C/S)–3HA/aip2-1</em> seeds showed a greater hypersensitivity to ABA. From these results we conclude that AIP2 is a negative regulator of ABA responses,
                     whose function is to keep ABI3 levels in check (<a class="xref-" href="#F7" id="xref-fig-7-6">Fig. 7G</a>).
                  </p>
                  <div id="F7" class="fig pos-float">
                     <div class="fig-inline"><a href="1532/F7.expansion.html"><img alt="Figure 7." src="1532/F7.small.gif"/></a><div class="callout"><span>View larger version:</span><ul class="callout-links"><li><a href="1532/F7.expansion.html">In this window</a></li>
                              <li><a class="in-nw" href="1532/F7.expansion.html">In a new window</a></li>
                           </ul><ul class="fig-services"><li class="ppt-link"><a href="/powerpoint/19/13/1532/F7">Download as PowerPoint Slide</a></li>
                           </ul></div>
                     </div>
                     <div class="fig-caption"><span class="fig-label"><strong>Figure 7.</strong></span> 
                        <p id="p-32" class="first-child">AIP2 is a negative regulator of ABI3 in ABA responses. (<em>A</em>) Five-day-old seedlings germinated in MS were transferred to media containing different concentrations of ABA from 0 to 50
                           μM, and the length of the primary root was measured 5 d later. The experiment was repeated twice. For each time point, <em>n</em> = 12 seedlings in each independent experiment. (<em>B</em>) The percentage of seeds showing root emergence was scored 6 d post-stratification. Standard error bars represent three independent
                           experiments. Because the germination frequency in MS medium alone is different for each line (lines <em>abi3-1, 35S-AIP2–3HA</em> #8 and #9 have a lower seed viability, ∼80%–95% of wild type), the percent germination without ABA was considered to be 100%
                           and the germination frequency in ABA for these lines was normalized based on this value. The <em>left</em> panels in <em>A</em> and <em>B</em> share the same symbols: Wild-type (Col-0) (square black), <em>aip2-1</em> (red triangle), <em>35S-ABI3–6myc</em> (light-gray diamond), <em>35S-AIP2–3HA</em> line #8 (dark-gray filled circle), and line #9 (dark-gray empty circle). The <em>right</em> panels in <em>A</em> and <em>B</em> show phenotypes of wild type (Ler) (square black) and <em>abi3-1</em> (blue triangle). <em>n</em> = 50–200. (<em>C</em>) AIP2 protein levels in <em>35S-AIP2–3HA</em> (#8 and #9) plants in experiments <em>A</em> and <em>B</em> were analyzed by Western blots using anti-HA antibody. Tubulin levels were used as loading controls. (<em>D</em>) Expression of <em>AIP2</em> and <em>AIP2-HA</em> in <em>aip2-1</em> mutant and transgenic lines. Two or three independent complemented lines were selected for RNA blots. Five micrograms of
                           total RNA was used in each lane. (<em>E</em>) Germination frequency of <em>aip2-1</em> mutants and transgenic lines. The percent of seeds showing root emergence was scored 6 d after stratification. Because seeds
                           from different lines for each construct showed similar phenotypes, only a representative line from each construct was graphed
                           for simplicity. The phenotype of <em>pBA002-HA/aip2-1</em> is the same as that of <em>aip2-1</em> (data not shown). Wild type (Col-0) (black square), <em>aip2-1</em> (red triangle), <em>AIP2/aip2-1</em> (dark-gray filled circle), <em>35S-AIP2–3HA/aip2-1</em> (dash line with dark-gray filled cycle), and <em>35S-AIP2(C/S)–3HA</em> (dashed line with light-gray diamond). Standard error bars represent three replications; <em>n</em> = 100–200. (<em>F</em>) Germinating seeds of the <em>aip2-1</em> mutant and transgenic lines on MS medium containing 2.5 μM ABA were photographed at 6 d post-stratification. (Panel <em>a</em>) Wild type (Col-0). (Panel <em>b</em>) <em>AIP2/aip2-1</em> line #1. (Panel <em>c</em>) <em>35S-AIP2–3HA/aip2-1</em> line #1. (Panel <em>d</em>) <em>aip2-1</em>. (Panel <em>e</em>) <em>35S-AIP2(C/S)–3HA/aip2-1</em> line #1. (Panel <em>f</em>) <em>pBA002–3HA/aip2-</em>1. (Panel <em>g</em>) Wild type (Ler). (Panel <em>h</em>) <em>abi3-1</em>. (<em>G</em>) A working model of the ABA signaling pathway in seed germination. AIP2 promotes ABI3 degradation in ABA signaling. ABI1/ABI2
                           phosphatases (<a class="xref-" href="#ref-25" id="xref-ref-25-2">Leung et al. 1994</a>; <a class="xref-" href="#ref-38" id="xref-ref-38-2">Rodriguez et al. 1998</a>) and ERA1 farnesyl transferase (<a class="xref-" href="#ref-5" id="xref-ref-5-2">Brady et al. 2003</a>) are negative regulators of ABA signaling and function at or upstream of the ABI3 transcription factor. ABI5, which is negatively
                           regulated by AFP, acts downstream of ABI3 (<a class="xref-" href="#ref-11" id="xref-ref-11-2">Finkelstein and Lynch 2000</a>; Lopez-Molina et al. <a class="xref-" href="#ref-30" id="xref-ref-30-4">2002</a>, <a class="xref-" href="#ref-31" id="xref-ref-31-4">2003</a>). ABI4 is another transcriptional factor that acts positively downstream in ABA signaling pathways (<a class="xref-" href="#ref-13" id="xref-ref-13-4">Finkelstein et al. 2002</a>).
                        </p>
                        <div class="sb-div caption-clear"/>
                     </div>
                  </div>
               </div>
               <div class="section" id="sec-10">
                  <div class="section-nav"><a href="#sec-2" title="Results" class="prev-section-link"><span>Previous Section</span></a><a href="#sec-15" title="Materials and methods" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>Discussion</h2>
                  <p id="p-33">Previous studies have addressed the role of the ubiquitin/26S proteasome pathway in the perception and transmission of various
                     environmental and hormonal signals (<a class="xref-" href="#ref-16" id="xref-ref-16-1">Hellmann and Estelle 2002</a>; <a class="xref-" href="#ref-15" id="xref-ref-15-1">Hare et al. 2003</a>). Two recent and independent lines of evidence implicate ubiquitin-mediated proteolysis in ABA responses. ABI5 has been identified
                     as a transcriptional activator downstream of ABI3 in ABA response pathways (Lopez-Molina et al. <a class="xref-" href="#ref-30" id="xref-ref-30-5">2002</a>, <a class="xref-" href="#ref-31" id="xref-ref-31-5">2003</a>). This factor is essential for certain germination responses to ABA, and it is rapidly destroyed by a proteasome-dependent
                     pathway that is inhibited by exogenous ABA. Furthermore, a novel protein, ABI FIVE INTERACTION PROTEIN (AFP), was isolated
                     as an ABI5-interacting partner that represses ABA signaling by promoting ABI5 turnover (<a class="xref-" href="#ref-15" id="xref-ref-15-2">Hare et al. 2003</a>; <a class="xref-" href="#ref-31" id="xref-ref-31-6">Lopez-Molina et al. 2003</a>). On the other hand, ABI5 degradation seems to be regulated at the level of proteasome activity as well. REGULATORY PARTICLE
                     NONATPASE 10 (RPN10) is a subunit of the 19S regulatory complex of 26S proteasomes (<a class="xref-" href="#ref-48" id="xref-ref-48-1">Smalle et al. 2003</a>). An <em>Arabidopsis</em> T-DNA insertion mutant, <em>rpn10-1</em>, exhibits pleiotropic phenotypes, most of which appear to be caused by ABA hypersensitivity. Mutant plants of <em>rpn10-1</em> accumulate ABI5, providing evidence that RPN10 is essential for its degradation by proteasomes (<a class="xref-" href="#ref-15" id="xref-ref-15-3">Hare et al. 2003</a>; <a class="xref-" href="#ref-48" id="xref-ref-48-2">Smalle et al. 2003</a>).
                  </p>
                  <h3>AIP2 is an E3 ligase for ABI3</h3>
                  <p id="p-34">The involvement of transcription factors <em>ABI3/VP1</em> in the activation of seed development and repression of germination is well established. However, there is little insight
                     as to how these factors themselves are regulated. Here we demonstrated that AIP2, an E3 ligase, is a novel negative regulator
                     of ABA signaling and that it executes its function by polyubiquitinating ABI3 and presumably targeting it to 26S proteasomes
                     for degradation. Several lines of biochemical evidence support this conclusion: (1) AIP2 physically interacts with ABI3 in
                     vitro (<a class="xref-" href="#F3" id="xref-fig-3-6">Fig. 3B,C</a>) and in yeast two-hybrid assays (<a class="xref-" href="#ref-24" id="xref-ref-24-4">Kurup et al. 2000</a>; data not shown). Moreover, direct evidence of ABI3–AIP2 interaction in vivo was obtained by coimmunoprecipitation (<a class="xref-" href="#F3" id="xref-fig-3-7">Fig. 3D</a>). (2) AIP2 is an E3 ligase and can polyubiquitinate ABI3 in vitro (Figs. <a class="xref-" href="#F2" id="xref-fig-2-3">2</a>, <a class="xref-" href="#F5" id="xref-fig-5-7">5</a>). The interaction of ABI3 and AIP2 in vitro and in vivo and the ubiquitination of ABI3 by AIP2 in vitro suggest that ABI3
                     is modified by AIP2 in vivo.
                  </p>
                  <p id="p-35">We further established the in vivo relationship between the two proteins by analysis of ABI3 levels in double-transgenic plants
                     expressing <em>35S-ABI3–6myc</em> and <em>XVE-AIP2–3HA</em>. We showed that induced expression of AIP2 reduced ABI3 abundance in several independent transgenic lines. In contrast, in
                     double-transgenic plants harboring <em>35S-ABI3–6myc</em> and <em>XVE-AIP2(C/S)–3HA</em>, enhanced expression of the AIP2 (C/S) mutant protein, which acts in a dominant-negative way, resulted in increased accumulation
                     of ABI3 protein. These results provide evidence that AIP2 serves as an E3 ligase to ubiquitinate ABI3 in vivo and targets
                     the latter for destruction by 26S proteasomes.
                  </p>
                  <h3>Multifaceted regulation of ABI3 expression by ABA</h3>
                  <p id="p-36">Endogenous ABI3 transcripts and proteins are abundant in maturing and mature seeds, but disappear soon after germination.
                     However, their levels are up-regulated by ABA during the time period when post-germination growth arrest occurs (Lopez-Molina
                     et al. <a class="xref-" href="#ref-29" id="xref-ref-29-5">2001</a>, <a class="xref-" href="#ref-30" id="xref-ref-30-6">2002</a>). Under this condition, ABA increases accumulation of endogenous ABI3 protein primarily by an ABA-induced increase in <em>ABI3</em> transcript abundance (<a class="xref-" href="#F4" id="xref-fig-4-6">Fig. 4C</a>; <a class="xref-" href="#ref-30" id="xref-ref-30-7">Lopez-Molina et al. 2002</a>). In our experiments, we showed that the effect of ABA on ABI3 protein accumulation also occurs at a post-translational level,
                     and that this modulation is related to the regulation of the endogenous AIP2 expression. During vegetative growth stages,
                     ABA induced AIP2 expression, apparently owing to enhanced <em>AIP2</em> transcript accumulation. The increased AIP2 protein levels resulted in a down-regulation of ABI3 protein levels in ABA-treated
                     <em>35S-ABI3–6myc</em> transgenic seedlings (<a class="xref-" href="#F6" id="xref-fig-6-11">Fig. 6E</a>). During seed germination, ABA did not alter <em>AIP2</em> transcript levels compared with untreated seeds; therefore, it is not surprising the enhanced accumulation of ABI3 protein
                     levels by ABA, owing to more abundant <em>ABI3</em> transcripts, is not affected. Taken together, our results indicate that ABA modulates ABI3 at the post-translational level,
                     mainly by regulating the expression levels of its cognate E3 ligase, AIP2.
                  </p>
                  <h3>AIP2 negatively regulates ABA signaling</h3>
                  <p id="p-37">Our genetic evidence also supports the conclusion that AIP2 is an E3 ligase for ABI3 in vivo. Since the main function of ABI3
                     is to block phase transitions during plant development, rapid ABI3 destruction is a prerequisite for seed germination after
                     stratification. ABA stabilizes ABI3 expression in <em>Arabidopsis</em> seeds, thus arresting growth and inhibiting their germination post-stratification. In our experiments, we observed elevated
                     accumulation of ABI3 in <em>aip2-1</em> seeds compared with that in wild-type (Col-0) seeds (<a class="xref-" href="#F6" id="xref-fig-6-12">Fig. 6A</a>). Therefore, it is not surprising that the <em>aip2-1</em> mutant is hypersensitive to ABA in assays of root growth inhibition and seed germination, somehow mimicking the phenotype
                     of the <em>35S-ABI3–6myc</em> transgenic plants. In contrast, transgenic plants expressing <em>35S-AIP2–3HA</em> were more resistant to ABA, phenocopying the <em>abi3-1</em> mutant. Furthermore, <em>aip2-1</em> mutants can be rescued by the native <em>AIP2</em> gene or a <em>35S-AIP2–3HA</em> transgene, but not by <em>35S-AIP2(C/S)–3HA</em> (<a class="xref-" href="#F7" id="xref-fig-7-7">Fig. 7D–F</a>). The complemented lines <em>35S-AIP2–3HA/aip2-1</em> showed even more resistance to ABA, obviously due to decreased levels of ABI3 compared with wild type (Col-0) (Figs. <a class="xref-" href="#F6" id="xref-fig-6-13">6C</a>, <a class="xref-" href="#F7" id="xref-fig-7-8">7D–F</a>). Besides similar resistance responses to ABA, AIP2-overexpressing seeds, like <em>abi3-1</em>, have reduced seed viability, suggesting a loss of drought tolerance, although we did not obtain green seeds. Additional
                     evidence is that induced <em>AIP2</em> expression could suppress the ABI3 overexpression phenotype and advance flowering time. Based on these results, we conclude
                     that AIP2 is a novel negative regulator of ABA response and that it executes its function through modulating the stability
                     of ABI3 (<a class="xref-" href="#F7" id="xref-fig-7-9">Fig. 7G</a>).
                  </p>
                  <h3>AIP2 may have other targets</h3>
                  <p id="p-38">Whereas in vitro assays and in vivo data with mutants and transgenic plants indicate that the AIP2 E3 ligase regulates ABI3
                     abundance, several lines of evidence suggest that AIP2 may also modify other substrates. First, expression patterns of AIP2
                     and ABI3 are not perfectly matched with each other as analyzed by Northern blots and GUS-reporter activity. AIP2 is ubiquitously
                     abundant in both vegetative and reproductive tissues, whereas ABI3 is preferentially seed-specific except in the presence
                     of exogenous ABA, when GUS expression was detected in seedling roots (<a class="xref-" href="#F4" id="xref-fig-4-7">Fig. 4</a>; <a class="xref-" href="#ref-36" id="xref-ref-36-3">Ng et al. 2004</a>; data not shown). Second, the AIP2 protein has a stronger binding affinity for the B2 + B3 domains of ABI3 than its A1 +
                     B1 domains in pull-down experiments, but this E3 ligase only modifies ABI3 A1 + B1 domains (Figs. <a class="xref-" href="#F3" id="xref-fig-3-8">3C</a>, <a class="xref-" href="#F5" id="xref-fig-5-8">5A</a>). In other words, their interaction domain and polyubiquitination sites are different from each other. This observation led
                     us to propose that the domain of a substrate that imparts interaction specificity with its E3 may sometimes be located in
                     a site distinct from where ubiquitination occurs. Provided that the requisite amount of bioinformatics and proteomics information
                     is accessible, it might be possible to identify other substrates that share the same E3. Considering that the <em>Arabidopsis</em> genome encodes 43 protein members of the B3 domain family, it will be interesting to explore whether AIP2 is also implicated
                     in the regulated proteolysis of these B3 domain factors in vivo.
                  </p>
                  <p id="p-39">The incomplete match of <em>AIP2</em> and <em>ABI3</em> expression profiles also raises the possibility that ABI3 degradation in vivo may require other pathways as well. There are
                     several examples of differential utilization of E3 ligases to modify one substrate depending on the physiological context
                     (<a class="xref-" href="#ref-15" id="xref-ref-15-4">Hare et al. 2003</a>). In our study, AIP2 turnover is much faster than ABI3 at the post-translational level, suggesting that other E3(s) may contribute
                     to ABI3 destruction (data not shown). Furthermore, the <em>aip2-1</em> mutant has a normal phenotype with respect to flowering time and seed development, and <em>AIP2</em>-overexpressing plants did not produce “green” seeds, a characteristic phenotype of the <em>abi3-4</em> mutant, a more severe allele than <em>abi3-1</em>. Finally, the greater ABA sensitivity of <em>35S-AIP2(C/S)/aip2-1</em> seeds compared with <em>aip2-1</em> mutant seeds can be explained if overexpression of the mutant AIP2 E3 ligase interferes with the activity of other negative
                     regulators in ABA signaling. Genomic analysis did not reveal any AIP2 homolog in <em>Arabidopsis thaliana</em>. We expect that exploring additional E3(s) that contribute to the modification of ABI3 will advance our insight in the regulation
                     of ABA signaling.
                  </p>
               </div>
               <div class="section" id="sec-15">
                  <div class="section-nav"><a href="#sec-10" title="Discussion" class="prev-section-link"><span>Previous Section</span></a><a href="#ack-1" title="Acknowledgments" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>Materials and methods</h2>
                  <h3>DNA construction</h3>
                  <p id="p-40">Most constructs were made using the Gateway system (Invitrogen). Several destination vectors (DC in our nomenclature) were
                     created for either protein expression in <em>Escherichia coli</em> or plant transformation. The pMAL-p2 containing maltose-binding protein (MBP) (NEB) was modified to give pMBP-DC-6myc, pMBP-DC-3HA,
                     and pMBP-DC. The pRSETc containing 6xHis (Invitrogen) was modified to obtain pRSETc-DC-6myc, and pRSETc-DC-3HA. The binary
                     vector pBA002 (<a class="xref-" href="#ref-23" id="xref-ref-23-1">Kost et al. 1998</a>) for constitutive expression in plants under the control of CaMV 35S promoter was modified to obtain pBA-DC-6myc and pBA-DC-3HA.
                     The vector pER8 (<a class="xref-" href="#ref-55" id="xref-ref-55-1">Zuo et al. 2000</a>) for inducible expression in plants under the control of the XVE promoter was modified to obtain pER8-DC-6myc and pER8-DC-3HA.
                  </p>
                  <p id="p-41"><em>AIP2</em> and <em>ABI3</em> full-length cDNAs were cloned from dry seeds by RT–PCR. For the <em>AIP2</em> gene, primers were 5′-CACCATG GATGCATCGTCTTCACC-3′ and 5′-TTAAACGTACATATA TTCACCTCCGC-3′. The <em>AIP2</em> mutant gene (C230S; C231S) was generated using the Quikchange site-directed mutagenesis kit (Stratagene), pENTR-AIP2 template,
                     and the following primers: 5′-GTTTGGAGCAGAGGCTGAATCTTCCATCTGCAA GGAG-3′ and 5′-CTCCTTGCAGATGGAAGATTCAGCCTC TGCTCCAAAC-3′.
                     For the <em>ABI3</em> gene, the primers were 5′-CACCATGAAAAGCTTGCATGTGG-3′ and 5′-TCCTTTAA CAGTTTGAGAAGTTGGTGAAGCGACCAC-3′. The N-terminal mutant
                     of ABI3 (ABI3 ΔC) includes the first 414 amino acid residues (<em>ABI3</em> full forward and 5′-AAACAGCTGCAG GAACATAAGGAT-3′). The ABI3 C-terminal mutant (ABI3 ΔN) includes the last 281 amino acid
                     residues (5′-CACCGCAG GACAAATGAGAGAT-3′, and <em>ABI3</em> full reverse). The <em>AIP2</em> promoter construct includes all the 469 bp upstream of the <em>AIP2</em> start codon, and the entire coding region. All cDNA or DNA fragments were cloned in pENTR/D vector (Invitrogen) and then
                     transferred to the appropriate vectors by recombination using the LR Clonase enzyme (Invitrogen). For promoter analysis, the
                     promoter sequence was transferred to the vector pK7WS7 to obtain the <em>AIP2-GUS-GFP</em> fusion (<a class="xref-" href="#ref-22" id="xref-ref-22-1">Karimi et al. 2002</a>). For subcellular analysis, full-length cDNAs for <em>AIP2</em> and <em>ABI3</em> were amplified without the stop codon and cloned inframe upstream of the <em>YFP</em> and <em>CFP</em> genes, respectively. Both constructs are under the control of the CaMV 35S promoter. All constructs were verified by sequencing.
                  </p>
                  <h3>Plant material and growth conditions</h3>
                  <p id="p-42">Seeds of <em>A. thaliana</em> ecotype Columbia 0 (Col-0) or Landsberg erecta (Ler) were treated and grown as described (<a class="xref-" href="#ref-29" id="xref-ref-29-6">Lopez-Molina et al. 2001</a>). The <em>aip2-1</em> T-DNA insertion line (SALK_143574) was obtained from the SALK collection, and plants homozygous for the T-DNA insertion were
                     selected by PCR following standard procedures (<a class="xref-" href="#ref-1" id="xref-ref-1-2">Alonso et al. 2003</a>). The T-DNA insertion site was confirmed by sequencing of PCR fragments. The absence of <em>AIP2</em> expression in these plants was confirmed by Northern and Western blot analyses. Transgenic plants were generated by the floral
                     dip transformation method (<a class="xref-" href="#ref-3" id="xref-ref-3-1">Bechtold et al. 1993</a>). Double-transgenic plants were obtained by the same method by coinfiltrating Col-0 plants with two constructs at the same
                     time. Seeds from infiltrated plants were selected on standard MS medium (<a class="xref-" href="#ref-34" id="xref-ref-34-1">Murashige and Skoog 1962</a>) containing the appropriate inhibitors: 50 mg/L kanamycin (Sigma), 10 mg/L gluphosinate ammonium (Crescent Chemical Co.),
                     and/or 25 mg/L hygromycin (Sigma) together with 100 mg/L cefotaxime (Sigma). All transgenic plants are in the Col-0 background
                     except for the <em>AIP2-GUS-GFP</em> fusion, which is in the Ler background.
                  </p>
                  <h3>MG132, cycloheximide, ABA, and β-estradiol treatments</h3>
                  <p id="p-43">Transgenic <em>Arabidopsis</em> seedlings expressing either <em>35S-ABI3–6myc</em> or <em>35S-AIP2–3HA</em>, or <em>35S-ABI3–6myc/XVE-AIP2–3HA</em> were germinated and grown on selective media for 2 or 3 wk (16 h light/8 h dark photoperiod) before transfer to liquid MS
                     medium supplemented with the indicated concentrations of MG132 (Calbiochem), and/or cycloheximide (Sigma), as well as ABA
                     (Sigma). Treated seedlings were harvested at the indicated times for Western and Northern blot analyses. For β-estradiol treatment,
                     double-transgenic plants harboring <em>35S-ABI3–6myc</em> and either <em>XVE-AIP2–3HA</em> or <em>XVE-AIP2(C/S)–3HA</em> were grown on MS solid medium containing 1% sucrose under continuous light at 22°C for 2–3 wk, and then incubated in MS liquid
                     medium supplemented with 25 μM β-estradiol (Sigma) for 16 h before Northern and Western blot analyses.
                  </p>
                  <h3>Expression and purification of recombinant proteins</h3>
                  <p id="p-44">All recombinant proteins were prepared according to the manufacturer's protocols. Briefly, for purification of MBP fusion
                     proteins, BL21 bacteria were lysed in a buffer containing 50 mM Tris-HCl (pH 7.4), 200 mM NaCl, 1 mM EDTA, 10 mM 2-mercaptoethanol,
                     2 mM phenylmethylsulphonyl fluoride (PMSF), and a complete proteinase inhibitor cocktail (Roche), and MBP fusion proteins
                     were purified using amylose resins (New England Biolabs). For purification of 6xHis fusion proteins, bacteria were lysed in
                     a buffer containing 50 mM Tris-HCl (pH 8.0), 300 mM NaCl, 1% Triton X-100, 10 mM imidazole, 5 mM 2-mercaptoethanol, 2 mM PMSF,
                     and an EDTA-free proteinase inhibitor cocktail (Roche), and proteins were purified using nitrilotriacetate (NTA) resin (QIAGEN).
                     All proteins were dialyzed against dialysis buffer (20 mM Tris-HCl at pH 7.4, 2 mM MgCl<sub>2</sub>, 150 mM NaCl, 2.5 mM 2-mercaptoethanol, and 10% glycerol).
                  </p>
                  <h3>Ubiquitination assay</h3>
                  <p id="p-45">For the AIP2 autoubiquitination assay, each reaction (30 μL final volume) contained 10 μg of recombinant Ub (Sigma), 0.1 μg
                     rabbit E1 (Boston Biochemicals), 0.22 μg E2 UbcH5b (Boston Biochemicals), 200 ng purified MBP-AIP2–3HA, 2 mM ATP, 50 mM Tris-HCl
                     (pH 7.4), 5 mM MgCl, and 2 mM DTT. After incubation at 30°C for 2 h, the reaction was stopped with 2× SDS-PAGE loading buffer
                     at 100°C for 5 min. Ten microliters of each reaction was analyzed by electrophoresis on 6% SDS-PAGE gels. Ubiquitinated proteins
                     were detected by Western blotting using anti-HA antibody. ABI3 ubiquitination was performed similarly except the reaction
                     mix contained 400 ng E3 (MBP-AIP2–3HA, or MBP-AIP2) and 50 ng of purified 6xHis-ABI3–6myc or truncated ABI3 mutant proteins.
                     Polyubiquitinated ABI3 was detected by Western blotting using anti-myc antibody.
                  </p>
                  <h3>In vitro pull-down assays and in vivo coimmunoprecipitation</h3>
                  <p id="p-46">Prey proteins of 2.5 μg (6xHis-ABI3–6myc or truncated ABI3 mutants) were preabsorbed for 1 h at room temperature in 1 mL of
                     binding buffer (50 μL of amylose resin, 50 mM Tris at pH 7.5, 100 mM NaCl, 0.2% glycerol, 0.6% Triton X-100, 0.5 mM β-mercaptoethanol).
                     The mixture was cleared by centrifugation at 12,000<em>g</em> for 2 min. The resulting supernatant was transferred to a new tube containing 2.5 μg of the bait protein (MBP-AIP2-HA3).
                     After incubation at room temperature for 2 h, amylose resin beads were added, and the incubation continued in the same conditions.
                     Finally, six further vigorous washes were performed with the washing buffer (50 mM Tris at pH 7.5, 100 mM NaCl, 0.6% Triton
                     X-100). Pulled-down proteins were resolved by 6% SDS-PAGE and detected by Western blotting using the appropriate antibody.
                     For coimmunoprecipitation, 2-wk-old <em>35S-ABI3–6myc/XVE-AIP2(C/S)–3HA</em> transgenic seedlings were transferred into a medium supplemented with MG132 (50 μM) in the presence or absence of β-estradiol
                     (25 μM) for 20 h. Total proteins were extracted in 50 mM Tris-HCl (pH 7.0), 150 mM NaCl, 10 mM MgCl<sub>2</sub>, and 0.2% Triton-100 supplemented with a EDTA-free protease inhibitor cocktail (Amersham Pharmacia). The protein extracts
                     were immunoprecipitated (IP) with monoclonal antibody to HA or His at 4°C for 3 h. Protein A beads were then added and incubated
                     for another 3 h. Beads were washed six times with the same buffer before an equal volume of SDS-loading buffer was added.
                     Western blots were analyzed with a polyclonal antibody to myc to detect coimmunoprecipitated ABI3–6myc. In a parallel experiment,
                     immunoprecipitation was performed using a polyclonal antibody to myc. Western blots were analyzed with a monoclonal antibody
                     to HA to detect coimmunoprecipitated AIP2–3HA protein. An ECL plus kit (Amersham Biosciences) was used for visualization of
                     membrane-associated peroxidase activity.
                  </p>
                  <h3>RNA extraction, antibody production, Northern blot, and Western blot analyses</h3>
                  <p id="p-47">RNA extraction and RNA blot analyses were performed as described (<a class="xref-" href="#ref-51" id="xref-ref-51-1">Vicient and Delseny 1999</a>) using gene-specific probes. Polyclonal anti-ABI3 and AIP2 antiserum was generated in rabbits immunized with the recombinant
                     full-length ABI3 or AIP2 proteins fused to 6xHis tag at their N termini. For Western blot analyses, ABI3 and AIP2 antibodies
                     were affinity-purified from sera using the same recombinant proteins used for rabbit immunizations (<a class="xref-" href="#ref-20" id="xref-ref-20-1">Jedd et al. 1995</a>).
                  </p>
                  <h3>Image quantification</h3>
                  <p id="p-48">Northern and Western blots were scanned using Quantity One (Bio-Rad), and image intensity was quantified using the software
                     Image Gauge version 3.12 (Fuji). Fold changes in levels of transcripts and proteins were normalized using 18S rRNA, and tubulin
                     or cross-reaction bands as controls.
                  </p>
                  <h3>Subcellular localization experiments, confocal microscopy, and GUS staining</h3>
                  <p id="p-49">Three-week-old tobacco plants (<em>Nicotiana benthamiana</em>) were agroinfiltrated by using a syringe without needle as previously described (<a class="xref-" href="#ref-43" id="xref-ref-43-1">Schob et al. 1997</a>). The <em>Agrobacterium</em> suspension was supplemented with 25 μM MG132 during infiltration. Plants were maintained for 2 d at 24°C (16 h light/8 h
                     dark). Confocal images for YFP and CFP were obtained with an Axiovert 200 inverted, LMS 510 multiphoton confocal Microscope
                     (Zeiss). An Argon laser was used, and the chlorophyll autoflorescence was filtered out by using a 540/520-nm filter. Images
                     were processed using Spot Advance and Adobe Photoshop software.
                  </p>
                  <p id="p-50">Transgenic plants harboring <em>AIP2-GUS-GFP</em> were stained for GUS activity as described (<a class="xref-" href="#ref-4" id="xref-ref-4-1">Beeckman 1994</a>). Stained plants were visualized using an Axioskop microscope (Zeiss) coupled to an Insight digital camera.
                  </p>
                  <h3>Germination and root sensitivity to ABA</h3>
                  <p id="p-51">For germination assays, 50–200 seeds were placed on MS medium plates containing 1% sucrose and the different concentrations
                     of ABA. Germinated seeds (radicle protruding) were counted daily for 7 d. Germination ratio refers to the number of germinated
                     seed as a proportion of the total number of seed. The final number was normalized to the germination frequency on an ABA-free
                     medium. The effect of ABA on root growth was analyzed as described (<a class="xref-" href="#ref-37" id="xref-ref-37-3">Parcy et al. 1994</a>) with some modifications. Wild-type, mutant, and transgenic plants were germinated and grown for 5 d on ABA-free medium.
                     Seedlings (12 seedlings per plate) were then transferred to plates containing the indicated concentrations of ABA, and the
                     primary roots lengths were scored (day 0). After 5 d, the primary root lengths were measured again to determine the additional
                     root growth. In these experiments, plates were placed in a vertical position and were kept in growth chambers at 22°C under
                     continuous light.
                  </p>
               </div>
               <div class="section ack" id="ack-1">
                  <div class="section-nav"><a href="#sec-15" title="Materials and methods" class="prev-section-link"><span>Previous Section</span></a><a href="#fn-group-1" title="Footnotes" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>Acknowledgments</h2>
                  <p id="p-58">We thank Dr. Peter Hare for critical reading of the manuscript, Dr. Hak Soo Seo for technical advice, Dr. Shih-Shun Lin for
                     sharing his Gateway vectors, and Dr. Allison North for advice on confocal microscopy. Confocal images were collected at the
                     Rockefeller University Bio-Imaging Resource Center. This work was supported by DOE grant DE-FG02-94ER20142 to N.-H.C.
                  </p>
               </div>
               <div class="section fn-group" id="fn-group-1">
                  <div class="section-nav"><a href="#ack-1" title="Acknowledgments" class="prev-section-link"><span>Previous Section</span></a><a href="#ref-list-1" title="References" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>Footnotes</h2>
                  <ul><li class="fn" id="fn-1">
                        <p id="p-52">Supplemental material is available at http://www.genesdev.org.</p>
                     </li>
                     <li class="fn" id="fn-2">
                        <p id="p-53">Article and publication are at http://www.genesdev.org/cgi/doi/10.1101/gad.1318705.</p>
                     </li>
                     <li class="fn" id="fn-3">
                        <p id="p-54"><a class="rev-xref" href="#xref-fn-3-1">↵</a><span class="fn-label">1</span> <strong>These authors contributed equally to this work.</strong></p>
                     </li>
                     <li class="fn" id="fn-4">
                        <p id="p-55"><a class="rev-xref" href="#xref-fn-4-1">↵</a><span class="fn-label">2</span> <strong>Present address: AustralBiotech, San Sebastian 2952, Piso 5, Las Condes, Santiago, Chile.</strong></p>
                     </li>
                     <li class="fn" id="fn-5">
                        <p id="p-56"><a class="rev-xref" href="#xref-fn-5-1">↵</a><span class="fn-label">3</span> <strong>Corresponding author.</strong></p>
                        <p id="p-57"><a class="rev-xref" href="#xref-fn-5-1">↵</a><span class="fn-label">3</span> <strong>E-MAIL <span class="em-link"><span class="em-addr">chua{at}mail.rockefeller.edu</span></span>; FAX (212) 327-8327.</strong></p>
                     </li>
                     <li class="history" id="history-1">
                        <ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2005-05-16">Accepted May 16, 2005.</li>
                           <li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2005-03-28">Received March 28, 2005.</li>
                        </ul></li>
                     <li class="copyright-statement" id="copyright-statement-1">Cold Spring Harbor Laboratory Press</li>
                  </ul></div>
               <div class="section ref-list" id="ref-list-1">
                  <div class="section-nav"><a href="#fn-group-1" title="Footnotes" class="prev-section-link"><span>Previous Section</span></a><div class="nav-placeholder"> </div>
                  </div>
                  <h2>References</h2>
                  <ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference in text" id="ref-1">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.1">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Alonso</span> <span class="cit-name-given-names">J.M.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Alonso</span>  <span class="cit-name-given-names">J.M.</span></span>, Stepanova, A.N., Leisse, T.J., Kim, C.J., Chen, H.M., Shinn, P., Stevenson, D.K., Zimmerman, J., Barajas, P., Cheuk, R.,
                                 et al. <span class="cit-pub-date">2003</span>. Genome-wide Insertional mutagenesis of Arabidopsis thaliana. <span class="cit-source">Science</span> <span class="cit-vol">301</span>: <span class="cit-fpage">653</span>–657.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=301/5633/653" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference in text" id="ref-2">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.2">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Beaudoin</span> <span class="cit-name-given-names">N.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Beaudoin</span>  <span class="cit-name-given-names">N.</span></span>, Serizet, C., Gosti, F., and Giraudat, J. <span class="cit-pub-date">2000</span>. Interactions between abscisic acid and ethylene signaling cascades. <span class="cit-source">Plant Cell</span> <span class="cit-vol">12</span>: <span class="cit-fpage">1103</span>–1115.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=12/7/1103" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference in text" id="ref-3">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.3">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Bechtold</span> <span class="cit-name-given-names">N.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Bechtold</span>  <span class="cit-name-given-names">N.</span></span>, Ellis, J., and Pelletier, G. <span class="cit-pub-date">1993</span>. In planta <em>Agrobacterium</em>-mediated gene-transfer by infiltration of adult <em>Arabidopsis thaliana</em> plants. <span class="cit-source">C. R. Acad. Sci. III</span> <span class="cit-vol">316</span>: <span class="cit-fpage">1194</span>–1199.</cite></div>
                           <div class="cit-extra"/>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference in text" id="ref-4">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.4">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Beeckman</span> <span class="cit-name-given-names">T.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Beeckman</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-pub-date">1994</span>. An easy technique for clearing of histochemically stained plant tissue. <span class="cit-source">Plant Mol. Biol. Rep.</span> <span class="cit-vol">82</span>: <span class="cit-fpage">259</span>–266.</cite></div>
                           <div class="cit-extra"/>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference in text" id="ref-5">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.5">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Brady</span> <span class="cit-name-given-names">S.M.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Brady</span>  <span class="cit-name-given-names">S.M.</span></span>, Sarkar, S.F., Bonetta, D., and McCourt., P. <span class="cit-pub-date">2003</span>. The ABSCISIC ACID INSENSITIVE 3 (ABI3) gene is modulated by farnesylation and is involved in auxin signaling and lateral
                                 root development in Arabidopsis. <span class="cit-source">Plant J.</span> <span class="cit-vol">34</span>: <span class="cit-fpage">67</span>–75.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1046/j.1365-313X.2003.01707.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=12662310&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000181862400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference in text" id="ref-6">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.6">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Brocard-Gifford</span> <span class="cit-name-given-names">I.M.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Brocard-Gifford</span>  <span class="cit-name-given-names">I.M.</span></span>, Lynch, T.J., and Finkelstein, R.R. <span class="cit-pub-date">2003</span>. Regulatory networks in seeds integrating developmental, abscisic acid, sugar, and light signaling. <span class="cit-source">Plant Physiol.</span> <span class="cit-vol">131</span>: <span class="cit-fpage">78</span>–92.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=131/1/78" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference in text" id="ref-7">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.7">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Brocard-Gifford</span> <span class="cit-name-given-names">I.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Brocard-Gifford</span>  <span class="cit-name-given-names">I.</span></span>, Lynch, T.J., Garcia, M.E., Malhotra, B., and Finkelstein, R.R. <span class="cit-pub-date">2004</span>. The <em>Arabidopsis thaliana</em> ABSCISIC ACID-INSENSITIVE8 locus encodes a novel protein mediating abscisic acid and sugar responses essential for growth.
                                 <span class="cit-source">Plant Cell</span> <span class="cit-vol">16</span>: <span class="cit-fpage">406</span>–421.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=16/2/406" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference in text" id="ref-8">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.8">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Carson</span> <span class="cit-name-given-names">C.B.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Carson</span>  <span class="cit-name-given-names">C.B.</span></span>, Hattori, T., Rosenkrans, L., Vasil, V., Vasil, I.-K., Peterson, P.A., and McCarty, D.R. <span class="cit-pub-date">1997</span>. The quiescent/colorless alleles of viviparous1 show that the conserved B3 domain of VP1 is not essential for ABA-regulated
                                 gene expression in the seed. <span class="cit-source">Plant J.</span> <span class="cit-vol">12</span>: <span class="cit-fpage">1231</span>–1240.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1046/j.1365-313x.1997.12061231.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=9450338&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000071560900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference in text" id="ref-9">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.9">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Cutler</span> <span class="cit-name-given-names">S.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Cutler</span>  <span class="cit-name-given-names">S.</span></span>, Ghassemian, M., Bonetta, D., Cooney, S., and McCourt, P. <span class="cit-pub-date">1996</span>. A protein farnesyl transferase involved in abscisic acid signal transduction in Arabidopsis. <span class="cit-source">Science</span> <span class="cit-vol">273</span>: <span class="cit-fpage">1239</span>–1241.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=273/5279/1239" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference in text" id="ref-10">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.10">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Ezcurra</span> <span class="cit-name-given-names">I.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Ezcurra</span>  <span class="cit-name-given-names">I.</span></span>, Wycliffe, P., Nehlin, L., Ellerstrom, M., and Rask, L. <span class="cit-pub-date">2000</span>. Transactivation of the <em>Brassica napus</em> napin promoter by ABI3 requires interaction of the conserved B2 and B3 domains of ABI3 with different <em>cis</em>-elements: B2 mediates activation through an ABRE, whereas B3 interacts with an RY/G-box. <span class="cit-source">Plant J.</span> <span class="cit-vol">24</span>: <span class="cit-fpage">57</span>–66.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1046/j.1365-313x.2000.00857.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=11029704&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000089935800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference in text" id="ref-11">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.11">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Finkelstein</span> <span class="cit-name-given-names">R.R.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Finkelstein</span>  <span class="cit-name-given-names">R.R.</span></span> and Lynch, T.J. <span class="cit-pub-date">2000</span>. The <em>Arabidopsis</em> abscisic acid response gene ABI5 encodes a basic leucine zipper transcription factor. <span class="cit-source">Plant Cell</span> <span class="cit-vol">12</span>: <span class="cit-fpage">599</span>–609.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=12/4/599" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference in text" id="ref-12">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.12">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Finkelstein</span> <span class="cit-name-given-names">R.R.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Finkelstein</span>  <span class="cit-name-given-names">R.R.</span></span>, Wang, M.L., Lynch, T.J., Rao, S., and Goodman, H.M. <span class="cit-pub-date">1998</span>. The <em>Arabidopsis</em> abscisic acid response locus ABI4 encodes an APETALA2 domain protein. <span class="cit-source">Plant Cell</span> <span class="cit-vol">10</span>: <span class="cit-fpage">1043</span>–1054.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=10/6/1043" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference in text" id="ref-13">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.13">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Finkelstein</span> <span class="cit-name-given-names">R.R.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Finkelstein</span>  <span class="cit-name-given-names">R.R.</span></span>, Gampala, S.S.L., and Rock, C.D. <span class="cit-pub-date">2002</span>. Abscisic acid signaling in seeds and seedlings. <span class="cit-source">Plant Cell</span> <span class="cit-vol">14</span>: <span class="cit-fpage">S15</span>–S45.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=FULL&amp;journalCode=plantcell&amp;resid=14/suppl_1/S15" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference in text" id="ref-14">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.14">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Giraudat</span> <span class="cit-name-given-names">J.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Giraudat</span>  <span class="cit-name-given-names">J.</span></span>, Hauge, B.M., Valon, C., Smalle, J., Parcy, F., and Goodman, H.M. <span class="cit-pub-date">1992</span>. Isolation of the <em>Arabidopsis</em>-ABI3 gene by positional cloning. <span class="cit-source">Plant Cell</span> <span class="cit-vol">4</span>: <span class="cit-fpage">1251</span>–1261.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=4/10/1251" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference in text" id="ref-15">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.15">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Hare</span> <span class="cit-name-given-names">P.D.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Hare</span>  <span class="cit-name-given-names">P.D.</span></span>, Seo, H.S., Yang, J.Y., and Chua, N.H. <span class="cit-pub-date">2003</span>. Modulation of sensitivity and selectivity in plant signaling by proteasomal destabilization. <span class="cit-source">Curr. Opin. Plant Biol.</span> <span class="cit-vol">6</span>: <span class="cit-fpage">453</span>–462.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1016/S1369-5266(03)00080-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=12972046&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000185580800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference in text" id="ref-16">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.16">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Hellmann</span> <span class="cit-name-given-names">H.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Hellmann</span>  <span class="cit-name-given-names">H.</span></span> and Estelle, M. <span class="cit-pub-date">2002</span>. Plant development: Regulation by protein degradation. <span class="cit-source">Science</span> <span class="cit-vol">297</span>: <span class="cit-fpage">793</span>–797.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=297/5582/793" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference in text" id="ref-17">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.17">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Hoecker</span> <span class="cit-name-given-names">U.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Hoecker</span>  <span class="cit-name-given-names">U.</span></span>, Vasil, I.K., and McCarty, D.R. <span class="cit-pub-date">1995</span>. Integrated control of seed maturation and germination programs by activator and repressor functions of viviparous-1 of maize.
                                 <span class="cit-source">Genes &amp; Dev.</span> <span class="cit-vol">9</span>: <span class="cit-fpage">2459</span>–2469.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=genesdev&amp;resid=9/20/2459" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference in text" id="ref-18">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.18">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Hugouvieux</span> <span class="cit-name-given-names">V.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Hugouvieux</span>  <span class="cit-name-given-names">V.</span></span>, Kwak, J.M., and Schroeder, J.I. <span class="cit-pub-date">2001</span>. An mRNA cap binding protein, ABH1, modulates early abscisic acid signal transduction in Arabidopsis. <span class="cit-source">Cell</span> <span class="cit-vol">106</span>: <span class="cit-fpage">477</span>–487.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1016/S0092-8674(01)00460-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=11525733&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000170665200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference in text" id="ref-19">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.19">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Jang</span> <span class="cit-name-given-names">I.C.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Jang</span>  <span class="cit-name-given-names">I.C.</span></span>, Yang, J.Y., Seo, H.S., and Chua, N.H. <span class="cit-pub-date">2005</span>. HFR1 is targeted by COP1 E3 ligase for post-translational proteolysis during phytochrome A signaling. <span class="cit-source">Genes &amp; Dev.</span> <span class="cit-vol">19</span>: <span class="cit-fpage">593</span>–602.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=genesdev&amp;resid=19/5/593" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference in text" id="ref-20">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.20">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Jedd</span> <span class="cit-name-given-names">G.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Jedd</span>  <span class="cit-name-given-names">G.</span></span>, Richardson, C., Litt, R., and Segev, N. <span class="cit-pub-date">1995</span>. The Ypt1 GTPase is essential for the first 2 steps of the yeast secretory pathway. <span class="cit-source">J. Cell Biol.</span> <span class="cit-vol">131</span>: <span class="cit-fpage">583</span>–590.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=jcb&amp;resid=131/3/583" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference in text" id="ref-21">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.21">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Jones</span> <span class="cit-name-given-names">H.D.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">H.D.</span></span>, Kurup, S., Peters, N.C.B., and Holdsworth, M.J. <span class="cit-pub-date">2000</span>. Identification and analysis of proteins that interact with the <em>Avena fatua</em> homologue of the maize transcription factor VIVIPAROUS 1. <span class="cit-source">Plant J.</span> <span class="cit-vol">21</span>: <span class="cit-fpage">133</span>–142.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1046/j.1365-313x.2000.00662.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=10743654&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000085607300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference in text" id="ref-22">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.22">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Karimi</span> <span class="cit-name-given-names">M.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Karimi</span>  <span class="cit-name-given-names">M.</span></span>, Inze, D., and Depicker, A. <span class="cit-pub-date">2002</span>. GATEWAY(TM) vectors for <em>Agrobacterium</em>-mediated plant transformation. <span class="cit-source">Trends Plant Sci.</span> <span class="cit-vol">7</span>: <span class="cit-fpage">193</span>–195.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1016/S1360-1385(02)02251-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=11992820&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000175361600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference in text" id="ref-23">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.23">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Kost</span> <span class="cit-name-given-names">B.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Kost</span>  <span class="cit-name-given-names">B.</span></span>, Spielhofer, P., and Chua, N.H. <span class="cit-pub-date">1998</span>. A GFP-mouse talin fusion protein labels plant actin filaments in vivo and visualizes the actin cytoskeleton in growing pollen
                                 tubes. <span class="cit-source">Plant J.</span> <span class="cit-vol">16</span>: <span class="cit-fpage">393</span>–401.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1046/j.1365-313x.1998.00304.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=9881160&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000077513000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference in text" id="ref-24">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.24">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Kurup</span> <span class="cit-name-given-names">S.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Kurup</span>  <span class="cit-name-given-names">S.</span></span>, Jones, H.D., and Holdsworth, M.J. <span class="cit-pub-date">2000</span>. Interactions of the developmental regulator ABI3 with proteins identified from developing <em>Arabidopsis</em> seeds. <span class="cit-source">Plant J.</span> <span class="cit-vol">21</span>: <span class="cit-fpage">143</span>–155.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1046/j.1365-313x.2000.00663.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=10743655&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000085607300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference in text" id="ref-25">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.25">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Leung</span> <span class="cit-name-given-names">J.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">J.</span></span>, Bouvierdurand, M., Morris, P.C., Guerrier, D., Chefdor, F., and Giraudat, J. <span class="cit-pub-date">1994</span>. <em>Arabidopsis</em> AbA response gene AbI1—Features of a calcium-modulated protein phosphatase. <span class="cit-source">Science</span> <span class="cit-vol">264</span>: <span class="cit-fpage">1448</span>–1452.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=264/5164/1448" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference in text" id="ref-26">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.26">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Li</span> <span class="cit-name-given-names">G.F.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G.F.</span></span>, Bishop, K.J., Chandrasekharan, M.B., and Hall, T.C. <span class="cit-pub-date">1999</span>. β-Phaseolin gene activation is a two-step process: PvALF-facilitated chromatin modification followed by abscisic acid-mediated
                                 gene activation. <span class="cit-source">Proc. Natl. Acad. Sci.</span> <span class="cit-vol">96</span>: <span class="cit-fpage">7104</span>–7109.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=96/12/7104" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference in text" id="ref-27">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.27">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Li</span> <span class="cit-name-given-names">G.F.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G.F.</span></span>, Chandrasekharan, M.B., Wolffe, A.P., and Hall, T.C. <span class="cit-pub-date">2001</span>. Chromatin structure and phaseolin gene regulation. <span class="cit-source">Plant Mol. Biol.</span> <span class="cit-vol">46</span>: <span class="cit-fpage">121</span>–129.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1023/A:1010693703421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=11442053&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000169130800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference in text" id="ref-28">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.28">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Lopez-Molina</span> <span class="cit-name-given-names">L.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Lopez-Molina</span>  <span class="cit-name-given-names">L.</span></span> and Chua, N.H. <span class="cit-pub-date">2000</span>. A null mutation in a bZIP factor confers ABA-insensitivity in Arabidopsis thaliana. <span class="cit-source">Plant Cell Physiol.</span> <span class="cit-vol">41</span>: <span class="cit-fpage">541</span>–547.</cite></div>
                           <div class="cit-extra"/>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference in text" id="ref-29">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.29">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Lopez-Molina</span> <span class="cit-name-given-names">L.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Lopez-Molina</span>  <span class="cit-name-given-names">L.</span></span>, Mongrand, S., and Chua, N.H. <span class="cit-pub-date">2001</span>. A postgermination developmental arrest checkpoint is mediated by abscisic acid and requires the AB15 transcription factor
                                 in Arabidopsis. <span class="cit-source">Proc. Natl. Acad. Sci.</span> <span class="cit-vol">98</span>: <span class="cit-fpage">4782</span>–4787.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=98/8/4782" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference in text" id="ref-30">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.30">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Lopez-Molina</span> <span class="cit-name-given-names">L.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Lopez-Molina</span>  <span class="cit-name-given-names">L.</span></span>, Mongrand, S., McLachlin, D.T., Chait, B.T., and Chua, N.H. <span class="cit-pub-date">2002</span>. ABI5 acts downstream of ABI3 to execute an ABA-dependent growth arrest during germination. <span class="cit-source">Plant J.</span> <span class="cit-vol">32</span>: <span class="cit-fpage">317</span>–328.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1046/j.1365-313X.2002.01430.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=12410810&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000178965600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference in text" id="ref-31">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.31">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Lopez-Molina</span> <span class="cit-name-given-names">L.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Lopez-Molina</span>  <span class="cit-name-given-names">L.</span></span>, Mongrand, S., Kinoshita, N., and Chua, N.H. <span class="cit-pub-date">2003</span>. AFP is a novel negative regulator of ABA signaling that promotes ABI5 protein degradation. <span class="cit-source">Genes &amp; Dev.</span> <span class="cit-vol">17</span>: <span class="cit-fpage">410</span>–418.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=genesdev&amp;resid=17/3/410" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference in text" id="ref-32">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.32">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Lu</span> <span class="cit-name-given-names">C.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">C.</span></span> and Fedoroff, N. <span class="cit-pub-date">2000</span>. A mutation in the <em>Arabidopsis</em> HYL1 gene encoding a dsRNA binding protein affects responses to abscisic acid, auxin, and cytokinin. <span class="cit-source">Plant Cell</span> <span class="cit-vol">12</span>: <span class="cit-fpage">2351</span>–2365.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=12/12/2351" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference in text" id="ref-33">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.33">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">McCarty</span> <span class="cit-name-given-names">D.R.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">McCarty</span>  <span class="cit-name-given-names">D.R.</span></span>, Hattori, T., Carson, C.B., Vasil, V., Lazar, M., and Vasil, I.K. <span class="cit-pub-date">1991</span>. The viviparous-1 developmental gene of maize encodes a novel transcriptional activator. <span class="cit-source">Cell</span> <span class="cit-vol">66</span>: <span class="cit-fpage">895</span>–905.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1016/0092-8674(91)90436-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=1889090&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=A1991GE46000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference in text" id="ref-34">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.34">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Murashige</span> <span class="cit-name-given-names">T.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Murashige</span>  <span class="cit-name-given-names">T.</span></span> and Skoog, F. <span class="cit-pub-date">1962</span>. A revised medium for rapid growth and bio assays with tobacco tissue cultures. <span class="cit-source">Physiol. Plant</span> <span class="cit-vol">15</span>: <span class="cit-fpage">473</span>–497.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1111/j.1399-3054.1962.tb08052.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference in text" id="ref-35">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.35">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Nakamura</span> <span class="cit-name-given-names">S.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">S.</span></span>, Lynch, T.J., and Finkelstein, R.R. <span class="cit-pub-date">2001</span>. Physical interactions between ABA response loci of Arabidopsis. <span class="cit-source">Plant J.</span> <span class="cit-vol">26</span>: <span class="cit-fpage">627</span>–635.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1046/j.1365-313x.2001.01069.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=11489176&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000170103400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference in text" id="ref-36">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.36">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Ng</span> <span class="cit-name-given-names">D.W.K.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">D.W.K.</span></span>, Chandrasekharan, M.B., and Hall, T.C. <span class="cit-pub-date">2004</span>. The 5′ UTR negatively regulates quantitative and spatial expression from the ABI3 promoter. <span class="cit-source">Plant Mol. Biol.</span> <span class="cit-vol">54</span>: <span class="cit-fpage">25</span>–38.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1023/B:PLAN.0000028767.06820.34&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=15159632&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000221591000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference in text" id="ref-37">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.37">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Parcy</span> <span class="cit-name-given-names">F.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Parcy</span>  <span class="cit-name-given-names">F.</span></span>, Valon, C., Raynal, M., Gaubiercomella, P., Delseny, M., and Giraudat, J. <span class="cit-pub-date">1994</span>. Regulation of gene-expression programs during <em>Arabidopsis</em> seed development—Roles of the Abi3 locus and of endogenous abscisic-acid. <span class="cit-source">Plant Cell</span> <span class="cit-vol">6</span>: <span class="cit-fpage">1567</span>–1582.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=6/11/1567" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference in text" id="ref-38">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.38">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Rodriguez</span> <span class="cit-name-given-names">P.L.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">P.L.</span></span>, Benning, G., and Grill, E. <span class="cit-pub-date">1998</span>. ABI2, a second protein phosphatase 2C involved in abscisic acid signal transduction in Arabidopsis. <span class="cit-source">FEBS Lett.</span> <span class="cit-vol">421</span>: <span class="cit-fpage">185</span>–190.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1016/S0014-5793(97)01558-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=9468303&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000071737700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference in text" id="ref-39">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.39">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Rohde</span> <span class="cit-name-given-names">A.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Rohde</span>  <span class="cit-name-given-names">A.</span></span>, Van Montagu, M., and Boerjan, W. <span class="cit-pub-date">1999</span>. The ABSCISIC ACID-INSENSITIVE 3 (ABI3) gene is expressed during vegetative quiescence processes in Arabidopsis. <span class="cit-source">Plant Cell Environ.</span> <span class="cit-vol">22</span>: <span class="cit-fpage">261</span>–270.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1046/j.1365-3040.1999.00428.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference in text" id="ref-40">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.40">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Rohde</span> <span class="cit-name-given-names">A.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Rohde</span>  <span class="cit-name-given-names">A.</span></span>, De Rycke, R., Beeckman, T., Engler, G., Van Montagu, M., and Boerjan, W. <span class="cit-pub-date">2000a</span>. ABI3 affects plastid differentiation in dark-grown <em>Arabidopsis</em> seedlings. <span class="cit-source">Plant Cell</span> <span class="cit-vol">12</span>: <span class="cit-fpage">35</span>–52.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=12/1/35" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference in text" id="ref-41">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.41">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Rohde</span> <span class="cit-name-given-names">A.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Rohde</span>  <span class="cit-name-given-names">A.</span></span>, Kurup, S., and Holdsworth, M. <span class="cit-pub-date">2000b</span>. ABI3 emerges from the seed. <span class="cit-source">Trends Plant Sci.</span> <span class="cit-vol">5</span>: <span class="cit-fpage">418</span>–419.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1016/S1360-1385(00)01736-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=11203275&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000089920700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference in text" id="ref-42">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.42">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Rohde</span> <span class="cit-name-given-names">A.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Rohde</span>  <span class="cit-name-given-names">A.</span></span>, Prinsen, E., De Rycke, R., Engler, G., Van Montagu, M., and Boerjan, W. <span class="cit-pub-date">2002</span>. PtABI3 impinges on the growth and differentiation of embryonic leaves during bud set in poplar. <span class="cit-source">Plant Cell</span> <span class="cit-vol">14</span>: <span class="cit-fpage">1885</span>–1901.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=14/8/1885" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference in text" id="ref-43">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.43">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Schob</span> <span class="cit-name-given-names">H.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Schob</span>  <span class="cit-name-given-names">H.</span></span>, Kunz, C., and Meins Jr., F. <span class="cit-pub-date">1997</span>. Silencing of transgenes introduced into leaves by agroinfiltration: A simple, rapid method for investigating sequence requirements
                                 for gene silencing. <span class="cit-source">Mol. Gen. Genet.</span> <span class="cit-vol">256</span>: <span class="cit-fpage">581</span>–585.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1007/s004380050604&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=9413443&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=A1997YJ11300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference in text" id="ref-44">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.44">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Schultz</span> <span class="cit-name-given-names">T.F.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Schultz</span>  <span class="cit-name-given-names">T.F.</span></span> and Quatrano, R.S. <span class="cit-pub-date">1997</span>. Characterization and expression of a rice RAD23 gene. <span class="cit-source">Plant Mol. Biol.</span> <span class="cit-vol">34</span>: <span class="cit-fpage">557</span>–562.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1023/A:1005870227632&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=9225866&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=A1997XK29000018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference in text" id="ref-45">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.45">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Seo</span> <span class="cit-name-given-names">H.S.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Seo</span>  <span class="cit-name-given-names">H.S.</span></span>, Yang, J.Y., Ishikawa, M., Bolle, C., Ballesteros, M.L., and Chua, N.H. <span class="cit-pub-date">2003</span>. LAF1 ubiquitination by COP1 controls photomorphogenesis and is stimulated by SPA1. <span class="cit-source">Nature</span> <span class="cit-vol">423</span>: <span class="cit-fpage">995</span>–999.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1038/nature01696&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=12827204&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference in text" id="ref-46">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.46">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Seo</span> <span class="cit-name-given-names">H.S.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Seo</span>  <span class="cit-name-given-names">H.S.</span></span>, Watanabe, E., Tokutomi, S., Nagatani, A., and Chua, N.H. <span class="cit-pub-date">2004</span>. Photoreceptor ubiquitination by COP1 E3 ligase desensitizes phytochrome A signaling. <span class="cit-source">Genes &amp; Dev.</span> <span class="cit-vol">18</span>: <span class="cit-fpage">617</span>–622.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=genesdev&amp;resid=18/6/617" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference in text" id="ref-47">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.47">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Signora</span> <span class="cit-name-given-names">L.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Signora</span>  <span class="cit-name-given-names">L.</span></span>, De Smet, I., Foyer, C.H., and Zhang, H.M. <span class="cit-pub-date">2001</span>. ABA plays a central role in mediating them regulatory effects of nitrate on root branching in Arabidopsis. <span class="cit-source">Plant J.</span> <span class="cit-vol">28</span>: <span class="cit-fpage">655</span>–662.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1046/j.1365-313x.2001.01185.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=11851911&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000173421000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference in text" id="ref-48">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.48">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Smalle</span> <span class="cit-name-given-names">J.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Smalle</span>  <span class="cit-name-given-names">J.</span></span>, Kurepa, J., Yang, P.Z., Emborg, T.J., Babiychuk, E., Kushnir, S., and Vierstra, R.D. <span class="cit-pub-date">2003</span>. The pleiotropic role of the 26S proteasome subunit RPN10 in <em>Arabidopsis</em> growth and development supports a substrate-specific function in abscisic acid signaling. <span class="cit-source">Plant Cell</span> <span class="cit-vol">15</span>: <span class="cit-fpage">965</span>–980.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=15/4/965" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference in text" id="ref-49">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.49">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Stone</span> <span class="cit-name-given-names">S.L.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">S.L.</span></span>, Hauksdottir, H., Troy, A., Herschleb, J., Kraft, E., and Callis, J. <span class="cit-pub-date">2005</span>. Functional analysis of the RING-type ubiquitin ligase family of Arabidopsis. <span class="cit-source">Plant Physiol.</span> <span class="cit-vol">137</span>: <span class="cit-fpage">13</span>–30.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=137/1/13" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference in text" id="ref-50">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.50">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Suzuki</span> <span class="cit-name-given-names">M.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">M.</span></span>, Kao, C.Y., and McCarty, D.R. <span class="cit-pub-date">1997</span>. The conserved B3 domain of VIVIPAROUS1 has a cooperative DNA binding activity. <span class="cit-source">Plant Cell</span> <span class="cit-vol">9</span>: <span class="cit-fpage">799</span>–807.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=9/5/799" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference in text" id="ref-51">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.51">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Vicient</span> <span class="cit-name-given-names">C.M.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Vicient</span>  <span class="cit-name-given-names">C.M.</span></span> and Delseny, M. <span class="cit-pub-date">1999</span>. Isolation of total RNA from Arabidopsis thaliana seeds. <span class="cit-source">Anal. Biochem.</span> <span class="cit-vol">268</span>: <span class="cit-fpage">412</span>–413.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1006/abio.1998.3045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=10075836&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000079508100033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference in text" id="ref-52">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.52">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Xie</span> <span class="cit-name-given-names">Q.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Q.</span></span>, Guo, H.S., Dallman, G., Fang, S.Y., Weissman, A.M., and Chua, N.H. <span class="cit-pub-date">2002</span>. SINAT5 promotes ubiquitin-related degradation of NAC1 to attenuate auxin signals. <span class="cit-source">Nature</span> <span class="cit-vol">419</span>: <span class="cit-fpage">167</span>–170.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1038/nature00998&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=12226665&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference in text" id="ref-53">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.53">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Xiong</span> <span class="cit-name-given-names">L.M.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">L.M.</span></span>, Gong, Z.Z., Rock, C.D., Subramanian, S., Guo, Y., Xu, W.Y., Galbraith, D., and Zhu, J.K. <span class="cit-pub-date">2001a</span>. Modulation of abscisic acid signal transduction and biosynthesis by an Sm-like protein in Arabidopsis. <span class="cit-source">Dev. Cell</span> <span class="cit-vol">1</span>: <span class="cit-fpage">771</span>–781.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1016/S1534-5807(01)00087-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=11740939&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000175301800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference in text" id="ref-54">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.54">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Xiong</span> <span class="cit-name-given-names">L.M.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">L.M.</span></span>, Lee, B.H., Ishitani, M., Lee, H., Zhang, C.Q., and Zhu, J.K. <span class="cit-pub-date">2001b</span>. FIERY1 encoding an inositol polyphosphate 1-phosphatase is a negative regulator of abscisic acid and stress signaling in
                                 <em>Arabidopsis.</em> <span class="cit-source">Genes &amp; Dev.</span> <span class="cit-vol">15</span>: <span class="cit-fpage">1971</span>–1984.</cite></div>
                           <div class="cit-extra"><a href="/cgi/ijlink?linkType=ABST&amp;journalCode=genesdev&amp;resid=15/15/1971" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"/><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference in text" id="ref-55">↵</a><div class="cit ref-cit ref-" id="cit-19.13.1532.55">
                           <div class="cit-metadata">
                              <ol class="cit-authors-list duplicate"><li><span class="cit-authors"><span class="cit-name-surname">Zuo</span> <span class="cit-name-given-names">J.R.</span></span></li>
                              </ol><cite><span class="cit-auth"><span class="cit-name-surname">Zuo</span>  <span class="cit-name-given-names">J.R.</span></span>, Niu, Q.W., and Chua, N.H. <span class="cit-pub-date">2000</span>. An estrogen receptor-based transactivator XVE mediates highly inducible gene expression in transgenic plants. <span class="cit-source">Plant J.</span> <span class="cit-vol">24</span>: <span class="cit-fpage">265</span>–273.</cite></div>
                           <div class="cit-extra"><a href="/external-ref?access_num=10.1046/j.1365-313x.2000.00868.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="/external-ref?access_num=11069700&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="/external-ref?access_num=000089935900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                  </ol></div><span class="highwire-journal-article-marker-end"/></div>
  </body>
</html>
