<?xml version="1.0"?>
<html>
  <body>
    <div itemprop="articleBody">
      <span class="highwire-journal-article-marker-start"/>
      <div class="hw-optimized-content">
                  <div class="auto-clean">
                     
                     
                     
                     
                     <span style="  font-size: 83.33%"><em>The Plant Cell</em> 18:626-638 (2006)<br/>
                        © 2006 <a href="/misc/terms.shtml">American Society of Plant Biologists</a><br/></span></div>
                  <h1 id="article-title-1" itemprop="headline">
                     The <em>Arabidopsis</em> SOMATIC EMBRYOGENESIS RECEPTOR-LIKE KINASE1 Protein Complex Includes BRASSINOSTEROID-INSENSITIVE1<a name="RFN1" id="RFN1"/><sup><a href="#FN1">[W]</a></sup></h1>
                  <div class="contributors">
                     Rumyana Karlova<sup>a</sup><sup>,b</sup>, 
                     Sjef Boeren<sup>a</sup>, 
                     Eugenia Russinova<sup>a</sup>, 
                     José Aker<sup>a</sup>, 
                     Jacques Vervoort<sup>a</sup> and 
                     Sacco de Vries<sup>a</sup><sup>,1</sup></div>
                  <p>
                     <span style="  font-size: 83.33%"><sup>a</sup> Laboratory of Biochemistry, Wageningen University, 6703 HA Wageningen, The Netherlands<br/><sup>b</sup> Center for BioSystems Genomics, 6700 AB Wageningen, The Netherlands
                        </span></p>
                  <p>
                     
                     <span style="  font-size: 83.33%"><sup><sup>1</sup></sup> To whom correspondence should be addressed. E-mail <span class="em-link"><span class="em-addr">sacco.devries{at}wur.nl</span></span>; fax 31-317-484801.</span></p>
                  <p>
                     <a name="ABS" id="ABS"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        ABSTRACT
                        
                        
                     </h2>
                     
                  </div>
                  <div class="section abstract" id="abstract-1" itemprop="description">
                     
                     <p>
                        <em>Arabidopsis thaliana</em> SOMATIC EMBRYOGENESIS RECEPTOR-LIKE KINASE1 (SERK1) is a leucine-rich repeat receptor-like kinase (LRR-RLK) involved in the
                        acquisition of embryogenic competence and in male sporogenesis. To determine the composition of the SERK1 signaling complex
                        in vivo, we generated plants expressing the SERK1 protein fused to cyan fluorescent protein under SERK1 promoter control.
                        The membrane receptor complex was immunoprecipitated from seedlings, and the coimmunoprecipitating proteins were identified
                        using liquid chromatography/matrix-assisted laser desorption ionization–time of flight/mass spectrometry of the trypsin-released
                        peptides. This approach identified two other LRR-RLKs, the BRASSINOSTEROID-INSENSITIVE1 (BRI1) receptor and its coreceptor,
                        the SERK3 or BRI1-ASSOCIATED KINASE1 protein. In addition, KINASE-ASSOCIATED PROTEIN PHOSPHATASE, CDC48A, and 14-3-3ν were
                        found. Finally, the MADS box transcription factor AGAMOUS-LIKE15 and an uncharacterized zinc finger protein, a member
                        of the CONSTANS family, were identified as part of the SERK1 complex. Using blue native gel electrophoresis, we show that
                        SERK1 and SERK3 are part of BRI1-containing multiple protein complexes with relative masses between 300 and 500 kD. The SERK1
                        mutant allele <em>serk1-1</em> enhances the phenotype of the weak BRI1 allele <em>bri1-119</em>. Collectively, these results suggest that apart from SERK3, SERK1 is also involved in the brassinolide signaling pathway.
                        
                        
                     </p>
                     
                  </div>
                  <div class="auto-clean">
                     
                     
                     <a name="SEC1" id="SEC1"/>
                     
                     
                  </div>
                  <div class="section-nav">
                     
                     <h2>
                        INTRODUCTION
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     Membrane-located leucine-rich repeat receptor-like kinases (LRR-RLKs) play important roles in plant signaling pathways. For
                     example, the <em>Arabidopsis thaliana</em> CLAVATA1 (CLV1), BRASSINOSTEROID-INSENSITIVE1 (BRI1), ERECTA, rice (<em>Oryza sativa</em>) Xa21, HAESA-RLK5, and FLS2 genes were shown to function in shoot meristem maintenance, hormone perception, organ elongation,
                     disease resistance, abscission, and flagellin signaling, respectively (reviewed in Torii, 2004<a href="#BIB42"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Extensive genetic and biochemical studies have been undertaken to identify additional components of different signaling
                     pathways. For example, negative and positive regulators of brassinosteroid (BR) signaling were identified as the GSK-3/Shaggy-like
                     kinase called BRASSINOSTEROID-INSENSITIVE2 (BIN2) (Li and Nam, 2002<a href="#BIB17"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) and the nucleus-localized Ser/Thr phosphatase <em>bri1</em> Suppressor1 family (Mora-Garcia et al., 2004<a href="#BIB20"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), respectively. Their potential substrates are the nuclear proteins BRASSINAZOLE-RESISTANT1 (BZR1) (Wang et al., 2002b<a href="#BIB48"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) and <em>bri1-</em>EMS-SUPPRESSOR1 (BES1) (Yin et al., 2002<a href="#BIB50"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     We previously identified <em>Arabidopsis</em> SOMATIC EMBRYOGENESIS RECEPTOR-LIKE KINASE1 (SERK1), an LRR-RLK that marks the formation of embryogenic cells in culture
                     and is expressed in ovule primordia and in both male and female gametophytes. In sporophytic tissues, SERK1 has a complex
                     expression pattern, with expression being highest in the vascular tissue of all organs (Hecht et al., 2001<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Albrecht et al., 2005<a href="#BIB1"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Kwaaitaal et al., 2005<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Ectopic expression of <em>SERK1</em> does not result in an obvious phenotype in <em>Arabidopsis</em> plants but increases somatic embryo formation in culture (Hecht et al., 2001<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). SERK1 is a member of a small family of five related RLKs, all of which have five LRRs and a typical Ser-Pro–rich juxtamembrane
                     region (Hecht et al., 2001<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). The SERK1 knockout alleles <em>serk1-1</em> and <em>serk1-2</em> did not have a morphological phenotype but in combination with a <em>serk2</em> null mutant resulted in complete male-sterile plants (Albrecht et al., 2005<a href="#BIB1"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Colcombet et al., 2005<a href="#BIB4"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). SERK3 or BRI1-ASSOCIATED RECEPTOR KINASE1 (BAK1) is part of the BR receptor complex and is proposed to function as a coreceptor
                     of the BRI1 protein (Li et al., 2002<a href="#BIB18"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Nam and Li, 2002<a href="#BIB23"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Recently, we showed that in living plant cells BRI1 and SERK3 interaction occurred in restricted areas of the membrane
                     and recycled upon internalization by endocytosis (Russinova et al., 2004<a href="#BIB36"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     SERK1 was shown to interact with the KINASE-ASSOCIATED PROTEIN PHOSPHATASE (KAPP) that is postulated to play a role in receptor
                     internalization (Shah et al., 2002<a href="#BIB39"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Using a yeast two-hybrid screen, two additional SERK1-interacting proteins, CDC48A and 14-3-3λ, were found (Rienties et
                     al., 2005<a href="#BIB32"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). These interactions suggested analogy with the mammalian CDC48 homologue p97/VCP complex, in which p97/VCP can be phosphorylated
                     by the JAK-2 kinase and dephosphorylated by the phosphatase PTPH1 that associates with a 14-3-3 protein (Zhang et al., 1997<a href="#BIB51"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>, 1999<a href="#BIB52"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). KAPP has also been reported to interact with other LRR-RLKs such as HAESA, WALL-ASSOCIATED KINASE1, FLS2, and CLV1 (reviewed
                     in Becraft, 2002<a href="#BIB3"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Transgenic studies indicate that KAPP functions as a negative regulator of CLV1 and FLS2 signaling (Williams et al., 1997<a href="#BIB49"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Stone et al., 1998<a href="#BIB41"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Gomez-Gomez et al., 2001<a href="#BIB9"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     Identification of proteins can now be performed at high sensitivity by specific proteolytic digestion and determination of
                     the peptide masses by mass spectrometry. This technique has been used for a systematic study of multiprotein complexes. For
                     the epidermal growth factor receptor, a human receptor Tyr kinase, immunoprecipitation combined with matrix-assisted laser
                     desorption ionization–time of flight/mass spectrometry (MALDI-TOF/MS) was used to identify components of the epidermal growth
                     factor receptor signaling complex (Pandey et al., 2000<a href="#BIB26"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). For only a few plant RLKs, the signaling complex has been defined. One example is the 105-kD CLV1 receptor that is found
                     in two distinct protein complexes of 450 and 185 kD. The larger 450-kD complex requires functional CLV1 and CLV3 proteins
                     for assembly and includes KAPP and a Rho GTPase-related protein (Trotochaud et al., 1999<a href="#BIB43"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     In this study, we have determined the composition of the SERK1 complex(es) in vivo. We combined immunoprecipitation of cyan
                     fluorescent protein (CFP)–tagged SERK1 with rapid liquid chromatography (LC)/MALDI-TOF/MS–based protein identification. Using
                     this method, we confirmed the presence in the SERK1 signaling complex of CDC48A (Rienties et al., 2005<a href="#BIB32"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) and KAPP (Shah et al., 2002<a href="#BIB39"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Additional proteins were also found, such as another member of the 14-3-3 family, 14-3-3ν, SERK3 (BAK1) and BRI1, the MADS
                     box transcription factor AGAMOUS-LIKE15 (AGL15), and an uncharacterized zinc finger protein.
                     In particular, the interaction between SERK1 and BRI1 was confirmed by a genetic experiment and fluorescence lifetime imaging
                     microscopy (FLIM) to determine Förster resonance energy transfer (FRET) between fluorescently tagged receptors. Our data show
                     that the method used here can distinguish between individual isoforms of related members of the same protein family, confirming
                     and extending previous screens performed using the more commonly applied yeast two-hybrid method to find interacting partners.
                     
                     
                  </p>
                  <p>
                     
                     <a name="SEC2" id="SEC2"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        RESULTS
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     <strong>Immunoprecipitation and Detection of SERK1-CFP Protein</strong><br/>
                     
                     To purify and detect the SERK1 protein and its interacting partners directly from seedlings, C-terminal SERK1-CFP fusion proteins
                     were stably expressed in <em>Arabidopsis</em> wild-type plants under the control of the SERK1 promoter as described (Kwaaitaal et al., 2005<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). The expression pattern of SERK1-CFP was identical to that of a previously characterized SERK1–yellow fluorescent protein
                     (YFP) fusion protein (Kwaaitaal et al., 2005<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Similar to SERK1-YFP–expressing plants, no unusual phenotype was observed, whereas introduction of the same SERK1 promoter-SERK1-CFP
                     construct used here in the SERK1 knockout mutant background restored the male sterility phenotype of the double <em>serk1-1 serk2-2</em> mutant (Albrecht et al., 2005<a href="#BIB1"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). This finding confirms that the fusion protein did not interfere with its endogenous counterpart and was fully functional.
                     Total proteins extracted from 7-d-old SERK1-CFP–expressing seedlings were separated by SDS-PAGE, and after blotting on a membrane,
                     the fusion proteins were detected with anti–green fluorescent protein (GFP) antibody (<a href="#FIG1">Figure 1</a>, lane 1). The same protein extract was detected with anti-GST-SERK1 kinase antibodies, which recognize endogenous SERK1 as
                     well as the SERK1-CFP fusion proteins, confirming the specificity of the anti-GFP antibodies (<a href="#FIG1">Figure 1</a>, lane 2). After immunoprecipitation, an aliquot was analyzed with anti-GFP antibody to confirm that SERK1-CFP was indeed
                     recovered in the immunoprecipitation. A band of 100 kD was recognized only in protein extracts of the transgenic line (<a href="#FIG1">Figure 1</a>, lane 4) and not in extracts of wild-type seedlings (<a href="#FIG1">Figure 1</a>, lane 3). 
                     
                  </p>
                  <p>
                     
                     <a name="FIG1" id="FIG1"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="626/F1.expansion.html"><img src="626/F1.small.gif"/></a><br/><strong>View larger version</strong> (31K):<br/><a href="626/F1.expansion.html">In this window</a><br/><a href="626/F1.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 1.</strong></span> Detection of SERK1 and SERK1-CFP from Wild-Type and Transgenic Plants.
                        
                        <p>
                           
                           Total protein extracts from SERK1-CFP–transformed seedlings and wild-type seedlings were immunoprecipitated with anti-GFP
                           antibodies. Thirty micrograms of total protein extracted from SERK1-CFP seedlings (lanes 1 and 2) and the immunoprecipitation
                           products from SERK1-CFP seedlings (lane 4) and wild-type seedlings (lane 3) were separated on a 10% SDS-PAGE gel. The blots
                           were incubated with anti-GFP antibodies (lanes 1, 3, and 4) or with anti–GST-SERK1 kinase antibodies (lane 2). The sizes of
                           the marker proteins are indicated by arrows.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     <strong>LC/MALDI-TOF/MS Identification of SERK1 and Its Interacting Partners in Vivo</strong><br/>
                     
                     To identify proteins that bind to SERK1 as extracted from SERK1-CFP–expressing seedlings, we subjected the immunoprecipitated
                     proteins (bound to the protein A–Sepharose beads) to trypsin digestion. To reduce the influence of ion-suppression effects
                     in MALDI-TOF/MS measurements and obtain more peptide peaks, separation of the tryptic peptides by reverse-phase column chromatography
                     was performed before subjection to MALDI-TOF/MS measurements. As a negative control, samples obtained from an immunoprecipitated
                     protein extract from wild-type seedlings with anti-GFP antibody were analyzed in the same way by LC/MALDI-TOF/MS. 
                     
                  </p>
                  <p>
                     
                     A representative LC/MALDI-TOF/MS spectrum, recorded for fraction 9 from the 24 LC fractions obtained after trypsin digestion
                     of the immunoprecipitated SERK1 complex, is shown in <a href="#FIG2">Figure 2A</a>. In <a href="#FIG2">Figure 2B</a>, a spectrum recorded for fraction 9 of the trypsin-digested immunoprecipitated negative control extract is shown. All mass-to-charge
                     ratio (<em>m</em>/<em>z</em>) values also present in the negative control (i.e., values belonging to protein A, GFP antibodies, and proteins aspecifically
                     immunoprecipitated by the GFP antibodies) were removed from the <em>m</em>/<em>z</em> list. The remaining values were then compared with the predicted tryptic peptide masses for the <em>Arabidopsis</em> proteome as described in Methods. The top panels in <a href="#FIG2">Figures 2C to 2F</a> show four representative spectra, expanded from the spectrum shown in <a href="#FIG2">Figure 2A</a>, that led to the identification of the SERK1-interacting partner, 14-3-3ν. The peaks with <em>m</em>/<em>z</em> 1492.7 (<a href="#FIG2">Figure 2C</a>) and 1611.8 (<a href="#FIG2">Figure 2E</a>) are predicted with one phosphorylated residue present. The peak with <em>m</em>/<em>z</em> 1531.8 (<a href="#FIG2">Figure 2D</a>) has the same predicted peptide sequence as that predicted for <em>m</em>/<em>z</em> 1611.8 (<a href="#FIG2">Figure 2E</a>) but without phosphorylation of the Ser residue. The same peptide sequence, but now extended by two missed tryptic cleavage
                     sites, is predicted for the peak with <em>m</em>/<em>z</em> 1716.8 (<a href="#FIG2">Figure 2F</a>). The bottom panels in <a href="#FIG2">Figures 2C to 2F</a> show the corresponding expanded spectra recorded for the negative control sample shown in <a href="#FIG2">Figure 2B</a>. 
                     
                  </p>
                  <p>
                     
                     <a name="FIG2" id="FIG2"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="626/F2.expansion.html"><img src="626/F2.small.gif"/></a><br/><strong>View larger version</strong> (29K):<br/><a href="626/F2.expansion.html">In this window</a><br/><a href="626/F2.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 2.</strong></span> LC/MALDI-TOF/MS Spectra Obtained after Trypsin Digestion of SERK1-GFP Immunoprecipitated Complex.
                        
                        <p>
                           
                           <strong>(A)</strong> and <strong>(B)</strong> MALDI-TOF/MS spectra from SERK1-CFP immunoprecipitated complex <strong>(A)</strong> (fraction 9 of 24) compared with the corresponding spectra (fraction 9) from the control immunoprecipitation performed with
                           wild-type plants <strong>(B)</strong>. The spectra were obtained after LC separation from the trypsin-digested proteins after immunoprecipitation. The <em>x</em> axis represents <em>m</em>/<em>z</em> values. The <em>y</em> axis represents the intensity of the peaks in arbitrary units.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(C)</strong> to <strong>(F)</strong> Four representative spectra expanded from <strong>(A)</strong> (top panels) and <strong>(B)</strong> (bottom panels), which led to the identification of one of the SERK1-interacting partners, 14-3-3ν. The corresponding predicted
                           peptides are also shown. The peaks are present only in the spectra from SERK1-CFP immunoprecipitation
                           (top panels) and not in the controls (bottom panels).
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     The combined results of comparable analyses performed for the list of remaining <em>m</em>/<em>z</em> values (see supplemental material online) were used to compile the list of identified proteins; these are presented in <a href="#TBL1">Table 1</a>. As expected, SERK1 was identified as part of the complex. This identification was based on 13 tryptic peptide matches and
                     a Mascot score of 84. All of the peptides found covered the kinase domain of SERK1 and are located on the exterior of the
                     protein, including the phosphorylated peptides, based on the structural model for SERK1 (Shah et al., 2001b<a href="#BIB40"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). None of the peptides matched the sequence of the extracellular domain, which may be attributable to posttranslational modification
                     by glycosylation (Shah et al., 2001a<a href="#BIB38"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) or to the presence of two disulfide bounds between the four Cys residues present, making trypsin digestion more difficult.
                     One peptide could be identified in which the oxidation of Met had occurred, whereas three other peptides were identified that
                     were modified by phosphorylation. The second receptor in the SERK1 complex was identified as the SERK3 or BAK1 protein. LC/MALDI-TOF/MS
                     fingerprint analysis revealed tryptic peptide <em>m</em>/<em>z</em> values from both SERK1 and SERK3, but because of the high homology in their protein sequences (78%), 8 of 13 of the predicted
                     <em>m</em>/<em>z</em> values match with the sequences of both proteins. Five peptides unique for SERK3 were identified, three without and two with
                     modification by phosphorylation. Based on inspection of the model of SERK1 structure, four of the five SERK3-specific peptides
                     identified are from a region also yielding peptides used for the identification of SERK1. This finding corroborates the identification
                     of SERK1 and SERK3. 
                     
                  </p>
                  <p>
                     
                     <a name="TBL1" id="TBL1"/>
                     
                     
                  </p>
                  <div class="table pos-float">
                     
                     <div class="table-inline">
                        <strong>View this table:</strong><br/><a href="626/T1.expansion.html">In this window</a><br/><a href="626/T1.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="table-caption">
                        <span class="fig-label"><strong>Table 1.</strong></span> SERK1-Interacting Proteins Identified by MALDI-TOF/MS 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     For SERK2, which is 90% homologous with SERK1 at the protein level, five <em>m</em>/<em>z</em> values are identical for both proteins. No <em>m</em>/<em>z</em> values unique for SERK2 were found. The third receptor found in the SERK1 complex was the BRI1 receptor (Li and Chory, 1997<a href="#BIB16"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) identified on the basis of 17 peptides, most of them covering the kinase domain. Most of the peptides identified with phosphorylations
                     were also found as nonphosphorylated or as oxidized peptides, which provides additional evidence for identification as the
                     BRI1 protein (see Supplemental Table 1 online). Because SERK3 (BAK1) is reported to be the coreceptor of BRI1 (Li et al.,
                     2002<a href="#BIB18"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Nam and Li, 2002<a href="#BIB23"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), the presence of both of these receptors suggests that there is a direct link between SERK1 activity and brassinolide signaling.
                     In addition to SERK1, SERK3, and BRI1, several other proteins were found that were previously reported to interact with SERK1
                     and therefore were expected to be present in the SERK1 immunoprecipitated complex. One of them is <em>Arabidopsis</em> KAPP, which was shown to interact only with a catalytically active kinase domain of SERK1, involving Thr-468 (Shah et al.,
                     2001b<a href="#BIB40"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     In a yeast two-hybrid screen, two other proteins were found to interact with the kinase domain of SERK1, CDC48A and 14-3-3λ
                     (Rienties et al., 2005<a href="#BIB32"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). MALDI-TOF/MS fingerprint analysis indeed revealed tryptic peptides derived from CDC48A (<a href="#TBL1">Table 1</a>). In our search, 14-3-3ν, rather then 14-3-3λ, was identified with a high score after searching even with the complete <em>m</em>/<em>z</em> list. Apparently, only these two members of the large <em>Arabidopsis</em> family of 14-3-3 proteins can interact with the SERK1 receptor. Finally, two unexpected proteins were found. The first is
                     the MADS box transcription factor AGL15 (Heck et al., 1995<a href="#BIB13"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), and the second is a putative CONSTANS (CO)-like B-box zinc finger protein (At2g33500). The zinc finger protein is annotated
                     as one large 401–amino acid protein and also as a shorter protein of 186 amino acids. We only found peptides covering the
                     shorter protein. To validate that the proteins described above indeed interacted with SERK1, we treated control samples with
                     SDS or with 0.5 M NaCl before coimmunoprecipitation to destroy protein–protein interactions. The resulting list of <em>m</em>/<em>z</em> values revealed only the presence of the SERK1 protein (data not shown). Finally, we used our coimmunoprecipitation procedure
                     with the GFP-tagged BRI1 receptor and the GFP-tagged integral membrane protein PIN1. For BRI1, the resulting <em>m</em>/<em>z</em> lists did not reveal the presence of any of the SERK1-interacting proteins listed in <a href="#TBL1">Table 1</a> (data not shown). Therefore, we conclude that the coimmunoprecipitation/LC/MALDI-TOF/MS procedure for identifying protein
                     complexes in <em>Arabidopsis</em> presented here is reliable and identifies specific interacting partners. 
                     
                  </p>
                  <p>
                     
                     <strong>BRI1, SERK3, CDC48A, and 14-3-3ν Interact in Vivo with SERK1</strong><br/>
                     
                     To confirm our results obtained by LC/MALDI-TOF/MS, we used coimmunoprecipitation and FLIM-based analysis to detect the interaction
                     between SERK1 and its interacting partners in vivo. CFP-tagged SERK1 was immunoprecipitated with anti-GFP antibodies, and
                     the precipitated proteins were detected using anti-BRI1 antibodies. A band with the expected mobility of BRI1 was visualized
                     (<a href="#FIG3">Figure 3A</a>, lane 1) that was absent in an immunoprecipitation of wild-type plants with the GFP antibody (<a href="#FIG3">Figure 3A</a>, lane 2). BRI1 antibodies recognize endogenous BRI1 as well as BRI1-GFP fusion proteins and can precipitate BRI1 from a total
                     plant protein extract (<a href="#FIG3">Figure 3B</a>, lanes 1 to 3). The BRI1 antibodies are specific for BRI1 because they do not detect a band of the expected mobility for
                     BRI1 in a protein extract from seedlings of the strong <em>bri1-4</em> allele (data not shown). Reciprocal coimmunoprecipitation with anti-BRI1 antibodies revealed the presence of SERK1-CFP in
                     the coimmunoprecipitated proteins (<a href="#FIG3">Figure 3C</a>, lane 1). 
                     
                  </p>
                  <p>
                     
                     <a name="FIG3" id="FIG3"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="626/F3.expansion.html"><img src="626/F3.small.gif"/></a><br/><strong>View larger version</strong> (47K):<br/><a href="626/F3.expansion.html">In this window</a><br/><a href="626/F3.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 3.</strong></span> Direct Interaction of SERK1 with BRI1 and SERK3.
                        
                        <p>
                           
                           <strong>(A)</strong> Total protein extract from SERK1-CFP transgenic seedlings was immunoprecipitated using anti-GFP antibodies (lane 1). As a
                           negative control, proteins extracted from wild-type plants were precipitated using anti-GFP antibodies (lane 2). WB, protein
                           gel blot.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(B)</strong> Total protein extracts from BRI1-GFP transgenic seedlings (lane 1) and wild-type (lane 2) and immunoprecipitated (lane 3)
                           products from wild-type seedlings were separated on an 8% SDS-PAGE gel and detected using anti-BRI1 antibodies.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(C)</strong> A protein extract from SERK1-CFP transgenic plants was precipitated using anti-BRI1 antibodies (lane 1). As a negative control,
                           a protein extract from wild-type plants was precipitated with the same antibodies (lane 2).
                           
                        </p>
                        
                        <p>
                           
                           The resulting immunoprecipitates were separated by 8% SDS-PAGE, transferred to a membrane, and probed with anti-GFP antibodies
                           for detection of SERK1-CFP fusion proteins <strong>(C)</strong> or with anti-BRI1 antibodies for detection of BRI1 protein (<strong>[A]</strong> and <strong>[B]</strong>). Arrows indicate the sizes of the marker proteins. IP, immunoprecipitate.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(D)</strong> FRET between SERK1, SERK3, and BRI1 imaged by FLIM on cowpea protoplasts transiently expressing SERK1-CFP (I and IV), BRI1-YFP/SERK1-CFP
                           (II and V), and SERK3-CFP/SERK1-YFP (III and VI) for 16 h. Fluorescence intensity images representing steady states of the
                           donor SERK1-CFP or SERK3-CFP fluorescence are presented in I to III. Mean fluorescence lifetime values (τ) and lifetime distribution
                           for the images in I to III are presented as pseudocolor images in IV to VI, respectively. The
                           arrowhead points to an area with a short lifetime, indicative of FRET. Note the color bar, where dark blue is used to show
                           a τ of 2.5 ns (no interaction) and light green is used to show a τ of 1.9 ns (interaction). The red to dark orange areas corresponding
                           to a τ of 1.5 ns are attributable to autofluorescence effects.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     Previously, we showed that FRET can be detected between CFP and YFP of SERK3 and BRI1 when transiently expressed in cowpea
                     (<em>Vigna unguiculata</em>) protoplasts. Coexpression of BRI1 and SERK3 results in a change of the steady state distribution of both receptors as a
                     result of accelerated endocytosis (Russinova et al., 2004<a href="#BIB36"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). To determine whether SERK1 can form heterodimers with either BRI1 or SERK3 in the plasma membrane, we used a similar transient
                     assay in protoplasts to detect FRET by FLIM. In <a href="#FIG3">Figure 3D</a>, we show the fluorescence intensity images of protoplasts transfected with SERK1-CFP alone (<a href="#FIG3">Figure 3D, I</a>), SERK1-CFP together with BRI1-YFP (<a href="#FIG3">Figure 3D, II</a>), and SERK1-YFP together with SERK3-CFP (<a href="#FIG3">Figure 3D, III</a>). The corresponding fluorescence lifetime images are compared in panels IV to VI in <a href="#FIG3">Figure 3D</a>. The results presented in <a href="#FIG3">Figure 3D</a> (panel V) show that SERK1 and BRI1 heterodimerize in small patches in the plasma membrane, as indicated by the significant
                     reduction in the average lifetime (τ) from 2.5 to 2.1 ns of the donor (SERK1-CFP) molecules in those areas indicated by the
                     arrowhead. A similar phenomenon was
                     described for the SERK3/BRI1 heterodimers (Russinova et al., 2004<a href="#BIB36"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). SERK1-YFP also formed heterodimers with SERK3-CFP in the plasma membrane, as shown by the green plasma membrane in <a href="#FIG3">Figure 3D</a> (panel VI) that corresponds to an average lifetime of 1.9 ns for the donor (SERK3-CFP) molecules. Thus, colocalization in
                     the plasma membrane and the ability to form heterodimers in plant protoplasts strongly suggest that SERK1, SERK3, and BRI1
                     can be part of the same complex in intact plant membranes. 
                     
                  </p>
                  <p>
                     
                     To confirm the in vivo interaction of CDC48A with SERK1, we coimmunoprecipitated SERK1 from <em>Arabidopsis</em> tissue-cultured cells with anti-GST-SERK1 kinase antibodies (Rienties et al., 2005<a href="#BIB32"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). The precipitated proteins were analyzed by immunoblotting using anti-CDC48 antibodies (Rancour et al., 2002<a href="#BIB30"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). A band with the expected mobility for monomeric CDC48A was detected in the total protein extract (<a href="#FIG4">Figure 4A</a>, lane 1) and in the immunoprecipitated sample (<a href="#FIG4">Figure 4A</a>, lane 3), showing that <em>Arabidopsis</em> CDC48A indeed coimmunoprecipitates with SERK1. No CDC48A band was detected in the negative control using rabbit preimmune
                     serum for immunoprecipitation (<a href="#FIG4">Figure 4A</a>, lane 2). Similarly, the presence of a 14-3-3 protein in the complex with SERK1 in vivo was verified by coimmunoprecipitation
                     with GFP antibodies of CFP-tagged SERK1 followed by immunodetection with anti–14-3-3 antibodies (<a href="#FIG4">Figure 4B</a>, lane 1). No 14-3-3 band was detected in the negative control using wild-type seedlings for immunoprecipitation (<a href="#FIG4">Figure 4B</a>, lane 2). The 14-3-3 antiserum can recognize in vitro–produced 14-3-3ν (data not shown) as well as 14-3-3λ, confirming that
                     SERK1 can interact with both 14-3-3λ (Rienties et al., 2005<a href="#BIB32"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) and 14-3-3ν isoforms. A summary of the experiments providing additional evidence for the presence of proteins in the SERK1
                     complex is
                     shown in <a href="#TBL1">Table 1</a>. 
                     
                  </p>
                  <p>
                     
                     <a name="FIG4" id="FIG4"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="626/F4.expansion.html"><img src="626/F4.small.gif"/></a><br/><strong>View larger version</strong> (19K):<br/><a href="626/F4.expansion.html">In this window</a><br/><a href="626/F4.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 4.</strong></span> SERK1 Interacts in Vivo with <em>Arabidopsis</em> CDC48A and 14-3-3.
                        
                        <p>
                           
                           <strong>(A)</strong> Coimmunoprecipitation of SERK1 and CDC48A. Total protein extract was isolated from <em>Arabidopsis</em> tissue cell culture, analyzed directly by SDS-PAGE, and immunodetected with anti-CDC48 antibody (lane 1), or immunoprecipitated
                           with anti-SERK1 kinase antibody (lane 3) or preimmune serum (lane 2) and detected with anti-CDC48 antibody.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(B)</strong> Coimmunoprecipitation of SERK1 and 14-3-3. Total protein extract was isolated from SERK1-CFP transgenic plants (lane 1) or
                           wild-type plants (lane 2) and precipitated with anti-GFP antibody. Precipitated proteins were immunodetected with anti–14-3-3
                           antibodies.
                           
                        </p>
                        
                        <p>
                           
                           The sizes of the marker proteins are indicated by arrows. IP, immunoprecipitate.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     <strong>SERK1 Interacts Genetically with BRI1</strong><br/>
                     
                     Analysis of the single T-DNA insertion mutant <em>serk1-1</em>, an allele that produces a kinase-dead receptor protein, did not reveal a morphological plant phenotype (Albrecht et al.,
                     2005<a href="#BIB1"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Therefore, we crossed the <em>serk1-1</em> allele into the weak BRI1 mutant allele <em>bri1-119</em>. This allele contains the G644D missense mutation in the island domain of BRI1 (Friedrichsen et al., 2000<a href="#BIB7"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). At the seedling level, 7-d-old <em>bri1-119</em> mutants exhibit mild morphological characteristics of the phenotype conferred by <em>bri1</em>, such as small seedlings, short petioles, and round, dark green, inwardly curved true leaves (<a href="#FIG5">Figure 5A</a>). <em>bri1-119</em> plants developed smaller, compact rosettes (<a href="#FIG5">Figure 5D</a>) compared with wild-type plants. The weak <em>bri1-119</em> allele is a semifertile dwarf that has less reduced stature (<a href="#FIG5">Figure 5D</a>) compared with severe <em>bri1</em> alleles (Noguchi et al., 1999<a href="#BIB24"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). The <em>serk1-1</em> mutation clearly enhanced all BRI1-associated phenotypes, especially the length of the petioles and excessive inward curling,
                     now seen in 7-d-old seedlings of the double mutant <em>bri1-119 serk1-1</em> (<a href="#FIG5">Figure 5B</a>), as well as a reduction in the size of the rosette (<a href="#FIG5">Figure 5E</a>; see Supplemental Table 2 online). The inflorescence length of the double <em>bri1-119 serk1-1</em> mutant was also reduced (<a href="#FIG5">Figure 5H</a>; see Supplemental Table 2 online). For comparison, we also introduced the <em>serk3</em> mutation (Russinova et al., 2004<a href="#BIB36"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), an allele of the BAK1 coreceptor, into <em>bri1-119</em>. The resulting <em>bri1-119 serk3</em> double mutant had even smaller rosettes (<a href="#FIG5">Figure 5F</a>) and grew with a severely reduced stature (<a href="#FIG5">Figure 5I</a>; see Supplemental Table 2 online) resembling strong <em>bri1</em> alleles. This finding indicates that the <em>serk3</em> mutation had an even stronger effect on <em>bri1-119</em> than did <em>serk1-1</em>. To rule out the possibility that the observed modification of the phenotype conferred by <em>bri1-119</em> by the <em>serk1-1</em> allele is attributable to the introduction of the Columbia ecotype into the Enkheim-2 background, we also combined the <em>serk2-2</em> mutant allele, which shows no morphological alterations from the wild type (Albrecht et al., 2005<a href="#BIB1"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), with <em>bri1-119</em> (see Supplemental Table 2 online). The resulting <em>bri1-119 serk2-2</em> double mutants were indistinguishable from <em>bri1-119</em> in terms of inflorescence length and rosette width, demonstrating that there is no effect of the Columbia ecotype. These
                     results provide genetic evidence that SERK1 is involved in BRI1 signaling and confirm the role of the <em>SERK3</em> (<em>BAK1</em>) gene as a BRI1 coreceptor. 
                     
                  </p>
                  <p>
                     
                     <a name="FIG5" id="FIG5"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="626/F5.expansion.html"><img src="626/F5.small.gif"/></a><br/><strong>View larger version</strong> (84K):<br/><a href="626/F5.expansion.html">In this window</a><br/><a href="626/F5.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 5.</strong></span> SERK1, SERK3, and BRI1 Function Together.
                        
                        <p>
                           
                           <em>serk1-1</em> and <em>serk3</em> mutations enhance the weak <em>bri1-119</em> mutation. Seven-day-old seedlings are shown in <strong>(A)</strong> to <strong>(C)</strong>, 3-week-old plants are shown in <strong>(D)</strong> to <strong>(F)</strong>, and 6-week-old plants are shown in <strong>(G)</strong> to <strong>(I)</strong>. <strong>(A)</strong>, <strong>(D)</strong>, and <strong>(G)</strong> show <em>bri1-119</em>, <strong>(B)</strong>, <strong>(E)</strong>, and <strong>(H)</strong> show <em>bri1-119 serk1-1</em>, and <strong>(C)</strong>, <strong>(F)</strong>, and <strong>(I)</strong> show <em>bri1-119 serk3</em> mutations. Note the increase in the curliness of the first true leaves and the reduction in the petiole length in the double
                           mutant seedlings as well as the reduced rosette width of the double mutant plants. Bars = 1 mm in <strong>(B)</strong> and <strong>(C)</strong> and 2 cm in <strong>(F)</strong>.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     <strong>Identification of a Membrane-Associated SERK1-Containing Complex Using Blue Native PAGE</strong><br/>
                     
                     In previous studies, we showed that SERK1 is located at the membrane in protoplasts and in plants (Russinova et al., 2004<a href="#BIB36"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Kwaaitaal et al., 2005<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) and can directly interact in vitro and in vivo with KAPP and 14-3-3λ and in vitro with CDC48A (Shah et al., 2002<a href="#BIB39"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Rienties et al., 2005<a href="#BIB32"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). The results presented here confirm the previously observed interactions and show that SERK1 can be present in a complex
                     with several other proteins. It is not clear whether the SERK1 complex harbors all of the proteins listed in <a href="#TBL1">Table 1</a> at the same time or that we immunoprecipitated multiple forms of the SERK1 complex. Different forms could exist in different
                     cell types or in different subcellular locations, such as plasma membrane or after internalization (Russinova et al., 2004<a href="#BIB36"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). To determine the size of the SERK1 complex in planta, microsomal preparations from SERK1-CFP– and BAK1-GFP–expressing seedlings
                     and wild-type seedlings were solubilized with Triton X-100 and subjected to blue native (BN) PAGE. In this technique, Coomassie
                     dyes and aminocaproic acid are used to induce a charge shift and improve the solubilization of membrane-associated proteins
                     (Rivas et al., 2002<a href="#BIB33"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). To detect SERK1 within the potential protein complex, single lanes of the BN-PAGE gel were cut and mounted on a denaturing
                     SDS gel. After electrophoresis, the gel was subjected to immunoblot analysis using anti-GFP antibody or anti-GST-SERK1 kinase
                     antibody. Using mass standards, ∼85% of the protein complex containing SERK1 was determined to have a mean relative mass of
                     350 kD. The remaining 15% appears
                     to exist as a complex with a mean mass of 450 kD. The maximum size is ∼500 kD (<a href="#FIG6">Figure 6A</a>). This variability in protein complex size is also seen when the tagged SERK1-CFP complex is detected by anti-GFP antibodies
                     (<a href="#FIG6">Figure 6B</a>). Approximately 90% of the CFP-tagged complex has a mean mass of 400 kD and 10% has a mass of ∼500 kD. Although there is
                     a clear difference in the mean size of the complex with and without the CFP tag, we cannot precisely
                     estimate whether this difference is 25 kD (one molecule of CFP protein) or 50 kD (two molecules of CFP protein). 
                     
                  </p>
                  <p>
                     
                     <a name="FIG6" id="FIG6"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="626/F6.expansion.html"><img src="626/F6.small.gif"/></a><br/><strong>View larger version</strong> (56K):<br/><a href="626/F6.expansion.html">In this window</a><br/><a href="626/F6.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 6.</strong></span> BN-PAGE of Solubilized Wild-Type and SERK1-CFP Microsomes.
                        
                        <p>
                           
                           Microsomes were isolated from wild-type plants (<strong>[A]</strong> and <strong>[E]</strong>) and SERK1-CFP–expressing (<strong>[B]</strong> and <strong>[C]</strong>) or BAK1-GFP–expressing <strong>(D)</strong> transgenic plants and subjected to BN-PAGE in the first dimension and SDS-PAGE in the second dimension. Proteins were immunodetected
                           using anti-SERK1 kinase antibodies <strong>(A)</strong>, anti-GFP antibodies (<strong>[B]</strong>, <strong>[D]</strong>, and <strong>[E]</strong>), or anti-BRI1 antibodies <strong>(C)</strong>. In <strong>(C)</strong>, panel 1 shows part of the membrane exposed for 3 h, and 2 shows part of the same membrane exposed for 3 min. The molecular
                           masses of standard proteins used are indicated at top (BN-PAGE) and at right (SDS-PAGE). Arrows in <strong>(A)</strong> indicate the positions of the main SERK1 complex sizes of 350 and 450 kD, and arrows in <strong>(B)</strong> indicate the positions of the main SERK1-CFP complexes of 400 and 500 kD. Arrows in <strong>(C)</strong> indicate the positions of the main BRI1 complexes of 250, 350, and 450 kD, and arrows in <strong>(D)</strong> indicate the main BAK1-GFP complexes ranging from 350 to 450 kD.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     The results show that there is one abundant core complex for SERK1 of ∼350 kD. We first examined whether BRI1 was part of
                     the core SERK1 complex. Therefore, a microsomal fraction prepared from
                     SERK1-CFP transgenic plants was examined for the presence of BRI1 proteins. BRI1 was found in complex with the same size range
                     of 400 to 500 kD (<a href="#FIG6">Figure 6C</a>, panel 1) as the SERK1-CFP proteins. In contrast with SERK1, most of the BRI1 protein was found as probably a homodimeric
                     form of ∼250 kD (<a href="#FIG6">Figure 6C</a>, panel 2). The occurrence of potential BRI1 homodimers is consistent with our observation of BRI1 homodimerization in protoplast
                     (Russinova et al., 2004<a href="#BIB36"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) and was recently shown in plants (Wang et al., 2005b<a href="#BIB47"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). The observed smaller and higher forms of BRI1 are probably attributable to the recognition of differentially processed
                     forms (<a href="#FIG6">Figure 6C</a>). The distribution of BAK1 in the microsomes prepared from BAK1-GFP transgenic plants was then studied by BN-PAGE. BAK1-GFP
                     was found in complex(es) with the same size range as SERK1-CFP (<a href="#FIG6">Figure 6D</a>). These data suggest that BRI1, SERK1, and SERK3 are indeed present in the main 350-kD core complex and are also present
                     in complexes of higher molecular mass. No signal was detected when solubilized microsomes from wild-type seedlings were analyzed
                     for the presence of GFP, confirming that the signals observed in <a href="#FIG6">Figures 6B and 6D</a> are indeed derived from microsomal SERK1-CFP or BAK1-GFP (<a href="#FIG6">Figure 6E</a>). Collectively, the results obtained from BN-PAGE suggest that the core SERK1 complex of ∼350 kD as well as the larger complexes
                     consist of the membrane receptors SERK1, SERK3, and BRI1. Assuming that all proteins
                     listed in <a href="#TBL1">Table 1</a> are present as monomers, a predicted size of ∼500 kD is obtained. Although this is about the maximum size of the SERK1 complex
                     as estimated by BN-PAGE, it exceeds the size
                     of the predominant complex of 350 kD. 
                     
                  </p>
                  <p>
                     
                     In summary, we propose that in <em>Arabidopsis</em> seedlings, a core SERK1 membrane complex consists of SERK1, BRI1, and SERK3. Collectively, this would amount to ∼270 kD when
                     all proteins are present as monomers, or between 350 and 400 kD in a more conventional tetrameric arrangement.
                     This is clearly in the range of the observed sizes on BN-PAGE gels. KAPP interacts only with the active SERK1 receptor (Shah
                     et al., 2002<a href="#BIB39"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) and could represent one of the less abundant higher molecular mass forms. 
                     
                  </p>
                  <p>
                     
                     <a name="SEC3" id="SEC3"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        DISCUSSION
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     In this work, we used a combination of coimmunoprecipitation based on the GFP variant CFP as a protein tag and LC/MALDI-TOF/MS
                     to identify proteins associated with the membrane receptor SERK1. Based on our data with other membrane proteins from <em>Arabidopsis</em>, this technique appears to be of general use. Compared with other techniques to identify interacting proteins, such as in
                     vitro pulldown assays and yeast two-hybrid screens, the main advantage is that the immunoprecipitation combined with LC/MALDI-TOF/MS
                     allows protein complex isolation under native conditions, therefore allowing functional studies in which the activity or posttranslational
                     modification of one or several proteins in the complex can be examined (Drewes and Bouwmeester, 2003<a href="#BIB5"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Another advantage is that the CFP tag allows one to directly follow all subcellular localizations of the fusion proteins.
                     This allows predictions concerning the nature of the proteins recovered, especially in the case of membrane receptors undergoing
                     recycling (Russinova et al., 2004<a href="#BIB36"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Other tags have been used, such as tandem affinity purification tags for plant protein complex isolation in a transient
                     expression system or in stably transformed transgenic lines, using the cauliflower mosaic virus 35S promoter (Rohila et al.,
                     2004<a href="#BIB34"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Rubio et al., 2005<a href="#BIB35"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Our procedure differs in several respects: we used an endogenous promoter-based construct and only used CFP as a protein
                     tag, thereby minimizing potential biological problems arising from the use of strong constitutive promoters. Also, we used
                     integral membrane protein as a bait, rather than the soluble proteins used previously. 
                     
                  </p>
                  <p>
                     
                     <strong>SERK1 and BR Signaling</strong><br/>
                     
                     A highly interesting finding was the identification of two members of the BR signaling pathway, the main BR receptor BRI1
                     (Li and Chory, 1997<a href="#BIB16"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) and its coreceptor BAK1 (SERK3) (Li et al., 2002<a href="#BIB18"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Nam and Li, 2002<a href="#BIB23"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), as components of the SERK1 complex. This finding implies that SERK1 is a previously unrecognized element of BR signaling.
                     Recently, we showed that the SERK3 receptor may have a role in the internalization of BRI1 after heterodimerization and that
                     BRI1 can also form homodimers in the plasma membrane (Russinova et al., 2004<a href="#BIB36"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Further biochemical evidence comes from the finding here that BRI1 and SERK1 as well as SERK1 and SERK3 can interact in
                     the plasma membrane of protoplasts. FRET studies between SERK1-CFP and SERK1-YFP have shown that only 15% of SERK1 exists
                     as homodimers on the plasma membrane of protoplast cells (Shah et al., 2001a<a href="#BIB38"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Our results showed that both SERK1 and SERK3 heterodimerize with the BRI1 receptor, suggesting that both coreceptors can
                     have comparable activity in terms of BRI1 signaling and/or internalization. Another piece of evidence is that a <em>bak1</em> null allele shows less severe phenotypes than <em>bri1</em> loss-of-function alleles, indicating that partially redundant activity could be attributed to different SERK family members
                     (Li et al., 2002<a href="#BIB18"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Nam and Li, 2002<a href="#BIB23"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Genetic analysis of the <em>serk1-1 bri1-119</em> double mutant suggests that SERK1–BRI1 interactions indeed affect BRI1-mediated signaling in planta, but to a lesser extent
                     than SERK3–BRI1 interactions. Notably, none of the proposed downstream components of BRI1, such as BES1, BZR1, or BIN2 (Li
                     and Nam, 2002<a href="#BIB17"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Wang et al., 2002b<a href="#BIB48"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Yin et al., 2002<a href="#BIB50"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), was detected in the SERK1 complex. 
                     
                  </p>
                  <p>
                     
                     It is unlikely that all proteins (as identified by MALDI-TOF/MS) listed in <a href="#TBL1">Table 1</a> are present in a complex together with SERK1 at the same time. Especially because proteins such as receptors and CDC48A (Rancour
                     et al., 2002<a href="#BIB30"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>, 2004<a href="#BIB31"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) are known to shuttle between monomeric and multimeric forms and can change their interaction properties upon activation
                     and relocalization, multiple forms of the same complex are to be expected. An indication of the presence of different complexes
                     is the localization of SERK1 in the plasma membrane as well as in internalized membrane compartments (Russinova et al., 2004<a href="#BIB36"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Kwaaitaal et al., 2005<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     It was proposed that BRI1 forms heterodimers with SERK3 to initiate BR signaling upon BR binding (Li et al., 2002<a href="#BIB18"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Nam and Li, 2002<a href="#BIB23"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). This was recently confirmed in an elegant demonstration of the in vivo phosphorylation properties of the BRI1 receptor
                     (Wang et al., 2005a<a href="#BIB46"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). In that work, it was clearly shown that the interaction between the BAK1 (SERK3) proteins and BRI1 is brassinolide-dependent.
                     Because we did not apply exogenous ligands before isolation of the SERK1 complex, it is likely that only a small fraction
                     of the complexes we isolated are actively signaling via the SERK1 protein. This notion seems to be supported by the fact that
                     most of the BRI1 receptors were unassociated with SERK1 in seedlings. Although our results do not allow us to precisely predict
                     the composition, it is likely that SERK1, SERK3, and BRI1 receptors can form tetrameric complexes analogous to the functional
                     complex of transforming growth factor-ß (TGF-ß) receptors. Each type of TGF-ß receptor is also present as a ligand-independent
                     dimer on the cell surface. TGF-ß binding to the TßRII homodimers promotes the formation of (TßRI)<sub>2</sub>/(TßRII)<sub>2</sub> heterodimers, in which TßRI is phosphorylated by the constitutively active TßRII and becomes activated to propagate the TGF-ß
                     signal. Combinatorial interactions in the tetrameric receptor complex allow differential ligand binding or differential signaling
                     in response to the same ligand. One receptor combination often binds different ligands, and patterns of ligand and receptor
                     expression dictate which receptor–ligand combination is activated (Feng and Derynck, 1997<a href="#BIB6"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). This so-called tetrameric model was also recently proposed for BRI1 and BAK1 (SERK3) (Wang et al., 2005b<a href="#BIB47"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     <strong>Other SERK1 Complex Components</strong><br/>
                     
                     The presence of the PP2C phosphatase KAPP, 14-3-3ν, and CDC48A in the SERK1 complex confirms previously obtained results using
                     yeast two-hybrid screening and in vitro interaction
                     studies (Rienties et al., 2005<a href="#BIB32"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Collectively, these proteins appear to be involved in dephosphorylation, protein interaction and membrane interaction,
                     and protein degradation, respectively. It is likely that they represent receptor maintenance or trafficking functions. Previously,
                     it was shown that SERK1 could interact with 14-3-3λ (Rienties et al., 2005<a href="#BIB32"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). The results presented here suggest that the receptor preferentially interacts with 14-3-3ν in seedlings and perhaps with
                     14-3-3λ in siliques, given the origin of the <em>Arabidopsis</em> cDNA library from young silique tissue that was used for yeast two-hybrid screening. The kinase domain of SERK1 is able to
                     transphosphorylate and bind in vitro to CDC48A, 14-3-3λ, and KAPP (Rienties et al., 2005<a href="#BIB32"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). In vitro, the interactions are all phosphorylation-dependent. In vivo, it was shown that SERK1 interacts with KAPP only
                     in intracellular vesicles, and it was proposed to play a role in receptor internalization as well as in dephosphorylation
                     (Shah et al., 2002<a href="#BIB39"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). For two other LRR-RLKs, CLV1 (Williams et al., 1997<a href="#BIB49"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) and FLS2 (Gomez-Gomez et al., 2001<a href="#BIB9"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), it has been shown that KAPP functions as a negative regulator, and it may have the same role in controlling signaling through
                     the SERK1 receptor. 
                     
                  </p>
                  <p>
                     
                     CDC48 protein assembles mainly in hexameric forms (Rancour et al., 2002<a href="#BIB30"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). In <em>Arabidopsis</em> suspension-cultured cells, soluble (cytosolic) CDC48A was found in a high molecular mass protein of 640 kD. Recently, it
                     was proposed that CDC48A interacts with PUX1 as a monomer and may function in regulating plant growth (Rancour et al., 2004<a href="#BIB31"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) and also in the plant endoplasmic reticulum–associated protein degradation system (Muller et al., 2005<a href="#BIB21"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     Plant 14-3-3 proteins have also been found associated with G-box transcription factors (Lu et al., 1992<a href="#BIB19"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Five <em>Arabidopsis</em> 14-3-3s, including 14-3-3λ, have been shown to interact with other transcription factors (Pan et al., 1999<a href="#BIB25"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), and current models propose that the interaction with members of the 14-3-3 family is client-driven (Paul et al., 2005<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     Two transcription factors were found to be associated with the SERK1 complex: AGL15 and a putative CO-like B-box zinc finger
                     protein. Most likely, these two proteins do not interact directly with the receptor but require 14-3-3 proteins as adaptor
                     proteins. It has been shown that 14-3-3 proteins can also promote the cytoplasmic localization or, conversely, the nuclear
                     localization, of transcription factors (reviewed in Muslin and Xing, 2000<a href="#BIB22"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). One of the other MADS box transcription factors, AGL24, was shown to interact directly with the kinase domain of the <em>Arabidopsis</em> meristematic receptor-like kinase and to be phosphorylated by the kinase domain of the receptor in vitro (Fujita et al.,
                     2003<a href="#BIB8"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). AGL15 was shown to accumulate in nuclei but also to be present in the cytoplasm (Perry et al., 1996<a href="#BIB29"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). SERK1 and AGL15 are highly expressed during embryogenic cell formation in culture and during early embryogenesis. As found
                     for SERK1 (Hecht et al., 2001<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), AGL15 promotes somatic embryo production from the shoot apical meristem in liquid culture when ectopically overexpressed
                     (Perry et al., 1996<a href="#BIB29"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Harding et al., 2003<a href="#BIB11"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Interestingly, AGL15-overexpressing tissues also had increased expression of SERK1 (Harding et al., 2003<a href="#BIB11"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). It was shown that AGL15 could bind directly to the promoter regions of different targets (Wang et al., 2002a<a href="#BIB45"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Those authors described how after chromatin immunoprecipitation, they obtained DNA fragments containing <em>cis</em>-regulatory elements targeted by AGL15, which may contribute to the regulation of the <em>SERK</em> gene by directly binding to its promoter. These data and our findings suggest that SERK1 and AGL15 can be involved in the
                     same signaling pathway. 
                     
                  </p>
                  <p>
                     
                     The other putative transcription factor found in the SERK1 complex belongs to a large family of CO-like zinc finger transcription
                     factors, in which the zinc finger region regulates protein–protein interactions, as found for several animal transcription
                     factors (reviewed in Griffiths et al., 2003<a href="#BIB10"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Recently, it was shown that CO accumulation is regulated by photoreceptors in photoperiodic flowering (Valverde et al.,
                     2004<a href="#BIB44"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     In conclusion, we propose that in <em>Arabidopsis</em> seedlings, signaling mediated by the SERK1 receptor combines elements of the BR pathway with a short signal transduction
                     chain, in which the plasma membrane receptor is in a complex with its cognate transcriptional regulators, such as AGL15. Further
                     studies are needed to evaluate this model and to map the phosphorylation sites that are responsible for the activation of
                     receptor kinases and the recruitment of downstream signaling components. 
                     
                  </p>
                  <p>
                     
                     <a name="SEC4" id="SEC4"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        METHODS
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     <strong>Plant Materials and Growth Conditions</strong><br/><em>Arabidopsis thaliana</em> ecotype Columbia was used as the wild type. Seeds were surface-sterilized and sown on germination medium (half-strength Murashige
                     and Skoog salt and vitamins medium [Duchefa] supplemented with 0.8% [w/v] agar [Daishin] and 2% [w/v] sucrose). Seeds were
                     kept for 2 d at 4°C and then transferred to a growth chamber. Seedlings were grown at 22°C under 16-h light/8-h dark periods
                     for 7 d. <em>Arabidopsis</em> plants expressing the SERK1 receptor fused to CFP under the control of its native promoter were similar to the SERK1-YFP–expressing
                     plants described previously by Kwaaitaal et al. (2005)<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. BRI1-GFP transgenic plants used for immunoprecipitation and BAK1-GFP plants used for BN-PAGE were described previously (Nam
                     and Li., 2002<a href="#BIB23"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Russinova et al., 2004<a href="#BIB36"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). The <em>bri1-4</em> strong allele (No. 3953; obtained from Frans Tax via the Arabidopsis Stock Center) was described previously by Noguchi et
                     al. (1999)<a href="#BIB24"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. 
                     
                  </p>
                  <p>
                     
                     <strong>Generation of <em>bri1-119 serk1-1</em>, <em>bri1-119 serk3</em>, and <em>bri1-119 serk2-2</em> Double Mutants</strong><br/>
                     
                     The <em>bri1-119</em> mutant was obtained from the ABRC (Ohio State University; stock No. CS399, ecotype Enkheim-2) and was described previously
                     by Noguchi et al. (1999)<a href="#BIB24"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a> and Friedrichsen et al. (2000)<a href="#BIB7"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. The SERK1 and SERK3 knockout lines, <em>serk1-1</em> and <em>serk3</em>, were obtained from the SIGnAL TDNA-Express collection (Alonso et al., 2003<a href="#BIB2"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) as insertion lines SALK_544330 and SALK_034523.56.00, respectively, and the <em>serk2-2</em> insertion line (No. 119-G03) was from the Syngenta <em>Arabidopsis</em> insertion lines, described previously by Albrecht et al. (2005)<a href="#BIB1"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>, in the Columbia background. <em>bri1-119 serk1-1</em> and <em>bri1-119 serk2-2</em> were generated by crossing <em>bri1-119</em> with <em>serk1-1</em> or <em>serk2-2</em>, respectively; <em>bri1-119 serk3</em> was generated by crossing <em>bri1-119</em> with <em>serk3</em>. In each case, F1 plants were allowed to self-fertilize, and candidate double F2 plants were genotyped phenotypically and
                     by PCR. 
                     
                  </p>
                  <p>
                     
                     <strong>Transient Expression in Protoplasts and FLIM</strong><br/>
                     
                     The full-length SERK1, SERK3, and BRI1 CFP and YFP fusion constructs were described previously (Shah et al., 2001a<a href="#BIB38"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Russinova et al., 2004<a href="#BIB36"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). FLIM was performed using a Bio-Rad Radiance 2100 MP system in combination with a Nikon TE 300 inverted microscope as described
                     (Russinova et al., 2004<a href="#BIB36"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). From the intensity images obtained, complete fluorescence lifetime decays were calculated per pixel and fitted using a
                     double exponential decay model. The fluorescence lifetime of one component was fixed to the value found for SERK1-CFP (2.5
                     ns). 
                     
                  </p>
                  <p>
                     
                     <strong>Protein Extraction, SDS-PAGE, and Protein Gel Blot Analysis</strong><br/>
                     
                     To prepare total protein extracts, plant material was ground in liquid nitrogen and thawed in extraction buffer containing
                     50 mM Tris, pH 7.5, 150 mM NaCl, 0.5% Triton X-100, and a protease inhibitor cocktail according to the manufacturer's instructions
                     (Roche). Samples were incubated on ice for 30 min. The suspension was centrifuged at 200<em>g</em> (Beckman Microfuge 18 centrifuge) at 4°C for 3 min. The concentration of extracted proteins was determined with the Bio-Rad
                     protein assay. After boiling 30 µg of total protein for 5 min in sample buffer containing 100 mM Tris-HCl, pH 6.8, 4% SDS,
                     0.2% bromphenol blue, 20% glycerol, and 100 mM DTT, the proteins were separated on 8 or 12% SDS-polyacrylamide gels (Laemmli,
                     1970<a href="#BIB15"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) and transferred onto nitrocellulose (Schleicher &amp; Schuell) by wet electroblotting (Mini-Protean II system; Bio-Rad). Proteins
                     were analyzed by immunoblotting with anti-GFP (IgG fraction, 1500<span style="font-family: arial,helvetica;  ">x</span> dilution; Molecular Probes), and a polyclonal antibody was generated against the GST-SERK1 kinase domain fusion protein (3000<span style="font-family: arial,helvetica;  ">x</span> dilution) (Rienties et al., 2005<a href="#BIB32"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Anti-BRI1 specific antibodies were kindly provided by Joanne Chory (Salk Institute for Biological Studies; used at 1000<span style="font-family: arial,helvetica;  ">x</span> dilution), anti-CDC48A antibodies (Rancour et al., 2002<a href="#BIB30"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) were kindly provided by Sebastian Bednarek (Department of Biochemistry, University of Wisconsin; used at 1000<span style="font-family: arial,helvetica;  ">x</span> dilution), and-14-3-3 antibodies have been described previously (Rienties et al., 2005<a href="#BIB32"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Bound primary antibodies were detected with horseradish peroxidase–conjugated IgG purified donkey anti-rabbit secondary
                     antibody at 10,000<span style="font-family: arial,helvetica;  ">x</span> dilution (Rockland Laboratories) or goat anti-chicken secondary antibody at 5000<span style="font-family: arial,helvetica;  ">x</span> dilution (Rockland Laboratories) and visualized using reagents for chemiluminescent detection (ECL Plus reagent; Amersham
                     Biosciences). 
                     
                  </p>
                  <p>
                     
                     <strong>BN-PAGE</strong><br/>
                     
                     Preparation of microsomes and BN-PAGE were performed as described (Rivas et al., 2002<a href="#BIB33"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Microsomes from 7-d-old seedlings from wild-type plants or transgenic plants expressing the SERK1 promoter-SERK1-CFP construct
                     were solubilized in extraction buffer supplemented with 750 mM aminocaproic acid and incubated for 30 min on ice. Aliquots
                     of supernatants were incubated with 0.5% Coomassie Brilliant Blue G 250 and 5% (v/v) glycerol and loaded on the gel. BN-PAGE
                     was then performed according to Schagger and von Jagow (1991)<a href="#BIB37"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a> on a 5 to 15% gradient polyacrylamide gel. Albumin bovine monomer (66 kD), albumin bovine dimer (132 kD), urease trimer (272
                     kD), and urease hexamer (545 kD) from Sigma-Aldrich were used as marker proteins. 
                     
                  </p>
                  <p>
                     
                     <strong>Immunoprecipitations</strong><br/>
                     
                     Routinely, we used four plates (9 cm) with ∼400 7-d-old seedlings (100 per plate) to prepare a protein extract suitable for
                     immunoprecipitation. Ten milliliters of total
                     plant protein extract (1 mg/mL) in extraction buffer (isolated as described above) was incubated for 1 h at 4°C with 100 µL
                     of 50% (v/v) protein A–Sepharose CL-4B beads slurry (Amersham Biosciences) in 50 mM Tris and 150 mM NaCl, pH 7.5. After centrifugation
                     at 200<em>g</em> (Beckman Microfuge 18 centrifuge) for 2 min at 4°C, the supernatant was incubated with 7 µL of anti-GFP, anti-BRI1, anti–14-3-3,
                     anti-CDC48A, or rabbit preimmune serum. After incubation with gentle mixing for 1 h at 4°C, 200 µL of fresh 50% slurry of
                     protein A beads was added, and incubation was continued for 4 h. Protein A beads were pelleted by centrifugation at 200<em>g</em> for 1 min, and the supernatant was removed. The beads were washed three times with 1 mL of wash buffer (50 mM Tris, pH 7.5,
                     and 150 mM NaCl) and once with 10 mM NH<sub>4</sub>HCO<sub>3</sub>, pH 8. Before the last wash step, the beads together with the wash buffer were moved to a new tube. After centrifugation,
                     the wash buffer was removed, and the beads were used in this form directly for digestion with trypsin and subsequent MALDI-TOF/MS
                     analysis. 
                     
                  </p>
                  <p>
                     
                     For immunodetection, the proteins were eluted from the beads by adding SDS sample buffer and boiling for 5 min at 95°C. In
                     the case of coimmunoprecipitation of SERK1-CFP with rabbit anti-GFP antibodies and detection with rabbit anti–14-3-3 antibodies,
                     no DTT was included in the SDS sample buffer. This prevents separation of the heavy and light chains of the antibodies and
                     improves the resolution of the blots, because the detection system also recognized IgGs whose light chains overlap with the
                     14-3-3 proteins. 
                     
                  </p>
                  <p>
                     
                     <strong>Protein Digestion</strong><br/>
                     
                     Proteins bound to protein A beads in 50 µL of 10 mM NH<sub>4</sub>HCO<sub>3</sub> were incubated with 100 ng of trypsin (sequencing grade; Boehringer Mannheim). After overnight incubation with gentle shaking
                     at room temperature, the pH was decreased to 2 by adding trifluoroacetic acid (TFA) in water up to 0.4% (v/v). 
                     
                  </p>
                  <p>
                     
                     <strong>LC/MALDI-TOF/MS Analysis</strong><br/>
                     
                     Fifty microliters of peptide sample was injected into a 100 <span style="font-family: arial,helvetica;  ">x</span> 0.25-mm Prontosil 3-µm C18H column (prepared in-house) and eluted at a flow rate of 4 µL/min with 5% (v/v) acetonitrile in
                     water containing 0.05% TFA. A gradient from 5 to 20% (v/v) acetonitrile in water containing 0.05% (v/v) TFA was applied in
                     90 s, followed by a slower linear increase to 40% (v/v) acetonitrile in water containing 0.05% (v/v) TFA in another 4 min
                     (Alliance 2690 running at 0.25 mL/min equipped with a flow splitter; Waters). Samples were collected directly onto a sample
                     plate (MTP AnchorChip 200/384; Bruker Daltonics) in fractions of 15 s, taking into account the system gradient delay time
                     and column volume before starting collection. After air-drying, 1 µL of a 20<span style="font-family: arial,helvetica;  ">x</span> diluted α-cyano-4-hydroxy-cinnamic acid solution was added to each spot. This solution was prepared by diluting a saturated
                     α-cyano-4-hydroxy-cinnamic acid solution (prepared in 50% [v/v] acetonitrile, 49% [v/v] water, and 1% [v/v] TFA) 20 times
                     into
                     the same solution. Sample spots were recrystallized by applying 1.5 µL of 60% (v/v) ethanol, 30% (v/v) acetone, 10% (v/v)
                     water, and 0.1% (v/v) TFA to all samples and allowing them to air-dry. 
                     
                  </p>
                  <p>
                     
                     Mass spectra were measured with a Bruker Ultraflex MALDI-TOF/MS apparatus equipped with a reflectron (Bruker Daltonics) in
                     positive mode with a high precision method. Spectra were measured after external quadratic calibration with a peptide standard
                     mixture containing nine peptides in the mass range between 757 and 3146 D (Bruker Daltonics). In general, spectra were summed
                     for ∼500 to 1000 laser pulses (20 Hz; nitrogen laser at 337 nm) and internally recalibrated with protein A–derived peptides
                     and/or
                     trypsin autolysis peaks when present. All <em>m</em>/<em>z</em> values also present in the control peptide sample were subsequently removed from the <em>m</em>/<em>z</em> list obtained for the immunoprecipitated SERK1 protein complex. Remaining <em>m</em>/<em>z</em> values were checked for sodium and/or potassium adducts (differences of 22.990 and 38.946 D, respectively). When found, these
                     <em>m</em>/<em>z</em> values were deleted as well. For identification of SERK1, the program GPMAW (<a href="http://welcome.to/gpmaw/">http://welcome.to/gpmaw/</a>) was used to determine which <em>m</em>/<em>z</em> values corresponded to tryptic peptides from the predicted protein sequence of SERK1. These selected <em>m</em>/<em>z</em> values were used as inputs for the MASCOT program to search either the <em>Arabidopsis</em> database or all entries without selecting a taxonomy. The posttranslational modifications phosphorylation (of Ser, Thr, and
                     Tyr) and oxidation of Met were allowed, and a peptide <em>m</em>/<em>z</em> tolerance of 100 ppm was used. After identification, the <em>m</em>/<em>z</em> values that matched the SERK1 protein were removed from the list before searching for other proteins. Alternatively, the
                     searches were performed with the complete <em>m</em>/<em>z</em> list directly using the MASCOT search engine (<a href="http://www.matrixscience.com">http://www.matrixscience.com</a>) (Perkins et al., 1999<a href="#BIB28"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     <strong>Accession Numbers</strong><br/>
                     
                     Sequence data and seed stocks from this article can be found in the GenBank/EMBL data libraries under the following accession
                     numbers: <em>SERK1</em> cDNA clone (<a href="/cgi/external_ref?access_num=A67827&amp;link_type=GEN">A67827</a>, <a href="/cgi/external_ref?access_num=At1g71830&amp;link_type=GEN">At1g71830</a>), <em>SERK3/BAK1</em> cDNA clone (<a href="/cgi/external_ref?access_num=AF384970&amp;link_type=GEN">AF384970</a>, <a href="/cgi/external_ref?access_num=At4g33430&amp;link_type=GEN">At4g33430</a>), and <em>BRI1</em> cDNA clone (<a href="/cgi/external_ref?access_num=AF017056&amp;link_type=GEN">AF017056</a>, <a href="/cgi/external_ref?access_num=At4g39400&amp;link_type=GEN">At4g39400</a>). The <em>serk1-1</em> and <em>serk3</em> insertion lines were obtained from the SIGnAL TDNA-Express collection, with accession numbers SALK_544330 and SALK_034523.56.00,
                     respectively. The <em>serk2-2</em> insertion line was obtained from the Syngenta <em>Arabidopsis</em> insertion lines, with accession number 119-G03. The <em>bri1-119</em> and <em>bri1-4</em> mutants were obtained from the ABRC, with stock numbers CS399 and 3953, respectively. 
                     
                  </p>
                  <p>
                     
                     <strong>Supplemental Data</strong><br/>
                     
                     The following materials are available in the online version of this article.
                     
                  </p>
                  <dl><dd><strong>Supplemental Table 1.</strong><em>m</em>/<em>z</em> Values of the Peptides Obtained after Immunoprecipitation of a Total Protein Extract from 10-d-Old Transgenic Seedlings Expressing
                        the SERK1-CFP Fusion Protein. <br/></dd>
                     <dd><strong>Supplemental Table 2.</strong> Numeric Representation of the Inflorescence Length and Rosette Width of <em>bri1-119</em> Mutant Plants and <em>bri1-119 serk1-1</em>, <em>bri1-119 serk2-2</em>, and <em>bri1-119 serk3</em> Double Mutants. <br/></dd>
                  </dl><div class="auto-clean"> </div>
                  <p>
                     
                     
                     <a name="ACK" id="ACK"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        Acknowledgments
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     We thank Jan-Willem Borst (Center for Microspectroscopy, Wageningen University) for help with the FLIM, Boudewijn van Veen
                     (Center for Microspectroscopy) for help in editing and formatting the images shown here, and the Nottingham Arabidopsis Stock
                     Centre for supplying the Salk insertion seed lines. We thank Jianming Li (Department of Molecular, Cellular, and Developmental
                     Biology, University of Michigan) for kindly providing the BAK1-GFP transgenic plants. This work was cofinanced by the Center
                     for BioSystems Genomics, which is part of the Netherlands Genomics Initiative/Netherlands Organization for Scientific Research
                     (NWO), by Grant ALW 812.06.004 from the Dutch Organization for Research, NWO (J.A.), and by the Agrotechnology and Food Sciences
                     Group of Wageningen University (E.R., J.V., S.B., and S.d.V.). 
                     
                  </p>
                  <p>
                     
                     
                     <a name="FN" id="FN"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        Footnotes
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     <a name="" id=""/>
                     The author responsible for distribution of materials integral to the findings presented in this article in accordance with
                     the policy described in the Instructions for Authors (<a href="http://www.plantcell.org">www.plantcell.org</a>) is: Sacco de Vries (<span class="em-link"><span class="em-addr">sacco.devries{at}wur.nl</span></span>). 
                     
                  </p>
                  <p>
                     
                     <a name="FN1" id="FN1"/>
                     <sup>[W]</sup> Online version contains Web-only data. <a href="#RFN1"><img width="12" height="12" alt="Back" src="/icons/back.gif"/></a></p>
                  <p>
                     <a/>
                     Article, publication date, and citation information can be found at <a href="http://www.plantcell.org/cgi/doi/10.1105/tpc.105.039412">www.plantcell.org/cgi/doi/10.1105/tpc.105.039412</a>. 
                     
                  </p>
                  <p>
                     
                     
                     Received November 10, 2005; 
                     Revision received January  4, 2006. 
                     accepted January 16, 2006.
                     
                  </p>
                  <p>
                     
                     <a name="BIBL" id="BIBL"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        REFERENCES
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     
                     <a name="BIB1" id="BIB1"/><strong>Albrecht, C., Russinova, E., Hecht, V., Baaijens, E., and de Vries, S.C.</strong> (2005). The <em>Arabidopsis thaliana</em> somatic embryogenesis receptor-like kinases 1 and 2 control male sporogenesis. Plant Cell <strong>17,</strong> 3337–3349.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=17/12/3337">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB2" id="BIB2"/><strong>Alonso, J.M., et al.</strong> (2003). Genome-wide insertional mutagenesis of <em>Arabidopsis thaliana.</em> Science <strong>301,</strong> 653–657.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=301/5633/653">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB3" id="BIB3"/><strong>Becraft, P.W.</strong> (2002). Receptor kinase signaling in plant development. Annu. Rev. Cell Dev. Biol. <strong>18,</strong> 163–192.<a href="/cgi/external_ref?access_num=10.1146%2Fannurev.cellbio.18.012502.083431&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000179413400008&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=12142267&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB4" id="BIB4"/><strong>Colcombet, J., Boisson-Dernier, A., Ros-Palau, R., Vera, C.E., and Schroeder, J.</strong> (2005). <em>Arabidopsis thaliana</em> somatic embryogenesis receptor kinases 1 and 2 are essential for tapetum development and microspore maturation. Plant Cell
                     <strong>17,</strong> 3350–3361.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=17/12/3350">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB5" id="BIB5"/><strong>Drewes, G., and Bouwmeester, T.</strong> (2003). Global approaches to protein-protein interactions. Curr. Opin. Cell Biol. <strong>15,</strong> 199–205.<a href="/cgi/external_ref?access_num=10.1016%2FS0955-0674%2803%2900005-X&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000182183300011&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=12648676&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB6" id="BIB6"/><strong>Feng, X.H., and Derynck, R.</strong> (1997). A kinase subdomain of transforming growth factor-beta (TGF-beta) type I receptor determines the TGF-beta intracellular
                     signaling specificity. EMBO J. <strong>16,</strong> 3912–3923.<a href="/cgi/external_ref?access_num=10.1093%2Femboj%2F16.13.3912&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=A1997XJ99000017&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=9233801&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB7" id="BIB7"/><strong>Friedrichsen, D.M., Joazeiro, C.A., Li, J., Hunter, T., and Chory, J.</strong> (2000). Brassinosteroid-insensitive-1 is a ubiquitously expressed leucine-rich repeat receptor serine/threonine kinase. Plant
                     Physiol. <strong>123,</strong> 1247–1256.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=123/4/1247">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB8" id="BIB8"/><strong>Fujita, H., Takemura, M., Tani, E., Nemoto, K., Yokota, A., and Kohchi, T.</strong> (2003). An Arabidopsis MADS-box protein, AGL24, is specifically bound to and phosphorylated by meristematic receptor-like
                     kinase (MRLK). Plant Cell Physiol. <strong>44,</strong> 735–742.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pcellphys&amp;resid=44/7/735">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB9" id="BIB9"/><strong>Gomez-Gomez, L., Bauer, Z., and Boller, T.</strong> (2001). Both the extracellular leucine-rich repeat domain and the kinase activity of FSL2 are required for flagellin binding
                     and signaling in Arabidopsis. Plant Cell <strong>13,</strong> 1155–1163.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=13/5/1155">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB10" id="BIB10"/><strong>Griffiths, S., Dunford, R.P., Coupland, G., and Laurie, D.A.</strong> (2003). The evolution of CONSTANS-like gene families in barley, rice, and Arabidopsis. Plant Physiol. <strong>131,</strong> 1855–1867.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=131/4/1855">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB11" id="BIB11"/><strong>Harding, E.W., Tang, W., Nichols, K.W., Fernandez, D.E., and Perry, S.E.</strong> (2003). Expression and maintenance of embryogenic potential is enhanced through constitutive expression of AGAMOUS-Like 15.
                     Plant Physiol. <strong>133,</strong> 653–663.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=133/2/653">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB12" id="BIB12"/><strong>Hecht, V., Vielle-Calzada, J.P., Hartog, M.V., Schmidt, E.D., Boutilier, K., Grossniklaus, U., and de Vries, S.C.</strong> (2001). The Arabidopsis SOMATIC EMBRYOGENESIS RECEPTOR KINASE 1 gene is expressed in developing ovules and embryos and enhances
                     embryogenic competence in culture. Plant Physiol. <strong>127,</strong> 803–816.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=127/3/803">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB13" id="BIB13"/><strong>Heck, G.R., Perry, S.E., Nichols, K.W., and Fernandez, D.E.</strong> (1995). AGL15, a MADS domain protein expressed in developing embryos. Plant Cell <strong>7,</strong> 1271–1282.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=7/8/1271">[Abstract]</a></p>
                  <p><a name="BIB14" id="BIB14"/><strong>Kwaaitaal, M.A.C.J., de Vries, S.C., and Russinova, E.</strong> (2005). The Arabidopsis somatic embryogenesis receptor kinase 1 protein is present in sporophytic and gametophytic cells
                     and undergoes endocytosis. Protoplasma <strong>226,</strong> 55–65.<a href="/cgi/external_ref?access_num=10.1007%2Fs00709-005-0111-9&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000232662500006&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=16231101&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB15" id="BIB15"/><strong>Laemmli, U.K.</strong> (1970). Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature <strong>227,</strong> 680–685.<a href="/cgi/external_ref?access_num=10.1038%2F227680a0&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=5432063&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB16" id="BIB16"/><strong>Li, J., and Chory, J.</strong> (1997). A putative leucine-rich repeat receptor kinase involved in brassinosteroid signal transduction. Cell <strong>90,</strong> 929–938.<a href="/cgi/external_ref?access_num=10.1016%2FS0092-8674%2800%2980357-8&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=A1997XV56300013&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=9298904&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB17" id="BIB17"/><strong>Li, J., and Nam, K.H.</strong> (2002). Regulation of brassinosteroid signaling by a GSK3/SHAGGY-like kinase. Science <strong>295,</strong> 1299–1310.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=295/5558/1299">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB18" id="BIB18"/><strong>Li, J., Wen, J., Lease, K.A., Doke, J.T., Tax, F.E., and Walker, J.C.</strong> (2002). BAK1, an Arabidopsis LRR receptor-like protein kinase, interacts with BRI1 and modulates brassinosteroid signaling.
                     Cell <strong>110,</strong> 213–222.<a href="/cgi/external_ref?access_num=10.1016%2FS0092-8674%2802%2900812-7&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000177105200009&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=12150929&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB19" id="BIB19"/><strong>Lu, G., DeLisle, A.J., de Vetten, N.C., and Ferl, R.J.</strong> (1992). Brain proteins in plants: An Arabidopsis homolog to neurotransmitter pathway activators is part of a DNA binding
                     complex. Proc. Natl. Acad. Sci. USA <strong>89,</strong> 11490–11494.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=89/23/11490">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB20" id="BIB20"/><strong>Mora-Garcia, S., Vert, G., Yin, Y., Cano-Delgado, A., Cheong, H., and Chory, J.</strong> (2004). Nuclear protein phosphatases with Kelch-repeat domains modulate the response to brassinosteroids in Arabidopsis.
                     Genes Dev. <strong>18,</strong> 448–460.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=genesdev&amp;resid=18/4/448">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB21" id="BIB21"/><strong>Muller, J., Piffanelli, P., Devoto, A., Miklis, M., Elliott, C., Ortmann, B., Schulze-Lefert, P., and Panstruga, R.</strong> (2005). Conserved ERAD-like quality control of a plant polytopic membrane protein. Plant Cell <strong>17,</strong> 149–163.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=17/1/149">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB22" id="BIB22"/><strong>Muslin, A.J., and Xing, H.</strong> (2000). 14-3-3 proteins: Regulation of subcellular localization by molecular interference. Cell. Signal. <strong>12,</strong> 703–709.<a href="/cgi/external_ref?access_num=10.1016%2FS0898-6568%2800%2900131-5&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000166048900001&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=11152955&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB23" id="BIB23"/><strong>Nam, K.H., and Li, J.</strong> (2002). BRI1/BAK1, a receptor kinase pair mediating brassinosteroid signaling. Cell <strong>110,</strong> 203–212.<a href="/cgi/external_ref?access_num=10.1016%2FS0092-8674%2802%2900814-0&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000177105200008&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=12150928&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB24" id="BIB24"/><strong>Noguchi, T., Fujioka, S., Choe, S., Takatsuto, S., Yoshida, S., Yuan, H., Feldmann, K.A., and Tax, F.E.</strong> (1999). Brassinosteroid-insensitive dwarf mutants of Arabidopsis accumulate brassinosteroids. Plant Physiol. <strong>121,</strong> 743–752.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=121/3/743">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB25" id="BIB25"/><strong>Pan, S., Sehnke, P.C., Ferl, R.J., and Gurley, W.B.</strong> (1999). Specific interactions with TBP and TFIIB in vitro suggest that 14-3-3 proteins may participate in the regulation
                     of transcription when part of a DNA binding complex. Plant Cell <strong>11,</strong> 1591–1602.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=11/8/1591">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB26" id="BIB26"/><strong>Pandey, A., Podtelejnikov, A.V., Blagoev, B., Bustelo, X.R., Mann, M., and Lodish, H.F.</strong> (2000). Analysis of receptor signaling pathways by mass spectrometry: Identification of vav-2 as a substrate of the epidermal
                     and platelet-derived growth factor receptors. Proc. Natl. Acad. Sci. USA <strong>97,</strong> 179–184.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=97/1/179">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB27" id="BIB27"/><strong>Paul, A.L., Sehnke, P.C., and Ferl, R.J.</strong> (2005). Isoform-specific subcellular localization among 14-3-3 proteins in Arabidopsis seems to be driven by client interactions.
                     Mol. Biol. Cell <strong>16,</strong> 1735–1743.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=molbiolcell&amp;resid=16/4/1735">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB28" id="BIB28"/><strong>Perkins, D.N., Pappin, D.J., Creasy, D.M., and Cottrell, J.S.</strong> (1999). Probability-based protein identification by searching sequence databases using mass spectrometry data. Electrophoresis
                     <strong>20,</strong> 3551–3567.<a href="/cgi/external_ref?access_num=10.1002%2F%28SICI%291522-2683%2819991201%2920%3A18%3C3551%3A%3AAID-ELPS3551%3E3.0.CO%3B2-2&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000085044900009&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=10612281&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB29" id="BIB29"/><strong>Perry, S.E., Nichols, K.W., and Fernandez, D.E.</strong> (1996). The MADS domain protein AGL15 localizes to the nucleus during early stages of seed development. Plant Cell <strong>8,</strong> 1977–1989.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=8/11/1977">[Abstract]</a></p>
                  <p><a name="BIB30" id="BIB30"/><strong>Rancour, D.M., Dickey, C.E., Park, S., and Bednarek, S.Y.</strong> (2002). Characterization of AtCDC48. Evidence for multiple membrane fusion mechanisms at the plane of cell division in plants.
                     Plant Physiol. <strong>130,</strong> 1241–1253.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=130/3/1241">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB31" id="BIB31"/><strong>Rancour, D.M., Park, S., Knight, S.D., and Bednarek, S.Y.</strong> (2004). Plant UBX domain-containing protein 1, PUX1, regulates the oligomeric structure and activity of Arabidopsis CDC48.
                     J. Biol. Chem. <strong>279,</strong> 54264–54274.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=jbc&amp;resid=279/52/54264">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB32" id="BIB32"/><strong>Rienties, I.M., Vink, J., Borst, J.W., Russinova, E., and de Vries, S.C.</strong> (2005). The Arabidopsis SERK1 protein interacts with the AAA-ATPase AtCDC48, the 14-3-3 protein GF14lambda and the PP2C phosphatase
                     KAPP. Planta <strong>221,</strong> 394–405.<a href="/cgi/external_ref?access_num=10.1007%2Fs00425-004-1447-7&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000229578800010&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=15592873&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB33" id="BIB33"/><strong>Rivas, S., Romeis, T., and Jones, J.D.</strong> (2002). The Cf-9 disease resistance protein is present in an approximately 420-kilodalton heteromultimeric membrane-associated
                     complex at one molecule per complex. Plant Cell <strong>14,</strong> 689–702.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=14/3/689">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB34" id="BIB34"/><strong>Rohila, J.S., Chen, M., Cerny, R., and Fromm, M.E.</strong> (2004). Improved tandem affinity purification tag and methods for isolation of protein heterocomplexes from plants. Plant
                     J. <strong>38,</strong> 172–181.<a href="/cgi/external_ref?access_num=10.1111%2Fj.1365-313X.2004.02031.x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000220521800016&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=15053770&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB35" id="BIB35"/><strong>Rubio, V., Shen, Y., Saijo, Y., Liu, Y., Gusmaroli, G., Dinesh-Kumar, S.P., and Deng, X.W.</strong> (2005). An alternative tandem affinity purification strategy applied to Arabidopsis protein complex isolation. Plant J. <strong>41,</strong> 767–778.<a href="/cgi/external_ref?access_num=10.1111%2Fj.1365-313X.2004.02328.x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000226917400011&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=15703063&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB36" id="BIB36"/><strong>Russinova, E., Borst, J.W., Kwaaitaal, M., Cano-Delgado, A., Yin, Y., Chory, J., and de Vries, S.C.</strong> (2004). Heterodimerization and endocytosis of Arabidopsis brassinosteroid receptors BRI1 and AtSERK3 (BAK1). Plant Cell <strong>16,</strong> 3216–3229.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=16/12/3216">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB37" id="BIB37"/><strong>Schagger, H., and von Jagow, G.</strong> (1991). Blue native electrophoresis for isolation of membrane protein complexes in enzymatically active form. Anal. Biochem.
                     <strong>199,</strong> 223–231.<a href="/cgi/external_ref?access_num=10.1016%2F0003-2697%2891%2990094-A&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=A1991GU88000013&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=1812789&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB38" id="BIB38"/><strong>Shah, K., Gadella, T.W., Jr., van Erp, H., Hecht, V., and de Vries, S.C.</strong> (2001a). Subcellular localization and oligomerization of the <em>Arabidopsis thaliana</em> somatic embryogenesis receptor kinase 1 protein. J. Mol. Biol. <strong>309,</strong> 641–655.<a href="/cgi/external_ref?access_num=10.1006%2Fjmbi.2001.4706&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000169302900010&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=11397085&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB39" id="BIB39"/><strong>Shah, K., Russinova, E., Gadella, T.W., Jr., Willemse, J., and De Vries, S.C.</strong> (2002). The Arabidopsis kinase-associated protein phosphatase controls internalization of the somatic embryogenesis receptor
                     kinase 1. Genes Dev. <strong>16,</strong> 1707–1720.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=genesdev&amp;resid=16/13/1707">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB40" id="BIB40"/><strong>Shah, K., Vervoort, J., and de Vries, S.C.</strong> (2001b). Role of threonines in the <em>Arabidopsis thaliana</em> somatic embryogenesis receptor kinase 1 activation loop in phosphorylation. J. Biol. Chem. <strong>276,</strong> 41263–41269.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=jbc&amp;resid=276/44/41263">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB41" id="BIB41"/><strong>Stone, J.M., Trotochaud, A.E., Walker, J.C., and Clark, S.E.</strong> (1998). Control of meristem development by CLAVATA1 receptor kinase and kinase-associated protein phosphatase interactions.
                     Plant Physiol. <strong>117,</strong> 1217–1225.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=117/4/1217">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB42" id="BIB42"/><strong>Torii, K.U.</strong> (2004). Leucine-rich repeat receptor kinases in plants: Structure, function, and signal transduction pathways. Int. Rev.
                     Cytol. <strong>234,</strong> 1–46.<a href="/cgi/external_ref?access_num=000221668300001&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=15066372&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB43" id="BIB43"/><strong>Trotochaud, A.E., Hao, T., Wu, G., Yang, Z., and Clark, S.E.</strong> (1999). The CLAVATA1 receptor-like kinase requires CLAVATA3 for its assembly into a signaling complex that includes KAPP
                     and a Rho-related protein. Plant Cell <strong>11,</strong> 393–406.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=11/3/393">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB44" id="BIB44"/><strong>Valverde, F., Mouradov, A., Soppe, W., Ravenscroft, D., Samach, A., and Coupland, G.</strong> (2004). Photoreceptor regulation of CONSTANS protein in photoperiodic flowering. Science <strong>303,</strong> 1003–1006.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=303/5660/1003">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB45" id="BIB45"/><strong>Wang, H., Tang, W., Zhu, C., and Perry, S.E.</strong> (2002a). A chromatin immunoprecipitation (ChIP) approach to isolate genes regulated by AGL15, a MADS domain protein that
                     preferentially accumulates in embryos. Plant J. <strong>32,</strong> 831–843.<a href="/cgi/external_ref?access_num=10.1046%2Fj.1365-313X.2002.01455.x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000179643100017&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=12472697&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB46" id="BIB46"/><strong>Wang, X., Goshe, M.B., Soderblom, E.J., Phinney, B.S., Kuchar, J.A., Li, J., Asami, T., Yoshida, S., Huber, S.C., and Clouse,
                        S.D.</strong> (2005a). Identification and functional analysis of in vivo phosphorylation sites of the Arabidopsis BRASSINOSTEROID-INSENSITIVE1
                     receptor kinase. Plant Cell <strong>17,</strong> 1685–1703.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=17/6/1685">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB47" id="BIB47"/><strong>Wang, X., Li, X., Meisenhelder, J., Hunter, T., Yoshida, S., Asami, T., and Chory, J.</strong> (2005b). Autoregulation and homodimerization are involved in the activation of the plant steroid receptor BRI1. Dev. Cell
                     <strong>8,</strong> 855–865.<a href="/cgi/external_ref?access_num=10.1016%2Fj.devcel.2005.05.001&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000230006200010&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=15935775&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB48" id="BIB48"/><strong>Wang, Z.Y., Nakano, T., Gendron, J., He, J., Chen, M., Vafeados, D., Yang, Y., Fujioka, S., Yoshida, S., Asami, T., and Chory,
                        J.</strong> (2002b). Nuclear-localized BZR1 mediates brassinosteroid-induced growth and feedback suppression of brassinosteroid biosynthesis.
                     Dev. Cell <strong>2,</strong> 505–513.<a href="/cgi/external_ref?access_num=10.1016%2FS1534-5807%2802%2900153-3&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000175386100017&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=11970900&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB49" id="BIB49"/><strong>Williams, R.W., Wilson, J.M., and Meyerowitz, E.M.</strong> (1997). A possible role for kinase-associated protein phosphatase in the Arabidopsis CLAVATA1 signaling pathway. Proc. Natl.
                     Acad. Sci. USA <strong>94,</strong> 10467–10472.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=94/19/10467">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB50" id="BIB50"/><strong>Yin, Y., Wang, Z.Y., Mora-Garcia, S., Li, J., Yoshida, S., Asami, T., and Chory, J.</strong> (2002). BES1 accumulates in the nucleus in response to brassinosteroids to regulate gene expression and promote stem elongation.
                     Cell <strong>109,</strong> 181–191.<a href="/cgi/external_ref?access_num=10.1016%2FS0092-8674%2802%2900721-3&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000175082600008&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=12007405&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB51" id="BIB51"/><strong>Zhang, S.H., Kobayashi, R., Graves, P.R., Piwnica-Worms, H., and Tonks, N.K.</strong> (1997). Serine phosphorylation-dependent association of the band 4.1-related protein-tyrosine phosphatase PTPH1 with 14-3-3beta
                     protein. J. Biol. Chem. <strong>272,</strong> 27281–27287.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=jbc&amp;resid=272/43/27281">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB52" id="BIB52"/><strong>Zhang, S.H., Liu, J., Kobayashi, R., and Tonks, N.K.</strong> (1999). Identification of the cell cycle regulator VCP (p97/CDC48) as a substrate of the band 4.1-related protein-tyrosine
                     phosphatase PTPH1. J. Biol. Chem. <strong>274,</strong> 17806–17812.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=jbc&amp;resid=274/25/17806">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
               </div>
      <span class="highwire-journal-article-marker-end"/>
    </div>
  </body>
</html>
