<?xml version="1.0"?>
<html>
  <body>
    <div itemprop="articleBody">
      <span class="highwire-journal-article-marker-start"/>
      <div class="hw-optimized-content">
                  <div class="auto-clean">
                     
                     
                     
                     
                     <span style="  font-size: 83.33%"><em>The Plant Cell</em> 18:1642-1651 (2006)<br/>
                        © 2006 <a href="/misc/terms.shtml">American Society of Plant Biologists</a><br/></span></div>
                  <h1 id="article-title-1" itemprop="headline">
                     A Network of Local and Redundant Gene Regulation Governs <em>Arabidopsis</em> Seed Maturation
                     
                     
                  </h1>
                  <div class="contributors">
                     Alexandra To<sup>a</sup><a name="RFN1" id="RFN1"/><sup>,<a href="#FN1">1</a></sup>, 
                     Christiane Valon<sup>a</sup><a/><sup>,<a href="#FN1">1</a></sup>, 
                     Gil Savino<sup>a</sup>, 
                     Jocelyne Guilleminot<sup>b</sup>, 
                     Martine Devic<sup>b</sup>, 
                     Jérôme Giraudat<sup>a</sup> and 
                     François Parcy<sup>a</sup><sup>,c</sup><sup>,d</sup><sup>,2</sup></div>
                  <p>
                     <span style="  font-size: 83.33%"><sup>a</sup> Institut des Sciences du Végétal, Unité Propre de Recherche 2355, Centre National de la Recherche Scientifique, 91190 Gif-sur-Yvette,
                        France<br/><sup>b</sup> Laboratoire Génome et Développement des Plantes, Centre National de la Recherche Scientifique, Unité Mixte de Recherche 5096,
                        Université de Perpignan, 66860 Perpignan Cedex, France<br/><sup>c</sup> Instituto de Biologia Molecular y Celular de Plantas, Universidad Politécnica de Valencia, Consejo Superior de Investigaciones
                        Científicas, 46022 Valencia, Spain<br/><sup>d</sup> Physiologie Cellulaire Végétale, Unité Mixte de Recherche, Centre National de la Recherche Scientifique 5168, Commissariat
                        à l'Energie Atomique, Université Joseph Fourier, Institut National de la Recherche Agronomique, 38054 Grenoble Cedex 9, France
                        </span></p>
                  <p>
                     
                     <span style="  font-size: 83.33%"><sup><sup>2</sup></sup> To whom correspondence should be addressed. E-mail <span class="em-link"><span class="em-addr">francois.parcy{at}cea.fr</span></span>; fax 33-4-38-50-91.</span></p>
                  <p>
                     <a name="ABS" id="ABS"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        ABSTRACT
                        
                        
                     </h2>
                     
                  </div>
                  <div class="section abstract" id="abstract-1" itemprop="description">
                     
                     <p>
                        In <em>Arabidopsis</em> <em>thaliana</em>, four major regulators (ABSCISIC ACID INSENSITIVE3 [ABI3], FUSCA3 [FUS3], LEAFY COTYLEDON1 [LEC1], and LEC2) control most
                        aspects of seed maturation, such as accumulation of storage compounds, cotyledon identity, acquisition of desiccation tolerance,
                        and dormancy. The molecular basis for complex genetic interactions among these regulators is poorly understood. By analyzing
                        <em>ABI3</em> and <em>FUS3</em> expression in various single, double, and triple maturation mutants, we have identified multiple regulatory links among all
                        four genes. We found that one of the major roles of LEC2 was to upregulate <em>FUS3</em> and <em>ABI3</em>. The <em>lec2</em> mutation is responsible for a dramatic decrease in <em>ABI3</em> and <em>FUS3</em> expression, and most <em>lec2</em> phenotypes can be rescued by <em>ABI3</em> or <em>FUS3</em> constitutive expression. In addition, <em>ABI3</em> and <em>FUS3</em> positively regulate themselves and each other, thereby forming feedback loops essential for their sustained and uniform expression
                        in the embryo. Finally, <em>LEC1</em> also positively regulates <em>ABI3</em> and <em>FUS3</em> in the cotyledons. Most of the genetic controls discovered were found to be local and redundant, explaining why they had
                        previously been overlooked. This works establishes a genetic framework for seed maturation, organizing the key regulators
                        of this process into a hierarchical network. In addition, it offers a molecular explanation for the puzzling variable features
                        of <em>lec2</em> mutant embryos. 
                        
                     </p>
                     
                  </div>
                  <div class="auto-clean">
                     
                     
                     <a name="SEC1" id="SEC1"/>
                     
                     
                  </div>
                  <div class="section-nav">
                     
                     <h2>
                        INTRODUCTION
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     The conquest of most terrestrial niches by land plants has been greatly facilitated by the appearance of seeds (Steeves, 1983<a href="#BIB29"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Seeds offer plants a unique opportunity to interrupt their life cycle by withstanding adverse environmental conditions
                     in a desiccated state and then resuming growth using endogenous storage compounds. Seed-specific traits, such as desiccation
                     tolerance, reserve accumulation, and entry into quiescence, are acquired during a developmental phase called seed maturation
                     (Goldberg et al., 1994<a href="#BIB10"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Wobus and Weber, 1999<a href="#BIB39"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Vicente-Carbajosa and Carbonero, 2005<a href="#BIB34"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). In <em>Arabidopsis thaliana</em>, this phase is genetically controlled by at least four genes, <em>FUSCA3</em> (<em>FUS3</em>), <em>ABSCISIC ACID INSENSITIVE3</em> (<em>ABI3</em>), <em>LEAFY COTYLEDON1</em> (<em>LEC1</em>), and <em>LEC2. ABI3</em>, <em>FUS3</em>, and <em>LEC2</em> encode related plant-specific transcription factors containing the conserved B3 DNA binding domain (Giraudat et al., 1992<a href="#BIB9"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Luerssen et al., 1998<a href="#BIB18"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Stone et al., 2001<a href="#BIB30"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), whereas <em>LEC1</em> encodes a CBF transcription factor (Lotan et al., 1998<a href="#BIB17"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). All four <em>abi3</em>, <em>lec1</em>, <em>lec2</em>, and <em>fus3</em> mutants are severely affected in seed maturation and share some common phenotypes, such as reduced expression of seed storage
                     proteins (SSPs) (<a href="#TBL1">Table 1</a>
                     
                     ). However, they also show some specific phenotypes, such as the absence of chlorophyll degradation in the dry seed (<em>abi3</em>), a reduced sensitivity to abscisic acid (ABA) (<em>abi3</em> and, to a lesser extent, <em>lec1</em>), the accumulation of anthocyanins (<em>fus3</em>, <em>lec1</em>, and, to a lesser extent, <em>lec2</em>), an intolerance to desiccation (<em>abi3</em>, <em>fus3</em>, and <em>lec1</em>), or defects in cotyledon identity (<em>lec1</em>, <em>fus3</em>, and <em>lec2</em>) (Meinke, 1992<a href="#BIB20"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Bäumlein et al., 1994<a href="#BIB2"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Keith et al., 1994<a href="#BIB13"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Meinke et al., 1994<a href="#BIB21"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Parcy et al., 1994<a href="#BIB25"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>, 1997<a href="#BIB24"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; West et al., 1994<a href="#BIB38"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Nambara et al., 1995<a href="#BIB23"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Lotan et al., 1998<a href="#BIB17"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Luerssen et al., 1998<a href="#BIB18"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Vicient et al., 2000a<a href="#BIB35"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Raz et al., 2001<a href="#BIB26"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Stone et al., 2001<a href="#BIB30"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Kroj et al., 2003<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Despite numerous studies, the mechanisms through which these genes interact to control the various facets of seed maturation
                     remain poorly understood (Bäumlein et al., 1994<a href="#BIB2"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Keith et al., 1994<a href="#BIB13"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Meinke et al., 1994<a href="#BIB21"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; West et al., 1994<a href="#BIB38"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Parcy et al., 1997<a href="#BIB24"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Nambara et al., 2000<a href="#BIB22"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Vicient et al., 2000a<a href="#BIB35"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Raz et al., 2001<a href="#BIB26"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Since <em>lec2</em>, <em>fus3</em>, and <em>abi3</em> single mutants have similar but distinct phenotypic traits, which are additive in double mutants, <em>ABI3</em>, <em>LEC2</em>, and <em>FUS3</em> may work in parallel pathways (Keith et al., 1994<a href="#BIB13"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Meinke et al., 1994<a href="#BIB21"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; West et al., 1994<a href="#BIB38"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Other genetic analyses have also suggested the existence of interactions between <em>ABI3</em> and <em>FUS3</em> but without elucidating their molecular nature (Parcy et al., 1997<a href="#BIB24"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Nambara et al., 2000<a href="#BIB22"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Vicient et al., 2000a<a href="#BIB35"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Recently, LEC1 was shown to regulate expression of <em>ABI3</em> and <em>FUS3</em> (Kagaya et al., 2005<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Also, <em>FUS3</em> and <em>LEC2</em> were shown to act in a partially redundant manner to control SSP gene expression, and <em>LEC2</em> was shown to locally regulate <em>FUS3</em> expression in regions of the cotyledons (Kroj et al., 2003<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). These finding suggests that local and redundant regulation within this group of genes, which had been previously overlooked,
                     might be a central requisite for the correct establishment of seed maturation and prompted us to systematically analyze their
                     expression in various mutant combinations. 
                     
                  </p>
                  <p>
                     
                     <a name="TBL1" id="TBL1"/>
                     
                     
                  </p>
                  <div class="table pos-float">
                     
                     <div class="table-inline">
                        <strong>View this table:</strong><br/><a href="1642/T1.expansion.html">In this window</a><br/><a href="1642/T1.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="table-caption">
                        <span class="fig-label"><strong>Table 1.</strong></span> Main Phenotypes of Maturation Mutant Seeds 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     <a name="SEC2" id="SEC2"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        RESULTS
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     <strong><em>FUS3</em> Expression Is Regulated by <em>LEC2</em>, <em>ABI3</em>, and <em>FUS3</em></strong><br/>
                     
                     We tested the effect of the <em>abi3</em>, <em>fus3</em>, and <em>lec2</em> mutations, alone and in combination, on the expression of <em>ABI3</em>, <em>FUS3</em>, and <em>LEC2</em>. We performed whole-mount ß-glucuronidase (GUS) staining because this allowed direct visualization of the staining patterns
                     in a large number of embryos. For the <em>FUS3</em> gene, we used a <em>FUS3:GUS</em> reporter construct and analyzed its expression in mature embryos (10 d after pollination [DAP]) (<a href="#FIG1">Figure 1A</a>
                     
                     ). As demonstrated previously, <em>FUS3</em> expression is detectable throughout the various parts of the wild-type embryo (root tip, embryo axis, and cotyledons) (Kroj
                     et al., 2003<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Tsuchiya et al., 2004<a href="#BIB33"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). It was recently shown that this expression mainly originates from the protoderm of the embryo (Tsuchiya et al., 2004<a href="#BIB33"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Introduction of the <em>FUS3:GUS</em> reporter transgene in <em>abi3</em>, <em>fus3</em>, and <em>abi3 fus3</em> mutants showed that the spatial expression pattern of <em>FUS3</em> is not altered in these backgrounds (<a href="#FIG1">Figure 1A</a>). As previously demonstrated, <em>FUS3</em> expression is absent from sectors of <em>lec2</em> mutant cotyledons (<a href="#FIG1">Figure 1A</a>; Kroj et al., 2003<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Strong alterations of <em>FUS3</em> expression were revealed by crossing the <em>lec2</em> mutant with <em>abi3</em> or <em>fus3</em>. In the <em>lec2 abi3</em> mutant, <em>FUS3:GUS</em> expression was restricted to the root meristem; in <em>fus3 lec2</em>, it was confined to the embryo axis; and in the <em>fus3 lec2 abi3</em> triple mutant, <em>FUS3:GUS</em> expression was completely lost (<a href="#FIG1">Figure 1A</a>). Comparison of <em>fus3 lec2 abi3</em> with <em>fus3 lec2</em> shows that <em>ABI3</em> regulates <em>FUS3</em> in the embryo axis. Comparison of <em>lec2 abi3</em> with <em>lec2</em> shows that <em>ABI3</em> regulates <em>FUS3</em> expression in the embryo axis and in the cotyledons. Similarly, the effect of the <em>fus3</em> mutation in the <em>lec2</em> mutant (by comparing <em>lec2</em> with <em>lec2 fus3</em>) or in the <em>lec2 abi3</em> background (by comparing <em>lec2 abi3</em> with <em>lec2 abi3 fus3</em>) shows that <em>FUS3</em> positively regulates its own expression in the root meristem and the cotyledons. Finally, the effect of the <em>lec2</em> mutation in the <em>fus3</em>, <em>abi3</em>, or <em>fus3 abi3</em> backgrounds shows that <em>LEC2</em> regulates <em>FUS3</em> expression throughout the embryo. 
                     
                  </p>
                  <p>
                     
                     <a name="FIG1" id="FIG1"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="1642/F1.expansion.html"><img src="1642/F1.small.gif"/></a><br/><strong>View larger version</strong> (53K):<br/><a href="1642/F1.expansion.html">In this window</a><br/><a href="1642/F1.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 1.</strong></span> ABI3:GUS, FUS3:GUS, and LEC2:GUS Activities in Wild-Type and Mutant Embryos.
                        
                        <p>
                           
                           Expression patterns of <em>FUS3:GUS</em> <strong>(A)</strong>, <em>ABI3:GUS</em> <strong>(B)</strong>, and LEC2:GUS <strong>(C)</strong> in wild-type, <em>abi3</em> (<em>a3</em>), <em>fus3</em> (<em>f3</em>), <em>lec2</em> (<em>l2</em>), <em>fus3 abi3</em> (<em>fa</em>), <em>lec2 abi3</em> (<em>la</em>), <em>fus3 lec2</em> (<em>fl</em>), and <em>fus3 lec2 abi3</em> (<em>fla</em>) 10-DAP embryos and LEC2:GUS in 14-DAP embryos. The size and shape of the sectors with reduced <em>ABI3</em> or <em>FUS3</em> expression in <em>lec2</em> mutant embryos are extremely variable: two examples of phenotypes are shown for <em>ABI3:GUS</em> and one for <em>FUS3:GUS. LEC2:GUS</em> persistent expression is also variable and not always as pronounced as shown here for <em>lec2</em>, <em>fus3</em>, or <em>fus3 abi3</em>. Bars = 50 µm.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     We tried to confirm some of these expression patterns using in situ hybridization. As opposed to the signal obtained with
                     the highly expressed <em>Cruciferin C</em> (<em>CRC</em>) storage protein probe (<a href="#FIG2">Figure 2A</a>
                     
                     ), the <em>FUS3</em> probe in the wild-type background gave a very weak and uniform signal (data not shown) and was not restricted to the epidermis
                     as previously reported (Tsuchiya et al., 2004<a href="#BIB33"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). We concluded that our experimental conditions were not sufficient to detect the weak level of <em>FUS3</em> expression. However, based on in situ hybridization (Tsuchiya et al., 2004<a href="#BIB33"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), mutant complementation (Gazzarrini et al., 2004<a href="#BIB8"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), and functional evidence (Kroj et al., 2003<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), <em>FUS3</em> expression appears to be faithfully reproduced by the sensitive <em>FUS3:GUS</em> reporter. Furthermore, this construct offers the unique opportunity to monitor <em>FUS3</em> expression in the <em>fus3</em> mutant backgrounds (<em>fus3</em>, <em>fus3 abi3</em>, <em>fus3 lec2</em>, and <em>fus3 abi3 lec2</em>) and to establish <em>FUS3</em> autoregulation. 
                     
                  </p>
                  <p>
                     
                     <a name="FIG2" id="FIG2"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="1642/F2.expansion.html"><img src="1642/F2.small.gif"/></a><br/><strong>View larger version</strong> (101K):<br/><a href="1642/F2.expansion.html">In this window</a><br/><a href="1642/F2.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 2.</strong></span> Analysis of <em>ABI3</em> and <em>CRC</em> Expression by in Situ Hybridization in Developing Seeds.
                        
                        <p>
                           
                           <strong>(A)</strong> <em>CRC</em> storage protein gene expression was analyzed in wild-type and <em>lec2</em> backgrounds.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(B)</strong> <em>ABI3</em> expression analyzed in wild-type and <em>lec2</em> 10-DAP embryos. Note the presence of variable sectors devoid of staining.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(C)</strong> Expression of <em>ABI3</em> in early stages of embryo development.
                           
                        </p>
                        
                        <p>
                           
                           a, embryo axis; c, cotyledons. Bars = 100 µm in <strong>(A)</strong> and <strong>(B)</strong> and 20 µm in <strong>(C)</strong>.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     <strong><em>ABI3</em> Expression Is Regulated by <em>LEC2</em>, <em>ABI3</em>, and <em>FUS3</em></strong><br/>
                     
                     We also investigated <em>ABI3</em> expression using the <em>ABI3:GUS</em> construct. As previously reported (Parcy et al., 1994<a href="#BIB25"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Lara et al., 2003<a href="#BIB15"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), <em>ABI3</em> expression was detected throughout the embryo axis and cotyledons in wild-type seeds (<a href="#FIG1">Figure 1B</a>). The root meristem staining was usually weak. In the <em>lec2</em> mutant, the <em>ABI3</em> expression pattern was modified in the cotyledons where variable regions were devoid of staining (<a href="#FIG1">Figure 1B</a>). These regions varied in shape and size as described for <em>AT2S3</em>, <em>CRC</em>, and <em>FUS3</em> expression in <em>lec2</em> (<a href="#FIG2">Figure 2A</a>; Kroj et al., 2003<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). We concluded that <em>LEC2</em> also controls <em>ABI3</em> expression. <em>ABI3:GUS</em> expression was not reduced in <em>abi3</em> or <em>fus3</em> single mutants but appeared weaker in the periphery of the cotyledons of the <em>abi3 fus3</em> double mutant, suggesting that <em>ABI3</em> might also be regulated by <em>FUS3</em> and by <em>ABI3</em> itself. The effect of <em>abi3</em> and <em>fus3</em> mutations was very apparent in the <em>lec2</em> mutant backgrounds. <em>ABI3:GUS</em> expression was absent from the lateral parts of the cotyledons of the <em>fus3 lec2</em> mutants. The strong phenotypic variability observed in the <em>lec2</em> mutant was abolished in <em>fus3 lec2</em>, and the pattern shown in <a href="#FIG1">Figure 1B</a> was systematically observed, thereby demonstrating that <em>FUS3</em> is involved in <em>ABI3</em> regulation. In the <em>lec2 abi3</em> double mutant, <em>ABI3:GUS</em> expression was completely absent from the cotyledons, confirming that <em>ABI3</em> positively regulates its own expression in cotyledons. We complemented the analysis of <em>ABI3:GUS</em> transgenic plants by monitoring <em>ABI3</em> mRNA expression by in situ hybridization (<a href="#FIG2">Figures 2B</a> and <a href="#FIG2">2C</a>). In wild-type embryos, we confirmed that <em>ABI3</em> expression was uniform throughout the cotyledons and embryo axis (<a href="#FIG2">Figures 2B</a> and <a href="#FIG2">2C</a>). In <em>lec2</em> embryos, <em>ABI3</em> mRNA was absent from variable sectors of the cotyledons, exactly as shown by the <em>ABI3:GUS</em> constructs, thereby indicating that the <em>ABI3:GUS</em> construct faithfully reproduces the <em>ABI3</em> mRNA pattern. 
                     
                  </p>
                  <p>
                     
                     We also analyzed <em>LEC2:GUS</em> expression patterns in all single and double mutant backgrounds. In these experiments, we never observed any decrease in
                     LEC2:GUS staining at any developmental stages examined (from the globular stage to desiccating embryos). At late developmental
                     stages, when <em>LEC2</em> expression disappears from the wild-type cotyledons (Kroj et al., 2003<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), we sometimes observed a sustained LEC2:GUS activity in the <em>lec2</em>, <em>fus3</em>, and <em>lec2 fus3</em> mutant backgrounds, suggesting that <em>LEC2</em> might be repressed by FUS3 and LEC2 (<a href="#FIG1">Figure 1C</a>). Since we did not succeed in detecting <em>LEC2</em> expression by in situ hybridization and since no obvious phenotypes are associated with the potentially extended <em>LEC2</em> expression, we did not investigate this point further. 
                     
                  </p>
                  <p>
                     
                     <strong>Confirmation of the Regulatory Links with Functional Tests</strong><br/>
                     
                     Cross-regulation between ABI3, FUS3, and LEC2, as established from expression analyses in wild-type and mutant backgrounds,
                     is summarized in <a href="#FIG3">Figure 3A</a>. These regulatory controls are largely corroborated by phenotypic analysis. Indeed, phenotypes characteristic of <em>ABI3</em> (or <em>FUS3</em>) loss of function are obvious in several mutant backgrounds with a functional <em>ABI3</em> (or <em>FUS3</em>) gene. For instance, the appearance of chlorophyll in dry embryos, which is one signature of <em>ABI3</em> loss of function, was visible in sectors of <em>lec2</em> (<a href="#FIG3">Figure 3B</a>) and <em>fus3 lec2</em> mutants (data not shown), consistent with the <em>ABI3:GUS</em> expression pattern. Similarly, anthocyanin accumulation in cotyledons, a characteristic of <em>FUS3</em> loss of function, coincided with the loss of <em>FUS3:GUS</em> expression in <em>lec2</em> and <em>lec2 abi3</em> mutants (<a href="#FIG3">Figure 3B</a>; data not shown; Meinke et al., 1994<a href="#BIB21"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Also, storage protein gene expression, monitored using the <em>AT2S3:GFP</em> (green fluorescent protein) reporter (Kroj et al., 2003<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), precisely coincided with the domains expressing both <em>ABI3</em> and <em>FUS3</em> in the <em>lec2</em> mutant (Kroj et al., 2003<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; <a href="#FIG3">Figure 3B</a>). In summary, phenotypic data provide evidence for the functional significance of mutual regulation within a network involving
                     <em>LEC2</em>, <em>ABI3</em>, and <em>FUS3</em>. 
                     
                  </p>
                  <p>
                     
                     <a name="FIG3" id="FIG3"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="1642/F3.expansion.html"><img src="1642/F3.small.gif"/></a><br/><strong>View larger version</strong> (48K):<br/><a href="1642/F3.expansion.html">In this window</a><br/><a href="1642/F3.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 3.</strong></span> Network between B3 Transcription Factors (<em>LEC2</em>, <em>ABI3</em>, and <em>FUS3</em>).
                        
                        <p>
                           
                           <strong>(A)</strong> Positive interactions between <em>ABI3</em>, <em>FUS3</em>, and <em>LEC2</em> as deduced from <a href="#FIG1">Figure 1</a>.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(B)</strong> Spatial coincidence of the domains lacking <em>ABI3</em> or <em>FUS3</em> expression and the phenotypic traits in <em>lec2</em>, such as anthocyanin accumulation or reduced AT2S3:GFP fluorescence at 10 DAP or absence of chlorophyll degradation at 14
                           to 15 DAP.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(C)</strong> Rescue of the <em>lec2</em> mutant phenotypes by constitutive expression of <em>FUS3</em>. The same groups of 15-DAP seeds are shown under white or blue illumination to observe the seed shape and color (top panels)
                           and the AT2S3:GFP fluorescence (bottom panels). Dark spots on the <em>lec2</em> seeds result from the combined presence of anthocyanin and chlorophyll.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(D)</strong> Rescue of the <em>lec2</em> mutant phenotypes by constitutive expression of <em>ABI3.</em> Anthocyanin and chlorophyll accumulations in the <em>lec2</em> mutant are suppressed by the <em>35S:ABI3</em> transgene.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(E)</strong> Rescue of ABI3:GUS and FUS3:GUS expression in 10-DAP <em>lec2</em> embryos by <em>ABI3</em> or <em>FUS3</em> constitutive expression. The regions lacking <em>ABI3</em> or <em>FUS3</em> promoter activity observed in <em>lec2</em> mutant (<a href="#FIG1">Figure 1</a>) have disappeared when <em>ABI3</em> or <em>FUS3</em> are constitutively expressed.
                           
                        </p>
                        
                        <p>
                           
                           Bars = 100 µm in <strong>(B)</strong>, 500 µm in <strong>(C)</strong> and <strong>(D)</strong>, and 50 µm in <strong>(E)</strong>.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     Our model suggests that part of <em>LEC2</em> action might be indirect and that LEC2 controls several aspects of seed maturation, including the prevention of anthocyanin
                     and chlorophyll accumulation, through the positive regulation of <em>FUS3</em> and <em>ABI3</em> (<a href="#FIG3">Figure 3A</a>). If this model is correct, constitutive expression of <em>FUS3</em> or <em>ABI3</em> should complement some of the defects caused by the <em>LEC2</em> loss of function. As shown in <a href="#FIG3">Figures 3C</a> and <a href="#FIG3">3E</a>, the constitutive expression of <em>FUS3</em> (directed by the <em>35S</em> promoter) into <em>lec2 AT2S3:GFP</em> plants allowed a nearly complete phenotypic rescue of <em>lec2</em>, including the recovery of uniform <em>AT2S3:GFP</em> expression, the absence of anthocyanin and chlorophyll accumulation, and the suppression of ectopic trichomes (<a href="#TBL2">Table 2</a>
                     
                     ). <em>35S:FUS3</em> also restored a uniform <em>FUS3:GUS</em> expression (<a href="#FIG3">Figure 3E</a>). Only the irregular shape of some of the <em>lec2</em> embryo axes was unaffected by <em>FUS3</em> expression (<a href="#FIG3">Figure 3E</a>). Interestingly, constitutive expression of <em>FUS3</em> also complemented defects associated with the lack of <em>ABI3</em> (such as chlorophyll accumulation, which is not under FUS3 control), suggesting either that constitutively expressed <em>FUS3</em> can fulfill ABI3 functions or that FUS3 might be able to restore uniform <em>ABI3</em> expression in the <em>lec2</em> mutant. To test the latter possibility, we introduced an <em>ABI3:GUS</em> reporter construct into the <em>35S:FUS3 lec2</em> background and observed that <em>ABI3</em> expression indeed recovered a wild-type expression pattern in cotyledons (<a href="#FIG3">Figure 3E</a>). These experiments suggest that many defects present in the <em>lec2</em> mutant might indeed be indirectly due to the loss of <em>FUS3</em> and <em>ABI3</em> expression. They also show that <em>FUS3</em> can positively regulate its own expression and that of <em>ABI3</em>, thereby confirming the conclusions previously inferred from loss-of-function analysis (<a href="#FIG3">Figure 3A</a>). 
                     
                  </p>
                  <p>
                     
                     <a name="TBL2" id="TBL2"/>
                     
                     
                  </p>
                  <div class="table pos-float">
                     
                     <div class="table-inline">
                        <strong>View this table:</strong><br/><a href="1642/T2.expansion.html">In this window</a><br/><a href="1642/T2.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="table-caption">
                        <span class="fig-label"><strong>Table 2.</strong></span> Constitutive <em>FUS3</em> or <em>ABI3</em> Expression Suppresses Ectopic Trichome Formation in the <em>lec2</em> Mutant 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     As for <em>FUS3</em>, constitutive expression of <em>ABI3</em> in <em>lec2</em> suppressed chlorophyll and anthocyanin accumulation and ectopic trichome formation (<a href="#TBL2">Table 2</a>) and restored uniform <em>FUS3:GUS</em> expression in the <em>lec2</em> mutant embryos (<a href="#FIG3">Figures 3D</a> and <a href="#FIG3">3E</a>), confirming that <em>ABI3</em> positively regulates <em>FUS3 (</em><a href="#FIG3">Figure 3A</a>). It is thus likely that <em>ABI3</em> or <em>FUS3</em> are sufficient to complement most of the <em>lec2</em> mutant defects because constitutive expression of one of these two factors is sufficient to restore the uniform expression
                     of the other. 
                     
                  </p>
                  <p>
                     
                     <strong>LEC2 Is Involved in <em>FUS3</em> Initiation and <em>ABI3</em> Maintenance</strong><br/>
                     
                     Cotyledons of <em>lec2</em> exhibit variable sectors that are devoid of <em>ABI3</em> and <em>FUS3</em> expression. To explain this phenomenon, we reasoned that LEC2 might either play an essential role early on, during the initial
                     induction phase of <em>ABI3</em> and <em>FUS3</em> expression, or alternatively, that it might be required only later to ensure uniform <em>ABI3</em> and <em>FUS3</em> expression patterns. To distinguish between these two possibilities, we analyzed the expression of <em>ABI3</em> and <em>FUS3</em> in early stages of embryo development. In wild-type embryos, <em>ABI3</em> and <em>FUS3</em> expression is first detected at the globular stage. <em>ABI3</em> expression starts in the embryo proper at the globular stage (<a href="#FIG2">Figures 2C</a> and <a href="#FIG4">4A</a>
                     
                     
                     ), while <em>FUS3</em> expression initiates in the suspensor and, by the transition stage (between globular and heart stages), covers the whole
                     embryo (<a href="#FIG4">Figure 4B</a>; Parcy et al., 1994<a href="#BIB25"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Kroj et al., 2003<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Tsuchiya et al., 2004<a href="#BIB33"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). In the <em>lec2</em> mutant, <em>ABI3</em> expression was the same as that in the wild type in transition stage embryos (<a href="#FIG4">Figure 4A</a>), whereas <em>FUS3</em> expression was abolished in the embryo proper, though not in the suspensor (<a href="#FIG4">Figure 4B</a>). This experiment indicates that initiation of <em>FUS3</em> expression depends on LEC2, whereas that of <em>ABI3</em> does not. Later on, in the heart and torpedo stages of embryo development, <em>ABI3</em> expression started to fade from sectors of the cotyledons (<a href="#FIG4">Figure 4A</a>), while <em>FUS3</em> expression appeared in patches at variable locations (<a href="#FIG4">Figure 4B</a>). By the bent cotyledon stage, <em>ABI3</em> and <em>FUS3</em> expression coincided, as deduced from the fact that they both shared the same pattern as the <em>AT2S3:GFP</em> marker (<a href="#FIG3">Figure 3B</a>; Kroj et al., 2003<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Therefore, LEC2 is required early for <em>FUS3</em> initiation in the embryo proper and later to ensure ABI3 uniform expression (but not for <em>ABI3</em> initiation). Since ABI3 regulates <em>FUS3</em> in mature cotyledons, we wondered whether it was necessary for the appearance of <em>FUS3</em> in patches in heart stage embryos of the <em>lec2</em> mutant. We therefore analyzed <em>FUS3:GUS</em> expression in <em>lec2 abi3</em> heart and torpedo stages and did not detect any FUS3:GUS activity in the embryo axis or cotyledons (<a href="#FIG4">Figure 4B</a>), confirming that the late induction of <em>FUS3</em> in <em>lec2</em> mutants depends on <em>ABI3</em>. 
                     
                  </p>
                  <p>
                     
                     <a name="FIG4" id="FIG4"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="1642/F4.expansion.html"><img src="1642/F4.small.gif"/></a><br/><strong>View larger version</strong> (119K):<br/><a href="1642/F4.expansion.html">In this window</a><br/><a href="1642/F4.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 4.</strong></span> ABI3:GUS and FUS3:GUS Expression during Early Embryo Development.
                        
                        <p>
                           
                           <strong>(A)</strong> ABI3:GUS expression during wild-type and <em>lec2</em> early embryo development. In the <em>lec2</em> embryo, the ABI3:GUS expression pattern is the same as in wild-type embryos at globular stage but reduced in emerging cotyledons
                           from heart stage on.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(B)</strong> FUS3:GUS expression during wild-type and <em>lec2</em> early development. In the <em>lec2</em> embryo, FUS3:GUS expression does not uniformly appear in the globular stage embryo (as in wild-type embryos) but in patches
                           at heart stage. Patches are absent in the <em>lec2 abi3</em> double mutant.
                           
                        </p>
                        
                        <p>
                           
                           Bars = 50 µm.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     <strong><em>LEC1</em> Controls <em>ABI3</em> and <em>FUS3</em> Expression in Cotyledons</strong><br/><em>LEC1</em> is the fourth master regulator of seed maturation in addition to LEC2, FUS3, and ABI3 and encodes the CBF-A subunit of the
                     CCAAT binding trimeric transcription factor (Lotan et al., 1998<a href="#BIB17"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Based on single mutant phenotypes, <em>LEC1</em> had been proposed to positively regulate <em>LEC2</em> and <em>FUS3</em> and to act independently of <em>ABI3</em> (Meinke et al., 1994<a href="#BIB21"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Vicient et al., 2000a<a href="#BIB35"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Kagaya et al., 2005<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Recently, LEC1 has been proposed to regulate <em>ABI3</em> and <em>FUS3</em> expression (Kagaya et al., 2005<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). However, double mutant analyses showed that <em>ABI3</em>, <em>LEC2</em>, and <em>FUS3</em> were still active in the <em>lec1</em> background (West et al., 1994<a href="#BIB38"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Parcy et al., 1997<a href="#BIB24"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Raz et al., 2001<a href="#BIB26"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). We examined <em>ABI3</em>, <em>LEC2</em>, and <em>FUS3</em> promoter activity in the <em>lec1</em> mutant. We found a drastic local reduction of <em>FUS3:GUS</em> activity in cotyledons and root tips and a more moderate one for <em>ABI3:GUS</em> (<a href="#FIG5">Figure 5</a>
                     
                     ). <em>LEC2</em> expression did not decrease but showed a moderate increase in late stages as observed in <em>lec2</em> and <em>fus3</em> mutants (<a href="#FIG1">Figure 1</a>). Again, these modifications of reporter gene expression were consistent with local phenotypes. <em>lec1</em> mutant embryos showed a marked chlorophyll accumulation at the periphery of the cotyledons, where <em>ABI3</em> expression was most reduced (<a href="#FIG5">Figure 5</a>). <em>lec1</em> cotyledons have reduced ABA sensitivity, consistent with <em>ABI3</em> reduced expression (Parcy et al., 1997<a href="#BIB24"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Finally, the <em>lec1</em> mutant accumulated anthocyanin and showed reduced storage protein expression, consistent with the <em>FUS3</em> expression pattern (<a href="#FIG5">Figure 5</a>). This set of data indicates that LEC1 affects <em>ABI3</em> and <em>FUS3</em> expression locally (with a major effect on <em>FUS3</em> expression in cotyledons) and suggests some of the <em>lec1-</em>associated phenotypes might actually be indirectly due to a reduction in the expression of <em>ABI3</em> and <em>FUS3</em>. 
                     
                  </p>
                  <p>
                     
                     <a name="FIG5" id="FIG5"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="1642/F5.expansion.html"><img src="1642/F5.small.gif"/></a><br/><strong>View larger version</strong> (71K):<br/><a href="1642/F5.expansion.html">In this window</a><br/><a href="1642/F5.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 5.</strong></span> <em>lec1</em> Mutation Affects the <em>ABI3</em>, <em>FUS3</em>, and <em>LEC2</em> Expression Pattern.
                        
                        <p>
                           
                           All embryos are <em>lec1</em> mutants and show reduced <em>ABI3:GUS</em>, <em>FUS3:GUS</em>, and <em>AT2S3:GFP</em> expression at 10 DAP and increased chlorophyll fluorescence at 14 to 15 DAP. Chlorophyll was present mainly in the regions
                           where <em>ABI3:GUS</em> was reduced. <em>AT2S3:GFP</em> persisted in regions expressing <em>FUS3:GUS</em>. Bars = 100 µm.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     <a name="SEC3" id="SEC3"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        DISCUSSION
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     <strong>A Regulatory Network for Seed Maturation</strong><br/>
                     
                     In this study, we have unraveled the regulatory network linking ABI3, FUS3, LEC1, and LEC2, four major regulators of seed
                     maturation (<a href="#FIG6">Figure 6A</a>
                     
                     ). In a previous study, we showed that <em>LEC2</em> was necessary for uniform <em>FUS3</em> expression in parts of the cotyledons (Kroj et al., 2003<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). We show here that <em>FUS3</em> is controlled by a set of local and redundant regulations that vary spatially throughout the embryo and involve ABI3, LEC2,
                     and FUS3 itself. In the root tip, <em>FUS3</em> expression is redundantly controlled by <em>LEC2</em> and by <em>FUS3</em> itself; in the embryo axis, <em>FUS3</em> expression is redundantly controlled by <em>LEC2</em> and <em>ABI3</em>; in the cotyledons, <em>FUS3</em> expression is under the control of all three regulators (<a href="#FIG6">Figure 6B</a>). In addition to this set of regulatory controls, LEC1 also locally regulates <em>FUS3</em> expression in cotyledons. Our data are consistent with recently published analyses showing that LEC2 and LEC1 can induce
                     <em>FUS3</em> expression in vegetative tissues (Kagaya et al., 2005<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Santos Mendoza et al., 2005<a href="#BIB28"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) and that <em>FUS3</em> expression is reduced in whole-seed mRNA of the <em>lec1</em> mutant (Brocard et al., 2002<a href="#BIB5"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Kagaya et al., 2005<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     <a name="FIG6" id="FIG6"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="1642/F6.expansion.html"><img src="1642/F6.small.gif"/></a><br/><strong>View larger version</strong> (27K):<br/><a href="1642/F6.expansion.html">In this window</a><br/><a href="1642/F6.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 6.</strong></span> Gene Network Architecture.
                        
                        <p>
                           
                           <strong>(A)</strong> Schematic description of the network. Relations between the regulators are depicted as dashed lines, whereas proposed downstream
                           actions are show as solid lines. X represents the unknown factor inducing <em>ABI3</em> expression at the globular stage.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(B)</strong> Summary of <em>FUS3</em> spatial regulation by various combinations of the B3 transcription factors.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     We also showed that <em>ABI3</em> expression is controlled by LEC1, LEC2, FUS3, and by ABI3 itself in the developing cotyledons. The decreased <em>ABI3</em> expression in <em>fus3 lec2</em> and <em>fus3 abi3</em> double mutants is consistent with previously obtained ABI3 protein level data (Parcy et al., 1997<a href="#BIB24"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Kroj et al., 2003<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). In <em>lec1</em> mutants, a reduction of ABI3 expression was not systematically observed (Parcy et al., 1997<a href="#BIB24"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Kagaya et al., 2005<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). As suggested by Kagaya et al. (2005)<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>, this discrepancy very likely arose from differences in seed stages examined. In this study, we indeed observed that ABI3:GUS
                     was weaker in <em>lec1</em> embryos than in wild-type embryos at 10 DAP (<a href="#FIG5">Figure 5</a>) but close to the wild-type expression in desiccating seeds (14 to 15 DAP; data not shown), which is the stage used by Parcy
                     et al. (1997)<a href="#BIB24"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a> for protein analysis. Interestingly, <em>ABI3</em> regulation differs from that of <em>FUS3</em>, as it was not abolished in the embryo axis of the double mutants tested (<em>abi3 fus3</em>, <em>abi3 lec2</em>, and <em>fus3 lec2</em>). This expression therefore appears to be independent of FUS3 and LEC2. Consistent with this hypothesis, a 5' deletion of
                     the <em>ABI3</em> promoter that removed all RY elements (likely binding sites for LEC2 and FUS3) resulted in an ABI3:GUS pattern confined to
                     the embryo axis (F. Parcy, unpublished data). 
                     
                  </p>
                  <p>
                     
                     Several of the features of the regulatory network studied here clearly rendered its analysis difficult. First, the gene regulatory
                     controls in this network act locally, rather than on the whole embryo, thereby complicating genetic analyses that are often
                     based on whole embryo characteristics (such as gene expression). Second, functional redundancies exist (for example, between
                     ABI3 and FUS3 or between FUS3 and LEC2) that, again, are tissue dependent. <em>FUS3</em> expression is regulated redundantly by <em>ABI3</em> and <em>LEC2</em> in the embryo axis but by <em>LEC2</em> and <em>FUS3</em> in the root tip (<a href="#FIG6">Figure 6B</a>). Third, <em>FUS3</em> and <em>ABI3</em> autoregulate. Hints of such types of regulation were previously provided by the analysis of the weak <em>abi3-1</em> allele (Parcy et al., 1997<a href="#BIB24"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) or from the ectopic activation of seed regulators in vegetative tissues (Kagaya et al., 2005<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Santos Mendoza et al., 2005<a href="#BIB28"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). However, reporter constructs have been necessary to firmly establish these cases of autoregulation in strong mutant backgrounds.
                     This network illustrates the intricate nature of regulatory circuitry and the difficulty of interpretation of global analyses
                     of gene expression in whole seedlings or organs. 
                     
                  </p>
                  <p>
                     
                     <strong>Functional Implications of the Network</strong><br/>
                     
                     Our results, together with other published data (Kroj et al., 2003<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Kagaya et al., 2005<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), establish that LEC2 and LEC1 act on <em>ABI3</em> and <em>FUS3</em> expression. Moreover, we observed that <em>FUS3</em> or <em>ABI3</em> constitutive expression could rescue most aspects of the <em>lec2</em> mutant phenotype, suggesting that <em>ABI3</em> and <em>FUS3</em> act downstream of <em>LEC2</em>. However, we should point out that interpretation of this result is not straightforward because <em>ABI3</em>, <em>FUS3</em>, and <em>LEC2</em> encode homologous transcription factors, whose specificity might be altered by constitutive expression. For example, we found
                     that <em>35S:FUS3</em> suppressed chlorophyll accumulation, a function known to be normally performed by ABI3. Since we also showed that <em>35S:FUS3</em> was capable of restoring <em>ABI3</em> uniform expression in <em>lec2</em> cotyledons, the simplest explanation is that <em>FUS3</em> constitutive expression induces <em>ABI3</em>, which suppresses chlorophyll accumulation. However, we cannot exclude that constitutive expression of FUS3 acts on chlorophyll
                     independently of ABI3. One way to test this hypothesis would be to introduce the <em>35S:FUS3</em> transgene in a <em>lec2 abi3</em> double mutant and observe if chlorophyll accumulation remains. Similarly, it is conceivable that <em>FUS3</em> or <em>ABI3</em> constitutive expression complement the <em>lec2</em> mutant because of the lack of specificity. We do not favor this hypothesis because FUS3 and ABI3 only complement phenotypic
                     defects that are related to their endogenous function (such as chlorophyll breakdown, anthocyanin repression, or SSP gene
                     expression). The abnormal shape of the <em>lec2</em> mutant embryo axis is not complemented by <em>FUS3</em> or <em>ABI3</em> (<a href="#FIG3">Figure 3</a>). Also, neither <em>35S:ABI3</em> nor <em>35S:FUS</em>3 induces the formation of ectopic embryos as <em>LEC2</em> does (Stone et al., 2001<a href="#BIB30"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). These results suggest that ABI3, FUS3, and LEC2 retain some specificity even when overexpressed. We concluded that <em>LEC2</em> is upstream of <em>ABI3</em> and <em>FUS3</em> because (1) <em>ABI3</em> and <em>FUS3</em> expression are absent from sectors of <em>lec2</em> embryos; (2) some mutant phenotypes, typical of <em>ABI3</em> and <em>FUS3</em> loss of function, appear precisely in these sectors; and (3) these defects can be complemented by constitutive expression
                     of <em>ABI3</em> or <em>FUS3</em>. However, it is clear that LEC2 does not act only via ABI3 or FUS3. Several complementary studies have shown that LEC2 is
                     capable of activating directly 12S and 2S storage protein genes (Kroj et al., 2003<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Santos Mendoza et al., 2005<a href="#BIB28"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Braybrook et al., 2006<a href="#BIB4"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). It is thus likely that, in wild-type plants, LEC2 controls SSP gene expression through two different mechanisms: directly,
                     by binding the SSP promoter, and indirectly, by activating <em>FUS3</em> and <em>ABI3</em> as indicated in <a href="#FIG6">Figure 6A</a>. 
                     
                  </p>
                  <p>
                     
                     Expression analyses of <em>FUS3</em> and <em>ABI3</em> in <em>lec1</em> suggest that LEC1 also might indirectly control many aspects of seed maturation through regulation of <em>ABI3</em> and <em>FUS3</em> expression. Testing this hypothesis will require analyzing the extent to which the <em>lec1</em> mutant phenotype can be complemented by <em>ABI3</em> or <em>FUS3</em> constitutive expression, as we did for the <em>lec2</em> mutant. The question of whether LEC1 participates in ABA sensitivity has previously been a matter of debate (West et al.,
                     1994<a href="#BIB38"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Parcy et al., 1997<a href="#BIB24"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Brocard et al., 2002<a href="#BIB5"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Based on observation of radicle growth, <em>lec1</em> has been shown to be ABA sensitive, but <em>lec1</em> cotyledon expansion is partially insensitive to ABA (West et al., 1994<a href="#BIB38"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Parcy et al., 1997<a href="#BIB24"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). The observation that <em>ABI3</em> expression is predominantly reduced in cotyledons offers a simple explanation for this apparent contradiction: LEC1 might
                     affect ABA sensitivity only indirectly, through ABI3 regulation in the cotyledons. As recently suggested, LEC1 might also
                     control storage protein expression through <em>ABI3</em> and <em>FUS3</em> control (Kagaya et al., 2005<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Testing the direct and indirect target genes of LEC2 and LEC1 will require further experiments, possibly using versions
                     of these transcription factors that can be posttranslationally activated (Kagaya et al., 2005<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Santos Mendoza et al., 2005<a href="#BIB28"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     <strong>The Network Structure and Its Implications: A Confirmation of Theoretical Predictions?</strong><br/>
                     
                     The maturation network presented in <a href="#FIG6">Figure 6A</a> resembles modules that are known to drive developmental progression or patterning in other higher eukaryotes (Davidson et
                     al., 2002<a href="#BIB7"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>, 2003<a href="#BIB6"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Rudel and Sommer, 2003<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Levine and Davidson, 2005<a href="#BIB16"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). The network analyzed here appears to function to ensure a uniform expression of essential regulators, such as ABI3 and
                     FUS3, during the entire phase of seed maturation in a manner that is independent of their initial activation. Our analysis
                     at early developmental stages indicates that LEC2 induces <em>FUS3</em> in the embryo at the globular stage (<em>FUS3</em> is thus the earliest known LEC2 target gene), whereas <em>ABI3</em> is induced independently of LEC2 (by an unknown factor X depicted in <a href="#FIG6">Figure 6A</a>). However, as soon as the heart stage, mutual activation loops between ABI3 and FUS3 play a pivotal role in maintaining their
                     uniform expression patterns. This maintenance might be of particular importance toward the end of seed maturation, when <em>LEC2</em> expression decreases, and ABI3 and FUS3 are required to uniformly establish desiccation tolerance. 
                     
                  </p>
                  <p>
                     
                     Interestingly, the structure of the network revealed here provides a possible explanation for the observed phenotypic variability
                     of the <em>lec2</em> mutant (<a href="#FIG3">Figure 3C</a>; Meinke et al., 1994<a href="#BIB21"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Kroj et al., 2003<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Our analyses at early stages of embryo development allowed us to trace back the origin of the observed variable <em>lec2</em> defects to the heart stage, where <em>ABI3</em> expression disappears and <em>FUS3</em> expression appears in randomly positioned patches. These expression patterns seem surprising but are easily explained by
                     the structure of the network. Modules with autoregulatory loops have been the focus of intense studies, and theoretical analyses
                     have predicted that small variations (or noise) can be stabilized as a result of bistability generated by the network (Thattai
                     and van Oudenaarden, 2001<a href="#BIB32"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Blake et al., 2003<a href="#BIB3"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Isaacs et al., 2003<a href="#BIB11"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Applying these principles to the situation analyzed here, we can propose the following explanation. In <em>lec2</em> mutant embryo development, there might be a critical moment where <em>ABI3</em> initial induction is fading. At this moment, in some localized parts of the <em>lec2</em> embryo, ABI3 residual expression levels might be sufficient to trigger <em>FUS3</em> expression, thereby feeding into the positive loop and leading to stabilization of the expression of both regulators (first
                     stable state). In other parts of the <em>lec2</em> embryo, ABI3 may not succeed in inducing <em>FUS3</em>, leading to the local loss of expression of both regulators (second stable state). The observed phenotype (embryo regions
                     with both <em>FUS3</em> and <em>ABI3</em> expression, other regions lacking both) is the exact theoretical outcome of this bistable network (Thattai and van Oudenaarden,
                     2001<a href="#BIB32"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Blake et al., 2003<a href="#BIB3"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Isaacs et al., 2003<a href="#BIB11"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). To our knowledge, our observations are one of the few experimental confirmations of such predictions. Our results therefore
                     not only provide an explanation for a puzzling seed phenotype but also represent an experimental confirmation of theoretical
                     studies of general interest. 
                     
                  </p>
                  <p>
                     
                     <strong>Molecular Mechanism and Appearance in Evolution</strong><br/>
                     
                     What could be the molecular mechanisms underlying this network? Since ABI3, FUS3, and LEC2 are B3 transcription factors, and
                     since RY elements are present in the promoters of <em>FUS3</em> and <em>ABI3</em>, some of the regulatory controls might be direct and involve physical binding of the B3 transcription factor to the <em>FUS3</em> and <em>ABI3</em> promoters. However, FUS3 has been shown to act from the embryo epidermis and to regulate storage protein gene expression
                     in internal cell layers using ABA as a mobile mediator (Gazzarrini et al., 2004<a href="#BIB8"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Thus, this indirect mode of action could also apply to <em>ABI3</em> regulation by FUS3. This point will require further analysis. 
                     
                  </p>
                  <p>
                     
                     Finally, it would be interesting to understand when and how this regulatory network was first established over the course
                     of plant evolution. A likely mechanism (Wagner, 2001<a href="#BIB37"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Amoutzias et al., 2004<a href="#BIB1"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Teichmann and Babu, 2004<a href="#BIB31"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) for the generation of this complex regulatory network of B3 transcription factors could be the repeated duplication of an
                     ancestral B3 gene with autoregulatory properties, thereby generating three B3 transcription factors that regulate their own
                     and each other's expression. The existence of <em>ABI3</em> orthologs in monocotyledonous species (McCarty et al., 1991<a href="#BIB19"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), together with the recent discovery of <em>FUS3</em> and <em>LEC2</em> homologs in cereals (F. Parcy and J. Vicente-Carbajosa, unpublished data), suggests that this network may also function in
                     cereal grains and that it originated before the taxonomic split between the monocotyledons and eudicotyledons. Analysis of
                     components of the network in basal angiosperms or gymnosperms might allow the identification of the ancestral autoregulatory
                     gene. 
                     
                  </p>
                  <p>
                     
                     <a name="SEC4" id="SEC4"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        METHODS
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     <strong>Plant Material</strong><br/>
                     
                     We used the following <em>Arabidopsis thaliana</em> strains: <em>lec1-1</em> (Meinke, 1992<a href="#BIB20"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) in Wassilewskija, <em>AT2S3:GFP</em> in Columbia-0 (Col-0) (line FP91.54.3) (Kroj et al., 2003<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), and <em>lec2-1</em>, <em>abi3-6 gl1</em>, and <em>fus3-3 gl1</em> introduced in the <em>AT2S3:GFP</em> background (Kroj et al., 2003<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Except for <em>lec1-1</em> and <em>abi3 fus3 lec2</em> triple mutants, all strains used in this study (including those mentioned as wild types in the figures) contained the <em>AT2S3:GFP</em> transgene. <em>FUS3:GUS</em> (line 3.2 in Col-0), <em>LEC2:GUS</em> (line 3.2 in Col-0), and <em>ABI3:GUS</em> (line LAG3-4 in Landsberg <em>erecta</em>) have been described (Parcy et al., 1994<a href="#BIB25"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Kroj et al., 2003<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     <strong>Generation and Characterization of Transgenic Plants</strong><br/>
                     
                     A double enhanced 35S promoter–NOS terminator cassette (Parcy et al., 1994<a href="#BIB25"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) and a Gateway cassette (Invitrogen) were successively introduced in the pZP312 vector (pZP derivative conferring Basta resistance
                     in plants built by C. Fankhauser) to generate the pFP108-rfA plasmid in which <em>FUS3</em> and <em>ABI3</em> cDNAs were recombined according to Invitrogen's recommendations. <em>Arabidopsis</em> plants were grown and transformed as described (Kroj et al., 2003<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). The presence of mutations and transgenes was verified by PCR genotyping as described below. 
                     
                  </p>
                  <p>
                     
                     Nineteen <em>35S:FUS3</em> lines in <em>lec2 AT2S3:GFP</em> were generated. Visual inspection of the T2 seeds (under white and blue illumination) showed <em>lec2</em> phenotypic rescue in 15 of them. Four single-locus lines were followed until the homozygous stage and showed a degree of
                     rescue similar to the line <em>35S:FUS3</em> #19, shown in <a href="#FIG3">Figure 3</a> and used for crosses to <em>lec2 ABI3:GUS</em> and <em>lec2 FUS3:GUS. 35S:ABI3</em> (pFP111) was introduced directly into the <em>lec2 FUS3:GUS</em> background. Among 18 T1 lines generated, six showed apparent rescue of the <em>lec2</em> phenotype, and five were followed until homozygous stage and showed a similar degree of rescue as line FP111 #3 shown in
                     <a href="#FIG3">Figure 3</a>. 
                     
                  </p>
                  <p>
                     
                     <strong>Expression Analysis</strong><br/>
                     
                     The GUS staining and AT2S3:GFP fluorescence analyses were performed as described (Kroj et al., 2003<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Individual embryos (at least 20 for each genotype) were observed and stained in rudimentary flow cells to directly compare
                     AT2S3:GFP, GUS activity, and chlorophyll or anthocyanin accumulation as described by Kroj et al. (2003)<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. When needed, anthocyanin was washed in 90% acetone (to allow observation of chlorophyll), and chlorophyll was washed with
                     95% ethanol (observation of anthocyanin). In situ hybridization was performed as described by Lara et al. (2003)<a href="#BIB15"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a> using <em>CRC</em>, <em>ABI3</em>, or <em>FUS3</em> as probes except for <a href="#FIG2">Figure 2C</a> (performed as in Vicient et al., 2000b<a href="#BIB36"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     <strong>Counting of Trichomes on Cotyledons</strong><br/>
                     
                     To prevent ectopic trichomes on cotyledons to desiccate, siliques were harvested when still green (before the first sign of
                     valve dehiscence) and were surface sterilized, and seeds were dissected out and sowed on germination medium (Parcy et al.,
                     1994<a href="#BIB25"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Seedlings bearing trichomes were counted 5 to 7 d after stratification. Genotypes used were the following: <em>AT2S3:GFP</em> in Col-0 (indicated as wild type in <a href="#TBL2">Table 2</a>), <em>35S:FUS3 lec2 AT2S3:GFP</em> line 19, <em>lec2 AT2S3:GFP</em>, <em>35S:ABI3 FUS3:GUS lec2</em> line 8, and <em>35S:ABI3 FUS3:GUS</em> line 18. The wild type and the <em>lec2</em> mutant are presented twice because experiments were performed in two independent growth conditions. 
                     
                  </p>
                  <p>
                     
                     <strong>PCR Genotyping of Mutations and the Transgene</strong><br/>
                     
                     Transgenes and mutations were followed by PCR amplification of genomic DNA using the oligonucleotides listed in <a href="#TBL3">Table 3</a>
                     
                     . 
                     
                  </p>
                  <p>
                     
                     <a name="TBL3" id="TBL3"/>
                     
                     
                  </p>
                  <div class="table pos-float">
                     
                     <div class="table-inline">
                        <strong>View this table:</strong><br/><a href="1642/T3.expansion.html">In this window</a><br/><a href="1642/T3.expansion.html" class="in-nw">In a new window</a><br/> 
                        
                        
                     </div>
                     
                     <div class="table-caption">
                        <span class="fig-label"><strong>Table 3.</strong></span> Oligonucleotides Used for PCR Genotyping 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     <strong>Accession Numbers</strong><br/>
                     
                     Sequence data from this article can be found in the GenBank/EMBL data libraries under accession numbers <a href="/cgi/external_ref?access_num=At4g27160&amp;link_type=GEN">At4g27160</a> (<em>AT2S3</em>), <a href="/cgi/external_ref?access_num=At4g28520&amp;link_type=GEN">At4g28520</a> (<em>CRC</em>), <a href="/cgi/external_ref?access_num=At3g24650&amp;link_type=GEN">At3g24650</a> (<em>ABI3</em>), <a href="/cgi/external_ref?access_num=At3g26790&amp;link_type=GEN">At3g26790</a> (<em>FUS3</em>), <a href="/cgi/external_ref?access_num=At1g28300&amp;link_type=GEN">At1g28300</a> (<em>LEC2</em>), and <a href="/cgi/external_ref?access_num=At1g21970&amp;link_type=GEN">At1g21970</a> (<em>LEC1</em>). 
                     
                  </p>
                  <p>
                     
                     
                     <a name="ACK" id="ACK"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        Acknowledgments
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     We thank H. Baumlein, P. McCourt, J. Harada, and the ABRC for providing seeds and cDNAs, C. Fankauser and Plant Genetic Systems
                     for vectors, A. Marinière for generating <em>FUS3:GUS fus3 lec2 abi3</em>, M. Blazquez, J. Vicente, H. Meinhardt, and F. Berger for stimulating feedback, C. Scutt and R. Benlloch for proofreading,
                     and M. Blazquez and C. Ferrandiz for hosting F.P. at the Instituto de Biologia Molecular y Celular de Plantas. F.P. was supported
                     by the Centre National de la Recherche Scientifique and by a fellowship from the Spanish Ministry of Science, and G. Savino
                     was supported by the REGIA FP5 European Program. 
                     
                  </p>
                  <p>
                     
                     
                     <a name="FN" id="FN"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        Footnotes
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     <a name="FN1" id="FN1"/>
                     <sup>1</sup> These authors contributed equally to this work. <a href="#RFN1"><img width="12" height="12" alt="Back" src="/icons/back.gif"/></a></p>
                  <p>
                     <a name="" id=""/>
                     The author responsible for distribution of materials integral to the findings presented in this article in accordance with
                     the policy described in the Instructions for Authors (<a href="http://www.plantcell.org">www.plantcell.org</a>) is: François Parcy (<span class="em-link"><span class="em-addr">francois.parcy{at}cea.fr</span></span>). 
                     
                  </p>
                  <p>
                     
                     <a/>
                     Article, publication date, and citation information can be found at <a href="http://www.plantcell.org/cgi/doi/10.1105/tpc.105.039925">www.plantcell.org/cgi/doi/10.1105/tpc.105.039925</a>. 
                     
                  </p>
                  <p>
                     
                     
                     Received December  5, 2005; 
                     Revision received April 14, 2006. 
                     accepted May  3, 2006.
                     
                  </p>
                  <p>
                     
                     <a name="BIBL" id="BIBL"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        REFERENCES
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     
                     <a name="BIB1" id="BIB1"/><strong>Amoutzias, G.D., Robertson, D.L., Oliver, S.G., and Bornberg-Bauer, E.</strong> (2004). Convergent networks by single-gene duplications in higher eukaryotes. EMBO Rep. <strong>5,</strong> 274–279.<a href="/cgi/external_ref?access_num=10.1038/sj.embor.7400096&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000189344900013&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=14968135&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB2" id="BIB2"/><strong>Bäumlein, H., Miséra, S., Luerssen, H., Kölle, K., Horstmann, C., Wobus, U., and Müller, A.J.</strong> (1994). The <em>FUS3</em> gene of <em>Arabidopsis thaliana</em> is a regulator of gene expression during late embryogenesis. Plant J. <strong>6,</strong> 379–387.<a href="/cgi/external_ref?access_num=10.1046/j.1365-313X.1994.06030379.x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=A1994PF50000009&amp;link_type=ISI">[Web of Science]</a></p>
                  <p><a name="BIB3" id="BIB3"/><strong>Blake, W.J., Kaern, M, Cantor, C.R., and Collins, J.J.</strong> (2003). Noise in eukaryotic gene expression. Nature <strong>422,</strong> 633–637.<a href="/cgi/external_ref?access_num=10.1038/nature01546&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=12687005&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB4" id="BIB4"/><strong>Braybrook, S.A., Stone, S.L., Park, S., Bui, A.Q., Le, B.H., Fischer, R.L., Goldberg, R.B., and Harada, J.J.</strong> (2006). Genes directly regulated by LEAFY COTYLEDON2 provide insight into the control of embryo maturation and somatic embryogenesis.
                     Proc. Natl. Acad. Sci. USA <strong>103,</strong> 3468–3473.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=103/9/3468">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB5" id="BIB5"/><strong>Brocard, I.M., Lynch, T.J., and Finkelstein, R.R.</strong> (2002). Regulation and role of the Arabidopsis abscisic acid-insensitive 5 gene in abscisic acid, sugar, and stress response.
                     Plant Physiol. <strong>129,</strong> 1533–1543.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=129/4/1533">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB6" id="BIB6"/><strong>Davidson, E.H., McClay, D.R., and Hood, L.</strong> (2003). Regulatory gene networks and the properties of the developmental process. Proc. Natl. Acad. Sci. USA <strong>100,</strong> 1475–1480.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=100/4/1475">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB7" id="BIB7"/><strong>Davidson, E.H., et al.</strong> (2002). A genomic regulatory network for development. Science <strong>295,</strong> 1669–1678.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=295/5560/1669">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB8" id="BIB8"/><strong>Gazzarrini, S., Tsuchiya, Y., Lumba, S., Okamoto, M., and McCourt, P.</strong> (2004). The transcription factor FUSCA3 controls developmental timing in <em>Arabidopsis</em> through the hormones gibberellin and abscisic acid. Dev. Cell <strong>7,</strong> 373–385.<a href="/cgi/external_ref?access_num=10.1016/j.devcel.2004.06.017&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000224082900012&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=15363412&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB9" id="BIB9"/><strong>Giraudat, J., Hauge, B.M., Valon, C., Smalle, J., Parcy, F., and Goodman, H.M.</strong> (1992). Isolation of the <em>Arabidopsis ABI3</em> gene by positional cloning. Plant Cell <strong>4,</strong> 1251–1261.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=4/10/1251">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB10" id="BIB10"/><strong>Goldberg, R.B., de Paiva, G., and Yadegari, R.</strong> (1994). Plant embryogenesis: Zygote to seed. Science <strong>266,</strong> 605–614.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=266/5185/605">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB11" id="BIB11"/><strong>Isaacs, F.J., Hasty, J., Cantor, C.R., and Collins, J.J.</strong> (2003). Prediction and measurement of an autoregulatory genetic module. Proc. Natl. Acad. Sci. USA <strong>100,</strong> 7714–7719.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=100/13/7714">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB12" id="BIB12"/><strong>Kagaya, Y., Toyoshima, R., Okuda, R., Usui, H., Yamamoto, A., and Hattori, T.</strong> (2005). <em>LEAFY COTYLEDON1</em> controls seed storage protein genes through its regulation of <em>FUSCA3</em> and <em>ABSCISIC ACID INSENSITIVE3.</em> Plant Cell Physiol. <strong>46,</strong> 399–406.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pcellphys&amp;resid=46/3/399">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB13" id="BIB13"/><strong>Keith, K., Kraml, M., Dengler, N.G., and McCourt, P.</strong> (1994). <em>fusca3</em>: A heterochronic mutation affecting late embryo development in <em>Arabidopsis.</em> Plant Cell <strong>6,</strong> 589–600.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=6/5/589">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB14" id="BIB14"/><strong>Kroj, T., Savino, G., Valon, C., Giraudat, J., and Parcy, F.</strong> (2003). Regulation of storage protein gene expression in <em>Arabidopsis.</em> Development <strong>130,</strong> 6065–6073.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=130/24/6065">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB15" id="BIB15"/><strong>Lara, P., Onate-Sanchez, L., Abraham, Z., Ferrandiz, C., Diaz, I., Carbonero, P., and Vicente-Carbajosa, J.</strong> (2003). Synergistic activation of seed storage protein gene expression in <em>Arabidopsis</em> by ABI3 and two bZIPs related to OPAQUE2. J. Biol. Chem. <strong>278,</strong> 21003–21011.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=jbc&amp;resid=278/23/21003">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB16" id="BIB16"/><strong>Levine, M., and Davidson, E.H.</strong> (2005). Gene regulatory networks for development. Proc. Natl. Acad. Sci. USA <strong>102,</strong> 4936–4942.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=102/14/4936">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB17" id="BIB17"/><strong>Lotan, T., Ohto, M., Yee, K.M., West, M.A., Lo, R., Kwong, R.W., Yamagishi, K., Fischer, R.L., Goldberg, R.B., and Harada,
                        J.J.</strong> (1998). <em>Arabidopsis</em> LEAFY COTYLEDON1 is sufficient to induce embryo development in vegetative cells. Cell <strong>93,</strong> 1195–1205.<a href="/cgi/external_ref?access_num=10.1016/S0092-8674(00)81463-4&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000074491100014&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=9657152&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB18" id="BIB18"/><strong>Luerssen, H., Kirik, V., Herrmann, P., and Misera, S.</strong> (1998). <em>FUSCA3</em> encodes a protein with a conserved VP1/AB13-like B3 domain which is of functional importance for the regulation of seed maturation
                     in <em>Arabidopsis thaliana.</em> Plant J. <strong>15,</strong> 755–764.<a href="/cgi/external_ref?access_num=10.1046/j.1365-313X.1998.00259.x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000076587300004&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=9807814&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB19" id="BIB19"/><strong>McCarty, D.R., Hattori, T., Carson, C.B., Vasil, V., Lazar, M., and Vasil, I.K.</strong> (1991). The <em>viviparous-1</em> developmental gene of maize encodes a novel transcriptional activator. Cell <strong>66,</strong> 895–905.<a href="/cgi/external_ref?access_num=10.1016/0092-8674(91)90436-3&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=A1991GE46000009&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=1889090&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB20" id="BIB20"/><strong>Meinke, D.W.</strong> (1992). A homeotic mutant of <em>Arabidopsis thaliana</em> with leafy cotyledons. Science <strong>258,</strong> 1647–1650.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=258/5088/1647">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB21" id="BIB21"/><strong>Meinke, D.W., Franzmann, L.H., Nickle, T.C., and Yeung, E.C.</strong> (1994). <em>Leafy cotyledon</em> mutants of <em>Arabidopsis.</em> Plant Cell <strong>6,</strong> 1049–1064.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=6/8/1049">[Abstract]</a></p>
                  <p><a name="BIB22" id="BIB22"/><strong>Nambara, E., Hayama, R., Tsuchiya, Y., Nishimura, M., Kawaide, H., Kamiya, Y., and Naito, S.</strong> (2000). The role of <em>ABI3</em> and <em>FUS3</em> loci in <em>Arabidopsis thaliana</em> on phase transition from late embryo development to germination. Dev. Biol. <strong>220,</strong> 412–423.<a href="/cgi/external_ref?access_num=10.1006/dbio.2000.9632&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000086633600024&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=10753527&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB23" id="BIB23"/><strong>Nambara, E., Keith, K., McCourt, P., and Naito, S.</strong> (1995). A regulatory role for the <em>ABI3</em> gene in the establishment of embryo maturation in <em>Arabidopsis thaliana.</em> Development <strong>121,</strong> 629–636.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=121/3/629">[Abstract]</a></p>
                  <p><a name="BIB24" id="BIB24"/><strong>Parcy, F., Valon, C., Kohara, A., Miséra, S., and Giraudat, J.</strong> (1997). The <em>ABSCISIC ACID-INSENSITIVE 3</em> (<em>ABI3</em>), <em>FUSCA 3</em> (<em>FUS3</em>) and <em>LEAFY COTYLEDON 1</em> (<em>LEC1</em>) loci act in concert to control multiple aspects of <em>Arabidopsis</em> seed development. Plant Cell <strong>9,</strong> 1265–1277.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=9/8/1265">[Abstract]</a></p>
                  <p><a name="BIB25" id="BIB25"/><strong>Parcy, F., Valon, C., Raynal, M., Gaubier-Comella, P., Delseny, M., and Giraudat, J.</strong> (1994). Regulation of gene expression programs during <em>Arabidopsis</em> seed development: Roles of the <em>ABI3</em> locus and of endogenous abscisic acid. Plant Cell <strong>6,</strong> 1567–1582.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=6/11/1567">[Abstract]</a></p>
                  <p><a name="BIB26" id="BIB26"/><strong>Raz, V., Bergervoet, J.H., and Koornneef, M.</strong> (2001). Sequential steps for developmental arrest in <em>Arabidopsis</em> seeds. Development <strong>128,</strong> 243–252.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=128/2/243">[Abstract]</a></p>
                  <p><a name="BIB27" id="BIB27"/><strong>Rudel, D., and Sommer, R.J.</strong> (2003). The evolution of developmental mechanisms. Dev. Biol. <strong>264,</strong> 15–37.<a href="/cgi/external_ref?access_num=10.1016/S0012-1606(03)00353-1&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000186723300002&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=14623229&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB28" id="BIB28"/><strong>Santos Mendoza, M., Dubreucq, B., Miquel, M., Caboche, M., and Lepiniec, L.</strong> (2005). <em>LEAFY COTYLEDON 2</em> activation is sufficient to trigger the accumulation of oil and seed specific mRNAs in <em>Arabidopsis</em> leaves. FEBS Lett. <strong>579,</strong> 4666–4670.<a href="/cgi/external_ref?access_num=000231625800019&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=16107256&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB29" id="BIB29"/><strong>Steeves, T.A.</strong> (1983). The evolution and biological significance of seeds. Can. J. Bot. <strong>61,</strong> 3550–3560.
                     
                  </p>
                  <p><a name="BIB30" id="BIB30"/><strong>Stone, S.L., Kwong, L.W., Yee, K.M., Pelletier, J., Lepiniec, L., Fischer, R.L., Goldberg, R.B., and Harada, J.J.</strong> (2001). <em>LEAFY COTYLEDON2</em> encodes a B3 domain transcription factor that induces embryo development. Proc. Natl. Acad. Sci. USA <strong>98,</strong> 11806–11811.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=98/20/11806">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB31" id="BIB31"/><strong>Teichmann, S.A., and Babu, M.M.</strong> (2004). Gene regulatory network growth by duplication. Nat. Genet. <strong>36,</strong> 492–496.<a href="/cgi/external_ref?access_num=10.1038/ng1340&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000221183000022&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=15107850&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB32" id="BIB32"/><strong>Thattai, M., and van Oudenaarden, A.</strong> (2001). Intrinsic noise in gene regulatory networks. Proc. Natl. Acad. Sci. USA <strong>98,</strong> 8614–8619.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=98/15/8614">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB33" id="BIB33"/><strong>Tsuchiya, Y., Nambara, E., Naito, S., and McCourt, P.</strong> (2004). The FUS3 transcription factor functions through the epidermal regulator TTG1 during embryogenesis in <em>Arabidopsis.</em> Plant J. <strong>37,</strong> 73–81.<a href="/cgi/external_ref?access_num=10.1046/j.1365-313X.2003.01939.x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000187257500007&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=14675433&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB34" id="BIB34"/><strong>Vicente-Carbajosa, J., and Carbonero, P.</strong> (2005). Seed maturation: Developing an intrusive phase to accomplish a quiescent state. Int. J. Dev. Biol. <strong>49,</strong> 645–651.<a href="/cgi/external_ref?access_num=10.1387/ijdb.052046jc&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000233266200017&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=16096971&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB35" id="BIB35"/><strong>Vicient, C.M., Bies-Etheve, N., and Delseny, M.</strong> (2000a). Changes in gene expression in the <em>leafy cotyledon1</em> (<em>lec1</em>) and <em>fusca3</em> (<em>fus3</em>) mutants of <em>Arabidopsis thaliana</em> L. J. Exp. Bot. <strong>51,</strong> 995–1003.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=jexbot&amp;resid=51/347/995">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB36" id="BIB36"/><strong>Vicient, C.M., Hull, G., Guilleminot, J., Devic, M., and Delseny, M.</strong> (2000b). Differential expression of the <em>Arabidopsis</em> genes coding for Em-like proteins. J. Exp. Bot. <strong>51,</strong> 1211–1220.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=jexbot&amp;resid=51/348/1211">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB37" id="BIB37"/><strong>Wagner, A.</strong> (2001). Birth and death of duplicated genes in completely sequenced eukaryotes. Trends Genet. <strong>17,</strong> 237–239.<a href="/cgi/external_ref?access_num=10.1016/S0168-9525(01)02243-0&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000168718500003&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=11335019&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB38" id="BIB38"/><strong>West, M.A.L., Matsuderai Yee, K., Danao, J., Zimmerman, J.L., Fisher, R.L., Goldberg, R.B., and Harada, J.J.</strong> (1994). <em>LEAFY COTYLEDON1</em> is an essential regulator of late embryogenesis and cotyledon identity in <em>Arabidopsis.</em> Plant Cell <strong>6,</strong> 1731–1745.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=6/12/1731">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB39" id="BIB39"/><strong>Wobus, U., and Weber, H.</strong> (1999). Seed maturation: Genetic programmes and control signals. Curr. Opin. Plant Biol. <strong>2,</strong> 33–38.<a href="/cgi/external_ref?access_num=11658141&amp;link_type=MED">[Medline]</a></p>
               </div>
      <span class="highwire-journal-article-marker-end"/>
    </div>
  </body>
</html>
