<?xml version="1.0"?>
<html>
  <body>
    <div itemprop="articleBody">
      <span class="highwire-journal-article-marker-start"/>
      <div class="hw-optimized-content">
                  <div class="auto-clean">
                     
                     
                     
                     
                     <span style="  font-size: 83.33%"><em>The Plant Cell</em> 21:1747-1761 (2009)<br/>
                        © 2009 <a href="/misc/terms.shtml">American Society of Plant Biologists</a><br/></span></div>
                  <h1 id="article-title-1" itemprop="headline">
                     A Pivotal Role of the Basic Leucine Zipper Transcription Factor bZIP53 in the Regulation of <em>Arabidopsis</em> Seed Maturation Gene Expression Based on Heterodimerization and Protein Complex Formation<a name="RFN2" id="RFN2"/><sup><a href="#FN2">[W]</a></sup></h1>
                  <div class="contributors">
                     Rosario Alonso<sup>a</sup>, 
                     Luis Oñate-Sánchez<sup>a</sup>, 
                     Fridtjof Weltmeier<sup>b</sup>, 
                     Andrea Ehlert<sup>b</sup>, 
                     Isabel Diaz<sup>a</sup>, 
                     Katrin Dietrich<sup>b</sup>, 
                     Jesús Vicente-Carbajosa<sup>a</sup><a name="RFN1" id="RFN1"/><sup>,<a href="#FN1">1</a></sup> and 
                     Wolfgang Dröge-Laser<sup>b</sup><a/><sup>,<a href="#FN1">1</a></sup><sup>,2</sup></div>
                  <p>
                     <span style="  font-size: 83.33%"><sup>a</sup> Centro de Biotecnología y Genómica de Plantas Universidad Politécnica de Madrid, Escuela Técnica Superior Ingenieros Agrónomos,
                        28223 Pozuelo de Alarcón, Madrid, Spain<br/><sup>b</sup> Albrecht-von-Haller Institut, Universität Göttingen, D-37073 Göttingen, Germany
                        </span></p>
                  <p>
                     
                     <span style="  font-size: 83.33%"><sup><sup>2</sup></sup> Address correspondence to <span class="em-link"><span class="em-addr">wdroege{at}gwdg.de</span></span>.</span></p>
                  <p>
                     <a name="ABS" id="ABS"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        ABSTRACT
                        
                        
                     </h2>
                     
                  </div>
                  <div class="section abstract" id="abstract-1" itemprop="description">
                     
                     <p>
                        Transcription of <em>Arabidopsis thaliana</em> seed maturation (MAT) genes is controlled by members of several transcription factor families, such as basic leucine zippers
                        (bZIPs), B3s, MYBs, and DOFs. In this work, we identify <em>Arabidopsis</em> bZIP53 as a novel transcriptional regulator of MAT genes. <em>bZIP53</em> expression in developing seeds precedes and overlaps that of its target genes. Gain- and loss-of-function approaches indicate
                        a correlation between the amount of bZIP53 protein and MAT gene expression. Specific in vivo and in vitro binding of bZIP53
                        protein to a G-box element in the albumin <em>2S2</em> promoter is demonstrated. Importantly, heterodimerization with bZIP10 or bZIP25, previously described bZIP regulators of
                        MAT gene expression, significantly enhances DNA binding activity and produces a synergistic increase in target gene activation.
                        Full-level target gene activation is strongly correlated with the ratio of the correspondent bZIP heterodimerization partners.
                        Whereas bZIP53 does not interact with ABI3, a crucial transcriptional regulator in <em>Arabidopsis</em> seeds, ternary complex formation between the bZIP heterodimers and ABI3 increases the expression of MAT genes in planta.
                        We therefore propose that heterodimers containing bZIP53 participate in enhanceosome formation to produce a dramatic increase
                        in MAT gene transcription. 
                        
                     </p>
                     
                  </div>
                  <div class="auto-clean">
                     
                     
                     <a name="SEC1" id="SEC1"/>
                     
                     
                  </div>
                  <div class="section-nav">
                     
                     <h2>
                        INTRODUCTION
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     As an outstanding adaptation of terrestrial plants, seed formation favors dispersal of species and allows the interruption
                     of the life cycle and its resumption once optimal growth conditions are newly established (for review, see Vicente-Carbajosa
                     and Carbonero, 2005<a href="#BIB49"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Weber et al., 2005<a href="#BIB52"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Santos-Mendoza et al., 2008<a href="#BIB41"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Seeds are formed after a double fertilization event, triggering the development of a complex organ, which comprises the
                     embryo, the endosperm, and the seed coat derived from the integuments and other surrounding layers of maternal origin. Seed
                     development can be divided into three phases: first, embryogenesis is characterized by cell division and differentiation until
                     embryo morphology is established. Second, the maturation phase is dominated by storage compound accumulation, growth arrest,
                     and acquisition of desiccation tolerance. Third, the embryo can enter into a dormancy state that is broken upon germination.
                     With respect to seed morphology, physiology, and gene regulation, considerable variations occur among species. <em>Arabidopsis thaliana</em> has been developed as a well-established model system for dicot seed development, and several similarities and differences
                     with monocot model systems have been described (Vicente-Carbajosa and Carbonero, 2005<a href="#BIB49"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Santos-Mendoza et al., 2008<a href="#BIB41"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     Important programs of gene expression related to the metabolic changes that occur during seed maturation are highly coordinated
                     and tightly regulated (Gutierrez et al., 2007<a href="#BIB17"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). An understanding of gene expression control in the seed was tackled from early studies in plant molecular biology, with
                     maize (<em>Zea mays</em>) Opaque2 (O2) representing a hallmark as one of the first plant transcription factor (TF) genes cloned and characterized
                     (Hartings et al., 1989<a href="#BIB20"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Schmidt et al., 1990<a href="#BIB44"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Similarly, orthologous genes from wheat (<em>Triticum aestivum</em>) (SPA) and barley (<em>Hordeum vulgare</em>) (BLZ2) play the same roles as O2 in their corresponding species (Albani et al., 1997<a href="#BIB1"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Oñate et al., 1999<a href="#BIB35"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). In dicot species, key TFs have been characterized that control gene expression programs during seed maturation. 
                     
                  </p>
                  <p>
                     
                     The class of maturation genes (MAT) expressed during seed maturation typically includes seed storage protein (SSP) genes,
                     such as albumin and cruciferin genes, which are induced in early or mid-maturation phase. The late embryogenesis abundant
                     (LEA) genes are induced at later stages of maturation and include genes proposed to be involved in acquisition of desiccation
                     tolerance (for review, see Tunnacliffe and Wise, 2007<a href="#BIB47"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). MAT promoter analyses have revealed several conserved <em>cis-</em>regulatory elements with functional relevance in the control of gene expression during seed maturation. Among them, G-box-related
                     ACGT elements, RY (CATGCA), AACA, and CTTT motifs are the best described examples (for review, see Vicente-Carbajosa and Carbonero,
                     2005<a href="#BIB49"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). The corresponding associated TFs belong to the basic leucine zipper (bZIP), B3, MYB, and DOF TF families, respectively.
                     Cooperation of these regulatory units in the control of gene expression appears to be an evolutionarily conserved pattern
                     that can be traced back to the origins of the Spermaphyta (Vicente-Carbajosa and Carbonero, 2005<a href="#BIB49"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Schallau et al., 2008<a href="#BIB43"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     TFs of the bZIP class, related to cereal O2-type TFs, have been identified in <em>Arabidopsis</em> (Lara et al., 2003<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), namely, bZIP10 and bZIP25, which have been classified into group C of the <em>Arabidopsis</em> bZIP TF family (Jakoby et al., 2002<a href="#BIB22"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Expression during seed development, specific binding to G-box-like ACGT elements of the albumin and cruciferin promoters,
                     and in vivo regulation of these target genes have been demonstrated (Lara et al., 2003<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). A peculiarity exists in that none of the four genes in the bZIP group C in <em>Arabidopsis</em> shows seed-specific expression, in contrast with characterized members of the cereal O2-type TFs. This suggests that in <em>Arabidopsis</em>, combinatorial interactions and expression levels of different TFs may be of major relevance in the induction of seed-specific
                     gene expression patterns. 
                     
                  </p>
                  <p>
                     
                     ABSCISIC ACID INSENSITIVE3 (ABI3), FUSCA3 (FUS3), and LEAFY COTYLEDON2 (LEC2) have been implicated in seed development and
                     belong to the B3 family of TFs (Parcy et al., 1994<a href="#BIB39"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Kroj et al., 2003<a href="#BIB25"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Braybrook et al., 2006<a href="#BIB6"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Santos-Mendoza et al., 2008<a href="#BIB41"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Mutations in any of these genes result in pleiotropic phenotypes affecting SSP accumulation and acquisition of desiccation
                     tolerance. Recent genetic analysis demonstrated that these B3-type TFs are controlled by a complex self-regulating network
                     including LEC1, LEC2, FUS3, and ABI3 (To et al., 2006<a href="#BIB46"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). The corresponding genes clearly differ in expression and function despite being partially redundant. Although binding of
                     ABI3, FUS3, and LEC2 to the RY element, mediated by the B3 DNA binding domain, has been shown in vitro and in yeast systems
                     (Kroj et al., 2003<a href="#BIB25"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Mönke et al., 2004<a href="#BIB29"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Braybrook et al., 2006<a href="#BIB6"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), RY motifs are not sufficient to confer seed-specific expression patterns to a target promoter (Ezcurra et al., 2000<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Nakashima et al., 2006<a href="#BIB32"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Cooperation with neighboring <em>cis</em>-elements and the cognate TFs has been demonstrated to be important to establish seed-specific transcriptional patterns. bZIP10
                     and bZIP25 physically interact with ABI3, which in turn enhances in vitro DNA binding of the bZIP proteins to SSP promoters
                     as well as their in vivo activation capacity (Lara et al., 2003<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Therefore, cooperation of transcriptional regulators by protein–protein interactions provides an efficient mechanism to
                     control gene expression in seeds and explains some of the molecular mechanisms underlying SSP gene expression. However, our
                     knowledge on the protein partners and their interplay remains limited. 
                     
                  </p>
                  <p>
                     
                     In general, bZIP TFs bind DNA as homo- or heterodimers (Ellenberger et al., 1992<a href="#BIB11"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Recently, we demonstrated that group C bZIPs, such as bZIP10 and bZIP25, preferentially interact with group S1 bZIPs, resulting
                     in a set of specific heterodimers designated the C/S1 network of bZIP TFs (Ehlert et al., 2006<a href="#BIB10"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). In particular, members of the C/S1 network have been implicated in energy homeostasis (Baena-Gonzalez et al., 2007<a href="#BIB2"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Usadel et al., 2008<a href="#BIB48"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), amino acid metabolism (Weltmeier et al., 2006<a href="#BIB54"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Hanson et al., 2008<a href="#BIB19"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), stress response (Kaminaka et al., 2006<a href="#BIB23"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), and sink-specific gene expression (Rook et al., 1998<a href="#BIB40"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Weltmeier et al., 2009<a href="#BIB55"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Importantly, these heterodimers were shown to convey synergistic activation properties to target genes, suggesting that
                     heterodimerization serves as an efficient mechanism of signal integration (Weltmeier et al., 2006<a href="#BIB54"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). With respect to the regulation of the seed gene <em>Em</em>, the importance of bZIP heterodimers and the ABI3 othologous regulator VP1 has already been proposed in rice (<em>Oryza sativa</em>) (Nantel and Quatrano, 1996<a href="#BIB33"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     In this work, we aimed to identify S1 bZIP members that could be involved in seed gene regulation as important potential partners
                     of previously described group C bZIPs. We define the S1 TF bZIP53 as a key regulator of MAT gene transcription. bZIP53 enhances
                     MAT gene expression by specific heterodimerization with bZIP10 or bZIP25. Furthermore, these bZIP heterodimers interact with
                     ABI3, which further increases MAT gene activation. Therefore, we propose that bZIP53 plays a pivotal and crucial role in quantitative
                     control of MAT gene transcription levels by cooperation with several TFs forming enhanceosome-like protein complexes. 
                     
                  </p>
                  <p>
                     
                     <a name="SEC2" id="SEC2"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        RESULTS
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     <strong><em>bZIP53</em> Is Expressed in Seed Tissues during Maturation</strong><br/>
                     
                     We have previously shown that <em>Arabidopsis</em> S1 bZIP proteins support strong heterodimerization with members of the group C bZIPs using two-hybrid systems in yeast and
                     protoplasts and bimolecular fluorescence complementation (BiFC) techniques (Ehlert et al., 2006<a href="#BIB10"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Weltmeier et al., 2006<a href="#BIB54"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Since two group C members, bZIP10 and bZIP25, are known activators of SSP genes (Lara et al., 2003<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), we surveyed the expression patterns of S1 members during seed development to identify possible partners for C-S1 interactions
                     in this process. According to the mRNA profiles of S1 bZIPs during silique and seed development derived from public AtGenExpress
                     microarray data (<a href="http://www.genomforschung.uni-bielefeld.de/GF-research/AtGenExpress-SeedsSiliques.html">http://www.genomforschung.uni-bielefeld.de/GF-research/AtGenExpress-SeedsSiliques.html</a>), <em>bZIP53</em> is the S1 member with the highest expression in mid and late maturation phases. Induction of <em>bZIP53</em> precedes and overlaps the activation of typical SSP and LEA genes, supporting the hypothesis that bZIP53 might be involved
                     in MAT gene regulation. 
                     
                  </p>
                  <p>
                     
                     Confirmation of the <em>bZIP53</em> temporal pattern of expression and its localization within the seed were obtained by in situ hybridization studies (<a href="#FIG1">Figure 1A</a>
                     
                     ). <em>bZIP53</em> mRNAs produced a strong signal in the embryo cotyledons during late maturation. In addition, we performed histochemical localization
                     of β-glucuronidase (GUS) activity in seeds of <em>Arabidopsis</em> transgenic plants expressing the <em>GUS</em> reporter gene under control of the <em>bZIP53</em> gene promoter (ProbZIP53:GUS). The results presented here were confirmed in three independent transgenic lines, all of which
                     showed the same profiles of GUS expression. As seen in <a href="#FIG1">Figure 1B</a>, GUS staining in the developing embryo was observed from the torpedo to the green cotyledon stages during seed development
                     (Bowman, 1994<a href="#BIB5"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Altogether, these results demonstrate that <em>bZIP53</em> expression increases during seed development and localizes to the embryo and endosperm during the maturation phase. 
                     
                  </p>
                  <p>
                     
                     <a name="FIG1" id="FIG1"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="1747/F1.expansion.html"><img src="1747/F1.small.gif"/></a><br/><strong>View larger version</strong> (72K):<br/><a href="1747/F1.expansion.html">In this window</a><br/><a href="1747/F1.expansion.html" class="in-nw">In a new window</a><br/><a href="1747/F1.expansion.html">As a PowerPoint slide</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 1.</strong></span> Expression of <em>bZIP53</em> in Seeds.
                        
                        <p>
                           
                           <strong>(A)</strong> In situ mRNA hybridization for <em>bZIP53</em> at different stages of seed development. Longitudinal sections of siliques with seeds at heart (I and I'), early torpedo
                           (II and II'), and green cotyledon (III and III') stages of development are shown. Sections were hybridized with a <em>bZIP53</em> antisense (I, II, and III) or a sense probe (I', II', and III').
                           
                        </p>
                        
                        <p>
                           
                           <strong>(B)</strong> Histochemical analysis of Columbia (Col-0) seeds harboring a <em>bZIP53</em> promoter fused to GUS (ProbZIP53:GUS). GUS staining of late torpedo stage (II) and green cotyledon stage (III) embryo development
                           are shown. The arrow indicates the developing endosperm; to facilitate viewing, the seed has been pressed to push out the
                           embryo.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     <strong>Ectopic Expression of <em>bZIP53</em> Results in Abnormal Plant Growth and Expression of Seed-Specific Genes in Leaf Tissue</strong><br/>
                     
                     Because of its particularly high levels of expression during seed development, bZIP53 could be expected to participate in
                     seed gene regulation. To address this question, we followed a gain-of-function approach and obtained transgenic <em>Arabidopsis</em> plants with ectopic expression of <em>bZIP53</em> driven by the cauliflower mosaic virus 35S promoter (Pro35S:bZIP53) (Weltmeier et al., 2006<a href="#BIB54"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). As seen in <a href="#FIG2">Figure 2A</a>
                     
                     , strong overexpression of <em>bZIP53</em> resulted in phenotypic alterations, including dwarfism and delayed bolting compared with the wild-type plants. To test whether
                     these modifications were associated to altered patterns of seed gene expression, mRNA isolated from wild-type and Pro35S:bZIP53
                     plants was subjected to quantitative RT-PCR (qRT-PCR) to check for <em>bZIP53</em>, MAT, and LEA gene expression. <a href="#FIG2">Figure 2B</a> shows data of wild-type and two independent Pro35S:bZIP53 transgenic lines with increased levels of <em>bZIP53</em> mRNA as well as induced expression of seed MAT and LEA genes, indicating that bZIP53 is able to activate seed-specific expression
                     in <em>Arabidopsis</em> leaves. Accumulation of SSP transcripts in leaves, as exemplified by the cruciferin (<em>CRU3</em>) and albumin (<em>2S2</em>) genes, was also obtained by overexpression of HA-tagged bZIP53 (see Supplemental Figures 1A and 1B online). The presence
                     of the transgene-encoded protein was confirmed by immunoblot analysis. Further examples of seed genes misexpressed in leaves
                     are legumin-like <em>CRA1</em> (Wang et al., 2007<a href="#BIB51"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) and the 11-β-hydroxysteroid dehydrogenase gene (<em>HSD1</em>) (Li et al., 2007<a href="#BIB28"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; see Supplemental Figure 1C online). In addition, these plants also show enhanced expression of the proline dehydrogenase
                     (<em>ProDH</em>) gene, as previously reported (Weltmeier et al., 2006<a href="#BIB54"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), or the <em>Asparagine Synthetase1</em> (<em>ASN1</em>) gene, both typical genes involved in amino acid metabolism (Lam et al., 2003<a href="#BIB26"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) (<a href="#FIG2">Figure 2B</a>; see Supplemental Figure 1C online). Altogether, these data support the assumption that overexpression of bZIP53 in leaves
                     triggers the misexpression of a substantial fraction of seed-specific genes. 
                     
                  </p>
                  <p>
                     
                     <a name="FIG2" id="FIG2"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="1747/F2.expansion.html"><img src="1747/F2.small.gif"/></a><br/><strong>View larger version</strong> (85K):<br/><a href="1747/F2.expansion.html">In this window</a><br/><a href="1747/F2.expansion.html" class="in-nw">In a new window</a><br/><a href="1747/F2.expansion.html">As a PowerPoint slide</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 2.</strong></span> Plants with Constitutive Expression of <em>bZIP53</em> Have Growth Defects and Increased Levels of Seed Maturation Transcripts.
                        
                        <p>
                           
                           <strong>(A)</strong> Plants overexpressing <em>bZIP53</em> (Pro35S:bZIP53) have a stunted and late flowering phenotype compared with Col wild-type (Col-0 wt) plants.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(B)</strong> mRNA samples from 2-week-old wild-type and two Pro35S:bZIP53 lines (#1 and #2) were analyzed by qRT-PCR to quantify the transcript
                           levels of <em>bZIP53</em>, <em>LEA76</em>, <em>CRU3</em>, and <em>ProDH</em>. Expression levels are given relative to a <em>UBIQUITIN</em> gene for normalization. Given are mean values and standard deviation of two to three replicates.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     <strong>The Albumin <em>2S2</em> Gene Is a Direct Target of bZIP53</strong><br/>
                     
                     To elucidate whether the SSP promoters were directly targeted by bZIP53, we performed chromatin immunoprecipitation (ChIP)
                     from leaves of plants expressing ectopical bZIP53 tagged at the N terminus with a 3xHA epitope. These plants have been previously
                     described and display the same phenotype as the untagged Pro35S:bZIP53 plants described in <a href="#FIG2">Figure 2A</a> (Weltmeier et al., 2006<a href="#BIB54"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     <a href="#FIG3">Figure 3A</a>
                     
                     shows the results of qPCR analyses for the <em>2S2</em> promoter on chromatin samples isolated from wild-type and Pro35S:HA-bZIP53 plants. The relative enrichment of the <em>2S2</em> gene promoter in Pro35S:HA-bZIP53 immunoprecipitated samples supports direct binding of a protein complex that includes bZIP53.
                     
                     
                  </p>
                  <p>
                     
                     <a name="FIG3" id="FIG3"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="1747/F3.expansion.html"><img src="1747/F3.small.gif"/></a><br/><strong>View larger version</strong> (21K):<br/><a href="1747/F3.expansion.html">In this window</a><br/><a href="1747/F3.expansion.html" class="in-nw">In a new window</a><br/><a href="1747/F3.expansion.html">As a PowerPoint slide</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 3.</strong></span> In Vivo and in Vitro Binding of the bZIP53 Protein to the Albumin <em>2S2</em> Promoter and Interaction with bZIP10 and bZIP25.
                        
                        <p>
                           
                           <strong>(A)</strong> Chromatin extracts from wild-type plants and plants overexpressing a HA-tagged bZIP53 protein (Pro35S:HA-bZIP53) were subjected
                           to qRT-PCR analysis with <em>2S2</em> promoter-specific primers before (input) and after immunoprecipitation with an anti-HA antibody (α-HA). Ct values for Pro35S:HA-bZIP53
                           samples were subtracted from the Ct values of the equivalent wild type, and the differentials
                           are shown on top of the right bars in the graph. A value of 1 was assigned to the Col-0 samples. For normalization, an actin
                           (<em>ACT7</em>) gene was used (see Methods).
                           
                        </p>
                        
                        <p>
                           
                           <strong>(B)</strong> Schematic view of the <em>2S2</em> promoter fused to GUS used as reporter in transient expression analysis. Depicted are RY (black) and G-box (white) elements.
                           The sequence of the wild type and mutated G-box (mt) is shown.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(C)</strong> <em>Arabidopsis</em> leaves were transformed with the reporter constructs containing sequences described in <strong>(B)</strong> and effector constructs containing specific <em>bZIP</em> genes under control of the Pro35S in cobombardment experiments as described by Lara et al. (2003)<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. Three microliters of control plasmid Pro<em>35S</em>:NAN was included in all the experiments to normalize GUS expression values for differences in bombardment efficiencies (Kirby
                           and Kavanagh, 2002<a href="#BIB24"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). The <em>x</em> axis values are expressed as GUS activity relative to NAN. Given are mean values and standard deviation of three independent
                           experiments.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(D)</strong> In vitro binding of bZIP53 to the G-box sequence from the <em>2S2</em> promoter. A biotinylated oligonucleotide containing the G-box sequence was bound to streptavidin-coated wells and incubated
                           with increasing amounts of a T7-tagged bZIP53 protein (1 to 1:90). Nonbound proteins were removed from the reaction wells,
                           and the amount of T7-bZIP53 protein was quantified by immunodetection with an anti-T7 antibody (α-T7; left panel). The binding
                           specificity of bZIP53 to the <em>2S2</em> G-box was analyzed in competition experiments where increasing amounts of unlabeled oligonucleotides (as indicated) containing
                           a wild-type (<em>2S2 wt</em>; black bars) or a mutated 2S2 <em>mt</em> version (gray bars) were incubated with a fixed amount of the T7-bZIP53 protein and the biotinylated oligonucleotide containing
                           the wild-type G-box sequence (right panel).
                           
                        </p>
                        
                        <p>
                           
                           <strong>(E)</strong> Effect of heterodimerization on bZIP53 and bZIP10 binding affinity. A biotinylated oligonucleotide containing the G-box sequence
                           was bound to streptavidin-coated wells and incubated with increasing amounts (1 to 1:30) of a T7-bZIP53 protein in the absence
                           (filled squares) or presence of a fixed amount of bZIP10 protein (open squares). Nonbound proteins were removed from the reaction
                           wells, and the amount of T7-bZIP53 protein was quantified by immunodetection with an α-T7 antibody (left panel). The reciprocal
                           experiment was performed with a T7-bZIP10 protein and a nontagged bZIP53 protein
                           (right panel). Given are mean values and standard deviation of three repetitions.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     <strong>Heterodimerization of bZIP53 with Group C bZIP10 or bZIP25 Promotes Strong Activation of Seed-Specific Genes</strong><br/>
                     
                     Activation of seed-specific genes in leaves of Pro35S:bZIP53 plants as well as ChIP analyses suggest that bZIP53 is directly
                     involved in this regulation. We studied the activation properties of bZIP53 in <em>Arabidopsis</em> leaves transformed by particle bombardment (Lara et al., 2003<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). <a href="#FIG3">Figures 3B and 3C</a> show an example of <em>2S2</em> promoter constructs driving the expression of a GUS reporter that is cotransformed with effector plasmids for bZIP53, bZIP10,
                     and bZIP25. Individually, none of these bZIPs is able to produce a significant increase in the basal activity of the reporter.
                     Previous studies showed no significant heterodimerization for bZIP10 and bZIP25, whereas strong heterodimerization of group
                     C bZIPs and bZIP53 was reported (Ehlert et al., 2006<a href="#BIB10"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Accordingly, when bZIP53 was cotransformed with bZIP10 or bZIP25, a dramatic increase in the reporter activity was observed.
                     These results were confirmed in transiently transformed leaf mesophyll protoplasts using two different seed-specific reporter
                     constructs, driven by the <em>2S2</em> or the <em>CRU3</em> promoter (see Supplemental Figure 2 online). Similarly, immunoblot analysis confirmed expression of the bZIP genes (see Supplemental
                     Figure 3A online) in transient assays in protoplasts. Unexpectedly, we observed that coexpression of bZIP10 and bZIP53 leads
                     to enhanced protein levels, suggesting that heterodimer formation might stabilize the bZIP proteins from degradation. This
                     data emphasize the importance of heterodimer formation between bZIP53 and group C bZIPs in the observed enhancement of gene
                     activation. 
                     
                  </p>
                  <p>
                     
                     <strong>Activation of the <em>2S2</em> Seed Storage Protein Gene by bZIP53 and Its Heterodimers Relies on the G-Box Promoter Element</strong><br/>
                     
                     The in vivo regulation of bZIP53 and its heterodimers was further investigated on the <em>2S2</em> promoter. A modified version of the native promoter was used, in which the G-box element was mutated to a sequence that prevents
                     bZIP DNA binding (<a href="#FIG3">Figure 3B</a>; Lara et al., 2003<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Different from the behavior of the native promoter, the activation mediated by the bZIP TFs is completely abolished in
                     this mutant (<a href="#FIG3">Figure 3C</a>), indicating that the regulation of the <em>2S2</em> promoter by these bZIP proteins requires an intact G-box motif. 
                     
                  </p>
                  <p>
                     
                     <strong>bZIP53 Binds in Vitro to the G-Box Present in the <em>2S2</em> Promoter, and Heterodimerization with bZIP10 or bZIP25 Increases Its Binding Activity</strong><br/>
                     
                     To establish if bZIP53 was able to bind to seed-specific gene promoters, we performed in vitro DNA binding experiments with
                     an epitope-tagged bZIP53 (T7-bZIP53) protein. A native-sequence oligonucleotide (wild type) containing a functional bZIP (G-box)
                     binding site (Ezcurra et al., 2000<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) derived from the <em>2S2</em> promoter and a mutated version were used (<a href="#FIG3">Figure 3B</a>). Double-stranded oligonucleotides were covalently attached to 96-well ELISA plates before doing the binding assays as described
                     in Methods. <a href="#FIG3">Figure 3D</a> shows a positive correlation between the amount of T7-bZIP53 detected with the epitope-specific antibody and the amount of
                     T7-bZIP53 protein extract added to the wells, indicating T7-bZIP53 binding to the wild-type oligonucleotide in a concentration-dependent
                     manner. To analyze the specificity of this binding, competition experiments were performed with increasing amounts of the
                     wild-type or mutated oligonucleotides added to the binding reactions. As shown in <a href="#FIG3">Figure 3D</a>, the wild-type, but not the mutated oligonucleotide, was able to reduce the amount of T7-bZIP53 bound to the plate, thus
                     confirming the specificity of the binding. 
                     
                  </p>
                  <p>
                     
                     We have shown in previous work and in this study that bZIP53 heterodimerization with bZIP10 or bZIP25 increases its transactivation
                     properties (Ehlert et al., 2006<a href="#BIB10"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Weltmeier et al., 2006<a href="#BIB54"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). To determine if the heterodimerization was also affecting the binding capabilities of these proteins, we incubated increasing
                     amounts of T7-bZIP53 in the wild-type oligo-coated wells in the presence or absence of fixed amounts of bZIP10. <a href="#FIG3">Figure 3E</a> illustrates the effect of heterodimerization between bZIP53 and bZIP10 and shows that the binding activity for the concentration
                     series of bZIP53 is higher in the presence (than in the absence) of bZIP10. Likewise, the same behavior is observed in the
                     reciprocal experiment for a bZIP10 series, in support of an increased binding activity of the heterodimers. 
                     
                  </p>
                  <p>
                     
                     <strong>ABI3 Does Not Interact with bZIP53 Alone, but Heterodimers with bZIP10 or bZIP25 Form a Ternary Complex in Yeast and in Planta</strong><br/>
                     
                     ABI3 is an important regulator of gene expression in seeds of <em>Arabidopsis</em> (Giraudat et al., 1992<a href="#BIB15"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Parcy et al., 1994<a href="#BIB39"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) and is able to interact with bZIP10 or bZIP25 to increase their activation capacity on the <em>2S2</em> promoter (Lara et al., 2003<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). We used a yeast two-hybrid system to determine if bZIP53 was also able to interact with ABI3. We expressed the ABI3 and
                     bZIP53 proteins as fusions to the GAL4 binding (BD) or activation (AD) domains, respectively, and introduced them into yeast
                     strains containing <em>LacZ</em> or <em>HIS</em> reporter genes under the control of GAL4 binding sites. Different combinations of ABI3 and bZIP53 were tested for possible
                     interactions in this system, alongside ABI5, another bZIP TF that has been implicated in seed gene expression serves here
                     as a negative control (Finkelstein and Lynch, 2000<a href="#BIB14"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). As seen in <a href="#FIG4">Figure 4A</a>
                     
                     , no interaction could be detected between bZIP53 and ABI3 or between bZIP53 and ABI5 in any of the experimental systems used.
                     As previously reported, positive interactions between bZIP53 and bZIP10 or bZIP25 confirmed their capacity to heterodimerize.
                     To further analyze the possibility that these heterodimers could still interact with ABI3, three-hybrid system experiments
                     were devised. Yeast cells expressing BD-bZIP53 were cotransformed with either AD-bZIP10 or AD-bZIP25 in the presence or absence
                     of ABI3, and β-galactosidase reporter activities were quantified (see Supplemental Figure 4 online). As previously observed,
                     the interaction between BD-bZIP53 and either AD-bZIP10 or AD-bZIP25 was confirmed by an increase of the reporter activity.
                     However, when a construct designed to express ABI3 was cotransformed into the strains, a further increase of the β-galactosidase
                     activity was observed, in support of a trimeric interaction. As expected for a negative control, the bZIP factor ABI5, which
                     interacts with ABI3 (Nakamura et al., 2001<a href="#BIB31"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) but does not heterodimerize with AtbZIP53, did not show activation indicative of ternary complex formation. 
                     
                  </p>
                  <p>
                     
                     <a name="FIG4" id="FIG4"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="1747/F4.expansion.html"><img src="1747/F4.small.gif"/></a><br/><strong>View larger version</strong> (25K):<br/><a href="1747/F4.expansion.html">In this window</a><br/><a href="1747/F4.expansion.html" class="in-nw">In a new window</a><br/><a href="1747/F4.expansion.html">As a PowerPoint slide</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 4.</strong></span> Interaction of bZIP53, bZIP10, and bZIP25 Homo- or Heterodimers with ABI3.
                        
                        <p>
                           
                           <strong>(A)</strong> Protein interactions in yeast two- and three-hybrid systems. ABI3, ABI5, bZIP10, and bZIP25 proteins were fused to the GAL4
                           DNA BD or AD and introduced separately into a yeast strain containing the AD-bZIP53 (HF7c) or BD-bZIP53 (SFY526) protein,
                           respectively. Activation of the reporter genes HIS3 (growth in a His-depleted medium; left panel) and LacZ (blue colored colonies;
                           right panel) indicates positive protein–protein interactions.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(B)</strong> Yeast strains (SFY526) expressing different combinations of BD-bZIP53, AD-ABI3, and bZIP10 or bZIP25 were assayed for β-galactosidase
                           activity. The latter were provided in the three-hybrid vector pTFT (Egea-Cortines et al., 1999<a href="#BIB9"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Average values (Miller units) and standard errors from six replicates and two independent experiments are shown.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(C)</strong> Schematic overview of a three-hybrid assay in <em>Arabidopsis</em> protoplasts. Constructs are shown on the left, and a model of reporter gene activation is shown on the right.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(D)</strong> Activation of the GAL4-UAS<sub>4</sub>:GUS reporter after cotransfection with the constructs indicated in <strong>(C)</strong>. The <em>x</em> axis values are expressed as GUS activity relative to NAN (Ehlert et al., 2006<a href="#BIB10"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Average values and standard errors from four transfections are shown. Numbers along the <em>y</em> axis represent fold induction values relative to nontransfected control cells. The experiments were repeated three times
                           with similar results.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     Reciprocal three-hybrid analysis where the noninteracting proteins bZIP53 and ABI3 were fused to the Gal4BD and Gal4AD, respectively,
                     showed also an increase of reporter activity when assayed in the presence of bZIP10 or bZIP25 (<a href="#FIG4">Figure 4</a>), in support of the proposed trimeric interaction. Altogether, these results indicate that ABI3 can be brought into the heterodimer
                     complex, probably by its interaction with bZIP10 and/or bZIP25. Importantly, bZIP53 does not interfere with the interaction
                     between ABI3 and bZIP10 or bZIP25. 
                     
                  </p>
                  <p>
                     
                     Ternary complex formation in plant cells was supported by a similar three-hybrid approach developed in <em>Arabidopsis</em> protoplasts. <a href="#FIG4">Figure 4C</a> shows a schematic overview of the experimental setup and the structure of the expressed proteins used to transactivate a
                     reporter construct containing the <em>GUS</em> gene under the control of a minimal promoter bearing a tetramer of the GAL4 upstream activating sequence (GAL4-UAS<sub>4</sub>:GUS) (Ehlert et al., 2006<a href="#BIB10"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). As shown in <a href="#FIG4">Figure 4D</a>, comparable results to the yeast system were achieved. Transactivation of the reporter with BD-bZIP53 alone results in low
                     levels of GUS activity, whereas cotransformation with either HA-bZIP10 or AD-bZIP10 displayed enhanced activity indicative
                     of heterodimer formation as previously observed in the yeast two-hybrid system. Expression of the corresponding proteins was
                     assayed by immunoblot analysis with HA-tag and BD-specific antibodies (see Supplemental Figure 3B online). Equally, in both
                     systems, the supplementary inclusion of HA-ABI3 resulted in an additional increase of reporter activity, in support of its
                     capacity to interact with the heterodimers in plant cells. 
                     
                  </p>
                  <p>
                     
                     BiFC (Walter et al., 2004<a href="#BIB50"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) was used as an additional experimental system to verify the detected ternary interactions in plant cells. bZIP53 and ABI3
                     proteins fused to different domains of yellow fluorescent protein (YFP) were coexpressed in onion cells in distinct combinations,
                     rendering no reconstruction of YFP activity. However, when a third construct expressing bZIP10 protein was cotransfected,
                     YFP fluorescence was observed in support of a tethering function of this protein to promote a ternary complex formation (<a href="#FIG5">Figure 5</a>
                     
                     ). 
                     
                  </p>
                  <p>
                     
                     <a name="FIG5" id="FIG5"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="1747/F5.expansion.html"><img src="1747/F5.small.gif"/></a><br/><strong>View larger version</strong> (64K):<br/><a href="1747/F5.expansion.html">In this window</a><br/><a href="1747/F5.expansion.html" class="in-nw">In a new window</a><br/><a href="1747/F5.expansion.html">As a PowerPoint slide</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 5.</strong></span> bZIP53, bZIP10, and ABI3 Protein–Protein Interaction Studied by BiFC (Walter et al., 2004<a href="#BIB50"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>).
                        
                        <p>
                           
                           Onion epidermis cells have been transiently transformed by particle bombardment. Fusion proteins of the C-terminal part of
                           YFP (CtYFP) and ABI3 and the N-terminal part of YFP (NtYFP) and bZIP53 have been coexpressed with HA-tagged bZIP10 (HA-bZIP10)
                           (a and a') or an empty vector control (b and b'). Bright-field images (top panels) and epifluorescence images taken by confocal
                           microscopy (bottom panels) are shown.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     <strong>Heterodimer Formation and Interaction with ABI3 Regulate the Activation Properties of bZIP53 on SSP Gene Expression in Plant
                        Cells</strong><br/>
                     
                     To study the functional implications of the interaction between ABI3 and bZIP heterodimers in plant cells, we performed transient
                     expression analysis in <em>Arabidopsis</em> protoplasts using two different SSP promoters, corresponding to the <em>2S2</em> and <em>CRU3</em> genes. <a href="#FIG6">Figure 6A</a>
                     
                     shows that the use of bZIP effectors individually or in combination with ABI3 resulted in limited activation capacity on both
                     promoters. Combination of bZIP53 and bZIP10 led to an important increase in the activity, in accordance with previous results.
                     Moreover, the inclusion of ABI3 in this combination produced an additional increase in the activation of both promoters. 
                     
                  </p>
                  <p>
                     
                     <a name="FIG6" id="FIG6"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="1747/F6.expansion.html"><img src="1747/F6.small.gif"/></a><br/><strong>View larger version</strong> (28K):<br/><a href="1747/F6.expansion.html">In this window</a><br/><a href="1747/F6.expansion.html" class="in-nw">In a new window</a><br/><a href="1747/F6.expansion.html">As a PowerPoint slide</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 6.</strong></span> Effect of ABI3 on the Transcriptional Activation Mediated by the bZIP53/bZIP10 Heterodimers.
                        
                        <p>
                           
                           <strong>(A)</strong> GUS reporter activity under the control of the <em>2S2</em> or <em>CRU3</em> promoter was measured in transiently transformed <em>Arabidopsis</em> protoplasts after cotransfection of the effector constructs indicated. bZIP10, bZIP53, and ABI3 are expressed under the control
                           of Pro35S. For immunodetection, 3xHA-epitope-tagged derivatives were used (see Supplemental Figure 3 online). The importance
                           of bZIP heterodimerization was demonstrated by including bZIP10pp, which is impaired in zipper-mediated dimerization (Weltmeier
                           et al., 2006<a href="#BIB54"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Given are mean values and standard deviation of four independent transfections.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(B)</strong> Immunoblot analysis of transiently transformed protoplasts confirms that expression of HA-bZIP10 is comparable to HA-bZIP10pp-untransformed
                           control protoplasts. HA epitope-tagged proteins were detected using an α-HA antiserum. The arrow indicates HA-bZIP10 and bZIP10pp
                           proteins, and the asterisk shows degradation products.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(C)</strong> Transient expression using a GUS reporter gene under the control of the <em>2S2</em> promoter. Combinations of bZIP53 and bZIP10 at different ratios (indicated are ratios of input DNA) in the presence (black
                           bars) or absence (white bars) of ABI3 were used as effectors. The <em>x</em> axis values are expressed as GUS activity relative to NAN as described by Ehlert et al. (2006)<a href="#BIB10"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. Average values and standard errors from four transfections are shown.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     This reporter gene activation cannot be observed when using an effector construct expressing bZIP10 with two Pro exchange
                     mutations (bZIP10pp); these mutations disrupt the zipper dimerization surface and therefore abolish the interaction with bZIP53
                     (Weltmeier et al., 2006<a href="#BIB54"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Expression of bZIP10pp was found to be comparable to the wild-type protein as confirmed by immunoblot analysis (<a href="#FIG6">Figure 6B</a>). Accordingly, disruption of the bZIP heterodimer using bZIP10pp abolishes this induced gene activation both in the presence
                     or absence of ABI3. These results indicate a functional interaction between bZIP53/bZIP10 heterodimer and ABI3 and confirm
                     a positive effect of ABI3 on heterodimer-mediated transcription. 
                     
                  </p>
                  <p>
                     
                     To further investigate this effect, a comprehensive analysis of ABI3 action on the heterodimer was performed. In <a href="#FIG6">Figure 6C</a>, different ratios of bZIP53 and bZIP10 were used to transactivate the <em>2S2</em> promoter in <em>Arabidopsis</em> protoplasts, either in the presence or in the absence of ABI3. In all cases, the activation effect of the heterodimer was
                     enhanced in the presence of ABI3 by a similar increment. However, maximum promoter activity was greatly determined by the
                     ratio of the heterodimerization partners. A 1:1 ratio, which enables optimal heterodimerization, results in maximum promoter
                     activation. 
                     
                  </p>
                  <p>
                     
                     <strong>bZIP53 Loss-of-Function Plants Display Reduced Expression of SSP Genes</strong><br/>
                     
                     Results from the analysis of ectopic expression of <em>bZIP53</em> suggested a regulatory role in SSP gene expression. We used a complementary loss-of-function strategy making use of a <em>bzip53</em> T-DNA insertion mutant (Weltmeier et al., 2006<a href="#BIB54"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) to corroborate the functional relevance of bZIP53 for SSP expression. We quantified SSP gene expression in <em>bzip53</em> plants during seed development. Siliques of wild-type and <em>bzip53</em> plants were collected at different stages of seed development, and total RNA was isolated and used in qRT-PCR experiments.
                     <a href="#FIG7">Figure 7A</a>
                     
                     shows that levels of <em>bZIP53</em> mRNA were greatly reduced throughout all the stages of silique development in the <em>bzip53</em> plants. However, residual levels of <em>bZIP53</em> were detected, probably due to the leaky nature of the <em>bzip53</em> mutation since the T-DNA is inserted in the promoter region (Weltmeier et al., 2006<a href="#BIB54"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Observed differences in the timing of RNA accumulation between <em>CRU3</em>, <em>2S2</em>, and <em>LEA76</em> genes were in accordance with those previously described for MAT and LEA genes (Parcy et al., 1994<a href="#BIB39"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Comparisons between wild-type and <em>bzip53</em> plants show significantly reduced levels of SSP transcript in the latter. Although the timing of mRNA accumulation for these
                     genes in the <em>bzip53</em> samples was not significantly altered, the mRNA levels were reduced throughout the maturation stages and no peaks of RNA
                     accumulation were detected, in contrast with the wild-type samples (<a href="#FIG7">Figure 7A</a>). 
                     
                  </p>
                  <p>
                     
                     <a name="FIG7" id="FIG7"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="1747/F7.expansion.html"><img src="1747/F7.small.gif"/></a><br/><strong>View larger version</strong> (20K):<br/><a href="1747/F7.expansion.html">In this window</a><br/><a href="1747/F7.expansion.html" class="in-nw">In a new window</a><br/><a href="1747/F7.expansion.html">As a PowerPoint slide</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 7.</strong></span> Effect of a bZIP53 T-DNA Insertion Mutant on Seed-Specific Gene Regulation.
                        
                        <p>
                           
                           <strong>(A)</strong> mRNA levels detected of the seed maturation genes indicated by qRT-PCR in Col wild-type and <em>bzip53</em> siliques at different stages of development. Lane 1, 0 d after flowering (DAF); lane 2, 2 DAF; lane 3, 4 DAF; lane 4, 6 DAF;
                           lane 5, 9 DAF; lane 6, 12 DAF; lane 7, 15 DAF; lane 8, ≥18 DAF. Data are normalized using <em>UBIQUITIN</em> expression values. Average values and standard errors from at least two technical replicates are shown.
                           
                        </p>
                        
                        <p>
                           
                           <strong>(B)</strong> Transient expression by microparticle bombardment of <em>Arabidopsis</em> leaves from the wild type (white bars) and plants with increased (<em>Pro</em>35S<em>:bZIP53</em>; gray bars) or decreased (<em>bzip53</em>; black bars) expression of <em>bZIP53.</em> Effector constructs containing bZIP10 or ABI3 under the control of a 35S promoter (Pro35S) and a <em>GUS</em> reporter gene under the control of the <em>2S2</em> promoter were used. The <em>x</em> axis values are expressed as GUS activity relative to NAN (Kirby and Kavanagh, 2002<a href="#BIB24"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Average values and standard errors from four replicates and two different experiments are shown.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     As seen above, impaired bZIP53 activity leads to decreased SSP gene expression in the seed, probably by limiting the activation
                     capacity of the regulatory complex involving bZIP10/bZIP25 and ABI3. To test this hypothesis, we performed transactivation
                     experiments using a <em>2S2</em> promoter driving a GUS reporter in different plant backgrounds. <a href="#FIG7">Figure 7B</a> shows results of particle bombardment experiments on <em>Arabidopsis</em> leaves from wild-type, Pro35S:bZIP53, and <em>bzip53</em> plants. As previously reported (Lara et al., 2003<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), the use of bZIP10 and ABI3 as effectors in this system resulted in the activation of the <em>2S2</em> promoter. This activation was significantly enhanced in Pro35S:bZIP53 plants and reduced in the <em>bzip53</em> knockdown mutant line, consistent with the participation of bZIP53 in this regulatory complex. The same results were obtained
                     for bZIP25 in combination with ABI3 (see Supplemental Figure 5 online). 
                     
                  </p>
                  <p>
                     
                     <a name="SEC3" id="SEC3"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        DISCUSSION
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     Seed gene expression relies on specific TFs acting in a combinatorial fashion. Both in monocot and dicot species, group C
                     bZIP factors related to maize O2 have been shown to be important players in this regulation. However, expression of the corresponding
                     <em>Arabidopsis</em> bZIP genes is not seed specific. The recently described functional interaction between group C and S1 bZIPs (Ehlert et al.,
                     2006<a href="#BIB10"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) prompted us to investigate the possible participation of S1 members in seed gene regulation. In this study, we identified
                     bZIP53 as the major S1 member expressed during seed maturation. Evidence of its participation in seed gene regulation was
                     obtained by different approaches, including the analysis of plants with bZIP53 gain and loss of function. Moreover, functional
                     studies show that bZIP53 plays a key role in the strong activation of seed-specific genes, directly involved in a seed regulatory
                     protein complex encompassing the group C bZIP10 and bZIP25 and the B3 seed regulator ABI3. 
                     
                  </p>
                  <p>
                     
                     <strong>bZIP53 Constitutes a Transcriptional Regulator of Seed Maturation Genes</strong><br/>
                     
                     Analyses of transcriptomic data of S1 bZIPs from repository databases show an increase in expression of <em>bZIP53</em> (and to a lesser extent for <em>bZIP1</em>) during intermediate and late stages of seed development. This suggests a prominent role of bZIP53 in regulating seed gene
                     expression in conjunction with the group C members. We further confirmed the precise pattern of expression of <em>bZIP53</em> in seed tissues by mRNA in situ hybridization and promoter reporter fusions in transgenic <em>Arabidopsis</em> plants. 
                     
                  </p>
                  <p>
                     
                     Gain-of-function approaches in transgenic plants and <em>Arabidopsis</em> leaf protoplasts ectopically expressing <em>bZIP53</em> demonstrate that bZIP53 is able to activate several seed MAT and LEA genes, such as <em>2S2</em>, <em>CRU3</em>, <em>CRA1</em>, <em>HSD1</em>, and <em>LEA76</em>. However, as observed before, the presence of MAT RNA does not necessarily result in the detection of the corresponding proteins.
                     Additional cellular requirements and tissue-specific protein degradation determine high-level accumulation of MAT proteins
                     (Gruis et al., 2004<a href="#BIB18"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) 
                     
                  </p>
                  <p>
                     
                     Complementing the gain-of-function results, reduced expression of <em>bZIP53</em> in a knockdown line that harbors a T-DNA insertion in the promoter leads to strong reduction in the expression of the corresponding
                     target genes. However, these plants still set viable seeds, which might be due to a residual amount of bZIP53 protein. The
                     lack of complete knockout lines in mutant collections might be explained by an essential function of this gene. Alternatively,
                     other genes among the 75 bZIP TFs identified in <em>Arabidopsis</em> (Jakoby et al., 2002<a href="#BIB22"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) may partly substitute for bZIP53 function. Within the group S1, which harbors the closest homologs of bZIP53, only <em>bZIP1</em> is intermediately induced during seed maturation with timing comparable to <em>bZIP53</em>. However, ectopic overexpression of At <em>bZIP1</em> does not lead to activation of MAT genes, indicating a lack of functional redundancy of these genes in MAT gene regulation
                     (Weltmeier et al., 2009<a href="#BIB55"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     High-level ectopic expression of <em>bZIP53</em> results also in a dwarf growth phenotype in vegetative phases. Based on these observations, it can be assumed that regulation
                     of SSP genes reflects only a subset of bZIP53 targets. In previous work, we and others demonstrated that bZIP53 is also involved
                     in regulating hypo-osmolarity responses (Satoh et al., 2004<a href="#BIB42"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Weltmeier et al., 2006<a href="#BIB54"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). In particular, the promoter of the <em>ProDH</em> gene, which is involved in regulating amino acid metabolism, has been defined as an in vivo target of bZIP53. Furthermore,
                     a second amino acid metabolism gene, <em>ASN1</em>, was shown to be regulated by bZIP53 (this study) and other members of the C/S1 network (Baena-Gonzalez et al., 2007<a href="#BIB2"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Hanson et al., 2008<a href="#BIB19"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Amino acid synthesis, transport, and SSP synthesis are linked processes that have to be coordinated in vegetative tissues
                     as well as in the seed. In accordance, ectopic overexpression of <em>ASN1</em> enhances the SSP content and nitrogen status of the seed (Lam et al., 2003<a href="#BIB26"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Further studies are necessary to test whether bZIP53 is coordinating other pathways in addition to SSP synthesis, such
                     as amino acid metabolism during seed and vegetative development. 
                     
                  </p>
                  <p>
                     
                     <em>bZIP53</em> expression is not restricted to the seed and can be observed in vegetative tissues, particularly under certain stress conditions.
                     Interestingly, salt treatment dramatically induces <em>bZIP53</em> transcription in roots (Weltmeier et al., 2009<a href="#BIB55"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). In the context of the seed, it is worth mentioning that besides storage compound accumulation, dehydration occurs during
                     late stages of the maturation phase. Accordingly, stress-related programs of gene expression are also important in the establishment
                     of desiccation tolerance. We have determined that bZIP53 can also control the expression of the seed-specific gene <em>LEA76</em>. Although our knowledge of LEA76 function is limited, LEA proteins are known to accumulate in seed and vegetative plant tissues
                     following environmental stress (Xu et al., 1996<a href="#BIB56"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) and also in desiccation-tolerant bacteria and invertebrates (Bies-Etheve et al., 2008<a href="#BIB4"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Hundertmark and Hincha, 2008<a href="#BIB21"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). The hypothesis whether bZIP53 is involved in establishing stress tolerance in vegetative tissues requires further testing.
                     
                     
                  </p>
                  <p>
                     
                     <strong>Specific Heterodimerization of bZIP53 and Group C bZIPs Executes an Efficient Mechanism to Enhance Transcription of Seed Maturation
                        Genes</strong><br/>
                     
                     MAT gene regulation is one of the early investigated areas in plant transcriptional control. The importance of G-box <em>cis</em>-elements in MAT gene promoters has been determined both in monocot and dicot species (Vicente-Carbajosa and Carbonero, 2005<a href="#BIB49"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Santos-Mendoza et al., 2008<a href="#BIB41"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), and its regulation by seed-specific O2-type bZIPs has been firmly established in cereals. By contrast, genome scale transcriptomic
                     data in <em>Arabidopsis</em> do not reveal a seed-specific candidate among all bZIPs, indicating that in this species (and probably in nonendospermic
                     seeds from dicot plants), combinatorial effects are likely of major importance in the regulation of seed gene expression.
                     In this framework, the group C bZIPs related to O2, bZIP10, and bZIP25 are able to interact with ABI3 and have been proposed
                     as <em>Arabidopsis</em> counterparts of the monocot bZIP seed regulator (Schmidt et al., 1990<a href="#BIB44"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Lara et al., 2003<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). The corresponding genes, which are not seed specific, are constitutively expressed at low levels in other tissues, and
                     bZIP10 has been implicated in plant stress responses to pathogen attack in vegetative tissues (Kaminaka et al., 2006<a href="#BIB23"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Previous work has defined a specific heterodimerization network of group C members with group S1 bZIPs, such as bZIP53
                     (Ehlert et al., 2006<a href="#BIB10"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). The expression of this gene increases during seed maturation, and it strongly heterodimerizes with bZIP10 and bZIP25 as
                     demonstrated in yeast and plant protoplasts (Ehlert et al., 2006<a href="#BIB10"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), as well as by BiFC. Additionally, in vitro binding studies show that they can bind DNA as heterodimers (Weltmeier et al.,
                     2006<a href="#BIB54"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     Efficient methods for in vivo DNA binding studies in <em>Arabidopsis</em> seeds have not been established. However, here, we demonstrated binding of bZIP53 to the 2S2 promoter using leaf-derived
                     chromatin from Pro35S:HA-bZIP53 plants, which in turn show vegetative transcription of the <em>2S2</em> gene. Although this approach does not unequivocally confirm participation of bZIP53 in regulation of MAT genes, it provides
                     circumstantial evidence in line with other results presented in this study. Mutational approaches confirm the <em>2S2</em> G-box as the in vitro binding site and also as the relevant in vivo <em>cis-</em>regulatory element. At high protein concentration, bZIP53 displays interaction with G-box elements in vitro, but titration
                     experiments demonstrate that binding of bZIP53 in the presence of bZIP10 or bZIP25, presumably by heterodimer formation, is
                     significantly enhanced. The in vivo relevance of heterodimer formation is further supported by expression of bZIP mutants,
                     which are impaired in zipper-mediated dimerization, and consequently target gene activation. Detailed titration experiments
                     in protoplasts also demonstrate that the level of target gene expression is determined by the ratio of the bZIP heterodimerization
                     partners. The 1:1 ratio of bZIP53 and bZIP10 leads to the highest target gene activity. Altogether, these data provide conclusive
                     evidence that bZIP heterodimers show enhanced binding to and gene activation through G-box-like <em>cis</em>-elements. Importantly, heterodimerization provides an efficient mechanism by which bZIP53 drives MAT gene expression. 
                     
                  </p>
                  <p>
                     
                     Possible mechanisms of heterodimer-enhanced transactivation have been described elsewhere (Weltmeier et al., 2006<a href="#BIB54"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), including (1) increased affinity for binding target sites, (2) stabilization of dimer structures (as effective DNA binding
                     forms) via equilibrium displacement in dimer-monomer concentrations, and (3) heterodimer interactions with other protein factors
                     that increase activation. Here, we showed that the second and third possibilities must be operating in vivo in the regulation
                     of MAT genes. In support of this is the increased activation observed in protoplast systems for the heterodimers, which is
                     independent of their DNA binding when used as GAL4-BD protein fusions (<a href="#FIG4">Figure 4E</a>), and their interaction and enhanced gene activation when assayed with ABI3 (<a href="#FIG6">Figure 6</a>). In addition, heterodimer formation seems to protect bZIP proteins from degradation (see Supplemental Figure 3 online) and
                     therefore provides an additional mechanism that contributes to the observed synergistic effect on transcription. Although
                     not investigated in this study, the first possibility might also explain how diversification in target binding sites can lead
                     to variations in the expression of different SSP genes (Conceicao Ada and Krebbers, 1994<a href="#BIB8"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), depending on the binding properties of the heterodimers prevailing temporally and spatially in the seed. In conclusion,
                     the concurrence of several regulatory mechanisms must be operating on the underlying bZIP network. 
                     
                  </p>
                  <p>
                     
                     <strong>bZIP53, Group C bZIPs, and ABI3 Physically and Functionally Interact in the Control of Seed Maturation Genes</strong><br/>
                     
                     Current models of regulation of MAT gene expression in the seed entail the participation of bZIP and B3 interacting proteins,
                     particularly bZIP10 and 25 and ABI3. We have demonstrated that bZIP53 is a positive factor in this regulation and that its
                     interaction with group C bZIPs results in enhanced activation of the heterodimers. Although yeast and protoplast two-hybrid
                     data indicate that bZIP53 on its own is not able to interact with ABI3, heterodimerizations should not preclude the group
                     C bZIP-ABI3 interactions. Three-hybrid and BiFC analyses confirmed that bZIP heterodimerization of bZIP10 or bZIP25 with bZIP53
                     does not interfere with ABI3 interaction and supports the existence of a ternary complex. ABI3 has been shown to bind to the
                     RY element (Mönke et al., 2004<a href="#BIB29"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), and a functional interplay between G-box and RY-element in MAT gene regulation has been previously described (Ezcurra et
                     al., 2000<a href="#BIB12"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Lara et al., 2003<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). In this report, we clearly demonstrate functional interplay between bZIP heterodimers and ABI3, presumably by ternary complex
                     formation. 
                     
                  </p>
                  <p>
                     
                     By altering the bZIP expression levels in leaves, we can still see functionality of the bZIP heterodimers despite the absence
                     of the ABI3 protein (Parcy et al., 1994<a href="#BIB39"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>), though their mediated activation occurs at a significantly lower level. Further experiments in bZIP53 overexpressing and
                     knockdown plants show that although bZIP10 and/or ABI3 can enhance MAT gene activation to a certain extent, full-level expression
                     clearly depends on the presence of bZIP53 (see <a href="#FIG7">Figure 7B</a>). In this respect, nonseed model systems, such as protoplasts and bombarded leaves, might not fully reflect the native situation
                     in seeds. Nevertheless, all results obtained in these systems are in agreement with the intrinsic reduction of MAT gene expression
                     observed in seeds of mutant <em>bzip53</em> plants. On the other hand, similar experiments performed in cereal systems that allowed the postulation of regulatory models
                     have been confirmed in null mutants and transgenic plants (Norre et al., 2002<a href="#BIB34"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). In summary, the output in MAT gene expression depends on all partners, ABI3 and bZIPs, and is quantitatively triggered
                     by the amount of bZIP heterodimers. 
                     
                  </p>
                  <p>
                     
                     Complex formation between the bZIP TF ABI5 and ABI3 have been recognized in LEA but not in MAT gene regulation (Bensmihen
                     et al., 2002<a href="#BIB3"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Our findings suggest that the same principle based on ABI3 protein interactions operates on MAT expression, but inclusion
                     of different bZIP partners defines distinct subsets of target genes. Therefore, this study provides further insight into the
                     synergistic action of TFs and formation of enhanceosome-like protein-DNA complexes, which control seed gene transcription.
                     
                     
                  </p>
                  <p>
                     
                     <strong>A Model of Seed Maturation Gene Expression Triggered by bZIP53 Heterodimerization and Protein Complex Formation</strong><br/>
                     
                     Based on previous findings and data from this study, the working model in <a href="#FIG8">Figure 8</a>
                     
                     is proposed. The function of the previously described group C bZIP factors bZIP10 and bZIP25 in seed maturation strongly depends
                     on bZIP53 as a heterodimerization partner. Spatial and developmentally controlled expression of <em>bZIP53</em> is precisely timed, leading to increased amounts in the seed from mid maturation. Subsequent accumulation of bZIP53 would
                     trigger heterodimer formation with bZIP10 or bZIP25 as prerequisite for efficient promoter binding and target SSP gene activation.
                     ABI3 protein, which also accumulates during seed maturation, interacts with the bZIP heterodimer via bZIP10 or bZIP25 and
                     provides an additional increment in promoter activation. Hence, SSP gene regulation is driven by the interplay of (at least)
                     two maturation-dependent transcriptional regulators, namely, bZIP53 and ABI3. The crucial role of ABI3 in this process was
                     largely uncovered through the characterization of <em>abi3</em> loss-of-function mutants (Parcy et al., 1994<a href="#BIB39"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). This factor has a prominent role in seed maturation and germination gene expression programs. In this respect, its participation
                     in a ternary complex with C/S bZIPs might partly explain many of numerous pleotropic effects observed in <em>abi3</em> mutants. Interestingly, transcriptomic data suggest that <em>bZIP53</em> and <em>bZIP10</em> expression levels are reduced in mutant <em>abi3-4</em> developing seeds (Carrera et al., 2008<a href="#BIB7"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     <a name="FIG8" id="FIG8"/>
                     
                     
                  </p>
                  <div class="fig pos-float odd">
                     
                     <div class="fig-inline">
                        <a href="1747/F8.expansion.html"><img src="1747/F8.small.gif"/></a><br/><strong>View larger version</strong> (52K):<br/><a href="1747/F8.expansion.html">In this window</a><br/><a href="1747/F8.expansion.html" class="in-nw">In a new window</a><br/><a href="1747/F8.expansion.html">As a PowerPoint slide</a><br/> 
                        
                        
                     </div>
                     
                     <div class="fig-caption">
                        <span class="fig-label"><strong>Figure 8.</strong></span> Model for bZIP53 Regulation of Seed Maturation Gene Expression.
                        
                        <p>
                           
                           Yellow and blue backgrounds represent expression in the seed and leaf, respectively. Positions of the indicated TFs were represented
                           in a coordinate system according to relative expression values in seeds (<em>y</em> axis) and leaves (<em>x</em> axis) (data derived from AtGenExpress). Structure of a typical SSP promoter-like albumin (<em>2S2</em>) or cruciferin (<em>CRU3</em>) is depicted, indicating regulatory elements in their promoters (RY, RY-box; G, G-box; TATA, TATA-box). Continuous and dotted
                           lines indicate DNA–protein interactions and protein–protein interactions, respectively. A displacement arrow (open line) indicates
                           overexpression of bZIP53 in Pro35S:bZIP53 plants. bZIP53 increased expression in leaves favors heterodimer formation (hd)
                           with group C bZIP10 and bZIP25 and triggers induction of seed gene expression in this organ.
                           
                        </p> 
                        
                        <div class="sb-div caption-clear"/>
                     </div>
                     
                  </div>
                  <p>
                     
                     
                     In vegetative tissues, where bZIP10 and bZIP25 are constitutively present at low levels, the ectopic high-level expression
                     of ABI3 can lead to the activation of SSP genes (Parcy et al., 1994<a href="#BIB39"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>; Parcy and Giraudat, 1997<a href="#BIB38"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Likewise, ectopic high-level expression of bZIP53 can substitute for the presence of ABI3, triggering SSP gene expression
                     even more effectively than this factor. In summary, we propose that bZIP53 acts as a key regulator, which drives a subset
                     of MAT genes mainly by the formation of heterodimers with a high activation capacity. Although further research is needed
                     to elucidate distinct interacting partners and target genes both in vegetative and seed tissues, bZIP53 might represent an
                     important target for manipulating seed storage compound accumulation and establishment of desiccation tolerance. 
                     
                  </p>
                  <p>
                     
                     <a name="SEC4" id="SEC4"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        METHODS
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     <strong>Plant Materials and Transformation</strong><br/>
                     
                     For plant and protoplast transformation, particle bombardment, and ChIP experiments, <em>Arabidopsis thaliana</em> ecotype Col-0 was grown on soil or Murashige and Skoog plates under controlled environmental conditions at 16-h-light/8-h-dark
                     cycles. To improve germination uniformity, seeds were pretreated at 4°C for 2 to 4 d. Floral dip transformations were performed
                     using the <em>Agrobacterium tumefaciens</em> strain GV3101 (Weigel and Glazebrook, 2002<a href="#BIB53"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Pro35S:bZIP53, Pro35S:HA-bZIP53, ProbZIP53:GUS, and <em>bzip53</em> (NASC ID: N569883) plants were described by Weltmeier et al. (2006)<a href="#BIB54"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. 
                     
                  </p>
                  <p>
                     
                     <strong>In Situ Hybridization and Histochemistry</strong><br/>
                     
                     In situ hybridization was performed according to Ferrandiz et al. (2000)<a href="#BIB13"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. Forward 5'-TTGTCCAATGCAACCCAATCA-3' and reverse 5'-ACAAGACTAGAGGACTGAGGCT-3' <em>bZIP53</em>-specific oligonucleotides were used to amplify a 200-bp fragment from the 3' untranslated region. The fragment was inserted
                     into pGEM-T easy (Promega), and the sense and antisense probes were generated as specified by Lara et al. (2003)<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. The construction of ProbZIP53:GUS plants has been described by Weltmeier et al. (2006)<a href="#BIB54"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. GUS staining was performed according to Stangeland and Salehian (2002)<a href="#BIB45"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a> with some modifications. The siliques were incubated in GUS assay buffer overnight at room temperature under vacuum and transferred
                     to an ethanol/acetic acid (1:1) solution for 4 h (young seeds) or 12 h (mature seeds). Clearing was performed by incubating
                     tissues in Hoyer's light medium (100 g chloral hydrate in 60 mL water) for 12 h (young seeds) or 24 h (mature seeds) at room
                     temperature. Samples were analyzed with a Zeiss Axiophot microscope. 
                     
                  </p>
                  <p>
                     
                     <strong>qRT-PCR</strong><br/>
                     
                     RNA was isolated from seedlings and siliques as described by Oñate-Sánchez and Vicente-Carbajosa (2008)<a href="#BIB36"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. cDNA synthesis was performed according to Oñate-Sánchez and Singh (2002)<a href="#BIB37"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>, and between 8 and 16 ng were used as a template for qRT-PCR together with forward and reverse oligonucleotides (0.5 µM each)
                     in 1<span style="font-family: arial,helvetica;  ">x</span> Power SYBR Green PCR Master mix (Applied Biosystems). Cycling conditions (ABI Prism 7300; Applied Biosystems) were as follows:
                     10 min at 95°C, 50 cycles of 15 s at 95°C, and 60 s at 60°C, linked to a default dissociation stage program to detect nonspecific
                     amplification. Primers are provided in Supplemental Table 1 online. At least two technical and two biological replicates were
                     analyzed in all the experiments 
                     
                  </p>
                  <p>
                     
                     <strong>Transient Expression Assays and Constructs</strong><br/>
                     
                     Protoplast isolation, transformation, construction of effector plasmids, and immunoblot analysis were performed according
                     to Ehlert et al. (2006)<a href="#BIB10"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a> and Weltmeier et al. (2006)<a href="#BIB54"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. For reporter gene assays, 9 µg reporter and 14 µg of each effector were used if not stated otherwise. Three micrograms of
                     Pro35S:NAN plasmid was added for normalization (Kirby and Kavanagh, 2002<a href="#BIB24"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). For three-hybrid analysis, 9 µg of each reporter, BD-, AD-, and HA-plasmid were applied. The overall amount of DNA was
                     set to 40 µg by adding pUC19 plasmid DNA. Particle bombardment was performed according to Lara et al. (2003)<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. Data presented for the different assayed constructs are derived from three independent experiments, including six plates
                     with 10 leaves per plate. Pro2S2:GUS reporter vector is a pUC19-derived plasmid containing the GUS reporter gene under the
                     control of the <em>2S2</em> promoter and fused to the 3'-terminator of the nopaline synthase gene (3'-nos) (Lara et al., 2003<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Effector plasmids for the expression of bZIP53, bZIP10, bZIP25, and ABI3 were generated by cloning the corresponding coding
                     sequence under the control of the Pro35S and a 3'-nos terminator (Lara et al., 2003<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>) in the pJIT60 plasmid (Guerineau, 1995<a href="#BIB16"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Three micrograms of Pro35S:NAN plasmid was added for normalization. GUS and NAN enzyme assays were performed according
                     to Kirby and Kavanagh (2002)<a href="#BIB24"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. Immunoblot analysis was performed according to Weltmeier et al. (2006)<a href="#BIB54"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a> using α-HA and α-BD antisera (SantaCruz). 
                     
                  </p>
                  <p>
                     
                     <strong>ChIP</strong><br/>
                     
                     ChIP was performed as described by Weltmeier et al. (2006)<a href="#BIB54"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a> using an HA-specific antibody (SantaCruz). <em>2S2</em> primers were used to amplify a 306-bp fragment of the <em>2S2</em> promoter (5'-GACCGGTGACCTGCGTGTA-3' and 5'-GACTTGCATGGAGTTCACGTG-3'). The difference between the resulting C(t) values of
                     wild-type and Pro35S:HA-bZIP53 overexpressor was calculated and normalized with the input controls of these samples that were
                     analyzed with the same primers. For normalization, PCR was performed with unspecific actin (<em>ACT7</em>) promoter primers (5'-CGTTTCGCTTTCCTTAGTGTTAGCT-3' and 5'-AGCGAACGGATCTAGAGACTCACCTTG-3'). 
                     
                  </p>
                  <p>
                     
                     <strong>ELISA-Based Protein-DNA Binding Assays</strong><br/>
                     
                     The cDNAs encoding bZIP10, bZIP25, bZIP53, and ABI3 proteins were cloned into the expression vectors pET23a (Novagen) to generate
                     T7-epitope tagged proteins (Lara et al., 2003<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Expression in <em>Escherichia coli</em> and preparation of protein extracts were performed as previously described (Lara et al., 2003<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). Biotinylated complementary oligonucleotides derived from the <em>At2S2</em> promoter containing a G-box sequence (5'-CATGCAATGCATTCTTAC<strong>ACGT</strong>GATTGCATGCAAA-3' and 5'-TTTGCATGCAATC<strong>ACGT</strong>GTAAGAATGCATTGCATG-3') or a mutated version (5'-CATGCAATGCATTCTTAC<strong>AaGg</strong>GATTGCATGCAAA-3' and 5'-TTTGCATGCAATCC<strong>CtTg</strong>TAAGAATGCATTGCATG-3') were annealed in TES buffer (10 mM Tris, pH 8, 1 mM EDTA, and 300 mM NaCl). The appropriate double-stranded
                     DNA (2 pmoles; target) in 60 µL of TBS-T buffer (20 mM Tris, pH 7.4, 180 mM NaCl, and 0.1% Tween 20) was added to each well
                     of a streptavidin-coated plate (Nunc Inmobilizer) and incubated 1 h at 37°C, and nonbound targets were removed by washing
                     three times with TBS-T. Wells were blocked by incubating 30 min at room temperature and gentle shaking (150 rpm) with 60 µL
                     of 1% blocking agent (Roche) in TBS-T and washed three times with TBS-T. Twenty microliters of the protein extracts were mixed
                     with 40 µL of binding buffer (4 mM HEPES, pH 7.5, 100 mM KCl, 0.2% BSA, 8% glycerol, and 5 mM DTT) in microfuge tubes and
                     incubated 15 min on ice. The mixes were added to the wells of the plate, the binding reactions incubated for 1 h at room temperature
                     with gentle shaking, and well washed three times with TBS-T. For epitope detection, we added 60 µL of a 1:5000 dilution of
                     the antibody (T7 tag horseradish peroxidase conjugate; Novagen) in TBS-T to each well. Following incubation for 1 h at room
                     temperature at 150 rpm, the wells were four times washed with TBS-T. The antibody peroxidase-conjugated reaction was performed
                     according to manufacturer's instructions, and the activity was measured at 492 nm using an ELISA plate reader (Tecan) with
                     filter setting at 650 for reference. 
                     
                  </p>
                  <p>
                     
                     <strong>Yeast Two- and Three-Hybrid Analyses</strong><br/>
                     
                     Yeast two-hybrid analyses were done as described previously (Lara et al., 2003<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). BD and AD constructs were generated using appropriate restriction enzymes and cloning the indicated open reading frames
                     into the pGBT9 and pGAD424 plasmids (Clontech), respectively, as follows: <em>Eco</em>RI/<em>Sal</em>I sites for the BD-bZIP53, AD-bZIP53, BD-ABI5, and AD-ABI5 constructs and <em>Bam</em>HI/<em>Sal</em>I sites for the BD-ABI3 and AD-ABI3 constructs. BD-bZIP10, AD-bZIP10, BD-bZIP25, and AD-bZIP25 were described by Lara et al.
                     (2003)<a href="#BIB27"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. To analyze ternary complex formation in yeast, we used BD and AD constructs from the two-hybrid assays and expressed a third
                     protein whose coding sequences were inserted into the <em>Eco</em>RI/<em>Sal</em>I sites of the pTFT1 plasmid (Egea-Cortines et al., 1999<a href="#BIB9"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>). 
                     
                  </p>
                  <p>
                     
                     <strong>BiFC</strong><br/>
                     
                     Experimental procedures were according to Moreno-Risueno et al. (2007)<a href="#BIB30"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. In general, at least two independent assays were performed that included four independent plates bombarded for every individual
                     construct or the different combinations. Comparisons were based on transformation efficiencies estimated to Pro35S:green fluorescent
                     protein as a reference. Vectors used have been described by Weltmeier et al. (2006)<a href="#BIB54"><img width="8" height="7" alt="Go" src="/icons/fig-down.gif"/></a>. 
                     
                  </p>
                  <p>
                     
                     <strong>Accession Numbers</strong><br/>
                     
                     Arabidopsis Genome Initiative identifiers for the genes mentioned in this article are as follows: b<em>ZIP53</em> (<a href="/cgi/external_ref?access_num=At3g62420&amp;link_type=GEN">At3g62420</a>), b<em>ZIP10</em> (<a href="/cgi/external_ref?access_num=At4g02640&amp;link_type=GEN">At4g02640</a>), b<em>ZIP25</em> (<a href="/cgi/external_ref?access_num=At3g54620&amp;link_type=GEN">At3g54620</a>), b<em>ZIP1</em> (<a href="/cgi/external_ref?access_num=At5g49450&amp;link_type=GEN">At5g49450</a>), <em>2S2</em> (<a href="/cgi/external_ref?access_num=At4g27150&amp;link_type=GEN">At4g27150</a>), <em>CRU3</em> (<a href="/cgi/external_ref?access_num=At4g28520&amp;link_type=GEN">At4g28520</a>), <em>LEA76</em> (<a href="/cgi/external_ref?access_num=At3g15670&amp;link_type=GEN">At3g15670</a>), <em>ProDH</em> (<a href="/cgi/external_ref?access_num=At5g38710&amp;link_type=GEN">At5g38710</a>), <em>UBI</em> (<a href="/cgi/external_ref?access_num=At5g25760&amp;link_type=GEN">At5g25760</a>), <em>ACT7</em> (<a href="/cgi/external_ref?access_num=At5g09810&amp;link_type=GEN">At5g09810</a>), <em>ABI3</em> (<a href="/cgi/external_ref?access_num=At3g24650&amp;link_type=GEN">At3g24650</a>), <em>ABI5</em> (<a href="/cgi/external_ref?access_num=At2g36270&amp;link_type=GEN">At2g36270</a>), <em>ASN1</em> (<a href="/cgi/external_ref?access_num=At3g47340&amp;link_type=GEN">At3g47340</a>), <em>CRA1</em> (<a href="/cgi/external_ref?access_num=At5g44120&amp;link_type=GEN">At5g44120</a>), and At <em>HSD1</em> (<a href="/cgi/external_ref?access_num=At5g50600&amp;link_type=GEN">At5g50600</a>). 
                     
                  </p>
                  <p>
                     
                     <strong>Supplemental Data</strong><br/>
                     
                     The following materials are available in the online version of this article.
                     
                  </p>
                  <dl><dd><strong>Supplemental Figure 1.</strong> Analysis of Putative bZIP53 Target Genes in Plants Constitutively Expressing HA-bZIP53 or bZIP53. <br/></dd>
                     <dd><strong>Suplemental Figure 2.</strong> Transient Expression of bZIP53 in Combination with bZIP10 or bZIP25 Synergistically Enhances Transactivation of <em>CRU3</em> and <em>2S2</em> Promoters. <br/></dd>
                     <dd><strong>Supplemental Figure 3.</strong> Analysis of Effector Proteins Transiently Expressed in <em>Arabidopsis</em> Protoplasts. <br/></dd>
                     <dd><strong>Supplemental Figure 4.</strong> Ternary Protein Interaction Studied in a Yeast Three-Hybrid System. <br/></dd>
                     <dd><strong>Supplemental Figure 5.</strong> Transient Expression by Microparticle Bombardment of <em>Arabidopsis</em> Leaves from Columbia Wild Type and Plants with Increased (Pro35S:bZIP53) or Decreased (<em>bzip53</em>) Expression of <em>bZIP53.</em><br/></dd>
                     <dd><strong>Supplemental Table 1.</strong> Primers Used in This Study. <br/></dd>
                  </dl><div class="auto-clean"> </div>
                  <p>
                     
                     
                     <a name="ACK" id="ACK"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        Acknowledgments
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     We thank Mar Gonzalez and Anna Herman for their valuable technical assistance and Caroline Carsjens and Louise Thatcher for
                     proofreading. Critical comments and suggestions from anonymous referees and the Scientific Editor greatly improved previous
                     versions of this study. Financial support from the Ministerio de Educación y Ciencia, Spain (GEN2003-20859 and BIO2007-68073)
                     is acknowledged. R.A. is the recipient of a predoctoral fellowship from the Spanish Agency for International Cooperation.
                     L.O.-S. was supported by the Spanish Ministry for Education and Science with a Ramón and Cajal Contract and funded with a
                     Marie Curie International Reintegration Grant (036524). W.D.-L. is supported by Deutsche Forschungsgemeinschaft and Deurscher
                     Akademische Auslausik Dienst. 
                     
                  </p>
                  <p>
                     
                     
                     <a name="FN" id="FN"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        Footnotes
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     <a name="FN1" id="FN1"/>
                     <sup>1</sup> These authors contributed equally to this work. <a href="#RFN1"><img width="12" height="12" alt="Back" src="/icons/back.gif"/></a></p>
                  <p>
                     <a name="" id=""/>
                     The author responsible for distribution of materials integral to the findings presented in this article in accordance with
                     the policy described in the Instructions for Authors (<a href="http://www.plantcell.org">www.plantcell.org</a>) is: Wolfgang Dröge-Laser (<span class="em-link"><span class="em-addr">wdroege{at}gwdg.de</span></span>). 
                     
                  </p>
                  <p>
                     
                     <a name="FN2" id="FN2"/>
                     <sup>[W]</sup> Online version contains Web-only data. <a href="#RFN2"><img width="12" height="12" alt="Back" src="/icons/back.gif"/></a></p>
                  <p>
                     <a/>
                     <a href="http://www.plantcell.org/cgi/doi/10.1105/tpc.108.062968">www.plantcell.org/cgi/doi/10.1105/tpc.108.062968</a> 
                     
                  </p>
                  <p>
                     
                     
                     Received September  2, 2008; 
                     Revision received May  6, 2009. 
                     accepted May 26, 2009.
                     
                  </p>
                  <p>
                     
                     <a name="BIBL" id="BIBL"/>
                     
                     
                  </p>
                  <div class="section-nav">
                     
                     <h2>
                        REFERENCES
                        
                        
                     </h2>
                     
                  </div>
                  <p>
                     
                     
                     <a name="BIB1" id="BIB1"/><strong>Albani, D., Hammond Kosack, M.C., Smith, C., Conlan, S., Colot, V., Holdsworth, M., and Bevan, M.W.</strong> (1997). The wheat transcriptional activator SPA: A seed-specific bZIP protein that recognizes the GCN4-like motif in the
                     bifactorial endosperm box of prolamin genes. Plant Cell <strong>9:</strong> 171–184.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=9/2/171">[Abstract]</a></p>
                  <p><a name="BIB2" id="BIB2"/><strong>Baena-Gonzalez, E., Rolland, F., Thevelein, J.M., and Sheen, J.</strong> (2007). A central integrator of transcription networks in plant stress and energy signalling. Nature <strong>448:</strong> 938–942.<a href="/cgi/external_ref?access_num=10.1038%2Fnature06069&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=17671505&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB3" id="BIB3"/><strong>Bensmihen, S., Rippa, S., Lambert, G., Jublot, D., Pautot, V., Granier, F., Giraudat, J., and Parcy, F.</strong> (2002). The homologous ABI5 and EEL transcription factors function antagonistically to fine-tune gene expression during late
                     embryogenesis. Plant Cell <strong>14:</strong> 1391–1403.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=14/6/1391">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB4" id="BIB4"/><strong>Bies-Etheve, N., Gaubier-Comella, P., Debures, A., Lasserre, E., Jobet, E., Raynal, M., Cooke, R., and Delseny, M.</strong> (2008). Inventory, evolution and expression profiling diversity of the LEA (late embryogenesis abundant) protein gene family
                     in <em>Arabidopsis thaliana</em>. Plant Mol. Biol. <strong>67:</strong> 107–124.<a href="/cgi/external_ref?access_num=10.1007%2Fs11103-008-9304-x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000254965000008&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=18265943&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB5" id="BIB5"/><strong>Bowman, J.</strong> (1994). Arabidopsis: An Atlas of Morphology and Development. (New York: Springer-Verlag).
                     
                  </p>
                  <p><a name="BIB6" id="BIB6"/><strong>Braybrook, S.A., Stone, S.L., Park, S., Bui, A.Q., Le, B.H., Fischer, R.L., Goldberg, R.B., and Harada, J.J.</strong> (2006). Genes directly regulated by LEAFY COTYLEDON2 provide insight into the control of embryo maturation and somatic embryogenesis.
                     Proc. Natl. Acad. Sci. USA <strong>103:</strong> 3468–3473.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=103/9/3468">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB7" id="BIB7"/><strong>Carrera, E., Holman, T., Medhurst, A., Dietrich, D., Footitt, S., Theodoulou, F.L., and Holdsworth, M.J.</strong> (2008). Seed after-ripening is a discrete developmental pathway associated with specific gene networks in Arabidopsis. Plant
                     J. <strong>53:</strong> 214–224.<a href="/cgi/external_ref?access_num=10.1111%2Fj.1365-313X.2007.03331.x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000252109500002&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=18028281&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB8" id="BIB8"/><strong>Conceicao Ada, S., and Krebbers, E.</strong> (1994). A cotyledon regulatory region is responsible for the different spatial expression patterns of Arabidopsis 2S albumin
                     genes. Plant J. <strong>5:</strong> 493–505.<a href="/cgi/external_ref?access_num=A1994NG46200005&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=8012403&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB9" id="BIB9"/><strong>Egea-Cortines, M., Saedler, H., and Sommer, H.</strong> (1999). Ternary complex formation between the MADS-box proteins SQUAMOSA, DEFICIENS and GLOBOSA is involved in the control
                     of architecture in <em>Antirrhinum majus.</em> EMBO J. <strong>18:</strong> 5370–5379.<a href="/cgi/external_ref?access_num=10.1093%2Femboj%2F18.19.5370&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000083140800021&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=10508169&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB10" id="BIB10"/><strong>Ehlert, A., Weltmeier, F., Wang, X., Mayer, C.S., Smeekens, S., Vicente-Carbajosa, J., and Dröge-Laser, W.</strong> (2006). Two-hybrid protein-protein interaction analysis in Arabidopsis protoplasts: Establishment of a heterodimerisation
                     map of group-C and S bZIP transcription factors. Plant J. <strong>46:</strong> 890–900.<a href="/cgi/external_ref?access_num=10.1111%2Fj.1365-313X.2006.02731.x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000237634900014&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=16709202&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB11" id="BIB11"/><strong>Ellenberger, T.E., Brandl, C.J., Struhl, K., and Harrison, S.C.</strong> (1992). The GCN4 basic region leucine zipper binds DNA as a dimer of uninterrupted alpha helices: Crystal structure of the
                     protein-DNA complex. Cell <strong>71:</strong> 1223–1237.<a href="/cgi/external_ref?access_num=10.1016%2FS0092-8674%2805%2980070-4&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=A1992KE60400016&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=1473154&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB12" id="BIB12"/><strong>Ezcurra, I., Wycliffe, P., Nehlin, L., Ellerstrom, M., and Rask, L.</strong> (2000). Transactivation of the <em>Brassica napus</em> napin promoter by ABI3 requires interaction of the conserved B2 and B3 domains of ABI3 with different cis-elements: B2 mediates
                     activation through an ABRE, whereas B3 interacts with an RY/G-box. Plant J. <strong>24:</strong> 57–66.<a href="/cgi/external_ref?access_num=10.1046%2Fj.1365-313x.2000.00857.x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000089935800006&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=11029704&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB13" id="BIB13"/><strong>Ferrandiz, C., Gu, Q., Martienssen, R., and Yanofsky, M.F.</strong> (2000). Redundant regulation of meristem identity and plant architecture by FRUITFULL, APETALA1 and CAULIFLOWER. Development
                     <strong>127:</strong> 725–734.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=127/4/725">[Abstract]</a></p>
                  <p><a name="BIB14" id="BIB14"/><strong>Finkelstein, R.R., and Lynch, T.J.</strong> (2000). The <em>Arabidopsis</em> abscisic acid response gene ABI5 encodes a basic leucine zipper transcription factor. Plant Cell <strong>12:</strong> 599–609.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=12/4/599">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB15" id="BIB15"/><strong>Giraudat, J., Hauge, B.M., Valon, C., Smalle, J., Parcy, F., and Goodman, H.M.</strong> (1992). Isolation of the <em>Arabidopsis</em> ABI3 gene by positional cloning. Plant Cell <strong>4:</strong> 1251–1261.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=4/10/1251">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB18" id="BIB18"/><strong>Gruis, D., Schulze, J., and Jung, R.</strong> (2004). Storage protein accumulation in the absence of the vacuolar processing enzyme family of cysteine proteases. Plant
                     Cell <strong>16:</strong> 270–290.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=16/1/270">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB16" id="BIB16"/><strong>Guerineau, F.</strong> (1995). Tools for expressing foreign genes in plants. Methods Mol. Biol. <strong>49:</strong> 1–32.<a href="/cgi/external_ref?access_num=8563796&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB17" id="BIB17"/><strong>Gutierrez, L., Van Wuytswinkel, O., Castelain, M., and Bellini, C.</strong> (2007). Combined networks regulating seed maturation. Trends Plant Sci. <strong>12:</strong> 294–300.<a href="/cgi/external_ref?access_num=10.1016%2Fj.tplants.2007.06.003&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000248466400005&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=17588801&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB19" id="BIB19"/><strong>Hanson, J., Hanssen, M., Wiese, A., Hendriks, M.M.W.B., and Smeekens, S.</strong> (2008). The sucrose regulated transcription factor bZIP11 affects amino acid metabolism by regulating the expression of Asparagin
                     Synthase1 and Proline Dehydrogenase 2. Plant J. <strong>53:</strong> 935–949.<a href="/cgi/external_ref?access_num=10.1111%2Fj.1365-313X.2007.03385.x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000253827600005&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=18088315&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB20" id="BIB20"/><strong>Hartings, H., Maddaloni, M., Lazzaroni, N., Di Fonzo, N., Motto, M., Salamini, F., and Thompson, R.</strong> (1989). The O2 gene which regulates zein deposition in maize endosperm encodes a protein with structural homologies to transcriptional
                     activators. EMBO J. <strong>8:</strong> 2795–2801.<a href="/cgi/external_ref?access_num=A1989AR14000003&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=2479535&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB21" id="BIB21"/><strong>Hundertmark, M., and Hincha, D.K.</strong> (2008). LEA (late embryogenesis abundant) proteins and their encoding genes in <em>Arabidopsis thaliana</em>. BMC Genomics <strong>9:</strong> 1–18.<a href="/cgi/external_ref?access_num=10.1186%2F1471-2164-9-1&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000257872300001&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=18171476&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB22" id="BIB22"/><strong>Jakoby, M., Weisshaar, B., Dröge-Laser, W., Vicente-Carbajosa, J., Tiedemann, J., Kroj, T., and Parcy, F.</strong> (2002). bZIP transcription factors in Arabidopsis. Trends Plant Sci. <strong>7:</strong> 106–111.<a href="/cgi/external_ref?access_num=10.1016%2FS1360-1385%2801%2902223-3&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000174264600005&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=11906833&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB23" id="BIB23"/><strong>Kaminaka, H., Nake, C., Epple, P., Dittgen, J., Schütze, K., Chaban, C., Holt III, B.F., Merkle, T., Schafer, E., Harter,
                        K., and Dangl, J.L.</strong> (2006). bZIP10-LSD1 antagonism modulates basal defense and cell death in Arabidopsis following infection. EMBO J. <strong>25:</strong> 4400–4411.<a href="/cgi/external_ref?access_num=10.1038%2Fsj.emboj.7601312&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000241077900024&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=16957775&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB24" id="BIB24"/><strong>Kirby, J., and Kavanagh, T.A.</strong> (2002). NAN fusions: A synthetic sialidase reporter gene as a sensitive and versatile partner for GUS. Plant J. <strong>32:</strong> 391–400.<a href="/cgi/external_ref?access_num=10.1046%2Fj.1365-313X.2002.01422.x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000178965600011&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=12410816&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB25" id="BIB25"/><strong>Kroj, T., Savino, G., Valon, C., Giraudat, J., and Parcy, F.</strong> (2003). Regulation of storage protein gene expression in Arabidopsis. Development <strong>130:</strong> 6065–6073.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=130/24/6065">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB26" id="BIB26"/><strong>Lam, H.M., Wong, P., Chan, H.K., Yam, K.M., Chen, L., Chow, C.M., and Coruzzi, G.M.</strong> (2003). Overexpression of the ASN1 gene enhances nitrogen status in seeds of Arabidopsis. Plant Physiol. <strong>132:</strong> 926–935.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=132/2/926">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB27" id="BIB27"/><strong>Lara, P., Oñate-Sánchez, L., Abraham, Z., Ferrandiz, C., Diaz, I., Carbonero, P., and Vicente-Carbajosa, J.</strong> (2003). Synergistic activation of seed storage protein gene expression in Arabidopsis by ABI3 and two bZIPs related to OPAQUE2.
                     J. Biol. Chem. <strong>278:</strong> 21003–21011.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=jbc&amp;resid=278/23/21003">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB28" id="BIB28"/><strong>Li, F., Asami, T., Wu, X., Tsang, E.W., and Cutler, A.J.</strong> (2007). A putative hydroxysteroid dehydrogenase involved in regulating plant growth and development. Plant Physiol. <strong>145:</strong> 87–97.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=145/1/87">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB29" id="BIB29"/><strong>Mönke, G., Altschmied, L., Tewes, A., Reidt, W., Mock, H.P., Bäumlein, H., and Conrad, U.</strong> (2004). Seed-specific transcription factors ABI3 and FUS3: Molecular interaction with DNA. Planta <strong>219:</strong> 158–166.<a href="/cgi/external_ref?access_num=10.1007%2Fs00425-004-1206-9&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000221192000019&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=14767767&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB30" id="BIB30"/><strong>Moreno-Risueno, M.A., Diaz, I., Carrillo, L., Fuentes, R., and Carbonero, P.</strong> (2007). The HvDOF19 transcription factor mediates the abscisic acid-dependent repression of hydrolase genes in germinating
                     barley aleurone. Plant J. <strong>51:</strong> 352–365.<a href="/cgi/external_ref?access_num=10.1111%2Fj.1365-313X.2007.03146.x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000248790500002&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=17565581&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB31" id="BIB31"/><strong>Nakamura, S., Lynch, T.J., and Finkelstein, R.R.</strong> (2001). Physical interactions between ABA response loci of Arabidopsis. Plant J. <strong>26:</strong> 627–635.<a href="/cgi/external_ref?access_num=10.1046%2Fj.1365-313x.2001.01069.x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000170103400007&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=11489176&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB32" id="BIB32"/><strong>Nakashima, K., Fujita, Y., Katsura, K., Maruyama, K., Narusaka, Y., Seki, M., Shinozaki, K., and Yamaguchi-Shinozaki, K.</strong> (2006). Transcriptional regulation of ABI3- and ABA-responsive genes including RD29B and RD29A in seeds, germinating embryos,
                     and seedlings of Arabidopsis. Plant Mol. Biol. <strong>60:</strong> 51–68.<a href="/cgi/external_ref?access_num=10.1007%2Fs11103-005-2418-5&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000235202000004&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=16463099&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB33" id="BIB33"/><strong>Nantel, A., and Quatrano, R.S.</strong> (1996). Characterization of three rice basic/leucine zipper factors, including two inhibitors of EmBP-1 DNA binding activity.
                     J. Biol. Chem. <strong>271:</strong> 31296–31305.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=jbc&amp;resid=271/49/31296">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB34" id="BIB34"/><strong>Norre, F., Peyrot, C., Garcia, C., Rance, I., Drevet, J., Theisen, M., and Gruber, V.</strong> (2002). Powerful effect of an atypical bifactorial endosperm box from wheat HMWG-Dx5 promoter in maize endosperm. Plant Mol.
                     Biol. <strong>50:</strong> 699–712.<a href="/cgi/external_ref?access_num=10.1023%2FA%3A1019953914467&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000177679200009&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=12374301&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB35" id="BIB35"/><strong>Oñate, L., Vicente Carbajosa, J., Lara, P., Diaz, I., and Carbonero, P.</strong> (1999). Barley BLZ2, a seed-specific bZIP protein that interacts with BLZ1 in vivo and activates transcription from the GCN4-like
                     motif of B-hordein promoters in barley endosperm. J. Biol. Chem. <strong>274:</strong> 9175–9182.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=jbc&amp;resid=274/14/9175">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB37" id="BIB37"/><strong>Oñate-Sánchez, L., and Singh, K.B.</strong> (2002). Identification of Arabidopsis ethylene-responsive element binding factors with distinct induction kinetics after
                     pathogen infection. Plant Physiol. <strong>128:</strong> 1313–1322.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=128/4/1313">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB36" id="BIB36"/><strong>Oñate-Sánchez, L., and Vicente-Carbajosa, J.</strong> (2008). DNA-free RNA isolation protocols for <em>Arabidopsis thaliana</em>, including seeds and siliques. BMC Res Notes. <strong>1:</strong> 93.<a href="/cgi/external_ref?access_num=10.1186%2F1756-0500-1-93&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=18937828&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB38" id="BIB38"/><strong>Parcy, F., and Giraudat, J.</strong> (1997). Interactions between the ABI1 and the ectopically expressed ABI3 genes in controlling abscisic acid responses in
                     Arabidopsis vegetative tissues. Plant J. <strong>11:</strong> 693–702.<a href="/cgi/external_ref?access_num=10.1046%2Fj.1365-313X.1997.11040693.x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=A1997WY17000006&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=9161030&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB39" id="BIB39"/><strong>Parcy, F., Valon, C., Raynal, M., Gaubier-Comella, P., Delseny, M., and Giraudat, J.</strong> (1994). Regulation of gene expression programs during Arabidopsis seed development: Roles of the ABI3 locus and of endogenous
                     abscisic acid. Plant Cell <strong>6:</strong> 1567–1582.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=6/11/1567">[Abstract]</a></p>
                  <p><a name="BIB40" id="BIB40"/><strong>Rook, F., Gerrits, N., Kortstee, A., van Kampen, M., Borrias, M., Weisbeek, P., and Smeekens, S.</strong> (1998). Sucrose-specific signalling represses translation of the Arabidopsis ATB2 bZIP transcription factor gene. Plant J.
                     <strong>15:</strong> 253–263.<a href="/cgi/external_ref?access_num=10.1046%2Fj.1365-313X.1998.00205.x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000075283900010&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=9721683&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB41" id="BIB41"/><strong>Santos-Mendoza, M., Dubreucq, B., Baud, S., Parcy, F., Caboche, M., and Lepiniec, L.</strong> (2008). Deciphering gene regulatory networks that control seed development and maturation in Arabidopsis. Plant J. <strong>54:</strong> 608–620.<a href="/cgi/external_ref?access_num=10.1111%2Fj.1365-313X.2008.03461.x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000255755000008&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=18476867&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB42" id="BIB42"/><strong>Satoh, R., Fujita, Y., Nakashima, K., Shinozaki, K., and Yamaguchi-Shinozaki, K.</strong> (2004). A novel subgroup of bZIP proteins functions as transcriptional activators in hypoosmolarity-responsive expression
                     of the ProDH gene in Arabidopsis. Plant Cell Physiol. <strong>45:</strong> 309–317.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pcellphys&amp;resid=45/3/309">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB43" id="BIB43"/><strong>Schallau, A., Kakhovskaya, I., Tewes, A., Czihal, A., Tiedemann, J., Mohr, M., Grosse, I., Manteuffel, R., and Bäumlein, H.</strong> (2008). Phylogenetic footprints in fern spore- and seed-specific gene promoters. Plant J. <strong>53:</strong> 414–424.<a href="/cgi/external_ref?access_num=10.1111%2Fj.1365-313X.2007.03354.x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000253457300002&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=18086283&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB44" id="BIB44"/><strong>Schmidt, R.J., Burr, F.A., Aukerman, M.J., and Burr, B.</strong> (1990). Maize regulatory gene opaque-2 encodes a protein with a "leucine-zipper" motif that binds to zein DNA. Proc. Natl.
                     Acad. Sci. USA <strong>87:</strong> 46–50.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=87/1/46">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB45" id="BIB45"/><strong>Stangeland, B., and Salehian, Z.</strong> (2002). An improved clearing method for GUS assay in Arabidopsis endosperm and seeds. Plant Mol. Biol. Rep. <strong>20:</strong> 107–114.<a href="/cgi/external_ref?access_num=10.1007%2FBF02799426&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000177202200001&amp;link_type=ISI">[Web of Science]</a></p>
                  <p><a name="BIB46" id="BIB46"/><strong>To, A., Valon, C., Savino, G., Guilleminot, J., Devic, M., Giraudat, J., and Parcy, F.</strong> (2006). A network of local and redundant gene regulation governs <em>Arabidopsis</em> seed maturation. Plant Cell <strong>18:</strong> 1642–1651.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantcell&amp;resid=18/7/1642">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB47" id="BIB47"/><strong>Tunnacliffe, A., and Wise, M.J.</strong> (2007). The continuing conundrum of the LEA proteins. Naturwissenschaften <strong>94:</strong> 791–812.<a href="/cgi/external_ref?access_num=10.1007%2Fs00114-007-0254-y&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000249263900001&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=17479232&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB48" id="BIB48"/><strong>Usadel, B., Blasing, O.E., Gibon, Y., Retzlaff, K., Hohne, M., Gunther, M., and Stitt, M.</strong> (2008). Global transcript levels respond to small changes of the carbon status during progressive exhaustion of carbohydrates
                     in Arabidopsis rosettes. Plant Physiol. <strong>146:</strong> 1834–1861.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=146/4/1834">[Abstract/<span style=" color: CC0000; ">Free</span> Full Text]</a></p>
                  <p><a name="BIB49" id="BIB49"/><strong>Vicente-Carbajosa, J., and Carbonero, P.</strong> (2005). Seed maturation: Developing an intrusive phase to accomplish a quiescent state. Int. J. Dev. Biol. <strong>49:</strong> 645–651.<a href="/cgi/external_ref?access_num=10.1387%2Fijdb.052046jc&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000233266200017&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=16096971&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB50" id="BIB50"/><strong>Walter, M., Chaban, C., Schütze, K., Batistic, O., Weckermann, K., Nake, C., Blazevic, D., Grefen, C., Schumacher, K., Oecking,
                        C., Harter, K., and Kudla, J.</strong> (2004). Visualization of protein interactions in living plant cells using bimolecular fluorescence complementation. Plant
                     J. <strong>40:</strong> 428–438.<a href="/cgi/external_ref?access_num=10.1111%2Fj.1365-313X.2004.02219.x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000224397700010&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=15469500&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB51" id="BIB51"/><strong>Wang, H.W., Zhang, B., Hao, Y.J., Huang, J., Tian, A.G., Liao, Y., Zhang, J.S., and Chen, S.Y.</strong> (2007). The soybean Dof-type transcription factor genes, GmDof4 and GmDof11, enhance lipid content in the seeds of transgenic
                     Arabidopsis plants. Plant J. <strong>52:</strong> 716–729.<a href="/cgi/external_ref?access_num=10.1111%2Fj.1365-313X.2007.03268.x&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000250730600009&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=17877700&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB52" id="BIB52"/><strong>Weber, H., Borisjuk, L., and Wobus, U.</strong> (2005). Molecular physiology of legume seed development. Annu. Rev. Plant Biol. <strong>56:</strong> 253–279.<a href="/cgi/external_ref?access_num=10.1146%2Fannurev.arplant.56.032604.144201&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=15862096&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB53" id="BIB53"/><strong>Weigel, R., and Glazebrook, J.</strong> (2002). Arabidopsis: A Laboratory Manual. (Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press).
                     
                  </p>
                  <p><a name="BIB54" id="BIB54"/><strong>Weltmeier, F., Ehlert, A., Mayer, C.S., Dietrich, K., Wang, X., Schütze, K., Harter, K., Vicente-Carbajosa, J., and Dröge-Laser,
                        W.</strong> (2006). Combinatorial control of Arabidopsis proline dehydrogenase transcription by specific heterodimerisation bZIP transcription
                     factors. EMBO J. <strong>25:</strong> 3133–3143.<a href="/cgi/external_ref?access_num=10.1038%2Fsj.emboj.7601206&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000239625800017&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=16810321&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB55" id="BIB55"/><strong>Weltmeier, F., Rahmani, F., Ehlert, A., Dietrich, K., Schütze, K., Wang, X., Chaban, C., Hanson, J., Teige, M., Harter, K.,
                        Vicente-Carbajosa, J., Smeekens, S., and Dröge-Laser, W.</strong> (2009). Expression patterns within the Arabidopsis C/S1 bZIP transcription factor network: Availability of heterodimerization
                     partners controls gene expression during stress response and development. Plant Mol. Biol. <strong>69:</strong> 107–119.<a href="/cgi/external_ref?access_num=10.1007%2Fs11103-008-9410-9&amp;link_type=DOI">[CrossRef]</a><a href="/cgi/external_ref?access_num=000261181700009&amp;link_type=ISI">[Web of Science]</a><a href="/cgi/external_ref?access_num=18841482&amp;link_type=MED">[Medline]</a></p>
                  <p><a name="BIB56" id="BIB56"/><strong>Xu, D., Duan, X., Wang, B., Hong, B., Ho, T., and Wu, R.</strong> (1996). Expression of a late embryogenesis abundant protein gene, HVA1, from barley confers tolerance to water deficit and
                     salt stress in transgenic rice. Plant Physiol. <strong>110:</strong> 249–257.<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=plantphysiol&amp;resid=110/1/249">[Abstract]</a></p>
               </div>
      <span class="highwire-journal-article-marker-end"/>
    </div>
  </body>
</html>
