<?xml version="1.0"?>
<html>
  <body>
    <div class="padded-inner">&#13;
                            &#13;
&#13;
&#13;
<div id="topmatter">&#13;
	&#13;
	   &#13;
&#13;
&#13;
&#13;
&#13;
&#13;
&#13;
&#13;
 &#13;
&#13;
				<a href="/about/access"><img height="24" width="102" alt="Open Access" src="/graphics/flashes/openaccess-large.png" class="access mr15"/></a>&#13;
			<a href="http://www.biomedcentral.com/about/mostviewed/"><img height="24" width="130" alt="Highly Access" src="/graphics/flashes/highlyaccessed-large.png" class="access mr15"/></a>&#13;
				<span class="articletype">Research article</span>&#13;
<h1>Gene coexpression clusters and putative regulatory elements underlying seed storage reserve accumulation in Arabidopsis</h1>&#13;
&#13;
        &#13;
	<div class="singleins">&#13;
	&#13;
		&#13;
	<p class="authors">&#13;
        																																																																	        <strong>Fred Y Peng</strong> and <strong>Randall J Weselake</strong><sup>*</sup></p>&#13;
&#13;
			<div id="affiliations">&#13;
		            <div class="module gray  inner">&#13;
                <div class="module-inner padded-inner">&#13;
		<ul><li>                                    &#13;
                            		<p>&#13;
							<span>*</span>&#13;
									 Corresponding author:										            Randall J Weselake <a href="mailto:Randall.Weselake@ualberta.ca">Randall.Weselake@ualberta.ca</a>    					</p>&#13;
      &#13;
</li>&#13;
			&#13;
                                                                        		</ul><p class="options"><a class="affiliations-toggle"><i class="arrow"/>Author Affiliations</a></p>&#13;
				<div id="ins_container" style="display: block;">&#13;
										    						    <p class="singleInstitute">&#13;
						                                        Agricultural Lipid Biotechnology Program, Department of Agricultural, Food and Nutritional Science, University of Alberta, Edmonton, Alberta, T6G 2P5, Canada&#13;
                            </p>&#13;
																&#13;
				</div>&#13;
										<p id="authoremails">            For all author emails, please <a href="/bmcgenomics/logon">log on</a>.&#13;
    </p>&#13;
						&#13;
				                </div>&#13;
                    <div class="module-tl"/>&#13;
                    <div class="module-tr"/>&#13;
                    <div class="module-bl"/>&#13;
                    <div class="module-br"/>&#13;
            </div>&#13;
		</div>&#13;
	  </div>&#13;
    <p><em>BMC Genomics</em> 2011, <strong>12</strong>:286 <span class="pseudotab">doi:10.1186/1471-2164-12-286</span></p>&#13;
&#13;
    &#13;
    		&#13;
        		<br/><p>The electronic version of this article is the complete one and can be found online at: <a href="http://www.biomedcentral.com/1471-2164/12/286">http://www.biomedcentral.com/1471-2164/12/286</a></p>&#13;
		<br/><table cellspacing="0" cellpadding="0"><tbody><tr><td>Received:</td><td>24 August 2010</td></tr><tr><td>Accepted:</td><td>2 June 2011</td></tr><tr><td>Published:</td><td>2 June 2011</td></tr></tbody></table><br/><p>&#13;
            © 2011 Peng and Weselake; licensee BioMed Central Ltd. <br/></p>&#13;
					<p>&#13;
				This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/2.0">http://creativecommons.org/licenses/by/2.0</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.&#13;
			</p>&#13;
		    &#13;
&#13;
</div>&#13;
    			&#13;
    <a name="abs"/><h3>Abstract</h3>
<h4>Background</h4>
<p style="line-height:160%">In Arabidopsis, a large number of genes involved in the accumulation of seed storage
   reserves during seed development have been characterized, but the relationship of
   gene expression and regulation underlying this physiological process remains poorly
   understood. A more holistic view of this molecular interplay will help in the further
   study of the regulatory mechanisms controlling seed storage compound accumulation.
</p>
<h4>Results</h4>
<p style="line-height:160%">We identified gene coexpression networks in the transcriptome of developing Arabidopsis
   (<em>Arabidopsis thaliana</em>) seeds from the globular to mature embryo stages by analyzing publicly accessible
   microarray datasets. Genes encoding the known enzymes in the fatty acid biosynthesis
   pathway were found in one coexpression subnetwork (or cluster), while genes encoding
   oleosins and seed storage proteins were identified in another subnetwork with a distinct
   expression profile. In the triacylglycerol assembly pathway, only the genes encoding
   diacylglycerol acyltransferase 1 (DGAT1) and a putative cytosolic "type 3" DGAT exhibited
   a similar expression pattern with genes encoding oleosins. We also detected a large
   number of putative <em>cis</em>-acting regulatory elements in the promoter regions of these genes, and promoter motifs
   for LEC1 (LEAFY COTYLEDON 1), DOF (DNA-binding-with-One-Finger), GATA, and MYB transcription
   factors (TF), as well as SORLIP5 (Sequences Over-Represented in Light-Induced Promoters
   5), are overrepresented in the promoter regions of fatty acid biosynthetic genes.
   The conserved CCAAT motifs for B3-domain TFs and binding sites for bZIP (basic-leucine
   zipper) TFs are enriched in the promoters of genes encoding oleosins and seed storage
   proteins.
</p>
<h4>Conclusions</h4>
<p style="line-height:160%">Genes involved in the accumulation of seed storage reserves are expressed in distinct
   patterns and regulated by different TFs. The gene coexpression clusters and putative
   regulatory elements presented here provide a useful resource for further experimental
   characterization of protein interactions and regulatory networks in this process.
</p><a name="sec1"/><h3>Background</h3>
<p style="line-height:160%">Seed storage reserves accumulated during embryogenesis in higher plants are crucial
   for plant propagation, providing carbon and energy during germination prior to seedling
   establishment. In mature Arabidopsis seeds, storage lipids and proteins are the major
   storage compounds, each accounting for 30% - 45% of the seed dry weight <a name="d14582e130"/>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>]. The past decade has witnessed a substantial progress in identification and characterization
   of genes involved in the <em>de novo </em>fatty acid (FA) biosynthesis and triacylglycerol (TAG) assembly pathways [<a name="d14582e137"/><a name="d14582e139"/>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>,<a onclick="LoadInParent('#B4'); return false;" href="#B4">4</a>] and references therein]. This advancement is particularly evident in the model plant
   Arabidopsis, largely owing to the sequencing and release of its relatively compact
   genome in the year 2000 <a name="d14582e143"/>[<a onclick="LoadInParent('#B5'); return false;" href="#B5">5</a>]. Moreover, characterization of transcription factors (TFs) has led to the identification
   of several master regulator genes that play critical regulatory roles in this biological
   process, including <em>ABI3 </em>(<em>ABSCISIC ACID INSENSITIVE 3</em>), <em>LEC1 </em>(<em>LEAFY COTYLEDON 1</em>), <em>LEC2 </em>and <em>FUS3 </em>(<em>FUSCA 3</em>) <a name="d14582e170"/><a name="d14582e172"/><a name="d14582e174"/><a name="d14582e176"/><a name="d14582e178"/><a name="d14582e180"/><a name="d14582e182"/><a name="d14582e184"/><a name="d14582e186"/><a name="d14582e188"/><a name="d14582e190"/><a name="d14582e193"/>[<a onclick="LoadInParent('#B6'); return false;" href="#B6">6</a>-<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a>]. These TFs interact with each other and form complex regulatory networks <a name="d14582e197"/><a name="d14582e199"/><a name="d14582e201"/><a name="d14582e203"/><a name="d14582e205"/><a name="d14582e207"/>[<a onclick="LoadInParent('#B18'); return false;" href="#B18">18</a>-<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a>], regulating multiple aspects of seed development including storage reserve accumulation
   through interaction with cognate <em>cis</em>-acting DNA elements in the promoter regions of target genes. ABI3, FUS3 and LEC2
   contain a plant-specific 'B3' DNA-binding domain which targets RY-repeat regulatory
   elements, whereas LEC1 and L1L (LEC1-LIKE) contain a NF-YB domain binding to the CCAAT
   boxes in the promoter region <a name="d14582e214"/><a name="d14582e216"/>[<a onclick="LoadInParent('#B24'); return false;" href="#B24">24</a>,<a onclick="LoadInParent('#B25'); return false;" href="#B25">25</a>]. Additional TFs such as WRINKLED 1 (WRI1), a member of plant-specific APETALA 2 (AP2)
   - ethylene response element binding factor (EREB) family, is also known to control
   transcription of many FA biosynthetic genes <a name="d14582e220"/>[<a onclick="LoadInParent('#B26'); return false;" href="#B26">26</a>], and recent studies show it acts via binding to the AW-box motif present in the promoter
   region of 19 FA biosynthetic genes <a name="d14582e224"/>[<a onclick="LoadInParent('#B27'); return false;" href="#B27">27</a>]. Moreover, ABI4 (an AP2 family protein) and various basic-leucine zipper (bZIP) TFs
   including ABI5 or EEL (ENHANCED EM [EMBRYO MORPHORGENESIS] LEVEL) are known regulators
   of the expression of <em>SEED STORAGE PROTEIN </em>(<em>SSP</em>) genes, which act in the same signalling network but downstream of ABI3 <a name="d14582e235"/><a name="d14582e237"/>[<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a>,<a onclick="LoadInParent('#B29'); return false;" href="#B29">29</a>]. Distinct regulatory mechanisms are present in controlling the accumulation processes
   of oils and proteins, perhaps with cross-talk to coordinate the synthesis of seed
   storage compounds. This coordination could help to explain the well-documented negative
   correlation (correlation coefficient ranging from -0.60 to -0.90) between oil and
   protein content in seeds of many oleaginous species [<a name="d14582e241"/>[<a onclick="LoadInParent('#B3'); return false;" href="#B3">3</a>] and references therein]. Moreover, several TFs, such as LEC1, ABI3 and FUS3, have
   been demonstrated to regulate many genes in the synthesis of both oils and storage
   proteins in developing seeds <a name="d14582e245"/><a name="d14582e247"/><a name="d14582e249"/>[<a onclick="LoadInParent('#B30'); return false;" href="#B30">30</a>-<a onclick="LoadInParent('#B32'); return false;" href="#B32">32</a>].
</p>
<p style="line-height:160%">In contrast to the great advancement in characterizing individual genes involved in
   the accumulation of seed storage reserves, the relationship of their expression and
   regulation is not well understood. A more holistic view of this biological process
   at the systems level would prove beneficial in developing strategies to further enhance
   seed yield and oil content, as well as in the modification of oil composition. To
   gain insights into global transcriptional dynamics in key cellular processes, microarray
   is an effective method for analyzing the transcript abundance of a large number of
   genes simultaneously. Datasets obtained from profiling experiments can be further
   used to infer gene regulatory networks. In Arabidopsis, two cDNA microarrays were
   designed several years ago based on the expressed sequence tag (EST) sequences available
   at the time. One array was used for tissue-specific expression profiling to identify
   genes that are preferentially expressed in developing seeds compared with vegetative
   leaves and roots <a name="d14582e255"/>[<a onclick="LoadInParent('#B33'); return false;" href="#B33">33</a>], and the other was used to study the temporal pattern of gene expression during the
   critical period of seed filling <a name="d14582e259"/>[<a onclick="LoadInParent('#B34'); return false;" href="#B34">34</a>]. These transcriptional profiling studies in Arabidopsis seeds have greatly increased
   our understanding of overall alterations of gene expression during seed development
   and storage reserve accumulation. These two early cDNA-based microarrays, however,
   surveyed &lt;3500 unique Arabidopsis genes.
</p>
<p style="line-height:160%">More recently, Schmid et al. <a name="d14582e265"/>[<a onclick="LoadInParent('#B35'); return false;" href="#B35">35</a>] created a global gene expression atlas AtGenExpress (Expression Atlas of Arabidopsis
   development) representing the Arabidopsis life cycle using the Arabidopsis ATH1 genome
   array (Affymetrix, Santa Clara, CA), which can measure nearly 24,000 genes in a single
   assay. In AtGenExpress, 237 chips were hybridized for 79 different samples collected
   from various organs, growth stages and under various environmental conditions, including
   24 arrays for eight stages of maturing seeds. Since its release, this exceptionally
   large transcriptome dataset has been a goldmine for plant biologists to identify candidate
   genes for molecular characterization. A number of studies have further "mined" this
   dataset within different contexts of plant biology. Wang et al. <a name="d14582e269"/>[<a onclick="LoadInParent('#B36'); return false;" href="#B36">36</a>] extracted the expression data for several TFs experimentally determined to regulate
   seed development and genes that code for enzymes in the FA biosynthesis pathway. Volodarsky
   et al. <a name="d14582e273"/>[<a onclick="LoadInParent('#B37'); return false;" href="#B37">37</a>] utilized the dataset to analyze hormone-related transcriptional activities in Arabidopsis.
   Vandepoele et al. <a name="d14582e277"/>[<a onclick="LoadInParent('#B38'); return false;" href="#B38">38</a>] constructed coexpression networks and predicted <em>cis</em>-regulatory elements for the cell cycle-related TF OBP1. Recently, the identification
   of gene coexpression networks has emerged as a popular method for predicting gene
   functions and interactions <a name="d14582e285"/><a name="d14582e287"/><a name="d14582e289"/><a name="d14582e291"/>[<a onclick="LoadInParent('#B38'); return false;" href="#B38">38</a>-<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>], and web-based tools such as Genevestigator <a name="d14582e295"/>[<a onclick="LoadInParent('#B42'); return false;" href="#B42">42</a>] and CressExpress <a name="d14582e299"/>[<a onclick="LoadInParent('#B43'); return false;" href="#B43">43</a>] have been developed to facilitate such analyses at a small scale for plant biologists.
   Transcriptional coordination, or coexpression, of genes may be an indication of functional
   relatedness, based on the "guilt-by-association" principle <a name="d14582e303"/>[<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>]. In a coexpression network, a vertex or node represents a gene whereas an edge is
   a connection inferred from the correlation coefficient calculated from the gene expression
   data. Although the relationship between coexpression networks and true biological
   networks is often not clear, it has been shown that gene groups identified from modular
   (cluster) analysis in coexpression networks often exhibit an enrichment of certain
   Gene Ontology (GO) categories <a name="d14582e307"/>[<a onclick="LoadInParent('#B45'); return false;" href="#B45">45</a>], suggesting the functional association of genes connected in a coexpression network.
   Hence, a coexpression edge can be considered a putative interaction between two genes.
   Genes in a coexpression network, particularly those expressed in a specific tissue
   or sharing a semantic similarity in the GO 'Biological Process' aspect, might be co-regulated
   through common TF binding sites in their upstream regions, leading to many attempts
   to identify overrepresented <em>cis</em>-motifs in coexpressed genes <a name="d14582e315"/><a name="d14582e317"/><a name="d14582e319"/><a name="d14582e321"/><a name="d14582e323"/>[<a onclick="LoadInParent('#B46'); return false;" href="#B46">46</a>-<a onclick="LoadInParent('#B50'); return false;" href="#B50">50</a>].
</p>
<p style="line-height:160%">In the current study, we took advantage of this public transcriptome dataset in Arabidopsis
   <a name="d14582e329"/>[<a onclick="LoadInParent('#B35'); return false;" href="#B35">35</a>], analyzed the raw data thoroughly in the context of seed storage reserve accumulation
   during seed development, and constructed coexpression networks for seed-expressed
   genes. We focused on genes involved in FA biosynthesis and the accumulation of storage
   lipids and proteins in developing seeds. This comprehensive analysis has resulted
   in the identification of a large number of genes that are possibly coexpressed and
   function cooperatively during seed maturation. Furthermore, we predicted a large number
   of <em>cis</em>-regulatory elements for key seed-expressed genes. This information could be useful
   in designing experiments to probe regulatory mechanisms underlying seed storage reserve
   accumulation.
</p><a name="sec2"/><h3>Results and Discussion</h3>
<h4>Association of seed transcriptome with embryo morphology in developing Arabidopsis
   seeds
</h4>
<p style="line-height:160%">Using the raw intensity data generated by AtGenExpress for a global gene expression
   atlas throughout the Arabidopsis life cycle <a name="d14582e346"/>[<a onclick="LoadInParent('#B35'); return false;" href="#B35">35</a>], we performed a detailed analysis of gene expression pertaining to seed storage reserve
   accumulation during the eight stages of seed development, ranging from globular embryo
   to mature embryo stages (Table <a name="d14582e349"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/table/T1">1</a>). Of the nearly 24,000 genes represented on the Affymetrix GeneChip ATH1 genome array,
   we estimated that approximately 12,353 genes (or ~54%) were expressed in at least
   one of the eight development stages. Our analysis took into account the fact that
   certain genes might be transiently expressed at only one stage during seed development.
   The relatively high log2 intensity value of 6.0 was chosen as the threshold to focus
   on the genes with at least a modest level of expression. The global transcriptional
   activity in the developing Arabidopsis seed is higher than in the leaf, lower than
   in the flower, and comparable to that in the apex, root or stem (data not shown).
</p><div class="figs">
<div class="table"><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/table/T1','T1',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/table/T1"><strong>Table 1.</strong></a> Arabidopsis developing seed samples used for AtGenExpress microarray experiments.
   
</p></div></div>
<p style="line-height:160%">To examine the overall transcriptome changes across the eight seed development stages,
   we performed a principal component analysis (PCA) in the 'Sample' space, and the results
   indicated that the global transcriptional program changes constantly during seed maturation
   (Figure <a name="d14582e529"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F1">1</a>). In PCA, the first principal component (i.e., development stage) was estimated to
   explain ~83% of variance in the seed transcriptome, indicating that embryogenesis
   is the predominant cause for the substantial variation observed in the transcript
   population. The differences in the global gene expression patterns among the eight
   developing stages were cross validated by a global association test <a name="d14582e533"/>[<a onclick="LoadInParent('#B51'); return false;" href="#B51">51</a>], showing that the seed transcriptome varied across the eight developmental stages
   in a statistically significant manner (<em>P </em>&lt; 0.0001). The presence of siliques in the young seeds (S3 to S5; Table <a name="d14582e539"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/table/T1">1</a>) may have had an effect on global transcript profiles in the seeds of earlier development
   stages, but its minor effect cannot be dissected from that of seed development under
   the experimental design in <a name="d14582e543"/>[<a onclick="LoadInParent('#B35'); return false;" href="#B35">35</a>]. Additionally, Figure <a name="d14582e547"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F1">1</a> shows that each stage has a distinct transcriptome signature that generally corresponds
   to its seed development stage defined by the embryo's morphology. For instance, as
   shown in Figure <a name="d14582e550"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F1">1</a>, the globular embryo stage (with three replicates) grouped tightly, the two samples
   from the bilateral stage clustered together but separately from other stages, and
   in general, samples from the expanded cotyledon stage and the mature embryo stage
   also clustered corresponding to their morphological stages, respectively. The transcriptome
   signature for one expanded cotyledon stage (with an asterisk in Figure <a name="d14582e553"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F1">1</a>), however, was closer to the two samples of the mature cotyledon stage, rather than
   the expanded cotyledon stage defined by embryo morphology. This result suggests that
   staging of seed development based on the embryo's morphological shape alone may not
   necessarily reflect the transcriptome state in the seed, which is attributable to
   the fact that molecular events, such as gene expression, occur prior to morphological
   changes. Consistent with the highly dynamic landscape in global gene expression, our
   analysis on individual genes using the method in <a name="d14582e557"/>[<a onclick="LoadInParent('#B52'); return false;" href="#B52">52</a>] indicated that nearly all the genes expressed in developing Arabidopsis seeds are
   differentially transcribed under a stringent false discovery rate (FDR) threshold
   of 0.01 (data not shown). This lack of stably expressed genes with adequate transcript
   abundance brings into focus the challenge of determining reference genes that can
   be used for normalization in quantifying mRNAs in developing seeds <a name="d14582e561"/>[<a onclick="LoadInParent('#B53'); return false;" href="#B53">53</a>]. In summary, this analysis demonstrates that the transcriptional program is subject
   to constant alterations during seed development as many other studies have shown,
   suggesting its tight regulation at the transcriptional level.
</p><div class="figs">
<div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F1','F1',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F1"><img align="top" src="/content/figures/1471-2164-12-286-1.gif" alt="thumbnail" class="thumbnail"/><strong>Figure 1.</strong></a> <strong>The transcriptome dynamics during Arabidopsis seed development</strong>. The normalized, log<sub>2</sub>-transformed expression data for the 24 samples were subjected to principal component
   analysis (PCA) using the R prcomp function <a name="d14582e580"/>[<a onclick="LoadInParent('#B83'); return false;" href="#B83">83</a>]. PC1 and PC2 are the first two principal components in the dataset. Different symbols
   and colours shown at the bottom of the figure were used for different seed developmental
   stages to show the relationship between molecular and morphological phenotypes. As
   in Table <a name="d14582e583"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/table/T1">1</a>, the different samples are as follows: S3: C globular stage; S4: D bilateral stage;
   S5: D bilateral stage; S6: E expanded cotyledon stage; S7: E expanded cotyledon stage;
   S8: E expanded cotyledon stage; S9: F mature embryo stage; S10: F mature embryo stage.
</p></div></div>
<h4>Construction of gene coexpression networks in the Arabidopsis seed transcriptome</h4>
<p style="line-height:160%">To infer the gene coexpression network in the transcriptome of developing Arabidopsis
   seeds, we focused on the 12,353 genes with moderate or high expression levels. The
   Pearson-based correlation coefficient was used as a measure of expression coherence,
   and we applied a correlation threshold of 0.90 and retained over 1.7 million correlated
   gene pairs representing 11,698 distinct genes. The resulting coexpression networks
   encompassed approximately 95% of seed- expressed genes, indicating that the majority
   of expressed genes in Arabidopsis seeds act in a concerted manner. We chose such a
   stringent correlation threshold considering the relatively small sample size in the
   analysis so that gene pairs in the coexpression network are statistically significant
   (<em>P </em>= 0.0005 using Fisher's Z transformation), meaning the probability of randomly obtaining
   a correlation coefficient of â¥ 0.90 in this seed transcriptome dataset is small. The
   frequency distribution of the number of connections is shown in Figure <a name="d14582e596"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F2">2</a>. Nayak et al. <a name="d14582e600"/>[<a onclick="LoadInParent('#B40'); return false;" href="#B40">40</a>] used the absolute correlation (|r|) to construct a gene coexpression network in human
   immortalized B cells, but we believe that positive and negative correlations in gene
   expression may indicate different biological interactions (synergistic or antagonistic),
   and therefore we only included gene pairs with positive correlation coefficients above
   the threshold for the coexpression analysis. Nevertheless, gene pairs consistently
   expressed in a negatively correlated manner can also be of great interest to biologists.
</p><div class="figs">
<div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F2','F2',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F2"><img align="top" src="/content/figures/1471-2164-12-286-2.gif" alt="thumbnail" class="thumbnail"/><strong>Figure 2.</strong></a> <strong>Summary of the gene coexpression network in developing Arabidopsis seeds</strong>. Distribution of the number of genes in different bins of edge numbers in the coexpression
   network of seed-expressed genes. The edge numbers were divided into different ranges
   and the frequency of nodes in each bin was found to summarize the coexpression network.
   The bin categories are as follows: 1; 2 - 49; 50 - 99; 100 - 149; 150 - 199; 200 -
   249; 250 - 299; and â¥300.
</p></div></div>
<p style="line-height:160%">We also used a complementary clustering approach to identify gene clusters with similar
   expression profiles during seed maturation (Figure <a name="d14582e618"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F3">3</a>). We found six clusters could sufficiently represent the distinct patterns inherent
   in this seed transcriptome dataset, with some clusters being the "mirror images" of
   others. The first two clusters included the majority of genes related to the accumulation
   of seed storage reserves, which will be described in more detail below. It is important
   to point out that the method for identifying coexpression networks is computationally
   similar to various clustering approaches, using correlation coefficient (r) as the
   similarity measure, or alternately 1 - |r| as the distance measure. An important difference
   exists, however, in the parameters used in the two processes: the number of clusters
   is often specified in clustering although certain assessment can be performed beforehand,
   whereas the correlation threshold is chosen in the coexpression network analysis.
   We believe our approach of coexpression network identification, coupled with clustering,
   is advantageous for identification of genes in the same coexpression cluster with
   visible expression patterns during seed maturation, enabling easier biological interpretation
   and various complementary analyses.
</p><div class="figs">
<div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F3','F3',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F3"><img align="top" src="/content/figures/1471-2164-12-286-3.gif" alt="thumbnail" class="thumbnail"/><strong>Figure 3.</strong></a> <strong>Fuzzy clustering of the expression data along seed development series</strong>. The six clusters showing the expression patterns during Arabidopsis seed development.
   The gene expression values were standardized to have a mean value of zero and a standard
   deviation of one for each gene profile. The transformed expressions were then clustered
   using the fuzzy c-means (FCM) clustering algorithm implemented in the Bioconductor
   Mfuzz package <a name="d14582e634"/>[<a onclick="LoadInParent('#B89'); return false;" href="#B89">89</a>]. Based on preliminary analysis, we found six clusters can well represent different
   expression patterns inherent in the dataset, and another FCM parameter m = 1.75. A
   membership value in the range of 0-1 was assigned in clustering and the cluster cores
   consisting of genes with membership value &gt; = 0.90 were coloured pink.
</p></div></div>
<p style="line-height:160%">Several parameters can be used to describe a biological network, including the clustering
   coefficient and scale-free topoplogy criterion. The scale-free topology criterion
   ranges from zero to one for typical biological networks under investigation <a name="d14582e641"/><a name="d14582e643"/><a name="d14582e645"/>[<a onclick="LoadInParent('#B54'); return false;" href="#B54">54</a>-<a onclick="LoadInParent('#B56'); return false;" href="#B56">56</a>]. The clustering coefficient and scale-free topology criterion were 0.73 and 0.68,
   respectively, in this Arabidopsis seed coexpression network (Table <a name="d14582e648"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/table/T2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/table/T2">2</a>), indicating topological similarity to other biological networks. As shown in Figure
   <a name="d14582e651"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F2">2</a>, the network is comprised of many genes with few links (e.g., most genes have two
   to 100 putative coexpression partners) but relatively few genes with many connections,
   which is consistent with the power-law distribution widely present in biological networks.
   In the coexpression network, each gene has a median of 71 edges. It is notable that
   a relatively large number of genes have â¥300 edges (Figure <a name="d14582e654"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F2">2</a>), which is at least partly due to this larger range containing all remaining numbers
   of connections. We observed the edge numbers for genes in different Gene Ontology
   (GO) 'Biological Process' categories and did not find any association between the
   number of coexpression partners and obvious functional significance (data not shown);
   TF gene <em>LEC1 </em>and a ribosomal protein S18 gene (<em>RPS18</em>), for instance, were found to connect with 38 and 178 coexpression partners, respectively.
   This indicates that, while the number of edges for a node may suggest the functional
   significance of the gene, the centrality (or location) of a node in the network can
   be more important. This aspect has been well described in social network analysis
   <a name="d14582e665"/>[<a onclick="LoadInParent('#B57'); return false;" href="#B57">57</a>].
</p><div class="figs">
<div class="table"><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/table/T2','T2',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/table/T2"><strong>Table 2.</strong></a> Network characteristics in the Arabidopsis seed coexpression network.
</p></div></div>
<h4>Genes encoding fatty acid biosynthetic genes and seed storage reserve associated proteins
   are located in different subnetworks
</h4>
<p style="line-height:160%">While the entire coexpression network is useful for network topology analysis, isolation
   of a subnetwork (or cluster) makes it more accessible to biologists <a name="d14582e790"/><a name="d14582e792"/>[<a onclick="LoadInParent('#B40'); return false;" href="#B40">40</a>,<a onclick="LoadInParent('#B58'); return false;" href="#B58">58</a>]. More importantly, a subnetwork in the large coexpression network is often more biologically
   relevant in a pathway context. Hence, we extracted subnetworks from this gene coexpression
   network for genes relevant to the accumulation of seed storage reserves (Figure <a name="d14582e795"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F4">4</a>). Of the 48 genes known to encode enzymes involved in FA biosynthesis <a name="d14582e799"/><a name="d14582e801"/>[<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a>,<a onclick="LoadInParent('#B59'); return false;" href="#B59">59</a>], we identified 44 (or ~92%) genes represented on the ATH1 array, and all of them
   were found in one subnetwork (Figure <a name="d14582e804"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F4">4A</a>). This subnetwork cluster consists of 1854 genes (Additional File <a name="d14582e807"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/suppl/S1">1</a>), which is in general agreement with an interactive correlation network generated
   genome-wide in Arabidopsis using a heuristic clustering algorithm <a name="d14582e812"/>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>]. Such a gene list can be used to identify interactors of genes in FA synthesis in
   developing seeds. Consistent with the coexpression subnetwork analysis, the majority
   of genes involved in FA biosynthesis were associated with Cluster 1 (Figure <a name="d14582e815"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F3">3</a>). Their expression levels increased steadily from the globular embryo stage, generally
   reached the peak at the expanded cotyledon stage, and dramatically declined subsequently
   throughout late seed maturation (Figure <a name="d14582e818"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F4">4B</a>). Such a unified expression pattern for most FA biosynthetic genes supports earlier
   studies showing that FA supply can be a limiting factor for triacylglycerol (TAG)
   accumulation in developing embryos of <em>Brassica napus </em><a name="d14582e824"/>[<a onclick="LoadInParent('#B60'); return false;" href="#B60">60</a>], olive (<em>Olea europaea </em>L.) and oil palm (<em>Elaeis guineensis </em>Jacq.)<a name="d14582e835"/>[<a onclick="LoadInParent('#B61'); return false;" href="#B61">61</a>], as well as <em>cuphea lanceolata </em>and other oil species <a name="d14582e842"/>[<a onclick="LoadInParent('#B62'); return false;" href="#B62">62</a>]. Recent studies of metabolic flux in developing embryos of <em>B. napus</em>, however, indicated that TAG assembly was more limiting than FA biosynthesis in regulating
   the flow of carbon into TAG <a name="d14582e849"/>[<a onclick="LoadInParent('#B63'); return false;" href="#B63">63</a>]. The majority of genes encoding oilbody oleosins and SSPs were found in another subnetwork
   with a distinct expression pattern (Figure <a name="d14582e853"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F4">4C</a>). The subnetwork encompassing genes encoding oleosins and SSPs is comprised of 1392
   genes (Additional File <a name="d14582e856"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/suppl/S2','',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/suppl/S2">2</a>). Genes encoding oleosins and SSPs were in Cluster 2 (Figure <a name="d14582e859"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F3">3</a>), and their expression profiles were strikingly similar. These genes were virtually
   unexpressed at the globular stage, increased rapidly (&gt;1000-fold in many cases) from
   the globular stage to the bilaternal stage, and remained at the elevated expression
   level throughout the remaining stages of seed maturation (Figure <a name="d14582e862"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F4">4D</a>). Transcripts for <em>OLEOSIN </em>and <em>SSP </em>genes are most abundant in the seed transcriptome late during seed development. In
   contrast, most genes in the TAG assembly pathway were found in different subnetworks,
   exhibiting various expression profiles during seed development (Figure <a name="d14582e872"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F5','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F5">5</a>). <em>DIACYLGLYCEROL ACYLTRANSFERASE 1 (DGAT1</em>), <em>FATTY ACID DESATURASE 2 (FAD2</em>), <em>FATTY ACID ELONGASE 1 </em>(<em>FAE1</em>) and <em>STEAROYL DESATURASE </em>(<em>SAD</em>) genes were identified in this subnetwork, albeit expressed at substantially lower
   levels compared to genes encoding oleosins and SSPs (Additional File <a name="d14582e894"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/suppl/S3','',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/suppl/S3">3</a>). DGAT catalyzes the acyl-CoA-dependent acylation of <em>sn</em>-1,2-diacylglycerol to produce TAG and CoA <a name="d14582e901"/>[<a onclick="LoadInParent('#B64'); return false;" href="#B64">64</a>]. FAD2 catalyzes the introduction of a second double bond into acyl groups in phospholipid
   whereas SAD catalyzes the formation of monounsaturated FA in the plastid <a name="d14582e905"/>[<a onclick="LoadInParent('#B65'); return false;" href="#B65">65</a>]. FAE1 catalyzes the elongation oleoyl-CoA in the endoplasmic reticulum <a name="d14582e909"/>[<a onclick="LoadInParent('#B65'); return false;" href="#B65">65</a>]. Our analysis determined that AT1G48300, which was named <em>DGAT3</em>, is the putative gene encoding a cytosolic DGAT in Arabidopsis. The amino acid sequence
   of AT1G48300 has a significantly high degree of similarity (expect value &lt; 1 Ã 10<sup>-21</sup>) to the soluble DGAT in peanut (<em>Arachis hypogaea</em>), where the cytosolic <em>DGAT </em>gene in plants was first discovered <a name="d14582e926"/>[<a onclick="LoadInParent('#B66'); return false;" href="#B66">66</a>]. Notably, <em>DGAT3 </em>exhibited a similar expression pattern with <em>DGAT1</em>, but expressed higher during late seed maturation. In earlier studies, quantification
   of DGAT activity during seed maturation in <em>B. napus </em>indicated that enzyme activity was maximal during the rapid phase of oil accumulation
   with a substantial decrease in activity occurring as oil levels reached a plateau
   <a name="d14582e940"/><a name="d14582e942"/>[<a onclick="LoadInParent('#B67'); return false;" href="#B67">67</a>,<a onclick="LoadInParent('#B68'); return false;" href="#B68">68</a>]. Assuming DGAT activity shows a similar profile during seed development in Arabidopsis,
   this suggests that DGAT may be down-regulated post-transcriptionally and/or post-translationally
   during the latter stages of seed development.
</p><div class="figs">
<div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F4','F4',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F4"><img align="top" src="/content/figures/1471-2164-12-286-4.gif" alt="thumbnail" class="thumbnail"/><strong>Figure 4.</strong></a> <strong>Subnetwork and temporal expression profiles for genes involved in seed storage reserve
      accumulation in developing Arabidopsis seeds</strong>. A is the subnetwork for genes including those in fatty acid (FA) biosynthesis, and
   B depicts the expression profiles of FA biosynthetic genes identified in the analysis.
   C is another subnetork including genes encoding oleosins and seed storage proteins
   (SSP), and D depicts the expression profiles of genes encoding oleosin and SSP. In
   B and D, the expression values, AGI identifiers of the genes depicted are listed in
   Additional File <a name="d14582e957"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/suppl/S3','',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/suppl/S3">3</a>, and the log<sub>2 </sub>expression values were standardised by subtracting the value at the first S3 stage
   for each gene. Dashed red, blue lines indicate 2-fold up- or down-regulation, respectively.
</p></div>
<div class="addfile "><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/suppl/S1','S1',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/suppl/S1"/><strong>Additional File 1.</strong> <strong>A list of select genes identified in the subnetwork including the majority of fatty
      acid biosynthetic genes</strong>. Genes without informative annotation such as hypothetical proteins were excluded.</p><p>
   	Format: XLS 
   	Size: 308KB <a href="/content/supplementary/1471-2164-12-286-s1.xls">Download file</a></p><p>This file can be viewed with: <a onclick="largepopup('http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=1cd6acf9-ce06-4e1c-8dcf-f33f669dbc3a','Help');return false;" href="http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=1cd6acf9-ce06-4e1c-8dcf-f33f669dbc3a">Microsoft Excel Viewer</a><span class="open-data-icon"><a href="/about/access/"><img title="Open Data" alt="Open Data" src="/bmcimages/article/opendata.gif"/></a></span></p></div>
<div class="addfile "><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/suppl/S2','S2',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/suppl/S2"/><strong>Additional File 2.</strong> <strong>A list of select genes identified in the subnetwork including those encoding oleosins
      and seed storage proteins</strong>. Genes without informative annotation such as hypothetical proteins were excluded.</p><p>
   	Format: XLS 
   	Size: 221KB <a href="/content/supplementary/1471-2164-12-286-s2.xls">Download file</a></p><p>This file can be viewed with: <a onclick="largepopup('http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=1cd6acf9-ce06-4e1c-8dcf-f33f669dbc3a','Help');return false;" href="http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=1cd6acf9-ce06-4e1c-8dcf-f33f669dbc3a">Microsoft Excel Viewer</a><span class="open-data-icon"><a href="/about/access/"><img title="Open Data" alt="Open Data" src="/bmcimages/article/opendata.gif"/></a></span></p></div>
<div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F5','F5',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F5"><img align="top" src="/content/figures/1471-2164-12-286-5.gif" alt="thumbnail" class="thumbnail"/><strong>Figure 5.</strong></a> <strong>Expression profiles of genes including homologues in the triacylglycerol assembly
      pathway</strong>. The dash line at 6.0 is often used as the cutoff for present (expressed; above the
   line) or absent (unexpressed; below the line). All expression data were transformed
   to the log<sub>2 </sub>scale for plotting the profiles. Genes and homologs in the triacylglycerol (TAG) assembly
   pathway were identified based on an early survey of Arabidopsis genes involved in
   acyl lipid metabolism <a name="d14582e1004"/>[<a onclick="LoadInParent('#B59'); return false;" href="#B59">59</a>], and their AGI identifiers listed in Additional File <a name="d14582e1007"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/suppl/S3','',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/suppl/S3">3</a>. Refer to <a name="d14582e1011"/>[<a onclick="LoadInParent('#B64'); return false;" href="#B64">64</a>] for their roles in TAG assembly. The abbreviations of these genes and their encoded
   enzymes (EC numbers) are as follows: <em>GPAT</em>: <em>sn</em>-glycerol-3-phospahte acyltransferase (EC 2.3.1.15); <em>LPAAT</em>: lysophosphatidic acid acyltransferase (EC 2.3.1.15); <em>PAP</em>: PA phosphatase (EC 3.1.3.4), including <em>LIPIN </em>(<em>PAP1</em>) and <em>LPP </em>(<em>PAP2</em>); <em>AAPT</em>: Aminoalcoholphosphotransferases (EC 2.7.8.1 and EC 2.7.8.2); <em>CPT</em>: cytidine diphosphate (CDP)-choline: 1, 2-diacylglycerol cholinephosphotransferase
   (EC 2.7.8.2); LPCAT: lysophosphatidylcholine acyltransferase (EC 2.3.1.23); <em>PLA</em><sub><em>2</em></sub>: Phospholipase A<sub>2 </sub>(EC 3.1.1.4); <em>PDAT</em>: phospholipid:diacylglycerol acyltransferase (EC 2.3.1.158); <em>LCAT</em>: lechitin:cholesterol acyltransferase (EC 2.3.1.43), these three shown here are <em>PDAT </em>homologs; <em>DGAT</em>: Diacylglycerol acyltransferase (E.C. 2.3.1.20).
</p></div>
<div class="addfile "><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/suppl/S3','S3',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/suppl/S3"/><strong>Additional File 3.</strong> <strong>The log<sub>2 </sub>expression values of genes involved in storage reserve accumulation across the eight
      seed development stages</strong>. Genes involved in seed storage reserve accumulation were adopted from early surveys
   <a name="d14582e1082"/><a name="d14582e1084"/>[<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a>,<a onclick="LoadInParent('#B59'); return false;" href="#B59">59</a>], and additional genes implicated in this process were identified through sequence
   analysis using BLAST <a name="d14582e1088"/>[<a onclick="LoadInParent('#B87'); return false;" href="#B87">87</a>].</p><p>
   	Format: XLS 
   	Size: 26KB <a href="/content/supplementary/1471-2164-12-286-s3.xls">Download file</a></p><p>This file can be viewed with: <a onclick="largepopup('http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=1cd6acf9-ce06-4e1c-8dcf-f33f669dbc3a','Help');return false;" href="http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=1cd6acf9-ce06-4e1c-8dcf-f33f669dbc3a">Microsoft Excel Viewer</a><span class="open-data-icon"><a href="/about/access/"><img title="Open Data" alt="Open Data" src="/bmcimages/article/opendata.gif"/></a></span></p></div></div>
<p style="line-height:160%">In summary, our new results suggest that genes acting in a biological process (FA
   biosynthesis) can be indicated by their presence in the same coexpression network
   cluster, but genes involved in the same pathway (TAG assembly) may not necessarily
   exhibit expression coherence. As a result, computational approaches using coexpression
   network to predict gene function, such as in <a name="d14582e1097"/>[<a onclick="LoadInParent('#B40'); return false;" href="#B40">40</a>], will undoubtedly have limitations.
</p>
<h4>Putative regulatory elements underlying seed storage reserve accumulation</h4>
<p style="line-height:160%">To gain insight into possible relationships in gene coexpression and regulation, we
   first identified the expression patterns for several TFs known to regulate the accumulation
   of seed storage reserves (Figure <a name="d14582e1106"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F6">6</a>). <em>AGL15 (AGAMOUS-LIKE 15)</em>, <em>GL2 (GLABRA2)</em>, <em>LEC1</em>, <em>L1L</em>, and <em>WRI1 </em>exhibited similar expression patterns with most genes encoding proteins involved in
   FA biosynthesis (Figure <a name="d14582e1125"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F6">6A</a>) whereas <em>ABI3, EEL</em>, and <em>FUS3 </em>all have similar expression profiles with genes encoding oleosins and SSPs (Figure
   <a name="d14582e1134"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F6">6B</a>). Two repressors of seed maturation genes, <em>ASIL1 </em>(<em>ARABIDOPSIS 6B-INTERACTING PROTEIN 1-LIKE 1</em>) <a name="d14582e1145"/>[<a onclick="LoadInParent('#B69'); return false;" href="#B69">69</a>] and <em>PICKLE </em>(<em>PKL</em>) <a name="d14582e1155"/>[<a onclick="LoadInParent('#B70'); return false;" href="#B70">70</a>], were modestly expressed and exhibited a stable expression pattern throughout seed
   maturation (Figure <a name="d14582e1158"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F6">6C</a>). Surprisingly, <em>LEC2</em>, a TF gene known to regulate oil accumulation in leaves and somatic embryogenesis
   <a name="d14582e1166"/><a name="d14582e1168"/><a name="d14582e1170"/>[<a onclick="LoadInParent('#B10'); return false;" href="#B10">10</a>,<a onclick="LoadInParent('#B14'); return false;" href="#B14">14</a>,<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a>], was barely detectable in these developing seeds. Although this result requires verification
   with other molecular methods, it was previously reported that <em>LEC2 </em>might be only expressed during early embryo morphogenesis <a name="d14582e1177"/>[<a onclick="LoadInParent('#B15'); return false;" href="#B15">15</a>]. Additionally, based on phenotype descriptions of <em>LEC1</em>, <em>LEC2 </em>mutants in the Arabidopsis Information Resource (TAIR) <a name="d14582e1188"/>[<a onclick="LoadInParent('#B71'); return false;" href="#B71">71</a>], the accumulation of storage compounds in the mature <em>lec2 </em>mutant seeds is only slightly defective when compared to <em>lec1 </em>mutant seeds. Therefore, the role of <em>LEC2 </em>as a regulator in the synthesis of seed storage reserves during late stages of zygotic
   embryo development might not be as important as currently thought. Likewise, <em>ABI4 </em>was also essentially unexpressed in these seed samples. The expression similarity
   between genes encoding TFs and their target genes is suggestive of regulatory relationships.
   Both <em>LEC1 </em>and <em>WRI1 </em>were clustered with most FA biosynthetic genes, while <em>ABI5 </em>was clustered with the majority of <em>LATE-EMBRYOGENESIS ABUNDANT </em>(<em>LEA</em>) genes (Figure <a name="d14582e1219"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F3">3</a> Cluster 3). <em>LEC1 </em>and <em>WRI1 </em>are known to regulate many FA biosynthetic genes <a name="d14582e1230"/><a name="d14582e1232"/><a name="d14582e1234"/>[<a onclick="LoadInParent('#B25'); return false;" href="#B25">25</a>-<a onclick="LoadInParent('#B27'); return false;" href="#B27">27</a>], and <em>ABI5 </em>regulates a subset of <em>LEAs </em><a name="d14582e1243"/>[<a onclick="LoadInParent('#B72'); return false;" href="#B72">72</a>].
</p><div class="figs">
<div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/figure/F6','F6',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/figure/F6"><img align="top" src="/content/figures/1471-2164-12-286-6.gif" alt="thumbnail" class="thumbnail"/><strong>Figure 6.</strong></a> <strong>Expression profiles of several well-characterized transcription factor genes</strong>. The dashed line at 6.0 on the y-axis is often used as the cutoff for present (expressed;
   above the line) or absent (unexpressed; below the line). All expression data were
   transformed to the log<sub>2 </sub>scale for plotting the profiles. Refer to Additional File <a name="d14582e1261"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/suppl/S3','',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/suppl/S3">3</a> for their AGI identifiers and full name of each transcription factor gene. The gene
   abbreviations are as follows: <em>AGL5: AGAMOUS-LIKE 5</em>; <em>GL2: GLABRA 2</em>; <em>LEC1: LEAFY COTELYDON 1</em>; <em>L1L: LEAFY COTELYDON 1 LIKE</em>; <em>WRI1: WRINKLED 1</em>; <em>ABI3: ABSCISIC ACID-INSENSITIVE 3</em>; <em>EEL: ENHANCED EM (EMBRYO MORPHOGENESIS) LEVEL</em>; <em>FUS3: FUSCA 3</em>; <em>ABI5: ABSCISIC ACID-INSENSITIVE 5</em>; <em>PKL: PICKLE</em>; <em>ASIL1: ARABIDOPSIS 6B-INTERACTING PROTEIN 1-LIKE 1</em>.
</p></div></div>
<p style="line-height:160%">To computationally identify <em>cis</em>-acting regulatory elements, the upstream promoter sequences for the genes involved
   in storage reserve biosynthesis were extracted from the RSAT server <a name="d14582e1306"/>[<a onclick="LoadInParent('#B73'); return false;" href="#B73">73</a>]. We included some 5'-UTR sequences as certain TF binding sites can be located within
   this region of a gene <a name="d14582e1310"/><a name="d14582e1312"/>[<a onclick="LoadInParent('#B27'); return false;" href="#B27">27</a>,<a onclick="LoadInParent('#B74'); return false;" href="#B74">74</a>]. On average, the G-C content in the promoter sequences of the gene set was found
   to be &lt;35%, which is consistent with the compositional bias of nucleotides towards
   A-T enrichment observed in plant promoter regions <a name="d14582e1316"/><a name="d14582e1318"/>[<a onclick="LoadInParent('#B74'); return false;" href="#B74">74</a>,<a onclick="LoadInParent('#B75'); return false;" href="#B75">75</a>]. Two software tools, TFBS <a name="d14582e1322"/>[<a onclick="LoadInParent('#B76'); return false;" href="#B76">76</a>] and fdrMotif <a name="d14582e1327"/>[<a onclick="LoadInParent('#B77'); return false;" href="#B77">77</a>], were used to search for putative TF-binding sites on both strands. Both tools depend
   on TF- binding profiles (Position Weight Matrix, or PWM) derived from experimentally
   determined binding sites for the prediction, we thus compiled 118 PWMs from the literature
   <a name="d14582e1331"/><a name="d14582e1333"/>[<a onclick="LoadInParent('#B27'); return false;" href="#B27">27</a>,<a onclick="LoadInParent('#B74'); return false;" href="#B74">74</a>] and the JASPAR database <a name="d14582e1337"/>[<a onclick="LoadInParent('#B78'); return false;" href="#B78">78</a>] (Additional File <a name="d14582e1340"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/suppl/S4','',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/suppl/S4">4</a>). In the JASPAR database, we only considered the binding profiles for plant-specific
   TFs because of their potential critical roles in regulating the accumulation of storage
   reserves during seed development, a unique physiological process in higher plants.
</p><div class="figs">
<div class="addfile "><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/suppl/S4','S4',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/suppl/S4"/><strong>Additional File 4.</strong> <strong>The 118 high-quality position weight matrices (PWMs) compiled for the analysis</strong>. In each matrix the definition line starts with a '&gt;' sign and an identifier (ID),
   followed by the description including the transcription factor (TF) name. A row represents
   a position in the motif sequence, and the four columns represent nucleotides A, C,
   G, T respectively. Each matrix was standardized such that its frequency sum for the
   four nucleotides at each position (row) is 1.0000. A blank line was added between
   every two matrices. One matrix (MA17) was excluded from the file via a manual examination
   due to ambiguous TF description for the matrix existing in JASPAR <a name="d14582e1353"/>[<a onclick="LoadInParent('#B78'); return false;" href="#B78">78</a>].</p><p>
   	Format: TXT 
   	Size: 43KB <a href="/content/supplementary/1471-2164-12-286-s4.txt">Download file</a><span class="open-data-icon"><a href="/about/access/"><img title="Open Data" alt="Open Data" src="/bmcimages/article/opendata.gif"/></a></span></p></div></div>
<p style="line-height:160%">We predicted a total of 1770 binding motifs in the promoter regions of genes involved
   in FA biosyntheis, TAG assembly, and genes encoding oleosins and seed storage proteins
   (Additional File <a name="d14582e1361"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/suppl/S5','',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/suppl/S5">5</a>). Each TF can have more than one putative binding site in each gene. As our approach
   of using two predictive tools already filtered out a large number of potentially false
   predictions, the remaining number of putative motifs was relatively small, making
   it difficult to perform statistical analysis of motif enrichment. Therefore, we used
   a simple approach to determine overrepresentation of a TF binding motif in the gene
   set, and defined the number of the motifs for a particular TF as overrepresented if
   it is greater than the sum of the average plus the standard deviation of all predicted
   motifs in a gene set. Sequence logos are used to show the degree of conservation,
   indicated by the height of each nucleotide, at each position (Table <a name="d14582e1364"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/table/T3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/table/T3">3</a>). For the Aw-box motif interacting with WRI1, which possesses a sequence pattern
   of [CnTnG](n)<sub>7</sub>[CG] (where n is any nucleotide), we predicted binding sites in 26 of 44 FA genes
   identified, seven more than reported recently in <a name="d14582e1371"/>[<a onclick="LoadInParent('#B27'); return false;" href="#B27">27</a>]. The highly conserved CCAAT motifs for LEC1 (and L1L) binding are significantly enriched
   in promoters of all FA biosynthetic genes identified. Motifs that interact with TF
   genes known to regulate light-induced genes, such as Zinc-finger proteins DOF1 (MNB1A)
   and DOF2 <a name="d14582e1375"/>[<a onclick="LoadInParent('#B79'); return false;" href="#B79">79</a>], as well as GATA TFs and SORLIP 5 (Sequences Over-Represented in Light-Induced Promoter
   5) <a name="d14582e1380"/>[<a onclick="LoadInParent('#B80'); return false;" href="#B80">80</a>], are overrepresented in the promoters of FA biosynthetic genes. <em>DOF1 </em>(AT1G51700), however, was expressed only at the early globular embryo stage. <em>DOF2 </em>(AT4G38000) exhibited a similar expression profile during seed development as for
   FA biosynthetic genes (data not shown). <em>ARR </em>(Arabidopsis Response Regulator) genes encode ARR7 and ARR15, which have been shown
   to regulate the interaction of cytokinin and auxin in root stem-cell specification
   during early embryogenesis <a name="d14582e1393"/>[<a onclick="LoadInParent('#B81'); return false;" href="#B81">81</a>]. We found no binding matrices for these two regulators, but the binding matrix for
   ARR10 is present in our compiled matrix set and ARR10 motifs are overrepresented in
   the promoter regions of FA biosynthetic genes. We also found no binding matrices for
   AGL 5 or GL2; binding profiles for AGL 3 and AGL 15 were present in our analysis but
   no enriched motifs were identified in the promoter sequences of these FA biosynthetic
   genes.
</p><div class="figs">
<div class="addfile "><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/suppl/S5','S5',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/suppl/S5"/><strong>Additional File 5.</strong> <strong>Predicted <em>cis</em>-acting promoter motifs for genes involved in seed storage reserve accumulation</strong>. The start and end positions of an predicted transcription factor (TF) binding site
   are relative to the start of each promoter sequence, not to the transcription start
   site (TSS) of the gene. The numbers +1 and -1 indicate sense and antisense strand,
   respectively. The score was determined by the transcription factor binding site analysis
   tool TFBS <a name="d14582e1409"/>[<a onclick="LoadInParent('#B76'); return false;" href="#B76">76</a>].</p><p>
   	Format: XLS 
   	Size: 252KB <a href="/content/supplementary/1471-2164-12-286-s5.xls">Download file</a></p><p>This file can be viewed with: <a onclick="largepopup('http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=1cd6acf9-ce06-4e1c-8dcf-f33f669dbc3a','Help');return false;" href="http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=1cd6acf9-ce06-4e1c-8dcf-f33f669dbc3a">Microsoft Excel Viewer</a><span class="open-data-icon"><a href="/about/access/"><img title="Open Data" alt="Open Data" src="/bmcimages/article/opendata.gif"/></a></span></p></div>
<div class="table"><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/table/T3','T3',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/table/T3"><strong>Table 3.</strong></a> Overrepresented motifs identified in promoters of genes involved in fatty acid synthesis,
   and oleosin and seed storage protein accumulation. 
</p></div></div>
<p style="line-height:160%">For the genes and isoforms in the TAG assembly pathway, no overrepresented motifs
   have been found. Our goal was to identify putative promoter elements that can be used
   for experimental studies (Additional File <a name="d14582e1585"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/suppl/S5','',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/suppl/S5">5</a>). Interestingly, promoter motifs for B3 domain TFs, such as ABI3, FUS3 and LEC2,
   were found to be overrepresented in promoters of genes encoding oleosins and SSPs.
   Motifs for bZIP factors (e.g., bZIP67) also appeared to be overrepresented in the
   promoter regions of these genes, but there were no binding matrices for bZIP ABI5
   or EEL.
</p>
<p style="line-height:160%">Our approach of computational promoter analysis was limited by the availability of
   experimentally determined TF-binding sites for deriving binding profiles of additional
   TFs. We compiled a list of 118 binding matrices for this analysis, but if binding
   profiles for other TFs can be generated from a reasonable number of known binding
   sites, we could identify more TFs that possibly regulate the accumulation of seed
   storage reserves. In addition, we only considered upstream sequences of 1000 bp plus
   200 bp 5'-UTR for each gene, because the majority of <em>cis</em>-acting regulatory elements are located in this region <a name="d14582e1594"/>[<a onclick="LoadInParent('#B74'); return false;" href="#B74">74</a>]. Other genomic regions including the 3'-UTR, or even introns, however, can also harbour
   TF binding sites.
</p><a name="sec3"/><h3>Conclusions</h3>
<p style="line-height:160%">Our analyses indicate that genes involved in the accumulation of seed storage reserves,
   along with known <em>TF </em>genes, are expressed in distinct patterns during seed maturation. Promoter motifs
   for CCAAT binding factors LEC1 and L1L, DOF and GATA factors, AP2 WRI1 as well as
   MYB factors are enriched in the promoter regions of genes involved in FA biosynthesis.
   Binding sites for B3-domain factors (ABI3/VP1 TF family) and bZIP factors are overrepresented
   in the promoter regions of genes encoding oleosins and seed storage proteins. When
   binding profiles for additional TFs become available, more putative regulatory elements
   will be detected, which in turn can be validated for functionality.
</p><a name="sec4"/><h3>Methods</h3>
<h4>Retrieval and processing of raw hybridization data</h4>
<p style="line-height:160%">The 24 raw hybridization intensity data files (.CEL files) for Arabidopsis seed development
   were retrieved from The Arabidopsis Information Resource (TAIR) gene expression data
   repository (<a class="longstring" href="http://www.arabidopsis.org/servlets/TairObject?type=hyb_descr_collection&amp;id=1006710873">http:/ / www.arabidopsis.org/ servlets/ TairObject?type=hyb_descr_collectio n&amp;id=1006710873</a> <a href="http://www.webcitation.org/query.php?url=http://www.arabidopsis.org/servlets/TairObject?type=hyb_descr_collection&amp;id=1006710873&amp;refdoi=10.1186/1471-2164-12-286" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a>) <a name="d14582e1620"/>[<a onclick="LoadInParent('#B71'); return false;" href="#B71">71</a>]. Microarray gene expression data analyses were performed using Bioconductor packages
   <a name="d14582e1624"/>[<a onclick="LoadInParent('#B82'); return false;" href="#B82">82</a>] in the open-source statistical R environment <a name="d14582e1628"/>[<a onclick="LoadInParent('#B83'); return false;" href="#B83">83</a>]. The raw data files were imported into Bioconductor using the Simpleaffy package
   <a name="d14582e1632"/>[<a onclick="LoadInParent('#B84'); return false;" href="#B84">84</a>]. The hybridization and RNA sample qualities were assessed using a number of quality
   control metrics (data not shown), and the raw data were background corrected, normalized
   and transformed to the log<sub>2 </sub>values using the GCRMA package <a name="d14582e1640"/>[<a onclick="LoadInParent('#B85'); return false;" href="#B85">85</a>]. This normalization method is developed on another normalization approach robust
   multi-array average (RMA; <a name="d14582e1644"/>[<a onclick="LoadInParent('#B86'); return false;" href="#B86">86</a>]), and uses probe sequence information (G-C content) for estimating hybridization
   affinity. The number of genes expressed in seeds was filtered using a log2 value of
   6.0 as the cutoff for the binary 'present' or 'absent' calls, and any gene with 'present'
   calls in less than three samples (corresponding to one seed development stage) was
   considered as "unexpressed" in these seed samples. After filtering, 12,353 genes expressed
   in at least one of the eight development stages in developing Arabidopsis seeds were
   used for subsequent high-level analyses. Custom Perl scripts were written to find
   the annotation of each gene in the latest CSV file ATH1-121501.na30.annot.csv (November
   15 2009) released by Affymetrix for the ATH1 Genome Array and revised in some cases
   through sequence analysis using BLAST <a name="d14582e1648"/>[<a onclick="LoadInParent('#B87'); return false;" href="#B87">87</a>]. For example, the TF gene <em>WRINKLED1 </em>(AT3G54320) was incorrectly annotated in the Affymetrix file as an aintegumaenta-like
   protein or ovule development protein aintegumenta (Additional File <a name="d14582e1654"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/suppl/S1">1</a>).
</p>
<h4>Principal component analysis and association test of global gene expression with seed
   development
</h4>
<p style="line-height:160%">The normalized, log<sub>2</sub>-transformed gene expression data were used for principal component analysis (PCA)
   using the R prcomp function <a name="d14582e1667"/>[<a onclick="LoadInParent('#B83'); return false;" href="#B83">83</a>]. For this analysis, expression values of the three replicates for each seed development
   stage were not combined in order to assess the reproducibility of biological replication.
   Global testing of the transcriptome with a particular variable (e.g., seed development
   stage) was carried out using the Globaltest package <a name="d14582e1671"/>[<a onclick="LoadInParent('#B51'); return false;" href="#B51">51</a>]. This package tests the overall gene expression in group(s) of genes for significant
   association with a given variable. The test gives one <em>P</em>-value for the whole group instead of one <em>P</em>-value for each gene to avoid the issue of multiple testing corrections.
</p>
<h4>Gene expression correlation analysis and construction of coexpression networks</h4>
<p style="line-height:160%">For the inference of gene coexpression networks in the transcriptome of developing
   Arabidopsis seeds, we used the 12,353 genes expressed at moderate or high levels and
   used the Pearson-based correlation coefficient to measure their expression coherence.
   We first used the median expression data of the genes in the eight samples to compute
   pairwise correlation coefficients in the R statistical environment, resulting in a
   correlation matrix of 12353 Ã 12353. Then we removed self-pairing and duplication,
   and applied a correlation cutoff of 0.90, which retained over 1.7 million gene pairs
   representing 11,698 distinct genes for construction of the coexpression network for
   the Arabidopsis seed genes. This stringent correlation threshold was chosen to eliminate
   potential spurious correlations in a coexpression network. Network properties were
   determined using custom scripts. Coexpression networks are visualized using Cytoscape
   <a name="d14582e1687"/>[<a onclick="LoadInParent('#B88'); return false;" href="#B88">88</a>]. For time-course clustering analysis, the gene expression values were standardized
   to have a mean value of zero and a standard deviation of one for each gene profile.
   This standardization of data ensures that genes with similar temporal profiles are
   close in Euclidean space during clustering, regardless of their absolute expression
   levels. The transformed expressions were then clustered using the fuzzy c-means (FCM)
   clustering algorithm in the Bioconductor Mfuzz package <a name="d14582e1691"/>[<a onclick="LoadInParent('#B89'); return false;" href="#B89">89</a>]. We determined six clusters can well separate the expression patterns inherent in
   the dataset, and another FCM parameter m = 1.75, which allows for investigation of
   the clustering robustness. FCM assigns a membership value in the range of 0-1 for
   each gene as an indicator of how representative a gene profile is for a specific cluster,
   and profiles with different membership values were differently coloured.
</p>
<h4>Computational analyses of transcription factor binding sites</h4>
<p style="line-height:160%">The genomic sequences 1000 bp upstream plus 200 bp 5' untranslated regions (UTR) for
   the genes involved in storage reserve biosynthesis were retrieved from the RSAT server
   <a name="d14582e1701"/>[<a onclick="LoadInParent('#B73'); return false;" href="#B73">73</a>]. If the intergenic region with the upstream neighbouring gene is &lt;1000 bp long, we
   only retrieved upstream sequence available in order to prevent using the 3'-end sequence
   of the adjacent gene in the upstream. Putative TF binding sites on both strands were
   identified with two software tools, TFBS <a name="d14582e1705"/>[<a onclick="LoadInParent('#B76'); return false;" href="#B76">76</a>] and fdrMotif <a name="d14582e1709"/>[<a onclick="LoadInParent('#B77'); return false;" href="#B77">77</a>]. Briefly, the 118 TF binding profiles (position-specific weight matrix, or PWM) were
   compiled from the literature <a name="d14582e1713"/><a name="d14582e1715"/>[<a onclick="LoadInParent('#B27'); return false;" href="#B27">27</a>,<a onclick="LoadInParent('#B74'); return false;" href="#B74">74</a>] and the JASPAR database <a name="d14582e1719"/>[<a onclick="LoadInParent('#B78'); return false;" href="#B78">78</a>], and converted into a format suitable for each software tool (Additional File <a name="d14582e1723"/><a onclick="popup('http://www.biomedcentral.com/1471-2164/12/286/suppl/S4','',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/12/286/suppl/S4">4</a>). In the TFBS search, an 80% similarity cutoff was adopted. In fdrMotif search, for
   each input sequence 10 background sequences were generated from a 4th-order Markov
   model and an upper boundary of false discovery rate (FDR) of 0.15 as suggested by
   fdrMotif was adopted to control FDR. Only putative binding sites predicted by both
   tools were retained for subsequent analysis. To ascertain the predictive performance,
   detected motifs were compared with curated motifs in AtcisDB and AGRIS databases <a name="d14582e1727"/><a name="d14582e1729"/>[<a onclick="LoadInParent('#B90'); return false;" href="#B90">90</a>,<a onclick="LoadInParent('#B91'); return false;" href="#B91">91</a>]. Sequence logos for the predicted motifs for a TF binding profile were created with
   WebLogo <a name="d14582e1733"/>[<a onclick="LoadInParent('#B92'); return false;" href="#B92">92</a>].
</p><a name="sec5"/><h3>Authors' contributions</h3>
<p style="line-height:160%">FYP carried out the data analysis and prepared the first draft of the manuscript.
   RJW supervised the analysis and revised the manuscript. Both authors read and approved
   the final version of the manuscript.
</p><a name="ack"/><h3>Acknowledgements</h3>
<p style="line-height:160%">The authors are grateful for the financial support provided by Genome Canada, Genome
   Alberta, Alberta Advanced Education and Technology, and the Canada Research Chairs
   Program. We also thank three anonymous reviewers for their helpful comments and suggestions.
</p><a name="refs"/><h3>References</h3>
<div id="article-references">
   <ol id="references"><li id="B1">
         <p><a name="B1"/> Baud S,  Lepiniec L: <strong> Regulation of de novo fatty acid synthesis in maturing oilseeds of Arabidopsis. </strong></p><p><em>Plant Physiol Biochem</em> 2009,  <strong>47</strong>(6)<strong>:</strong>448-455.  <a target="_blank" href="/pubmed/19136270">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19136270">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B1" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B1','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B2">
         <p><a name="B2"/> Verdier J,  Thompson RD: <strong> Transcriptional regulation of storage protein synthesis during dicotyledon seed filling. </strong></p><p><em>Plant Cell Physiol</em> 2008,  <strong>49</strong>(9)<strong>:</strong>1263-1271.  <a target="_blank" href="/pubmed/18701524">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18701524">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B2" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B2','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B3">
         <p><a name="B3"/> Weselake RJ,  Taylor DC,  Rahman MH,  Shah S,  Laroche A,  McVetty PB,  Harwood JL: <strong> Increasing the flow of carbon into seed oil. </strong></p><p><em>Biotechnol Adv</em> 2009,  <strong>27</strong>(6)<strong>:</strong>866-878.  <a target="_blank" href="/pubmed/19625012">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19625012">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B3" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B3','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B4">
         <p><a name="B4"/> North H,  Baud S,  Debeaujon I,  Dubos C,  Dubreucq B,  Grappin P,  Jullien M,  Lepiniec L,  Marion-Poll A,  Martine M,  LoÃ¯c Rajjou L,  Routaboul JM,  Caboche M: <strong> Arabidopsis seed secrets unravelled after a decade of genetic and omics-driven research. </strong></p><p><em>Plant J</em> 2010,  <strong>61</strong>(6)<strong>:</strong>971-981.  <a target="_blank" href="/pubmed/20409271">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=20409271">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B4" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B4','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B5">
         <p><a name="B5"/>Arabidopsis Genome Initiative: <strong> Analysis of the genome sequence of the flowering plant Arabidopsis thaliana. </strong></p><p><em>Nature</em> 2000,  <strong>408</strong><strong>:</strong>796-815.  <a target="_blank" href="/pubmed/11130711">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11130711">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B5" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B5','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B6">
         <p><a name="B6"/> West M,  Yee KM,  Danao J,  Zimmerman JL,  Fischer RL,  Goldberg RB,  Harada JJ: <strong> LEAFY COTYLEDON1 Is an Essential Regulator of Late Embryogenesis and Cotyledon Identity
               in Arabidopsis. </strong></p><p><em>Plant Cell</em> 1994,  <strong>6</strong>(12)<strong>:</strong>1731-1745.  <a target="_blank" href="/pubmed/12244233">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12244233">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=12244233">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B6" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B6','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B7">
         <p><a name="B7"/> Parcy F,  Valon C,  Kohara A,  Misera S,  Giraudat J: <strong> The ABSCISIC ACID-INSENSITIVE3, FUSCA3, and LEAFY COTYLEDON1 loci act in concert to
               control multiple aspects of Arabidopsis seed development. </strong></p><p><em>Plant Cell</em> 1997,  <strong>9</strong>(8)<strong>:</strong>1265-1277.  <a target="_blank" href="/pubmed/9286105">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9286105">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=9286105">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B7" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B7','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B8">
         <p><a name="B8"/> Lotan T,  Ohto M,  Yee KM,  West MA,  Lo R,  Kwong RW,  Yamagishi K,  Fischer RL,  Goldberg RB,  Harada JJ: <strong> Arabidopsis LEAFY COTYLEDON1 is sufficient to induce embryo development in vegetative
               cells. </strong></p><p><em>Cell</em> 1998,  <strong>93</strong>(7)<strong>:</strong>1195-1205.  <a target="_blank" href="/pubmed/9657152">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9657152">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B8" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B8','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B9">
         <p><a name="B9"/> Luerssen H,  Kirik V,  Herrmann P,  Misera S: <strong> FUSCA3 encodes a protein with a conserved VP1/AB13-like B3 domain which is of functional
               importance for the regulation of seed maturation in Arabidopsis thaliana. </strong></p><p><em>Plant J</em> 1998,  <strong>15</strong>(6)<strong>:</strong>755-764.  <a target="_blank" href="/pubmed/9807814">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9807814">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B9" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B9','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B10">
         <p><a name="B10"/> Stone SL,  Kwong LW,  Yee KM,  Pelletier J,  Lepiniec L,  Fischer RL,  Goldberg RB,  Harada JJ: <strong> LEAFY COTYLEDON2 encodes a B3 domain transcription factor that induces embryo development. </strong></p><p><em>Proc Natl Acad Sci USA</em> 2001,  <strong>98</strong>(20)<strong>:</strong>11806-11811.  <a target="_blank" href="/pubmed/11573014">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11573014">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=11573014">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B10" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B10','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B11">
         <p><a name="B11"/> Kwong RW,  Bui AQ,  Lee H,  Kwong LW,  Fischer RL,  Goldberg RB,  Harada JJ: <strong> LEAFY COTYLEDON1-LIKE defines a class of regulators essential for embryo development. </strong></p><p><em>Plant Cell</em> 2003,  <strong>15</strong>(1)<strong>:</strong>5-18.  <a target="_blank" href="/pubmed/12509518">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12509518">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=12509518">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B11" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B11','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B12">
         <p><a name="B12"/> Lee H,  Fischer RL,  Goldberg RB,  Harada JJ: <strong> Arabidopsis LEAFY COTYLEDON1 represents a functionally specialized subunit of the
               CCAAT binding transcription factor. </strong></p><p><em>Proc Natl Acad Sci USA</em> 2003,  <strong>100</strong>(4)<strong>:</strong>2152-2156.  <a target="_blank" href="/pubmed/12578989">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12578989">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=12578989">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B12" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B12','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B13">
         <p><a name="B13"/> Monke G,  Altschmied L,  Tewes A,  Reidt W,  Mock HP,  Baumlein H,  Conrad U: <strong> Seed-specific transcription factors ABI3 and FUS3: molecular interaction with DNA. </strong></p><p><em>Planta</em> 2004,  <strong>219</strong>(1)<strong>:</strong>158-166.  <a target="_blank" href="/pubmed/14767767">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=14767767">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B13" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B13','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B14">
         <p><a name="B14"/> Santos-Mendoza M,  Dubreucq B,  Miquel M,  Caboche M,  Lepiniec L: <strong> LEAFY COTYLEDON 2 activation is sufficient to trigger the accumulation of oil and
               seed specific mRNAs in Arabidopsis leaves. </strong></p><p><em>FEBS Lett</em> 2005,  <strong>579</strong>(21)<strong>:</strong>4666-4670.  <a target="_blank" href="/pubmed/16107256">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16107256">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B14" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B14','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B15">
         <p><a name="B15"/> Braybrook SA,  Stone SL,  Park S,  Bui AQ,  Le BH,  Fischer RL,  Goldberg RB,  Harada JJ: <strong> Genes directly regulated by LEAFY COTYLEDON2 provide insight into the control of embryo
               maturation and somatic embryogenesis. </strong></p><p><em>Proc Natl Acad Sci USA</em> 2006,  <strong>103</strong>(9)<strong>:</strong>3468-3473.  <a target="_blank" href="/pubmed/16492731">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16492731">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=16492731">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B15" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B15','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B16">
         <p><a name="B16"/> Braybrook SA,  Harada JJ: <strong> LECs go crazy in embryo development. </strong></p><p><em>Trends Plant Sci</em> 2008,  <strong>13</strong>(12)<strong>:</strong>624-630.  <a target="_blank" href="/pubmed/19010711">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19010711">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B16" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B16','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B17">
         <p><a name="B17"/> Mu J,  Tan H,  Zheng Q,  Fu F,  Liang Y,  Zhang J,  Yang X,  Wang T,  Chong K,  Wang XJ,  Zuo J: <strong> LEAFY COTYLEDON1 is a key regulator of fatty acid biosynthesis in Arabidopsis. </strong></p><p><em>Plant Physiol</em> 2008,  <strong>148</strong>(2)<strong>:</strong>1042-1054.  <a target="_blank" href="/pubmed/18689444">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18689444">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=18689444">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B17" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B17','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B18">
         <p><a name="B18"/> Baud S,  Mendoza MS,  To A,  Harscoet E,  Lepiniec L,  Dubreucq B: <strong> WRINKLED1 specifies the regulatory action of LEAFY COTYLEDON2 towards fatty acid metabolism
               during seed maturation in Arabidopsis. </strong></p><p><em>Plant J</em> 2007,  <strong>50</strong>(5)<strong>:</strong>825-838.  <a target="_blank" href="/pubmed/17419836">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17419836">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B18" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B18','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B19">
         <p><a name="B19"/> Brocard-Gifford IM,  Lynch TJ,  Finkelstein RR: <strong> Regulatory networks in seeds integrating developmental, abscisic acid, sugar, and
               light signaling. </strong></p><p><em>Plant Physiol</em> 2003,  <strong>131</strong>(1)<strong>:</strong>78-92.  <a target="_blank" href="/pubmed/12529517">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12529517">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=12529517">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B19" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B19','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B20">
         <p><a name="B20"/> To A,  Valon C,  Savino G,  Guilleminot J,  Devic M,  Giraudat J,  Parcy F: <strong> A network of local and redundant gene regulation governs Arabidopsis seed maturation. </strong></p><p><em>Plant Cell</em> 2006,  <strong>18</strong>(7)<strong>:</strong>1642-1651.  <a target="_blank" href="/pubmed/16731585">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16731585">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=16731585">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B20" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B20','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B21">
         <p><a name="B21"/> Gutierrez L,  Van Wuytswinkel O,  Castelain M,  Bellini C: <strong> Combined networks regulating seed maturation. </strong></p><p><em>Trends Plant Sci</em> 2007,  <strong>12</strong>(7)<strong>:</strong>294-300.  <a target="_blank" href="/pubmed/17588801">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17588801">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B21" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B21','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B22">
         <p><a name="B22"/> Santos-Mendoza M,  Dubreucq B,  Baud S,  Parcy F,  Caboche M,  Lepiniec L: <strong> Deciphering gene regulatory networks that control seed development and maturation
               in Arabidopsis. </strong></p><p><em>Plant J</em> 2008,  <strong>54</strong>(4)<strong>:</strong>608-620.  <a target="_blank" href="/pubmed/18476867">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18476867">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B22" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B22','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B23">
         <p><a name="B23"/> Angelovici R,  Fait A,  Zhu X,  Szymanski J,  Feldmesser E,  Fernie AR,  Galili G: <strong> Deciphering transcriptional and metabolic networks associated with lysine metabolism
               during Arabidopsis seed development. </strong></p><p><em>Plant Physiol</em> 2009,  <strong>151</strong>(4)<strong>:</strong>2058-2072.  <a target="_blank" href="/pubmed/19783646">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19783646">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19783646">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B23" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B23','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B24">
         <p><a name="B24"/> Reidt W,  Wohlfarth T,  Ellerstrom M,  Czihal A,  Tewes A,  Ezcurra I,  Rask L,  Baumlein H: <strong> Gene regulation during late embryogenesis: the RY motif of maturation-specific gene
               promoters is a direct target of the FUS3 gene product. </strong></p><p><em>Plant J</em> 2000,  <strong>21</strong>(5)<strong>:</strong>401-408.  <a target="_blank" href="/pubmed/10758492">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10758492">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B24" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B24','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B25">
         <p><a name="B25"/> Yamamoto A,  Kagaya Y,  Toyoshima R,  Kagaya M,  Takeda S,  Hattori T: <strong> Arabidopsis NF-YB subunits LEC1 and LEC1-LIKE activate transcription by interacting
               with seed-specific ABRE-binding factors. </strong></p><p><em>Plant J</em> 2009,  <strong>58</strong>(5)<strong>:</strong>843-856.  <a target="_blank" href="/pubmed/19207209">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19207209">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B25" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B25','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B26">
         <p><a name="B26"/> Cernac A,  Benning C: <strong> WRINKLED1 encodes an AP2/EREB domain protein involved in the control of storage compound
               biosynthesis in Arabidopsis. </strong></p><p><em>Plant J</em> 2004,  <strong>40</strong>(4)<strong>:</strong>575-585.  <a target="_blank" href="/pubmed/15500472">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15500472">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B26" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B26','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B27">
         <p><a name="B27"/> Maeo K,  Tokuda T,  Ayame A,  Mitsui N,  Kawai T,  Tsukagoshi H,  Ishiguro S,  Nakamura K: <strong> An AP2-type transcription factor, WRINKLED1, of Arabidopsis thaliana binds to the
               AW-box sequence conserved among proximal upstream regions of genes involved in fatty
               acid synthesis. </strong></p><p><em>Plant J</em> 2009,  <strong>60</strong>(3)<strong>:</strong>476-487.  <a target="_blank" href="/pubmed/19594710">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19594710">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B27" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B27','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B28">
         <p><a name="B28"/> Bensmihen S,  Rippa S,  Lambert G,  Jublot D,  Pautot V,  Granier F,  Giraudat J,  Parcy F: <strong> The homologous ABI5 and EEL transcription factors function antagonistically to fine-tune
               gene expression during late embryogenesis. </strong></p><p><em>Plant Cell</em> 2002,  <strong>14</strong>(6)<strong>:</strong>1391-1403.  <a target="_blank" href="/pubmed/12084834">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12084834">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=12084834">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B28" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B28','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B29">
         <p><a name="B29"/> Alonso R,  Onate-Sanchez L,  Weltmeier F,  Ehlert A,  Diaz I,  Dietrich K,  Vicente-Carbajosa J,  Droge-Laser W: <strong> A pivotal role of the basic leucine zipper transcription factor bZIP53 in the regulation
               of Arabidopsis seed maturation gene expression based on heterodimerization and protein
               complex formation. </strong></p><p><em>Plant Cell</em> 2009,  <strong>21</strong>(6)<strong>:</strong>1747-1761.  <a target="_blank" href="/pubmed/19531597">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19531597">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19531597">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B29" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B29','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B30">
         <p><a name="B30"/> Kroj T,  Savino G,  Valon C,  Giraudat J,  Parcy F: <strong> Regulation of storage protein gene expression in Arabidopsis. </strong></p><p><em>Development</em> 2003,  <strong>130</strong>(24)<strong>:</strong>6065-6073.  <a target="_blank" href="/pubmed/14597573">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=14597573">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B30" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B30','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B31">
         <p><a name="B31"/> Kagaya Y,  Okuda R,  Ban A,  Toyoshima R,  Tsutsumida K,  Usui H,  Yamamoto A,  Hattori T: <strong> Indirect ABA-dependent regulation of seed storage protein genes by FUSCA3 transcription
               factor in Arabidopsis. </strong></p><p><em>Plant Cell Physiol</em> 2005,  <strong>46</strong>(2)<strong>:</strong>300-311.  <a target="_blank" href="/pubmed/15695463">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15695463">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B31" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B31','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B32">
         <p><a name="B32"/> Kagaya Y,  Toyoshima R,  Okuda R,  Usui H,  Yamamoto A,  Hattori T: <strong> LEAFY COTYLEDON1 controls seed storage protein genes through its regulation of FUSCA3
               and ABSCISIC ACID INSENSITIVE3. </strong></p><p><em>Plant Cell Physiol</em> 2005,  <strong>46</strong>(3)<strong>:</strong>399-406.  <a target="_blank" href="/pubmed/15695450">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15695450">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B32" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B32','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B33">
         <p><a name="B33"/> Girke T,  Todd J,  Ruuska S,  White J,  Benning C,  Ohlrogge J: <strong> Microarray analysis of developing Arabidopsis seeds. </strong></p><p><em>Plant Physiol</em> 2000,  <strong>124</strong>(4)<strong>:</strong>1570-1581.  <a target="_blank" href="/pubmed/11115875">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11115875">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=11115875">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B33" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B33','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B34">
         <p><a name="B34"/> Ruuska SA,  Girke T,  Benning C,  Ohlrogge JB: <strong> Contrapuntal networks of gene expression during Arabidopsis seed filling. </strong></p><p><em>Plant Cell</em> 2002,  <strong>14</strong>(6)<strong>:</strong>1191-1206.  <a target="_blank" href="/pubmed/12084821">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12084821">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=12084821">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B34" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B34','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B35">
         <p><a name="B35"/> Schmid M,  Davison TS,  Henz SR,  Pape UJ,  Demar M,  Vingron M,  Scholkopf B,  Weigel D,  Lohmann JU: <strong> A gene expression map of Arabidopsis thaliana development. </strong></p><p><em>Nat Genet</em> 2005,  <strong>37</strong>(5)<strong>:</strong>501-506.  <a target="_blank" href="/pubmed/15806101">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15806101">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B35" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B35','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B36">
         <p><a name="B36"/> Wang H,  Guo J,  Lambert KN,  Lin Y: <strong> Developmental control of Arabidopsis seed oil biosynthesis. </strong></p><p><em>Planta</em> 2007,  <strong>226</strong>(3)<strong>:</strong>773-783.  <a target="_blank" href="/pubmed/17522888">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17522888">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B36" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B36','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B37">
         <p><a name="B37"/> Volodarsky D,  Leviatan N,  Otcheretianski A,  Fluhr R: <strong> HORMONOMETER: a tool for discerning transcript signatures of hormone action in the
               Arabidopsis transcriptome. </strong></p><p><em>Plant Physiol</em> 2009,  <strong>150</strong>(4)<strong>:</strong>1796-1805.  <a target="_blank" href="/pubmed/19535475">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19535475">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19535475">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B37" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B37','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B38">
         <p><a name="B38"/> Vandepoele K,  Quimbaya M,  Casneuf T,  De Veylder L,  Van de Peer Y: <strong> Unraveling transcriptional control in Arabidopsis using cis-regulatory elements and
               coexpression networks. </strong></p><p><em>Plant Physiol</em> 2009,  <strong>150</strong>(2)<strong>:</strong>535-546.  <a target="_blank" href="/pubmed/19357200">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19357200">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19357200">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B38" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B38','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B39">
         <p><a name="B39"/> Obayashi T,  Hayashi S,  Shibaoka M,  Saeki M,  Ohta H,  Kinoshita K: <strong> COXPRESdb: a database of coexpressed gene networks in mammals. </strong></p><p><em>Nucleic Acids Res</em> 2008,  <strong/>(36 Database)<strong>:</strong>D77-82.  <a target="_blank" href="/pubmed/    17932064">PubMed Abstract</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B39" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B39','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B40">
         <p><a name="B40"/> Nayak RR,  Kearns M,  Spielman RS,  Cheung VG: <strong> Coexpression network based on natural variation in human gene expression reveals gene
               interactions and functions. </strong></p><p><em>Genome Res</em> 2009,  <strong>19</strong>(11)<strong>:</strong>1953-1962.  <a target="_blank" href="/pubmed/19797678">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19797678">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19797678">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B40" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B40','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B41">
         <p><a name="B41"/> Mutwil M,  Usadel B,  Schutte M,  Loraine A,  Ebenhoh O,  Persson S: <strong> Assembly of an interactive correlation network for the Arabidopsis genome using a
               novel heuristic clustering algorithm. </strong></p><p><em>Plant Physiol</em> 2010,  <strong>152</strong>(1)<strong>:</strong>29-43.  <a target="_blank" href="/pubmed/19889879">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19889879">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19889879">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B41" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B41','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B42">
         <p><a name="B42"/> Hruz T,  Laule O,  Szabo G,  Wessendorp F,  Bleuler S,  Oertle L,  Widmayer P,  Gruissem W,  Zimmermann P: <strong> Genevestigator v3: a reference expression database for the meta-analysis of transcriptomes. </strong></p><p><em>Adv Bioinformatics 2008</em> 2008,  <strong>147</strong>(3)<strong>:</strong>1004-1016.  <a target="_blank" href="/pubmed/19956698">PubMed Abstract</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B42" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B42','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B43">
         <p><a name="B43"/> Srinivasasainagendra V,  Page GP,  Mehta T,  Coulibaly I,  Loraine AE: <strong> CressExpress: a tool for large-scale mining of expression data from Arabidopsis. </strong></p><p><em>Plant Physiol</em> 2008,  <strong>147</strong>(3)<strong>:</strong>1004-1016.  <a target="_blank" href="/pubmed/18467456">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18467456">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=18467456">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B43" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B43','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B44">
         <p><a name="B44"/> Wolfe CJ,  Kohane IS,  Butte AJ: <strong> Systematic survey reveals general applicability of "guilt-by-association" within gene
               coexpression networks. </strong></p><p><em>BMC Bioinformatics</em> 2005,  <strong>6</strong><strong>:</strong>227.  <a target="_blank" href="/pubmed/16162296">PubMed Abstract</a> | <a target="_blank" href="http://dx.doi.org/10.1186/1471-2105-6-227">BioMed Central Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=16162296">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B44" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B44','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B45">
         <p><a name="B45"/> Marco A,  Konikoff C,  Karr TL,  Kumar S: <strong> Relationship between gene co-expression and sharing of transcription factor binding
               sites in Drosophila melanogaster. </strong></p><p><em>Bioinformatics</em> 2009,  <strong>25</strong>(19)<strong>:</strong>2473-2477.  <a target="_blank" href="/pubmed/19633094">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19633094">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19633094">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B45" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B45','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B46">
         <p><a name="B46"/> Xulvi-Brunet R,  Li H: <strong> Co-expression networks: graph properties and topological comparisons. </strong></p><p><em>Bioinformatics</em> 2010,  <strong>26</strong>(2)<strong>:</strong>205-214.  <a target="_blank" href="/pubmed/19910304">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19910304">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19910304">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B46" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B46','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B47">
         <p><a name="B47"/> Kreiman G: <strong> Identification of sparsely distributed clusters of cis-regulatory elements in sets
               of co-expressed genes. </strong></p><p><em>Nucleic Acids Res</em> 2004,  <strong>32</strong>(9)<strong>:</strong>2889-2900.  <a target="_blank" href="/pubmed/15155858">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15155858">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=15155858">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B47" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B47','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B48">
         <p><a name="B48"/> Haberer G,  Mader MT,  Kosarev P,  Spannagl M,  Yang L,  Mayer KF: <strong> Large-scale cis-element detection by analysis of correlated expression and sequence
               conservation between Arabidopsis and Brassica oleracea. </strong></p><p><em>Plant Physiol</em> 2006,  <strong>142</strong>(4)<strong>:</strong>1589-1602.  <a target="_blank" href="/pubmed/17028152">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17028152">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17028152">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B48" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B48','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B49">
         <p><a name="B49"/> Obayashi T,  Kinoshita K,  Nakai K,  Shibaoka M,  Hayashi S,  Saeki M,  Shibata D,  Saito K,  Ohta H: <strong> ATTED-II: a database of co-expressed genes and cis elements for identifying co-regulated
               gene groups in Arabidopsis. </strong></p><p><em>Nucleic Acids Res</em> 2007,  <strong/>(35 Database)<strong>:</strong>D863-9.  <a target="_blank" href="/pubmed/17130150">PubMed Abstract</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B49" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B49','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B50">
         <p><a name="B50"/> Lenka SK,  Lohia B,  Kumar A,  Chinnusamy V,  Bansal KC: <strong> Genome-wide targeted prediction of ABA responsive genes in rice based on overrepresented
               cis-motif in co-expressed genes. </strong></p><p><em>Plant Mol Biol</em> 2009,  <strong>69</strong>(3)<strong>:</strong>261-271.  <a target="_blank" href="/pubmed/18998058">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18998058">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B50" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B50','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B51">
         <p><a name="B51"/> Goeman JJ,  van de Geer SA,  de Kort F,  van Houwelingen HC: <strong> A global test for groups of genes: testing association with a clinical outcome. </strong></p><p><em>Bioinformatics</em> 2004,  <strong>20</strong>(1)<strong>:</strong>93-99.  <a target="_blank" href="/pubmed/14693814">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=14693814">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B51" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B51','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B52">
         <p><a name="B52"/> Yang YH,  Xiao Y,  Segal MR: <strong> Identifying differentially expressed genes from microarray experiments via statistic
               synthesis. </strong></p><p><em>Bioinformatics</em> 2005,  <strong>21</strong>(7)<strong>:</strong>1084-1093.  <a target="_blank" href="/pubmed/15513985">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15513985">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B52" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B52','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B53">
         <p><a name="B53"/> Chen X,  Truksa M,  Shah S,  Weselake RJ: <strong> A survey of quantitative real-time PCR internal reference genes for expression studies
               in Brassica napus. </strong></p><p><em>Anal Biochem</em> 2010,  <strong>405</strong>(1)<strong>:</strong>138-140.  <a target="_blank" href="/pubmed/20522329">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=20522329">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B53" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B53','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B54">
         <p><a name="B54"/> Watts DJ,  Strogatz SH: <strong> Collective dynamics of 'small-world' networks. </strong></p><p><em>Nature</em> 1998,  <strong>393</strong>(6684)<strong>:</strong>440-442.  <a target="_blank" href="/pubmed/9623998">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9623998">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B54" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B54','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B55">
         <p><a name="B55"/> Zhang B,  Horvath S: <strong> A general framework for weighted gene co-expression network analysis. </strong></p><p><em>Stat Appl Genet Mol Biol</em> 2005,  <strong>4</strong><strong>:</strong>17.  <a target="_blank" href="/pubmed/16646834">PubMed Abstract</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B55" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B55','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B56">
         <p><a name="B56"/> Albert R: <strong> Scale-free networks in cell biology. </strong></p><p><em>J Cell Sci</em> 2005,  <strong>118</strong>(Pt 21)<strong>:</strong>4947-4957.  <a target="_blank" href="/pubmed/16254242">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16254242">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B56" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B56','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B57">
         <p><a name="B57"/> Borgatti SP,  Mehra A,  Brass DJ,  Labianca G: <strong> Network analysis in the social sciences. </strong></p><p><em>Science</em> 2009,  <strong>323</strong>(5916)<strong>:</strong>892-895.  <a target="_blank" href="/pubmed/19213908">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19213908">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B57" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B57','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B58">
         <p><a name="B58"/> Aoki K,  Ogata Y,  Shibata D: <strong> Approaches for extracting practical information from gene co-expression networks in
               plant biology. </strong></p><p><em>Plant Cell Physiol</em> 2007,  <strong>48</strong>(3)<strong>:</strong>381-390.  <a target="_blank" href="/pubmed/17251202">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17251202">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B58" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B58','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B59">
         <p><a name="B59"/> Beisson F,  Koo AJ,  Ruuska S,  Schwender J,  Pollard M,  Thelen JJ,  Paddock T,  Salas JJ,  Savage L,  Milcamps A,  Mhaske VB,  Cho Y,  Ohlrogge JB: <strong> Arabidopsis genes involved in acyl lipid metabolism. A 2003 census of the candidates,
               a study of the distribution of expressed sequence tags in organs, and a web-based
               database. </strong></p><p><em>Plant Physiol</em> 2003,  <strong>132</strong>(2)<strong>:</strong>681-697.  <a target="_blank" href="/pubmed/12805597">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12805597">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=12805597">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B59" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B59','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B60">
         <p><a name="B60"/> Eccleston VS,  Ohlrogge JB: <strong> Expression of lauroyl-acyl carrier protein thioesterase in brassica napus seeds induces
               pathways for both fatty acid oxidation and biosynthesis and implies a set point for
               triacylglycerol accumulation. </strong></p><p><em>Plant Cell</em> 1998,  <strong>10</strong>(4)<strong>:</strong>613-622.  <a target="_blank" href="/pubmed/9548986">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9548986">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=9548986">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B60" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B60','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B61">
         <p><a name="B61"/> Ramli US,  Baker DS,  Quant PA,  Harwood JL: <strong> Control analysis of lipid biosynthesis in tissue cultures from oil crops shows that
               flux control is shared between fatty acid synthesis and lipid assembly. </strong></p><p><em>Biochem J</em> 2002,  <strong>364</strong>(Pt 2)<strong>:</strong>393-401.  <a target="_blank" href="/pubmed/12023882">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12023882">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=12023882">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B61" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B61','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B62">
         <p><a name="B62"/> Bao X,  Ohlrogge J: <strong> Supply of fatty acid is one limiting factor in the accumulation of triacylglycerol
               in developing embryos. </strong></p><p><em>Plant Physiol</em> 1999,  <strong>120</strong>(4)<strong>:</strong>1057-1062.  <a target="_blank" href="/pubmed/10444089">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10444089">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=10444089">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B62" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B62','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B63">
         <p><a name="B63"/> Weselake RJ,  Shah S,  Tang M,  Quant PA,  Snyder CL,  Furukawa-Stoffer TL,  Zhu W,  Taylor DC,  Zou J,  Kumar A,  Hall L,  Laroche A,  Rakow G,  Raney P,  Moloney MM,  Harwood JL: <strong> Metabolic control analysis is helpful for informed genetic manipulation of oilseed
               rape (Brassica napus) to increase seed oil content. </strong></p><p><em>J Exp Bot</em> 2008,  <strong>59</strong>(13)<strong>:</strong>3543-3549.  <a target="_blank" href="/pubmed/18703491">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18703491">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=18703491">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B63" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B63','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B64">
         <p><a name="B64"/> Weselake RJ: <strong> Storage lipids. </strong>In <em>Plant Lipids - Biology, Utilization and Manipulation</em>.  Edited by Murphy DJ.  Oxford, UK: Blackwell Publishing;  2005:162-221.  <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B64" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B64','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B65">
         <p><a name="B65"/> Ohlrogge J,  Browse J: <strong> Lipid biosynthesis. </strong></p><p><em>Plant Cell</em> 1995,  <strong>7</strong>(7)<strong>:</strong>957-970.  <a target="_blank" href="/pubmed/7640528">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=7640528">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=7640528">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B65" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B65','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B66">
         <p><a name="B66"/> Saha S,  Enugutti B,  Rajakumari S,  Rajasekharan R: <strong> Cytosolic triacylglycerol biosynthetic pathway in oilseeds. Molecular cloning and
               expression of peanut cytosolic diacylglycerol acyltransferase. </strong></p><p><em>Plant Physiol</em> 2006,  <strong>141</strong>(4)<strong>:</strong>1533-1543.  <a target="_blank" href="/pubmed/16798944">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16798944">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=16798944">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B66" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B66','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B67">
         <p><a name="B67"/> Tzen J,  Cao Y,  Laurent P,  Ratnayake C,  Huang A: <strong> Lipids, Proteins, and Structure of Seed Oil Bodies from Diverse Species. </strong></p><p><em>Plant Physiol</em> 1993,  <strong>101</strong>(1)<strong>:</strong>267-276.  <a target="_blank" href="/pubmed/12231682">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12231682">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=12231682">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B67" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B67','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B68">
         <p><a name="B68"/> Weselake RJ,  Pomeroy MK,  Furukawa TL,  Golden JL,  Little DB,  Laroche A: <strong> Developmental profile of diacylglycerol acyltransferase in maturing seeds of oilseed
               rape and safflower and microspore-derived cultures of oilseed rape. </strong></p><p><em>Plant Physiol</em> 1993,  <strong>102</strong>(2)<strong>:</strong>565-571.  <a target="_blank" href="/pubmed/12231845">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12231845">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=12231845">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B68" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B68','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B69">
         <p><a name="B69"/> Gao MJ,  Lydiate DJ,  Li X,  Lui H,  Gjetvaj B,  Hegedus DD,  Rozwadowski K: <strong> Repression of seed maturation genes by a trihelix transcriptional repressor in Arabidopsis
               seedlings. </strong></p><p><em>Plant Cell</em> 2009,  <strong>21</strong>(1)<strong>:</strong>54-71.  <a target="_blank" href="/pubmed/19155348">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19155348">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19155348">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B69" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B69','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B70">
         <p><a name="B70"/> Ogas J,  Kaufmann S,  Henderson J,  Somerville C: <strong> PICKLE is a CHD3 chromatin-remodeling factor that regulates the transition from embryonic
               to vegetative development in Arabidopsis. </strong></p><p><em>Proc Natl Acad Sci USA</em> 1999,  <strong>96</strong>(24)<strong>:</strong>13839-13844.  <a target="_blank" href="/pubmed/10570159">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10570159">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=10570159">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B70" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B70','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B71">
         <p><a name="B71"/> Swarbreck D,  Wilks C,  Lamesch P,  Berardini TZ,  Garcia-Hernandez M,  Foerster H,  Li D,  Meyer T,  Muller R,  Ploetz L,  Radenbaugh A,  Singh S,  Swing V,  Tissier C,  Zhang P,  Huala E: <strong> The Arabidopsis Information Resource (TAIR): gene structure and function annotation. </strong></p><p><em>Nucleic Acids Res</em> 2008,  <strong>36</strong>(Database issue)<strong>:</strong>D1009-14.  <a target="_blank" href="/pubmed/17986450">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17986450">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17986450">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B71" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B71','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B72">
         <p><a name="B72"/> Finkelstein RR,  Lynch TJ: <strong> The Arabidopsis abscisic acid response gene ABI5 encodes a basic leucine zipper transcription
               factor. </strong></p><p><em>Plant Cell</em> 2000,  <strong>12</strong>(4)<strong>:</strong>599-609.  <a target="_blank" href="/pubmed/10760247">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10760247">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=10760247">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B72" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B72','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B73">
         <p><a name="B73"/> Thomas-Chollier M,  Sand O,  Turatsinze JV,  Janky R,  Defrance M,  Vervisch E,  Brohee S,  van Helden J: <strong> RSAT: regulatory sequence analysis tools. </strong></p><p><em>Nucleic Acids Res</em> 2008,  <strong>36</strong>(Web Server Issue)<strong>:</strong>W119-27.  <a target="_blank" href="/pubmed/18495751">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18495751">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=18495751">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B73" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B73','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B74">
         <p><a name="B74"/> Megraw M,  Baev V,  Rusinov V,  Jensen ST,  Kalantidis K,  Hatzigeorgiou AG: <strong> MicroRNA promoter element discovery in Arabidopsis. </strong></p><p><em>RNA</em> 2006,  <strong>12</strong>(9)<strong>:</strong>1612-1619.  <a target="_blank" href="/pubmed/16888323">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16888323">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=16888323">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B74" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B74','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B75">
         <p><a name="B75"/> Pandey SP,  Krishnamachari A: <strong> Computational analysis of plant RNA Pol-II promoters. </strong></p><p><em>BioSystems</em> 2006,  <strong>83</strong><strong>:</strong>38-50.  <a target="_blank" href="/pubmed/16236422">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16236422">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B75" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B75','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B76">
         <p><a name="B76"/> Lenhard B,  Wasserman WW: <strong> TFBS: Computational framework for transcription factor binding site analysis. </strong></p><p><em>Bioinformatics</em> 2002,  <strong>18</strong>(8)<strong>:</strong>1135-1136.  <a target="_blank" href="/pubmed/12176838">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12176838">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B76" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B76','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B77">
         <p><a name="B77"/> Li L,  Bass RL,  Liang Y: <strong> fdrMotif: identifying cis-elements by an EM algorithm coupled with false discovery
               rate control. </strong></p><p><em>Bioinformatics</em> 2008,  <strong>24</strong>(5)<strong>:</strong>629-636.  <a target="_blank" href="/pubmed/18296465">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18296465">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=18296465">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B77" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B77','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B78">
         <p><a name="B78"/> Bryne JC,  Valen E,  Tang MH,  Marstrand T,  Winther O,  da Piedade I,  Krogh A,  Lenhard B,  Sandelin A: <strong> JASPAR, the open access database of transcription factor-binding profiles: new content
               and tools in the 2008 update. </strong></p><p><em>Nucleic Acids Res</em> 2008,  <strong>36</strong>(Database Issue)<strong>:</strong>D102-106.  <a target="_blank" href="/pubmed/18006571">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18006571">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=18006571">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B78" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B78','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B79">
         <p><a name="B79"/> Yanagisawa S,  Sheen J: <strong> Involvement of maize Dof zinc finger proteins in tissue-specific and light-regulated
               gene expression. </strong></p><p><em>Plant Cell</em> 1998,  <strong>10</strong>(1)<strong>:</strong>75-89.  <a target="_blank" href="/pubmed/9477573">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9477573">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=9477573">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B79" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B79','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B80">
         <p><a name="B80"/> Jiao Y,  Ma L,  Strickland E,  Deng XW: <strong> Conservation and divergence of light-regulated genome expression patterns during seedling
               development in rice and Arabidopsis. </strong></p><p><em>Plant Cell</em> 2005,  <strong>17</strong>(12)<strong>:</strong>3239-3256.  <a target="_blank" href="/pubmed/16284311">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16284311">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=16284311">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B80" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B80','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B81">
         <p><a name="B81"/> Muller B,  Sheen J: <strong> Cytokinin and auxin interaction in root stem-cell specification during early embryogenesis. </strong></p><p><em>Nature</em> 2008,  <strong>453</strong>(7198)<strong>:</strong>1094-1097.  <a target="_blank" href="/pubmed/18463635">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18463635">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=18463635">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B81" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B81','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B82">
         <p><a name="B82"/> Gentleman RC,  Carey VJ,  Bates DM,  Bolstad B,  Dettling M,  Dudoit S,  Ellis B,  Gautier L,  Ge Y,  Gentry J,  Hornik K,  Hothorn T,  Huber W,  Iacus S,  Irizarry R,  Leisch F,  Li C,  Maechler M,  Rossini AJ,  Sawitzki G,  Smith C,  Smyth G,  Tierney L,  Yang JY,  Zhang J: <strong> Bioconductor: open software development for computational biology and bioinformatics. </strong></p><p><em>Genome Biol</em> 2004,  <strong>5</strong>(10)<strong>:</strong>R80.  <a target="_blank" href="/pubmed/15461798">PubMed Abstract</a> | <a target="_blank" href="http://dx.doi.org/10.1186/gb-2004-5-10-r80">BioMed Central Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=15461798">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B82" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B82','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B83">
         <p><a name="B83"/>R Development Core Team: <strong> R: a language and environment for statistical computing. </strong>In <em>The R Foundation for Statistical Computing</em>.  Vienna, Austria;  2008. <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B83" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B83','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B84">
         <p><a name="B84"/> Wilson CL,  Miller CJ: <strong> Simpleaffy: a BioConductor package for Affymetrix Quality Control and data analysis. </strong></p><p><em>Bioinformatics</em> 2005,  <strong>21</strong>(18)<strong>:</strong>3683-3685.  <a target="_blank" href="/pubmed/16076888">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16076888">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B84" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B84','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B85">
         <p><a name="B85"/> Wu ZJ,  Irizarry RA,  Gentleman R,  Martinez-Murillo F,  Spencer F: <strong> A model-based background adjustment for oligonucleotide expression arrays. </strong></p><p><em>J Am Stat Assoc</em> 2003,  <strong>99</strong><strong>:</strong>909-917.  <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B85" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B85','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B86">
         <p><a name="B86"/> Irizarry RA,  Hobbs B,  Collin F,  Beazer-Barclay YD,  Antonellis KJ,  Scherf U,  Speed TP: <strong> Exploration, normalization, and summaries of high density oligonucleotide array probe
               level data. </strong></p><p><em>Biostatistics</em> 2003,  <strong>4</strong>(2)<strong>:</strong>249-264.  <a target="_blank" href="/pubmed/12925520">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12925520">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B86" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B86','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B87">
         <p><a name="B87"/> Altschul SF,  Madden TL,  Schaffer AA,  Zhang JH,  Zhang Z,  Miller W,  Lipman DJ: <strong> Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. </strong></p><p><em>Nucleic Acids Res</em> 1997,  <strong>25</strong><strong>:</strong>3389-3402.  <a target="_blank" href="/pubmed/9254694">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9254694">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=9254694">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B87" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B87','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B88">
         <p><a name="B88"/> Cline MS,  Smoot M,  Cerami E,  Kuchinsky A,  Landys N,  Workman C,  Christmas R,  Avila-Campilo I,  Creech M,  Gross B,  Hanspers K,  Isserlin R,  Kelley R,  Killcoyne S,  Lotia S,  Maere S,  Morris J,  Ono K,  Pavlovic V,  Pico AR,  Vailaya A,  Wang PL,  Adler A,  Conklin BR,  Hood L,  Kuiper M,  Sander C,  Schmulevich I,  Schwikowski B,  Warner GJ,  Ideker T,  Bader GD: <strong> Integration of biological networks and gene expression data using Cytoscape. </strong></p><p><em>Nat Protoc</em> 2007,  <strong>2</strong>(10)<strong>:</strong>2366-2382.  <a target="_blank" href="/pubmed/17947979">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17947979">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B88" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B88','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B89">
         <p><a name="B89"/> Futschik ME,  Carlisle B: <strong> Noise-robust soft clustering of gene expression time-course data. </strong></p><p><em>J Bioinform Comput Biol</em> 2005,  <strong>3</strong>(4)<strong>:</strong>965-988.  <a target="_blank" href="/pubmed/16078370">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16078370">Publisher Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B89" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B89','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B90">
         <p><a name="B90"/> Molina C,  Grotewold E: <strong> Genome wide analysis of Arabidopsis core promoters. </strong></p><p><em>BMC Genomics</em> 2005,  <strong>6</strong>(1)<strong>:</strong>25.  <a target="_blank" href="/pubmed/15733318">PubMed Abstract</a> | <a target="_blank" href="http://dx.doi.org/10.1186/1471-2164-6-25">BioMed Central Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=15733318">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B90" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B90','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B91">
         <p><a name="B91"/> Palaniswamy SK,  James S,  Sun H,  Lamb RS,  Davuluri RV,  Grotewold E: <strong> AGRIS and AtRegNet. A platform to link cis-regulatory elements and transcription factors
               into regulatory networks. </strong></p><p><em>Plant Physiol</em> 2006,  <strong>140</strong>(3)<strong>:</strong>818-829.  <a target="_blank" href="/pubmed/16524982">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16524982">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=16524982">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B91" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B91','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
      <li id="B92">
         <p><a name="B92"/> Crooks GE,  Hon G,  Chandonia JM,  Brenner SE: <strong> WebLogo: a sequence logo generator. </strong></p><p><em>Genome Res</em> 2004,  <strong>14</strong>(6)<strong>:</strong>1188-1190.  <a target="_blank" href="/pubmed/15173120">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15173120">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=15173120">PubMed Central Full Text</a> <a href="/sfx_links?ui=1471-2164-12-286&amp;bibl=B92" onclick="popup('/sfx_links?ui=1471-2164-12-286&amp;bibl=B92','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"/></a></p>
            <p class="totext"><script type="text/javascript">totext()</script></p>
         
      </li>
   </ol></div>&#13;
                    </div>
  </body>
</html>
