<?xml version="1.0"?>
<html>
  <body>
    <div id="centerPane" class="centerPane svBigBox js" aria-label="Article" role="main">
      <script>prs.rt('data_start');</script><div class="head"><div class="publicationHead titleBanner"><div class="cover"><div class="title"><a title="Go to Cell on SciVerse ScienceDirect" href="http://www.sciencedirect.com/science/journal/00928674"><img alt="Cell" src="S0092867400814634_files/B00928674.gif"/></a></div><p><a title="Go to table of contents for this volume/issue" href="http://www.sciencedirect.com/science/journal/00928674/93/7">Volume 93, Issue 7<!--comment--></a>, 26 June 1998, Pages 1195–1205<!--comment--></p></div></div><div class="publicationCover"><img alt="Cover image" src="S0092867400814634_files/Dcellpress.gif" class="toprightlogo"/></div></div><div class="publicationType">Article</div><h1 class="svTitle"><em>Arabidopsis</em> LEAFY COTYLEDON1 Is Sufficient to Induce Embryo Development in Vegetative Cells</h1><ul class="authorGroup"><li>Tamar Lotan<a title="Affiliation: 1" href="#AFF1" class="intra_ref" id="bAFF1"><sup>1</sup></a>, </li><li>Masa-aki Ohto<a title="Affiliation: 1" href="#AFF1" class="intra_ref" id="bAFF1"><sup>1</sup></a><sup>, </sup><a title="Footnote: 2" href="#FN2" class="intra_ref" id="bFN2"><sup>2</sup></a>, </li><li>Kelly Matsudaira Yee<a title="Affiliation: 1" href="#AFF1" class="intra_ref" id="bAFF1"><sup>1</sup></a>, </li><li>Marilyn A.L West<a title="Affiliation: 1" href="#AFF1" class="intra_ref" id="bAFF1"><sup>1</sup></a><sup>, </sup><a title="Footnote: 3" href="#FN3" class="intra_ref" id="bFN3"><sup>3</sup></a>, </li><li>Russell Lo<a title="Affiliation: 1" href="#AFF1" class="intra_ref" id="bAFF1"><sup>1</sup></a><sup>, </sup><a title="Footnote: 4" href="#FN4" class="intra_ref" id="bFN4"><sup>4</sup></a>, </li><li>Raymond W Kwong<a title="Affiliation: 1" href="#AFF1" class="intra_ref" id="bAFF1"><sup>1</sup></a>, </li><li>Kazutoshi Yamagishi<a title="Affiliation: 1" href="#AFF1" class="intra_ref" id="bAFF1"><sup>1</sup></a>, </li><li>Robert L Fischer<a title="Affiliation: 2" href="#AFF2" class="intra_ref" id="bAFF2"><sup>2</sup></a>, </li><li>Robert B Goldberg<a title="Affiliation: 3" href="#AFF3" class="intra_ref" id="bAFF3"><sup>3</sup></a>, </li><li>John J Harada<a title="Affiliation: 1" href="#AFF1" class="intra_ref" id="bAFF1"><sup>1</sup></a><sup>, </sup><a title="Footnote: 1" href="#FN1" class="intra_ref" id="bFN1"><sup>1</sup></a><sup>, </sup><a title="Corresponding author contact information" href="#COR1" id="bCOR1" class="intra_ref"><img alt="Corresponding author contact information" src="S0092867400814634_files/REcor.gif"/></a><sup>, </sup><a href="mailto:jjharada@ucdavis.edu"><img src="S0092867400814634_files/REemail.gif" alt="E-mail the corresponding author"/></a></li></ul><ul class="affiliation"><li id="AFF1"><sup>1</sup> Section of Plant Biology, Division of Biological Sciences, University of California, Davis, California 95616, USA</li><li id="AFF2"><sup>2</sup> Department of Plant and Microbial Biology, University of California, Berkeley, California 94720, USA</li><li id="AFF3"><sup>3</sup> Department of Molecular, Cell, and Developmental Biology, University of California, Los Angeles, California 90024-1606, USA</li></ul><!--comment--><ul class="articleDates"><li>Received 9 December 1997. Revised 7 May 1998. Available online 9 October 2000.</li></ul><ul class="articleDoi"><li><a id="ddDoi" href="http://dx.doi.org/10.1016/S0092-8674%2800%2981463-4" target="doilink" onclick="var doiWin; doiWin=window.open('http://dx.doi.org/10.1016/S0092-8674(00)81463-4','doilink','scrollbars=yes,resizable=yes,directories=yes,toolbar=yes,menubar=yes,status=yes'); doiWin.focus()">http://dx.doi.org/10.1016/S0092-8674(00)81463-4</a>, <a href="http://www.sciencedirect.com/science/help/doi.htm" target="sdhelp" onmouseover="window.status='Help is Available';return true" onclick="var helpWin; helpWin=window.open('http://www.sciencedirect.com/science/help/doi.htm','sdhelp','scrollbars=yes,resizable=yes,directories=no,toolbar=no,menubar=no,status=no,width=760,height=570'); helpWin.focus()">How to Cite or Link Using DOI</a></li></ul><ul class="articleInfo"><li class="permissions"><div class="boxLink"><a href="http://www.sciencedirect.com/science?_ob=RedirectURL&amp;_method=outwardLink&amp;_partnerName=936&amp;_eid=1-s2.0-S0092867400814634&amp;_pii=S0092867400814634&amp;_origin=article&amp;_zone=art_page&amp;_targetURL=https%3A%2F%2Fs100.copyright.com%2FAppDispatchServlet%3FpublisherName%3DELS%26contentID%3DS0092867400814634%26orderBeanReset%3Dtrue&amp;_acct=C000228598&amp;_version=1&amp;_userid=10&amp;md5=8a6249cc65fa09e70896cf1904d2275d" rel="nofollow" class="olSpawnWindow" target="outwardLink">Permissions &amp; Reprints</a></div></li></ul><hr id="Abstract"/><div class="abstract svAbstract"><h2 class="secHeading" id="section_abstract">Abstract</h2><p id="">The <em>Arabidopsis LEAFY COTYLEDON1</em> (<em>LEC1</em>) gene is required for the specification of cotyledon identity and the completion of embryo maturation. We isolated the <em>LEC1</em> gene and showed that it functions at an early developmental stage to maintain embryonic cell fate. The <em>LEC1</em> gene encodes a transcription factor homolog, the CCAAT box–binding factor HAP3 subunit. <em>LEC1</em>
 RNA accumulates only during seed development in embryo cell types and 
in endosperm tissue. Ectopic postembryonic expression of the <em>LEC1</em>
 gene in vegetative cells induces the expression of embryo-specific 
genes and initiates formation of embryo-like structures. Our results 
suggest that LEC1 is an important regulator of embryo development that 
activates the transcription of genes required for both embryo 
morphogenesis and cellular differentiation.</p></div><script>prs.rt("abs_end");</script><div id="SD_BA1P" class="sgfNoTitleBar sgfNoGadgetBorder svDoNotLink"/><hr/><h2 class="svArticle" id="section_lbl1">Introduction</h2><p class="svArticle section" id="">Higher
 plant embryogenesis is divided conceptually into two distinct phases: 
early morphogenetic processes that give rise to embryonic cell types, 
tissues, and organ systems, and late maturation events that allow the 
fully developed embryo to enter a desiccated and metabolically quiescent
 state (<a href="#BIB52" id="bBIB52" class="intra_ref">
                  [52]</a> and <a href="#BIB13" id="bBIB13" class="intra_ref">[13]</a>).
 Upon reception of the appropriate signals, the dormant embryo 
germinates, and seedling development begins. Thus, seed maturation and 
metabolic quiescence interrupt the morphogenetic processes that occur 
during embryogenesis and seedling development. This unique form of 
development underlies, in part, a plant's ability to make seeds, a trait
 that has conferred significant selective advantages to higher plants (<span id="bBIB46"><a href="#BIB46" id="ancbBIB46" class="intra_ref">Steeves 1983</a></span>).
 Because lower plants do not make seeds and do not undergo embryo 
maturation, this bipartite mode of embryogenesis is thought to have 
resulted from the insertion of maturation events into the higher plant 
life cycle (reviewed by<a href="#BIB51" id="bBIB51" class="intra_ref">
                  [51]</a>, <a href="#BIB47" id="bBIB47" class="intra_ref">
                  [47]</a> and <a href="#BIB16" id="bBIB16" class="intra_ref">[16]</a>).
 Little is known at the mechanistic level about how distinct processes 
that occur during the morphogenesis and seed maturation phases are 
coordinated.</p><p class="svArticle section" id="">Genetic strategies employed to identify regulators of <em>Arabidopsis thaliana</em> embryo development have distinguished several gene classes that affect embryogenesis. One class of mutations, including <em>raspberry</em>, <em>tinman</em>, and <em>abnormal suspensor</em> ( <a href="#BIB41" id="bBIB41" class="intra_ref">
                  [41]</a>, <a href="#BIB56" id="bBIB56" class="intra_ref">
                  [56]</a>, <a href="#BIB7" id="bBIB7" class="intra_ref">
                  [7]</a> and <a href="#BIB49" id="bBIB49" class="intra_ref">[49]</a>;
 T. L. et al, unpublished data), causes the arrest of embryo 
morphogenesis. Many of the corresponding proteins are involved in basic 
cellular functions and probably do not perform direct regulatory roles. A
 second class of genes predicted to play a role in embryonic pattern 
formation has been identified (<span id="bBIB26"><a href="#BIB26" id="ancbBIB26" class="intra_ref">Mayer et al. 1991</a></span>). The expression patterns of two such genes, <em>EMB30</em> (<em>GNOM</em>) and <em>KNOLLE</em>, suggest that they do not function specifically during embryogenesis ( <a href="#BIB42" id="bBIB42" class="intra_ref">
                  [42]</a> and <a href="#BIB23" id="bBIB23" class="intra_ref">[23]</a>). Another class, including <em>SHOOTMERISTEMLESS</em> and <em>SCARECROW</em>, is active in the developing shoot or root apical meristems ( <a href="#BIB9" id="bBIB9" class="intra_ref">
                  [9]</a> and <a href="#BIB22" id="bBIB22" class="intra_ref">[22]</a>).
 Although these genes play key roles in meristem function, they are 
required for meristem formation during both embryogenesis and 
postembryonic development. <em>ABSCISIC ACID INSENSITIVE3</em> (<em>ABI3</em>), a fourth gene class, encodes an embryo-specific transcription factor that regulates genes expressed during seed maturation ( <a href="#BIB20" id="bBIB20" class="intra_ref">
                  [20]</a> and <a href="#BIB38" id="bBIB38" class="intra_ref">[38]</a>).
 However, its function is limited to the late stages of embryogenesis. 
Although each of these gene classes is essential for embryo development,
 none appears to act specifically during embryogenesis to control both 
the morphogenesis and maturation phases.</p><p class="svArticle section" id="">The <em>LEAFY COTYLEDON1</em> (<em>LEC1</em>) gene, by contrast, controls many distinct aspects of embryogenesis (<span id="bBIB28"><a href="#BIB28" id="ancbBIB28" class="intra_ref">Meinke 1992</a></span>Meinke, 1992, 1994;  <a href="#BIB53" id="bBIB53" class="intra_ref">
                  [53]</a> and <a href="#BIB39" id="bBIB39" class="intra_ref">[39]</a>). As summarized in <span id="bFIG1"><a href="#FIG1" id="ancbFIG1" class="intra_ref">Figure 1</a></span>, the <em>lec1</em> mutation is pleiotropic, indicating several roles in late embryo development. <em>LEC1</em> is required for specific aspects of seed maturation. <em>lec1</em>
 mutant embryos are intolerant of desiccation and show defects in the 
expression of some, but not all, maturation-specific genes ( <a href="#BIB28" id="bBIB28" class="intra_ref">
                  [28]</a>, <a href="#BIB53" id="bBIB53" class="intra_ref">
                  [53]</a> and <a href="#BIB39" id="bBIB39" class="intra_ref">[39]</a>). <em>LEC1</em> is also involved in inhibiting premature germination. <em>lec1</em>
 mutant embryos exhibit morphological and molecular characteristics of 
both embryogenesis and postgerminative seedling development, showing 
that aspects of both programs can occur simultaneously (<span id="bBIB28"><a href="#BIB28" id="ancbBIB28" class="intra_ref">Meinke 1992</a></span>Meinke, 1992, 1994; <span id="bBIB53"><a href="#BIB53" id="ancbBIB53" class="intra_ref">West et al. 1994</a></span>). <em>LEC1</em> plays a role in the specification of embryonic organ identity as well. Embryonic leaves or cotyledons of <em>lec1</em> mutants possess trichomes, epidermal hairs, which normally form only on leaves and stems in <em>Arabidopsis</em> ( <a href="#BIB28" id="bBIB28" class="intra_ref">
                  [28]</a> and <a href="#BIB53" id="bBIB53" class="intra_ref">[53]</a>). The anatomy of <em>lec1</em> mutant cotyledons is intermediate between those of a wild-type cotyledon and a leaf (<span id="bBIB53"><a href="#BIB53" id="ancbBIB53" class="intra_ref">West et al. 1994</a></span>). Finally, <em>LEC1</em> appears to act only during embryo development. Desiccation-intolerant <em>lec1</em>
 embryos can be rescued from plants before desiccation and germinated to
 produce homozygous mutant plants that are fertile and that do not 
display any obvious vegetative or floral mutant phenotypes ( <a href="#BIB28" id="bBIB28" class="intra_ref">
                  [28]</a> and <a href="#BIB53" id="bBIB53" class="intra_ref">[53]</a>). Two other <em>LEC</em> class genes, <em>LEC2</em> and <em>FUSCA3</em> (<em>FUS3</em>), are thought to share similar or overlapping functions with <em>LEC1</em>, including the specification or cotyledon identity and the maintenance of maturation ( <a href="#BIB2" id="bBIB2" class="intra_ref">
                  [2]</a>, <a href="#BIB18" id="bBIB18" class="intra_ref">
                  [18]</a> and <a href="#BIB29" id="bBIB29" class="intra_ref">[29]</a>). Although nothing has been reported about how <em>LEC</em>
 class genes act at the molecular level, their involvement in many 
diverse aspects of embryogenesis suggests that these genes serve as 
regulators of higher plant embryonic processes.</p><div class="figTblUpiOuter svArticle" id="figure_FIG1"><div class="&#10;                    textboxdefault&#10;                "><dl class="figure" id="FIG1"><dt style="" class="autoScroll" data-style="height:413px;width:640px;"><a onclick="openStrippedNS('/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0092867400814634&amp;_image=1-s2.0-S0092867400814634-gr1.jpg&amp;_ba=&amp;_rdoc=1&amp;_fmt=full&amp;_orig=na&amp;_issn=00928674&amp;_pii=S0092867400814634&amp;view=c&amp;_acct=C000228598&amp;_version=1&amp;_urlVersion=0&amp;_userid=10&amp;md5=1ddeb02de2dfcd7d0885489d967eaba6','labelCaptionFIG1','','S0092867400814634');  return false;" href="#gr1" title="Full-size image (44K)"><img data-loaded="true" src="S0092867400814634_files/1-s2.jpg" data-fullsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr1.jpg" data-thumbsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr1.sml" alt="Full-size image" class="figure large" border="0"/><noscript class="figure large" border="0" alt="Full-size image" data-thumbsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr1.sml" data-fullsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr1.jpg"><img class="figure large" border="0" alt="Full-size image (44K)" src="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr1.jpg" data-thumbsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr1.sml" data-fullsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr1.jpg"/></noscript></a></dt><dd id="labelCaptionFIG1"><p class="caption">Figure 1. <span id="">Pleiotropic Effects of the <em>lec1</em> Mutation on Embryo Development<!--Comment--></span><span id="">Major differences between wild-type and <em>lec1</em>
 mutant embryos are as follows. Embryo shape: the axes of mutant embryos
 are short, and their cotyledons are round and do not curl. Anthocyanin 
generally accumulates at the tips of mutant cotyledons. Precocious 
germination: the shoot apical meristems of <em>lec1</em> embryos are 
activated in that they are domed and possess leaf primordia, unlike 
their wild-type counterparts that are flat and do not contain leaf 
primordia. Defects in seed maturation: <em>lec1</em> mutant embryos are intolerant of desiccation and normally die if dried on the plant. However, <em>lec1</em>
 embryos isolated before desiccation can be germinated to produce 
fertile homozygous mutant plants. The promoter of a 7S storage protein 
gene that is normally active during wild-type embryogenesis is not 
active in the <em>lec1</em> mutant. Incomplete specification of cotyledon identity: <em>lec1</em> seedlings possess trichomes on cotyledons. Trichomes are present on <em>Arabidopsis</em> leaves and stems but not on wild-type cotyledons. a, axis; c, cotyledon; SAM, shoot apical meristem.<!--Comment--></span><!--caption--></p><a style="display: none;" class="linkText" title="View thumbnail images" href="#gr1">View thumbnail images</a><p><a onclick="openStrippedNS('/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0092867400814634&amp;_image=1-s2.0-S0092867400814634-gr1_lrg.jpg&amp;_ba=&amp;_rdoc=1&amp;_fmt=full&amp;_orig=na&amp;_issn=00928674&amp;_pii=S0092867400814634&amp;view=c&amp;_isHiQual=Y&amp;_acct=C000228598&amp;_version=1&amp;_urlVersion=0&amp;_userid=10&amp;md5=3e48b58ba8f7e4092747aef1c45977b5','labelCaptionFIG1','','S0092867400814634'); return false;" title="View high quality image - Opens in new Window" href="#gr1">View high quality image (116K)</a></p></dd></dl></div></div><p class="svArticle section" id="">In this paper, we report the isolation of the <em>LEC1</em> gene and show that it encodes a homolog of a conserved eukaryotic transcription factor. Expression studies showed that the <em>LEC1</em> gene is active only within seeds during both early and late seed development. Ectopic expression of the <em>LEC1</em>
 gene induces embryonic programs and embryo development in vegetative 
cells. We suggest that LEC1 is an important transcriptional regulator 
required for both early and late embryogenesis that controls and 
coordinates higher plant embryo development.</p><h2 class="svArticle" id="section_lbl2">Results</h2><h3 class="svArticle" id="section_lbl2.1"><em>LEC1</em> Functions Early in Embryogenesis</h3><p class="svArticle section" id="">The Lec1<sup>−</sup> phenotype indicates that the gene plays a significant role in controlling late embryo development (<span id="bFIG1"><a href="#FIG1" id="ancbFIG1" class="intra_ref">Figure 1</a></span>). To determine whether the gene is also required for early embryonic events, we analyzed early-stage <em>lec1</em> mutant embryos and detected defects in suspensor morphology. The wild-type suspensor, shown in <span id="bFIG2"><a href="#FIG2" id="ancbFIG2" class="intra_ref">Figure 2</a></span>D,
 is a transient embryonic structure consisting of a single file of six 
to eight cells that are identical genotypically to embryo proper cells. 
By contrast, <span id="bFIG2"><a href="#FIG2" id="ancbFIG2" class="intra_ref">Figure 2</a></span>A and <span id="bFIG2"><a href="#FIG2" id="ancbFIG2" class="intra_ref">Figure 2</a></span>B show that globular- and transition-stage embryos homozygous for either of the two <em>lec1</em>
 mutant alleles had abnormal suspensors. Cell walls parallel to the 
suspensor axis were observed, suggesting that aberrant cell divisions 
occurred in the mutant suspensors. As summarized in <span id="bTBL1"><a href="#TBL1" id="ancbTBL1" class="intra_ref">Table 1</a></span>, abnormal suspensors rarely observed in wild-type embryos were detected initially in globular/transition-stage <em>lec1</em> embryos and were represented in approximately 90% of mutant torpedo-stage embryos.</p><div class="figTblUpiOuter svArticle" id="figure_FIG2"><div class="&#10;                    textboxdefault&#10;                "><dl class="figure" id="FIG2"><dt style="height:1352px;width:640px;" class="autoScroll" data-style="height:1352px;width:640px;"><a onclick="openStrippedNS('/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0092867400814634&amp;_image=1-s2.0-S0092867400814634-gr2.jpg&amp;_ba=&amp;_rdoc=1&amp;_fmt=full&amp;_orig=na&amp;_issn=00928674&amp;_pii=S0092867400814634&amp;view=c&amp;_acct=C000228598&amp;_version=1&amp;_urlVersion=0&amp;_userid=10&amp;md5=1971dcf924738148d9adface133ec1e8','labelCaptionFIG2','','S0092867400814634');  return false;" href="#gr2" title="Full-size image (203K)"><img src="S0092867400814634_files/1-s2_003.gif" data-fullsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr2.jpg" data-thumbsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr2.sml" alt="Full-size image" class="figure large" border="0"/><noscript class="figure large" border="0" alt="Full-size image" data-thumbsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr2.sml" data-fullsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr2.jpg"><img class="figure large" border="0" alt="Full-size image (203K)" src="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr2.jpg" data-thumbsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr2.sml" data-fullsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr2.jpg"/></noscript></a></dt><dd id="labelCaptionFIG2"><p class="caption">Figure 2. <span id="">Embryogenic Potential of <em>lec1</em> Mutant Suspensor<!--Comment--></span><span id="">(A)–(D) Seeds from the indicated plants were cleared and photographed using Nomarski optics. <em>lec1-1</em> (A); <em>lec1-2</em> (B); <em>lec1-2 fus3-3</em>
 (C); wild type (D). Arrowheads indicate sites of abnormal suspensor 
cell divisions. Bar, 25 μm. (E) A primary (top) and a secondary (bottom)
 embryo isolated from a <em>lec1-2 fus3-3</em> double mutant. Bar, 0.1 mm. (F) A germinated <em>lec1-2 fus3-3</em> seed containing twin seedlings. Bar, 0.5 mm. ep, embryo proper; s, suspensor.<!--Comment--></span><!--caption--></p><a style="display: none;" class="linkText" title="View thumbnail images" href="#gr2">View thumbnail images</a><p><a onclick="openStrippedNS('/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0092867400814634&amp;_image=1-s2.0-S0092867400814634-gr2_lrg.jpg&amp;_ba=&amp;_rdoc=1&amp;_fmt=full&amp;_orig=na&amp;_issn=00928674&amp;_pii=S0092867400814634&amp;view=c&amp;_isHiQual=Y&amp;_acct=C000228598&amp;_version=1&amp;_urlVersion=0&amp;_userid=10&amp;md5=4b6e76a92d65436f4405522fa1c459cf','labelCaptionFIG2','','S0092867400814634'); return false;" title="View high quality image - Opens in new Window" href="#gr2">View high quality image (209K)</a></p></dd></dl></div></div><div class="figTblUpiOuter svArticle" id="table_TBL1"><div class="textboxdefault"><dl id="TBL1" class="table "><dd><p class="caption"><span class="label">Table 1.
        </span>Suspensor Abnormalities in Mutant Embryos</p></dd><dd class="table"><table><colgroup><col/><col/><col/><col/></colgroup><tbody><tr><td rowspan="1" colspan="1"/><td rowspan="1" colspan="3">Embryonic Stages</td></tr><tr><td rowspan="1" colspan="1"/><td rowspan="1" colspan="1">Globular/Transition</td><td rowspan="1" colspan="1">Heart</td><td rowspan="1" colspan="1">Torpedo</td></tr><tr><td rowspan="1" colspan="1">Wild type</td><td rowspan="1" colspan="1">1/162 (0.01%)<span id="bTBL1FNA"><a href="#TBL1FNA" id="ancbTBL1FNA" class="intra_ref"><sup>a</sup></a></span></td><td rowspan="1" colspan="1">2/78 (2.6%)</td><td rowspan="1" colspan="1">0/200 (0%)</td></tr><tr><td rowspan="1" colspan="1"><em>lec1-1</em></td><td rowspan="1" colspan="1">25/115 (22%)</td><td rowspan="1" colspan="1">87/128 (68%)</td><td rowspan="1" colspan="1">49/54 (91%)</td></tr><tr><td rowspan="1" colspan="1"><em>lec1-2</em></td><td rowspan="1" colspan="1">15/164 (9.0%)</td><td rowspan="1" colspan="1">48/80 (60%)</td><td rowspan="1" colspan="1">29/32 (90%)</td></tr><tr><td rowspan="1" colspan="1"><em>lec2</em></td><td rowspan="1" colspan="1">1/189 (0.01%)</td><td rowspan="1" colspan="1">4/47 (8.5%)</td><td rowspan="1" colspan="1">21/54 (39%)</td></tr><tr><td rowspan="1" colspan="1"><em>fus3-3</em></td><td rowspan="1" colspan="1">9/97 (9.3%)</td><td rowspan="1" colspan="1">21/112 (19%)</td><td rowspan="1" colspan="1">10/89 (11%)</td></tr><tr><td rowspan="1" colspan="1"><em>abi3-3</em></td><td rowspan="1" colspan="1">0/68 (0%)</td><td rowspan="1" colspan="1">0/62 (0%)</td><td rowspan="1" colspan="1">0/20 (0%)</td></tr></tbody></table></dd><dd><dl class="footnote"><dt id="TBL1FNA">a</dt><dd><p>Number of embryos with abnormal suspensors/total embryos analyzed (percent embryos with abnormal suspensors).</p></dd></dl></dd><dd class="fullsizeTable"><a onclick="openStrippedNS('/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0092867400814634&amp;_image=TBL1&amp;_ba=&amp;_rdoc=1&amp;_fmt=full&amp;_orig=na&amp;_issn=00928674&amp;_pii=S0092867400814634&amp;view=c&amp;_isTablePopup=Y&amp;_acct=C000228598&amp;_version=1&amp;_urlVersion=0&amp;_userid=10&amp;md5=c4d370e55e673dd5dcc38f54a8a1a9e3','');  return false;" title="Full-size table - Opens new window" href="#">Full-size table</a></dd></dl></div></div><p class="svArticle section" id="">We investigated a <em>lec1 fus3</em> double mutant (<span id="bBIB53"><a href="#BIB53" id="ancbBIB53" class="intra_ref">West et al. 1994</a></span>) to learn whether other <em>LEC</em> class genes could enhance the effect of the <em>lec1</em> mutation on suspensor development. <span id="bFIG2"><a href="#FIG2" id="ancbFIG2" class="intra_ref">Figure 2</a></span>C shows that suspensor abnormalities were observed in <em>lec1-2 fus3-3</em>
 double mutants at an early embryonic stage as with the single mutants. 
By contrast to wild-type suspensors that undergo a limited number of 
cell divisions, suspensor cells continued to proliferate in the double 
mutants. Subsequently, secondary embryos, shown in <span id="bFIG2"><a href="#FIG2" id="ancbFIG2" class="intra_ref">Figure 2</a></span>E, formed from these abnormal suspensor cell masses in approximately 20% (118/598) of <em>lec1-2 fus3-3</em> seeds. Primary embryos were attached to secondary, suspensor-derived embryos either at or near the latter's shoot apices (<span id="bFIG2"><a href="#FIG2" id="ancbFIG2" class="intra_ref">Figure 2</a></span>E)
 or at their root ends (data not shown). Both primary and secondary 
embryos were able to germinate, and viable seedings were produced (<span id="bFIG2"><a href="#FIG2" id="ancbFIG2" class="intra_ref">Figure 2</a></span>F). Secondary embryo formation was also observed in <em>lec1-1 fus3-3</em> and <em>lec1-2 abi3-3</em> double mutants, but not in <em>lec1</em> single mutants or in <em>lec1-2 lec2</em> double mutants, and only rarely formed in <em>fus3-3</em>
 monogenic mutants (e.g., 2 of 298 seeds had secondary embryos). These 
results showed that polyembryony was not limited to a particular <em>lec1</em> or <em>fus3</em> allele. Control experiments showed that abnormal suspensors were also detected in early-stage <em>lec2-1</em> and <em>fus3-3</em> mutant embryos but not in <em>abi3-3</em> mutant embryos (<span id="bTBL1"><a href="#TBL1" id="ancbTBL1" class="intra_ref">Table 1</a></span>). Together these results indicate that <em>LEC1</em> gene activity is required during early embryogenesis, in part, to suppress the embryogenic potential of the suspensor.</p><h3 class="svArticle" id="section_lbl2.2">Insertion and Deletion Mutations Identify the <em>LEC1</em> Gene</h3><p class="svArticle section" id=""><em>lec1-1</em> and <em>lec1-2</em> mutant alleles were derived from a population of plants mutagenized insertionally with T-DNA ( <a href="#BIB11" id="bBIB11" class="intra_ref">
                  [11]</a>, <a href="#BIB28" id="bBIB28" class="intra_ref">
                  [28]</a> and <a href="#BIB53" id="bBIB53" class="intra_ref">[53]</a>). The <em>lec1-1</em> mutation is not associated with T-DNA (<span id="bBIB28"><a href="#BIB28" id="ancbBIB28" class="intra_ref">Meinke 1992</a></span>; M. O. and J. J. H., unpublished data); however, we showed that a specific subset of the T-DNA fragments in <em>lec1-2</em> was within 1.5 cM of the <em>lec1</em> mutation. We identified genomic clones containing T-DNA sequences that cosegregated with the <em>lec1</em> mutation from a library of <em>lec1-2</em> DNA. The one clone that contained <em>Arabidopsis</em> DNA sequences identified restriction fragment length polymorphisms that distinguished wild-type, <em>lec1-1</em>, and <em>lec1-2</em> genomic DNAs. <span id="bFIG3"><a href="#FIG3" id="ancbFIG3" class="intra_ref">Figure 3</a></span>B shows that plant DNA sequences flanking the <em>lec1-2</em> T-DNA identified 2.25 kb, 1.2 kb, and 0.6 kb HindIII fragments in wild-type DNA that were replaced with a 2.0 kb fragment in <em>lec1-1</em> DNA. The two <em>lec1-2</em> DNA fragments marked by the arrows in <span id="bFIG3"><a href="#FIG3" id="ancbFIG3" class="intra_ref">Figure 3</a></span>B that were absent in wild-type DNA presumably represented T-DNA/plant DNA junction fragments.</p><div class="figTblUpiOuter svArticle" id="figure_FIG3"><div class="&#10;                    textboxdefault&#10;                "><dl class="figure" id="FIG3"><dt style="height:895px;width:640px;" class="autoScroll" data-style="height:895px;width:640px;"><a onclick="openStrippedNS('/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0092867400814634&amp;_image=1-s2.0-S0092867400814634-gr3.jpg&amp;_ba=&amp;_rdoc=1&amp;_fmt=full&amp;_orig=na&amp;_issn=00928674&amp;_pii=S0092867400814634&amp;view=c&amp;_acct=C000228598&amp;_version=1&amp;_urlVersion=0&amp;_userid=10&amp;md5=9a9f4487f4dc6ab22aac6cd4836d31ff','labelCaptionFIG3','','S0092867400814634');  return false;" href="#gr3" title="Full-size image (84K)"><img src="S0092867400814634_files/1-s2_007.gif" data-fullsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr3.jpg" data-thumbsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr3.sml" alt="Full-size image" class="figure large" border="0"/><noscript class="figure large" border="0" alt="Full-size image" data-thumbsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr3.sml" data-fullsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr3.jpg"><img class="figure large" border="0" alt="Full-size image (84K)" src="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr3.jpg" data-thumbsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr3.sml" data-fullsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr3.jpg"/></noscript></a></dt><dd id="labelCaptionFIG3"><p class="caption">Figure 3. <span id="">Structure and Expression of <em>LEC1</em> Gene Loci<!--Comment--></span><span id="">(A) Diagrammatic representation of the <em>LEC1</em> locus in wild-type (WT), <em>lec1-1</em>, and <em>lec1-2</em>
 chromosomes. The box represents the LEC1 open reading frame and its 
transcriptional orientation. The positions of the deletion in <em>lec1-1</em> and of the T-DNA insertion in <em>lec1-2</em>
 are indicated. The bars represent the 3.4 kb BstYI restriction fragment
 used for the transgene complementation test and the 7.4 kb EcoRI 
restriction fragment used in (B). Vertical lines represent HindIII 
restriction sites.<!--Comment--></span><span id="">(B) Gel blot of genomic DNA from wild-type (wt), <em>lec1-1</em>, and <em>lec1-2</em>
 digested with HindIII and hybridized with the wild-type 7.4 kb EcoRI 
fragment shown in (A). Arrows indicate restriction fragments in <em>lec1-2</em> that contain the T-DNA/plant DNA junctions.<!--Comment--></span><span id="">(C) Gel blot containing 2 μg polyadenylated RNA from homozygous <em>lec1-1</em> siliques (lane 1), homozygous <em>lec1-2</em>
 siliques (lane 2), wild-type siliques (lane 3), and the leaves and 
stems of wild-type seedlings grown for 2 weeks (lane 4) were hybridized 
with the <em>LEC1</em> cDNA clone. The 0.85 kb <em>LEC1</em> RNA was 
detected only in wild-type siliques. Hybridization of this blot and the 
blot shown in (D) with a probe for a constitutively expressed ribosomal 
protein showed that equal amounts of RNA were present in each lane (data
 not shown).<!--Comment--></span><span id="">(D) Gel blot analysis of 20
 μg total RNA from wild-type siliques at three different stages 
hybridized with LEC1 cDNA clone. Lane 1, preglobular to heart stage; 
lane 2, heart to curled cotyledon stage; lane 3, maturing embryo stage.<!--Comment--></span><!--caption--></p><a style="display: none;" class="linkText" title="View thumbnail images" href="#gr3">View thumbnail images</a><p><a onclick="openStrippedNS('/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0092867400814634&amp;_image=1-s2.0-S0092867400814634-gr3_lrg.jpg&amp;_ba=&amp;_rdoc=1&amp;_fmt=full&amp;_orig=na&amp;_issn=00928674&amp;_pii=S0092867400814634&amp;view=c&amp;_isHiQual=Y&amp;_acct=C000228598&amp;_version=1&amp;_urlVersion=0&amp;_userid=10&amp;md5=13ea553568ea033f36756e4e605996c6','labelCaptionFIG3','','S0092867400814634'); return false;" title="View high quality image - Opens in new Window" href="#gr3">View high quality image (253K)</a></p></dd></dl></div></div><p class="svArticle section" id="">Restriction mapping and nucleotide sequence analyses, summarized in <span id="bFIG3"><a href="#FIG3" id="ancbFIG3" class="intra_ref">Figure 3</a></span>A, indicated that one T-DNA complex in <em>lec1-2</em>
 was inserted 115 bp upstream of a 626 bp open reading frame (ORF). 
Several lines of evidence suggested that this ORF represented the <em>LEC1</em> gene. First, as shown diagrammatically in <span id="bFIG3"><a href="#FIG3" id="ancbFIG3" class="intra_ref">Figure 3</a></span>A, cloning and nucleotide sequencing studies showed that this ORF was part of a 2000 bp region deleted in <em>lec1-1</em>
 DNA. Second, this ORF was represented in embryo RNA. cDNA clones 
corresponding to the ORF were identified from a library of silique RNA 
that contained developing embryo mRNA. As shown in <span id="bFIG3"><a href="#FIG3" id="ancbFIG3" class="intra_ref">Figure 3</a></span>C, this cDNA clone identified a single 0.85 kb polyadenylated RNA detected in wild-type silique RNA (lane 3) but not <em>lec1-1</em> (lane 1) or <em>lec1-2</em>
 (lane 2) silique RNAs. This result suggested that both mutations 
compromised the expression of this ORF. Third, RNA gel blot analysis 
using the 7.4 kb EcoRI restriction fragment shown in <span id="bFIG3"><a href="#FIG3" id="ancbFIG3" class="intra_ref">Figure 3</a></span>A
 that spanned a region 4.4 kb upstream and 2.4 kp downstream of this ORF
 did not identify any other transcript in the silique RNA. Finally, 
nucleotide sequencing studies did not reveal any other extended ORFs 
within this 7.4 kb EcoRI restriction fragment.</p><p class="svArticle section" id="">To demonstrate directly that this ORF corresponded to the <em>LEC1</em> gene, we transferred the 3394 bp fragment that contained this ORF and its 5′ and 3′ flanking DNA sequences into homozygous <em>lec1-1</em> and <em>lec1-2</em> mutant plants (see <span id="bFIG3"><a href="#FIG3" id="ancbFIG3" class="intra_ref">Figure 3</a></span>A).
 Plants transformed with this ORF produced viable seeds that survived 
desiccation, germinated, and produced fertile plants with a wild-type 
vegetative phenotype. Because <em>lec1</em> mutant embryos are normally 
desiccation intolerant, this result indicated that the ORF complemented 
the mutation. PCR amplification experiments and segregation analyses 
verified that all viable transgenic plants were homozygous for the <em>lec1</em>
 mutation and contained 1–3 copies of the transgene (data not shown). 
These data showed that the 3.4 kb fragment containing this ORF 
complemented <em>lec1</em> mutations. Taken together, these results indicate that the ORF represents the <em>LEC1</em> gene.</p><h3 class="svArticle" id="section_lbl2.3">LEC1 Is a Homolog of CCAAT Box–Binding Factor</h3><p class="svArticle section" id="">The
 predicted LEC1 polypeptide shared significant sequence similarity with 
the HAP3 subunit of the CCAAT box–binding factor (CBF; <span id="bFIG4"><a href="#FIG4" id="ancbFIG4" class="intra_ref">Figure 4</a></span>).
 Based on amino acid sequence comparisons, the HAP3 subunit is divided 
into three domains: an amino-terminal A domain, a central B domain, and a
 carboxyl-terminal C domain, as shown diagrammatically in <span id="bFIG4"><a href="#FIG4" id="ancbFIG4" class="intra_ref">Figure 4</a></span>A (<span id="bBIB21"><a href="#BIB21" id="ancbBIB21" class="intra_ref">Li et al. 1992</a></span>). <span id="bFIG4"><a href="#FIG4" id="ancbFIG4" class="intra_ref">Figure 4</a></span>B
 shows that LEC1 shared between 75% and 85% similarity and between 55% 
and 63% identity with the B domains of other HAP3 homologs. This degree 
of similarity was comparable to that obtained in sequence comparisons 
among HAP3 subunits from other organisms (<span id="bBIB21"><a href="#BIB21" id="ancbBIB21" class="intra_ref">Li et al. 1992</a></span>).
 No significant sequence similarity was detected between the A and C 
domains of LEC1 and other HAP3 homologs, or with any other polypeptide 
sequence recorded in the databases. The high degree of sequence 
conservation in the B domain strongly suggests that LEC1 is part of an 
oligomeric CBF transcriptional activator.</p><div class="figTblUpiOuter svArticle" id="figure_FIG4"><div class="&#10;                    textboxdefault&#10;                "><dl class="figure" id="FIG4"><dt style="height:755px;width:640px;" class="autoScroll" data-style="height:755px;width:640px;"><a onclick="openStrippedNS('/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0092867400814634&amp;_image=1-s2.0-S0092867400814634-gr4.jpg&amp;_ba=&amp;_rdoc=1&amp;_fmt=full&amp;_orig=na&amp;_issn=00928674&amp;_pii=S0092867400814634&amp;view=c&amp;_acct=C000228598&amp;_version=1&amp;_urlVersion=0&amp;_userid=10&amp;md5=8e48044135bb083073022d6652a894e3','labelCaptionFIG4','BIB21','S0092867400814634');  return false;" href="#gr4" title="Full-size image (195K)"><img src="S0092867400814634_files/1-s2.gif" data-fullsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr4.jpg" data-thumbsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr4.sml" alt="Full-size image" class="figure large" border="0"/><noscript class="figure large" border="0" alt="Full-size image" data-thumbsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr4.sml" data-fullsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr4.jpg"><img class="figure large" border="0" alt="Full-size image (195K)" src="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr4.jpg" data-thumbsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr4.sml" data-fullsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr4.jpg"/></noscript></a></dt><dd id="labelCaptionFIG4"><p class="caption">Figure 4. <span id="">Amino Acid Sequence Similarity between LEC1 and Other CBF HAP3 Homologs<!--Comment--></span><span id="">(A) Schematic representation of the three domains of the predicted LEC1 polypeptide.<!--Comment--></span><span id="">(B) Comparison of the predicted amino acid sequence of the B domain encoded by <em>LEC1</em> with HAP3 homologs from maize, chicken, lamprey, <em>Xenopus laveis</em>, human, mouse (<span id="FIG4bBIB21"><a href="#BIB21" id="ancFIG4bBIB21" class="intra_ref">Li et al. 1992</a></span>), rat (<span id="FIG4bBIB50"><a href="#BIB50" id="ancFIG4bBIB50" class="intra_ref">Vuorio et al. 1990</a></span>), <em>Emericella nidulans</em> (<span id="FIG4bBIB37"><a href="#BIB37" id="ancFIG4bBIB37" class="intra_ref">Papagiannopoulos et al. 1996</a></span>), <em>Schizosaccharomyces pombe</em> (<span id="FIG4bBIB55"><a href="#BIB55" id="ancFIG4bBIB55" class="intra_ref">Xing et al. 1993</a></span>), <em>Saccharomyces cerevisiae</em> (<span id="FIG4bBIB15"><a href="#BIB15" id="ancFIG4bBIB15" class="intra_ref">Hahn et al. 1988</a></span>), and <em>Kluyveromyces lactis</em> (<span id="FIG4bBIB30"><a href="#BIB30" id="ancFIG4bBIB30" class="intra_ref">Mulder et al. 1994</a></span>).
 The DNA-binding region and the subunit interaction region are 
indicated. Numbers indicate amino acid positions of the B domains. The <em>Xenopus</em> sequence was derived from an incomplete cDNA clone.<!--Comment--></span><!--caption--></p><a style="display: none;" class="linkText" title="View thumbnail images" href="#gr4">View thumbnail images</a><p><a onclick="openStrippedNS('/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0092867400814634&amp;_image=1-s2.0-S0092867400814634-gr4_lrg.jpg&amp;_ba=&amp;_rdoc=1&amp;_fmt=full&amp;_orig=na&amp;_issn=00928674&amp;_pii=S0092867400814634&amp;view=c&amp;_isHiQual=Y&amp;_acct=C000228598&amp;_version=1&amp;_urlVersion=0&amp;_userid=10&amp;md5=e0439ac5b1e18f2682fa9c5c1d8a2460','labelCaptionFIG4','BIB21','S0092867400814634'); return false;" title="View high quality image - Opens in new Window" href="#gr4">View high quality image (253K)</a></p></dd></dl></div></div><h3 class="svArticle" id="section_lbl2.4"><em>LEC1</em> Is Expressed Specifically within Seeds during Early and Late Embryogenesis</h3><p class="svArticle section" id="">We analyzed <em>LEC1</em> RNA levels to determine the expression pattern of the gene during development. <span id="bFIG3"><a href="#FIG3" id="ancbFIG3" class="intra_ref">Figure 3</a></span>C shows that <em>LEC1</em>
 RNA was present in developing siliques of wild-type plants (lane 3) but
 was not detected in the leaves and stems of 2-week-old vegetative 
plants (lane 4). <em>LEC1</em> RNA was also not detected in gel blot 
hybridization experiments with polyadenylated RNAs from wild-type 
leaves, stems, roots, and flower buds (data not shown). Our previous 
observation that <em>lec1</em> mutant embryos rescued before desiccation
 occurred produce homozygous mutant plants with no obvious vegetative 
abnormalities is consistent with this apparent seed-specific expression 
pattern (<span id="bFIG1"><a href="#FIG1" id="ancbFIG1" class="intra_ref">Figure 1</a></span>; <span id="bBIB53"><a href="#BIB53" id="ancbBIB53" class="intra_ref">West et al. 1994</a></span>).</p><p class="svArticle section" id="">To define when the <em>LEC1</em> gene was active during embryogenesis, we measured <em>LEC1</em> RNA levels in siliques at different stages of development. As shown in <span id="bFIG3"><a href="#FIG3" id="ancbFIG3" class="intra_ref">Figure 3</a></span>D, <em>LEC1</em>
 RNA was present at higher levels in siliques containing early 
preglobular to heart-stage embryos (lane 1) and heart to curled 
cotyledon–stage embryos (lane 2) than in maturing embryos (lane 3). 
These results suggest that the <em>LEC1</em> gene is expressed at highest levels during early embryo development.</p><p class="svArticle section" id="">We hybridized a <em>LEC1</em> probe in situ with developing seed sections to determine the distribution of <em>LEC1</em> RNA within the seed. <span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>A and <span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>E show that <em>LEC1</em> RNA accumulated in both the embryo proper and the suspensor of an early eight-celled proembryo. Other experiments showed that <em>LEC1</em> RNA was present shortly after fertilization within a two-celled proembryo and its suspensor (data not shown). <em>LEC1</em> RNA was present at a higher level in early-stage embryos at the proembryo stage, globular stage (<span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>B and <span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>F), transition stage (<span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>C and <span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>G), heart stage (<span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>D and <span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>H), torpedo stage (<span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>I and <span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>M), and curled cotyledon stage (<span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>J and <span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>N) than in late maturing-stage embryos (<span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>K and <span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>O). This result was consistent with the RNA gel blot studies (<span id="bFIG3"><a href="#FIG3" id="ancbFIG3" class="intra_ref">Figure 3</a></span>D). Beginning with the globular and transition stages (<span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>F and <span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>G), <em>LEC1</em>
 RNA became restricted to the embryo periphery, primarily within the 
outer protoderm and the ground tissue cell layers. By contrast, the 
extent of hybridization was much less in the procambial tissue at the 
center of the embryo proper (<span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>G and <span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>H). The <em>LEC1</em>
 RNA accumulation pattern changed gradually during the progression from 
the linear to curled cotyledon stages when the RNA became distributed 
throughout the embryo (<span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>N).</p><div class="figTblUpiOuter svArticle" id="figure_FIG5"><div class="&#10;                    textboxdefault&#10;                "><dl class="figure" id="FIG5"><dt style="height:891px;width:640px;" class="autoScroll" data-style="height:891px;width:640px;"><a onclick="openStrippedNS('/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0092867400814634&amp;_image=1-s2.0-S0092867400814634-gr5.jpg&amp;_ba=&amp;_rdoc=1&amp;_fmt=full&amp;_orig=na&amp;_issn=00928674&amp;_pii=S0092867400814634&amp;view=c&amp;_acct=C000228598&amp;_version=1&amp;_urlVersion=0&amp;_userid=10&amp;md5=797310b36b4ed3365f1a86726255546f','labelCaptionFIG5','','S0092867400814634');  return false;" href="#gr5" title="Full-size image (259K)"><img src="S0092867400814634_files/1-s2_004.gif" data-fullsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr5.jpg" data-thumbsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr5.sml" alt="Full-size image" class="figure large" border="0"/><noscript class="figure large" border="0" alt="Full-size image" data-thumbsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr5.sml" data-fullsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr5.jpg"><img class="figure large" border="0" alt="Full-size image (259K)" src="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr5.jpg" data-thumbsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr5.sml" data-fullsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr5.jpg"/></noscript></a></dt><dd id="labelCaptionFIG5"><p class="caption">Figure 5. <span id="">Distribution of <em>LEC1</em> mRNA in Developing Embryos<!--Comment--></span><span id="">Wild-type and <em>lec1-1</em> mutant seed sections were hybridized with a <em>LEC1</em>
 antisense RNA probe and photographed following autoradiography. 
Sections (A)–(D) and (I)–(L) were stained with toluidine blue and 
photographed. Sections (E)–(H) and (M)–(P) were photographed with 
darkfield optics. As a negative control, the <em>LEC1</em> probe was hybridized to seed sections of <em>lec1-1</em> mutants. Hybridization with a <em>LEC1</em>
 sense RNA probe did not yield detectable signals (data not shown). 
Bars, 50 μm. a, embryonic axis; c, cotyledon; en, endosperm; ep, embryo 
proper; s, suspensor.<!--Comment--></span><span id="">(A and E) Preglobular embryo.<!--Comment--></span><span id="">(B and F) Globular-stage embryo.<!--Comment--></span><span id="">(C and G) Transition-stage embryo.<!--Comment--></span><span id="">(D and H) Heart-stage embryo.<!--Comment--></span><span id="">(I and M) Torpedo/linear cotyledon–stage embryo.<!--Comment--></span><span id="">(J and N) Bent cotyledon–stage embryo.<!--Comment--></span><span id="">(K and O) Maturing embryo.<!--Comment--></span><span id="">(L and P) <em>lec1-1</em> mutant heart–stage embryo.<!--Comment--></span><!--caption--></p><a style="display: none;" class="linkText" title="View thumbnail images" href="#gr5">View thumbnail images</a><p><a onclick="openStrippedNS('/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0092867400814634&amp;_image=1-s2.0-S0092867400814634-gr5_lrg.jpg&amp;_ba=&amp;_rdoc=1&amp;_fmt=full&amp;_orig=na&amp;_issn=00928674&amp;_pii=S0092867400814634&amp;view=c&amp;_isHiQual=Y&amp;_acct=C000228598&amp;_version=1&amp;_urlVersion=0&amp;_userid=10&amp;md5=3c1b85ef984b15bc7f35a97b2a4e8db7','labelCaptionFIG5','','S0092867400814634'); return false;" title="View high quality image - Opens in new Window" href="#gr5">View high quality image (381K)</a></p></dd></dl></div></div><p class="svArticle section" id="">As shown in <span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>H, <span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>M, and <span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>N, <em>LEC1</em>
 RNA also accumulated in the endosperm, a triploid nonembryonic seed 
tissue that originates from the fertilization of the central cell of a 
female gametophyte with a sperm nucleus. <span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span> also shows that <em>LEC1</em> RNA was not detected in maternally derived silique and seed coat tissues, indicating that the <em>LEC1</em> RNA detected in silique RNA gel blots was present primarily within the seed (<span id="bFIG3"><a href="#FIG3" id="ancbFIG3" class="intra_ref">Figure 3</a></span>C and <span id="bFIG3"><a href="#FIG3" id="ancbFIG3" class="intra_ref">Figure 3</a></span>D). Furthermore, <span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>L and <span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>P show that the hybridization reactions were specific for <em>LEC1</em> RNA; no appreciable hybridization was observed within an early-stage seed of the <em>lec1-1</em> null mutant. The pattern of <em>LEC1</em> gene expression and the <em>lec1</em> mutant phenotype suggest strongly that the <em>LEC1</em> gene functions specifically during embryogenesis, including the earliest embryonic period.</p><h3 class="svArticle" id="section_lbl2.5"><em>LEC1</em> Is Sufficient to Induce Embryonic Pathways</h3><p class="svArticle section" id="">Because
 LEC1 appears to function as a specific regulator of embryo development,
 we wanted to know whether ectopic expression of the <em>LEC1</em> gene after embryogenesis affected vegetative development. We transferred a <em>LEC1</em> cDNA clone under the control of the cauliflower mosaic virus <em>35S</em> promoter into <em>lec1-1</em> null mutants. The <em>35S</em> promoter is active at a high level in most plant tissues (<span id="bBIB34"><a href="#BIB34" id="ancbBIB34" class="intra_ref">Odell et al. 1985</a></span>).</p><p class="svArticle section" id="">We obtained viable, desiccated T1 seeds from <em>lec1-1</em> mutants transformed in planta with the 35S/<em>LEC1</em> construct. This result showed that the transgene complemented the mutation because <em>lec1</em> mutant seeds are intolerant of desiccation (<span id="bFIG1"><a href="#FIG1" id="ancbFIG1" class="intra_ref">Figure 1</a></span>).
 However, viable seed production was a relatively rare event; T1 seeds 
germinated with an efficiency of only 0.006%, much less than the 1% 
efficiency typically obtained from in planta transformation experiments.
 Three-fourths of the seeds that germinated (33 of 43) produced T1 
seedlings with abnormal terminal morphologies as shown in <span id="bFIG6"><a href="#FIG6" id="ancbFIG6" class="intra_ref">Figure 6</a></span>B and <span id="bFIG6"><a href="#FIG6" id="ancbFIG6" class="intra_ref">Figure 6</a></span>C. These 35S/<em>LEC1</em> seedlings were smaller than wild-type seedlings (<span id="bFIG6"><a href="#FIG6" id="ancbFIG6" class="intra_ref">Figure 6</a></span>A),
 and they possessed cotyledons that remained fleshy and failed to 
expand. Their roots often did not extend or extended only in sections 
and sometimes greened. As shown in <span id="bFIG6"><a href="#FIG6" id="ancbFIG6" class="intra_ref">Figure 6</a></span>C, /<em>LEC1</em>
 seedlings sometimes produced a single pair of cotyledon-like organs on 
the shoot apex at positions normally occupied by leaves. Unlike 
wild-type leaves, these organs did not expand and did not possess 
trichomes or mature stomatal structures (<span id="bFIG6"><a href="#FIG6" id="ancbFIG6" class="intra_ref">Figure 6</a></span>C; data not shown). Morphologically, these organs closely resembled embryonic cotyledons.</p><div class="figTblUpiOuter svArticle" id="figure_FIG6"><div class="&#10;                    textboxdefault&#10;                "><dl class="figure" id="FIG6"><dt style="height:646px;width:640px;" class="autoScroll" data-style="height:646px;width:640px;"><a onclick="openStrippedNS('/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0092867400814634&amp;_image=1-s2.0-S0092867400814634-gr6.jpg&amp;_ba=&amp;_rdoc=1&amp;_fmt=full&amp;_orig=na&amp;_issn=00928674&amp;_pii=S0092867400814634&amp;view=c&amp;_acct=C000228598&amp;_version=1&amp;_urlVersion=0&amp;_userid=10&amp;md5=2aa6fa7b715c6de83158ab7c7e67baa8','labelCaptionFIG6','','S0092867400814634');  return false;" href="#gr6" title="Full-size image (85K)"><img src="S0092867400814634_files/1-s2_002.gif" data-fullsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr6.jpg" data-thumbsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr6.sml" alt="Full-size image" class="figure large" border="0"/><noscript class="figure large" border="0" alt="Full-size image" data-thumbsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr6.sml" data-fullsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr6.jpg"><img class="figure large" border="0" alt="Full-size image (85K)" src="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr6.jpg" data-thumbsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr6.sml" data-fullsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr6.jpg"/></noscript></a></dt><dd id="labelCaptionFIG6"><p class="caption">Figure 6. <span id=""><em>35S/LEC1</em> Seedlings Have Embryonic Characteristics<!--Comment--></span><span id="">(A) Comparison of a wild-type seedling grown for 2 weeks (left) with two <em>35S/LEC1</em> seedlings (right) that are shown at a higher magnification in (B) and (C).<!--Comment--></span><span id="">(B) <em>35S/LEC1</em> seedling grown for 4 weeks that germinated but did not continue to develop.<!--Comment--></span><span id="">(C) Two-week-old <em>35S/LEC1</em> seedling that produced a pair of cotyledon-like organs at the shoot apex.<!--Comment--></span><span id="">(D–F) Darkfield micrographs of <em>35S/LEC1</em> seedling sections hybridized with probes for the following RNAs: (D) cruciferin A storage protein; (E) oleosin; (F) <em>LEC1</em>.<!--Comment--></span><span id="">c, cotyledon; cl, cotyledon-like organ; l, leaf; r, root. Bars, 1 mm for (A)–(C) and 0.1 mm for (D)–(F).<!--Comment--></span><!--caption--></p><a style="display: none;" class="linkText" title="View thumbnail images" href="#gr6">View thumbnail images</a><p><a onclick="openStrippedNS('/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0092867400814634&amp;_image=1-s2.0-S0092867400814634-gr6_lrg.jpg&amp;_ba=&amp;_rdoc=1&amp;_fmt=full&amp;_orig=na&amp;_issn=00928674&amp;_pii=S0092867400814634&amp;view=c&amp;_isHiQual=Y&amp;_acct=C000228598&amp;_version=1&amp;_urlVersion=0&amp;_userid=10&amp;md5=0d6e82a09e99dc920bbf50d702497b20','labelCaptionFIG6','','S0092867400814634'); return false;" title="View high quality image - Opens in new Window" href="#gr6">View high quality image (253K)</a></p></dd></dl></div></div><p class="svArticle section" id="">Ten of the T1 35S/<em>LEC1</em>
 seedlings produced plants that grew vegetatively. One plant was male 
sterile and did not produce progeny. Seven other plants flowered and 
produced T2 progeny that all displayed the Lec1<sup>−</sup> phenotype though PCR amplification experiments confirmed the presence of the <em>35S/LEC1</em> transgene (data not shown). This result suggested that the 35S/<em>LEC1</em> gene was initially active in developing T1 seeds to complement the <em>lec1</em> mutation but that the gene became inactive after germination. Others have observed transgene silencing in plants ( <a href="#BIB25" id="bBIB25" class="intra_ref">
                  [25]</a> and <a href="#BIB45" id="bBIB45" class="intra_ref">[45]</a>).
 Two of the ten plants that grew vegetatively were exceptional in that 
they produced progeny with variable phenotypes. Only 25% of total 
desiccated seed from one plant, 20-3, was able to germinate, and all 
seedlings initially recapitulated the embryo-like seedling phenotypes 
shown in <span id="bFIG6"><a href="#FIG6" id="ancbFIG6" class="intra_ref">Figure 6</a></span>B and <span id="bFIG6"><a href="#FIG6" id="ancbFIG6" class="intra_ref">Figure 6</a></span>C.
 All T2 progeny from the second, independently derived line, 21-4, also 
exhibited the embryo-like seedling phenotypes shown in <span id="bFIG6"><a href="#FIG6" id="ancbFIG6" class="intra_ref">Figure 6</a></span>B and <span id="bFIG6"><a href="#FIG6" id="ancbFIG6" class="intra_ref">Figure 6</a></span>C. Thus, the ability of <em>35S/LEC1</em> plants to produce seedlings with an embryonic morphology was heritable to the T2 generation. T2 <em>35S/LEC1</em> plants that were fertile only produced progeny with a Lec1<sup>−</sup> phenotype.</p><p class="svArticle section" id="">Because the <em>35S/LEC1</em>
 seedlings had embryonic morphological characteristics, we asked whether
 they express genes normally active only in developing seeds. <span id="bFIG6"><a href="#FIG6" id="ancbFIG6" class="intra_ref">Figure 6</a></span>D shows that cruciferin A storage protein RNA accumulated throughout 35S/<em>LEC1</em>
 T2 seedlings displaying embryonic characteristics, including the 
cotyledon-like organs at the position of leaves. Similar results were 
obtained in experiments with independently derived T1 embryo-like 
seedlings. We also showed that other embryo-specific RNAs encoding 
oleosin, an oil body protein (<span id="bFIG6"><a href="#FIG6" id="ancbFIG6" class="intra_ref">Figure 6</a></span>E), and two 2S storage proteins (data not shown) accumulated similarly in these embryo-like seedlings. We confirmed that <em>LEC1</em> RNA accumulated in these 35S/<em>LEC1</em> seedlings (<span id="bFIG6"><a href="#FIG6" id="ancbFIG6" class="intra_ref">Figure 6</a></span>F) but not in wild-type seedlings (data not shown; <span id="bFIG3"><a href="#FIG3" id="ancbFIG3" class="intra_ref">Figure 3</a></span>C). Thus, <em>35S/LEC1</em> seedlings displaying an embryo-like phenotype accumulated embryo-specific RNAs. Together, these results suggest that ectopic <em>LEC1</em> gene expression induces embryonic programs in vegetative cells.</p><p class="svArticle section" id="">Of the T2 <em>35S/LEC1</em> seedlings that displayed embryonic characteristics (<span id="bFIG6"><a href="#FIG6" id="ancbFIG6" class="intra_ref">Figure 6</a></span>B and <span id="bFIG6"><a href="#FIG6" id="ancbFIG6" class="intra_ref">Figure 6</a></span>C),
 most of the progeny from one line, 21-4, and approximately 10% from the
 second line, 20-3, continued to grow vegetatively, unlike the T1 
seedlings that were arrested developmentally. These T2 plants displayed 
morphological abnormalities ranging from plants shown in <span id="bFIG7"><a href="#FIG7" id="ancbFIG7" class="intra_ref">Figure 7</a></span>A
 with multiple embryonic cotyledon-like organs to plants with small, 
dark green abnormally shaped leaves that often produced callus-like 
cells. Immunochemical analysis showed that these plants accumulated 
cruciferin storage protein (data not shown), suggesting that these 
vegetatively growing plants expressed embryonic programs.</p><div class="figTblUpiOuter svArticle" id="figure_FIG7"><div class="&#10;                    textboxdefault&#10;                "><dl class="figure" id="FIG7"><dt style="height:497px;width:640px;" class="autoScroll" data-style="height:497px;width:640px;"><a onclick="openStrippedNS('/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0092867400814634&amp;_image=1-s2.0-S0092867400814634-gr7.jpg&amp;_ba=&amp;_rdoc=1&amp;_fmt=full&amp;_orig=na&amp;_issn=00928674&amp;_pii=S0092867400814634&amp;view=c&amp;_acct=C000228598&amp;_version=1&amp;_urlVersion=0&amp;_userid=10&amp;md5=f9022b0c1686ed9b0bcf869449bdb1e8','labelCaptionFIG7','','S0092867400814634');  return false;" href="#gr7" title="Full-size image (87K)"><img src="S0092867400814634_files/1-s2_005.gif" data-fullsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr7.jpg" data-thumbsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr7.sml" alt="Full-size image" class="figure large" border="0"/><noscript class="figure large" border="0" alt="Full-size image" data-thumbsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr7.sml" data-fullsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr7.jpg"><img class="figure large" border="0" alt="Full-size image (87K)" src="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr7.jpg" data-thumbsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr7.sml" data-fullsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0092867400814634-gr7.jpg"/></noscript></a></dt><dd id="labelCaptionFIG7"><p class="caption">Figure 7. <span id="">Embryo-like Structures on Transgenic Plants Ectopically Expressing the <em>LEC1</em> Gene<!--Comment--></span><span id="">(A) <em>35S/LEC1</em> seedling that grew vegetatively and produced multiple cotyledon-like organs.<!--Comment--></span><span id="">(B) Embryo-like structures on the leaf of a <em>35S/LEC1</em> plant that grew vegetatively.<!--Comment--></span><span id="">(C) Axes of embryo-like structures that “germinated” to produce roots.<!--Comment--></span><span id="">(D and E) SEM analysis of embryo-like structures. Structures resemble fused cotyledon-stage embryos with multiple cotyledons.<!--Comment--></span><span id="">(F) SEM of wild-type cotyledon-stage embryo.<!--Comment--></span><span id="">a, axis; c, cotyledon; l, leaf; r, root. Bars, 1 mm (A and C), 0.5 mm (B), 0.1 mm (D and E), and 0.05 mm (F).<!--Comment--></span><!--caption--></p><a style="display: none;" class="linkText" title="View thumbnail images" href="#gr7">View thumbnail images</a><p><a onclick="openStrippedNS('/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0092867400814634&amp;_image=1-s2.0-S0092867400814634-gr7_lrg.jpg&amp;_ba=&amp;_rdoc=1&amp;_fmt=full&amp;_orig=na&amp;_issn=00928674&amp;_pii=S0092867400814634&amp;view=c&amp;_isHiQual=Y&amp;_acct=C000228598&amp;_version=1&amp;_urlVersion=0&amp;_userid=10&amp;md5=cb4be36276e8caa0e5174d60579735c2','labelCaptionFIG7','','S0092867400814634'); return false;" title="View high quality image - Opens in new Window" href="#gr7">View high quality image (256K)</a></p></dd></dl></div></div><p class="svArticle section" id="">A striking phenotype of the T2 progeny is shown in <span id="bFIG7"><a href="#FIG7" id="ancbFIG7" class="intra_ref">Figure 7</a></span>B. We discovered embryo-like structures on the leaves of three progeny plants from the two independently derived <em>35S/LEC1</em> lines. As shown in <span id="bFIG7"><a href="#FIG7" id="ancbFIG7" class="intra_ref">Figure 7</a></span>D, these structures resembled fused wild-type cotyledon-stage embryos (<span id="bFIG7"><a href="#FIG7" id="ancbFIG7" class="intra_ref">Figure 7</a></span>F).
 Multiple embryonic cotyledon-like organs that lacked trichomes and 
mature stomata were attached to structures that resembled embryonic axes
 with elongated cells typical of wild-type embryos (<span id="bFIG7"><a href="#FIG7" id="ancbFIG7" class="intra_ref">Figure 7</a></span>E;
 data not shown). Histological analyses suggested that, like wild-type 
embryos, these embryo-like structures possessed an outer protoderm 
layer, a central procambium layer, and ground tissue that consisted of 
several distinct files of cells (data not shown). Similar to embryos, 
the ground tissue cells of the ectopic embryos were densely cytoplasmic.
 <span id="bFIG7"><a href="#FIG7" id="ancbFIG7" class="intra_ref">Figure 7</a></span>C
 shows roots that emerged from the axes tips, suggesting that these axes
 share similar functions with wild-type embryos. Finally, in situ 
hybridization experiments showed that embryo-specific RNAs encoding 
cruciferin A and 2S-1 storage proteins accumulated in these embryo-like 
structures but were not detected in the underlying leaf cells (data not 
shown). We conclude that postembryonic expression of the <em>LEC1</em> gene is sufficient to induce embryo formation in vegetative tissues of these two lines.</p><h2 class="svArticle" id="section_lbl3">Discussion</h2><h3 class="svArticle" id="section_lbl3.1">LEC1 Is a Transcriptional Activator Homolog</h3><p class="svArticle section" id="">We
 have shown that the LEC1 polypeptide is homologous to the HAP3 subunit 
of the CBF class of eukaryotic transcriptional activators that includes 
NF-Y, CP1, and HAP2/3/4/5 (<span id="bBIB17"><a href="#BIB17" id="ancbBIB17" class="intra_ref">Johnson and McKnight 1989</a></span>).
 The sequence similarity between LEC1 and other HAP3 subunits is 
restricted to the B domain, consistent with the finding that this domain
 is conserved evolutionarily (<span id="bBIB21"><a href="#BIB21" id="ancbBIB21" class="intra_ref">Li et al. 1992</a></span>).
 Furthermore, amino acid residues of yeast and mammalian HAP3 subunits 
required for DNA binding and for interactions with other CBF subunits 
are conserved in LEC1 (<span id="bFIG4"><a href="#FIG4" id="ancbFIG4" class="intra_ref">Figure 4</a></span>; <a href="#BIB55" id="bBIB55" class="intra_ref">
                  [55]</a> and <a href="#BIB44" id="bBIB44" class="intra_ref">[44]</a>).
 Experiments demonstrating that yeast and mammalian CBF subunits can be 
combined to form DNA-binding complexes indicate that this amino acid 
sequence similarity underlies functional conservation (<a href="#BIB6" id="bBIB6" class="intra_ref">
                  [6]</a> and <a href="#BIB43" id="bBIB43" class="intra_ref">[43]</a>).</p><p class="svArticle section" id="">CBFs
 are heteroligomeric transcription factors, but it is not known whether 
the plant CBF is organized as its yeast counterpart into four 
nonhomologous subunits, HAP2, 3, 4, and 5, or is similar to the trimeric
 mammalian CBF (<a href="#BIB24" id="bBIB24" class="intra_ref">
                  [24]</a> and <a href="#BIB27" id="bBIB27" class="intra_ref">[27]</a>). Oilseed rape, maize, and <em>Arabidopsis</em>
 DNA sequences encoding HAP2, HAP3, and HAP5 homologs have been 
identified, although the functional roles of the plant proteins have not
 been established ( <a href="#BIB21" id="bBIB21" class="intra_ref">
                  [21]</a>, <a href="#BIB33" id="bBIB33" class="intra_ref">
                  [33]</a> and <a href="#BIB1" id="bBIB1" class="intra_ref">[1]</a>). Several distinct <em>Arabidopsis</em>
 DNA sequences have been identified that correspond to each of these 
subunits, including the LEC1/HAP3 subunit, implicating the existence of 
gene families.</p><p class="svArticle section" id="">Because LEC1 is a 
component of a plant CBF, we predict that it regulates embryonic 
processes by activating the transcription of specific genes. Mammalian 
CBFs are thought to serve a general role in transcription by optimizing 
promoter efficiency through its binding with CCAAT DNA sequences that 
are found 50–100 bp upstream of many mammalian genes (<span id="bBIB31"><a href="#BIB31" id="ancbBIB31" class="intra_ref">Myers et al. 1986</a></span>).
 However, some mammalian CBFs have been shown to increase the 
transcriptional activities of specific genes by their association with 
other transcription factors (<a href="#BIB54" id="bBIB54" class="intra_ref">
                  [54]</a> and <a href="#BIB10" id="bBIB10" class="intra_ref">[10]</a>).
 The CBF containing LEC1 is unlikely to serve a general role in 
transcription for several reasons. First, CBFs in other organisms 
regulate specific gene sets. For example, yeast CBFs specifically 
activate genes encoding mitochondrial proteins involved in respiration (<a href="#BIB14" id="bBIB14" class="intra_ref">
                  [14]</a>, <a href="#BIB19" id="bBIB19" class="intra_ref">
                  [19]</a>, <a href="#BIB48" id="bBIB48" class="intra_ref">
                  [48]</a> and <a href="#BIB40" id="bBIB40" class="intra_ref">[40]</a>).
 Second, CCAAT boxes are not typically found upstream of most plant 
genes near position −80, and mutation of the CCAAT sequence within the 
35S promoter does not affect promoter activity or footprint formation (<span id="bBIB5"><a href="#BIB5" id="ancbBIB5" class="intra_ref">Benfey and Chua 1990</a></span>). Third, <em>lec1</em>
 null mutations, although pleiotropic, do not abrogate the transcription
 of many genes, including those encoding cruciferin A, oleosin, and late
 embryogenesis abundant proteins (<span id="bBIB53"><a href="#BIB53" id="ancbBIB53" class="intra_ref">West et al. 1994</a></span>). Thus, the Lec1<sup>−</sup>
 phenotype most likely results from abnormal transcription of specific 
genes regulated by the CBF containing LEC1. Finally, expression of the 
35S/<em>LEC1</em> gene in postembryonic plants induces embryonic 
processes, indicating that specific gene sets required for embryo 
development are activated by LEC1 (<span id="bFIG6"><a href="#FIG6" id="ancbFIG6" class="intra_ref">Figure 6</a></span> and <span id="bFIG7"><a href="#FIG7" id="ancbFIG7" class="intra_ref">Figure 7</a></span>). We conclude that LEC1 is a specific transcriptional regulator of genes required for normal <em>Arabidopsis</em> embryo development.</p><h3 class="svArticle" id="section_lbl3.2">LEC1 Is a Central Regulator of Embryogenesis</h3><p class="svArticle section" id="">A
 key to define the precise role of LEC1 in embryo development is to 
understand whether LEC1 functions throughout embryogenesis or 
specifically during either the morphogenesis phase or the maturation 
phase. We and others speculated previously that LEC1 might function 
solely during morphogenesis as a homeotic regulator of cotyledon 
identity (<a href="#BIB28" id="bBIB28" class="intra_ref">
                  [28]</a> and <a href="#BIB53" id="bBIB53" class="intra_ref">[53]</a>). Cotyledons and leaves are homologous organs. Incomplete specification of organ identity resulting from the <em>lec1</em> mutation could cause cotyledons to acquire leaf characteristics and defects in seed maturation (<span id="bFIG1"><a href="#FIG1" id="ancbFIG1" class="intra_ref">Figure 1</a></span>). However, <em>LEC1</em> RNA is distributed throughout the embryo and in the endosperm but not exclusively in developing cotyledons (<span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>). These results suggest that <em>LEC1</em>
 does not act solely in the specification of cotyledon identity. An 
alternative hypothesis is that LEC1 might act exclusively to regulate 
the maturation phase of embryogenesis ( <a href="#BIB53" id="bBIB53" class="intra_ref">
                  [53]</a> and <a href="#BIB39" id="bBIB39" class="intra_ref">[39]</a>). Because germination is actively suppressed during embryo development (<span id="bBIB16"><a href="#BIB16" id="ancbBIB16" class="intra_ref">Harada 1997</a></span>),
 defects in maturation processes are expected to cause premature 
activation of postgerminative development. Thus, the phenotype of <em>lec1</em> mutants (<span id="bFIG1"><a href="#FIG1" id="ancbFIG1" class="intra_ref">Figure 1</a></span>) could result from the heterochronic effects of the mutation. Our finding that <em>LEC1</em> is expressed early in embryogenesis and is required for early embryo development (<span id="bFIG2"><a href="#FIG2" id="ancbFIG2" class="intra_ref">Figure 2</a></span> and <span id="bFIG3"><a href="#FIG3" id="ancbFIG3" class="intra_ref">Figure 3</a></span>)
 indicates that LEC1 functions before the onset of maturation and, 
therefore, cannot be involved only in regulating this late embryonic 
phase. The simplest interpretation is that LEC1 plays a more central 
role in embryo development.</p><p class="svArticle section" id="">The ability of <em>LEC1</em>
 to induce embryonic programs in vegetative cells establishes the gene 
as a critical regulator of embryogenesis. The seed maturation phase of 
embryogenesis is induced, at least in part, in seedlings expressing the 
gene as indicated by the activation of cruciferin A and 2S storage 
protein genes and the oleosin gene (<span id="bFIG6"><a href="#FIG6" id="ancbFIG6" class="intra_ref">Figure 6</a></span>). The failure of <em>35S/LEC1</em> seedlings with embryonic characteristics (<span id="bFIG6"><a href="#FIG6" id="ancbFIG6" class="intra_ref">Figure 6</a></span>)
 to continue vegetative development is also consistent with induction of
 maturation because morphogenesis is normally arrested during the seed 
maturation phase (<span id="bBIB16"><a href="#BIB16" id="ancbBIB16" class="intra_ref">Harada 1997</a></span>). Because the <em>LEC1</em>
 gene is not normally expressed postembryonically, this result suggests 
that LEC1 may activate genes that suppress vegetative development. Our 
previous finding that postgerminative development is activated 
prematurely in <em>lec1</em> mutant embryos is consistent with this intepretation (<span id="bBIB53"><a href="#BIB53" id="ancbBIB53" class="intra_ref">West et al. 1994</a></span>).
 Alternatively, LEC1 may interfere with interactions between 
transcription factors in seedlings, creating a dominant-negative 
mutation that disrupts vegetative development. The effect of <em>LEC1</em> gene expression on postembryonic development may explain the low germination frequency of <em>35S/LEC1</em> T1 seeds and the rarity with which we recovered vegetative plants with embryonic structures. The <em>35S/LEC1</em> transgene must be active in the developing <em>lec1</em> mutant seeds to permit the completion of embryo development, yet the activity of the <em>LEC1</em> gene is antagonistic to vegetative development. Thus, continued overexpression of the <em>35S/LEC1</em>
 gene following germination is likely to inhibit vegetative development,
 and silencing of the transgene probably produced plants with the Lec1<sup>−</sup> phenotype. Because transgene activity is highly variable in independent transformants ( <a href="#BIB25" id="bBIB25" class="intra_ref">
                  [25]</a> and <a href="#BIB45" id="bBIB45" class="intra_ref">[45]</a>),
 only those rare transgenic plants producing LEC1 protein at a critical 
level may be competent to produce vegetative plants with embryonic 
structures. Together, these results indicate that LEC1 is sufficient to 
induce many aspects of the maturation phase of embryogenesis in 
vegetative cells.</p><p class="svArticle section" id="">The formation of embryo-like structures on the leaves of <em>35S/LEC1</em> plants (<span id="bFIG7"><a href="#FIG7" id="ancbFIG7" class="intra_ref">Figure 7</a></span>)
 strongly suggests that LEC1 is sufficient to induce the morphogenesis 
phase of embryo development, although it remains to be determined how 
closely ectopic embryo formation follows zygotic embryogenesis. 
Additional evidence that ectopic <em>LEC1</em> gene expression induces 
embryonic structures comes from preliminary experiments showing that 
embryonic cotyledon-like structures form on seedlings ectopically 
expressing the <em>LEC1</em> gene from a different promoter (R. W. K. and J. J. H., unpublished results). Our findings that the <em>lec1</em> mutation causes defects in suspensor morphology in early-stage embryos (<span id="bFIG2"><a href="#FIG2" id="ancbFIG2" class="intra_ref">Figure 2</a></span>) and that the <em>LEC1</em> gene is expressed at the earliest stages of embryo development (<span id="bFIG5"><a href="#FIG5" id="ancbFIG5" class="intra_ref">Figure 5</a></span>) provide independent evidence of <em>LEC1</em> function early in embryogenesis. This conclusion is also consistent with genetic analyses of <em>lec1 lec2</em> double mutants. Although <em>lec1</em> and <em>lec2</em>
 mutants at their terminal stages resemble late-stage embryos, the 
double mutant arrests with the morphology of an early torpedo-stage 
embryo, suggesting a role for both genes in early embryogenesis (data 
not shown; <span id="bBIB29"><a href="#BIB29" id="ancbBIB29" class="intra_ref">Meinke et al. 1994</a></span>).</p><p class="svArticle section" id="">The
 ability of LEC1 to induce both the morphogenesis and maturation phases 
of embryogenesis and to suppress vegetative development suggests its 
fundamental role in regulating different aspects of embryogenesis. As 
discussed above, other genes shown to be required for embryo development
 do not function seed-specifically throughout embryogenesis. <em>LEC1</em> is the only gene shown to be sufficient to induce embryo formation in vegetatively growing plants. For example, <em>ABI3</em>
 has been implicated to play a critical role in controlling the late 
seed maturation phase of embryogenesis. Transgenic plants containing a <em>35S/ABI3</em> gene express a subset of genes normally active during seed maturation when challenged with the hormone ABA (<span id="bBIB38"><a href="#BIB38" id="ancbBIB38" class="intra_ref">Parcy et al. 1994</a></span>). Ectopic <em>ABI3</em>
 gene expression does not cause visible defects in vegetative or 
reproductive development in these transgenic plants, suggesting that 
ABI3 is not sufficient to induce the maturation phase. This result 
suggests that, unlike <em>LEC1</em>, the role of <em>ABI3</em> is more limited to regulating gene sets expressed during seed maturation. Double mutant analyses indicating that <em>LEC1</em> and <em>ABI3</em> do not appear to act in series in the same genetic pathway are consistent with this interpretation (data not shown; <span id="bBIB29"><a href="#BIB29" id="ancbBIB29" class="intra_ref">Meinke et al. 1994</a></span>). We note that it has recently been reported that embryonic programs are induced in vegetative root cells of the <em>pickle</em> mutant (<span id="bBIB35"><a href="#BIB35" id="ancbBIB35" class="intra_ref">Ogas et al. 1997</a></span>).</p><p class="svArticle section" id="">Considering that LEC1 can induce embryo development in vegetative cells, why is the Lec1<sup>−</sup> phenotype not more severe? Although <em>lec1</em> mutant embryos are intolerant of desiccation, they continue to undergo many aspects of morphogenesis and seed maturation (see <span id="bFIG1"><a href="#FIG1" id="ancbFIG1" class="intra_ref">Figure 1</a></span>). A clue comes from analyses of cruciferin A, oleosin, and 2S storage protein gene expression. These genes are expressed in <em>35S/LEC1</em> seedlings, yet they are also active in <em>lec1</em> mutant embryos (<span id="bFIG6"><a href="#FIG6" id="ancbFIG6" class="intra_ref">Figure 6</a></span>; <span id="bBIB53"><a href="#BIB53" id="ancbBIB53" class="intra_ref">West et al. 1994</a></span>).
 Thus, LEC1 is sufficient but not necessary for their expression, 
implicating genetic redundancy. That is, another gene, possibly the 
other <em>LEC</em> genes <em>LEC2</em> and/or <em>FUS3</em>, may partially fulfill LEC1 function during embryogenesis.</p><p class="svArticle section" id="">In
 conclusion, we have shown that LEC1 is a transcription factor homolog 
that is required early and late in embryo development and that it is 
sufficient to induce embryonic programs in vegetative cells. Together, 
our results suggest that LEC1 is a major embryonic regulator that 
mediates the switch between embryo and vegetative development. LEC1's 
role in inducing and maintaining embryogenesis while suppressing 
vegetative development is likely to be critical for the establishment of
 the seed habit of higher plants. Given this central role, the <em>LEC1</em>
 gene and its protein will be important tools for the dissection of 
higher plant embryogenesis. In particular, one key will be to identify 
the downstream genes regulated by LEC1 and the protein(s) with which it 
interacts in establishing the regulatory circuits controlling embryonic 
development.</p><h2 class="svArticle" id="section_lbl4">Experimental Procedures</h2><h3 class="svArticle" id="section_lbl4.1">Plant Material</h3><p class="svArticle section" id=""><em>lec1-1</em>, <em>lec1-2</em>, and <em>lec2-1</em> mutants were derived from a population of <em>Arabidopsis thaliana</em> ecotype Wassilewskija (Ws-O) lines mutagenized with T-DNA insertions ( <a href="#BIB11" id="bBIB11" class="intra_ref">
                  [11]</a> and <a href="#BIB28" id="bBIB28" class="intra_ref">[28]</a>Meinke, 1992, 1994; <span id="bBIB53"><a href="#BIB53" id="ancbBIB53" class="intra_ref">West et al. 1994</a></span>). <em>abi3-3</em> and <em>fus3-3</em> mutants and <em>lec2-1</em> were provided by Peter McCourt (University of Toronto;  <a href="#BIB32" id="bBIB32" class="intra_ref">
                  [32]</a> and <a href="#BIB18" id="bBIB18" class="intra_ref">[18]</a>) and by David Meinke (Oklahoma State University). Plants were grown as described previously (<span id="bBIB53"><a href="#BIB53" id="ancbBIB53" class="intra_ref">West et al. 1994</a></span>).
 Digenic mutants were constructed and their genotypes were verified 
through backcrosses with each parental line as described previously (<span id="bBIB53"><a href="#BIB53" id="ancbBIB53" class="intra_ref">West et al. 1994</a></span>).</p><h3 class="svArticle" id="section_lbl4.2">Isolation and Sequence Analysis of Genomic and cDNA Clones</h3><p class="svArticle section" id="">Genomic DNA libraries from homozygous <em>lec1-1</em> and <em>lec1-2</em>
 mutants were constructed in the λGEM 11 vector (Promega). A wild-type 
Ws-O genomic library was provided by Ken Feldmann (University of 
Arizona). Two cDNA libraries were prepared according to manufacturers' 
specifications in the λZAPII bacteriophage vector (Stratagene) from 
wild-type siliques containing globular- to heart-stage embryos and 
heart- to young torpedo-stage embryos.</p><p class="svArticle section" id="">Clones were isolated from the <em>lec1-2</em>
 genomic library using probes for the right and left T-DNA borders. A 
7.1 kb XhoI fragment containing the plant DNA/T-DNA junction was 
isolated and cloned into the Bluescript-KS plasmid (Stratagene) to 
create pML7. A 4.2 kb <em>EcoR</em> I fragment containing plant DNA from pML7 was used to isolate genomic clones from a wild-type Ws-0 library. A 7.4 kb <em>EcoR</em>
 I fragment present in several overlapping clones was inserted into the 
Bluescript-KS plasmid and used to identify corresponding clones from a <em>lec1-1</em> genomic library and from the wild-type silique cDNA libraries. Eighteen <em>LEC1</em> cDNA clones were isolated and entirely or partially sequenced; all sequences were identical to corresponding regions of the <em>LEC1</em>
 gene. Additional details of the cloning experiments are available upon 
request. Nucleotide sequencing was done using the automated dideoxy 
chain termination method on an ABI Prism 377 DNA Sequencer. Database 
searches were performed at the National Center for Biotechnology 
Information by using the BLAST network service. Alignment of protein 
sequences was done using PILEUP program (Genetics Computer Group, 
Madison, WI).</p><h3 class="svArticle" id="section_lbl4.3">Production of Transgenic <em>lec1</em> Mutants</h3><p class="svArticle section" id="">A 3.4 kb BstYI fragment containing the wild-type <em>LEC1</em> gene was inserted into the plant transformation vector, pBIB-Hyg (<span id="bBIB4"><a href="#BIB4" id="ancbBIB4" class="intra_ref">Becker 1990</a></span>). A full-length <em>LEC1</em>
 cDNA was fused in the proper transcriptional orientation with the 35S 
promoter and the octopine synthase terminator into the plasmid, pART7 (<span id="bBIB12"><a href="#BIB12" id="ancbBIB12" class="intra_ref">Gleave 1992</a></span>).
 The entire fusion gene was transferred into the plant transformation 
vector BJ49. Constructs were transferred into homozygous <em>lec1-1</em> and/or <em>lec1-2</em> mutants using the in planta transformation procedure with <em>Agrobacterium tumefaciens</em> strain GV3101 (<span id="bBIB3"><a href="#BIB3" id="ancbBIB3" class="intra_ref">Bechtold et al. 1993</a></span>). Genotypes of the complemented plants were verified in DNA amplification experiments. T2 plants containing the <em>35S/LEC1</em> transgene were either germinated from dry seeds or rescued from siliques and grown on basal media (<span id="bBIB36"><a href="#BIB36" id="ancbBIB36" class="intra_ref">Olsen et al. 1993</a></span>).</p><h3 class="svArticle" id="section_lbl4.4">DNA and RNA Hybridization Analyses</h3><p class="svArticle section" id="">Nucleic acid isolation and gel blot hybridization experiments were done as described previously (<span id="bBIB53"><a href="#BIB53" id="ancbBIB53" class="intra_ref">West et al. 1994</a></span>). In situ hybridization experiments were performed as described previously (<span id="bBIB8"><a href="#BIB8" id="ancbBIB8" class="intra_ref">Dietrich et al. 1989</a></span>).</p><h3 class="svArticle" id="section_lbl4.5">Scanning Electron Microscopy</h3><p class="svArticle section" id="">SEM analysis was performed as described previously (<span id="bBIB56"><a href="#BIB56" id="ancbBIB56" class="intra_ref">Yadegari et al. 1994</a></span>).</p><div class="articleText svArticle"><div class="articleText_indent"><h2 class="h2 secHeading" id="ack001">Acknowledgements</h2><p class="svArticle section" id="">We
 thank Jay Danao, Minsung Kim, and Sara Wortley for their help with 
experiments, Bart Janssen for vectors, K.C. McFarland of the NSF-Plant 
Cell Biology Training Facility for help in generating figures, and 
Neelima Sinha for comments about the manuscript. T. L. was supported by a
 postdoctoral fellowship from the Human Frontiers Science Program, M. O.
 was supported by postdoctoral fellowships for research abroad from the 
Japanese Society for the Promotion of Science and from Kenzo Nakamura, 
National Institute of Basic Biology, and K. Y. was supported in part by a
 postdoctoral fellowship from the Japanese Society for the Promotion of 
Science. This work was supported by a grant from the Department of 
Energy.</p></div></div><div id="References"><script type="text/javascript">
            prs.rt("ref_start");
            </script><div class="refText svRefs" id="bibl001"><h2 id="section_references">References</h2><ol class="references"><li id="BIB1"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB1" class="label intra_ref">1</a></li><li class="author">D Albani, L.S Robert</li><li class="title"><p>Cloning and characterization of a <em>Brassica napus</em> gene encoding a homologue of the B subunit of a heteromeric CCAAT-binding factor</p></li><li class="source"><p>Gene, 167 (1995), pp. 209–213</p></li><li class="external refPlaceHolder" id="bibsbref1"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB2"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB2" class="label intra_ref">2</a></li><li class="author">H Baumlein, S Misera, H Luerben, K Kolle, C Horstmann, U Wobus, A.J Muller</li><li class="title"><p>The <em>FUS3</em> gene of <em>Arabidopsis thaliana</em> is a regulator of gene expression during late embryogenesis</p></li><li class="source"><p>Plant J., 6 (1994), pp. 379–387</p></li><li class="external refPlaceHolder" id="bibsbref2"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB3"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB3" class="label intra_ref">3</a></li><li class="author">N Bechtold, J Ellis, G Pelletier</li><li class="title"><p><em>In planta Agrobacterium</em> mediated gene transfer by infiltration of adult <em>Arabidopsis thaliana</em> plants</p></li><li class="source"><p>C.R. Acad. Sci. Paris, 316 (1993), pp. 1194–1199</p></li><li class="external refPlaceHolder" id="bibsbref3"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB4"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB4" class="label intra_ref">4</a></li><li class="author">D Becker</li><li class="title"><p>Binary vectors which allow the exchange of plant selectable markers and reporter genes</p></li><li class="source"><p>Nucleic Acids Res., 18 (1990), p. 203</p></li><li class="external refPlaceHolder" id="bibsbref4"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB5"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB5" class="label intra_ref">5</a></li><li class="author">P.N Benfey, N.-H Chua</li><li class="title"><p>The cauliflower mosaic virus 35S promotercombinatorial regulation of transcription in plants</p></li><li class="source"><p>Science, 250 (1990), pp. 959–966</p></li><li class="external refPlaceHolder" id="bibsbref5"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB6"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB6" class="label intra_ref">6</a></li><li class="author">L.A Chodosh, J Olesen, S Hahn, A.S Baldwin, L Guarente, P.A Sharp</li><li class="title"><p>A yeast and a human CCAAT-binding protein have heterologous subunits that are functionally interchangeable</p></li><li class="source"><p>Cell, 53 (1988), pp. 25–36</p></li><li class="external refPlaceHolder" id="bibsbref6"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB7"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB7" class="label intra_ref">7</a></li><li class="author">M Devic, S Albert, M Delseny</li><li class="title"><p>Induction and expression of seed-specific promoters in Arabidopsis embryo-defective mutants</p></li><li class="source"><p>Plant J., 9 (1996), pp. 205–215</p></li><li class="external refPlaceHolder" id="bibsbref7"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB8"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB8" class="label intra_ref">8</a></li><li class="author">R.A Dietrich, D.J Maslyar, R.C Heupel, J.J Harada</li><li class="title"><p>Spatial patterns of gene expression in <em>Brassica napus</em>
 seedlingsidentification of a cortex-specific gene and localization of 
messenger RNA encoding isocitrate lyase and a polypeptide homologous to 
proteinases</p></li><li class="source"><p>Plant Cell, 1 (1989), pp. 73–80</p></li><li class="external refPlaceHolder" id="bibsbref8"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB9"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB9" class="label intra_ref">9</a></li><li class="author">L Di Laurenzio, J Wysocka-Diller, J.E Malamy, L Pysh, Y Helariutta, G Freshour, M.G Hahn, K.A Feldmann, P.N Benfey</li><li class="title"><p>The SCARECROW gene regulates an asymmetric cell division that is essential for generating the radial organization of the <em>Arabidopsis</em> root</p></li><li class="source"><p>Cell, 86 (1996), pp. 423–433</p></li><li class="external refPlaceHolder" id="bibsbref9"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB10"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB10" class="label intra_ref">10</a></li><li class="author">J Ericsson, S.M Jackson, P.A Edwards</li><li class="title"><p>Synergistic
 binding of sterol regulatory element-binding protein and NF-Y to the 
farnesyl diphosphate synthase promoter is critical for sterol-regulated 
expression of the gene</p></li><li class="source"><p>J. Biol. Chem., 271 (1996), pp. 24359–24364</p></li><li class="external refPlaceHolder" id="bibsbref10"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB11"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB11" class="label intra_ref">11</a></li><li class="author">K.A Feldmann, M.D Marks</li><li class="title"><p>Agrobacterium-mediated transformation of germinating seeds of <em>Arabidopsis thaliana</em>a non-tissue culture approach</p></li><li class="source"><p>Mol. Gen. Genet., 208 (1987), pp. 1–9</p></li><li class="external refPlaceHolder" id="bibsbref11"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB12"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB12" class="label intra_ref">12</a></li><li class="author">A.P Gleave</li><li class="title"><p>A
 versatile binary vector system with a T-DNA organisational structure 
conducive to efficient integration of cloned DNA into the plant genome</p></li><li class="source"><p>Plant Mol. Biol., 20 (1992), pp. 1203–1207</p></li><li class="external refPlaceHolder" id="bibsbref12"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB13"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB13" class="label intra_ref">13</a></li><li class="author">R.B Goldberg, G De Paiva, R Yadegari</li><li class="title"><p>Plant embryogenesiszygote to seed</p></li><li class="source"><p>Science, 266 (1994), pp. 605–614</p></li><li class="external refPlaceHolder" id="bibsbref13"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB14"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB14" class="label intra_ref">14</a></li><li class="author">L Guarente, B Lalonde, P Gifford, E Alani</li><li class="title"><p>Distinctly regulated tandem upstream activation sites mediate catabolite repression of the CYC1 genes of <em>S. cerevisiae</em></p></li><li class="source"><p>Cell, 32 (1984), pp. 317–321</p></li><li class="external refPlaceHolder" id="bibsbref14"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB15"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB15" class="label intra_ref">15</a></li><li class="author">S Hahn, J Pinkham, R Wei, R Miller, L Guarente</li><li class="title"><p>The HAP3 regulatory locus of Saccharomyces cerevisiae encodes divergent overlapping transcripts</p></li><li class="source"><p>Mol. Cell. Biol., 8 (1988), pp. 655–663</p></li><li class="external refPlaceHolder" id="bibsbref15"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB16"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB16" class="label intra_ref">16</a></li><li class="source">Harada,
 J.J. (1997). Seed maturation and control of germination. In Advances in
 Cellular and Molecular Biology of Plants, Volume <em>4</em>, Cellular 
and Molecular Biology of Seed Development, B.A. Larkins and I.K. Vasi, 
eds. (Dordrecht: Kluwer Academic Publishers), pp. 545–592.</li><li class="external refPlaceHolder" id="bibsbref16"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB17"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB17" class="label intra_ref">17</a></li><li class="author">P.F Johnson, SL McKnight</li><li class="title"><p>Eukaryotic transcriptional regulatory proteins</p></li><li class="source"><p>Annu. Rev. Biochem., 58 (1989), pp. 799–840</p></li><li class="external refPlaceHolder" id="bibsbref17"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB18"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB18" class="label intra_ref">18</a></li><li class="author">K Keith, M Kraml, N.G Dengler, P McCourt</li><li class="title"><p><em>fusca3</em>a heterochronic mutation affecting late embryo development in Arabidopsis</p></li><li class="source"><p>Plant Cell, 6 (1994), pp. 589–600</p></li><li class="external refPlaceHolder" id="bibsbref18"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB19"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB19" class="label intra_ref">19</a></li><li class="author">T Keng, L Guarente</li><li class="title"><p>Constitutive expression of the yeast <em>HEM1</em> gene is actually a composite of activation and repression</p></li><li class="source"><p>Proc. Natl. Acad. Sci. USA, 84 (1987), pp. 9113–9117</p></li><li class="external refPlaceHolder" id="bibsbref19"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB20"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB20" class="label intra_ref">20</a></li><li class="author">M Koornneef, G Reuling, C.M Karssen</li><li class="title"><p>The isolation and characterization of abscisic acid-insensitive mutants of <em>Arabidopsis thaliana</em></p></li><li class="source"><p>Physiol. Plant, 61 (1984), pp. 377–383</p></li><li class="external refPlaceHolder" id="bibsbref20"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB21"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB21" class="label intra_ref">21</a></li><li class="author">X.-Y Li, R Mantvani, R Hooft Van Huijsduijnen, I Andre, C Benoist, D Mathis</li><li class="title"><p>Evolutionary variation of the CCAAT-binding transcription factor NF-Y</p></li><li class="source"><p>Nucleic Acids Res., 20 (1992), pp. 1087–1091</p></li><li class="external refPlaceHolder" id="bibsbref21"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB22"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB22" class="label intra_ref">22</a></li><li class="author">J.A Long, E.I Moan, J.I Medford, M.K Barton</li><li class="title"><p>A member of the KNOTTED class of homeodomain proteins encoded by the STM gene of Arabidopsis</p></li><li class="source"><p>Nature, 379 (1996), pp. 66–69</p></li><li class="external refPlaceHolder" id="bibsbref22"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB23"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB23" class="label intra_ref">23</a></li><li class="author">W Lukowitz, U Mayer, G Juergens</li><li class="title"><p>Cytokinesis in the <em>Arabidopsis</em> embryo involves the syntaxin-related KNOLLE gene product</p></li><li class="source"><p>Cell, 84 (1996), pp. 61–71</p></li><li class="external refPlaceHolder" id="bibsbref23"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB24"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB24" class="label intra_ref">24</a></li><li class="author">S.N Maity, S Sinha, E.C Ruteshouser, B De Crombrugghe</li><li class="title"><p>Three different polypeptides are necessary for DNA binding of the mammalian heteromeric CCAAT binding factor</p></li><li class="source"><p>J. Biol. Chem., 267 (1992), pp. 16574–16580</p></li><li class="external refPlaceHolder" id="bibsbref24"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB25"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB25" class="label intra_ref">25</a></li><li class="author">M.A Matzke, A.J.M Matzke</li><li class="title"><p>How and why do plants inactivate homologous (trans)genes?</p></li><li class="source"><p>Plant Physiol., 107 (1995), pp. 679–685</p></li><li class="external refPlaceHolder" id="bibsbref25"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB26"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB26" class="label intra_ref">26</a></li><li class="author">U Mayer, R.A.T Torres Ruiz, T Berleth, S Misera, G Jurgens</li><li class="title"><p>Mutations affecting body organization in the <em>Arabidopsis</em> embryo</p></li><li class="source"><p>Nature, 353 (1991), pp. 402–407</p></li><li class="external refPlaceHolder" id="bibsbref26"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB27"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB27" class="label intra_ref">27</a></li><li class="author">D.S McNabb, Y Xing, L Guarente</li><li class="title"><p>Cloning of yeast HAP5a novel subunit of a heterotrimeric complex required for CCAAT binding</p></li><li class="source"><p>Genes Dev., 9 (1994), pp. 47–58</p></li><li class="external refPlaceHolder" id="bibsbref27"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB28"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB28" class="label intra_ref">28</a></li><li class="author">D.W Meinke</li><li class="title"><p>A homoeotic mutant of <em>Arabidopsis thaliana</em> with leafy cotyledons</p></li><li class="source"><p>Science, 258 (1992), pp. 1647–1650</p></li><li class="external refPlaceHolder" id="bibsbref28"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB29"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB29" class="label intra_ref">29</a></li><li class="author">D.W Meinke, L.H Franzmann, T.C Nickle, E.C Yeung</li><li class="title"><p><em>leafy cotyledon</em> mutants of Arabidopsis</p></li><li class="source"><p>Plant Cell, 6 (1994), pp. 1049–1064</p></li><li class="external refPlaceHolder" id="bibsbref29"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB30"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB30" class="label intra_ref">30</a></li><li class="author">W Mulder, I.H Scholten, R.W de Boer, L.A Grivell</li><li class="title"><p>Sequence of the HAP3 transcription factor of <em>Kluyveromyces lactis</em> predicts the presence of a novel 4-cysteine zinc-finger motif</p></li><li class="source"><p>Mol. Gen. Genet., 245 (1994), pp. 96–106</p></li><li class="external refPlaceHolder" id="bibsbref30"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB31"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB31" class="label intra_ref">31</a></li><li class="author">R.M Myers, K Tilly, T Maniatis</li><li class="title"><p>Fine structure genetic analysis of a beta-globin promoter</p></li><li class="source"><p>Science, 232 (1986), pp. 613–618</p></li><li class="external refPlaceHolder" id="bibsbref31"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB32"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB32" class="label intra_ref">32</a></li><li class="author">E Nambara, S Naito, P McCourt</li><li class="title"><p>A mutant of Arabidopsis which is defective in seed development and storage protein accumulation is a new <em>abi3</em> allele</p></li><li class="source"><p>Plant J., 2 (1992), pp. 435–441</p></li><li class="external refPlaceHolder" id="bibsbref32"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB33"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB33" class="label intra_ref">33</a></li><li class="author">T Newman, F.J De Bruijn, P Green, K Keegstra, H Kende, L McIntosh, J Ohlrogge, N Raikhel, S Somerville, M Thomashow <em>et al.</em></li><li class="title"><p>Genes galorea summary of methods for accessing results from large-scale partial sequencing of anonymous Arabidopsis cDNA clones</p></li><li class="source"><p>Plant Physiol., 106 (1994), pp. 1241–1255</p></li><li class="external refPlaceHolder" id="bibsbref33"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB34"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB34" class="label intra_ref">34</a></li><li class="author">J.T Odell, F Nagy, N.-H Chua</li><li class="title"><p>Identification of DNA sequences required for activity of cauliflower mosaic virus 35S promoter</p></li><li class="source"><p>Nature, 313 (1985), pp. 810–812</p></li><li class="external refPlaceHolder" id="bibsbref34"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB35"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB35" class="label intra_ref">35</a></li><li class="author">J Ogas, J.-C Cheng, Z.R Sung, C Somerville</li><li class="title"><p>Cellular differentiation regulated by gibberellin in the <em>Arabidopsis thaliana pickle</em> mutant</p></li><li class="source"><p>Science, 277 (1997), pp. 91–94</p></li><li class="external refPlaceHolder" id="bibsbref35"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB36"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB36" class="label intra_ref">36</a></li><li class="author">L.J Olsen, W.F Ettinger, B Damsz, K.L Matsudaira, M.A Webb, J.J Harada</li><li class="title"><p>Targeting of glyoxysomal proteins to peroxisomes in leaves and roots of a higher plant</p></li><li class="source"><p>Plant Cell, 5 (1993), pp. 941–952</p></li><li class="external refPlaceHolder" id="bibsbref36"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB37"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB37" class="label intra_ref">37</a></li><li class="author">P Papagiannopoulos, A Andrianopoulos, J.A Sharp, M.A Davis, M.J Hynes</li><li class="title"><p>The hapC gene of Aspergillus nidulans is involved in the expression of CCAAT-containing promoters</p></li><li class="source"><p>Mol. Gen. Genet., 251 (1996), pp. 412–421</p></li><li class="external refPlaceHolder" id="bibsbref37"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB38"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB38" class="label intra_ref">38</a></li><li class="author">F Parcy, C Valon, M Raynal, P Gaubier-Comella, M Delseny, J Giraudat</li><li class="title"><p>Regulation of gene expression programs during Arabidopsis seed developmentroles of the <em>ABI3</em> locus and of endogenous abscisic acid</p></li><li class="source"><p>Plant Cell, 6 (1994), pp. 1567–1582</p></li><li class="external refPlaceHolder" id="bibsbref38"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB39"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB39" class="label intra_ref">39</a></li><li class="author">F Parcy, C Valon, A Kohara, S Misera, J Giraudat</li><li class="title"><p>The <em>ABSCISIC ACID-INSENSITIVE3</em>, <em>FUSCA3</em>, and <em>LEAFY COTYLEDON1</em> loci act in concert to control multiple aspects of Arabidopsis seed development</p></li><li class="source"><p>Plant Cell, 9 (1997), pp. 1265–1277</p></li><li class="external refPlaceHolder" id="bibsbref39"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB40"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB40" class="label intra_ref">40</a></li><li class="author">J.C Schneider, L Guarente</li><li class="title"><p>Regulation of the yeast <em>CYT1</em> gene encoding cytochrome c<sub>1</sub> by HAP1 and HAP2/3/4</p></li><li class="source"><p>Mol. Cell Biol., 11 (1991), pp. 4934–4942</p></li><li class="external refPlaceHolder" id="bibsbref40"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB41"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB41" class="label intra_ref">41</a></li><li class="author">B.W Schwartz, E.C Yeung, D.W Meinke</li><li class="title"><p>Disruption of morphogenesis and transformation of the suspensor in abnormal suspensor mutants of <em>Arabidopsis</em></p></li><li class="source"><p>Development, 120 (1994), pp. 3235–3245</p></li><li class="external refPlaceHolder" id="bibsbref41"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB42"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB42" class="label intra_ref">42</a></li><li class="author">D.E Shevell, W.-M Leu, C.S Gillmor, G Xia, K.A Feldmann, N.-H Chua</li><li class="title"><p><em>EMB30</em>
 is essential for normal cell division, cell expansion, and cell 
adhesion in arabidopsis and encodes a protein that has similarity to 
Sec7</p></li><li class="source"><p>Cell, 77 (1994), pp. 1051–1062</p></li><li class="external refPlaceHolder" id="bibsbref42"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB43"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB43" class="label intra_ref">43</a></li><li class="author">S Sinha, S.N Maity, J Lu, B de Crombrugghe</li><li class="title"><p>Recombinant
 rat CBF-C, the third subunit of CBF/NFY, allows formation of a 
protein-DNA complex with CBF-A and CBF-B and with yeast HAP2 and HAP3</p></li><li class="source"><p>Proc. Natl. Acad. Sci. USA, 92 (1995), pp. 1624–1628</p></li><li class="external refPlaceHolder" id="bibsbref43"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB44"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB44" class="label intra_ref">44</a></li><li class="author">S Sinha, I.S Kim, K.-Y Sohn, B de Crombrugghe, S.N Maity</li><li class="title"><p>Three
 classes of mutations in the A subunit of the CCAAT-binding factor CBF 
delineate functional domains involved in the three-step assembly of the 
CBF-DNA complex</p></li><li class="source"><p>Mol. Cell. Biol., 16 (1996), pp. 328–337</p></li><li class="external refPlaceHolder" id="bibsbref44"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB45"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB45" class="label intra_ref">45</a></li><li class="author">M Stam, J.N.M Mol, J.M Kooster</li><li class="title"><p>The silence of genes in transgenic plants</p></li><li class="source"><p>Ann. Bot., 79 (1997), pp. 3–12</p></li><li class="external refPlaceHolder" id="bibsbref45"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB46"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB46" class="label intra_ref">46</a></li><li class="author">T.A Steeves</li><li class="title"><p>The evolution and biological significance of seeds</p></li><li class="source"><p>Can. J. Bot., 61 (1983), pp. 3550–3560</p></li><li class="external refPlaceHolder" id="bibsbref46"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB47"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB47" class="label intra_ref">47</a></li><li class="author">T.A Steeves, I.M Sussex</li><li class="source"><p>Patterns in Plant Development, Second EditionCambridge University Press, Cambridge (1989)</p></li><li class="external refPlaceHolder" id="bibsbref47"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB48"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB48" class="label intra_ref">48</a></li><li class="author">C.E Trueblood, R.M Wright, R.O Poyton</li><li class="title"><p>Differential regulation of the two genes encoding <em>Saccharomyces cervisiae</em> cytochrome oxidase subunit V by heme and <em>HAP2</em> and <em>REO1</em> genes</p></li><li class="source"><p>Mol. Cell Biol., 8 (1988), pp. 4537–4740</p></li><li class="external refPlaceHolder" id="bibsbref48"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB49"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB49" class="label intra_ref">49</a></li><li class="author">R Tsugeki, E.Z Kochieva, N.V Fedoroff</li><li class="title"><p>A transposon insertion in the Arabidopsis SSR16 gene causes an embryo-defective lethal mutation</p></li><li class="source"><p>Plant J., 10 (1996), pp. 479–489</p></li><li class="external refPlaceHolder" id="bibsbref49"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB50"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB50" class="label intra_ref">50</a></li><li class="author">T Vuorio, S.N Maity, B de Crombrugghe</li><li class="title"><p>Purification and molecular cloning of the “A” chain of a rat</p></li><li class="source"><p>J. Biol. Chem., 265 (1990), pp. 22480–22486</p></li><li class="external refPlaceHolder" id="bibsbref50"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB51"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB51" class="label intra_ref">51</a></li><li class="source">Walbot,
 V. (1978). Control mechanisms for plant embryogeny. In Dormancy and 
Developmental Arrest, M.E. Clutter, ed. (New York: Academic Press, 
Inc.), pp. 113–166.</li><li class="external refPlaceHolder" id="bibsbref51"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB52"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB52" class="label intra_ref">52</a></li><li class="author">M.A West, J.J Harada</li><li class="title"><p>Embryogenesis in higher plantsan overview</p></li><li class="source"><p>Plant Cell, 5 (1993), pp. 1361–1369</p></li><li class="external refPlaceHolder" id="bibsbref52"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB53"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB53" class="label intra_ref">53</a></li><li class="author">M.A.L West, K.L Matsudaira Yee, J Danao, J.L Zimmerman, R.L Fischer, R.B Goldberg, J.J Harada</li><li class="title"><p><em>LEAFY COTYLEDON1</em> is an essential regulator of late embryogenesis and cotyledon identity in Arabidopsis</p></li><li class="source"><p>Plant Cell, 6 (1994), pp. 1731–1745</p></li><li class="external refPlaceHolder" id="bibsbref53"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB54"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB54" class="label intra_ref">54</a></li><li class="author">K.L Wright, B.J Vilen, Y Itoh-Lindstrom, T.L Moore, G Li, M Criscitiello, P Cogswell, J.B Clarke, J.P.-Y Ting</li><li class="title"><p>CCAAT box binding protein NF-Y facilitates in vivo recruitment of upstream DNA binding transcription factors</p></li><li class="source"><p>EMBO J., 13 (1994), pp. 4042–4053</p></li><li class="external refPlaceHolder" id="bibsbref54"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB55"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB55" class="label intra_ref">55</a></li><li class="author">Y Xing, J.D Fikes, L Guarente</li><li class="title"><p>Mutations in yeast HAP2/HAP3 define a hybrid CCAAT box binding domain</p></li><li class="source"><p>EMBO J., 12 (1993), pp. 4647–4655</p></li><li class="external refPlaceHolder" id="bibsbref55"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li><li id="BIB56"><ul class="reference" id=""><li class="referenceLabel"><a href="#bBIB56" class="label intra_ref">56</a></li><li class="author">R Yadegari, G.R De Paiva, T Laux, A.M Koltunow, N Apuya, J.L Zimmerman, R.L Fischer, J.J Harada, R.B Goldberg</li><li class="title"><p>Cell differentiation and morphogenesis are uncoupled in Arabidopsis <em>raspberry</em> embryos</p></li><li class="source"><p>Plant Cell, 6 (1994), pp. 1713–1729</p></li><li class="external refPlaceHolder" id="bibsbref56"><img class="refimgLoader" alt="" src="S0092867400814634_files/loading_icon.gif"/></li></ul></li></ol></div><div class="refText svRefs"><h3 id="N2091fa08N2cf7e078">Further reading</h3><p id="">The accession number for the sequence reported in this paper is <span class="interref" data-locatortype="genbank"><a href="http://www.sciencedirect.com/science?_ob=RedirectURL&amp;_method=externObjLink&amp;_locator=genbank&amp;_issn=00928674&amp;_origin=article&amp;_zone=art_page&amp;_plusSign=%2B&amp;_targetURL=http%253A%252F%252Fwww.ncbi.nlm.nih.gov%252Fentrez%252Fquery.fcgi%253Fcmd%253Dsearch%2526db%253Dnucleotide%2526doptcmdl%253Dgenbank%2526term%253DAF036684[accn]" target="externObjLink" onclick="var newWidth=((window.screen.availWidth*90)/100);var newHeight=((window.screen.availHeight*90)/100);var parms = 'status=yes,location=yes,' + 'scrollbars=yes,resizable=yes,directories=yes,' + 'toolbar=yes,menubar=yes,' + 'width=' + newWidth + ',height=' + newHeight + ',screenX=10,screenY=10';var externalWin; externalWin=window.open('','externObjLink',parms); externalWin.focus()">AF036684</a></span>.</p></div><div class="artFooterContent"><dl class="footnote" id="COR1"><dt class="label topPadd"><a href="#bCOR1" class="intra_ref"><sup><img src="S0092867400814634_files/REcor.gif" alt="Corresponding author contact information" title="Corresponding author contact information" border="0"/></sup></a></dt><dd>Correspondence: John J. Harada, 916 752 0673 (phone), 916 752 5410 (fax)</dd></dl><dl class="footnote" id="FN1"><dt class="label"><a href="#bFN1" class="intra_ref">1</a></dt><dd><p>To whom correspondence should be addressed.</p></dd></dl><dl class="footnote" id="FN2"><dt class="label"><a href="#bFN2" class="intra_ref">2</a></dt><dd><p>Present address: Department of Developmental Biology, National Institute for Basic Biology, Myodaijicho, Okazaki, 444 Japan.</p></dd></dl><dl class="footnote" id="FN3"><dt class="label"><a href="#bFN3" class="intra_ref">3</a></dt><dd><p>Present address: Department of Vegetable Crops, University of California, Davis, California 95616.</p></dd></dl><dl class="footnote" id="FN4"><dt class="label"><a href="#bFN4" class="intra_ref">4</a></dt><dd><p>Present address: Department of Genetics, University of Washington, Seattle, Washington 98195.</p></dd></dl><!--footerNotes--></div><p class="copyright">Copyright © 1998 Cell Press.  All rights reserved.</p></div>
      <script>prs.rt('data_end');</script></div>
  </body>
</html>
